{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "a8b92d8b",
   "metadata": {},
   "source": [
    "# 4. Spatio-temporal diversity of must "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "abc52439",
   "metadata": {},
   "outputs": [],
   "source": [
    "#!mkdir /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "cd670efc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries\n"
     ]
    }
   ],
   "source": [
    "workdir = '/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries'\n",
    "%cd $workdir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "413c885b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "env: TMPDIR=/scratch/lfloerl/tmpdata\n"
     ]
    }
   ],
   "source": [
    "%env TMPDIR=/scratch/lfloerl/tmpdata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "18e20f86",
   "metadata": {},
   "outputs": [],
   "source": [
    "import qiime2 as q2\n",
    "from qiime2 import Visualization\n",
    "from qiime2 import Artifact\n",
    "from qiime2 import Metadata\n",
    "from qiime2.plugins.diversity.visualizers import alpha_group_significance\n",
    "from qiime2.plugins import diversity as q2d\n",
    "\n",
    "\n",
    "import os\n",
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "import numpy as np\n",
    "from scipy.stats import spearmanr\n",
    "\n",
    "\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "eff46f25",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "md = Metadata.load('/home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv')\n",
    "taxonomy = q2.Artifact.load('/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/taxonomy.qza')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2914f57c",
   "metadata": {},
   "source": [
    "# Diversity "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "57999d3b",
   "metadata": {},
   "outputs": [],
   "source": [
    "table = q2.Artifact.load('/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/lavaux/must_filtered_table.qza')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "adc33f33",
   "metadata": {},
   "source": [
    "* 10'000: Retained 3’400’000 (26.33%) features in 340 (95.51%) samples\n",
    "\n",
    "* **15'000**: Retained 4’845’000 (37.51%) features in 323 (90.73%) samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "ad617760",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/2589a44c-545a-4201-8668-9cbc454920bd')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 2589a44c-545a-4201-8668-9cbc454920bd>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load('/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/lavaux/must_filtered_table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "e1175bf0",
   "metadata": {},
   "outputs": [],
   "source": [
    "depth = 10000 \n",
    "threads = 5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "d91c7235",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/scratch/lfloerl/.condaenvs/qiime2-amplicon-2024.10/lib/python3.10/site-packages/skbio/util/_warning.py:60: UserWarning: observed_otus is deprecated as of 0.6.0.\n",
      "  warn(f\"{func.__name__} is deprecated as of {ver}.\")\n",
      "/scratch/lfloerl/.condaenvs/qiime2-amplicon-2024.10/lib/python3.10/site-packages/skbio/stats/ordination/_principal_coordinate_analysis.py:146: RuntimeWarning: The result contains negative eigenvalues. Please compare their magnitude with the magnitude of some of the largest positive eigenvalues. If the negative ones are smaller, it's probably safe to ignore them, but if they are large in magnitude, the results won't be useful. See the Notes section for more details. The smallest eigenvalue is -0.44173564283507494 and the largest is 10.03622517708246.\n",
      "  warn(\n"
     ]
    }
   ],
   "source": [
    "core_div_dict = {}\n",
    "core_div = q2d.actions.core_metrics(table=table,\n",
    "                                    sampling_depth=depth,\n",
    "                                    metadata=md, \n",
    "                                    n_jobs=threads)\n",
    "# save outputs\n",
    "output_dir='must_cm{0}/'.format(depth)\n",
    "if not os.path.exists(output_dir):\n",
    "    os.makedirs(output_dir)\n",
    "for k, v in core_div.__dict__.items():\n",
    "    if k != '_fields':\n",
    "        v.save(output_dir + k)\n",
    "        core_div_dict[k] = v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "010456c4",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/6582572f-1d70-4e52-8e27-af16b2ec68e4')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 6582572f-1d70-4e52-8e27-af16b2ec68e4>"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{workdir}/{output_dir}/bray_curtis_emperor.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "1aff2c07",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/626b3668-f09d-4923-b481-1724038f2530')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 626b3668-f09d-4923-b481-1724038f2530>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{workdir}/{output_dir}/jaccard_emperor.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "27e37ac4",
   "metadata": {},
   "source": [
    "## Kmerizer "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "449282b2",
   "metadata": {},
   "outputs": [],
   "source": [
    "#!rm -r kmer-core-metrics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "7c480b75",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureData[Sequence] to: /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/must_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: must_kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: must_kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: must_kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: must_kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: must_kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: must_kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: must_kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: must_kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: must_kmer_cm10000/scatterplot.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "qiime feature-table filter-seqs \\\n",
    "    --i-data /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/Sven-denoising-test/ITS/revcomp-trimmed-denoised/with-adjusted-parameters/ITS-revcomp-trimmed-ER-0.05-denoised-trunc-190-PFA-4.0-maxee-4.0-rep-seqs.qza \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/must_filtered_table.qza \\\n",
    "    --o-filtered-data /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/must_filtered_rep_seqs.qza\n",
    "\n",
    "qiime kmerizer core-metrics \\\n",
    "    --i-sequences /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/must_filtered_rep_seqs.qza \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/must_filtered_table.qza \\\n",
    "    --p-sampling-depth 10000 \\\n",
    "    --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv \\\n",
    "    --p-n-jobs 'auto' \\\n",
    "    --output-dir must_kmer_cm10000/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "e90b9c36",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/a499e910-f677-4ef7-baa7-dc0820ae97e4')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: a499e910-f677-4ef7-baa7-dc0820ae97e4>"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load('must_kmer_cm10000/scatterplot.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fb6c27ed",
   "metadata": {},
   "source": [
    "### Alpha Group Significance "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "dd8cf4d5",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/scratch/lfloerl/.condaenvs/qiime2-amplicon-2024.10/lib/python3.10/site-packages/q2_types/sample_data/_deferred_setup/_transformers.py:28: FutureWarning: errors='ignore' is deprecated and will raise in a future version. Use to_numeric without passing `errors` and catch exceptions explicitly instead\n",
      "  df[cols] = df[cols].apply(pd.to_numeric, errors='ignore')\n",
      "/scratch/lfloerl/.condaenvs/qiime2-amplicon-2024.10/lib/python3.10/site-packages/q2_types/sample_data/_deferred_setup/_transformers.py:28: FutureWarning: errors='ignore' is deprecated and will raise in a future version. Use to_numeric without passing `errors` and catch exceptions explicitly instead\n",
      "  df[cols] = df[cols].apply(pd.to_numeric, errors='ignore')\n",
      "/scratch/lfloerl/.condaenvs/qiime2-amplicon-2024.10/lib/python3.10/site-packages/q2_types/sample_data/_deferred_setup/_transformers.py:28: FutureWarning: errors='ignore' is deprecated and will raise in a future version. Use to_numeric without passing `errors` and catch exceptions explicitly instead\n",
      "  df[cols] = df[cols].apply(pd.to_numeric, errors='ignore')\n"
     ]
    }
   ],
   "source": [
    "# metrics: \n",
    "alpha_metrics = ['shannon_vector', 'observed_features_vector', 'evenness_vector']\n",
    "\n",
    "for m in alpha_metrics:\n",
    "    ags, = alpha_group_significance(alpha_diversity = core_div_dict[m],\n",
    "                               metadata = md)\n",
    "    ags.save(output_dir + 'alpha_groupSig_{0}.qzv'.format(m))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "fd079ab6",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/4a90a9e7-e822-4199-86bc-667575f25597')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 4a90a9e7-e822-4199-86bc-667575f25597>"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{output_dir}/alpha_groupSig_evenness_vector.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "a475d39b",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/e9b699af-3089-41a9-9c46-a831182757b7')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: e9b699af-3089-41a9-9c46-a831182757b7>"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{output_dir}/alpha_groupSig_observed_features_vector.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "b9469709",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/c45b7be7-6a7d-4c9e-8499-964acfb01e51')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: c45b7be7-6a7d-4c9e-8499-964acfb01e51>"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{output_dir}/alpha_groupSig_shannon_vector.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c0c19d61",
   "metadata": {},
   "source": [
    "### PERMANOVA"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "1239daba",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The command being run is below. This command cannot be manually re-run as it will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: run_adonis.R /scratch/lfloerl/tmpdata/tmpushk_8lf/dm.tsv /scratch/lfloerl/tmpdata/tmpushk_8lf/md.tsv Plot_ID + Year + Time_point + Plot_ID:Year + Plot_ID:Time_point + Year:Time_point 999 3 /scratch/lfloerl/tmpdata/qiime2-temp-91a91_no/adonis.tsv\n",
      "\n",
      "R version 4.3.3 (2024-02-29) \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Lade nötiges Paket: permute\n",
      "Lade nötiges Paket: lattice\n",
      "This is vegan 2.6-8\n",
      "Warnmeldung:\n",
      "'adonis' ist veraltet.\n",
      "Benutzen Sie stattdessen 'adonis2'\n",
      "Siehe help(\"Deprecated\") und help(\"vegan-deprecated\"). \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The command being run is below. This command cannot be manually re-run as it will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: run_adonis.R /scratch/lfloerl/tmpdata/tmptt_i7mhx/dm.tsv /scratch/lfloerl/tmpdata/tmptt_i7mhx/md.tsv Plot_ID + Year + Time_point + Plot_ID:Year + Plot_ID:Time_point + Year:Time_point 999 3 /scratch/lfloerl/tmpdata/qiime2-temp-m_5kpgkp/adonis.tsv\n",
      "\n",
      "R version 4.3.3 (2024-02-29) \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Lade nötiges Paket: permute\n",
      "Lade nötiges Paket: lattice\n",
      "This is vegan 2.6-8\n",
      "Warnmeldung:\n",
      "'adonis' ist veraltet.\n",
      "Benutzen Sie stattdessen 'adonis2'\n",
      "Siehe help(\"Deprecated\") und help(\"vegan-deprecated\"). \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: must_kmer_cm10000/adonis_bray_curtis.qzv\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved Visualization to: must_kmer_cm10000/adonis_jaccard.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "formula = 'Plot_ID + Year + Time_point + Plot_ID:Year + Plot_ID:Time_point + Year:Time_point'\n",
    "\n",
    "beta_metrics = ['bray_curtis', 'jaccard']\n",
    "\n",
    "# PERMANOVA with core metrics \n",
    "for m in beta_metrics:\n",
    "    adonis_res, = q2d.actions.adonis(\n",
    "        distance_matrix=core_div_dict[m + '_distance_matrix'],\n",
    "        metadata=md,\n",
    "        formula=formula,\n",
    "        n_jobs=3)\n",
    "    adonis_res.save(output_dir + 'adonis_{0}.qzv'.format(m))\n",
    "    \n",
    "\n",
    "# PERMANOVA with kmer core metrics \n",
    "kmer_dir = 'must_kmer_cm10000'\n",
    "    \n",
    "for m in beta_metrics:\n",
    "    !qiime diversity adonis \\\n",
    "        --i-distance-matrix \"{kmer_dir}/{m}_distance_matrix.qza\" \\\n",
    "        --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv \\\n",
    "        --p-formula \"{formula}\" \\\n",
    "        --p-n-jobs 5 \\\n",
    "        --o-visualization \"{kmer_dir}/adonis_{m}.qzv\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "03924c1f",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/65a06767-3d5c-476f-8c7d-60f659f4023b')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 65a06767-3d5c-476f-8c7d-60f659f4023b>"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{output_dir}/adonis_bray_curtis.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "56e82da0",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/5ac933b0-01ef-49f7-ae2c-82427716336e')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 5ac933b0-01ef-49f7-ae2c-82427716336e>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{output_dir}/adonis_jaccard.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "6e32f2ce",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/ac878d2b-a929-4f78-a400-e7b066baf352')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: ac878d2b-a929-4f78-a400-e7b066baf352>"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{kmer_dir}/adonis_bray_curtis.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "46ea1f80",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/a21a0e04-5071-47a1-801d-0c63938ae5cb')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: a21a0e04-5071-47a1-801d-0c63938ae5cb>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{kmer_dir}/adonis_jaccard.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ab557f5c",
   "metadata": {},
   "source": [
    "## Mantel "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "7d2da626",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exported must_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory must_cm10000/bray_curtis_distance_matrix\n",
      "Exported must_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory must_cm10000/jaccard_distance_matrix\n",
      "Exported must_kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory must_kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported must_kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory must_kmer_cm10000/jaccard_distance_matrix\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "div_dir='must_cm10000'\n",
    "kmer_dir='must_kmer_cm10000'\n",
    "\n",
    "# geodisic distance\n",
    "#qiime tools export --input-path /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/geodesic_distance_ITS.qza --output-path geodesic_distance_ITS\n",
    "\n",
    "# distance matrices \n",
    "qiime tools export --input-path $div_dir/bray_curtis_distance_matrix.qza --output-path $div_dir/bray_curtis_distance_matrix\n",
    "qiime tools export --input-path $div_dir/jaccard_distance_matrix.qza --output-path $div_dir/jaccard_distance_matrix\n",
    "qiime tools export --input-path $kmer_dir/bray_curtis_distance_matrix.qza --output-path $kmer_dir/bray_curtis_distance_matrix\n",
    "qiime tools export --input-path $kmer_dir/jaccard_distance_matrix.qza --output-path $kmer_dir/jaccard_distance_matrix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "617c24dc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                           Test Mantel_statistic_r Significance\n",
      "1      must_cm10000-bray_curtis_distance_matrix         0.19169210        0.001\n",
      "2          must_cm10000-jaccard_distance_matrix         0.15309950        0.001\n",
      "3 must_kmer_cm10000-bray_curtis_distance_matrix         0.19026594        0.001\n",
      "4     must_kmer_cm10000-jaccard_distance_matrix         0.05474539        0.020\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "geodesic_path='/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/spatio-temporal/geodesic_distance_ITS/distance-matrix.tsv'\n",
    "\n",
    "folder_path_1=\"must_cm10000\"\n",
    "folder_path_2=\"must_kmer_cm10000\"\n",
    "\n",
    "# Call the R script and pass folder paths and geodesic matrix\n",
    "Rscript /home/lfloerl/microterroir/Microbiome/Other_scripts/mantel_test.R \"$folder_path_1\" \"$folder_path_2\" \"$geodesic_path\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "e89cd9c4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Test</th>\n",
       "      <th>Mantel_statistic_r</th>\n",
       "      <th>Significance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>must_cm10000-bray_curtis_distance_matrix</td>\n",
       "      <td>0.191692</td>\n",
       "      <td>0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>must_cm10000-jaccard_distance_matrix</td>\n",
       "      <td>0.153099</td>\n",
       "      <td>0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>must_kmer_cm10000-bray_curtis_distance_matrix</td>\n",
       "      <td>0.190266</td>\n",
       "      <td>0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>must_kmer_cm10000-jaccard_distance_matrix</td>\n",
       "      <td>0.054745</td>\n",
       "      <td>0.020</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                            Test  Mantel_statistic_r  \\\n",
       "0       must_cm10000-bray_curtis_distance_matrix            0.191692   \n",
       "1           must_cm10000-jaccard_distance_matrix            0.153099   \n",
       "2  must_kmer_cm10000-bray_curtis_distance_matrix            0.190266   \n",
       "3      must_kmer_cm10000-jaccard_distance_matrix            0.054745   \n",
       "\n",
       "   Significance  \n",
       "0         0.001  \n",
       "1         0.001  \n",
       "2         0.001  \n",
       "3         0.020  "
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mantel = pd.read_csv('must_kmer_cm10000/mantel_results.tsv', sep='\\t')\n",
    "mantel"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "24395fac",
   "metadata": {},
   "source": [
    "<hr>\n",
    "\n",
    "# SUBSET TO SINGLE YEAR \n",
    "\n",
    "# Diversity "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "5ecef7c3",
   "metadata": {},
   "outputs": [],
   "source": [
    "#!mkdir /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries/2021"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "519ca429",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries/2021\n"
     ]
    }
   ],
   "source": [
    "workdir = '/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries/2021'\n",
    "%cd $workdir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "33561db1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: must_21_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_21_filtered_rep_seqs.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# subset to only harvest samples before fitting \n",
    "qiime feature-table filter-samples \\\n",
    "  --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/lavaux/must_filtered_table.qza \\\n",
    "  --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv \\\n",
    "  --p-where \"[Collection_ID]='Harvest_2021'\" \\\n",
    "  --o-filtered-table must_21_filtered_table.qza\n",
    "\n",
    "qiime feature-table filter-seqs \\\n",
    "    --i-data /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/Sven-denoising-test/ITS/revcomp-trimmed-denoised/with-adjusted-parameters/ITS-revcomp-trimmed-ER-0.05-denoised-trunc-190-PFA-4.0-maxee-4.0-rep-seqs.qza \\\n",
    "    --i-table must_21_filtered_table.qza \\\n",
    "    --o-filtered-data must_21_filtered_rep_seqs.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "5eb27202",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/scratch/lfloerl/.condaenvs/qiime2-amplicon-2024.10/lib/python3.10/site-packages/skbio/stats/ordination/_principal_coordinate_analysis.py:146: RuntimeWarning: The result contains negative eigenvalues. Please compare their magnitude with the magnitude of some of the largest positive eigenvalues. If the negative ones are smaller, it's probably safe to ignore them, but if they are large in magnitude, the results won't be useful. See the Notes section for more details. The smallest eigenvalue is -0.17725876156831202 and the largest is 6.991737027796639.\n",
      "  warn(\n"
     ]
    }
   ],
   "source": [
    "table = q2.Artifact.load('must_21_filtered_table.qza')\n",
    "\n",
    "core_div_dict = {}\n",
    "core_div = q2d.actions.core_metrics(table=table,\n",
    "                                    sampling_depth=depth,\n",
    "                                    metadata=md, \n",
    "                                    n_jobs=threads)\n",
    "# save outputs\n",
    "output_dir='must_21_cm{0}/'.format(depth)\n",
    "if not os.path.exists(output_dir):\n",
    "    os.makedirs(output_dir)\n",
    "for k, v in core_div.__dict__.items():\n",
    "    if k != '_fields':\n",
    "        v.save(output_dir + k)\n",
    "        core_div_dict[k] = v"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dc23940d",
   "metadata": {},
   "source": [
    "## Kmerizer "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "9ac1a97a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: must_21_kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: must_21_kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: must_21_kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: must_21_kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: must_21_kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: must_21_kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: must_21_kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: must_21_kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: must_21_kmer_cm10000/scatterplot.qzv\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "qiime kmerizer core-metrics \\\n",
    "    --i-sequences must_21_filtered_rep_seqs.qza \\\n",
    "    --i-table must_21_filtered_table.qza \\\n",
    "    --p-sampling-depth 10000 \\\n",
    "    --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv \\\n",
    "    --p-n-jobs 'auto' \\\n",
    "    --output-dir must_21_kmer_cm10000/"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1b5b1b57",
   "metadata": {},
   "source": [
    "## Mantel "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "48c99dba",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Exported must_21_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory must_21_cm10000/bray_curtis_distance_matrix\n",
      "Exported must_21_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory must_21_cm10000/jaccard_distance_matrix\n",
      "Exported must_21_kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory must_21_kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported must_21_kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory must_21_kmer_cm10000/jaccard_distance_matrix\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "div_dir='must_21_cm10000'\n",
    "kmer_dir='must_21_kmer_cm10000'\n",
    "\n",
    "# geodisic distance\n",
    "#qiime tools export --input-path /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/geodesic_distance_ITS.qza --output-path geodesic_distance_ITS\n",
    "\n",
    "# distance matrices \n",
    "qiime tools export --input-path $div_dir/bray_curtis_distance_matrix.qza --output-path $div_dir/bray_curtis_distance_matrix\n",
    "qiime tools export --input-path $div_dir/jaccard_distance_matrix.qza --output-path $div_dir/jaccard_distance_matrix\n",
    "qiime tools export --input-path $kmer_dir/bray_curtis_distance_matrix.qza --output-path $kmer_dir/bray_curtis_distance_matrix\n",
    "qiime tools export --input-path $kmer_dir/jaccard_distance_matrix.qza --output-path $kmer_dir/jaccard_distance_matrix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "3fcab397",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                              Test Mantel_statistic_r\n",
      "1      must_21_cm10000-bray_curtis_distance_matrix         0.27458688\n",
      "2          must_21_cm10000-jaccard_distance_matrix         0.19530028\n",
      "3 must_21_kmer_cm10000-bray_curtis_distance_matrix         0.26730096\n",
      "4     must_21_kmer_cm10000-jaccard_distance_matrix         0.09606918\n",
      "  Significance\n",
      "1        0.001\n",
      "2        0.001\n",
      "3        0.001\n",
      "4        0.001\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "geodesic_path='/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/spatio-temporal/geodesic_distance_ITS/distance-matrix.tsv'\n",
    "\n",
    "folder_path_1=\"must_21_cm10000\"\n",
    "folder_path_2=\"must_21_kmer_cm10000\"\n",
    "\n",
    "# Call the R script and pass folder paths and geodesic matrix\n",
    "Rscript /home/lfloerl/microterroir/Microbiome/Other_scripts/mantel_test.R \"$folder_path_1\" \"$folder_path_2\" \"$geodesic_path\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "2750d294",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Test</th>\n",
       "      <th>Mantel_statistic_r</th>\n",
       "      <th>Significance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>must_21_cm10000-bray_curtis_distance_matrix</td>\n",
       "      <td>0.274587</td>\n",
       "      <td>0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>must_21_cm10000-jaccard_distance_matrix</td>\n",
       "      <td>0.195300</td>\n",
       "      <td>0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>must_21_kmer_cm10000-bray_curtis_distance_matrix</td>\n",
       "      <td>0.267301</td>\n",
       "      <td>0.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>must_21_kmer_cm10000-jaccard_distance_matrix</td>\n",
       "      <td>0.096069</td>\n",
       "      <td>0.001</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               Test  Mantel_statistic_r  \\\n",
       "0       must_21_cm10000-bray_curtis_distance_matrix            0.274587   \n",
       "1           must_21_cm10000-jaccard_distance_matrix            0.195300   \n",
       "2  must_21_kmer_cm10000-bray_curtis_distance_matrix            0.267301   \n",
       "3      must_21_kmer_cm10000-jaccard_distance_matrix            0.096069   \n",
       "\n",
       "   Significance  \n",
       "0         0.001  \n",
       "1         0.001  \n",
       "2         0.001  \n",
       "3         0.001  "
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mantel = pd.read_csv('must_21_kmer_cm10000/mantel_results.tsv', sep='\\t')\n",
    "mantel"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dabd1af7",
   "metadata": {},
   "source": [
    "<hr>\n",
    "\n",
    "# DENSE SAMPLING: SUBSET TO SINGLE PLOT IN HARVEST 2021  \n",
    "\n",
    "-> calculate the Mantel test for every single densely sampled plot "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "338e4994",
   "metadata": {},
   "outputs": [],
   "source": [
    "#!mkdir /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries/berries_2021_dense_sampling"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "09813e2b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries/berries_2021_dense_sampling\n"
     ]
    }
   ],
   "source": [
    "workdir = '/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/berries/berries_2021_dense_sampling'\n",
    "%cd $workdir"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "705e8afd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing Lavaux_Plot_4\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_4_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_4_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_5\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_5_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_5_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_9\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_9_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_9_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_11\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_11_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_11_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_12\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_12_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_12_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_13\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_13_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_13_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_14\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_14_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_14_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_15\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_15_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_15_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_18\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_18_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_18_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_19\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_19_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_19_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_20\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_20_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_20_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n",
      "Processing Lavaux_Plot_17\n",
      "Saved FeatureTable[Frequency] to: must_harvest21_Lavaux_Plot_17_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: must_harvest21_Lavaux_Plot_17_filtered_rep_seqs.qza\n",
      "Saved FeatureTable[Frequency] to: cm10000/rarefied_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/shannon_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: cm10000/evenness_vector.qza\n",
      "Saved DistanceMatrix to: cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: cm10000/jaccard_emperor.qzv\n",
      "Saved Visualization to: cm10000/bray_curtis_emperor.qzv\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/rarefied_table.qza\n",
      "Saved FeatureTable[Frequency] to: kmer_cm10000/kmer_table.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/observed_features_vector.qza\n",
      "Saved SampleData[AlphaDiversity] to: kmer_cm10000/shannon_vector.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/jaccard_distance_matrix.qza\n",
      "Saved DistanceMatrix to: kmer_cm10000/bray_curtis_distance_matrix.qza\n",
      "Saved PCoAResults to: kmer_cm10000/jaccard_pcoa_results.qza\n",
      "Saved PCoAResults to: kmer_cm10000/bray_curtis_pcoa_results.qza\n",
      "Saved Visualization to: kmer_cm10000/scatterplot.qzv\n",
      "Exported cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/bray_curtis_distance_matrix\n",
      "Exported cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory cm10000/jaccard_distance_matrix\n",
      "Exported kmer_cm10000/bray_curtis_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/bray_curtis_distance_matrix\n",
      "Exported kmer_cm10000/jaccard_distance_matrix.qza as DistanceMatrixDirectoryFormat to directory kmer_cm10000/jaccard_distance_matrix\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# Define the list of plots\n",
    "plots=('Lavaux_Plot_4' 'Lavaux_Plot_5' 'Lavaux_Plot_9' \n",
    "       'Lavaux_Plot_11' 'Lavaux_Plot_12' 'Lavaux_Plot_13' \n",
    "       'Lavaux_Plot_14' 'Lavaux_Plot_15' 'Lavaux_Plot_18' \n",
    "       'Lavaux_Plot_19' 'Lavaux_Plot_20' 'Lavaux_Plot_17')\n",
    "\n",
    "# Loop through each plot\n",
    "for plot in \"${plots[@]}\"\n",
    "do \n",
    "    echo \"Processing $plot\"\n",
    "    \n",
    "    # Create directory for the plot and navigate into it\n",
    "    mkdir -p \"./$plot\"\n",
    "    pushd \"./$plot\" > /dev/null\n",
    "\n",
    "    # Filter table and rep seqs\n",
    "    qiime feature-table filter-samples \\\n",
    "      --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/macrofungi/lavaux/must_filtered_table.qza \\\n",
    "      --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv \\\n",
    "      --p-where \"[Plot_ID]='$plot'\" \\\n",
    "      --o-filtered-table must_harvest21_${plot}_filtered_table.qza\n",
    "\n",
    "    qiime feature-table filter-seqs \\\n",
    "        --i-data /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/Sven-denoising-test/ITS/revcomp-trimmed-denoised/with-adjusted-parameters/ITS-revcomp-trimmed-ER-0.05-denoised-trunc-190-PFA-4.0-maxee-4.0-rep-seqs.qza \\\n",
    "        --i-table must_harvest21_${plot}_filtered_table.qza \\\n",
    "        --o-filtered-data must_harvest21_${plot}_filtered_rep_seqs.qza\n",
    "\n",
    "    # Core diversity metrics\n",
    "    qiime diversity core-metrics \\\n",
    "        --i-table must_harvest21_${plot}_filtered_table.qza \\\n",
    "        --p-sampling-depth 10000 \\\n",
    "        --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux_Harvest21.tsv \\\n",
    "        --p-n-jobs 10 \\\n",
    "        --p-ignore-missing-samples \\\n",
    "        --output-dir cm10000/\n",
    "\n",
    "    # Kmer diversity\n",
    "    qiime kmerizer core-metrics \\\n",
    "        --i-sequences must_harvest21_${plot}_filtered_rep_seqs.qza \\\n",
    "        --i-table must_harvest21_${plot}_filtered_table.qza \\\n",
    "        --p-sampling-depth 10000 \\\n",
    "        --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux_Harvest21.tsv \\\n",
    "        --p-n-jobs auto \\\n",
    "        --output-dir kmer_cm10000/\n",
    "\n",
    "    # Export all distance matrics for Mantel \n",
    "    qiime tools export --input-path cm10000/bray_curtis_distance_matrix.qza --output-path cm10000/bray_curtis_distance_matrix\n",
    "    qiime tools export --input-path cm10000/jaccard_distance_matrix.qza --output-path cm10000/jaccard_distance_matrix\n",
    "    qiime tools export --input-path kmer_cm10000/bray_curtis_distance_matrix.qza --output-path kmer_cm10000/bray_curtis_distance_matrix\n",
    "    qiime tools export --input-path kmer_cm10000/jaccard_distance_matrix.qza --output-path kmer_cm10000/jaccard_distance_matrix\n",
    "\n",
    "    # Return to the parent directory\n",
    "    popd > /dev/null\n",
    "done\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c3eb4599",
   "metadata": {},
   "source": [
    "###  Mantel tests "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "8a75210e",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                    Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_4/cm10000-bray_curtis_distance_matrix        -0.01413569\n",
      "2          Lavaux_Plot_4/cm10000-jaccard_distance_matrix        -0.01287396\n",
      "3 Lavaux_Plot_4/kmer_cm10000-bray_curtis_distance_matrix        -0.02367147\n",
      "4     Lavaux_Plot_4/kmer_cm10000-jaccard_distance_matrix        -0.03547038\n",
      "  Significance\n",
      "1        0.547\n",
      "2        0.539\n",
      "3        0.621\n",
      "4        0.634\n",
      "                                                    Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_5/cm10000-bray_curtis_distance_matrix         0.02794778\n",
      "2          Lavaux_Plot_5/cm10000-jaccard_distance_matrix        -0.20952540\n",
      "3 Lavaux_Plot_5/kmer_cm10000-bray_curtis_distance_matrix         0.06921815\n",
      "4     Lavaux_Plot_5/kmer_cm10000-jaccard_distance_matrix        -0.21764059\n",
      "  Significance\n",
      "1        0.388\n",
      "2        0.968\n",
      "3        0.253\n",
      "4        0.991\n",
      "                                                    Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_9/cm10000-bray_curtis_distance_matrix         0.10723853\n",
      "2          Lavaux_Plot_9/cm10000-jaccard_distance_matrix         0.02470799\n",
      "3 Lavaux_Plot_9/kmer_cm10000-bray_curtis_distance_matrix         0.02497219\n",
      "4     Lavaux_Plot_9/kmer_cm10000-jaccard_distance_matrix        -0.05460845\n",
      "  Significance\n",
      "1        0.137\n",
      "2        0.375\n",
      "3        0.370\n",
      "4        0.702\n",
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_11/cm10000-bray_curtis_distance_matrix        -0.11192311\n",
      "2          Lavaux_Plot_11/cm10000-jaccard_distance_matrix        -0.16303716\n",
      "3 Lavaux_Plot_11/kmer_cm10000-bray_curtis_distance_matrix        -0.05262983\n",
      "4     Lavaux_Plot_11/kmer_cm10000-jaccard_distance_matrix         0.02167810\n",
      "  Significance\n",
      "1        0.893\n",
      "2        0.982\n",
      "3        0.723\n",
      "4        0.353\n",
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_12/cm10000-bray_curtis_distance_matrix        -0.04819915\n",
      "2          Lavaux_Plot_12/cm10000-jaccard_distance_matrix        -0.05146824\n",
      "3 Lavaux_Plot_12/kmer_cm10000-bray_curtis_distance_matrix        -0.01561471\n",
      "4     Lavaux_Plot_12/kmer_cm10000-jaccard_distance_matrix        -0.07164257\n",
      "  Significance\n",
      "1        0.627\n",
      "2        0.652\n",
      "3        0.526\n",
      "4        0.704\n",
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_13/cm10000-bray_curtis_distance_matrix         0.09137709\n",
      "2          Lavaux_Plot_13/cm10000-jaccard_distance_matrix         0.27028766\n",
      "3 Lavaux_Plot_13/kmer_cm10000-bray_curtis_distance_matrix         0.24607465\n",
      "4     Lavaux_Plot_13/kmer_cm10000-jaccard_distance_matrix         0.04427284\n",
      "  Significance\n",
      "1        0.270\n",
      "2        0.058\n",
      "3        0.060\n",
      "4        0.398\n",
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_14/cm10000-bray_curtis_distance_matrix         0.34384933\n",
      "2          Lavaux_Plot_14/cm10000-jaccard_distance_matrix         0.26297277\n",
      "3 Lavaux_Plot_14/kmer_cm10000-bray_curtis_distance_matrix         0.30010496\n",
      "4     Lavaux_Plot_14/kmer_cm10000-jaccard_distance_matrix         0.05838939\n",
      "  Significance\n",
      "1        0.006\n",
      "2        0.004\n",
      "3        0.007\n",
      "4        0.272\n",
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_15/cm10000-bray_curtis_distance_matrix        -0.05057643\n",
      "2          Lavaux_Plot_15/cm10000-jaccard_distance_matrix         0.11827690\n",
      "3 Lavaux_Plot_15/kmer_cm10000-bray_curtis_distance_matrix        -0.01779333\n",
      "4     Lavaux_Plot_15/kmer_cm10000-jaccard_distance_matrix         0.09678172\n",
      "  Significance\n",
      "1        0.748\n",
      "2        0.043\n",
      "3        0.587\n",
      "4        0.098\n",
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_18/cm10000-bray_curtis_distance_matrix        -0.18973245\n",
      "2          Lavaux_Plot_18/cm10000-jaccard_distance_matrix        -0.01805363\n",
      "3 Lavaux_Plot_18/kmer_cm10000-bray_curtis_distance_matrix        -0.17220026\n",
      "4     Lavaux_Plot_18/kmer_cm10000-jaccard_distance_matrix        -0.06455863\n",
      "  Significance\n",
      "1        0.819\n",
      "2        0.543\n",
      "3        0.791\n",
      "4        0.608\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "'nperm' >= set of all permutations: complete enumeration.\n",
      "Set of permutations < 'minperm'. Generating entire set.\n",
      "'nperm' >= set of all permutations: complete enumeration.\n",
      "Set of permutations < 'minperm'. Generating entire set.\n",
      "'nperm' >= set of all permutations: complete enumeration.\n",
      "Set of permutations < 'minperm'. Generating entire set.\n",
      "'nperm' >= set of all permutations: complete enumeration.\n",
      "Set of permutations < 'minperm'. Generating entire set.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_19/cm10000-bray_curtis_distance_matrix          0.4857143\n",
      "2          Lavaux_Plot_19/cm10000-jaccard_distance_matrix         -0.2571429\n",
      "3 Lavaux_Plot_19/kmer_cm10000-bray_curtis_distance_matrix          0.4857143\n",
      "4     Lavaux_Plot_19/kmer_cm10000-jaccard_distance_matrix          0.4857143\n",
      "  Significance\n",
      "1    0.1666667\n",
      "2    0.8333333\n",
      "3    0.1666667\n",
      "4    0.1666667\n",
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_20/cm10000-bray_curtis_distance_matrix         0.02477141\n",
      "2          Lavaux_Plot_20/cm10000-jaccard_distance_matrix         0.05122138\n",
      "3 Lavaux_Plot_20/kmer_cm10000-bray_curtis_distance_matrix        -0.07192939\n",
      "4     Lavaux_Plot_20/kmer_cm10000-jaccard_distance_matrix         0.14662223\n",
      "  Significance\n",
      "1        0.383\n",
      "2        0.312\n",
      "3        0.755\n",
      "4        0.083\n",
      "                                                     Test Mantel_statistic_r\n",
      "1      Lavaux_Plot_17/cm10000-bray_curtis_distance_matrix         0.06149698\n",
      "2          Lavaux_Plot_17/cm10000-jaccard_distance_matrix         0.05899055\n",
      "3 Lavaux_Plot_17/kmer_cm10000-bray_curtis_distance_matrix         0.09162549\n",
      "4     Lavaux_Plot_17/kmer_cm10000-jaccard_distance_matrix         0.06731784\n",
      "  Significance\n",
      "1        0.146\n",
      "2        0.152\n",
      "3        0.048\n",
      "4        0.148\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "folders=('Lavaux_Plot_4' 'Lavaux_Plot_5' 'Lavaux_Plot_9' \n",
    "       'Lavaux_Plot_11' 'Lavaux_Plot_12' 'Lavaux_Plot_13' \n",
    "       'Lavaux_Plot_14' 'Lavaux_Plot_15' 'Lavaux_Plot_18' \n",
    "       'Lavaux_Plot_19' 'Lavaux_Plot_20' 'Lavaux_Plot_17')\n",
    "\n",
    "geodesic_path='geodesic_distance_ITS_HR/distance-matrix.tsv'\n",
    "\n",
    "for folder in \"${folders[@]}\"\n",
    "do\n",
    "    folder_path_1=\"$folder/cm10000\"\n",
    "    folder_path_2=\"$folder/kmer_cm10000\"\n",
    "\n",
    "    # Call the R script and pass folder paths and geodesic matrix\n",
    "    Rscript /home/lfloerl/microterroir/Microbiome/Other_scripts/mantel_test.R \"$folder_path_1\" \"$folder_path_2\" \"$geodesic_path\"\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "d976e467",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Main_Folder</th>\n",
       "      <th>Subfolder</th>\n",
       "      <th>Distance_Matrix</th>\n",
       "      <th>Mantel_statistic_r</th>\n",
       "      <th>Significance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Lavaux_Plot_4</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.014136</td>\n",
       "      <td>0.547000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Lavaux_Plot_4</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.012874</td>\n",
       "      <td>0.539000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Lavaux_Plot_4</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.023671</td>\n",
       "      <td>0.621000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Lavaux_Plot_4</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.035470</td>\n",
       "      <td>0.634000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Lavaux_Plot_5</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.027948</td>\n",
       "      <td>0.388000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Lavaux_Plot_5</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.209525</td>\n",
       "      <td>0.968000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Lavaux_Plot_5</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.069218</td>\n",
       "      <td>0.253000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Lavaux_Plot_5</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.217641</td>\n",
       "      <td>0.991000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Lavaux_Plot_9</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.107239</td>\n",
       "      <td>0.137000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Lavaux_Plot_9</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.024708</td>\n",
       "      <td>0.375000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Lavaux_Plot_9</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.024972</td>\n",
       "      <td>0.370000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Lavaux_Plot_9</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.054608</td>\n",
       "      <td>0.702000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Lavaux_Plot_11</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.111923</td>\n",
       "      <td>0.893000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Lavaux_Plot_11</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.163037</td>\n",
       "      <td>0.982000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Lavaux_Plot_11</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.052630</td>\n",
       "      <td>0.723000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Lavaux_Plot_11</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.021678</td>\n",
       "      <td>0.353000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Lavaux_Plot_12</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.048199</td>\n",
       "      <td>0.627000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Lavaux_Plot_12</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.051468</td>\n",
       "      <td>0.652000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Lavaux_Plot_12</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.015615</td>\n",
       "      <td>0.526000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Lavaux_Plot_12</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.071643</td>\n",
       "      <td>0.704000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Lavaux_Plot_13</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.091377</td>\n",
       "      <td>0.270000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>Lavaux_Plot_13</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.270288</td>\n",
       "      <td>0.058000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>Lavaux_Plot_13</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.246075</td>\n",
       "      <td>0.060000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Lavaux_Plot_13</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.044273</td>\n",
       "      <td>0.398000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>Lavaux_Plot_14</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.343849</td>\n",
       "      <td>0.006000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>Lavaux_Plot_14</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.262973</td>\n",
       "      <td>0.004000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>Lavaux_Plot_14</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.300105</td>\n",
       "      <td>0.007000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>Lavaux_Plot_14</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.058389</td>\n",
       "      <td>0.272000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>Lavaux_Plot_15</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.050576</td>\n",
       "      <td>0.748000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>Lavaux_Plot_15</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.118277</td>\n",
       "      <td>0.043000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>Lavaux_Plot_15</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.017793</td>\n",
       "      <td>0.587000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>Lavaux_Plot_15</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.096782</td>\n",
       "      <td>0.098000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>Lavaux_Plot_18</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.189732</td>\n",
       "      <td>0.819000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>Lavaux_Plot_18</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.018054</td>\n",
       "      <td>0.543000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>Lavaux_Plot_18</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.172200</td>\n",
       "      <td>0.791000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>Lavaux_Plot_18</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.064559</td>\n",
       "      <td>0.608000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>Lavaux_Plot_19</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.485714</td>\n",
       "      <td>0.166667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>Lavaux_Plot_19</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.257143</td>\n",
       "      <td>0.833333</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>Lavaux_Plot_19</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.485714</td>\n",
       "      <td>0.166667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>Lavaux_Plot_19</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.485714</td>\n",
       "      <td>0.166667</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>Lavaux_Plot_20</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.024771</td>\n",
       "      <td>0.383000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>Lavaux_Plot_20</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.051221</td>\n",
       "      <td>0.312000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>Lavaux_Plot_20</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>-0.071929</td>\n",
       "      <td>0.755000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>Lavaux_Plot_20</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.146622</td>\n",
       "      <td>0.083000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>Lavaux_Plot_17</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.061497</td>\n",
       "      <td>0.146000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>Lavaux_Plot_17</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.058991</td>\n",
       "      <td>0.152000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>Lavaux_Plot_17</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.091625</td>\n",
       "      <td>0.048000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>Lavaux_Plot_17</td>\n",
       "      <td>kmer_cm10000</td>\n",
       "      <td>jaccard_distance_matrix</td>\n",
       "      <td>0.067318</td>\n",
       "      <td>0.148000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       Main_Folder     Subfolder          Distance_Matrix  Mantel_statistic_r  \\\n",
       "0    Lavaux_Plot_4  kmer_cm10000  jaccard_distance_matrix           -0.014136   \n",
       "1    Lavaux_Plot_4  kmer_cm10000  jaccard_distance_matrix           -0.012874   \n",
       "2    Lavaux_Plot_4  kmer_cm10000  jaccard_distance_matrix           -0.023671   \n",
       "3    Lavaux_Plot_4  kmer_cm10000  jaccard_distance_matrix           -0.035470   \n",
       "4    Lavaux_Plot_5  kmer_cm10000  jaccard_distance_matrix            0.027948   \n",
       "5    Lavaux_Plot_5  kmer_cm10000  jaccard_distance_matrix           -0.209525   \n",
       "6    Lavaux_Plot_5  kmer_cm10000  jaccard_distance_matrix            0.069218   \n",
       "7    Lavaux_Plot_5  kmer_cm10000  jaccard_distance_matrix           -0.217641   \n",
       "8    Lavaux_Plot_9  kmer_cm10000  jaccard_distance_matrix            0.107239   \n",
       "9    Lavaux_Plot_9  kmer_cm10000  jaccard_distance_matrix            0.024708   \n",
       "10   Lavaux_Plot_9  kmer_cm10000  jaccard_distance_matrix            0.024972   \n",
       "11   Lavaux_Plot_9  kmer_cm10000  jaccard_distance_matrix           -0.054608   \n",
       "12  Lavaux_Plot_11  kmer_cm10000  jaccard_distance_matrix           -0.111923   \n",
       "13  Lavaux_Plot_11  kmer_cm10000  jaccard_distance_matrix           -0.163037   \n",
       "14  Lavaux_Plot_11  kmer_cm10000  jaccard_distance_matrix           -0.052630   \n",
       "15  Lavaux_Plot_11  kmer_cm10000  jaccard_distance_matrix            0.021678   \n",
       "16  Lavaux_Plot_12  kmer_cm10000  jaccard_distance_matrix           -0.048199   \n",
       "17  Lavaux_Plot_12  kmer_cm10000  jaccard_distance_matrix           -0.051468   \n",
       "18  Lavaux_Plot_12  kmer_cm10000  jaccard_distance_matrix           -0.015615   \n",
       "19  Lavaux_Plot_12  kmer_cm10000  jaccard_distance_matrix           -0.071643   \n",
       "20  Lavaux_Plot_13  kmer_cm10000  jaccard_distance_matrix            0.091377   \n",
       "21  Lavaux_Plot_13  kmer_cm10000  jaccard_distance_matrix            0.270288   \n",
       "22  Lavaux_Plot_13  kmer_cm10000  jaccard_distance_matrix            0.246075   \n",
       "23  Lavaux_Plot_13  kmer_cm10000  jaccard_distance_matrix            0.044273   \n",
       "24  Lavaux_Plot_14  kmer_cm10000  jaccard_distance_matrix            0.343849   \n",
       "25  Lavaux_Plot_14  kmer_cm10000  jaccard_distance_matrix            0.262973   \n",
       "26  Lavaux_Plot_14  kmer_cm10000  jaccard_distance_matrix            0.300105   \n",
       "27  Lavaux_Plot_14  kmer_cm10000  jaccard_distance_matrix            0.058389   \n",
       "28  Lavaux_Plot_15  kmer_cm10000  jaccard_distance_matrix           -0.050576   \n",
       "29  Lavaux_Plot_15  kmer_cm10000  jaccard_distance_matrix            0.118277   \n",
       "30  Lavaux_Plot_15  kmer_cm10000  jaccard_distance_matrix           -0.017793   \n",
       "31  Lavaux_Plot_15  kmer_cm10000  jaccard_distance_matrix            0.096782   \n",
       "32  Lavaux_Plot_18  kmer_cm10000  jaccard_distance_matrix           -0.189732   \n",
       "33  Lavaux_Plot_18  kmer_cm10000  jaccard_distance_matrix           -0.018054   \n",
       "34  Lavaux_Plot_18  kmer_cm10000  jaccard_distance_matrix           -0.172200   \n",
       "35  Lavaux_Plot_18  kmer_cm10000  jaccard_distance_matrix           -0.064559   \n",
       "36  Lavaux_Plot_19  kmer_cm10000  jaccard_distance_matrix            0.485714   \n",
       "37  Lavaux_Plot_19  kmer_cm10000  jaccard_distance_matrix           -0.257143   \n",
       "38  Lavaux_Plot_19  kmer_cm10000  jaccard_distance_matrix            0.485714   \n",
       "39  Lavaux_Plot_19  kmer_cm10000  jaccard_distance_matrix            0.485714   \n",
       "40  Lavaux_Plot_20  kmer_cm10000  jaccard_distance_matrix            0.024771   \n",
       "41  Lavaux_Plot_20  kmer_cm10000  jaccard_distance_matrix            0.051221   \n",
       "42  Lavaux_Plot_20  kmer_cm10000  jaccard_distance_matrix           -0.071929   \n",
       "43  Lavaux_Plot_20  kmer_cm10000  jaccard_distance_matrix            0.146622   \n",
       "44  Lavaux_Plot_17  kmer_cm10000  jaccard_distance_matrix            0.061497   \n",
       "45  Lavaux_Plot_17  kmer_cm10000  jaccard_distance_matrix            0.058991   \n",
       "46  Lavaux_Plot_17  kmer_cm10000  jaccard_distance_matrix            0.091625   \n",
       "47  Lavaux_Plot_17  kmer_cm10000  jaccard_distance_matrix            0.067318   \n",
       "\n",
       "    Significance  \n",
       "0       0.547000  \n",
       "1       0.539000  \n",
       "2       0.621000  \n",
       "3       0.634000  \n",
       "4       0.388000  \n",
       "5       0.968000  \n",
       "6       0.253000  \n",
       "7       0.991000  \n",
       "8       0.137000  \n",
       "9       0.375000  \n",
       "10      0.370000  \n",
       "11      0.702000  \n",
       "12      0.893000  \n",
       "13      0.982000  \n",
       "14      0.723000  \n",
       "15      0.353000  \n",
       "16      0.627000  \n",
       "17      0.652000  \n",
       "18      0.526000  \n",
       "19      0.704000  \n",
       "20      0.270000  \n",
       "21      0.058000  \n",
       "22      0.060000  \n",
       "23      0.398000  \n",
       "24      0.006000  \n",
       "25      0.004000  \n",
       "26      0.007000  \n",
       "27      0.272000  \n",
       "28      0.748000  \n",
       "29      0.043000  \n",
       "30      0.587000  \n",
       "31      0.098000  \n",
       "32      0.819000  \n",
       "33      0.543000  \n",
       "34      0.791000  \n",
       "35      0.608000  \n",
       "36      0.166667  \n",
       "37      0.833333  \n",
       "38      0.166667  \n",
       "39      0.166667  \n",
       "40      0.383000  \n",
       "41      0.312000  \n",
       "42      0.755000  \n",
       "43      0.083000  \n",
       "44      0.146000  \n",
       "45      0.152000  \n",
       "46      0.048000  \n",
       "47      0.148000  "
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# List of folders to loop through\n",
    "folders = ['Lavaux_Plot_4', 'Lavaux_Plot_5', 'Lavaux_Plot_9', \n",
    "           'Lavaux_Plot_11', 'Lavaux_Plot_12', 'Lavaux_Plot_13', \n",
    "           'Lavaux_Plot_14', 'Lavaux_Plot_15', 'Lavaux_Plot_18', \n",
    "           'Lavaux_Plot_19', 'Lavaux_Plot_20', 'Lavaux_Plot_17']\n",
    "\n",
    "# Initialize an empty list to store the results\n",
    "all_results = []\n",
    "\n",
    "# Loop through each folder and process the results\n",
    "for folder in folders:\n",
    "    folder_path_2 = os.path.join(folder, \"kmer_cm10000\")\n",
    "    mantel_path_2 = os.path.join(folder_path_2, \"mantel_results.tsv\")\n",
    "    \n",
    "    # Check if the mantel results file exists\n",
    "    if os.path.exists(mantel_path_2):\n",
    "        df_2 = pd.read_csv(mantel_path_2, sep=\"\\t\")\n",
    "\n",
    "        # Add metadata columns\n",
    "        df_2['Main_Folder'] = folder\n",
    "        df_2['Subfolder'] = 'kmer_cm10000'\n",
    "        df_2['Distance_Matrix'] = 'bray_curtis_distance_matrix' if 'bray_curtis' in mantel_path_2 else 'jaccard_distance_matrix'\n",
    "\n",
    "        all_results.append(df_2)\n",
    "\n",
    "# Concatenate all the results into a single dataframe\n",
    "if all_results:\n",
    "    final_results = pd.concat(all_results, ignore_index=True)\n",
    "    final_results = final_results[['Main_Folder', 'Subfolder', 'Distance_Matrix', 'Mantel_statistic_r', 'Significance']]\n",
    "else:\n",
    "    final_results = pd.DataFrame(columns=['Main_Folder', 'Subfolder', 'Distance_Matrix', 'Mantel_statistic_r', 'Significance'])\n",
    "\n",
    "# Display final results\n",
    "final_results\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b2c0ef22",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.14"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
