{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Importing and Denoising LP3\n",
    "\n",
    "> downloaded the demultiplexed files from FGCZ \n",
    "\n",
    "\n",
    "## 1. File Preprocessing for Import\n",
    "\n",
    "To import files to QIIME2 with Casava1.8 format we first have to add the Lane number (since it's a full flow cell this is not specified)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash \n",
    "\n",
    "## add lane number \n",
    "cd /home/lfloerl/cloud/lfloerl/Microterroir/o36452_NextSeq_241102_NS2k-400 \n",
    "\n",
    "# Loop through all fastq.gz files in the directory\n",
    "for file in *.fastq.gz; \n",
    "do \n",
    "  # Extract the parts of the filename\n",
    "  base_name=$(echo \"$file\" | sed -E 's/(.*)(_R[12]_001\\.fastq\\.gz)/\\1_L001\\2/')\n",
    "  \n",
    "  # Rename the file\n",
    "  mv \"$file\" \"$base_name\"\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Move the 16S and ITS reads respective folders\n",
    "\n",
    "FGCZ assigned new IDs, so we have to move the files to import fungal and bacterial data seperately."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "## split by 16S & ITS reads\n",
    "mkdir -p ITS\n",
    "mkdir -p 16S\n",
    "\n",
    "# Loop through all fastq.gz files in the current directory\n",
    "for file in *.fastq.gz; do\n",
    "  # Check if the filename contains 'ITS'\n",
    "  if [[ \"$file\" == *ITS* ]]; then\n",
    "    mv \"$file\" ITS/\n",
    "  elif [[ \"$file\" == *16S* ]]; then\n",
    "    mv \"$file\" 16S/\n",
    "  fi\n",
    "done"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Import files to QIIME2 "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash \n",
    "\n",
    "mkdir /home/lfloerl/cloud/lfloerl/Microterroir/artifacts\n",
    "cd /home/lfloerl/cloud/lfloerl/Microterroir/artifacts \n",
    "\n",
    "mkdir -p ITS\n",
    "mkdir -p 16S\n",
    "\n",
    "time qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /home/lfloerl/cloud/lfloerl/Microterroir/o36452_NextSeq_241102_NS2k-400/16S \\\n",
    "  --input-format CasavaOneEightSingleLanePerSampleDirFmt \\\n",
    "  --output-path 16S/16S-demux-paired-end.qza\n",
    "\n",
    "qiime demux summarize --i-data 16S/16S-demux-paired-end.qza --o-visualization 16S/16S-demux-paired-end.qzv\n",
    "\n",
    "time qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /home/lfloerl/cloud/lfloerl/Microterroir/o36452_NextSeq_241102_NS2k-400/ITS \\\n",
    "  --input-format CasavaOneEightSingleLanePerSampleDirFmt \\\n",
    "  --output-path ITS/ITS-demux-paired-end.qza\n",
    "\n",
    "qiime demux summarize --i-data ITS/ITS-demux-paired-end.qza --o-visualization ITS/ITS-demux-paired-end.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4. Denoise \n",
    "\n",
    "### 4.1. Denoise 16S paired end "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash \n",
    "\n",
    "cd /home/lfloerl/cloud/lfloerl/Microterroir/artifacts \n",
    "\n",
    "mkdir 16S/bac-dada2\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs 16S/16S-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 190 \\\n",
    "    --p-trunc-len-r 165 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences 16S/bac-dada2/dada-rep-seqs.qza \\\n",
    "    --o-table 16S/bac-dada2/dada-table.qza \\\n",
    "    --o-denoising-stats 16S/bac-dada2/dada-stats.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application(s). This may print messages to stdout and/or stderr.\n",
      "The command(s) being run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: run_dada.R --input_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/forward --input_directory_reverse /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/reverse --output_path /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/output.tsv.biom --output_track /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/track.tsv --filtered_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_f --filtered_directory_reverse /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_r --truncation_length 220 --truncation_length_reverse 190 --trim_left 0 --trim_left_reverse 0 --max_expected_errors 2.0 --max_expected_errors_reverse 2.0 --truncation_quality_score 2 --min_overlap 12 --pooling_method independent --chimera_method consensus --min_parental_fold 1.0 --allow_one_off False --num_threads 1 --learn_min_reads 1000000\n",
      "\n",
      "R version 4.3.3 (2024-02-29) \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Lade n√∂tiges Paket: Rcpp\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DADA2: 1.30.0 / Rcpp: 1.0.13.1 / RcppParallel: 5.1.9 \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_f/364527_045-LP3-16S-0045_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_r/364527_045-LP3-16S-0045_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_f/364527_162-LP3-16S-0162_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_r/364527_162-LP3-16S-0162_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_f/364527_165-LP3-16S-0165_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_r/364527_165-LP3-16S-0165_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_f/364527_258-LP3-16S-0258_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_r/364527_258-LP3-16S-0258_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_f/364527_417-LP3-16S-0417_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_r/364527_417-LP3-16S-0417_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_f/364528_431-LP3-16S-1007_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_r/364528_431-LP3-16S-1007_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_f/364528_432-LP3-16S-1008_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpofxg5r3v/filt_r/364528_432-LP3-16S-1008_L001_R2_001.fastq.gz not written.\n",
      "Some input samples had no reads pass the filter.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2) Filtering ...........................................x....................................................................................................................x..x............................................................................................x..............................................................................................................................................................x..........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................xx................................................................................................................................................\n",
      "3) Learning Error Rates\n",
      "228854340 total bases in 1040247 reads from 286 samples will be used for learning the error rates.\n",
      "197646930 total bases in 1040247 reads from 286 samples will be used for learning the error rates.\n",
      "3) Denoise samples .....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      ".....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      "5) Remove chimeras (method = consensus)\n",
      "6) Report read numbers through the pipeline\n",
      "7) Write output\n",
      "Saved FeatureTable[Frequency] to: 16S/denoised/dada-220_190-table.qza\n",
      "Saved FeatureData[Sequence] to: 16S/denoised/dada-220_190-rep-seqs.qza\n",
      "Saved SampleData[DADA2Stats] to: 16S/denoised/dada-220_190-stats.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t287m2.052s\n",
      "user\t285m17.362s\n",
      "sys\t1m29.416s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-220_190-rep-seqs.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m16.006s\n",
      "user\t0m9.335s\n",
      "sys\t0m1.110s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-220_190-table.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m12.773s\n",
      "user\t0m6.386s\n",
      "sys\t0m1.167s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-220_190-stats.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m10.883s\n",
      "user\t0m4.744s\n",
      "sys\t0m0.973s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs 16S/16S-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 220 \\\n",
    "    --p-trunc-len-r 190 \\\n",
    "    --o-representative-sequences 16S/denoised/dada-220_190-rep-seqs.qza \\\n",
    "    --o-table 16S/denoised/dada-220_190-table.qza \\\n",
    "    --o-denoising-stats 16S/denoised/dada-220_190-stats.qza \\\n",
    "    --verbose\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data 16S/denoised/dada-220_190-rep-seqs.qza \\\n",
    "    --o-visualization 16S/denoised/dada-220_190-rep-seqs.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table 16S/denoised/dada-220_190-table.qza \\\n",
    "    --o-visualization 16S/denoised/dada-220_190-table.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file 16S/denoised/dada-220_190-stats.qza \\\n",
    "    --o-visualization 16S/denoised/dada-220_190-stats.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application(s). This may print messages to stdout and/or stderr.\n",
      "The command(s) being run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: run_dada.R --input_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/forward --input_directory_reverse /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/reverse --output_path /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/output.tsv.biom --output_track /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/track.tsv --filtered_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_f --filtered_directory_reverse /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_r --truncation_length 210 --truncation_length_reverse 180 --trim_left 0 --trim_left_reverse 0 --max_expected_errors 2.0 --max_expected_errors_reverse 2.0 --truncation_quality_score 2 --min_overlap 12 --pooling_method independent --chimera_method consensus --min_parental_fold 1.0 --allow_one_off False --num_threads 1 --learn_min_reads 1000000\n",
      "\n",
      "R version 4.3.3 (2024-02-29) \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Lade n√∂tiges Paket: Rcpp\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DADA2: 1.30.0 / Rcpp: 1.0.13.1 / RcppParallel: 5.1.9 \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_f/364527_045-LP3-16S-0045_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_r/364527_045-LP3-16S-0045_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_f/364527_162-LP3-16S-0162_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_r/364527_162-LP3-16S-0162_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_f/364527_165-LP3-16S-0165_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_r/364527_165-LP3-16S-0165_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_f/364527_258-LP3-16S-0258_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_r/364527_258-LP3-16S-0258_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_f/364527_417-LP3-16S-0417_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_r/364527_417-LP3-16S-0417_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_f/364528_431-LP3-16S-1007_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_r/364528_431-LP3-16S-1007_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_f/364528_432-LP3-16S-1008_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpb5q4e66r/filt_r/364528_432-LP3-16S-1008_L001_R2_001.fastq.gz not written.\n",
      "Some input samples had no reads pass the filter.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2) Filtering ...........................................x....................................................................................................................x..x............................................................................................x..............................................................................................................................................................x..........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................xx................................................................................................................................................\n",
      "3) Learning Error Rates\n",
      "229171530 total bases in 1091293 reads from 286 samples will be used for learning the error rates.\n",
      "196432740 total bases in 1091293 reads from 286 samples will be used for learning the error rates.\n",
      "3) Denoise samples .....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      ".....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      "5) Remove chimeras (method = consensus)\n",
      "6) Report read numbers through the pipeline\n",
      "7) Write output\n",
      "Saved FeatureTable[Frequency] to: 16S/denoised/dada-210_180-table.qza\n",
      "Saved FeatureData[Sequence] to: 16S/denoised/dada-210_180-rep-seqs.qza\n",
      "Saved SampleData[DADA2Stats] to: 16S/denoised/dada-210_180-stats.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t373m15.450s\n",
      "user\t256m52.574s\n",
      "sys\t1m36.576s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-210_180-rep-seqs.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m11.701s\n",
      "user\t0m9.412s\n",
      "sys\t0m1.203s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-210_180-table.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m7.874s\n",
      "user\t0m6.424s\n",
      "sys\t0m1.208s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-210_180-stats.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m5.911s\n",
      "user\t0m4.754s\n",
      "sys\t0m0.984s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs 16S/16S-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 210 \\\n",
    "    --p-trunc-len-r 180 \\\n",
    "    --o-representative-sequences 16S/denoised/dada-210_180-rep-seqs.qza \\\n",
    "    --o-table 16S/denoised/dada-210_180-table.qza \\\n",
    "    --o-denoising-stats 16S/denoised/dada-210_180-stats.qza \\\n",
    "    --verbose\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data 16S/denoised/dada-210_180-rep-seqs.qza \\\n",
    "    --o-visualization 16S/denoised/dada-210_180-rep-seqs.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table 16S/denoised/dada-210_180-table.qza \\\n",
    "    --o-visualization 16S/denoised/dada-210_180-table.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file 16S/denoised/dada-210_180-stats.qza \\\n",
    "    --o-visualization 16S/denoised/dada-210_180-stats.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application(s). This may print messages to stdout and/or stderr.\n",
      "The command(s) being run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: run_dada.R --input_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/forward --input_directory_reverse /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/reverse --output_path /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/output.tsv.biom --output_track /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/track.tsv --filtered_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_f --filtered_directory_reverse /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_r --truncation_length 200 --truncation_length_reverse 170 --trim_left 0 --trim_left_reverse 0 --max_expected_errors 2.0 --max_expected_errors_reverse 2.0 --truncation_quality_score 2 --min_overlap 12 --pooling_method independent --chimera_method consensus --min_parental_fold 1.0 --allow_one_off False --num_threads 1 --learn_min_reads 1000000\n",
      "\n",
      "R version 4.3.3 (2024-02-29) \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Lade n√∂tiges Paket: Rcpp\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DADA2: 1.30.0 / Rcpp: 1.0.13.1 / RcppParallel: 5.1.9 \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_f/364527_045-LP3-16S-0045_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_r/364527_045-LP3-16S-0045_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_f/364527_165-LP3-16S-0165_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_r/364527_165-LP3-16S-0165_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_f/364527_258-LP3-16S-0258_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_r/364527_258-LP3-16S-0258_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_f/364528_431-LP3-16S-1007_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_r/364528_431-LP3-16S-1007_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_f/364528_432-LP3-16S-1008_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpetfmb2sw/filt_r/364528_432-LP3-16S-1008_L001_R2_001.fastq.gz not written.\n",
      "Some input samples had no reads pass the filter.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2) Filtering ...........................................x.......................................................................................................................x............................................................................................x.........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................xx................................................................................................................................................\n",
      "3) Learning Error Rates\n",
      "227483000 total bases in 1137415 reads from 287 samples will be used for learning the error rates.\n",
      "193360550 total bases in 1137415 reads from 287 samples will be used for learning the error rates.\n",
      "3) Denoise samples .......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      ".......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      "5) Remove chimeras (method = consensus)\n",
      "6) Report read numbers through the pipeline\n",
      "7) Write output\n",
      "Saved FeatureTable[Frequency] to: 16S/denoised/dada-200_170-table.qza\n",
      "Saved FeatureData[Sequence] to: 16S/denoised/dada-200_170-rep-seqs.qza\n",
      "Saved SampleData[DADA2Stats] to: 16S/denoised/dada-200_170-stats.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t258m2.751s\n",
      "user\t255m27.971s\n",
      "sys\t2m0.649s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-200_170-rep-seqs.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m12.147s\n",
      "user\t0m9.711s\n",
      "sys\t0m1.366s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-200_170-table.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m8.473s\n",
      "user\t0m6.797s\n",
      "sys\t0m1.404s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-200_170-stats.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m6.478s\n",
      "user\t0m5.119s\n",
      "sys\t0m1.153s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs 16S/16S-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 200 \\\n",
    "    --p-trunc-len-r 170 \\\n",
    "    --o-representative-sequences 16S/denoised/dada-200_170-rep-seqs.qza \\\n",
    "    --o-table 16S/denoised/dada-200_170-table.qza \\\n",
    "    --o-denoising-stats 16S/denoised/dada-200_170-stats.qza \\\n",
    "    --verbose\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data 16S/denoised/dada-200_170-rep-seqs.qza \\\n",
    "    --o-visualization 16S/denoised/dada-200_170-rep-seqs.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table 16S/denoised/dada-200_170-table.qza \\\n",
    "    --o-visualization 16S/denoised/dada-200_170-table.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file 16S/denoised/dada-200_170-stats.qza \\\n",
    "    --o-visualization 16S/denoised/dada-200_170-stats.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application(s). This may print messages to stdout and/or stderr.\n",
      "The command(s) being run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: run_dada.R --input_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/forward --input_directory_reverse /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/reverse --output_path /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/output.tsv.biom --output_track /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/track.tsv --filtered_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_f --filtered_directory_reverse /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_r --truncation_length 180 --truncation_length_reverse 150 --trim_left 0 --trim_left_reverse 0 --max_expected_errors 2.0 --max_expected_errors_reverse 2.0 --truncation_quality_score 2 --min_overlap 12 --pooling_method independent --chimera_method consensus --min_parental_fold 1.0 --allow_one_off False --num_threads 1 --learn_min_reads 1000000\n",
      "\n",
      "R version 4.3.3 (2024-02-29) \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Lade n√∂tiges Paket: Rcpp\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DADA2: 1.30.0 / Rcpp: 1.0.13.1 / RcppParallel: 5.1.9 \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_f/364527_165-LP3-16S-0165_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_r/364527_165-LP3-16S-0165_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_f/364527_258-LP3-16S-0258_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_r/364527_258-LP3-16S-0258_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_f/364528_431-LP3-16S-1007_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_r/364528_431-LP3-16S-1007_L001_R2_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_f/364528_432-LP3-16S-1008_L001_R1_001.fastq.gz and /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpqohwzsut/filt_r/364528_432-LP3-16S-1008_L001_R2_001.fastq.gz not written.\n",
      "Some input samples had no reads pass the filter.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2) Filtering ...................................................................................................................................................................x............................................................................................x.........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................xx................................................................................................................................................\n",
      "3) Learning Error Rates\n",
      "190821060 total bases in 1060117 reads from 287 samples will be used for learning the error rates.\n",
      "159017550 total bases in 1060117 reads from 287 samples will be used for learning the error rates.\n",
      "3) Denoise samples ........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      "........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      "5) Remove chimeras (method = consensus)\n",
      "6) Report read numbers through the pipeline\n",
      "7) Write output\n",
      "Saved FeatureTable[Frequency] to: 16S/denoised/dada-180_150-table.qza\n",
      "Saved FeatureData[Sequence] to: 16S/denoised/dada-180_150-rep-seqs.qza\n",
      "Saved SampleData[DADA2Stats] to: 16S/denoised/dada-180_150-stats.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t282m0.538s\n",
      "user\t233m5.808s\n",
      "sys\t1m46.011s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-180_150-rep-seqs.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m12.335s\n",
      "user\t0m10.132s\n",
      "sys\t0m1.335s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-180_150-table.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m8.906s\n",
      "user\t0m7.121s\n",
      "sys\t0m1.380s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: 16S/denoised/dada-180_150-stats.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m6.638s\n",
      "user\t0m5.322s\n",
      "sys\t0m1.123s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs 16S/16S-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 180 \\\n",
    "    --p-trunc-len-r 150 \\\n",
    "    --o-representative-sequences 16S/denoised/dada-180_150-rep-seqs.qza \\\n",
    "    --o-table 16S/denoised/dada-180_150-table.qza \\\n",
    "    --o-denoising-stats 16S/denoised/dada-180_150-stats.qza \\\n",
    "    --verbose\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data 16S/denoised/dada-180_150-rep-seqs.qza \\\n",
    "    --o-visualization 16S/denoised/dada-180_150-rep-seqs.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table 16S/denoised/dada-180_150-table.qza \\\n",
    "    --o-visualization 16S/denoised/dada-180_150-table.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file 16S/denoised/dada-180_150-stats.qza \\\n",
    "    --o-visualization 16S/denoised/dada-180_150-stats.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.2 Denoise ITS single-end, no cutadapt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/microterroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-single \\\n",
    "    --i-demultiplexed-seqs ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len 180 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences ITS/dada2-nocutadapt/dada-rep-seqs-180.qza \\\n",
    "    --o-table ITS/dada2-nocutadapt/dada-table-180.qza \\\n",
    "    --o-denoising-stats ITS/dada2-nocutadapt/dada-stats-180.qza \n",
    "    \n",
    "time qiime dada2 denoise-single \\\n",
    "    --i-demultiplexed-seqs ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len 190 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences ITS/dada2-nocutadapt/dada-rep-seqs-190.qza \\\n",
    "    --o-table ITS/dada2-nocutadapt/dada-table-190.qza \\\n",
    "    --o-denoising-stats ITS/dada2-nocutadapt/dada-stats-190.qza\n",
    "\n",
    "time qiime dada2 denoise-single \\\n",
    "    --i-demultiplexed-seqs ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len 200 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences ITS/dada2-nocutadapt/dada-rep-seqs-200.qza \\\n",
    "    --o-table ITS/dada2-nocutadapt/dada-table-200.qza \\\n",
    "    --o-denoising-stats ITS/dada2-nocutadapt/dada-stats-200.qza\n",
    "\n",
    "time qiime dada2 denoise-single \\\n",
    "    --i-demultiplexed-seqs ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len 210 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences ITS/dada2-nocutadapt/dada-rep-seqs-210.qza \\\n",
    "    --o-table ITS/dada2-nocutadapt/dada-table-210.qza \\\n",
    "    --o-denoising-stats ITS/dada2-nocutadapt/dada-stats-210.qza\n",
    "\n",
    "time qiime dada2 denoise-single \\\n",
    "    --i-demultiplexed-seqs ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len 220 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences ITS/dada2-nocutadapt/dada-rep-seqs-220.qza \\\n",
    "    --o-table ITS/dada2-nocutadapt/dada-table-220.qza \\\n",
    "    --o-denoising-stats ITS/dada2-nocutadapt/dada-stats-220.qza"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.3 ITS Cutadapt without Primer + denoising w/o truncation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/microterroir/artefacts\n",
    "\n",
    "time qiime cutadapt trim-single \\\n",
    "    --i-demultiplexed-sequences ITS-demux-single-end.qza \\\n",
    "    --o-trimmed-sequences ITS/cutadapt-noprimer/ITS-trimmed-noprimer.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: ITS-trimmed-noprimer-denoised-table.qza\n",
      "Saved FeatureData[Sequence] to: ITS-trimmed-noprimer-denoised-rep-seqs.qza\n",
      "Saved SampleData[DADA2Stats] to: ITS-trimmed-noprimer-denoised-stats.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t8m48.422s\n",
      "user\t20m6.053s\n",
      "sys\t0m17.536s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS-trimmed-noprimer-denoised-rep-seqs.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m13.296s\n",
      "user\t0m6.010s\n",
      "sys\t0m1.282s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS-trimmed-noprimer-denoised-table.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m12.117s\n",
      "user\t0m5.515s\n",
      "sys\t0m1.272s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS-trimmed-noprimer-denoised-stats.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m11.232s\n",
      "user\t0m4.903s\n",
      "sys\t0m1.133s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-single \\\n",
    "    --i-demultiplexed-seqs ITS-trimmed-noprimer.qza \\\n",
    "    --p-trunc-len 0 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences ITS-trimmed-noprimer-denoised-rep-seqs.qza \\\n",
    "    --o-table ITS-trimmed-noprimer-denoised-table.qza \\\n",
    "    --o-denoising-stats ITS-trimmed-noprimer-denoised-stats.qza\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data ITS-trimmed-noprimer-denoised-rep-seqs.qza \\\n",
    "    --o-visualization ITS-trimmed-noprimer-denoised-rep-seqs.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table ITS-trimmed-noprimer-denoised-table.qza \\\n",
    "    --o-visualization ITS-trimmed-noprimer-denoised-table.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file ITS-trimmed-noprimer-denoised-stats.qza \\\n",
    "    --o-visualization ITS-trimmed-noprimer-denoised-stats.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.4 Cutadapt with primer + denoising w/o truncation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_001-LP3-ITS-0001_318_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_001-LP3-ITS-0001_318_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_001-LP3-ITS-0001_318_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_001-LP3-ITS-0001_318_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.674 s (10.442 ¬µs/read; 5.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  64,556\n",
      "Reads with adapters:                       975 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        64,556 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,431,356 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     19,301,523 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 975 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.3%\n",
      "  C: 13.4%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1008.7\t0\t1\n",
      "9\t2\t0.2\t0\t2\n",
      "10\t1\t0.1\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t9\t0.0\t2\t0 0 9\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t171\t0.0\t2\t0 3 168\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t69\t0.0\t2\t0 0 69\n",
      "134\t119\t0.0\t2\t0 0 119\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t205\t0.0\t2\t0 0 205\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t317\t0.0\t2\t0 4 313\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_002-LP3-ITS-0002_689_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_002-LP3-ITS-0002_689_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_002-LP3-ITS-0002_689_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_002-LP3-ITS-0002_689_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.509 s (10.007 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  50,880\n",
      "Reads with adapters:                       462 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        50,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,314,880 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,254,174 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 462 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.2%\n",
      "  C: 18.4%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t795.0\t0\t3\n",
      "4\t2\t198.8\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "52\t5\t0.0\t2\t0 0 5\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t154\t0.0\t2\t0 2 152\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "130\t7\t0.0\t2\t0 0 7\n",
      "133\t39\t0.0\t2\t0 1 38\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t178\t0.0\t2\t0 0 178\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_003-LP3-ITS-0003_507_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_003-LP3-ITS-0003_507_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_003-LP3-ITS-0003_507_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_003-LP3-ITS-0003_507_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.446 s (10.198 ¬µs/read; 5.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,733\n",
      "Reads with adapters:                     1,005 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,733 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,163,633 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,124,211 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1005 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 51.4%\n",
      "  C: 29.7%\n",
      "  G: 1.0%\n",
      "  T: 17.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t481\t683.3\t0\t481\n",
      "4\t3\t170.8\t0\t3\n",
      "8\t1\t0.7\t0\t1\n",
      "9\t159\t0.2\t0\t151 8\n",
      "10\t108\t0.0\t1\t6 102\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 1 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t82\t0.0\t2\t0 1 81\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "134\t5\t0.0\t2\t0 1 4\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t30\t0.0\t2\t0 1 29\n",
      "150\t16\t0.0\t2\t0 0 16\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "160\t34\t0.0\t2\t0 0 34\n",
      "175\t20\t0.0\t2\t0 0 20\n",
      "219\t8\t0.0\t2\t0 0 8\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_004-LP3-ITS-0004_159_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_004-LP3-ITS-0004_159_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_004-LP3-ITS-0004_159_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_004-LP3-ITS-0004_159_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.505 s (9.662 ¬µs/read; 6.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,250\n",
      "Reads with adapters:                       542 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,250 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,727,250 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,658,990 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 542 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.2%\n",
      "  C: 14.4%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t816.4\t0\t3\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "87\t8\t0.0\t2\t0 0 8\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t321\t0.0\t2\t0 4 317\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "133\t40\t0.0\t2\t0 0 40\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t30\t0.0\t2\t0 0 30\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t48\t0.0\t2\t0 1 47\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "160\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_005-LP3-ITS-0005_1119_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_005-LP3-ITS-0005_1119_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_005-LP3-ITS-0005_1119_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_005-LP3-ITS-0005_1119_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.515 s (10.147 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  50,771\n",
      "Reads with adapters:                       559 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        50,771 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,282,071 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,207,676 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 559 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.3%\n",
      "  C: 16.5%\n",
      "  G: 0.4%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t793.3\t0\t3\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "109\t8\t0.0\t2\t0 0 8\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t242\t0.0\t2\t0 11 231\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t40\t0.0\t2\t0 1 39\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t26\t0.0\t2\t0 0 26\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t17\t0.0\t2\t0 0 17\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t31\t0.0\t2\t0 0 31\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "155\t21\t0.0\t2\t0 1 20\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t65\t0.0\t2\t0 0 65\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t54\t0.0\t2\t0 0 54\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "194\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_006-LP3-ITS-0006_337_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_006-LP3-ITS-0006_337_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_006-LP3-ITS-0006_337_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_006-LP3-ITS-0006_337_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.314 s (9.901 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,671\n",
      "Reads with adapters:                       339 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,671 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,532,971 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,498,246 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 339 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.3%\n",
      "  C: 10.3%\n",
      "  G: 0.3%\n",
      "  T: 2.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t494.9\t0\t28\n",
      "4\t2\t123.7\t0\t2\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t25\t0.1\t0\t25\n",
      "10\t1\t0.0\t1\t1\n",
      "84\t6\t0.0\t2\t0 0 6\n",
      "94\t8\t0.0\t2\t0 0 8\n",
      "106\t7\t0.0\t2\t0 0 7\n",
      "118\t203\t0.0\t2\t0 4 199\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t6\t0.0\t2\t0 1 5\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_007-LP3-ITS-0007_294_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_007-LP3-ITS-0007_294_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_007-LP3-ITS-0007_294_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_007-LP3-ITS-0007_294_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.408 s (9.820 ¬µs/read; 6.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,577\n",
      "Reads with adapters:                       468 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,577 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,514,677 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,455,521 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 468 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 62.0%\n",
      "  C: 37.6%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t649.6\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t213\t0.0\t2\t0 10 203\n",
      "133\t110\t0.0\t2\t0 0 110\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t23\t0.0\t2\t0 0 23\n",
      "151\t24\t0.0\t2\t0 0 24\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t34\t0.0\t2\t0 0 34\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_008-LP3-ITS-0008_692_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_008-LP3-ITS-0008_692_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_008-LP3-ITS-0008_692_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_008-LP3-ITS-0008_692_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.413 s (10.072 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,967\n",
      "Reads with adapters:                       461 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,967 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,331,067 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,273,593 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 461 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.3%\n",
      "  C: 25.4%\n",
      "  G: 0.4%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t640.1\t0\t1\n",
      "4\t1\t160.0\t0\t1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "46\t12\t0.0\t2\t0 1 11\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "59\t3\t0.0\t2\t0 1 2\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t169\t0.0\t2\t0 3 166\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t35\t0.0\t2\t0 0 35\n",
      "132\t54\t0.0\t2\t0 0 54\n",
      "133\t50\t0.0\t2\t0 0 50\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t16\t0.0\t2\t0 0 16\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t26\t0.0\t2\t0 0 26\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "160\t17\t0.0\t2\t0 0 17\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "165\t10\t0.0\t2\t0 1 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_009-LP3-ITS-0009_907_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_009-LP3-ITS-0009_907_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_009-LP3-ITS-0009_907_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_009-LP3-ITS-0009_907_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.388 s (10.245 ¬µs/read; 5.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,874\n",
      "Reads with adapters:                       518 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,874 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,400,074 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,329,540 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 518 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.9%\n",
      "  C: 19.9%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t147.9\t0\t1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "118\t145\t0.0\t2\t0 1 144\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "130\t11\t0.0\t2\t0 0 11\n",
      "133\t44\t0.0\t2\t0 0 44\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t6\t0.0\t2\t0 1 5\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t115\t0.0\t2\t0 3 112\n",
      "151\t58\t0.0\t2\t0 0 58\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t30\t0.0\t2\t0 0 30\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t7\t0.0\t2\t0 1 6\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t49\t0.0\t2\t0 2 47\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_010-LP3-ITS-0010_278_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_010-LP3-ITS-0010_278_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_010-LP3-ITS-0010_278_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_010-LP3-ITS-0010_278_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.593 s (10.068 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  58,852\n",
      "Reads with adapters:                       597 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        58,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,714,452 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,633,461 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 597 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.3%\n",
      "  C: 13.9%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t919.6\t0\t1\n",
      "4\t1\t229.9\t0\t1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t169\t0.0\t2\t0 0 169\n",
      "133\t49\t0.0\t2\t0 2 47\n",
      "134\t14\t0.0\t2\t0 1 13\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t172\t0.0\t2\t0 3 169\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t119\t0.0\t2\t0 2 117\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t8\t0.0\t2\t0 1 7\n",
      "160\t14\t0.0\t2\t0 0 14\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_011-LP3-ITS-0011_506_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_011-LP3-ITS-0011_506_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_011-LP3-ITS-0011_506_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_011-LP3-ITS-0011_506_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.523 s (10.230 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,167\n",
      "Reads with adapters:                       589 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,167 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,401,267 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,322,199 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 589 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.4%\n",
      "  C: 14.3%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t799.5\t0\t1\n",
      "6\t1\t12.5\t0\t1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t173\t0.0\t2\t0 2 171\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 1\n",
      "133\t50\t0.0\t2\t0 1 49\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t34\t0.0\t2\t0 0 34\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t200\t0.0\t2\t0 4 196\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t63\t0.0\t2\t0 0 63\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t10\t0.0\t2\t0 0 10\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_012-LP3-ITS-0012_236_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_012-LP3-ITS-0012_236_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_012-LP3-ITS-0012_236_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_012-LP3-ITS-0012_236_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.457 s (9.792 ¬µs/read; 6.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,712\n",
      "Reads with adapters:                       470 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,060,312 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,000,741 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 470 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.5%\n",
      "  C: 24.0%\n",
      "  G: 0.4%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t729.9\t0\t3\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "118\t222\t0.0\t2\t0 5 217\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t60\t0.0\t2\t0 0 60\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "142\t78\t0.0\t2\t0 1 77\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t41\t0.0\t2\t0 1 40\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t6\t0.0\t2\t0 0 6\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_013-LP3-ITS-0013_382_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_013-LP3-ITS-0013_382_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_013-LP3-ITS-0013_382_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_013-LP3-ITS-0013_382_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.346 s (10.877 ¬µs/read; 5.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,815\n",
      "Reads with adapters:                     8,127 (25.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,815 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,576,315 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,521,444 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8127 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.7%\n",
      "  C: 48.3%\n",
      "  G: 1.8%\n",
      "  T: 25.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6265\t497.1\t0\t6265\n",
      "4\t4\t124.3\t0\t4\n",
      "7\t1\t1.9\t0\t1\n",
      "8\t34\t0.5\t0\t34\n",
      "9\t61\t0.1\t0\t61\n",
      "10\t1613\t0.0\t1\t112 1501\n",
      "11\t6\t0.0\t1\t0 6\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "118\t64\t0.0\t2\t0 1 63\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "145\t27\t0.0\t2\t0 0 27\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "175\t7\t0.0\t2\t0 0 7\n",
      "212\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_014-LP3-ITS-0014_581_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_014-LP3-ITS-0014_581_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_014-LP3-ITS-0014_581_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_014-LP3-ITS-0014_581_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.409 s (8.227 ¬µs/read; 7.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  49,675\n",
      "Reads with adapters:                     4,504 (9.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        49,675 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,952,175 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,929,991 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4504 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.5%\n",
      "  C: 6.1%\n",
      "  G: 0.1%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4442\t776.2\t0\t4442\n",
      "4\t5\t194.0\t0\t5\n",
      "5\t4\t48.5\t0\t4\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "176\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_015-LP3-ITS-0015_271_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_015-LP3-ITS-0015_271_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_015-LP3-ITS-0015_271_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_015-LP3-ITS-0015_271_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.484 s (9.876 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,998\n",
      "Reads with adapters:                       672 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,998 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,748,398 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,659,619 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 672 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.5%\n",
      "  C: 15.0%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t765.6\t0\t23\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t10\t0.0\t2\t0 0 10\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t173\t0.0\t2\t0 1 172\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t35\t0.0\t2\t0 0 35\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t10\t0.0\t2\t0 1 9\n",
      "151\t308\t0.0\t2\t0 3 305\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "164\t19\t0.0\t2\t0 0 19\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_016-LP3-ITS-0016_246_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_016-LP3-ITS-0016_246_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_016-LP3-ITS-0016_246_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_016-LP3-ITS-0016_246_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.317 s (10.316 ¬µs/read; 5.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,728\n",
      "Reads with adapters:                       444 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,728 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,249,128 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,194,033 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 444 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.0%\n",
      "  C: 22.3%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t480.1\t0\t1\n",
      "5\t11\t30.0\t0\t11\n",
      "10\t1\t0.0\t1\t0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t4\t0.0\t2\t0 0 4\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t4\t0.0\t2\t0 1 3\n",
      "60\t7\t0.0\t2\t0 1 6\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t192\t0.0\t2\t0 4 188\n",
      "121\t4\t0.0\t2\t0 0 4\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t30\t0.0\t2\t0 0 30\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t116\t0.0\t2\t0 0 116\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_017-LP3-ITS-0017_327_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_017-LP3-ITS-0017_327_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_017-LP3-ITS-0017_327_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_017-LP3-ITS-0017_327_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.538 s (9.809 ¬µs/read; 6.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  54,817\n",
      "Reads with adapters:                       673 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        54,817 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,499,917 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,411,636 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 673 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.5%\n",
      "  C: 15.9%\n",
      "  G: 0.1%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t856.5\t0\t1\n",
      "4\t5\t214.1\t0\t5\n",
      "5\t7\t53.5\t0\t7\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "51\t9\t0.0\t2\t0 0 9\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t223\t0.0\t2\t0 3 220\n",
      "123\t6\t0.0\t2\t0 1 5\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "134\t4\t0.0\t2\t0 1 3\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t60\t0.0\t2\t0 0 60\n",
      "143\t31\t0.0\t2\t0 0 31\n",
      "145\t61\t0.0\t2\t0 0 61\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t165\t0.0\t2\t0 1 164\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_018-LP3-ITS-0018_950_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_018-LP3-ITS-0018_950_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_018-LP3-ITS-0018_950_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_018-LP3-ITS-0018_950_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.475 s (9.887 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,081\n",
      "Reads with adapters:                       429 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,081 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,472,381 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,418,573 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 429 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.7%\n",
      "  C: 18.6%\n",
      "  G: 0.2%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "118\t191\t0.0\t2\t0 5 186\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t46\t0.0\t2\t0 1 45\n",
      "134\t14\t0.0\t2\t0 1 13\n",
      "137\t15\t0.0\t2\t0 0 15\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t59\t0.0\t2\t0 1 58\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t42\t0.0\t2\t0 0 42\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 1 6\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_019-LP3-ITS-0019_598_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_019-LP3-ITS-0019_598_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_019-LP3-ITS-0019_598_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_019-LP3-ITS-0019_598_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.420 s (10.037 ¬µs/read; 5.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,882\n",
      "Reads with adapters:                       426 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,882 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,606,482 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,552,204 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 426 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.8%\n",
      "  C: 23.9%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t4\t2.6\t0\t4\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t200\t0.0\t2\t0 5 195\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t62\t0.0\t2\t0 0 62\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t11\t0.0\t2\t0 0 11\n",
      "145\t88\t0.0\t2\t0 1 87\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t22\t0.0\t2\t0 0 22\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_020-LP3-ITS-0020_1102_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_020-LP3-ITS-0020_1102_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_020-LP3-ITS-0020_1102_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_020-LP3-ITS-0020_1102_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.396 s (9.924 ¬µs/read; 6.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,885\n",
      "Reads with adapters:                       410 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,885 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,005,385 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,955,094 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 410 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.0%\n",
      "  C: 27.6%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t623.2\t0\t1\n",
      "4\t1\t155.8\t0\t1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "118\t240\t0.0\t2\t0 5 235\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t45\t0.0\t2\t0 0 45\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t21\t0.0\t2\t0 0 21\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "155\t35\t0.0\t2\t0 1 34\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_021-LP3-ITS-0021_39_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_021-LP3-ITS-0021_39_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_021-LP3-ITS-0021_39_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_021-LP3-ITS-0021_39_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.581 s (10.130 ¬µs/read; 5.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  57,359\n",
      "Reads with adapters:                       664 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        57,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,265,059 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,178,337 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 664 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.1%\n",
      "  C: 13.3%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t896.2\t0\t9\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t227\t0.0\t2\t0 8 219\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t57\t0.0\t2\t0 0 57\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "137\t85\t0.0\t2\t0 1 84\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t159\t0.0\t2\t0 1 158\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t59\t0.0\t2\t0 0 59\n",
      "151\t27\t0.0\t2\t0 0 27\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_022-LP3-ITS-0022_23_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_022-LP3-ITS-0022_23_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_022-LP3-ITS-0022_23_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_022-LP3-ITS-0022_23_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.403 s (10.266 ¬µs/read; 5.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,265\n",
      "Reads with adapters:                       426 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,265 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,818,765 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,764,303 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 426 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.1%\n",
      "  C: 16.4%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t613.5\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "118\t213\t0.0\t2\t0 2 211\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t96\t0.0\t2\t0 1 95\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "160\t12\t0.0\t2\t0 1 11\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_023-LP3-ITS-0023_783_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_023-LP3-ITS-0023_783_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_023-LP3-ITS-0023_783_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_023-LP3-ITS-0023_783_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.498 s (9.973 ¬µs/read; 6.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  49,901\n",
      "Reads with adapters:                       566 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        49,901 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,020,201 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,945,268 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 566 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.6%\n",
      "  C: 21.4%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t779.7\t0\t2\n",
      "4\t1\t194.9\t0\t1\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t5\t0.0\t2\t0 1 4\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "118\t189\t0.0\t2\t0 5 184\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t78\t0.0\t2\t0 0 78\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t97\t0.0\t2\t0 2 95\n",
      "146\t5\t0.0\t2\t0 0 5\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t84\t0.0\t2\t0 0 84\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "160\t48\t0.0\t2\t0 0 48\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_024-LP3-ITS-0024_140_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_024-LP3-ITS-0024_140_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_024-LP3-ITS-0024_140_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_024-LP3-ITS-0024_140_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.777 s (10.234 ¬µs/read; 5.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  75,912\n",
      "Reads with adapters:                       859 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        75,912 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,849,512 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     22,739,490 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 859 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.7%\n",
      "  C: 28.5%\n",
      "  G: 0.2%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1186.1\t0\t2\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "52\t50\t0.0\t2\t0 1 49\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t244\t0.0\t2\t0 4 240\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t13\t0.0\t2\t0 0 13\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t127\t0.0\t2\t0 1 126\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t25\t0.0\t2\t0 0 25\n",
      "143\t25\t0.0\t2\t0 0 25\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t11\t0.0\t2\t0 0 11\n",
      "151\t245\t0.0\t2\t0 2 243\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "164\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_025-LP3-ITS-0025_570_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_025-LP3-ITS-0025_570_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_025-LP3-ITS-0025_570_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_025-LP3-ITS-0025_570_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.380 s (9.944 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,185\n",
      "Reads with adapters:                       461 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,185 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,493,685 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,432,096 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 461 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.9%\n",
      "  C: 15.4%\n",
      "  G: 0.2%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t596.6\t0\t4\n",
      "5\t6\t37.3\t0\t6\n",
      "10\t1\t0.0\t1\t0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t144\t0.0\t2\t0 4 140\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t18\t0.0\t2\t0 0 18\n",
      "151\t201\t0.0\t2\t0 2 199\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "252\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_026-LP3-ITS-0026_857_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_026-LP3-ITS-0026_857_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_026-LP3-ITS-0026_857_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_026-LP3-ITS-0026_857_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.555 s (10.053 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  55,228\n",
      "Reads with adapters:                       472 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        55,228 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,623,628 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,562,348 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 472 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 61.9%\n",
      "  C: 37.7%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t862.9\t0\t3\n",
      "4\t1\t215.7\t0\t1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t10\t0.0\t2\t0 0 10\n",
      "53\t4\t0.0\t2\t0 0 4\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t168\t0.0\t2\t0 0 168\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t35\t0.0\t2\t0 2 33\n",
      "134\t10\t0.0\t2\t0 0 10\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t29\t0.0\t2\t0 2 27\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t28\t0.0\t2\t0 0 28\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t81\t0.0\t2\t0 1 80\n",
      "160\t21\t0.0\t2\t0 0 21\n",
      "198\t2\t0.0\t2\t0 1 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_027-LP3-ITS-0027_836_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_027-LP3-ITS-0027_836_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_027-LP3-ITS-0027_836_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_027-LP3-ITS-0027_836_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.594 s (9.775 ¬µs/read; 6.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  60,764\n",
      "Reads with adapters:                       576 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        60,764 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,289,964 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,225,939 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 576 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.4%\n",
      "  C: 22.4%\n",
      "  G: 0.0%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t949.4\t0\t4\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t4\t0.0\t2\t0 0 4\n",
      "47\t19\t0.0\t2\t0 0 19\n",
      "50\t8\t0.0\t2\t0 0 8\n",
      "51\t41\t0.0\t2\t0 0 41\n",
      "67\t5\t0.0\t2\t0 0 5\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t383\t0.0\t2\t0 5 378\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t31\t0.0\t2\t0 0 31\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t33\t0.0\t2\t0 1 32\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_028-LP3-ITS-0028_512_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_028-LP3-ITS-0028_512_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_028-LP3-ITS-0028_512_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_028-LP3-ITS-0028_512_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.317 s (9.998 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,677\n",
      "Reads with adapters:                       343 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,677 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,534,777 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,494,043 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 343 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.6%\n",
      "  C: 28.9%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t495.0\t0\t3\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "84\t23\t0.0\t2\t0 1 22\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t211\t0.0\t2\t0 4 207\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t54\t0.0\t2\t0 0 54\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t13\t0.0\t2\t0 0 13\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_029-LP3-ITS-0029_878_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_029-LP3-ITS-0029_878_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_029-LP3-ITS-0029_878_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_029-LP3-ITS-0029_878_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.472 s (10.095 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,712\n",
      "Reads with adapters:                       469 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,060,312 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,996,293 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 469 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.7%\n",
      "  C: 11.1%\n",
      "  G: 0.2%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t729.9\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "118\t157\t0.0\t2\t0 1 156\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t20\t0.0\t2\t0 0 20\n",
      "151\t188\t0.0\t2\t0 1 187\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "164\t16\t0.0\t2\t0 0 16\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_030-LP3-ITS-0030_663_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_030-LP3-ITS-0030_663_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_030-LP3-ITS-0030_663_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_030-LP3-ITS-0030_663_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.458 s (10.114 ¬µs/read; 5.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,301\n",
      "Reads with adapters:                     1,199 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,301 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,635,601 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,569,945 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1199 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.8%\n",
      "  C: 41.0%\n",
      "  G: 0.8%\n",
      "  T: 15.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t626\t707.8\t0\t626\n",
      "4\t3\t177.0\t0\t3\n",
      "9\t8\t0.2\t0\t8\n",
      "10\t104\t0.0\t1\t9 95\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "108\t6\t0.0\t2\t0 0 6\n",
      "118\t135\t0.0\t2\t0 4 131\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "133\t56\t0.0\t2\t0 0 56\n",
      "134\t12\t0.0\t2\t0 0 12\n",
      "137\t34\t0.0\t2\t0 1 33\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t53\t0.0\t2\t0 2 51\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "151\t112\t0.0\t2\t0 1 111\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t30\t0.0\t2\t0 0 30\n",
      "219\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_031-LP3-ITS-0031_223_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_031-LP3-ITS-0031_223_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_031-LP3-ITS-0031_223_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_031-LP3-ITS-0031_223_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.382 s (10.626 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,949\n",
      "Reads with adapters:                     1,386 (3.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,949 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,820,649 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,783,254 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1386 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 2.2%\n",
      "  C: 94.8%\n",
      "  G: 2.7%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t561.7\t0\t1\n",
      "4\t1\t140.4\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "25\t4\t0.0\t2\t0 0 4\n",
      "26\t1359\t0.0\t2\t1 50 1308\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "86\t13\t0.0\t2\t0 0 13\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_032-LP3-ITS-0032_301_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_032-LP3-ITS-0032_301_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_032-LP3-ITS-0032_301_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_032-LP3-ITS-0032_301_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.421 s (10.180 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,322\n",
      "Reads with adapters:                       507 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,322 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,437,922 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,369,250 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 507 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.1%\n",
      "  C: 8.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t199\t0.0\t2\t0 5 194\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t47\t0.0\t2\t0 0 47\n",
      "150\t26\t0.0\t2\t0 0 26\n",
      "151\t143\t0.0\t2\t0 2 141\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t12\t0.0\t2\t0 1 11\n",
      "160\t12\t0.0\t2\t0 1 11\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_033-LP3-ITS-0033_926_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_033-LP3-ITS-0033_926_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_033-LP3-ITS-0033_926_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_033-LP3-ITS-0033_926_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.476 s (10.087 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  47,189\n",
      "Reads with adapters:                       696 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        47,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,203,889 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,116,246 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 696 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.8%\n",
      "  C: 16.1%\n",
      "  G: 0.0%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t737.3\t0\t4\n",
      "9\t4\t0.2\t0\t4\n",
      "10\t2\t0.0\t1\t0 2\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "50\t6\t0.0\t2\t0 0 6\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t7\t0.0\t2\t0 1 6\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t319\t0.0\t2\t0 3 316\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t42\t0.0\t2\t0 0 42\n",
      "134\t38\t0.0\t2\t0 0 38\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "145\t129\t0.0\t2\t0 2 127\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t21\t0.0\t2\t0 0 21\n",
      "151\t31\t0.0\t2\t0 1 30\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t20\t0.0\t2\t0 1 19\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_034-LP3-ITS-0034_379_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_034-LP3-ITS-0034_379_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_034-LP3-ITS-0034_379_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_034-LP3-ITS-0034_379_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.370 s (9.938 ¬µs/read; 6.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,189\n",
      "Reads with adapters:                       315 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,193,889 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,157,811 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 315 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.6%\n",
      "  C: 30.2%\n",
      "  G: 0.0%\n",
      "  T: 2.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t581.1\t0\t11\n",
      "10\t4\t0.0\t1\t0 4\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t4\t0.0\t2\t0 0 4\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "55\t6\t0.0\t2\t0 0 6\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "81\t3\t0.0\t2\t0 0 3\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t147\t0.0\t2\t0 4 143\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t43\t0.0\t2\t0 0 43\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "145\t37\t0.0\t2\t0 0 37\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_035-LP3-ITS-0035_247_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_035-LP3-ITS-0035_247_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_035-LP3-ITS-0035_247_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_035-LP3-ITS-0035_247_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (21.784 ¬µs/read; 2.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     407\n",
      "Reads with adapters:                        12 (2.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           407 (100.0%)\n",
      "\n",
      "Total basepairs processed:       122,507 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        121,184 bp (98.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.7%\n",
      "  C: 8.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t10\t0.0\t2\t0 0 10\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_036-LP3-ITS-0036_66_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_036-LP3-ITS-0036_66_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_036-LP3-ITS-0036_66_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_036-LP3-ITS-0036_66_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.482 s (9.876 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,837\n",
      "Reads with adapters:                       450 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,837 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,699,937 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,643,440 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 450 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.9%\n",
      "  C: 25.1%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t763.1\t0\t2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "55\t3\t0.0\t2\t0 1 2\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t5\t0.0\t2\t0 1 4\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t221\t0.0\t2\t0 4 217\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t54\t0.0\t2\t0 0 54\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t38\t0.0\t2\t0 0 38\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t17\t0.0\t2\t0 0 17\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t25\t0.0\t2\t0 0 25\n",
      "160\t44\t0.0\t2\t0 1 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_037-LP3-ITS-0037_333_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_037-LP3-ITS-0037_333_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_037-LP3-ITS-0037_333_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_037-LP3-ITS-0037_333_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.331 s (9.756 ¬µs/read; 6.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,900\n",
      "Reads with adapters:                       314 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,203,900 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,165,986 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 314 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.2%\n",
      "  C: 11.8%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t529.7\t0\t2\n",
      "4\t3\t132.4\t0\t3\n",
      "48\t4\t0.0\t2\t0 0 4\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t223\t0.0\t2\t0 6 217\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t11\t0.0\t2\t0 0 11\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t22\t0.0\t2\t0 0 22\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_038-LP3-ITS-0038_811_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_038-LP3-ITS-0038_811_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_038-LP3-ITS-0038_811_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_038-LP3-ITS-0038_811_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.835 s (9.778 ¬µs/read; 6.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  85,369\n",
      "Reads with adapters:                       743 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        85,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,696,069 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     25,594,077 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 743 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.5%\n",
      "  C: 11.6%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t1333.9\t0\t11\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t174\t0.0\t2\t0 0 174\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "133\t30\t0.0\t2\t0 0 30\n",
      "134\t14\t0.0\t2\t0 0 14\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "143\t125\t0.0\t2\t0 1 124\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t90\t0.0\t2\t0 1 89\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t20\t0.0\t2\t0 0 20\n",
      "151\t147\t0.0\t2\t0 1 146\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "158\t16\t0.0\t2\t0 0 16\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t47\t0.0\t2\t0 0 47\n",
      "164\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_039-LP3-ITS-0039_695_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_039-LP3-ITS-0039_695_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_039-LP3-ITS-0039_695_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_039-LP3-ITS-0039_695_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.518 s (10.323 ¬µs/read; 5.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  50,135\n",
      "Reads with adapters:                       507 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        50,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,090,635 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,027,817 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 507 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.5%\n",
      "  C: 23.5%\n",
      "  G: 0.4%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t783.4\t0\t8\n",
      "10\t2\t0.0\t1\t0 2\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "45\t7\t0.0\t2\t0 0 7\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "118\t288\t0.0\t2\t1 4 283\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "132\t6\t0.0\t2\t0 0 6\n",
      "133\t35\t0.0\t2\t0 0 35\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "155\t27\t0.0\t2\t0 0 27\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t72\t0.0\t2\t0 1 71\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_040-LP3-ITS-0040_1131_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_040-LP3-ITS-0040_1131_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_040-LP3-ITS-0040_1131_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_040-LP3-ITS-0040_1131_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.309 s (10.398 ¬µs/read; 5.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,743\n",
      "Reads with adapters:                       422 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,743 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,952,643 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,895,056 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 422 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.2%\n",
      "  C: 11.8%\n",
      "  G: 0.2%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t464.7\t0\t2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t114\t0.0\t2\t0 3 111\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "134\t13\t0.0\t2\t0 0 13\n",
      "137\t33\t0.0\t2\t0 1 32\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t92\t0.0\t2\t1 2 89\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t92\t0.0\t2\t0 1 91\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t15\t0.0\t2\t0 1 14\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_041-LP3-ITS-0041_1041_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_041-LP3-ITS-0041_1041_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_041-LP3-ITS-0041_1041_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_041-LP3-ITS-0041_1041_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.237 s (10.246 ¬µs/read; 5.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,113\n",
      "Reads with adapters:                       287 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,957,013 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,920,294 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 287 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.5%\n",
      "  C: 18.1%\n",
      "  G: 0.3%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t361.1\t0\t1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t131\t0.0\t2\t0 0 131\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t16\t0.0\t2\t0 1 15\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t27\t0.0\t2\t0 0 27\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t14\t0.0\t2\t0 0 14\n",
      "164\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_042-LP3-ITS-0042_16_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_042-LP3-ITS-0042_16_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_042-LP3-ITS-0042_16_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_042-LP3-ITS-0042_16_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.345 s (10.373 ¬µs/read; 5.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,268\n",
      "Reads with adapters:                       476 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,013,668 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,968,714 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 476 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.4%\n",
      "  C: 15.8%\n",
      "  G: 0.4%\n",
      "  T: 9.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t82\t519.8\t0\t82\n",
      "5\t1\t32.5\t0\t1\n",
      "8\t5\t0.5\t0\t5\n",
      "9\t54\t0.1\t0\t49 5\n",
      "10\t11\t0.0\t1\t1 10\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "118\t85\t0.0\t2\t0 1 84\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "134\t16\t0.0\t2\t0 0 16\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t122\t0.0\t2\t0 2 120\n",
      "147\t18\t0.0\t2\t0 0 18\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t19\t0.0\t2\t0 1 18\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_043-LP3-ITS-0043_18_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_043-LP3-ITS-0043_18_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_043-LP3-ITS-0043_18_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_043-LP3-ITS-0043_18_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.575 s (9.851 ¬µs/read; 6.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  58,343\n",
      "Reads with adapters:                       609 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        58,343 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,561,243 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,484,356 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 609 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.7%\n",
      "  C: 21.8%\n",
      "  G: 0.2%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t911.6\t0\t1\n",
      "5\t1\t57.0\t0\t1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t2\t0.0\t2\t0 1 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "84\t7\t0.0\t2\t0 0 7\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "118\t267\t0.0\t2\t1 7 259\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t10\t0.0\t2\t0 0 10\n",
      "133\t54\t0.0\t2\t0 1 53\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t172\t0.0\t2\t0 3 169\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_044-LP3-ITS-0044_911_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_044-LP3-ITS-0044_911_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_044-LP3-ITS-0044_911_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_044-LP3-ITS-0044_911_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.368 s (8.529 ¬µs/read; 7.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,133\n",
      "Reads with adapters:                     3,472 (8.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,133 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,983,033 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,959,641 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3472 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.3%\n",
      "  C: 7.0%\n",
      "  G: 0.2%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3372\t674.0\t0\t3372\n",
      "4\t9\t168.5\t0\t9\n",
      "5\t1\t42.1\t0\t1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "118\t36\t0.0\t2\t0 1 35\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "176\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_045-LP3-ITS-0045_709_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_045-LP3-ITS-0045_709_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_045-LP3-ITS-0045_709_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_045-LP3-ITS-0045_709_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.458 s (9.801 ¬µs/read; 6.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,772\n",
      "Reads with adapters:                       434 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,772 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,078,372 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,019,737 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 434 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.8%\n",
      "  C: 17.7%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t150\t0.0\t2\t1 0 149\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t62\t0.0\t2\t0 1 61\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t103\t0.0\t2\t0 1 102\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t66\t0.0\t2\t0 0 66\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_046-LP3-ITS-0046_900_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_046-LP3-ITS-0046_900_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_046-LP3-ITS-0046_900_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_046-LP3-ITS-0046_900_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.321 s (9.871 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,519\n",
      "Reads with adapters:                       318 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,519 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,788,219 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,748,047 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 318 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.4%\n",
      "  C: 23.3%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t508.1\t0\t2\n",
      "4\t1\t127.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "118\t163\t0.0\t2\t0 1 162\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t49\t0.0\t2\t0 0 49\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t23\t0.0\t2\t0 0 23\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t19\t0.0\t2\t0 0 19\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "164\t18\t0.0\t2\t0 0 18\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "188\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_047-LP3-ITS-0047_501_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_047-LP3-ITS-0047_501_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_047-LP3-ITS-0047_501_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_047-LP3-ITS-0047_501_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.097 s (10.488 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   9,279\n",
      "Reads with adapters:                       240 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         9,279 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,792,979 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,788,552 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 240 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.7%\n",
      "  C: 49.2%\n",
      "  G: 2.1%\n",
      "  T: 17.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t114\t145.0\t0\t114\n",
      "5\t21\t9.1\t0\t21\n",
      "6\t6\t2.3\t0\t6\n",
      "9\t33\t0.0\t0\t29 4\n",
      "10\t16\t0.0\t1\t2 14\n",
      "43\t36\t0.0\t2\t0 0 36\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t8\t0.0\t2\t0 0 8\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_048-LP3-ITS-0048_42_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_048-LP3-ITS-0048_42_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_048-LP3-ITS-0048_42_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_048-LP3-ITS-0048_42_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.545 s (9.863 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  55,304\n",
      "Reads with adapters:                       676 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        55,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,646,504 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,567,818 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 676 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.8%\n",
      "  C: 25.6%\n",
      "  G: 0.1%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t864.1\t0\t18\n",
      "4\t2\t216.0\t0\t2\n",
      "5\t1\t54.0\t0\t1\n",
      "6\t8\t13.5\t0\t8\n",
      "9\t3\t0.2\t0\t2 1\n",
      "10\t3\t0.1\t1\t0 3\n",
      "32\t18\t0.0\t2\t0 0 18\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t17\t0.0\t2\t0 0 17\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "60\t7\t0.0\t2\t0 0 7\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t295\t0.0\t2\t0 4 291\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t58\t0.0\t2\t1 0 57\n",
      "134\t22\t0.0\t2\t0 0 22\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t149\t0.0\t2\t0 0 149\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t25\t0.0\t2\t0 0 25\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_049-LP3-ITS-0049_943_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_049-LP3-ITS-0049_943_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_049-LP3-ITS-0049_943_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_049-LP3-ITS-0049_943_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.327 s (10.152 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,228\n",
      "Reads with adapters:                       434 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,228 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,700,628 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,649,831 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 434 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.4%\n",
      "  C: 24.2%\n",
      "  G: 0.9%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t503.6\t0\t2\n",
      "4\t16\t125.9\t0\t16\n",
      "8\t1\t0.5\t0\t1\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "38\t1\t0.0\t2\t0 1\n",
      "45\t7\t0.0\t2\t0 0 7\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t9\t0.0\t2\t0 0 9\n",
      "67\t4\t0.0\t2\t0 0 4\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t220\t0.0\t2\t0 2 218\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "145\t21\t0.0\t2\t0 0 21\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t14\t0.0\t2\t0 0 14\n",
      "160\t13\t0.0\t2\t0 0 13\n",
      "198\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_050-LP3-ITS-0050_43_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_050-LP3-ITS-0050_43_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_050-LP3-ITS-0050_43_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_050-LP3-ITS-0050_43_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (29.164 ¬µs/read; 2.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     244\n",
      "Reads with adapters:                         5 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           244 (100.0%)\n",
      "\n",
      "Total basepairs processed:        73,444 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         72,882 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.0%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_051-LP3-ITS-0051_497_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_051-LP3-ITS-0051_497_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_051-LP3-ITS-0051_497_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_051-LP3-ITS-0051_497_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.396 s (10.262 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,547\n",
      "Reads with adapters:                       332 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,547 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,602,647 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,560,668 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 332 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.2%\n",
      "  C: 20.8%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t602.3\t0\t1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t189\t0.0\t2\t0 4 185\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t1\t0.0\t2\t0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t37\t0.0\t2\t0 0 37\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "150\t36\t0.0\t2\t0 0 36\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_052-LP3-ITS-0052_125_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_052-LP3-ITS-0052_125_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_052-LP3-ITS-0052_125_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_052-LP3-ITS-0052_125_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.383 s (10.037 ¬µs/read; 5.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,151\n",
      "Reads with adapters:                       440 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,151 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,483,451 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,426,743 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 440 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.3%\n",
      "  C: 22.0%\n",
      "  G: 0.2%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t596.1\t0\t1\n",
      "4\t1\t149.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t207\t0.0\t2\t0 4 203\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "133\t59\t0.0\t2\t0 1 58\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t10\t0.0\t2\t0 1 9\n",
      "142\t31\t0.0\t2\t0 0 31\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t13\t0.0\t2\t0 1 12\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t62\t0.0\t2\t0 0 62\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_053-LP3-ITS-0053_402_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_053-LP3-ITS-0053_402_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_053-LP3-ITS-0053_402_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_053-LP3-ITS-0053_402_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.361 s (10.060 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,909\n",
      "Reads with adapters:                       361 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,909 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,808,609 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,761,326 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 361 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.1%\n",
      "  C: 11.6%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "118\t155\t0.0\t2\t0 2 153\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t32\t0.0\t2\t0 1 31\n",
      "134\t24\t0.0\t2\t0 1 23\n",
      "137\t34\t0.0\t2\t0 1 33\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t82\t0.0\t2\t0 2 80\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_054-LP3-ITS-0054_551_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_054-LP3-ITS-0054_551_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_054-LP3-ITS-0054_551_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_054-LP3-ITS-0054_551_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.385 s (10.067 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,213\n",
      "Reads with adapters:                       409 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,502,113 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,452,591 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 409 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.1%\n",
      "  C: 24.2%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t597.1\t0\t6\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "45\t6\t0.0\t2\t0 0 6\n",
      "47\t6\t0.0\t2\t0 1 5\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t219\t0.0\t2\t0 5 214\n",
      "133\t37\t0.0\t2\t0 0 37\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t34\t0.0\t2\t0 1 33\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t35\t0.0\t2\t0 1 34\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_055-LP3-ITS-0055_237_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_055-LP3-ITS-0055_237_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_055-LP3-ITS-0055_237_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_055-LP3-ITS-0055_237_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.402 s (10.173 ¬µs/read; 5.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,493\n",
      "Reads with adapters:                       588 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,493 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,887,393 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,815,504 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 588 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.9%\n",
      "  C: 12.6%\n",
      "  G: 0.0%\n",
      "  T: 8.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t617.1\t0\t59\n",
      "5\t1\t38.6\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "118\t160\t0.0\t2\t0 2 158\n",
      "133\t41\t0.0\t2\t0 1 40\n",
      "137\t7\t0.0\t2\t0 1 6\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t121\t0.0\t2\t0 0 121\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t13\t0.0\t2\t0 0 13\n",
      "151\t141\t0.0\t2\t0 0 141\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_056-LP3-ITS-0056_433_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_056-LP3-ITS-0056_433_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_056-LP3-ITS-0056_433_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_056-LP3-ITS-0056_433_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.523 s (10.363 ¬µs/read; 5.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  50,513\n",
      "Reads with adapters:                       647 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        50,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,204,413 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,117,333 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 647 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.3%\n",
      "  C: 12.2%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t789.3\t0\t1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t9\t0.0\t2\t0 0 9\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "118\t201\t0.0\t2\t0 4 197\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "143\t125\t0.0\t2\t0 0 125\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t147\t0.0\t2\t0 0 147\n",
      "151\t46\t0.0\t2\t0 0 46\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t19\t0.0\t2\t0 0 19\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "164\t14\t0.0\t2\t0 0 14\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_057-LP3-ITS-0057_50_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_057-LP3-ITS-0057_50_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_057-LP3-ITS-0057_50_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_057-LP3-ITS-0057_50_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.996 s (9.950 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 100,109\n",
      "Reads with adapters:                     1,334 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):       100,109 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,132,809 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     29,966,553 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1334 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.1%\n",
      "  C: 39.2%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1564.2\t0\t3\n",
      "4\t3\t391.1\t0\t3\n",
      "10\t2\t0.1\t1\t0 2\n",
      "32\t18\t0.0\t2\t0 0 18\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "48\t12\t0.0\t2\t0 0 12\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t47\t0.0\t2\t0 0 47\n",
      "52\t9\t0.0\t2\t0 0 9\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t9\t0.0\t2\t0 0 9\n",
      "60\t9\t0.0\t2\t0 0 9\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t10\t0.0\t2\t0 0 10\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "108\t7\t0.0\t2\t0 0 7\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t465\t0.0\t2\t0 9 456\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t312\t0.0\t2\t0 0 312\n",
      "134\t10\t0.0\t2\t0 1 9\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t47\t0.0\t2\t0 0 47\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t21\t0.0\t2\t0 0 21\n",
      "145\t42\t0.0\t2\t0 0 42\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t47\t0.0\t2\t0 0 47\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t17\t0.0\t2\t0 0 17\n",
      "160\t176\t0.0\t2\t0 1 175\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "198\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_058-LP3-ITS-0058_154_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_058-LP3-ITS-0058_154_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_058-LP3-ITS-0058_154_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_058-LP3-ITS-0058_154_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.402 s (9.871 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,706\n",
      "Reads with adapters:                       312 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,252,506 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,212,910 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 312 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.2%\n",
      "  C: 28.5%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t636.0\t0\t1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "118\t162\t0.0\t2\t0 1 161\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t74\t0.0\t2\t0 0 74\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "151\t38\t0.0\t2\t0 0 38\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_059-LP3-ITS-0059_212_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_059-LP3-ITS-0059_212_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_059-LP3-ITS-0059_212_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_059-LP3-ITS-0059_212_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.389 s (10.134 ¬µs/read; 5.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,428\n",
      "Reads with adapters:                       411 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,566,828 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,518,553 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 411 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.3%\n",
      "  C: 17.8%\n",
      "  G: 0.0%\n",
      "  T: 2.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t600.4\t0\t17\n",
      "10\t6\t0.0\t1\t0 6\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "118\t217\t0.0\t2\t0 5 212\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "133\t30\t0.0\t2\t0 0 30\n",
      "134\t24\t0.0\t2\t0 0 24\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t18\t0.0\t2\t0 0 18\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t20\t0.0\t2\t0 0 20\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "160\t12\t0.0\t2\t0 0 12\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_060-LP3-ITS-0060_290_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_060-LP3-ITS-0060_290_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_060-LP3-ITS-0060_290_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_060-LP3-ITS-0060_290_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.510 s (9.945 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,296\n",
      "Reads with adapters:                       539 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,296 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,440,096 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,370,291 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 539 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.7%\n",
      "  C: 28.2%\n",
      "  G: 0.0%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t801.5\t0\t5\n",
      "4\t2\t200.4\t0\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "118\t197\t0.0\t2\t0 3 194\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t93\t0.0\t2\t0 0 93\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t42\t0.0\t2\t0 0 42\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t31\t0.0\t2\t0 1 30\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t87\t0.0\t2\t0 1 86\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_061-LP3-ITS-0061_630_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_061-LP3-ITS-0061_630_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_061-LP3-ITS-0061_630_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_061-LP3-ITS-0061_630_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (27.918 ¬µs/read; 2.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     269\n",
      "Reads with adapters:                        12 (4.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           269 (100.0%)\n",
      "\n",
      "Total basepairs processed:        80,969 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         80,283 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 33.3%\n",
      "  G: 8.3%\n",
      "  T: 8.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t4.2\t0\t5\n",
      "10\t2\t0.0\t1\t0 2\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_062-LP3-ITS-0062_61_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_062-LP3-ITS-0062_61_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_062-LP3-ITS-0062_61_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_062-LP3-ITS-0062_61_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.545 s (10.408 ¬µs/read; 5.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,371\n",
      "Reads with adapters:                    10,935 (20.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,371 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,763,671 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,697,866 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10935 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.4%\n",
      "  C: 60.5%\n",
      "  G: 1.8%\n",
      "  T: 19.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9043\t818.3\t0\t9043\n",
      "4\t3\t204.6\t0\t3\n",
      "6\t1\t12.8\t0\t1\n",
      "8\t3\t0.8\t0\t3\n",
      "9\t148\t0.2\t0\t136 12\n",
      "10\t1568\t0.0\t1\t76 1492\n",
      "11\t3\t0.0\t1\t0 3\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t85\t0.0\t2\t0 5 80\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t23\t0.0\t2\t0 2 21\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "212\t8\t0.0\t2\t0 0 8\n",
      "216\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_063-LP3-ITS-0063_105_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_063-LP3-ITS-0063_105_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_063-LP3-ITS-0063_105_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_063-LP3-ITS-0063_105_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.397 s (10.620 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,418\n",
      "Reads with adapters:                     8,159 (21.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,418 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,262,818 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,211,107 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8159 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.8%\n",
      "  C: 57.3%\n",
      "  G: 1.8%\n",
      "  T: 18.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6447\t584.7\t0\t6447\n",
      "4\t1\t146.2\t0\t1\n",
      "6\t1\t9.1\t0\t1\n",
      "8\t5\t0.6\t0\t5\n",
      "9\t126\t0.1\t0\t117 9\n",
      "10\t1460\t0.0\t1\t95 1365\n",
      "11\t3\t0.0\t1\t0 3\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "118\t43\t0.0\t2\t0 2 41\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "145\t4\t0.0\t2\t0 1 3\n",
      "151\t40\t0.0\t2\t0 0 40\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "175\t13\t0.0\t2\t0 0 13\n",
      "219\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_064-LP3-ITS-0064_341_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_064-LP3-ITS-0064_341_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_064-LP3-ITS-0064_341_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_064-LP3-ITS-0064_341_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.326 s (10.213 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,885\n",
      "Reads with adapters:                     1,170 (3.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,885 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,597,385 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,548,934 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1170 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.0%\n",
      "  C: 45.8%\n",
      "  G: 1.4%\n",
      "  T: 17.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t669\t498.2\t0\t669\n",
      "8\t3\t0.5\t0\t3\n",
      "9\t23\t0.1\t0\t21 2\n",
      "10\t132\t0.0\t1\t5 127\n",
      "11\t1\t0.0\t1\t0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "69\t3\t0.0\t2\t0 0 3\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t143\t0.0\t2\t0 3 140\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t73\t0.0\t2\t0 0 73\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t29\t0.0\t2\t0 0 29\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t33\t0.0\t2\t0 0 33\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t20\t0.0\t2\t0 0 20\n",
      "160\t14\t0.0\t2\t0 0 14\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_065-LP3-ITS-0065_826_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_065-LP3-ITS-0065_826_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_065-LP3-ITS-0065_826_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_065-LP3-ITS-0065_826_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.516 s (9.993 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,633\n",
      "Reads with adapters:                       633 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,633 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,541,533 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,460,878 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 633 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.8%\n",
      "  C: 26.9%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t201.7\t0\t1\n",
      "6\t3\t12.6\t0\t3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "59\t17\t0.0\t2\t0 0 17\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t9\t0.0\t2\t0 0 9\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t279\t0.0\t2\t0 3 276\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t76\t0.0\t2\t0 0 76\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t25\t0.0\t2\t0 0 25\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t32\t0.0\t2\t0 0 32\n",
      "160\t95\t0.0\t2\t0 2 93\n",
      "164\t40\t0.0\t2\t0 0 40\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_066-LP3-ITS-0066_928_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_066-LP3-ITS-0066_928_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_066-LP3-ITS-0066_928_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_066-LP3-ITS-0066_928_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.402 s (9.902 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,583\n",
      "Reads with adapters:                       322 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,583 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,215,483 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,178,732 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 322 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.7%\n",
      "  C: 35.7%\n",
      "  G: 0.6%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t634.1\t0\t12\n",
      "4\t6\t158.5\t0\t6\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "45\t3\t0.0\t2\t0 1 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t9\t0.0\t2\t0 0 9\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t136\t0.0\t2\t1 2 133\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t54\t0.0\t2\t0 0 54\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t15\t0.0\t2\t0 1 14\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "160\t19\t0.0\t2\t0 1 18\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t2\t0.0\t2\t0 0 2\n",
      "198\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_067-LP3-ITS-0067_217_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_067-LP3-ITS-0067_217_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_067-LP3-ITS-0067_217_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_067-LP3-ITS-0067_217_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.340 s (10.069 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,772\n",
      "Reads with adapters:                       369 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,772 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,165,372 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,126,165 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 369 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.1%\n",
      "  C: 17.3%\n",
      "  G: 0.8%\n",
      "  T: 6.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t527.7\t0\t53\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t16\t0.0\t1\t0 16\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t137\t0.0\t2\t0 2 135\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t28\t0.0\t2\t0 0 28\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t22\t0.0\t2\t0 1 21\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t43\t0.0\t2\t0 1 42\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_068-LP3-ITS-0068_849_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_068-LP3-ITS-0068_849_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_068-LP3-ITS-0068_849_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_068-LP3-ITS-0068_849_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.453 s (10.021 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,217\n",
      "Reads with adapters:                       512 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,217 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,610,317 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,547,877 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 512 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.8%\n",
      "  C: 22.9%\n",
      "  G: 0.6%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t706.5\t0\t3\n",
      "4\t1\t176.6\t0\t1\n",
      "5\t1\t44.2\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "47\t7\t0.0\t2\t0 0 7\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t7\t0.0\t2\t0 0 7\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t275\t0.0\t2\t0 7 268\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t62\t0.0\t2\t0 0 62\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t23\t0.0\t2\t0 0 23\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t17\t0.0\t2\t0 0 17\n",
      "145\t57\t0.0\t2\t0 1 56\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "158\t7\t0.0\t2\t0 0 7\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_069-LP3-ITS-0069_390_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_069-LP3-ITS-0069_390_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_069-LP3-ITS-0069_390_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_069-LP3-ITS-0069_390_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.410 s (9.854 ¬µs/read; 6.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,589\n",
      "Reads with adapters:                       402 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,518,289 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,468,578 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 402 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.4%\n",
      "  C: 24.4%\n",
      "  G: 0.2%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t649.8\t0\t1\n",
      "4\t1\t162.5\t0\t1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t8\t0.0\t2\t0 0 8\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t6\t0.0\t2\t0 0 6\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t248\t0.0\t2\t0 8 240\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t56\t0.0\t2\t0 0 56\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t38\t0.0\t2\t0 1 37\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_070-LP3-ITS-0070_915_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_070-LP3-ITS-0070_915_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_070-LP3-ITS-0070_915_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_070-LP3-ITS-0070_915_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.284 s (9.905 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,662\n",
      "Reads with adapters:                       291 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,662 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,627,262 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,593,254 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 291 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.9%\n",
      "  C: 21.6%\n",
      "  G: 1.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t447.8\t0\t19\n",
      "4\t6\t112.0\t0\t6\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t6\t0.0\t2\t0 0 6\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t145\t0.0\t2\t0 5 140\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t29\t0.0\t2\t0 1 28\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t16\t0.0\t2\t0 0 16\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "160\t38\t0.0\t2\t0 0 38\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_071-LP3-ITS-0071_1089_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_071-LP3-ITS-0071_1089_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_071-LP3-ITS-0071_1089_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_071-LP3-ITS-0071_1089_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (34.348 ¬µs/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     221\n",
      "Reads with adapters:                        26 (11.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           221 (100.0%)\n",
      "\n",
      "Total basepairs processed:        66,521 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         65,224 bp (98.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 26 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.5%\n",
      "  C: 53.8%\n",
      "  G: 0.0%\n",
      "  T: 7.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t3.5\t0\t14\n",
      "9\t1\t0.0\t0\t0 1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t8\t0.0\t2\t0 0 8\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_072-LP3-ITS-0072_589_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_072-LP3-ITS-0072_589_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_072-LP3-ITS-0072_589_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_072-LP3-ITS-0072_589_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (50.467 ¬µs/read; 1.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     122\n",
      "Reads with adapters:                         4 (3.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           122 (100.0%)\n",
      "\n",
      "Total basepairs processed:        36,722 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         36,399 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 25.0%\n",
      "  G: 0.0%\n",
      "  T: 25.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.9\t0\t1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_073-LP3-ITS-0073_809_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_073-LP3-ITS-0073_809_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_073-LP3-ITS-0073_809_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_073-LP3-ITS-0073_809_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.830 s (9.923 ¬µs/read; 6.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  83,620\n",
      "Reads with adapters:                       917 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        83,620 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,169,620 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     25,051,994 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 917 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.1%\n",
      "  C: 17.7%\n",
      "  G: 0.1%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1306.6\t0\t6\n",
      "4\t1\t326.6\t0\t1\n",
      "10\t4\t0.1\t1\t0 4\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "58\t12\t0.0\t2\t0 0 12\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t4\t0.0\t2\t0 1 3\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "84\t14\t0.0\t2\t0 0 14\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "96\t9\t0.0\t2\t0 0 9\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "118\t398\t0.0\t2\t0 5 393\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "130\t8\t0.0\t2\t0 0 8\n",
      "133\t68\t0.0\t2\t0 0 68\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t160\t0.0\t2\t0 3 157\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t35\t0.0\t2\t0 0 35\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t40\t0.0\t2\t0 0 40\n",
      "157\t2\t0.0\t2\t0 1 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t96\t0.0\t2\t0 0 96\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_074-LP3-ITS-0074_1032_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_074-LP3-ITS-0074_1032_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_074-LP3-ITS-0074_1032_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_074-LP3-ITS-0074_1032_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.362 s (10.009 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,213\n",
      "Reads with adapters:                       285 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,900,113 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,860,804 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 285 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.1%\n",
      "  C: 17.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t137\t0.0\t2\t0 4 133\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t31\t0.0\t2\t0 0 31\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t32\t0.0\t2\t0 1 31\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "164\t10\t0.0\t2\t0 0 10\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "234\t14\t0.0\t2\t0 0 14\n",
      "235\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_075-LP3-ITS-0075_739_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_075-LP3-ITS-0075_739_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_075-LP3-ITS-0075_739_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_075-LP3-ITS-0075_739_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.565 s (9.803 ¬µs/read; 6.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  57,670\n",
      "Reads with adapters:                       544 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        57,670 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,358,670 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,293,707 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 544 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.1%\n",
      "  C: 30.3%\n",
      "  G: 0.4%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t901.1\t0\t1\n",
      "5\t1\t56.3\t0\t1\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "32\t8\t0.0\t2\t0 0 8\n",
      "43\t5\t0.0\t2\t0 0 5\n",
      "49\t5\t0.0\t2\t0 1 4\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t15\t0.0\t2\t0 0 15\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "118\t280\t0.0\t2\t0 5 275\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t71\t0.0\t2\t0 2 69\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t27\t0.0\t2\t0 0 27\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t37\t0.0\t2\t0 1 36\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t40\t0.0\t2\t0 2 38\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_076-LP3-ITS-0076_522_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_076-LP3-ITS-0076_522_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_076-LP3-ITS-0076_522_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_076-LP3-ITS-0076_522_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (41.138 ¬µs/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     160\n",
      "Reads with adapters:                         3 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           160 (100.0%)\n",
      "\n",
      "Total basepairs processed:        48,160 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         47,805 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.5\t0\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_077-LP3-ITS-0077_967_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_077-LP3-ITS-0077_967_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_077-LP3-ITS-0077_967_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_077-LP3-ITS-0077_967_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.828 s (9.976 ¬µs/read; 6.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  83,031\n",
      "Reads with adapters:                       828 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        83,031 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,992,331 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     24,884,227 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 828 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.3%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1297.4\t0\t5\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 1 4\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "99\t5\t0.0\t2\t0 0 5\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t333\t0.0\t2\t0 6 327\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t13\t0.0\t2\t0 0 13\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "133\t134\t0.0\t2\t0 3 131\n",
      "134\t34\t0.0\t2\t0 0 34\n",
      "137\t24\t0.0\t2\t0 1 23\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t38\t0.0\t2\t0 0 38\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t31\t0.0\t2\t0 0 31\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t37\t0.0\t2\t0 0 37\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t58\t0.0\t2\t0 2 56\n",
      "160\t68\t0.0\t2\t0 3 65\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_078-LP3-ITS-0078_806_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_078-LP3-ITS-0078_806_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_078-LP3-ITS-0078_806_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_078-LP3-ITS-0078_806_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.470 s (9.949 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  47,260\n",
      "Reads with adapters:                       538 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        47,260 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,225,260 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,166,138 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 538 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.6%\n",
      "  C: 16.7%\n",
      "  G: 0.2%\n",
      "  T: 4.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t60\t738.4\t0\t60\n",
      "9\t2\t0.2\t0\t2\n",
      "10\t17\t0.0\t1\t1 16\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t280\t0.0\t2\t0 6 274\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t34\t0.0\t2\t0 0 34\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t8\t0.0\t2\t0 1 7\n",
      "151\t95\t0.0\t2\t0 1 94\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_079-LP3-ITS-0079_526_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_079-LP3-ITS-0079_526_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_079-LP3-ITS-0079_526_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_079-LP3-ITS-0079_526_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.345 s (10.019 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,389\n",
      "Reads with adapters:                       397 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,389 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,351,089 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,301,869 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 397 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.4%\n",
      "  C: 16.1%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t537.3\t0\t6\n",
      "10\t3\t0.0\t1\t0 3\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "81\t3\t0.0\t2\t0 0 3\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t233\t0.0\t2\t0 3 230\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t48\t0.0\t2\t0 0 48\n",
      "145\t26\t0.0\t2\t0 0 26\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "198\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_080-LP3-ITS-0080_535_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_080-LP3-ITS-0080_535_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_080-LP3-ITS-0080_535_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_080-LP3-ITS-0080_535_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (35.727 ¬µs/read; 1.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     187\n",
      "Reads with adapters:                         5 (2.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           187 (100.0%)\n",
      "\n",
      "Total basepairs processed:        56,287 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         55,809 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.0%\n",
      "  C: 40.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_081-LP3-ITS-0081_85_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_081-LP3-ITS-0081_85_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_081-LP3-ITS-0081_85_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_081-LP3-ITS-0081_85_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.659 s (9.873 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  66,712\n",
      "Reads with adapters:                       848 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        66,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,080,312 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     19,965,582 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 848 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.7%\n",
      "  C: 17.9%\n",
      "  G: 0.2%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1042.4\t0\t7\n",
      "10\t2\t0.1\t1\t1 1\n",
      "51\t16\t0.0\t2\t0 0 16\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t4\t0.0\t2\t0 0 4\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t240\t0.0\t2\t0 0 240\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t59\t0.0\t2\t0 1 58\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "145\t13\t0.0\t2\t0 1 12\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t408\t0.0\t2\t0 2 406\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t33\t0.0\t2\t0 0 33\n",
      "160\t13\t0.0\t2\t0 0 13\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_082-LP3-ITS-0082_1002_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_082-LP3-ITS-0082_1002_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_082-LP3-ITS-0082_1002_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_082-LP3-ITS-0082_1002_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.579 s (10.040 ¬µs/read; 5.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  57,694\n",
      "Reads with adapters:                       534 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        57,694 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,365,894 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,298,797 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 534 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.7%\n",
      "  C: 26.0%\n",
      "  G: 0.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t901.5\t0\t15\n",
      "9\t3\t0.2\t0\t3\n",
      "16\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t5\t0.0\t2\t0 0 5\n",
      "93\t11\t0.0\t2\t0 0 11\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t227\t0.0\t2\t0 5 222\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t59\t0.0\t2\t0 0 59\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t27\t0.0\t2\t0 0 27\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t20\t0.0\t2\t0 0 20\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t31\t0.0\t2\t0 1 30\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t30\t0.0\t2\t0 1 29\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t34\t0.0\t2\t0 0 34\n",
      "164\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_083-LP3-ITS-0083_669_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_083-LP3-ITS-0083_669_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_083-LP3-ITS-0083_669_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_083-LP3-ITS-0083_669_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.679 s (9.958 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  68,158\n",
      "Reads with adapters:                       679 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        68,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,515,558 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     20,421,644 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 679 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.2%\n",
      "  C: 17.5%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1065.0\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t160\t0.0\t2\t0 1 159\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "133\t73\t0.0\t2\t0 1 72\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "137\t24\t0.0\t2\t0 0 24\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t156\t0.0\t2\t0 3 153\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "150\t25\t0.0\t2\t0 1 24\n",
      "151\t169\t0.0\t2\t0 1 168\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t10\t0.0\t2\t0 0 10\n",
      "158\t10\t0.0\t2\t0 0 10\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_084-LP3-ITS-0084_463_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_084-LP3-ITS-0084_463_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_084-LP3-ITS-0084_463_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_084-LP3-ITS-0084_463_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.465 s (9.921 ¬µs/read; 6.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,902\n",
      "Reads with adapters:                       492 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,902 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,117,502 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,060,280 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 492 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.6%\n",
      "  C: 21.3%\n",
      "  G: 1.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t732.8\t0\t3\n",
      "11\t1\t0.0\t1\t0 1\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "58\t44\t0.0\t2\t0 0 44\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t267\t0.0\t2\t0 5 262\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t45\t0.0\t2\t0 1 44\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t16\t0.0\t2\t0 0 16\n",
      "143\t10\t0.0\t2\t0 0 10\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t14\t0.0\t2\t0 0 14\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t29\t0.0\t2\t0 0 29\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_085-LP3-ITS-0085_249_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_085-LP3-ITS-0085_249_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_085-LP3-ITS-0085_249_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_085-LP3-ITS-0085_249_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.527 s (9.886 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,314\n",
      "Reads with adapters:                       510 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,047,514 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,983,690 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 510 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.6%\n",
      "  C: 21.8%\n",
      "  G: 0.2%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t833.0\t0\t8\n",
      "10\t3\t0.1\t1\t0 3\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t230\t0.0\t2\t0 2 228\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "130\t5\t0.0\t2\t0 0 5\n",
      "133\t79\t0.0\t2\t0 2 77\n",
      "134\t48\t0.0\t2\t0 0 48\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t38\t0.0\t2\t0 0 38\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t44\t0.0\t2\t0 1 43\n",
      "151\t24\t0.0\t2\t0 0 24\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_086-LP3-ITS-0086_107_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_086-LP3-ITS-0086_107_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_086-LP3-ITS-0086_107_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_086-LP3-ITS-0086_107_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.577 s (9.996 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  57,741\n",
      "Reads with adapters:                       573 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        57,741 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,380,041 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,306,051 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 573 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.6%\n",
      "  C: 26.5%\n",
      "  G: 0.2%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t902.2\t0\t7\n",
      "10\t3\t0.1\t1\t0 3\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "32\t7\t0.0\t2\t0 0 7\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t5\t0.0\t2\t0 0 5\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t7\t0.0\t2\t0 0 7\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t177\t0.0\t2\t0 1 176\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t68\t0.0\t2\t0 0 68\n",
      "134\t75\t0.0\t2\t0 1 74\n",
      "137\t37\t0.0\t2\t0 0 37\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t44\t0.0\t2\t0 0 44\n",
      "151\t19\t0.0\t2\t0 0 19\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t19\t0.0\t2\t0 0 19\n",
      "160\t48\t0.0\t2\t0 0 48\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_087-LP3-ITS-0087_55_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_087-LP3-ITS-0087_55_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_087-LP3-ITS-0087_55_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_087-LP3-ITS-0087_55_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.659 s (10.239 ¬µs/read; 5.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  64,357\n",
      "Reads with adapters:                     1,579 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        64,357 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,371,457 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     19,266,775 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1579 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.2%\n",
      "  C: 33.6%\n",
      "  G: 1.1%\n",
      "  T: 13.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t671\t1005.6\t0\t671\n",
      "4\t1\t251.4\t0\t1\n",
      "8\t2\t1.0\t0\t2\n",
      "9\t20\t0.2\t0\t18 2\n",
      "10\t168\t0.1\t1\t9 159\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t132\t0.0\t2\t0 0 132\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t12\t0.0\t2\t0 0 12\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "134\t72\t0.0\t2\t0 0 72\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t143\t0.0\t2\t0 1 142\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t17\t0.0\t2\t0 0 17\n",
      "151\t164\t0.0\t2\t0 5 159\n",
      "155\t31\t0.0\t2\t0 0 31\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t54\t0.0\t2\t0 0 54\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "166\t4\t0.0\t2\t0 0 4\n",
      "175\t3\t0.0\t2\t0 0 3\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_088-LP3-ITS-0088_68_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_088-LP3-ITS-0088_68_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_088-LP3-ITS-0088_68_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_088-LP3-ITS-0088_68_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.440 s (10.599 ¬µs/read; 5.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,512\n",
      "Reads with adapters:                       415 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,512 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,495,112 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,439,713 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 415 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.7%\n",
      "  C: 18.1%\n",
      "  G: 0.2%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t648.6\t0\t4\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "84\t13\t0.0\t2\t0 0 13\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t117\t0.0\t2\t0 2 115\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t39\t0.0\t2\t0 2 37\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t24\t0.0\t2\t0 0 24\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "146\t4\t0.0\t2\t0 0 4\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "151\t124\t0.0\t2\t0 1 123\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "164\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_089-LP3-ITS-0089_804_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_089-LP3-ITS-0089_804_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_089-LP3-ITS-0089_804_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_089-LP3-ITS-0089_804_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.801 s (9.995 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  80,158\n",
      "Reads with adapters:                     1,127 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        80,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,127,558 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     24,086,124 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1127 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.2%\n",
      "  C: 11.5%\n",
      "  G: 0.3%\n",
      "  T: 14.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t312\t1252.5\t0\t312\n",
      "4\t6\t313.1\t0\t6\n",
      "5\t2\t78.3\t0\t2\n",
      "8\t4\t1.2\t0\t4\n",
      "9\t499\t0.3\t0\t471 28\n",
      "10\t14\t0.1\t1\t1 13\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "101\t6\t0.0\t2\t0 0 6\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t165\t0.0\t2\t0 0 165\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t38\t0.0\t2\t0 0 38\n",
      "134\t8\t0.0\t2\t0 2 6\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t24\t0.0\t2\t0 0 24\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_090-LP3-ITS-0090_970_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_090-LP3-ITS-0090_970_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_090-LP3-ITS-0090_970_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_090-LP3-ITS-0090_970_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.403 s (10.102 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,911\n",
      "Reads with adapters:                     3,513 (8.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,911 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,013,211 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,982,610 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3513 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.6%\n",
      "  C: 49.4%\n",
      "  G: 1.2%\n",
      "  T: 25.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2716\t623.6\t0\t2716\n",
      "4\t1\t155.9\t0\t1\n",
      "7\t7\t2.4\t0\t7\n",
      "8\t61\t0.6\t0\t61\n",
      "9\t168\t0.2\t0\t157 11\n",
      "10\t454\t0.0\t1\t25 429\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t57\t0.0\t2\t0 1 56\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t4\t0.0\t2\t0 0 4\n",
      "234\t19\t0.0\t2\t0 0 19\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_091-LP3-ITS-0091_1106_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_091-LP3-ITS-0091_1106_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_091-LP3-ITS-0091_1106_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_091-LP3-ITS-0091_1106_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.833 s (10.145 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  82,135\n",
      "Reads with adapters:                       774 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        82,135 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,722,635 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     24,617,898 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 774 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.4%\n",
      "  C: 21.7%\n",
      "  G: 0.1%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t1283.4\t0\t10\n",
      "10\t1\t0.1\t1\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t162\t0.0\t2\t0 3 159\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t117\t0.0\t2\t0 0 117\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t67\t0.0\t2\t0 0 67\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t188\t0.0\t2\t0 4 184\n",
      "151\t50\t0.0\t2\t0 2 48\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "158\t8\t0.0\t2\t0 0 8\n",
      "160\t110\t0.0\t2\t0 4 106\n",
      "198\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_092-LP3-ITS-0092_1093_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_092-LP3-ITS-0092_1093_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_092-LP3-ITS-0092_1093_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_092-LP3-ITS-0092_1093_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.567 s (10.009 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  56,600\n",
      "Reads with adapters:                       565 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        56,600 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,036,600 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,967,760 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 565 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 55.6%\n",
      "  C: 43.7%\n",
      "  G: 0.4%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t884.4\t0\t2\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t6\t0.0\t2\t0 1 5\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "59\t10\t0.0\t2\t0 0 10\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "72\t1\t0.0\t2\t0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t8\t0.0\t2\t0 0 8\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t67\t0.0\t2\t0 2 65\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t13\t0.0\t2\t0 1 12\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t164\t0.0\t2\t0 1 163\n",
      "119\t5\t0.0\t2\t0 0 5\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t99\t0.0\t2\t0 0 99\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t32\t0.0\t2\t0 1 31\n",
      "142\t33\t0.0\t2\t0 0 33\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t8\t0.0\t2\t0 0 8\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t36\t0.0\t2\t0 2 34\n",
      "153\t8\t0.0\t2\t0 1 7\n",
      "155\t16\t0.0\t2\t0 1 15\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_093-LP3-ITS-0093_665_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_093-LP3-ITS-0093_665_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_093-LP3-ITS-0093_665_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_093-LP3-ITS-0093_665_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.617 s (9.827 ¬µs/read; 6.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  62,742\n",
      "Reads with adapters:                       505 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        62,742 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,885,342 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,820,422 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 505 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.1%\n",
      "  C: 31.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t980.3\t0\t2\n",
      "10\t1\t0.1\t1\t0 1\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "52\t5\t0.0\t2\t0 0 5\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t206\t0.0\t2\t0 1 205\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t138\t0.0\t2\t0 3 135\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t37\t0.0\t2\t0 0 37\n",
      "151\t66\t0.0\t2\t0 0 66\n",
      "155\t4\t0.0\t2\t0 1 3\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 1 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_094-LP3-ITS-0094_773_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_094-LP3-ITS-0094_773_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_094-LP3-ITS-0094_773_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_094-LP3-ITS-0094_773_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.634 s (10.032 ¬µs/read; 5.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  63,243\n",
      "Reads with adapters:                       682 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        63,243 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,036,143 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,976,498 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 682 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.4%\n",
      "  C: 22.3%\n",
      "  G: 0.4%\n",
      "  T: 5.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t101\t988.2\t0\t101\n",
      "9\t101\t0.2\t0\t95 6\n",
      "10\t4\t0.1\t1\t0 4\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "58\t8\t0.0\t2\t0 0 8\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t286\t0.0\t2\t0 5 281\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t74\t0.0\t2\t0 0 74\n",
      "134\t26\t0.0\t2\t0 1 25\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "151\t13\t0.0\t2\t0 0 13\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_095-LP3-ITS-0095_437_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_095-LP3-ITS-0095_437_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_095-LP3-ITS-0095_437_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_095-LP3-ITS-0095_437_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.827 s (10.160 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  81,439\n",
      "Reads with adapters:                       931 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        81,439 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,513,139 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     24,385,880 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 931 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.1%\n",
      "  C: 16.1%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t1272.5\t0\t12\n",
      "10\t5\t0.1\t1\t0 5\n",
      "12\t1\t0.0\t1\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "55\t1\t0.0\t2\t0 1\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t169\t0.0\t2\t0 0 169\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "133\t73\t0.0\t2\t0 0 73\n",
      "134\t29\t0.0\t2\t0 0 29\n",
      "137\t53\t0.0\t2\t0 0 53\n",
      "142\t37\t0.0\t2\t0 0 37\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t20\t0.0\t2\t0 0 20\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t86\t0.0\t2\t0 1 85\n",
      "151\t301\t0.0\t2\t0 4 297\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t6\t0.0\t2\t0 0 6\n",
      "155\t24\t0.0\t2\t0 1 23\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t54\t0.0\t2\t0 0 54\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "194\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_096-LP3-ITS-0096_741_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_096-LP3-ITS-0096_741_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_096-LP3-ITS-0096_741_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_096-LP3-ITS-0096_741_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.515 s (9.649 ¬µs/read; 6.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,326\n",
      "Reads with adapters:                       386 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,051,126 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,003,726 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 386 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.4%\n",
      "  C: 24.6%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "60\t11\t0.0\t2\t0 0 11\n",
      "67\t16\t0.0\t2\t0 0 16\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "111\t5\t0.0\t2\t0 0 5\n",
      "118\t198\t0.0\t2\t0 3 195\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t45\t0.0\t2\t0 0 45\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t25\t0.0\t2\t0 1 24\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_097-LP3-ITS-0097_351_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_097-LP3-ITS-0097_351_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_097-LP3-ITS-0097_351_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_097-LP3-ITS-0097_351_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.530 s (9.967 ¬µs/read; 6.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,217\n",
      "Reads with adapters:                       523 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,217 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,018,317 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,949,346 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 523 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.5%\n",
      "  C: 14.9%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t831.5\t0\t7\n",
      "4\t1\t207.9\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "26\t3\t0.0\t2\t0 0 3\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "36\t3\t0.0\t2\t0 0 3\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t178\t0.0\t2\t0 3 175\n",
      "133\t34\t0.0\t2\t0 1 33\n",
      "134\t17\t0.0\t2\t0 0 17\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t25\t0.0\t2\t0 0 25\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t118\t0.0\t2\t0 4 114\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t73\t0.0\t2\t1 1 71\n",
      "181\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_098-LP3-ITS-0098_662_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_098-LP3-ITS-0098_662_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_098-LP3-ITS-0098_662_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_098-LP3-ITS-0098_662_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.426 s (9.347 ¬µs/read; 6.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,523\n",
      "Reads with adapters:                     1,888 (4.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,523 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,702,423 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,647,880 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1888 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.8%\n",
      "  C: 6.9%\n",
      "  G: 0.1%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1522\t711.3\t0\t1522\n",
      "4\t5\t177.8\t0\t5\n",
      "6\t1\t11.1\t0\t1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t100\t0.0\t2\t1 2 97\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t23\t0.0\t2\t0 0 23\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 1 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t180\t0.0\t2\t0 0 180\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t5\t0.0\t2\t0 1 4\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "176\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_099-LP3-ITS-0099_188_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_099-LP3-ITS-0099_188_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_099-LP3-ITS-0099_188_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_099-LP3-ITS-0099_188_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.688 s (9.894 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  69,491\n",
      "Reads with adapters:                       628 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        69,491 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,916,791 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     20,833,642 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 628 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.5%\n",
      "  C: 13.2%\n",
      "  G: 0.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t4\t271.4\t0\t4\n",
      "42\t3\t0.0\t2\t0 0 3\n",
      "46\t12\t0.0\t2\t0 0 12\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "118\t221\t0.0\t2\t0 2 219\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t47\t0.0\t2\t0 1 46\n",
      "134\t30\t0.0\t2\t0 0 30\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t65\t0.0\t2\t0 2 63\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t142\t0.0\t2\t0 1 141\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_100-LP3-ITS-0100_493_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_100-LP3-ITS-0100_493_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_100-LP3-ITS-0100_493_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_100-LP3-ITS-0100_493_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.664 s (9.889 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  67,115\n",
      "Reads with adapters:                       733 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        67,115 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,201,615 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     20,110,643 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 733 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.4%\n",
      "  C: 18.1%\n",
      "  G: 0.3%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1048.7\t0\t3\n",
      "5\t1\t65.5\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t3\t0.0\t1\t0 3\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "32\t6\t0.0\t2\t0 0 6\n",
      "58\t16\t0.0\t2\t0 0 16\n",
      "59\t11\t0.0\t2\t0 0 11\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "67\t6\t0.0\t2\t0 0 6\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t382\t0.0\t2\t1 10 371\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t67\t0.0\t2\t1 2 64\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t22\t0.0\t2\t0 0 22\n",
      "143\t13\t0.0\t2\t0 0 13\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t16\t0.0\t2\t0 0 16\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t48\t0.0\t2\t0 0 48\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t66\t0.0\t2\t0 1 65\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_101-LP3-ITS-0101_1090_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_101-LP3-ITS-0101_1090_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_101-LP3-ITS-0101_1090_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_101-LP3-ITS-0101_1090_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.535 s (10.034 ¬µs/read; 5.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,314\n",
      "Reads with adapters:                       545 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,314 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,047,514 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,987,144 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 545 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 61.8%\n",
      "  C: 37.2%\n",
      "  G: 0.2%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t833.0\t0\t6\n",
      "9\t2\t0.2\t0\t1 1\n",
      "26\t3\t0.0\t2\t0 0 3\n",
      "51\t10\t0.0\t2\t0 0 10\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "59\t80\t0.0\t2\t0 3 77\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "118\t263\t0.0\t2\t0 6 257\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t69\t0.0\t2\t0 0 69\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t17\t0.0\t2\t0 0 17\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t22\t0.0\t2\t0 1 21\n",
      "157\t11\t0.0\t2\t0 0 11\n",
      "160\t20\t0.0\t2\t0 1 19\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_102-LP3-ITS-0102_664_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_102-LP3-ITS-0102_664_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_102-LP3-ITS-0102_664_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_102-LP3-ITS-0102_664_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.414 s (9.898 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,829\n",
      "Reads with adapters:                       407 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,829 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,590,529 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,541,547 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 407 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.8%\n",
      "  C: 30.0%\n",
      "  G: 0.0%\n",
      "  T: 2.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t653.6\t0\t5\n",
      "4\t5\t163.4\t0\t5\n",
      "5\t1\t40.8\t0\t1\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "52\t7\t0.0\t2\t0 1 6\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t192\t0.0\t2\t0 3 189\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "131\t6\t0.0\t2\t0 0 6\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t72\t0.0\t2\t0 0 72\n",
      "134\t12\t0.0\t2\t0 0 12\n",
      "137\t3\t0.0\t2\t0 1 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t46\t0.0\t2\t0 0 46\n",
      "151\t13\t0.0\t2\t0 1 12\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_103-LP3-ITS-0103_185_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_103-LP3-ITS-0103_185_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_103-LP3-ITS-0103_185_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_103-LP3-ITS-0103_185_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.977 s (9.753 ¬µs/read; 6.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 100,170\n",
      "Reads with adapters:                       953 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):       100,170 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,151,170 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     30,025,120 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 953 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.2%\n",
      "  C: 18.6%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1565.2\t0\t8\n",
      "4\t2\t391.3\t0\t2\n",
      "10\t2\t0.1\t1\t1 1\n",
      "51\t11\t0.0\t2\t0 0 11\n",
      "52\t6\t0.0\t2\t0 0 6\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t329\t0.0\t2\t0 5 324\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t16\t0.0\t2\t0 0 16\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "132\t51\t0.0\t2\t0 1 50\n",
      "133\t50\t0.0\t2\t0 0 50\n",
      "134\t11\t0.0\t2\t0 1 10\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t30\t0.0\t2\t0 1 29\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t6\t0.0\t2\t0 0 6\n",
      "150\t100\t0.0\t2\t0 0 100\n",
      "151\t226\t0.0\t2\t0 1 225\n",
      "155\t18\t0.0\t2\t0 1 17\n",
      "160\t45\t0.0\t2\t0 2 43\n",
      "164\t2\t0.0\t2\t0 1 1\n",
      "188\t2\t0.0\t2\t0 0 2\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_104-LP3-ITS-0104_196_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_104-LP3-ITS-0104_196_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_104-LP3-ITS-0104_196_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_104-LP3-ITS-0104_196_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.394 s (10.194 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,661\n",
      "Reads with adapters:                       465 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,661 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,636,961 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,575,766 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 465 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.0%\n",
      "  C: 22.6%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t604.1\t0\t2\n",
      "4\t4\t151.0\t0\t4\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "84\t6\t0.0\t2\t0 0 6\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t6\t0.0\t2\t0 0 6\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t108\t0.0\t2\t0 4 104\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t58\t0.0\t2\t0 1 57\n",
      "134\t14\t0.0\t2\t0 0 14\n",
      "137\t22\t0.0\t2\t0 0 22\n",
      "143\t31\t0.0\t2\t0 0 31\n",
      "145\t128\t0.0\t2\t0 1 127\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t14\t0.0\t2\t0 1 13\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_105-LP3-ITS-0105_721_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_105-LP3-ITS-0105_721_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_105-LP3-ITS-0105_721_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_105-LP3-ITS-0105_721_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.638 s (10.523 ¬µs/read; 5.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  60,598\n",
      "Reads with adapters:                       713 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        60,598 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,239,998 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,143,683 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 713 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.0%\n",
      "  C: 11.9%\n",
      "  G: 0.0%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t946.8\t0\t8\n",
      "7\t1\t3.7\t0\t1\n",
      "10\t5\t0.1\t1\t0 5\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t192\t0.0\t2\t0 2 190\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t42\t0.0\t2\t0 2 40\n",
      "134\t14\t0.0\t2\t0 0 14\n",
      "137\t98\t0.0\t2\t0 0 98\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t116\t0.0\t2\t0 2 114\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t51\t0.0\t2\t0 0 51\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t110\t0.0\t2\t0 1 109\n",
      "164\t19\t0.0\t2\t0 0 19\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_106-LP3-ITS-0106_631_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_106-LP3-ITS-0106_631_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_106-LP3-ITS-0106_631_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_106-LP3-ITS-0106_631_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.707 s (9.878 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  71,577\n",
      "Reads with adapters:                       710 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        71,577 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,544,677 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     21,456,494 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 710 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.8%\n",
      "  C: 12.1%\n",
      "  G: 0.0%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t1118.4\t0\t7\n",
      "4\t2\t279.6\t0\t2\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "36\t2\t0.0\t2\t0 0 2\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t12\t0.0\t2\t0 0 12\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t439\t0.0\t2\t0 9 430\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "133\t29\t0.0\t2\t0 0 29\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "145\t23\t0.0\t2\t0 0 23\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t48\t0.0\t2\t0 0 48\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t70\t0.0\t2\t0 2 68\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_107-LP3-ITS-0107_693_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_107-LP3-ITS-0107_693_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_107-LP3-ITS-0107_693_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_107-LP3-ITS-0107_693_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (36.902 ¬µs/read; 1.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     184\n",
      "Reads with adapters:                        11 (6.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           184 (100.0%)\n",
      "\n",
      "Total basepairs processed:        55,384 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         54,297 bp (98.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.9%\n",
      "  C: 9.1%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.9\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_108-LP3-ITS-0108_193_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_108-LP3-ITS-0108_193_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_108-LP3-ITS-0108_193_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_108-LP3-ITS-0108_193_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.858 s (9.879 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  86,828\n",
      "Reads with adapters:                       924 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        86,828 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,135,228 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     26,015,956 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 924 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.9%\n",
      "  C: 33.1%\n",
      "  G: 0.3%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1356.7\t0\t3\n",
      "4\t3\t339.2\t0\t3\n",
      "6\t2\t21.2\t0\t2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t7\t0.0\t2\t0 0 7\n",
      "58\t3\t0.0\t2\t0 1 2\n",
      "59\t10\t0.0\t2\t0 0 10\n",
      "84\t7\t0.0\t2\t0 0 7\n",
      "87\t5\t0.0\t2\t0 0 5\n",
      "93\t8\t0.0\t2\t0 0 8\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t398\t0.0\t2\t0 5 393\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t121\t0.0\t2\t0 1 120\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t22\t0.0\t2\t0 0 22\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t24\t0.0\t2\t0 0 24\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t60\t0.0\t2\t0 0 60\n",
      "155\t49\t0.0\t2\t0 1 48\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t69\t0.0\t2\t0 0 69\n",
      "160\t70\t0.0\t2\t0 1 69\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "300\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_109-LP3-ITS-0109_192_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_109-LP3-ITS-0109_192_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_109-LP3-ITS-0109_192_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_109-LP3-ITS-0109_192_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (39.096 ¬µs/read; 1.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     158\n",
      "Reads with adapters:                         5 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           158 (100.0%)\n",
      "\n",
      "Total basepairs processed:        47,558 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         47,306 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.0%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2.5\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_110-LP3-ITS-0110_331_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_110-LP3-ITS-0110_331_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_110-LP3-ITS-0110_331_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_110-LP3-ITS-0110_331_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.601 s (9.879 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  60,887\n",
      "Reads with adapters:                       580 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        60,887 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,326,987 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,257,292 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 580 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.9%\n",
      "  C: 38.6%\n",
      "  G: 0.2%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t951.4\t0\t2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "39\t2\t0.0\t2\t0 0 2\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t8\t0.0\t2\t0 0 8\n",
      "60\t6\t0.0\t2\t0 0 6\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t14\t0.0\t2\t0 1 13\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t271\t0.0\t2\t0 5 266\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "133\t139\t0.0\t2\t0 0 139\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "142\t16\t0.0\t2\t0 0 16\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t16\t0.0\t2\t0 0 16\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t6\t0.0\t2\t0 1 5\n",
      "155\t16\t0.0\t2\t0 1 15\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_111-LP3-ITS-0111_964_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_111-LP3-ITS-0111_964_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_111-LP3-ITS-0111_964_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_111-LP3-ITS-0111_964_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.540 s (10.014 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,945\n",
      "Reads with adapters:                       595 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,945 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,237,445 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,160,446 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 595 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.1%\n",
      "  C: 10.8%\n",
      "  G: 0.0%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t842.9\t0\t6\n",
      "4\t6\t210.7\t0\t6\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t2\t0.1\t1\t0 2\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t227\t0.0\t2\t0 5 222\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t28\t0.0\t2\t0 0 28\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t14\t0.0\t2\t0 0 14\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t68\t0.0\t2\t0 0 68\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t37\t0.0\t2\t0 0 37\n",
      "151\t147\t0.0\t2\t0 1 146\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_112-LP3-ITS-0112_776_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_112-LP3-ITS-0112_776_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_112-LP3-ITS-0112_776_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_112-LP3-ITS-0112_776_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.399 s (10.251 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,934\n",
      "Reads with adapters:                       484 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,934 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,719,134 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,657,195 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 484 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.8%\n",
      "  C: 34.5%\n",
      "  G: 0.6%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t608.3\t0\t5\n",
      "4\t1\t152.1\t0\t1\n",
      "6\t1\t9.5\t0\t1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t1\t0.0\t2\t0 1\n",
      "53\t2\t0.0\t2\t0 1 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t8\t0.0\t2\t0 1 7\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t176\t0.0\t2\t0 4 172\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t79\t0.0\t2\t0 1 78\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t51\t0.0\t2\t0 0 51\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "150\t38\t0.0\t2\t0 0 38\n",
      "151\t15\t0.0\t2\t0 1 14\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t46\t0.0\t2\t0 0 46\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_113-LP3-ITS-0113_679_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_113-LP3-ITS-0113_679_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_113-LP3-ITS-0113_679_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_113-LP3-ITS-0113_679_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.473 s (10.532 ¬µs/read; 5.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,957\n",
      "Reads with adapters:                     6,010 (13.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,957 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,532,057 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,465,929 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6010 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.9%\n",
      "  C: 52.2%\n",
      "  G: 1.7%\n",
      "  T: 20.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4513\t702.5\t0\t4513\n",
      "4\t5\t175.6\t0\t5\n",
      "7\t4\t2.7\t0\t4\n",
      "8\t78\t0.7\t0\t78\n",
      "9\t68\t0.2\t0\t65 3\n",
      "10\t1040\t0.0\t1\t61 979\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "118\t119\t0.0\t2\t0 1 118\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t18\t0.0\t2\t0 1 17\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t14\t0.0\t2\t0 1 13\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t31\t0.0\t2\t0 0 31\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t9\t0.0\t2\t0 0 9\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t43\t0.0\t2\t0 0 43\n",
      "164\t12\t0.0\t2\t0 1 11\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_114-LP3-ITS-0114_326_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_114-LP3-ITS-0114_326_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_114-LP3-ITS-0114_326_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_114-LP3-ITS-0114_326_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.536 s (10.223 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,413\n",
      "Reads with adapters:                     1,795 (3.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,413 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,776,313 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,703,600 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1795 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.6%\n",
      "  C: 52.0%\n",
      "  G: 1.6%\n",
      "  T: 11.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1057\t819.0\t0\t1057\n",
      "4\t3\t204.7\t0\t3\n",
      "8\t10\t0.8\t0\t10\n",
      "9\t6\t0.2\t0\t6\n",
      "10\t195\t0.0\t1\t13 182\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t7\t0.0\t2\t0 0 7\n",
      "101\t10\t0.0\t2\t0 0 10\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t213\t0.0\t2\t0 1 212\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t67\t0.0\t2\t0 2 65\n",
      "134\t17\t0.0\t2\t0 0 17\n",
      "137\t26\t0.0\t2\t0 0 26\n",
      "142\t32\t0.0\t2\t0 1 31\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t33\t0.0\t2\t0 1 32\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "155\t35\t0.0\t2\t0 0 35\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t32\t0.0\t2\t0 0 32\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_115-LP3-ITS-0115_529_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_115-LP3-ITS-0115_529_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_115-LP3-ITS-0115_529_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_115-LP3-ITS-0115_529_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.408 s (9.986 ¬µs/read; 6.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,857\n",
      "Reads with adapters:                       370 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,857 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,297,957 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,252,643 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 370 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.5%\n",
      "  C: 13.2%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t638.4\t0\t1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t264\t0.0\t2\t0 2 262\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t6\t0.0\t2\t0 0 6\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t4\t0.0\t2\t0 1 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t17\t0.0\t2\t0 0 17\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t11\t0.0\t2\t0 0 11\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_116-LP3-ITS-0116_579_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_116-LP3-ITS-0116_579_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_116-LP3-ITS-0116_579_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_116-LP3-ITS-0116_579_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.423 s (10.021 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,167\n",
      "Reads with adapters:                       439 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,167 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,692,267 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,637,174 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 439 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.9%\n",
      "  C: 29.4%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t658.9\t0\t1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t6\t0.0\t2\t0 0 6\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t4\t0.0\t2\t0 1 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t210\t0.0\t2\t0 6 204\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t66\t0.0\t2\t0 1 65\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "142\t26\t0.0\t2\t0 0 26\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t26\t0.0\t2\t0 0 26\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t50\t0.0\t2\t0 1 49\n",
      "164\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_117-LP3-ITS-0117_730_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_117-LP3-ITS-0117_730_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_117-LP3-ITS-0117_730_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_117-LP3-ITS-0117_730_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.542 s (8.263 ¬µs/read; 7.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  65,602\n",
      "Reads with adapters:                     5,932 (9.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        65,602 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,746,202 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     19,718,593 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5932 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.4%\n",
      "  C: 7.0%\n",
      "  G: 0.2%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5857\t1025.0\t0\t5857\n",
      "4\t8\t256.3\t0\t8\n",
      "5\t1\t64.1\t0\t1\n",
      "118\t24\t0.0\t2\t0 0 24\n",
      "133\t4\t0.0\t2\t0 1 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "176\t35\t0.0\t2\t0 0 35\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_118-LP3-ITS-0118_716_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_118-LP3-ITS-0118_716_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_118-LP3-ITS-0118_716_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_118-LP3-ITS-0118_716_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.550 s (9.294 ¬µs/read; 6.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  59,170\n",
      "Reads with adapters:                     3,126 (5.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        59,170 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,810,170 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,766,905 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3126 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.4%\n",
      "  C: 7.0%\n",
      "  G: 0.1%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2835\t924.5\t0\t2835\n",
      "4\t4\t231.1\t0\t4\n",
      "5\t1\t57.8\t0\t1\n",
      "6\t1\t14.4\t0\t1\n",
      "9\t5\t0.2\t0\t4 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "99\t6\t0.0\t2\t0 0 6\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t120\t0.0\t2\t0 4 116\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t41\t0.0\t2\t0 1 40\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t24\t0.0\t2\t0 0 24\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t15\t0.0\t2\t0 0 15\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "176\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_119-LP3-ITS-0119_175_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_119-LP3-ITS-0119_175_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_119-LP3-ITS-0119_175_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_119-LP3-ITS-0119_175_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.496 s (9.971 ¬µs/read; 6.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  49,752\n",
      "Reads with adapters:                       397 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        49,752 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,975,352 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,925,677 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 397 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.1%\n",
      "  C: 17.4%\n",
      "  G: 0.3%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t777.4\t0\t8\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t14\t0.0\t2\t0 0 14\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t179\t0.0\t2\t1 5 173\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t26\t0.0\t2\t0 0 26\n",
      "133\t26\t0.0\t2\t0 1 25\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t36\t0.0\t2\t0 0 36\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t39\t0.0\t2\t0 0 39\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "198\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_120-LP3-ITS-0120_998_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_120-LP3-ITS-0120_998_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_120-LP3-ITS-0120_998_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_120-LP3-ITS-0120_998_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.484 s (10.050 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,181\n",
      "Reads with adapters:                       557 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,181 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,502,481 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,435,407 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 557 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.1%\n",
      "  C: 33.0%\n",
      "  G: 0.2%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t752.8\t0\t6\n",
      "4\t2\t188.2\t0\t2\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "32\t7\t0.0\t2\t0 0 7\n",
      "45\t4\t0.0\t2\t0 0 4\n",
      "51\t20\t0.0\t2\t0 1 19\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t8\t0.0\t2\t0 0 8\n",
      "67\t5\t0.0\t2\t0 0 5\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t234\t0.0\t2\t0 5 229\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t89\t0.0\t2\t0 0 89\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t40\t0.0\t2\t0 0 40\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t20\t0.0\t2\t0 0 20\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t68\t0.0\t2\t0 0 68\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_121-LP3-ITS-0121_894_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_121-LP3-ITS-0121_894_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_121-LP3-ITS-0121_894_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_121-LP3-ITS-0121_894_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.564 s (10.217 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  55,244\n",
      "Reads with adapters:                       573 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        55,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,628,444 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,552,951 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 573 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.4%\n",
      "  C: 11.9%\n",
      "  G: 0.0%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t863.2\t0\t19\n",
      "10\t7\t0.1\t1\t1 6\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t147\t0.0\t2\t0 6 141\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t56\t0.0\t2\t0 1 55\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t122\t0.0\t2\t0 0 122\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "151\t73\t0.0\t2\t0 0 73\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_122-LP3-ITS-0122_623_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_122-LP3-ITS-0122_623_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_122-LP3-ITS-0122_623_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_122-LP3-ITS-0122_623_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.540 s (10.417 ¬µs/read; 5.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,794\n",
      "Reads with adapters:                       565 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,589,994 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,518,032 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 565 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.9%\n",
      "  C: 23.9%\n",
      "  G: 0.2%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t809.3\t0\t4\n",
      "4\t3\t202.3\t0\t3\n",
      "5\t1\t50.6\t0\t1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t11\t0.0\t2\t0 0 11\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "103\t3\t0.0\t2\t0 1 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t29\t0.0\t2\t0 2 27\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t234\t0.0\t2\t0 4 230\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t54\t0.0\t2\t0 2 52\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t38\t0.0\t2\t0 1 37\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t18\t0.0\t2\t0 2 16\n",
      "151\t30\t0.0\t2\t0 0 30\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "160\t76\t0.0\t2\t0 1 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_123-LP3-ITS-0123_830_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_123-LP3-ITS-0123_830_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_123-LP3-ITS-0123_830_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_123-LP3-ITS-0123_830_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.582 s (10.064 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  57,815\n",
      "Reads with adapters:                       555 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        57,815 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,402,315 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,337,546 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 555 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.4%\n",
      "  C: 27.6%\n",
      "  G: 0.2%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t903.4\t0\t10\n",
      "4\t2\t225.8\t0\t2\n",
      "8\t5\t0.9\t0\t5\n",
      "10\t3\t0.1\t1\t1 2\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "32\t9\t0.0\t2\t0 0 9\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "42\t3\t0.0\t2\t0 0 3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t28\t0.0\t2\t0 0 28\n",
      "52\t5\t0.0\t2\t0 0 5\n",
      "53\t6\t0.0\t2\t0 0 6\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t5\t0.0\t2\t0 0 5\n",
      "86\t3\t0.0\t2\t0 0 3\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t235\t0.0\t2\t0 5 230\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t53\t0.0\t2\t0 3 50\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t36\t0.0\t2\t0 2 34\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t30\t0.0\t2\t0 0 30\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t58\t0.0\t2\t0 2 56\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_124-LP3-ITS-0124_26_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_124-LP3-ITS-0124_26_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_124-LP3-ITS-0124_26_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_124-LP3-ITS-0124_26_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.371 s (10.049 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,870\n",
      "Reads with adapters:                       427 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,870 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,097,870 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,044,311 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 427 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.3%\n",
      "  C: 22.2%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t576.1\t0\t1\n",
      "4\t1\t144.0\t0\t1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "73\t1\t0.0\t2\t0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t5\t0.0\t2\t0 0 5\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "108\t4\t0.0\t2\t0 0 4\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "118\t238\t0.0\t2\t0 2 236\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t46\t0.0\t2\t0 0 46\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t25\t0.0\t2\t0 0 25\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t40\t0.0\t2\t0 0 40\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_125-LP3-ITS-0125_1015_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_125-LP3-ITS-0125_1015_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_125-LP3-ITS-0125_1015_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_125-LP3-ITS-0125_1015_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 1.308 s (10.179 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 128,545\n",
      "Reads with adapters:                     1,158 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):       128,545 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,692,045 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     38,543,430 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1158 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.9%\n",
      "  C: 15.4%\n",
      "  G: 0.3%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t2008.5\t0\t27\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 1 1\n",
      "52\t8\t0.0\t2\t0 0 8\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t4\t0.0\t2\t0 1 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t506\t0.0\t2\t0 6 500\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t13\t0.0\t2\t0 0 13\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t97\t0.0\t2\t0 0 97\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 1 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t381\t0.0\t2\t0 1 380\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "155\t28\t0.0\t2\t0 0 28\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t14\t0.0\t2\t0 0 14\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_126-LP3-ITS-0126_852_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_126-LP3-ITS-0126_852_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_126-LP3-ITS-0126_852_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_126-LP3-ITS-0126_852_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.564 s (10.073 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  55,997\n",
      "Reads with adapters:                     1,169 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        55,997 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,855,097 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,817,648 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1169 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.7%\n",
      "  C: 25.7%\n",
      "  G: 0.6%\n",
      "  T: 13.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t490\t875.0\t0\t490\n",
      "4\t5\t218.7\t0\t5\n",
      "5\t1\t54.7\t0\t1\n",
      "6\t1\t13.7\t0\t1\n",
      "7\t1\t3.4\t0\t1\n",
      "8\t22\t0.9\t0\t22\n",
      "9\t332\t0.2\t0\t309 23\n",
      "10\t68\t0.1\t1\t2 66\n",
      "17\t1\t0.0\t1\t0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t127\t0.0\t2\t0 3 124\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t41\t0.0\t2\t0 0 41\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t20\t0.0\t2\t0 0 20\n",
      "151\t19\t0.0\t2\t0 0 19\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "160\t12\t0.0\t2\t0 1 11\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_127-LP3-ITS-0127_430_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_127-LP3-ITS-0127_430_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_127-LP3-ITS-0127_430_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_127-LP3-ITS-0127_430_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.641 s (9.997 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  64,133\n",
      "Reads with adapters:                       590 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        64,133 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,304,033 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     19,232,319 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 590 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.9%\n",
      "  C: 27.6%\n",
      "  G: 0.2%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t1002.1\t0\t20\n",
      "4\t1\t250.5\t0\t1\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 1 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t7\t0.0\t2\t0 0 7\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t5\t0.0\t2\t0 0 5\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t277\t0.0\t2\t1 7 269\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t82\t0.0\t2\t0 3 79\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t30\t0.0\t2\t0 0 30\n",
      "145\t12\t0.0\t2\t0 1 11\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t34\t0.0\t2\t0 1 33\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t31\t0.0\t2\t0 0 31\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t33\t0.0\t2\t0 0 33\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_128-LP3-ITS-0128_319_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_128-LP3-ITS-0128_319_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_128-LP3-ITS-0128_319_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_128-LP3-ITS-0128_319_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.550 s (10.588 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,962\n",
      "Reads with adapters:                       644 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,962 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,640,562 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,554,665 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 644 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.2%\n",
      "  C: 22.0%\n",
      "  G: 0.5%\n",
      "  T: 2.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t811.9\t0\t4\n",
      "4\t4\t203.0\t0\t4\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t14\t0.0\t2\t0 0 14\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t180\t0.0\t2\t0 5 175\n",
      "123\t12\t0.0\t2\t0 0 12\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t69\t0.0\t2\t0 0 69\n",
      "134\t27\t0.0\t2\t0 1 26\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t44\t0.0\t2\t0 0 44\n",
      "142\t21\t0.0\t2\t0 0 21\n",
      "145\t27\t0.0\t2\t0 0 27\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t31\t0.0\t2\t0 0 31\n",
      "152\t12\t0.0\t2\t0 0 12\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "155\t13\t0.0\t2\t0 2 11\n",
      "159\t11\t0.0\t2\t0 0 11\n",
      "160\t79\t0.0\t2\t0 0 79\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t56\t0.0\t2\t0 0 56\n",
      "194\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_129-LP3-ITS-0129_659_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_129-LP3-ITS-0129_659_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_129-LP3-ITS-0129_659_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_129-LP3-ITS-0129_659_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.595 s (9.918 ¬µs/read; 6.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  60,021\n",
      "Reads with adapters:                       610 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        60,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,066,321 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,988,937 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 610 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.2%\n",
      "  C: 33.4%\n",
      "  G: 0.3%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t937.8\t0\t1\n",
      "4\t3\t234.5\t0\t3\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 1\n",
      "87\t5\t0.0\t2\t0 0 5\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "118\t282\t0.0\t2\t0 5 277\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t101\t0.0\t2\t0 3 98\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t33\t0.0\t2\t0 0 33\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t17\t0.0\t2\t0 0 17\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t44\t0.0\t2\t0 0 44\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t47\t0.0\t2\t0 0 47\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_130-LP3-ITS-0130_103_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_130-LP3-ITS-0130_103_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_130-LP3-ITS-0130_103_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_130-LP3-ITS-0130_103_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.471 s (10.050 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,853\n",
      "Reads with adapters:                       478 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,102,753 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,046,661 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 478 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.2%\n",
      "  C: 16.9%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t732.1\t0\t10\n",
      "6\t1\t11.4\t0\t1\n",
      "9\t5\t0.2\t0\t4 1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "104\t30\t0.0\t2\t0 0 30\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t274\t0.0\t2\t1 6 267\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "132\t4\t0.0\t2\t0 1 3\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "134\t18\t0.0\t2\t0 0 18\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t25\t0.0\t2\t0 0 25\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t7\t0.0\t2\t0 1 6\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_131-LP3-ITS-0131_819_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_131-LP3-ITS-0131_819_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_131-LP3-ITS-0131_819_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_131-LP3-ITS-0131_819_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.507 s (9.657 ¬µs/read; 6.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,542\n",
      "Reads with adapters:                       430 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,542 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,815,142 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,762,769 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 430 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.4%\n",
      "  C: 12.1%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t821.0\t0\t4\n",
      "10\t2\t0.1\t1\t0 2\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "67\t6\t0.0\t2\t0 0 6\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t292\t0.0\t2\t0 8 284\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t34\t0.0\t2\t0 1 33\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "160\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_132-LP3-ITS-0132_494_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_132-LP3-ITS-0132_494_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_132-LP3-ITS-0132_494_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_132-LP3-ITS-0132_494_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.530 s (9.896 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,522\n",
      "Reads with adapters:                       559 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,522 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,110,122 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,038,215 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 559 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.2%\n",
      "  C: 30.4%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t836.3\t0\t2\n",
      "4\t1\t209.1\t0\t1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t7\t0.0\t2\t0 0 7\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t227\t0.0\t2\t0 6 221\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t2\t0.0\t2\t0 1 1\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "132\t31\t0.0\t2\t0 0 31\n",
      "133\t77\t0.0\t2\t0 2 75\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t21\t0.0\t2\t0 0 21\n",
      "151\t96\t0.0\t2\t0 0 96\n",
      "155\t17\t0.0\t2\t0 1 16\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t19\t0.0\t2\t0 0 19\n",
      "190\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_133-LP3-ITS-0133_92_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_133-LP3-ITS-0133_92_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_133-LP3-ITS-0133_92_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_133-LP3-ITS-0133_92_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (66.684 ¬µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      95\n",
      "Reads with adapters:                         1 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            95 (100.0%)\n",
      "\n",
      "Total basepairs processed:        28,595 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         28,477 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_134-LP3-ITS-0134_216_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_134-LP3-ITS-0134_216_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_134-LP3-ITS-0134_216_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_134-LP3-ITS-0134_216_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.711 s (9.602 ¬µs/read; 6.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  74,011\n",
      "Reads with adapters:                       575 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        74,011 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,277,311 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     22,211,338 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 575 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.8%\n",
      "  C: 14.4%\n",
      "  G: 0.2%\n",
      "  T: 1.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t1156.4\t0\t18\n",
      "8\t1\t1.1\t0\t1\n",
      "9\t17\t0.3\t0\t15 2\n",
      "10\t1\t0.1\t1\t0 1\n",
      "45\t7\t0.0\t2\t0 0 7\n",
      "52\t17\t0.0\t2\t0 0 17\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "64\t10\t0.0\t2\t0 0 10\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "108\t4\t0.0\t2\t0 0 4\n",
      "115\t1\t0.0\t2\t0 1\n",
      "118\t325\t0.0\t2\t0 4 321\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t34\t0.0\t2\t0 0 34\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t62\t0.0\t2\t0 0 62\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t13\t0.0\t2\t0 0 13\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_135-LP3-ITS-0135_747_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_135-LP3-ITS-0135_747_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_135-LP3-ITS-0135_747_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_135-LP3-ITS-0135_747_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.695 s (10.213 ¬µs/read; 5.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  68,022\n",
      "Reads with adapters:                       859 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        68,022 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,474,622 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     20,409,496 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 859 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.5%\n",
      "  C: 25.0%\n",
      "  G: 0.3%\n",
      "  T: 9.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t211\t1062.8\t0\t211\n",
      "7\t1\t4.2\t0\t1\n",
      "8\t4\t1.0\t0\t4\n",
      "9\t119\t0.3\t0\t109 10\n",
      "10\t30\t0.1\t1\t0 30\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "99\t9\t0.0\t2\t0 0 9\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t238\t0.0\t2\t0 4 234\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t67\t0.0\t2\t0 0 67\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "137\t19\t0.0\t2\t0 0 19\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t43\t0.0\t2\t0 1 42\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t42\t0.0\t2\t0 2 40\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t14\t0.0\t2\t0 0 14\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_136-LP3-ITS-0136_794_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_136-LP3-ITS-0136_794_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_136-LP3-ITS-0136_794_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_136-LP3-ITS-0136_794_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.249 s (10.248 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,287\n",
      "Reads with adapters:                       274 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,287 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,310,387 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,278,137 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 274 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.1%\n",
      "  C: 21.2%\n",
      "  G: 0.4%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t379.5\t0\t1\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "32\t10\t0.0\t2\t0 0 10\n",
      "52\t4\t0.0\t2\t0 1 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "118\t183\t0.0\t2\t0 0 183\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t28\t0.0\t2\t0 0 28\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "160\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_137-LP3-ITS-0137_626_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_137-LP3-ITS-0137_626_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_137-LP3-ITS-0137_626_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_137-LP3-ITS-0137_626_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.491 s (10.140 ¬µs/read; 5.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,467\n",
      "Reads with adapters:                       690 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,467 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,588,567 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,526,189 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 690 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.7%\n",
      "  C: 40.7%\n",
      "  G: 0.4%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t757.3\t0\t7\n",
      "4\t1\t189.3\t0\t1\n",
      "5\t180\t47.3\t0\t180\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t11\t0.0\t2\t0 0 11\n",
      "60\t11\t0.0\t2\t0 0 11\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t285\t0.0\t2\t0 5 280\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "145\t25\t0.0\t2\t0 0 25\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t61\t0.0\t2\t0 0 61\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_138-LP3-ITS-0138_986_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_138-LP3-ITS-0138_986_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_138-LP3-ITS-0138_986_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_138-LP3-ITS-0138_986_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.389 s (10.125 ¬µs/read; 5.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,396\n",
      "Reads with adapters:                       480 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,396 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,557,196 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,497,446 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 480 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.5%\n",
      "  C: 26.0%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t599.9\t0\t1\n",
      "4\t3\t150.0\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t6\t0.0\t2\t0 0 6\n",
      "60\t1\t0.0\t2\t0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t277\t0.0\t2\t0 5 272\n",
      "132\t6\t0.0\t2\t0 0 6\n",
      "133\t57\t0.0\t2\t0 0 57\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t34\t0.0\t2\t0 1 33\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_139-LP3-ITS-0139_532_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_139-LP3-ITS-0139_532_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_139-LP3-ITS-0139_532_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_139-LP3-ITS-0139_532_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.442 s (9.890 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,673\n",
      "Reads with adapters:                       376 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,673 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,446,573 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,401,621 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 376 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.8%\n",
      "  C: 7.7%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t698.0\t0\t2\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t324\t0.0\t2\t0 8 316\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t15\t0.0\t2\t0 0 15\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_140-LP3-ITS-0140_258_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_140-LP3-ITS-0140_258_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_140-LP3-ITS-0140_258_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_140-LP3-ITS-0140_258_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.345 s (9.707 ¬µs/read; 6.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,581\n",
      "Reads with adapters:                       374 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,581 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,709,881 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,663,161 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 374 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.5%\n",
      "  C: 15.0%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "118\t252\t0.0\t2\t0 7 245\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t25\t0.0\t2\t0 0 25\n",
      "175\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_141-LP3-ITS-0141_203_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_141-LP3-ITS-0141_203_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_141-LP3-ITS-0141_203_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_141-LP3-ITS-0141_203_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.402 s (9.869 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,686\n",
      "Reads with adapters:                       345 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,686 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,246,486 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,206,635 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 345 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.6%\n",
      "  C: 26.1%\n",
      "  G: 0.3%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t635.7\t0\t2\n",
      "32\t7\t0.0\t2\t0 0 7\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "52\t5\t0.0\t2\t0 0 5\n",
      "53\t7\t0.0\t2\t0 0 7\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "99\t10\t0.0\t2\t0 0 10\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t223\t0.0\t2\t0 1 222\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "133\t44\t0.0\t2\t0 2 42\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_142-LP3-ITS-0142_655_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_142-LP3-ITS-0142_655_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_142-LP3-ITS-0142_655_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_142-LP3-ITS-0142_655_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.260 s (10.080 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,745\n",
      "Reads with adapters:                       274 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,745 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,749,245 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,717,038 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 274 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.7%\n",
      "  C: 15.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t402.3\t0\t1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "59\t8\t0.0\t2\t0 1 7\n",
      "84\t12\t0.0\t2\t0 0 12\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t192\t0.0\t2\t0 3 189\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "150\t17\t0.0\t2\t0 1 16\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_143-LP3-ITS-0143_299_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_143-LP3-ITS-0143_299_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_143-LP3-ITS-0143_299_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_143-LP3-ITS-0143_299_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.616 s (9.784 ¬µs/read; 6.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  62,913\n",
      "Reads with adapters:                       590 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        62,913 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,936,813 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,858,770 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 590 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.7%\n",
      "  C: 14.6%\n",
      "  G: 0.3%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t983.0\t0\t5\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t2\t0.1\t1\t0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t228\t0.0\t2\t0 6 222\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "133\t46\t0.0\t2\t0 0 46\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t48\t0.0\t2\t0 0 48\n",
      "151\t175\t0.0\t2\t0 0 175\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "164\t16\t0.0\t2\t0 0 16\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_144-LP3-ITS-0144_531_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_144-LP3-ITS-0144_531_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_144-LP3-ITS-0144_531_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_144-LP3-ITS-0144_531_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.542 s (10.144 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,386\n",
      "Reads with adapters:                       473 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,069,186 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,022,535 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 473 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.7%\n",
      "  C: 34.2%\n",
      "  G: 0.4%\n",
      "  T: 4.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t834.2\t0\t43\n",
      "9\t58\t0.2\t0\t50 8\n",
      "10\t4\t0.1\t1\t1 3\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t9\t0.0\t2\t0 1 8\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t153\t0.0\t2\t0 2 151\n",
      "133\t102\t0.0\t2\t0 0 102\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t43\t0.0\t2\t0 0 43\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "159\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_145-LP3-ITS-0145_364_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_145-LP3-ITS-0145_364_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_145-LP3-ITS-0145_364_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_145-LP3-ITS-0145_364_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.467 s (10.180 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,833\n",
      "Reads with adapters:                       501 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,833 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,795,733 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,736,393 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 501 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.7%\n",
      "  C: 24.8%\n",
      "  G: 0.2%\n",
      "  T: 2.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t716.1\t0\t15\n",
      "4\t1\t179.0\t0\t1\n",
      "5\t2\t44.8\t0\t2\n",
      "10\t7\t0.0\t1\t0 7\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "84\t44\t0.0\t2\t0 0 44\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t199\t0.0\t2\t0 6 193\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t24\t0.0\t2\t0 0 24\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t3\t0.0\t2\t0 1 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t14\t0.0\t2\t0 0 14\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t32\t0.0\t2\t0 2 30\n",
      "164\t46\t0.0\t2\t0 0 46\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_146-LP3-ITS-0146_404_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_146-LP3-ITS-0146_404_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_146-LP3-ITS-0146_404_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_146-LP3-ITS-0146_404_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.381 s (9.722 ¬µs/read; 6.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,195\n",
      "Reads with adapters:                       412 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,195 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,797,695 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,748,124 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 412 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.0%\n",
      "  C: 6.8%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t153.1\t0\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t350\t0.0\t2\t0 12 338\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t9\t0.0\t2\t0 1 8\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t13\t0.0\t2\t0 1 12\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_147-LP3-ITS-0147_1075_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_147-LP3-ITS-0147_1075_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_147-LP3-ITS-0147_1075_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_147-LP3-ITS-0147_1075_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.525 s (9.734 ¬µs/read; 6.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,948\n",
      "Reads with adapters:                       438 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,948 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,238,348 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,182,467 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 438 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.5%\n",
      "  C: 17.8%\n",
      "  G: 0.2%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t842.9\t0\t1\n",
      "4\t3\t210.7\t0\t3\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t3\t0.0\t2\t0 1 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t164\t0.0\t2\t0 3 161\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "133\t48\t0.0\t2\t0 0 48\n",
      "134\t10\t0.0\t2\t0 0 10\n",
      "137\t22\t0.0\t2\t0 0 22\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t72\t0.0\t2\t0 1 71\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t68\t0.0\t2\t0 1 67\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_148-LP3-ITS-0148_369_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_148-LP3-ITS-0148_369_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_148-LP3-ITS-0148_369_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_148-LP3-ITS-0148_369_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.405 s (10.082 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,213\n",
      "Reads with adapters:                       497 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,104,113 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,039,054 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 497 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.3%\n",
      "  C: 18.1%\n",
      "  G: 0.0%\n",
      "  T: 1.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t628.3\t0\t4\n",
      "45\t3\t0.0\t2\t0 0 3\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "109\t5\t0.0\t2\t0 0 5\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t220\t0.0\t2\t0 3 217\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t62\t0.0\t2\t0 0 62\n",
      "134\t14\t0.0\t2\t0 0 14\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "151\t88\t0.0\t2\t0 2 86\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t54\t0.0\t2\t0 1 53\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_149-LP3-ITS-0149_262_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_149-LP3-ITS-0149_262_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_149-LP3-ITS-0149_262_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_149-LP3-ITS-0149_262_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.344 s (10.040 ¬µs/read; 5.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,307\n",
      "Reads with adapters:                       298 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,326,407 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,293,068 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 298 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.1%\n",
      "  C: 25.2%\n",
      "  G: 1.3%\n",
      "  T: 2.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t536.0\t0\t13\n",
      "4\t4\t134.0\t0\t4\n",
      "10\t2\t0.0\t1\t1 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t11\t0.0\t2\t0 0 11\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "118\t172\t0.0\t2\t0 5 167\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t28\t0.0\t2\t0 0 28\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_150-LP3-ITS-0150_407_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_150-LP3-ITS-0150_407_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_150-LP3-ITS-0150_407_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_150-LP3-ITS-0150_407_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.298 s (9.859 ¬µs/read; 6.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,208\n",
      "Reads with adapters:                       470 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,208 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,092,608 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,084,439 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 470 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.4%\n",
      "  C: 6.2%\n",
      "  G: 0.2%\n",
      "  T: 17.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t133\t472.0\t0\t133\n",
      "4\t3\t118.0\t0\t3\n",
      "8\t2\t0.5\t0\t2\n",
      "9\t283\t0.1\t0\t261 22\n",
      "10\t7\t0.0\t1\t0 7\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "118\t27\t0.0\t2\t0 1 26\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_151-LP3-ITS-0151_481_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_151-LP3-ITS-0151_481_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_151-LP3-ITS-0151_481_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_151-LP3-ITS-0151_481_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.485 s (9.936 ¬µs/read; 6.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,772\n",
      "Reads with adapters:                       456 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,772 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,680,372 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,627,967 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 456 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.5%\n",
      "  C: 20.6%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t762.1\t0\t19\n",
      "10\t11\t0.0\t1\t0 11\n",
      "46\t5\t0.0\t2\t0 0 5\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "118\t300\t0.0\t2\t0 4 296\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t60\t0.0\t2\t1 1 58\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "160\t12\t0.0\t2\t0 1 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_152-LP3-ITS-0152_444_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_152-LP3-ITS-0152_444_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_152-LP3-ITS-0152_444_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_152-LP3-ITS-0152_444_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.540 s (9.869 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  54,681\n",
      "Reads with adapters:                       531 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        54,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,458,981 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,392,332 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 531 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.4%\n",
      "  C: 18.1%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t854.4\t0\t3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t327\t0.0\t2\t0 6 321\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "133\t73\t0.0\t2\t0 0 73\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t47\t0.0\t2\t0 1 46\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t23\t0.0\t2\t0 0 23\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_153-LP3-ITS-0153_523_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_153-LP3-ITS-0153_523_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_153-LP3-ITS-0153_523_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_153-LP3-ITS-0153_523_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.559 s (9.928 ¬µs/read; 6.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  56,271\n",
      "Reads with adapters:                       598 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        56,271 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,937,571 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,869,149 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 598 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.4%\n",
      "  C: 23.9%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t879.2\t0\t3\n",
      "4\t11\t219.8\t0\t11\n",
      "8\t1\t0.9\t0\t1\n",
      "10\t2\t0.1\t1\t0 2\n",
      "11\t9\t0.0\t1\t0 9\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t7\t0.0\t2\t0 0 7\n",
      "64\t20\t0.0\t2\t0 1 19\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t2\t0.0\t2\t0 1 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "118\t376\t0.0\t2\t0 6 370\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t43\t0.0\t2\t0 0 43\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t6\t0.0\t2\t0 1 5\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "145\t24\t0.0\t2\t0 0 24\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t17\t0.0\t2\t0 0 17\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_154-LP3-ITS-0154_1047_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_154-LP3-ITS-0154_1047_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_154-LP3-ITS-0154_1047_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_154-LP3-ITS-0154_1047_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.460 s (9.873 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,546\n",
      "Reads with adapters:                       439 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,546 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,010,346 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,956,325 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 439 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.4%\n",
      "  C: 21.2%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t727.3\t0\t6\n",
      "4\t1\t181.8\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 1 1\n",
      "118\t257\t0.0\t2\t0 5 252\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t55\t0.0\t2\t0 1 54\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 1 1\n",
      "151\t40\t0.0\t2\t0 1 39\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_155-LP3-ITS-0155_1065_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_155-LP3-ITS-0155_1065_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_155-LP3-ITS-0155_1065_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_155-LP3-ITS-0155_1065_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.371 s (9.826 ¬µs/read; 6.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,783\n",
      "Reads with adapters:                       405 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,783 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,372,683 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,332,252 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 405 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.5%\n",
      "  C: 17.5%\n",
      "  G: 0.0%\n",
      "  T: 4.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t590.4\t0\t50\n",
      "10\t2\t0.0\t1\t0 2\n",
      "32\t14\t0.0\t2\t0 0 14\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "62\t4\t0.0\t2\t0 1 3\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t301\t0.0\t2\t1 3 297\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t10\t0.0\t2\t0 0 10\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_156-LP3-ITS-0156_451_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_156-LP3-ITS-0156_451_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_156-LP3-ITS-0156_451_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_156-LP3-ITS-0156_451_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.317 s (10.149 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,239\n",
      "Reads with adapters:                       375 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,239 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,402,939 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,359,029 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 375 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.1%\n",
      "  C: 29.6%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t488.1\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t14\t0.0\t2\t0 1 13\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t225\t0.0\t2\t1 3 221\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t66\t0.0\t2\t0 1 65\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_157-LP3-ITS-0157_228_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_157-LP3-ITS-0157_228_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_157-LP3-ITS-0157_228_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_157-LP3-ITS-0157_228_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.274 s (9.870 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,794\n",
      "Reads with adapters:                       214 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,365,994 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,340,011 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 214 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.3%\n",
      "  C: 10.3%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t434.3\t0\t7\n",
      "5\t1\t27.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "118\t151\t0.0\t2\t0 2 149\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_158-LP3-ITS-0158_336_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_158-LP3-ITS-0158_336_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_158-LP3-ITS-0158_336_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_158-LP3-ITS-0158_336_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.460 s (9.896 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,478\n",
      "Reads with adapters:                       451 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,478 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,989,878 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,938,953 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 451 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.7%\n",
      "  C: 21.3%\n",
      "  G: 0.4%\n",
      "  T: 1.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t726.2\t0\t17\n",
      "4\t1\t181.6\t0\t1\n",
      "5\t2\t45.4\t0\t2\n",
      "9\t9\t0.2\t0\t9\n",
      "10\t3\t0.0\t1\t1 2\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 1 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "93\t6\t0.0\t2\t0 0 6\n",
      "118\t300\t0.0\t2\t0 2 298\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "128\t10\t0.0\t2\t0 1 9\n",
      "133\t57\t0.0\t2\t0 0 57\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_159-LP3-ITS-0159_89_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_159-LP3-ITS-0159_89_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_159-LP3-ITS-0159_89_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_159-LP3-ITS-0159_89_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.298 s (10.642 ¬µs/read; 5.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,041\n",
      "Reads with adapters:                       329 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,440,341 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,403,139 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 329 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.7%\n",
      "  C: 9.7%\n",
      "  G: 0.3%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t438.1\t0\t9\n",
      "10\t4\t0.0\t1\t0 4\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "47\t12\t0.0\t2\t0 1 11\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "118\t262\t0.0\t2\t0 4 258\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "145\t16\t0.0\t2\t0 1 15\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_160-LP3-ITS-0160_1052_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_160-LP3-ITS-0160_1052_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_160-LP3-ITS-0160_1052_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_160-LP3-ITS-0160_1052_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.554 s (10.453 ¬µs/read; 5.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,041\n",
      "Reads with adapters:                       636 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,041 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,965,341 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,890,717 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 636 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 62.9%\n",
      "  C: 35.4%\n",
      "  G: 0.8%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t828.8\t0\t9\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t3\t0.1\t1\t0 3\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t25\t0.0\t2\t0 0 25\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t10\t0.0\t2\t0 1 9\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t25\t0.0\t2\t0 0 25\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t7\t0.0\t2\t0 0 7\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t307\t0.0\t2\t0 4 303\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t97\t0.0\t2\t0 0 97\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t23\t0.0\t2\t0 1 22\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t34\t0.0\t2\t0 1 33\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t37\t0.0\t2\t0 1 36\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_161-LP3-ITS-0161_1040_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_161-LP3-ITS-0161_1040_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_161-LP3-ITS-0161_1040_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_161-LP3-ITS-0161_1040_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.327 s (9.957 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,804\n",
      "Reads with adapters:                       382 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,874,004 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,829,209 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 382 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.4%\n",
      "  C: 23.3%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t128.1\t0\t1\n",
      "5\t9\t32.0\t0\t9\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t241\t0.0\t2\t0 3 238\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t41\t0.0\t2\t0 1 40\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "151\t5\t0.0\t2\t0 1 4\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_162-LP3-ITS-0162_919_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_162-LP3-ITS-0162_919_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_162-LP3-ITS-0162_919_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_162-LP3-ITS-0162_919_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.340 s (9.923 ¬µs/read; 6.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,303\n",
      "Reads with adapters:                       318 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,303 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,325,203 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,287,610 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 318 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.4%\n",
      "  C: 29.2%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t536.0\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t4\t0.0\t2\t0 0 4\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t22\t0.0\t2\t0 1 21\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t194\t0.0\t2\t0 4 190\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "133\t43\t0.0\t2\t0 1 42\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_163-LP3-ITS-0163_892_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_163-LP3-ITS-0163_892_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_163-LP3-ITS-0163_892_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_163-LP3-ITS-0163_892_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.464 s (9.761 ¬µs/read; 6.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  47,575\n",
      "Reads with adapters:                       382 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        47,575 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,320,075 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,270,818 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 382 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.4%\n",
      "  C: 11.5%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t743.4\t0\t3\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t196\t0.0\t2\t0 1 195\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t14\t0.0\t2\t0 0 14\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t113\t0.0\t2\t0 1 112\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_164-LP3-ITS-0164_715_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_164-LP3-ITS-0164_715_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_164-LP3-ITS-0164_715_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_164-LP3-ITS-0164_715_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.546 s (9.829 ¬µs/read; 6.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  55,541\n",
      "Reads with adapters:                       618 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        55,541 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,717,841 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,639,554 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 618 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.5%\n",
      "  C: 16.2%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t867.8\t0\t4\n",
      "9\t1\t0.2\t0\t0 1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t369\t0.0\t2\t0 4 365\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t73\t0.0\t2\t0 0 73\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t30\t0.0\t2\t0 1 29\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t76\t0.0\t2\t0 0 76\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_165-LP3-ITS-0165_935_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_165-LP3-ITS-0165_935_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_165-LP3-ITS-0165_935_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_165-LP3-ITS-0165_935_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.464 s (9.932 ¬µs/read; 6.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,745\n",
      "Reads with adapters:                       442 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,745 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,070,245 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,016,888 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 442 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.4%\n",
      "  C: 20.4%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t730.4\t0\t2\n",
      "16\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 1 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t11\t0.0\t2\t0 0 11\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t274\t0.0\t2\t0 5 269\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "133\t40\t0.0\t2\t0 1 39\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t11\t0.0\t2\t0 1 10\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t43\t0.0\t2\t0 0 43\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_166-LP3-ITS-0166_552_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_166-LP3-ITS-0166_552_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_166-LP3-ITS-0166_552_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_166-LP3-ITS-0166_552_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.258 s (9.901 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,083\n",
      "Reads with adapters:                       265 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,083 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,850,983 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,819,813 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 265 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.1%\n",
      "  C: 10.2%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t407.5\t0\t4\n",
      "4\t1\t101.9\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "53\t5\t0.0\t2\t0 0 5\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "118\t200\t0.0\t2\t0 3 197\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t22\t0.0\t2\t0 0 22\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_167-LP3-ITS-0167_795_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_167-LP3-ITS-0167_795_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_167-LP3-ITS-0167_795_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_167-LP3-ITS-0167_795_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.365 s (10.017 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,390\n",
      "Reads with adapters:                       370 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,390 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,953,390 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,908,543 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 370 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.2%\n",
      "  C: 11.9%\n",
      "  G: 0.5%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t568.6\t0\t14\n",
      "4\t1\t142.1\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "118\t208\t0.0\t2\t0 6 202\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t96\t0.0\t2\t0 3 93\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_168-LP3-ITS-0168_973_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_168-LP3-ITS-0168_973_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_168-LP3-ITS-0168_973_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_168-LP3-ITS-0168_973_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.523 s (9.993 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,286\n",
      "Reads with adapters:                       540 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,286 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,738,086 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,674,856 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 540 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.3%\n",
      "  C: 26.5%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t817.0\t0\t4\n",
      "10\t2\t0.0\t1\t0 2\n",
      "26\t5\t0.0\t2\t0 0 5\n",
      "42\t6\t0.0\t2\t0 0 6\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "67\t7\t0.0\t2\t0 0 7\n",
      "69\t3\t0.0\t2\t0 1 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t353\t0.0\t2\t0 7 346\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "133\t72\t0.0\t2\t0 0 72\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t20\t0.0\t2\t0 0 20\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "160\t11\t0.0\t2\t0 0 11\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_169-LP3-ITS-0169_838_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_169-LP3-ITS-0169_838_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_169-LP3-ITS-0169_838_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_169-LP3-ITS-0169_838_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.784 s (10.009 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  78,348\n",
      "Reads with adapters:                       822 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        78,348 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,582,748 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     23,473,029 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 822 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.2%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t1224.2\t0\t8\n",
      "10\t3\t0.1\t1\t0 3\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t232\t0.0\t2\t0 5 227\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t118\t0.0\t2\t0 1 117\n",
      "134\t18\t0.0\t2\t0 0 18\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t44\t0.0\t2\t0 0 44\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t184\t0.0\t2\t0 5 179\n",
      "146\t7\t0.0\t2\t0 0 7\n",
      "151\t141\t0.0\t2\t0 0 141\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t17\t0.0\t2\t0 2 15\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t12\t0.0\t2\t0 0 12\n",
      "165\t6\t0.0\t2\t0 0 6\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_170-LP3-ITS-0170_817_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_170-LP3-ITS-0170_817_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_170-LP3-ITS-0170_817_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_170-LP3-ITS-0170_817_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.630 s (10.054 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  62,631\n",
      "Reads with adapters:                       656 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        62,631 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,851,931 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,765,073 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 656 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.1%\n",
      "  C: 20.6%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t978.6\t0\t6\n",
      "4\t1\t244.7\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t197\t0.0\t2\t0 5 192\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "133\t90\t0.0\t2\t0 0 90\n",
      "134\t63\t0.0\t2\t0 0 63\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "142\t32\t0.0\t2\t0 0 32\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t127\t0.0\t2\t0 2 125\n",
      "146\t4\t0.0\t2\t0 0 4\n",
      "151\t44\t0.0\t2\t0 0 44\n",
      "155\t11\t0.0\t2\t0 1 10\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t4\t0.0\t2\t0 1 3\n",
      "160\t36\t0.0\t2\t0 1 35\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "165\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_171-LP3-ITS-0171_174_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_171-LP3-ITS-0171_174_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_171-LP3-ITS-0171_174_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_171-LP3-ITS-0171_174_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.165 s (10.985 ¬µs/read; 5.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,011\n",
      "Reads with adapters:                     7,365 (49.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,011 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,518,311 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,488,842 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 7365 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 14.7%\n",
      "  C: 59.1%\n",
      "  G: 1.5%\n",
      "  T: 24.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6375\t234.5\t0\t6375\n",
      "4\t1\t58.6\t0\t1\n",
      "9\t16\t0.1\t0\t15 1\n",
      "10\t966\t0.0\t1\t56 910\n",
      "11\t3\t0.0\t1\t0 3\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_172-LP3-ITS-0172_465_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_172-LP3-ITS-0172_465_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_172-LP3-ITS-0172_465_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_172-LP3-ITS-0172_465_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.381 s (10.353 ¬µs/read; 5.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,839\n",
      "Reads with adapters:                       571 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,839 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,088,539 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,027,852 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 571 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.2%\n",
      "  C: 29.1%\n",
      "  G: 0.2%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t575.6\t0\t2\n",
      "4\t17\t143.9\t0\t17\n",
      "5\t62\t36.0\t0\t62\n",
      "10\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "44\t34\t0.0\t2\t0 1 33\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t16\t0.0\t2\t0 0 16\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t174\t0.0\t2\t0 3 171\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t23\t0.0\t2\t0 0 23\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t181\t0.0\t2\t0 1 180\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "209\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_173-LP3-ITS-0173_840_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_173-LP3-ITS-0173_840_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_173-LP3-ITS-0173_840_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_173-LP3-ITS-0173_840_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.644 s (10.108 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  63,745\n",
      "Reads with adapters:                       876 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        63,745 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,187,245 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     19,112,235 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 876 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.7%\n",
      "  C: 31.2%\n",
      "  G: 0.6%\n",
      "  T: 9.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t263\t996.0\t0\t263\n",
      "4\t1\t249.0\t0\t1\n",
      "8\t9\t1.0\t0\t9\n",
      "9\t5\t0.2\t0\t5\n",
      "10\t49\t0.1\t1\t1 48\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "118\t181\t0.0\t2\t0 2 179\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t69\t0.0\t2\t0 0 69\n",
      "134\t24\t0.0\t2\t0 0 24\n",
      "137\t26\t0.0\t2\t0 0 26\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t132\t0.0\t2\t0 1 131\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t14\t0.0\t2\t0 0 14\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t32\t0.0\t2\t0 2 30\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_174-LP3-ITS-0174_251_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_174-LP3-ITS-0174_251_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_174-LP3-ITS-0174_251_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_174-LP3-ITS-0174_251_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.386 s (10.185 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,868\n",
      "Reads with adapters:                       476 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,868 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,398,268 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,347,495 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 476 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.1%\n",
      "  C: 30.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t37.0\t0\t1\n",
      "32\t8\t0.0\t2\t0 0 8\n",
      "38\t10\t0.0\t2\t0 1 9\n",
      "49\t7\t0.0\t2\t0 0 7\n",
      "51\t67\t0.0\t2\t0 2 65\n",
      "52\t13\t0.0\t2\t0 0 13\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t6\t0.0\t2\t0 1 5\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "118\t251\t0.0\t2\t0 6 245\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t14\t0.0\t2\t0 0 14\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "139\t15\t0.0\t2\t0 0 15\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t29\t0.0\t2\t0 0 29\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_175-LP3-ITS-0175_284_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_175-LP3-ITS-0175_284_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_175-LP3-ITS-0175_284_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_175-LP3-ITS-0175_284_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.635 s (10.378 ¬µs/read; 5.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  61,169\n",
      "Reads with adapters:                       695 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        61,169 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,411,869 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,332,769 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 695 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.3%\n",
      "  C: 7.5%\n",
      "  G: 0.0%\n",
      "  T: 4.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t955.8\t0\t40\n",
      "8\t1\t0.9\t0\t1\n",
      "9\t43\t0.2\t0\t42 1\n",
      "10\t4\t0.1\t1\t0 4\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "59\t4\t0.0\t2\t0 1 3\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t314\t0.0\t2\t0 2 312\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t121\t0.0\t2\t0 1 120\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t24\t0.0\t2\t0 0 24\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t88\t0.0\t2\t0 0 88\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "198\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_176-LP3-ITS-0176_517_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_176-LP3-ITS-0176_517_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_176-LP3-ITS-0176_517_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_176-LP3-ITS-0176_517_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.633 s (10.206 ¬µs/read; 5.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  61,985\n",
      "Reads with adapters:                       758 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        61,985 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,657,485 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,575,441 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 758 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.6%\n",
      "  C: 32.6%\n",
      "  G: 0.5%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t968.5\t0\t3\n",
      "5\t1\t60.5\t0\t1\n",
      "6\t34\t15.1\t0\t34\n",
      "10\t1\t0.1\t1\t0 1\n",
      "11\t6\t0.0\t1\t0 6\n",
      "32\t7\t0.0\t2\t0 0 7\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "50\t8\t0.0\t2\t0 0 8\n",
      "51\t16\t0.0\t2\t0 1 15\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t38\t0.0\t2\t0 1 37\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "69\t4\t0.0\t2\t0 0 4\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t6\t0.0\t2\t0 1 5\n",
      "91\t6\t0.0\t2\t0 0 6\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t6\t0.0\t2\t0 0 6\n",
      "118\t427\t0.0\t2\t0 7 420\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t3\t0.0\t2\t0 0 3\n",
      "123\t9\t0.0\t2\t0 0 9\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t52\t0.0\t2\t0 1 51\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t14\t0.0\t2\t0 0 14\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t43\t0.0\t2\t0 0 43\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_177-LP3-ITS-0177_1071_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_177-LP3-ITS-0177_1071_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_177-LP3-ITS-0177_1071_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_177-LP3-ITS-0177_1071_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.596 s (9.945 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  59,907\n",
      "Reads with adapters:                       556 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        59,907 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,032,007 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,958,937 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 556 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.4%\n",
      "  C: 24.3%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t936.0\t0\t2\n",
      "32\t1\t0.0\t2\t0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 1 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t269\t0.0\t2\t0 9 260\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t97\t0.0\t2\t0 0 97\n",
      "134\t10\t0.0\t2\t0 1 9\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t32\t0.0\t2\t0 0 32\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t33\t0.0\t2\t0 0 33\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t32\t0.0\t2\t0 0 32\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t14\t0.0\t2\t0 1 13\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_178-LP3-ITS-0178_78_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_178-LP3-ITS-0178_78_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_178-LP3-ITS-0178_78_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_178-LP3-ITS-0178_78_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 1.070 s (9.961 ¬µs/read; 6.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 107,412\n",
      "Reads with adapters:                     1,079 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):       107,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,331,012 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     32,182,510 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1079 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.2%\n",
      "  C: 15.7%\n",
      "  G: 0.1%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1678.3\t0\t1\n",
      "32\t7\t0.0\t2\t0 0 7\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t549\t0.0\t2\t0 11 538\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "130\t5\t0.0\t2\t0 0 5\n",
      "133\t128\t0.0\t2\t0 1 127\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t87\t0.0\t2\t0 0 87\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t43\t0.0\t2\t0 0 43\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t62\t0.0\t2\t0 0 62\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t29\t0.0\t2\t0 0 29\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "231\t2\t0.0\t2\t0 0 2\n",
      "232\t27\t0.0\t2\t0 1 26\n",
      "234\t65\t0.0\t2\t0 1 64\n",
      "235\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_179-LP3-ITS-0179_454_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_179-LP3-ITS-0179_454_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_179-LP3-ITS-0179_454_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_179-LP3-ITS-0179_454_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.579 s (10.116 ¬µs/read; 5.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  57,196\n",
      "Reads with adapters:                       468 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        57,196 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,215,996 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,156,534 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 468 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.1%\n",
      "  C: 20.7%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t893.7\t0\t4\n",
      "5\t1\t55.9\t0\t1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t248\t0.0\t2\t0 6 242\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t45\t0.0\t2\t0 1 44\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "142\t20\t0.0\t2\t0 0 20\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t75\t0.0\t2\t0 1 74\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t19\t0.0\t2\t0 0 19\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_180-LP3-ITS-0180_609_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_180-LP3-ITS-0180_609_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_180-LP3-ITS-0180_609_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_180-LP3-ITS-0180_609_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.455 s (10.350 ¬µs/read; 5.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,006\n",
      "Reads with adapters:                     6,154 (14.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,006 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,245,806 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,180,927 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6154 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.3%\n",
      "  C: 51.5%\n",
      "  G: 1.2%\n",
      "  T: 25.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4836\t687.6\t0\t4836\n",
      "8\t4\t0.7\t0\t4\n",
      "9\t97\t0.2\t0\t92 5\n",
      "10\t920\t0.0\t1\t45 875\n",
      "11\t1\t0.0\t1\t0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "118\t108\t0.0\t2\t0 4 104\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t29\t0.0\t2\t0 1 28\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t33\t0.0\t2\t0 0 33\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t13\t0.0\t2\t0 2 11\n",
      "151\t21\t0.0\t2\t0 0 21\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t50\t0.0\t2\t0 0 50\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_181-LP3-ITS-0181_925_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_181-LP3-ITS-0181_925_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_181-LP3-ITS-0181_925_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_181-LP3-ITS-0181_925_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.764 s (10.180 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  75,015\n",
      "Reads with adapters:                       649 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        75,015 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,579,515 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     22,499,448 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 649 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.9%\n",
      "  C: 33.0%\n",
      "  G: 0.6%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t1172.1\t0\t26\n",
      "10\t4\t0.1\t1\t0 4\n",
      "21\t1\t0.0\t2\t0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "46\t5\t0.0\t2\t0 1 4\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "118\t281\t0.0\t2\t0 4 277\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t121\t0.0\t2\t0 4 117\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t81\t0.0\t2\t0 1 80\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t28\t0.0\t2\t0 0 28\n",
      "160\t20\t0.0\t2\t0 0 20\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "257\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_182-LP3-ITS-0182_546_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_182-LP3-ITS-0182_546_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_182-LP3-ITS-0182_546_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_182-LP3-ITS-0182_546_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.392 s (10.082 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,908\n",
      "Reads with adapters:                       377 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,711,308 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,669,119 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 377 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.2%\n",
      "  C: 35.0%\n",
      "  G: 0.3%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t607.9\t0\t5\n",
      "5\t11\t38.0\t0\t11\n",
      "10\t2\t0.0\t1\t1 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t6\t0.0\t2\t0 0 6\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t14\t0.0\t2\t0 0 14\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "54\t4\t0.0\t2\t0 0 4\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t6\t0.0\t2\t0 1 5\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t199\t0.0\t2\t0 3 196\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t63\t0.0\t2\t0 0 63\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t6\t0.0\t2\t0 1 5\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_183-LP3-ITS-0183_181_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_183-LP3-ITS-0183_181_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_183-LP3-ITS-0183_181_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_183-LP3-ITS-0183_181_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.936 s (10.864 ¬µs/read; 5.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  86,192\n",
      "Reads with adapters:                    20,654 (24.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        86,192 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,943,792 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     25,828,319 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 20654 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.1%\n",
      "  C: 57.1%\n",
      "  G: 2.0%\n",
      "  T: 18.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16429\t1346.8\t0\t16429\n",
      "4\t13\t336.7\t0\t13\n",
      "8\t1\t1.3\t0\t1\n",
      "9\t313\t0.3\t0\t294 19\n",
      "10\t3685\t0.1\t1\t201 3484\n",
      "11\t9\t0.0\t1\t0 9\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "114\t8\t0.0\t2\t0 0 8\n",
      "118\t112\t0.0\t2\t0 3 109\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t36\t0.0\t2\t0 1 35\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_184-LP3-ITS-0184_24_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_184-LP3-ITS-0184_24_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_184-LP3-ITS-0184_24_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_184-LP3-ITS-0184_24_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.822 s (10.320 ¬µs/read; 5.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  79,663\n",
      "Reads with adapters:                    16,872 (21.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        79,663 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,978,563 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     23,872,842 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 16872 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.7%\n",
      "  C: 58.9%\n",
      "  G: 1.5%\n",
      "  T: 19.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13615\t1244.7\t0\t13615\n",
      "4\t3\t311.2\t0\t3\n",
      "5\t1\t77.8\t0\t1\n",
      "6\t1\t19.4\t0\t1\n",
      "7\t1\t4.9\t0\t1\n",
      "8\t6\t1.2\t0\t6\n",
      "9\t202\t0.3\t0\t188 14\n",
      "10\t2776\t0.1\t1\t113 2663\n",
      "11\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "118\t113\t0.0\t2\t0 1 112\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t46\t0.0\t2\t0 1 45\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t23\t0.0\t2\t0 0 23\n",
      "151\t25\t0.0\t2\t0 0 25\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t16\t0.0\t2\t0 0 16\n",
      "166\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_185-LP3-ITS-0185_702_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_185-LP3-ITS-0185_702_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_185-LP3-ITS-0185_702_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_185-LP3-ITS-0185_702_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.472 s (10.411 ¬µs/read; 5.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,374\n",
      "Reads with adapters:                       398 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,374 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,657,574 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,608,388 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 398 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.9%\n",
      "  C: 24.4%\n",
      "  G: 0.3%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t709.0\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t13\t0.0\t2\t0 0 13\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t197\t0.0\t2\t0 2 195\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t12\t0.0\t2\t0 0 12\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t6\t0.0\t2\t0 0 6\n",
      "133\t49\t0.0\t2\t0 0 49\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t61\t0.0\t2\t0 1 60\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_186-LP3-ITS-0186_714_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_186-LP3-ITS-0186_714_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_186-LP3-ITS-0186_714_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_186-LP3-ITS-0186_714_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.621 s (10.121 ¬µs/read; 5.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  61,359\n",
      "Reads with adapters:                       580 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        61,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,469,059 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,396,510 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 580 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.6%\n",
      "  C: 22.9%\n",
      "  G: 0.3%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t958.7\t0\t5\n",
      "9\t3\t0.2\t0\t3\n",
      "10\t1\t0.1\t1\t0 1\n",
      "47\t7\t0.0\t2\t0 0 7\n",
      "60\t11\t0.0\t2\t0 0 11\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t308\t0.0\t2\t0 4 304\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t66\t0.0\t2\t0 0 66\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "142\t45\t0.0\t2\t0 0 45\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "145\t16\t0.0\t2\t0 1 15\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t31\t0.0\t2\t0 0 31\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t50\t0.0\t2\t0 1 49\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_187-LP3-ITS-0187_933_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_187-LP3-ITS-0187_933_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_187-LP3-ITS-0187_933_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_187-LP3-ITS-0187_933_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.590 s (9.526 ¬µs/read; 6.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  61,974\n",
      "Reads with adapters:                       451 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        61,974 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,654,174 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,601,369 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 451 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.6%\n",
      "  C: 20.2%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t968.3\t0\t3\n",
      "32\t27\t0.0\t2\t0 0 27\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "118\t323\t0.0\t2\t0 8 315\n",
      "133\t38\t0.0\t2\t0 0 38\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t20\t0.0\t2\t0 0 20\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_188-LP3-ITS-0188_350_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_188-LP3-ITS-0188_350_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_188-LP3-ITS-0188_350_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_188-LP3-ITS-0188_350_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.468 s (9.721 ¬µs/read; 6.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,095\n",
      "Reads with adapters:                       472 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,095 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,476,595 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,418,951 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 472 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.0%\n",
      "  C: 20.8%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t751.5\t0\t2\n",
      "60\t16\t0.0\t2\t0 0 16\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t320\t0.0\t2\t0 5 315\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t48\t0.0\t2\t0 0 48\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t27\t0.0\t2\t0 1 26\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t15\t0.0\t2\t0 0 15\n",
      "160\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_189-LP3-ITS-0189_557_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_189-LP3-ITS-0189_557_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_189-LP3-ITS-0189_557_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_189-LP3-ITS-0189_557_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.583 s (10.411 ¬µs/read; 5.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  56,046\n",
      "Reads with adapters:                     1,973 (3.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        56,046 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,869,846 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,809,534 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1973 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.3%\n",
      "  C: 49.4%\n",
      "  G: 1.1%\n",
      "  T: 20.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1299\t875.7\t0\t1299\n",
      "4\t2\t218.9\t0\t2\n",
      "9\t25\t0.2\t0\t24 1\n",
      "10\t229\t0.1\t1\t13 216\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "93\t20\t0.0\t2\t0 1 19\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t172\t0.0\t2\t0 4 168\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t67\t0.0\t2\t0 1 66\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t36\t0.0\t2\t0 1 35\n",
      "143\t3\t0.0\t2\t0 1 2\n",
      "145\t11\t0.0\t2\t1 0 10\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t23\t0.0\t2\t0 0 23\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t39\t0.0\t2\t0 2 37\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_190-LP3-ITS-0190_370_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_190-LP3-ITS-0190_370_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_190-LP3-ITS-0190_370_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_190-LP3-ITS-0190_370_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.615 s (9.776 ¬µs/read; 6.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  62,909\n",
      "Reads with adapters:                       683 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        62,909 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,935,609 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,841,129 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 683 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.7%\n",
      "  C: 7.6%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t983.0\t0\t4\n",
      "4\t1\t245.7\t0\t1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "118\t189\t0.0\t2\t0 4 185\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "133\t37\t0.0\t2\t0 1 36\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t76\t0.0\t2\t0 2 74\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t344\t0.0\t2\t0 6 338\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_191-LP3-ITS-0191_1010_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_191-LP3-ITS-0191_1010_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_191-LP3-ITS-0191_1010_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_191-LP3-ITS-0191_1010_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 1.022 s (10.679 ¬µs/read; 5.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  95,726\n",
      "Reads with adapters:                    13,755 (14.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        95,726 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,813,526 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     28,696,659 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 13755 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.7%\n",
      "  C: 56.0%\n",
      "  G: 1.7%\n",
      "  T: 18.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10649\t1495.7\t0\t10649\n",
      "4\t4\t373.9\t0\t4\n",
      "8\t3\t1.5\t0\t3\n",
      "9\t277\t0.4\t0\t254 23\n",
      "10\t2385\t0.1\t1\t164 2221\n",
      "11\t2\t0.0\t1\t0 2\n",
      "17\t2\t0.0\t1\t0 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t148\t0.0\t2\t0 2 146\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "133\t52\t0.0\t2\t0 0 52\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t60\t0.0\t2\t0 0 60\n",
      "150\t29\t0.0\t2\t0 0 29\n",
      "151\t56\t0.0\t2\t0 4 52\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t43\t0.0\t2\t0 0 43\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_192-LP3-ITS-0192_963_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_192-LP3-ITS-0192_963_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_192-LP3-ITS-0192_963_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_192-LP3-ITS-0192_963_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 1.835 s (9.954 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 184,337\n",
      "Reads with adapters:                     7,849 (4.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):       184,337 (100.0%)\n",
      "\n",
      "Total basepairs processed:    55,485,437 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     55,294,416 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 7849 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.5%\n",
      "  C: 46.7%\n",
      "  G: 0.9%\n",
      "  T: 24.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5409\t2880.3\t0\t5409\n",
      "8\t2\t2.8\t0\t2\n",
      "9\t88\t0.7\t0\t80 8\n",
      "10\t1116\t0.2\t1\t49 1067\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "51\t76\t0.0\t2\t0 1 75\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "60\t10\t0.0\t2\t0 0 10\n",
      "79\t4\t0.0\t2\t0 0 4\n",
      "93\t6\t0.0\t2\t0 0 6\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t363\t0.0\t2\t0 6 357\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t17\t0.0\t2\t0 0 17\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t115\t0.0\t2\t0 0 115\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t32\t0.0\t2\t0 0 32\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t103\t0.0\t2\t0 0 103\n",
      "151\t339\t0.0\t2\t0 3 336\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "155\t42\t0.0\t2\t0 1 41\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t18\t0.0\t2\t0 0 18\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "164\t34\t0.0\t2\t0 0 34\n",
      "212\t2\t0.0\t2\t0 0 2\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "234\t9\t0.0\t2\t0 0 9\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_193-LP3-ITS-0193_1146_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_193-LP3-ITS-0193_1146_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_193-LP3-ITS-0193_1146_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_193-LP3-ITS-0193_1146_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.401 s (10.684 ¬µs/read; 5.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,553\n",
      "Reads with adapters:                     4,033 (10.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,553 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,303,453 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,264,857 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4033 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.3%\n",
      "  C: 54.6%\n",
      "  G: 1.5%\n",
      "  T: 22.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3224\t586.8\t0\t3224\n",
      "4\t2\t146.7\t0\t2\n",
      "5\t1\t36.7\t0\t1\n",
      "8\t12\t0.6\t0\t12\n",
      "9\t94\t0.1\t0\t88 6\n",
      "10\t535\t0.0\t1\t18 517\n",
      "11\t3\t0.0\t1\t0 3\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "118\t64\t0.0\t2\t0 1 63\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "145\t10\t0.0\t2\t0 1 9\n",
      "146\t1\t0.0\t2\t0 1\n",
      "150\t52\t0.0\t2\t0 0 52\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "175\t8\t0.0\t2\t0 0 8\n",
      "219\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_194-LP3-ITS-0194_298_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_194-LP3-ITS-0194_298_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_194-LP3-ITS-0194_298_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_194-LP3-ITS-0194_298_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.315 s (10.098 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,195\n",
      "Reads with adapters:                       306 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,195 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,389,695 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,351,968 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 306 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.9%\n",
      "  C: 24.8%\n",
      "  G: 0.7%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t487.4\t0\t3\n",
      "44\t5\t0.0\t2\t0 0 5\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "118\t169\t0.0\t2\t0 2 167\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t43\t0.0\t2\t0 0 43\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t7\t0.0\t2\t0 1 6\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_195-LP3-ITS-0195_144_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_195-LP3-ITS-0195_144_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_195-LP3-ITS-0195_144_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_195-LP3-ITS-0195_144_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.468 s (10.597 ¬µs/read; 5.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,124\n",
      "Reads with adapters:                       524 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,124 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,281,324 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,212,022 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 524 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.8%\n",
      "  C: 15.8%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t689.4\t0\t4\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t207\t0.0\t2\t0 6 201\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t48\t0.0\t2\t0 1 47\n",
      "134\t8\t0.0\t2\t0 1 7\n",
      "137\t48\t0.0\t2\t0 0 48\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "145\t28\t0.0\t2\t0 0 28\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t18\t0.0\t2\t0 0 18\n",
      "151\t36\t0.0\t2\t0 2 34\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "160\t51\t0.0\t2\t0 0 51\n",
      "164\t8\t0.0\t2\t0 0 8\n",
      "165\t17\t0.0\t2\t0 0 17\n",
      "175\t2\t0.0\t2\t0 1 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_196-LP3-ITS-0196_687_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_196-LP3-ITS-0196_687_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_196-LP3-ITS-0196_687_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_196-LP3-ITS-0196_687_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.419 s (9.889 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,333\n",
      "Reads with adapters:                       393 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,333 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,742,233 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,691,119 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 393 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.8%\n",
      "  C: 24.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t213\t0.0\t2\t0 7 206\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t42\t0.0\t2\t0 0 42\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t13\t0.0\t2\t0 0 13\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t38\t0.0\t2\t0 1 37\n",
      "160\t51\t0.0\t2\t0 1 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_197-LP3-ITS-0197_227_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_197-LP3-ITS-0197_227_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_197-LP3-ITS-0197_227_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_197-LP3-ITS-0197_227_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.283 s (9.878 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,677\n",
      "Reads with adapters:                       289 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,677 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,631,777 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,597,625 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 289 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.3%\n",
      "  C: 8.3%\n",
      "  G: 0.3%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t448.1\t0\t4\n",
      "61\t4\t0.0\t2\t0 0 4\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "118\t246\t0.0\t2\t0 2 244\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t6\t0.0\t2\t0 0 6\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_198-LP3-ITS-0198_1014_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_198-LP3-ITS-0198_1014_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_198-LP3-ITS-0198_1014_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_198-LP3-ITS-0198_1014_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.619 s (9.909 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  62,461\n",
      "Reads with adapters:                       664 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        62,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,800,761 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,723,912 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 664 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 55.9%\n",
      "  C: 43.2%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t976.0\t0\t2\n",
      "5\t59\t61.0\t0\t59\n",
      "11\t1\t0.0\t1\t0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "46\t5\t0.0\t2\t0 0 5\n",
      "51\t19\t0.0\t2\t0 1 18\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t10\t0.0\t2\t0 1 9\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "99\t3\t0.0\t2\t0 1 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "108\t3\t0.0\t2\t0 0 3\n",
      "118\t241\t0.0\t2\t0 5 236\n",
      "123\t25\t0.0\t2\t0 0 25\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t84\t0.0\t2\t0 0 84\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t14\t0.0\t2\t0 0 14\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t24\t0.0\t2\t0 0 24\n",
      "145\t15\t0.0\t2\t0 0 15\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 1 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t49\t0.0\t2\t0 0 49\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "160\t16\t0.0\t2\t0 0 16\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t40\t0.0\t2\t0 0 40\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_199-LP3-ITS-0199_1053_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_199-LP3-ITS-0199_1053_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_199-LP3-ITS-0199_1053_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_199-LP3-ITS-0199_1053_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.313 s (10.167 ¬µs/read; 5.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,796\n",
      "Reads with adapters:                       306 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,269,596 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,231,518 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 306 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.1%\n",
      "  C: 35.0%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t481.2\t0\t5\n",
      "4\t2\t120.3\t0\t2\n",
      "6\t1\t7.5\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "64\t5\t0.0\t2\t0 0 5\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t108\t0.0\t2\t0 0 108\n",
      "122\t4\t0.0\t2\t0 0 4\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "133\t37\t0.0\t2\t0 0 37\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t18\t0.0\t2\t0 0 18\n",
      "145\t32\t0.0\t2\t0 0 32\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "159\t19\t0.0\t2\t0 1 18\n",
      "160\t11\t0.0\t2\t0 0 11\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_200-LP3-ITS-0200_871_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_200-LP3-ITS-0200_871_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_200-LP3-ITS-0200_871_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_200-LP3-ITS-0200_871_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.379 s (10.202 ¬µs/read; 5.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,190\n",
      "Reads with adapters:                       419 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,190 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,194,190 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,143,613 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 419 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.2%\n",
      "  C: 32.7%\n",
      "  G: 0.2%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t581.1\t0\t30\n",
      "10\t4\t0.0\t1\t0 4\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t150\t0.0\t2\t0 5 145\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t48\t0.0\t2\t0 1 47\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t26\t0.0\t2\t0 0 26\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t41\t0.0\t2\t0 2 39\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "159\t26\t0.0\t2\t0 0 26\n",
      "160\t22\t0.0\t2\t0 0 22\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "273\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_201-LP3-ITS-0201_1130_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_201-LP3-ITS-0201_1130_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_201-LP3-ITS-0201_1130_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_201-LP3-ITS-0201_1130_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.118 s (10.509 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,214\n",
      "Reads with adapters:                       286 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,214 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,375,414 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,366,888 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 286 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 11.2%\n",
      "  C: 87.8%\n",
      "  G: 0.7%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t175.2\t0\t2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t260\t0.0\t2\t0 13 247\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "118\t19\t0.0\t2\t0 0 19\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_202-LP3-ITS-0202_1022_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_202-LP3-ITS-0202_1022_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_202-LP3-ITS-0202_1022_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_202-LP3-ITS-0202_1022_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.194 s (11.908 ¬µs/read; 5.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,294\n",
      "Reads with adapters:                       918 (5.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,294 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,904,494 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,889,694 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 918 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.8%\n",
      "  C: 38.9%\n",
      "  G: 1.6%\n",
      "  T: 21.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t581\t254.6\t0\t581\n",
      "7\t1\t1.0\t0\t1\n",
      "8\t8\t0.2\t0\t8\n",
      "9\t88\t0.1\t0\t82 6\n",
      "10\t164\t0.0\t1\t16 148\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t22\t0.0\t2\t0 0 22\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "160\t19\t0.0\t2\t0 0 19\n",
      "219\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_203-LP3-ITS-0203_883_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_203-LP3-ITS-0203_883_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_203-LP3-ITS-0203_883_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_203-LP3-ITS-0203_883_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.157 s (11.399 ¬µs/read; 5.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,758\n",
      "Reads with adapters:                     4,537 (33.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,758 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,141,158 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,118,545 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4537 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.4%\n",
      "  C: 49.3%\n",
      "  G: 1.7%\n",
      "  T: 22.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3450\t215.0\t0\t3450\n",
      "4\t3\t53.7\t0\t3\n",
      "9\t69\t0.1\t0\t66 3\n",
      "10\t1000\t0.0\t1\t69 931\n",
      "11\t1\t0.0\t1\t0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t11\t0.0\t2\t0 0 11\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_204-LP3-ITS-0204_426_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_204-LP3-ITS-0204_426_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_204-LP3-ITS-0204_426_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_204-LP3-ITS-0204_426_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.414 s (9.838 ¬µs/read; 6.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,093\n",
      "Reads with adapters:                       437 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,093 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,669,993 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,625,200 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 437 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.8%\n",
      "  C: 25.6%\n",
      "  G: 0.0%\n",
      "  T: 1.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t657.7\t0\t2\n",
      "4\t1\t164.4\t0\t1\n",
      "5\t36\t41.1\t0\t36\n",
      "6\t1\t10.3\t0\t1\n",
      "7\t19\t2.6\t0\t19\n",
      "8\t4\t0.6\t0\t4\n",
      "9\t1\t0.2\t0\t1\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t271\t0.0\t2\t0 8 263\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "133\t29\t0.0\t2\t0 1 28\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t7\t0.0\t2\t0 1 6\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_205-LP3-ITS-0205_383_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_205-LP3-ITS-0205_383_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_205-LP3-ITS-0205_383_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_205-LP3-ITS-0205_383_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.201 s (9.946 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,202\n",
      "Reads with adapters:                       170 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,080,802 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,059,920 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 170 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 96.5%\n",
      "  C: 3.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "118\t134\t0.0\t2\t0 0 134\n",
      "133\t4\t0.0\t2\t0 1 3\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "145\t13\t0.0\t2\t0 0 13\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_206-LP3-ITS-0206_1147_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_206-LP3-ITS-0206_1147_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_206-LP3-ITS-0206_1147_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_206-LP3-ITS-0206_1147_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.504 s (10.209 ¬µs/read; 5.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  49,319\n",
      "Reads with adapters:                       434 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        49,319 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,845,019 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,790,274 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 434 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 57.6%\n",
      "  C: 41.9%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t770.6\t0\t2\n",
      "12\t6\t0.0\t1\t0 6\n",
      "37\t5\t0.0\t2\t0 0 5\n",
      "77\t4\t0.0\t2\t0 0 4\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "99\t5\t0.0\t2\t0 0 5\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t198\t0.0\t2\t0 2 196\n",
      "123\t15\t0.0\t2\t0 0 15\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "133\t83\t0.0\t2\t0 0 83\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t6\t0.0\t2\t0 1 5\n",
      "155\t41\t0.0\t2\t0 0 41\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t19\t0.0\t2\t0 0 19\n",
      "160\t11\t0.0\t2\t0 0 11\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_207-LP3-ITS-0207_993_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_207-LP3-ITS-0207_993_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_207-LP3-ITS-0207_993_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_207-LP3-ITS-0207_993_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.356 s (10.091 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,261\n",
      "Reads with adapters:                       491 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,261 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,613,561 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,557,238 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 491 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.0%\n",
      "  C: 20.6%\n",
      "  G: 0.2%\n",
      "  T: 3.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t551.0\t0\t45\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t4\t0.0\t1\t1 3\n",
      "59\t5\t0.0\t2\t0 1 4\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "95\t7\t0.0\t2\t0 0 7\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "118\t154\t0.0\t2\t0 2 152\n",
      "133\t41\t0.0\t2\t0 0 41\n",
      "134\t63\t0.0\t2\t0 3 60\n",
      "137\t142\t0.0\t2\t0 3 139\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_208-LP3-ITS-0208_256_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_208-LP3-ITS-0208_256_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_208-LP3-ITS-0208_256_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_208-LP3-ITS-0208_256_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.267 s (10.375 ¬µs/read; 5.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,760\n",
      "Reads with adapters:                       338 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,760 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,753,760 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,714,024 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 338 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.1%\n",
      "  C: 73.1%\n",
      "  G: 0.9%\n",
      "  T: 3.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t402.5\t0\t2\n",
      "6\t1\t6.3\t0\t1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t9\t0.0\t2\t0 0 9\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "87\t22\t0.0\t2\t0 0 22\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t54\t0.0\t2\t0 0 54\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t7\t0.0\t2\t0 0 7\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "108\t4\t0.0\t2\t0 0 4\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t28\t0.0\t2\t0 2 26\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t11\t0.0\t2\t0 0 11\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t121\t0.0\t2\t0 2 119\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t14\t0.0\t2\t0 0 14\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_209-LP3-ITS-0209_243_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_209-LP3-ITS-0209_243_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_209-LP3-ITS-0209_243_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_209-LP3-ITS-0209_243_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.463 s (9.833 ¬µs/read; 6.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  47,075\n",
      "Reads with adapters:                       473 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        47,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,169,575 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,114,389 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 473 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.3%\n",
      "  C: 9.5%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "58\t34\t0.0\t2\t0 1 33\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "118\t379\t0.0\t2\t0 2 377\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "134\t17\t0.0\t2\t0 0 17\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "160\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_210-LP3-ITS-0210_906_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_210-LP3-ITS-0210_906_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_210-LP3-ITS-0210_906_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_210-LP3-ITS-0210_906_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.314 s (10.083 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,187\n",
      "Reads with adapters:                       317 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,187 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,387,287 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,351,582 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 317 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.4%\n",
      "  C: 39.7%\n",
      "  G: 0.6%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t487.3\t0\t7\n",
      "4\t1\t121.8\t0\t1\n",
      "7\t1\t1.9\t0\t1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "47\t10\t0.0\t2\t0 0 10\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t24\t0.0\t2\t0 0 24\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "118\t145\t0.0\t2\t0 2 143\n",
      "133\t52\t0.0\t2\t0 0 52\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "142\t6\t0.0\t2\t0 1 5\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_211-LP3-ITS-0211_902_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_211-LP3-ITS-0211_902_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_211-LP3-ITS-0211_902_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_211-LP3-ITS-0211_902_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.283 s (10.355 ¬µs/read; 5.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,327\n",
      "Reads with adapters:                       343 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,327 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,225,427 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,182,222 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 343 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.7%\n",
      "  C: 21.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.1\t0\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "84\t6\t0.0\t2\t0 0 6\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t169\t0.0\t2\t0 2 167\n",
      "133\t42\t0.0\t2\t0 0 42\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t71\t0.0\t2\t0 0 71\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t17\t0.0\t2\t0 0 17\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_212-LP3-ITS-0212_685_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_212-LP3-ITS-0212_685_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_212-LP3-ITS-0212_685_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_212-LP3-ITS-0212_685_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.325 s (9.855 ¬µs/read; 6.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,021\n",
      "Reads with adapters:                       344 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,939,321 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,897,856 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 344 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.5%\n",
      "  C: 12.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t516.0\t0\t1\n",
      "44\t4\t0.0\t2\t0 0 4\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t23\t0.0\t2\t0 0 23\n",
      "59\t4\t0.0\t2\t0 1 3\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "118\t224\t0.0\t2\t0 6 218\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t42\t0.0\t2\t0 1 41\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_213-LP3-ITS-0213_440_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_213-LP3-ITS-0213_440_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_213-LP3-ITS-0213_440_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_213-LP3-ITS-0213_440_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.343 s (9.864 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,726\n",
      "Reads with adapters:                       600 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,726 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,452,526 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,409,959 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 600 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 57.3%\n",
      "  C: 37.5%\n",
      "  G: 0.3%\n",
      "  T: 4.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t104\t542.6\t0\t104\n",
      "5\t126\t33.9\t0\t126\n",
      "6\t1\t8.5\t0\t1\n",
      "9\t2\t0.1\t0\t2\n",
      "10\t17\t0.0\t1\t1 16\n",
      "12\t5\t0.0\t1\t1 4\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "118\t293\t0.0\t2\t1 3 289\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t13\t0.0\t2\t0 0 13\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_214-LP3-ITS-0214_46_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_214-LP3-ITS-0214_46_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_214-LP3-ITS-0214_46_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_214-LP3-ITS-0214_46_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (22.906 ¬µs/read; 2.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     383\n",
      "Reads with adapters:                         6 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           383 (100.0%)\n",
      "\n",
      "Total basepairs processed:       115,283 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        114,432 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_215-LP3-ITS-0215_339_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_215-LP3-ITS-0215_339_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_215-LP3-ITS-0215_339_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_215-LP3-ITS-0215_339_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.296 s (10.485 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,251\n",
      "Reads with adapters:                       236 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,251 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,503,551 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,473,407 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 236 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.0%\n",
      "  C: 27.1%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t110.4\t0\t1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "118\t107\t0.0\t2\t0 2 105\n",
      "133\t58\t0.0\t2\t0 0 58\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t41\t0.0\t2\t0 0 41\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_216-LP3-ITS-0216_422_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_216-LP3-ITS-0216_422_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_216-LP3-ITS-0216_422_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_216-LP3-ITS-0216_422_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.413 s (8.045 ¬µs/read; 7.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,375\n",
      "Reads with adapters:                     5,015 (9.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,463,875 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,444,406 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5015 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 94.1%\n",
      "  C: 5.6%\n",
      "  G: 0.1%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4971\t802.7\t0\t4971\n",
      "4\t12\t200.7\t0\t12\n",
      "5\t1\t50.2\t0\t1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t9\t0.0\t2\t0 2 7\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "176\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_217-LP3-ITS-0217_868_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_217-LP3-ITS-0217_868_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_217-LP3-ITS-0217_868_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_217-LP3-ITS-0217_868_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.455 s (9.664 ¬µs/read; 6.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  47,100\n",
      "Reads with adapters:                       414 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        47,100 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,177,100 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,127,525 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 414 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 97.8%\n",
      "  C: 2.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t735.9\t0\t5\n",
      "10\t1\t0.0\t1\t1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t353\t0.0\t2\t0 6 347\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t37\t0.0\t2\t0 1 36\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_218-LP3-ITS-0218_391_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_218-LP3-ITS-0218_391_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_218-LP3-ITS-0218_391_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_218-LP3-ITS-0218_391_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.316 s (10.152 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,116\n",
      "Reads with adapters:                       306 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,116 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,365,916 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,328,317 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 306 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.1%\n",
      "  C: 22.5%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t486.2\t0\t1\n",
      "6\t1\t7.6\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "50\t5\t0.0\t2\t0 1 4\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t180\t0.0\t2\t0 3 177\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t16\t0.0\t2\t0 0 16\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t8\t0.0\t2\t0 1 7\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_219-LP3-ITS-0219_1111_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_219-LP3-ITS-0219_1111_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_219-LP3-ITS-0219_1111_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_219-LP3-ITS-0219_1111_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.443 s (10.184 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,470\n",
      "Reads with adapters:                       508 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,470 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,084,470 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,017,306 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 508 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.8%\n",
      "  C: 14.0%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t679.2\t0\t3\n",
      "10\t2\t0.0\t1\t0 2\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "118\t178\t0.0\t2\t0 2 176\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t56\t0.0\t2\t0 0 56\n",
      "134\t49\t0.0\t2\t0 0 49\n",
      "137\t18\t0.0\t2\t0 0 18\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t136\t0.0\t2\t0 0 136\n",
      "150\t13\t0.0\t2\t0 0 13\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t19\t0.0\t2\t0 0 19\n",
      "164\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_220-LP3-ITS-0220_2_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_220-LP3-ITS-0220_2_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_220-LP3-ITS-0220_2_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_220-LP3-ITS-0220_2_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.258 s (10.067 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,596\n",
      "Reads with adapters:                       308 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,596 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,704,396 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,672,921 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 308 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.6%\n",
      "  C: 31.2%\n",
      "  G: 0.3%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t399.9\t0\t17\n",
      "4\t3\t100.0\t0\t3\n",
      "5\t3\t25.0\t0\t3\n",
      "6\t7\t6.2\t0\t7\n",
      "7\t2\t1.6\t0\t2\n",
      "8\t2\t0.4\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t5\t0.0\t1\t0 5\n",
      "11\t1\t0.0\t1\t0 1\n",
      "51\t22\t0.0\t2\t0 0 22\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t8\t0.0\t2\t0 0 8\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t164\t0.0\t2\t0 8 156\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_221-LP3-ITS-0221_611_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_221-LP3-ITS-0221_611_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_221-LP3-ITS-0221_611_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_221-LP3-ITS-0221_611_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.263 s (10.076 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,128\n",
      "Reads with adapters:                       254 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,128 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,864,528 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,834,825 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 254 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.2%\n",
      "  C: 26.4%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t408.2\t0\t1\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "86\t5\t0.0\t2\t0 0 5\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "91\t4\t0.0\t2\t0 0 4\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "118\t173\t0.0\t2\t0 2 171\n",
      "122\t5\t0.0\t2\t0 0 5\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t25\t0.0\t2\t0 0 25\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_222-LP3-ITS-0222_314_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_222-LP3-ITS-0222_314_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_222-LP3-ITS-0222_314_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_222-LP3-ITS-0222_314_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.304 s (9.738 ¬µs/read; 6.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,194\n",
      "Reads with adapters:                       344 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,194 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,389,394 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,352,543 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 344 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.5%\n",
      "  C: 25.6%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t487.4\t0\t54\n",
      "4\t1\t121.9\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t156\t0.0\t2\t0 1 155\n",
      "133\t35\t0.0\t2\t0 0 35\n",
      "134\t10\t0.0\t2\t0 0 10\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t33\t0.0\t2\t0 1 32\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t22\t0.0\t2\t0 0 22\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_223-LP3-ITS-0223_124_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_223-LP3-ITS-0223_124_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_223-LP3-ITS-0223_124_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_223-LP3-ITS-0223_124_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.360 s (9.833 ¬µs/read; 6.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,567\n",
      "Reads with adapters:                       336 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,567 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,006,667 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,966,316 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 336 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.9%\n",
      "  C: 32.1%\n",
      "  G: 0.0%\n",
      "  T: 3.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t571.4\t0\t2\n",
      "4\t10\t142.8\t0\t10\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t4\t0.0\t2\t0 0 4\n",
      "43\t4\t0.0\t2\t0 0 4\n",
      "52\t7\t0.0\t2\t0 1 6\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t9\t0.0\t2\t0 0 9\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t129\t0.0\t2\t0 3 126\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "133\t67\t0.0\t2\t0 0 67\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "137\t43\t0.0\t2\t0 0 43\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_224-LP3-ITS-0224_839_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_224-LP3-ITS-0224_839_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_224-LP3-ITS-0224_839_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_224-LP3-ITS-0224_839_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.299 s (10.175 ¬µs/read; 5.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,421\n",
      "Reads with adapters:                       344 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,421 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,855,721 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,817,384 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 344 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.8%\n",
      "  C: 57.8%\n",
      "  G: 0.9%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t459.7\t0\t23\n",
      "4\t2\t114.9\t0\t2\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "78\t5\t0.0\t2\t0 0 5\n",
      "80\t3\t0.0\t2\t0 1 2\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t40\t0.0\t2\t0 0 40\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "93\t20\t0.0\t2\t0 0 20\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "118\t83\t0.0\t2\t0 0 83\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "133\t96\t0.0\t2\t0 0 96\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t12\t0.0\t2\t0 0 12\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "155\t26\t0.0\t2\t0 0 26\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "162\t3\t0.0\t2\t0 0 3\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_225-LP3-ITS-0225_1148_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_225-LP3-ITS-0225_1148_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_225-LP3-ITS-0225_1148_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_225-LP3-ITS-0225_1148_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.346 s (9.882 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,992\n",
      "Reads with adapters:                       346 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,992 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,532,592 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,492,731 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 346 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.6%\n",
      "  C: 11.6%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t546.8\t0\t9\n",
      "4\t1\t136.7\t0\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "52\t5\t0.0\t2\t0 0 5\n",
      "54\t5\t0.0\t2\t0 0 5\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "109\t7\t0.0\t2\t0 0 7\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t288\t0.0\t2\t0 5 283\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "227\t1\t0.0\t2\t0 0 1\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "272\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_226-LP3-ITS-0226_469_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_226-LP3-ITS-0226_469_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_226-LP3-ITS-0226_469_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_226-LP3-ITS-0226_469_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.289 s (10.082 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,632\n",
      "Reads with adapters:                       975 (3.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,632 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,618,232 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,584,588 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 975 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.2%\n",
      "  C: 40.1%\n",
      "  G: 1.5%\n",
      "  T: 19.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t560\t447.4\t0\t560\n",
      "4\t1\t111.8\t0\t1\n",
      "6\t2\t7.0\t0\t2\n",
      "8\t32\t0.4\t0\t32\n",
      "9\t6\t0.1\t0\t5 1\n",
      "10\t133\t0.0\t1\t5 128\n",
      "16\t1\t0.0\t1\t1\n",
      "99\t12\t0.0\t2\t0 0 12\n",
      "118\t137\t0.0\t2\t0 1 136\n",
      "133\t31\t0.0\t2\t0 0 31\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t13\t0.0\t2\t0 0 13\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t16\t0.0\t2\t0 0 16\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_227-LP3-ITS-0227_520_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_227-LP3-ITS-0227_520_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_227-LP3-ITS-0227_520_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_227-LP3-ITS-0227_520_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.363 s (10.161 ¬µs/read; 5.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,687\n",
      "Reads with adapters:                       435 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,687 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,741,787 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,691,595 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 435 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.5%\n",
      "  C: 56.3%\n",
      "  G: 0.9%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t557.6\t0\t8\n",
      "4\t1\t139.4\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t46\t0.0\t2\t0 1 45\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t7\t0.0\t2\t0 0 7\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "62\t5\t0.0\t2\t0 1 4\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t149\t0.0\t2\t0 3 146\n",
      "123\t13\t0.0\t2\t0 0 13\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t57\t0.0\t2\t0 0 57\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t22\t0.0\t2\t0 1 21\n",
      "158\t8\t0.0\t2\t0 0 8\n",
      "159\t54\t0.0\t2\t0 1 53\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_228-LP3-ITS-0228_138_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_228-LP3-ITS-0228_138_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_228-LP3-ITS-0228_138_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_228-LP3-ITS-0228_138_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.324 s (9.966 ¬µs/read; 6.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,471\n",
      "Reads with adapters:                       287 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,773,771 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,738,997 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 287 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.8%\n",
      "  C: 19.5%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t198\t0.0\t2\t1 5 192\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t34\t0.0\t2\t0 0 34\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_229-LP3-ITS-0229_1056_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_229-LP3-ITS-0229_1056_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_229-LP3-ITS-0229_1056_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_229-LP3-ITS-0229_1056_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.282 s (9.866 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,588\n",
      "Reads with adapters:                       258 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,588 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,604,988 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,576,190 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 258 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 54.7%\n",
      "  C: 44.2%\n",
      "  G: 0.4%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t446.7\t0\t7\n",
      "10\t3\t0.0\t1\t0 3\n",
      "11\t11\t0.0\t1\t0 11\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "50\t7\t0.0\t2\t0 0 7\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "118\t137\t0.0\t2\t0 3 134\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t40\t0.0\t2\t0 0 40\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "155\t31\t0.0\t2\t0 0 31\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_230-LP3-ITS-0230_957_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_230-LP3-ITS-0230_957_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_230-LP3-ITS-0230_957_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_230-LP3-ITS-0230_957_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.440 s (9.969 ¬µs/read; 6.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,145\n",
      "Reads with adapters:                       345 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,145 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,287,645 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,247,162 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 345 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 71.0%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t689.8\t0\t2\n",
      "20\t17\t0.0\t2\t0 0 17\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "32\t7\t0.0\t2\t0 0 7\n",
      "48\t10\t0.0\t2\t0 0 10\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "68\t2\t0.0\t2\t0 0 2\n",
      "87\t15\t0.0\t2\t0 0 15\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t7\t0.0\t2\t0 0 7\n",
      "99\t7\t0.0\t2\t0 0 7\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "118\t57\t0.0\t2\t0 0 57\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "133\t127\t0.0\t2\t0 0 127\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "155\t35\t0.0\t2\t0 0 35\n",
      "157\t1\t0.0\t2\t0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_231-LP3-ITS-0231_60_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_231-LP3-ITS-0231_60_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_231-LP3-ITS-0231_60_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_231-LP3-ITS-0231_60_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.283 s (10.030 ¬µs/read; 5.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,264\n",
      "Reads with adapters:                       342 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,264 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,507,464 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,478,063 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 342 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.7%\n",
      "  C: 26.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t441.6\t0\t95\n",
      "6\t1\t6.9\t0\t1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t16\t0.0\t2\t0 0 16\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t136\t0.0\t2\t0 2 134\n",
      "133\t47\t0.0\t2\t0 1 46\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_232-LP3-ITS-0232_1144_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_232-LP3-ITS-0232_1144_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_232-LP3-ITS-0232_1144_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_232-LP3-ITS-0232_1144_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.128 s (9.281 ¬µs/read; 6.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,826\n",
      "Reads with adapters:                     1,073 (7.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,826 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,161,626 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,155,462 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1073 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.6%\n",
      "  C: 6.6%\n",
      "  G: 0.2%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1052\t216.0\t0\t1052\n",
      "4\t4\t54.0\t0\t4\n",
      "176\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_233-LP3-ITS-0233_613_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_233-LP3-ITS-0233_613_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_233-LP3-ITS-0233_613_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_233-LP3-ITS-0233_613_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.370 s (9.876 ¬µs/read; 6.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,437\n",
      "Reads with adapters:                       388 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,437 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,268,537 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,221,901 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 388 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.2%\n",
      "  C: 9.3%\n",
      "  G: 0.3%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t146.2\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "79\t14\t0.0\t2\t0 0 14\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "118\t303\t0.0\t2\t0 8 295\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t4\t0.0\t2\t0 0 4\n",
      "133\t10\t0.0\t2\t0 0 10\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t26\t0.0\t2\t0 0 26\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_234-LP3-ITS-0234_1_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_234-LP3-ITS-0234_1_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_234-LP3-ITS-0234_1_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_234-LP3-ITS-0234_1_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.128 s (11.555 ¬µs/read; 5.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,043\n",
      "Reads with adapters:                     1,482 (13.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,043 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,323,943 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,309,183 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1482 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.3%\n",
      "  C: 47.4%\n",
      "  G: 1.7%\n",
      "  T: 22.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1086\t172.5\t0\t1086\n",
      "4\t1\t43.1\t0\t1\n",
      "8\t8\t0.2\t0\t8\n",
      "9\t36\t0.0\t0\t33 3\n",
      "10\t296\t0.0\t1\t12 284\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "118\t19\t0.0\t2\t0 0 19\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "219\t6\t0.0\t2\t0 0 6\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_235-LP3-ITS-0235_558_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_235-LP3-ITS-0235_558_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_235-LP3-ITS-0235_558_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_235-LP3-ITS-0235_558_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.323 s (10.263 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,466\n",
      "Reads with adapters:                       648 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,466 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,471,266 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,436,840 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 648 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.1%\n",
      "  C: 17.3%\n",
      "  G: 0.0%\n",
      "  T: 15.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t190\t491.7\t0\t190\n",
      "4\t1\t122.9\t0\t1\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t163\t0.1\t0\t153 10\n",
      "10\t29\t0.0\t1\t3 26\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t192\t0.0\t2\t0 4 188\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "145\t14\t0.0\t2\t0 0 14\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_236-LP3-ITS-0236_888_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_236-LP3-ITS-0236_888_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_236-LP3-ITS-0236_888_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_236-LP3-ITS-0236_888_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.213 s (9.850 ¬µs/read; 6.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,668\n",
      "Reads with adapters:                       200 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,668 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,522,068 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,497,639 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 200 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.5%\n",
      "  C: 33.5%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t338.6\t0\t4\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t97\t0.0\t2\t0 0 97\n",
      "133\t44\t0.0\t2\t0 0 44\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t15\t0.0\t2\t0 1 14\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_237-LP3-ITS-0237_895_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_237-LP3-ITS-0237_895_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_237-LP3-ITS-0237_895_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_237-LP3-ITS-0237_895_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.126 s (11.212 ¬µs/read; 5.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,218\n",
      "Reads with adapters:                       104 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,218 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,376,618 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,364,680 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 104 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.4%\n",
      "  C: 34.6%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t175.3\t0\t1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t67\t0.0\t2\t0 0 67\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_238-LP3-ITS-0238_565_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_238-LP3-ITS-0238_565_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_238-LP3-ITS-0238_565_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_238-LP3-ITS-0238_565_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.265 s (10.396 ¬µs/read; 5.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,457\n",
      "Reads with adapters:                       292 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,662,557 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,628,744 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 292 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 59.9%\n",
      "  C: 39.7%\n",
      "  G: 0.3%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t397.8\t0\t12\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t10\t0.0\t1\t0 10\n",
      "11\t1\t0.0\t1\t0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t104\t0.0\t2\t0 1 103\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t57\t0.0\t2\t0 0 57\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "137\t27\t0.0\t2\t0 0 27\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "219\t3\t0.0\t2\t0 0 3\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_239-LP3-ITS-0239_899_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_239-LP3-ITS-0239_899_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_239-LP3-ITS-0239_899_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_239-LP3-ITS-0239_899_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.211 s (10.128 ¬µs/read; 5.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,850\n",
      "Reads with adapters:                       190 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,275,850 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,251,712 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 190 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.8%\n",
      "  C: 34.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "93\t6\t0.0\t2\t0 0 6\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t67\t0.0\t2\t0 0 67\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "133\t46\t0.0\t2\t0 1 45\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "137\t32\t0.0\t2\t0 1 31\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_240-LP3-ITS-0240_438_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_240-LP3-ITS-0240_438_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_240-LP3-ITS-0240_438_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_240-LP3-ITS-0240_438_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.017 s (16.374 ¬µs/read; 3.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   1,040\n",
      "Reads with adapters:                        16 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         1,040 (100.0%)\n",
      "\n",
      "Total basepairs processed:       313,040 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        311,562 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.0%\n",
      "  C: 18.8%\n",
      "  G: 0.0%\n",
      "  T: 56.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t16.2\t0\t3\n",
      "102\t9\t0.0\t2\t0 0 9\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_241-LP3-ITS-0241_908_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_241-LP3-ITS-0241_908_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_241-LP3-ITS-0241_908_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_241-LP3-ITS-0241_908_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.292 s (10.043 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,071\n",
      "Reads with adapters:                       312 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,750,371 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,711,751 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 312 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.6%\n",
      "  C: 6.7%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t454.2\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t3\t0.0\t2\t0 1 2\n",
      "118\t205\t0.0\t2\t0 6 199\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t33\t0.0\t2\t0 0 33\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t38\t0.0\t2\t0 0 38\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_242-LP3-ITS-0242_580_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_242-LP3-ITS-0242_580_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_242-LP3-ITS-0242_580_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_242-LP3-ITS-0242_580_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (48.428 ¬µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     131\n",
      "Reads with adapters:                         6 (4.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           131 (100.0%)\n",
      "\n",
      "Total basepairs processed:        39,431 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         38,953 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2.0\t0\t2\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_243-LP3-ITS-0243_749_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_243-LP3-ITS-0243_749_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_243-LP3-ITS-0243_749_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_243-LP3-ITS-0243_749_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.978 s (10.006 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  97,756\n",
      "Reads with adapters:                       893 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        97,756 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,424,556 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     29,320,973 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 893 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.3%\n",
      "  C: 14.1%\n",
      "  G: 0.2%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1527.4\t0\t6\n",
      "5\t2\t95.5\t0\t2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "32\t22\t0.0\t2\t0 1 21\n",
      "58\t5\t0.0\t2\t0 0 5\n",
      "66\t19\t0.0\t2\t0 0 19\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t710\t0.0\t2\t0 8 702\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t46\t0.0\t2\t0 0 46\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t37\t0.0\t2\t0 0 37\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "155\t22\t0.0\t2\t0 1 21\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_244-LP3-ITS-0244_524_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_244-LP3-ITS-0244_524_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_244-LP3-ITS-0244_524_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_244-LP3-ITS-0244_524_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.326 s (9.919 ¬µs/read; 6.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,911\n",
      "Reads with adapters:                       392 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,911 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,906,211 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,863,551 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 392 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.0%\n",
      "  C: 25.8%\n",
      "  G: 0.3%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "50\t9\t0.0\t2\t0 0 9\n",
      "51\t52\t0.0\t2\t0 1 51\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "94\t6\t0.0\t2\t0 0 6\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t280\t0.0\t2\t0 8 272\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t24\t0.0\t2\t0 1 23\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_245-LP3-ITS-0245_33_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_245-LP3-ITS-0245_33_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_245-LP3-ITS-0245_33_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_245-LP3-ITS-0245_33_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.120 s (10.265 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,696\n",
      "Reads with adapters:                       124 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,696 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,520,496 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,506,205 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 124 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.2%\n",
      "  C: 18.5%\n",
      "  G: 0.0%\n",
      "  T: 3.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t182.8\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "118\t91\t0.0\t2\t0 2 89\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_246-LP3-ITS-0246_1134_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_246-LP3-ITS-0246_1134_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_246-LP3-ITS-0246_1134_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_246-LP3-ITS-0246_1134_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.228 s (9.995 ¬µs/read; 6.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,798\n",
      "Reads with adapters:                       215 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,798 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,862,198 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,836,823 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 215 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.2%\n",
      "  C: 27.9%\n",
      "  G: 0.5%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t356.2\t0\t4\n",
      "10\t1\t0.0\t1\t1\n",
      "36\t4\t0.0\t2\t0 0 4\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "116\t10\t0.0\t2\t0 0 10\n",
      "118\t132\t0.0\t2\t0 1 131\n",
      "133\t31\t0.0\t2\t0 0 31\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_247-LP3-ITS-0247_13_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_247-LP3-ITS-0247_13_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_247-LP3-ITS-0247_13_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_247-LP3-ITS-0247_13_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.410 s (10.165 ¬µs/read; 5.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,362\n",
      "Reads with adapters:                       421 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,362 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,148,962 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,096,412 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 421 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.7%\n",
      "  C: 27.8%\n",
      "  G: 0.5%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t630.7\t0\t1\n",
      "4\t1\t157.7\t0\t1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t12\t0.0\t2\t0 1 11\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t179\t0.0\t2\t0 2 177\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t12\t0.0\t2\t0 0 12\n",
      "132\t4\t0.0\t2\t0 1 3\n",
      "133\t49\t0.0\t2\t0 1 48\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "142\t22\t0.0\t2\t0 0 22\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t52\t0.0\t2\t0 0 52\n",
      "155\t22\t0.0\t2\t0 0 22\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "198\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_248-LP3-ITS-0248_743_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_248-LP3-ITS-0248_743_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_248-LP3-ITS-0248_743_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_248-LP3-ITS-0248_743_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.434 s (9.907 ¬µs/read; 6.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,811\n",
      "Reads with adapters:                       848 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,187,111 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,117,477 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 848 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 4.4%\n",
      "  C: 94.7%\n",
      "  G: 0.6%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t684.5\t0\t1\n",
      "4\t3\t171.1\t0\t3\n",
      "45\t56\t0.0\t2\t0 0 56\n",
      "51\t406\t0.0\t2\t0 1 405\n",
      "55\t32\t0.0\t2\t0 0 32\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 1\n",
      "87\t53\t0.0\t2\t0 2 51\n",
      "93\t25\t0.0\t2\t0 0 25\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "123\t10\t0.0\t2\t0 0 10\n",
      "133\t140\t0.0\t2\t0 0 140\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t16\t0.0\t2\t0 0 16\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t82\t0.0\t2\t0 0 82\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_249-LP3-ITS-0249_932_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_249-LP3-ITS-0249_932_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_249-LP3-ITS-0249_932_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_249-LP3-ITS-0249_932_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.228 s (10.700 ¬µs/read; 5.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,338\n",
      "Reads with adapters:                       190 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,338 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,422,738 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,401,327 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 190 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.4%\n",
      "  C: 21.6%\n",
      "  G: 0.0%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t333.4\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "103\t37\t0.0\t2\t0 1 36\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t138\t0.0\t2\t0 2 136\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_250-LP3-ITS-0250_758_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_250-LP3-ITS-0250_758_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_250-LP3-ITS-0250_758_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_250-LP3-ITS-0250_758_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.118 s (10.103 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,633\n",
      "Reads with adapters:                        95 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,633 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,501,533 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,490,288 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 95 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 96.8%\n",
      "  C: 3.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t181.8\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t74\t0.0\t2\t0 1 73\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_251-LP3-ITS-0251_921_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_251-LP3-ITS-0251_921_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_251-LP3-ITS-0251_921_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_251-LP3-ITS-0251_921_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.307 s (10.051 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,517\n",
      "Reads with adapters:                       341 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,517 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,185,617 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,149,155 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 341 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.9%\n",
      "  C: 13.5%\n",
      "  G: 0.3%\n",
      "  T: 2.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t476.8\t0\t20\n",
      "4\t3\t119.2\t0\t3\n",
      "10\t5\t0.0\t1\t0 5\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "51\t13\t0.0\t2\t0 0 13\n",
      "54\t5\t0.0\t2\t0 0 5\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t228\t0.0\t2\t0 12 216\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t9\t0.0\t2\t0 1 8\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "134\t10\t0.0\t2\t0 0 10\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_252-LP3-ITS-0252_762_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_252-LP3-ITS-0252_762_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_252-LP3-ITS-0252_762_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_252-LP3-ITS-0252_762_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.383 s (9.716 ¬µs/read; 6.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,471\n",
      "Reads with adapters:                       398 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,880,771 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,835,825 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 398 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.6%\n",
      "  C: 29.9%\n",
      "  G: 0.5%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t616.7\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "41\t3\t0.0\t2\t0 0 3\n",
      "45\t11\t0.0\t2\t0 0 11\n",
      "51\t25\t0.0\t2\t0 0 25\n",
      "60\t5\t0.0\t2\t0 1 4\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t269\t0.0\t2\t0 6 263\n",
      "133\t50\t0.0\t2\t0 1 49\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "155\t15\t0.0\t2\t0 1 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_253-LP3-ITS-0253_449_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_253-LP3-ITS-0253_449_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_253-LP3-ITS-0253_449_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_253-LP3-ITS-0253_449_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.228 s (9.845 ¬µs/read; 6.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,174\n",
      "Reads with adapters:                       199 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,174 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,975,374 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,952,808 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 199 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.8%\n",
      "  C: 30.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t362.1\t0\t3\n",
      "4\t1\t90.5\t0\t1\n",
      "11\t5\t0.0\t1\t0 5\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t132\t0.0\t2\t0 1 131\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t34\t0.0\t2\t0 0 34\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_254-LP3-ITS-0254_1063_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_254-LP3-ITS-0254_1063_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_254-LP3-ITS-0254_1063_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_254-LP3-ITS-0254_1063_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.344 s (9.956 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,537\n",
      "Reads with adapters:                       301 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,537 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,395,637 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,359,751 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 301 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.4%\n",
      "  C: 30.9%\n",
      "  G: 0.3%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t539.6\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t6\t0.0\t2\t0 0 6\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "48\t4\t0.0\t2\t0 0 4\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 1 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t168\t0.0\t2\t0 6 162\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t53\t0.0\t2\t0 1 52\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t17\t0.0\t2\t0 1 16\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_255-LP3-ITS-0255_896_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_255-LP3-ITS-0255_896_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_255-LP3-ITS-0255_896_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_255-LP3-ITS-0255_896_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.154 s (10.574 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,541\n",
      "Reads with adapters:                       206 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,541 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,376,841 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,353,760 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 206 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 14.6%\n",
      "  C: 83.5%\n",
      "  G: 1.5%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t227.2\t0\t1\n",
      "4\t3\t56.8\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "48\t6\t0.0\t2\t0 0 6\n",
      "51\t23\t0.0\t2\t0 0 23\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "87\t14\t0.0\t2\t0 0 14\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "103\t40\t0.0\t2\t0 0 40\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "116\t10\t0.0\t2\t0 0 10\n",
      "118\t6\t0.0\t2\t0 1 5\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t53\t0.0\t2\t0 1 52\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_256-LP3-ITS-0256_824_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_256-LP3-ITS-0256_824_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_256-LP3-ITS-0256_824_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_256-LP3-ITS-0256_824_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.365 s (10.514 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,682\n",
      "Reads with adapters:                       433 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,682 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,439,282 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,383,819 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 433 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.5%\n",
      "  C: 77.4%\n",
      "  G: 0.2%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t29\t0.0\t2\t0 2 27\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t12\t0.0\t2\t0 0 12\n",
      "80\t3\t0.0\t2\t0 0 3\n",
      "93\t8\t0.0\t2\t0 0 8\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t35\t0.0\t2\t0 0 35\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t205\t0.0\t2\t0 1 204\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t20\t0.0\t2\t0 0 20\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t80\t0.0\t2\t0 1 79\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_257-LP3-ITS-0257_222_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_257-LP3-ITS-0257_222_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_257-LP3-ITS-0257_222_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_257-LP3-ITS-0257_222_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.461 s (10.849 ¬µs/read; 5.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,483\n",
      "Reads with adapters:                       447 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,787,383 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,734,804 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 447 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.1%\n",
      "  C: 13.2%\n",
      "  G: 0.2%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t663.8\t0\t13\n",
      "9\t1\t0.2\t0\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t4\t0.0\t2\t0 0 4\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "99\t6\t0.0\t2\t0 0 6\n",
      "100\t1\t0.0\t2\t0 1\n",
      "118\t311\t0.0\t2\t0 2 309\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "134\t14\t0.0\t2\t0 0 14\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "145\t3\t0.0\t2\t0 1 2\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_258-LP3-ITS-0258_34_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_258-LP3-ITS-0258_34_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_258-LP3-ITS-0258_34_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_258-LP3-ITS-0258_34_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.138 s (10.043 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,768\n",
      "Reads with adapters:                       110 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,768 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,144,168 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,131,445 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 110 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.3%\n",
      "  C: 20.9%\n",
      "  G: 0.0%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t215.1\t0\t5\n",
      "10\t2\t0.0\t1\t0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t71\t0.0\t2\t0 1 70\n",
      "133\t8\t0.0\t2\t0 0 8\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t7\t0.0\t2\t0 1 6\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_259-LP3-ITS-0259_423_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_259-LP3-ITS-0259_423_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_259-LP3-ITS-0259_423_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_259-LP3-ITS-0259_423_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.378 s (10.433 ¬µs/read; 5.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,225\n",
      "Reads with adapters:                       371 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,225 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,903,725 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,863,052 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 371 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.7%\n",
      "  C: 24.8%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t566.0\t0\t2\n",
      "51\t28\t0.0\t2\t0 0 28\n",
      "52\t23\t0.0\t2\t0 0 23\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "100\t4\t0.0\t2\t0 0 4\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t260\t0.0\t2\t0 5 255\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 1 4\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_260-LP3-ITS-0260_782_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_260-LP3-ITS-0260_782_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_260-LP3-ITS-0260_782_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_260-LP3-ITS-0260_782_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.439 s (10.401 ¬µs/read; 5.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,218\n",
      "Reads with adapters:                       380 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,218 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,707,618 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,663,154 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 380 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 54.5%\n",
      "  C: 42.9%\n",
      "  G: 0.5%\n",
      "  T: 2.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t659.7\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "33\t2\t0.0\t2\t0 0 2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t18\t0.0\t2\t0 1 17\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "79\t4\t0.0\t2\t0 0 4\n",
      "84\t10\t0.0\t2\t0 0 10\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t9\t0.0\t2\t0 0 9\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t168\t0.0\t2\t0 3 165\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "133\t87\t0.0\t2\t0 1 86\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t15\t0.0\t2\t0 0 15\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "155\t25\t0.0\t2\t0 0 25\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_261-LP3-ITS-0261_1127_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_261-LP3-ITS-0261_1127_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_261-LP3-ITS-0261_1127_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_261-LP3-ITS-0261_1127_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.344 s (10.464 ¬µs/read; 5.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,840\n",
      "Reads with adapters:                       286 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,884,840 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,851,016 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 286 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.6%\n",
      "  C: 22.7%\n",
      "  G: 0.3%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t513.1\t0\t1\n",
      "61\t6\t0.0\t2\t0 0 6\n",
      "65\t4\t0.0\t2\t0 0 4\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "118\t211\t0.0\t2\t0 4 207\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "133\t38\t0.0\t2\t0 0 38\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_262-LP3-ITS-0262_1086_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_262-LP3-ITS-0262_1086_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_262-LP3-ITS-0262_1086_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_262-LP3-ITS-0262_1086_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.277 s (10.963 ¬µs/read; 5.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,304\n",
      "Reads with adapters:                       340 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,616,504 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,573,490 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 340 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.8%\n",
      "  C: 17.9%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t395.4\t0\t1\n",
      "4\t12\t98.8\t0\t12\n",
      "6\t1\t6.2\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t79\t0.0\t2\t0 2 77\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "137\t70\t0.0\t2\t0 0 70\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t98\t0.0\t2\t0 2 96\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_263-LP3-ITS-0263_165_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_263-LP3-ITS-0263_165_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_263-LP3-ITS-0263_165_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_263-LP3-ITS-0263_165_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.195 s (10.192 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,132\n",
      "Reads with adapters:                       146 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,132 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,758,732 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,740,487 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 146 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.2%\n",
      "  C: 67.8%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "101\t14\t0.0\t2\t0 0 14\n",
      "110\t5\t0.0\t2\t0 0 5\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "118\t30\t0.0\t2\t0 0 30\n",
      "133\t75\t0.0\t2\t0 0 75\n",
      "136\t6\t0.0\t2\t0 0 6\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_264-LP3-ITS-0264_461_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_264-LP3-ITS-0264_461_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_264-LP3-ITS-0264_461_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_264-LP3-ITS-0264_461_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (60.164 ¬µs/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      98\n",
      "Reads with adapters:                         5 (5.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            98 (100.0%)\n",
      "\n",
      "Total basepairs processed:        29,498 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         29,469 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.0%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1.5\t0\t3\n",
      "10\t2\t0.0\t1\t0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_265-LP3-ITS-0265_828_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_265-LP3-ITS-0265_828_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_265-LP3-ITS-0265_828_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_265-LP3-ITS-0265_828_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.398 s (11.172 ¬µs/read; 5.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,589\n",
      "Reads with adapters:                     7,845 (22.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,712,289 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,664,893 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 7845 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.0%\n",
      "  C: 54.5%\n",
      "  G: 1.6%\n",
      "  T: 24.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6434\t556.1\t0\t6434\n",
      "4\t4\t139.0\t0\t4\n",
      "8\t2\t0.5\t0\t2\n",
      "9\t88\t0.1\t0\t79 9\n",
      "10\t1197\t0.0\t1\t73 1124\n",
      "11\t4\t0.0\t1\t0 4\n",
      "16\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "118\t58\t0.0\t2\t0 1 57\n",
      "132\t9\t0.0\t2\t0 0 9\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "219\t9\t0.0\t2\t0 0 9\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_266-LP3-ITS-0266_73_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_266-LP3-ITS-0266_73_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_266-LP3-ITS-0266_73_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_266-LP3-ITS-0266_73_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.309 s (11.354 ¬µs/read; 5.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,209\n",
      "Reads with adapters:                     2,466 (9.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,209 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,189,909 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,163,733 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2466 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 45.5%\n",
      "  G: 2.3%\n",
      "  T: 20.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1704\t425.1\t0\t1704\n",
      "4\t1\t106.3\t0\t1\n",
      "6\t1\t6.6\t0\t1\n",
      "7\t1\t1.7\t0\t1\n",
      "8\t28\t0.4\t0\t28\n",
      "9\t120\t0.1\t0\t108 12\n",
      "10\t496\t0.0\t1\t31 465\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "118\t56\t0.0\t2\t0 2 54\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_267-LP3-ITS-0267_820_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_267-LP3-ITS-0267_820_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_267-LP3-ITS-0267_820_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_267-LP3-ITS-0267_820_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.479 s (10.510 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,589\n",
      "Reads with adapters:                     1,835 (4.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,722,289 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,673,357 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1835 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.4%\n",
      "  C: 48.2%\n",
      "  G: 1.1%\n",
      "  T: 24.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1229\t712.3\t0\t1229\n",
      "8\t6\t0.7\t0\t6\n",
      "9\t50\t0.2\t0\t47 3\n",
      "10\t221\t0.0\t1\t17 204\n",
      "11\t1\t0.0\t1\t1\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t154\t0.0\t2\t0 3 151\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t3\t0.0\t2\t0 0 3\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "133\t48\t0.0\t2\t0 1 47\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t25\t0.0\t2\t0 0 25\n",
      "159\t50\t0.0\t2\t0 0 50\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_268-LP3-ITS-0268_106_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_268-LP3-ITS-0268_106_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_268-LP3-ITS-0268_106_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_268-LP3-ITS-0268_106_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.417 s (10.145 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,092\n",
      "Reads with adapters:                       447 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,092 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,368,692 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,327,375 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 447 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.2%\n",
      "  C: 18.3%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t94\t642.1\t0\t94\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "53\t7\t0.0\t2\t0 0 7\n",
      "84\t12\t0.0\t2\t0 0 12\n",
      "87\t8\t0.0\t2\t0 0 8\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "118\t264\t0.0\t2\t0 2 262\n",
      "133\t39\t0.0\t2\t0 0 39\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_269-LP3-ITS-0269_1109_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_269-LP3-ITS-0269_1109_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_269-LP3-ITS-0269_1109_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_269-LP3-ITS-0269_1109_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.335 s (8.833 ¬µs/read; 6.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,982\n",
      "Reads with adapters:                     3,547 (9.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,982 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,432,582 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,415,040 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3547 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.3%\n",
      "  C: 7.0%\n",
      "  G: 0.3%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3498\t593.5\t0\t3498\n",
      "4\t4\t148.4\t0\t4\n",
      "5\t2\t37.1\t0\t2\n",
      "6\t1\t9.3\t0\t1\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "176\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_270-LP3-ITS-0270_332_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_270-LP3-ITS-0270_332_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_270-LP3-ITS-0270_332_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_270-LP3-ITS-0270_332_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.422 s (10.222 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,300\n",
      "Reads with adapters:                       757 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,300 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,431,300 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,377,692 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 757 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.1%\n",
      "  C: 11.1%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t355\t645.3\t0\t355\n",
      "4\t3\t161.3\t0\t3\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "118\t143\t0.0\t2\t0 4 139\n",
      "122\t3\t0.0\t2\t0 1 2\n",
      "133\t31\t0.0\t2\t0 0 31\n",
      "134\t24\t0.0\t2\t0 0 24\n",
      "137\t91\t0.0\t2\t0 0 91\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "143\t21\t0.0\t2\t0 0 21\n",
      "145\t27\t0.0\t2\t0 0 27\n",
      "150\t11\t0.0\t2\t0 0 11\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "160\t24\t0.0\t2\t0 0 24\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_271-LP3-ITS-0271_495_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_271-LP3-ITS-0271_495_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_271-LP3-ITS-0271_495_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_271-LP3-ITS-0271_495_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.383 s (10.372 ¬µs/read; 5.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,901\n",
      "Reads with adapters:                       329 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,901 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,107,201 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,067,055 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 329 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.1%\n",
      "  C: 54.4%\n",
      "  G: 0.6%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t144.1\t0\t1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "87\t37\t0.0\t2\t0 0 37\n",
      "93\t9\t0.0\t2\t0 0 9\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t113\t0.0\t2\t0 5 108\n",
      "123\t30\t0.0\t2\t0 0 30\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "133\t72\t0.0\t2\t0 0 72\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "155\t34\t0.0\t2\t0 0 34\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_272-LP3-ITS-0272_93_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_272-LP3-ITS-0272_93_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_272-LP3-ITS-0272_93_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_272-LP3-ITS-0272_93_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (70.115 ¬µs/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      97\n",
      "Reads with adapters:                         3 (3.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            97 (100.0%)\n",
      "\n",
      "Total basepairs processed:        29,197 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         29,039 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t0.4\t0\t1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_273-LP3-ITS-0273_365_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_273-LP3-ITS-0273_365_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_273-LP3-ITS-0273_365_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_273-LP3-ITS-0273_365_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.550 s (10.038 ¬µs/read; 5.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  54,821\n",
      "Reads with adapters:                       527 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        54,821 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,501,121 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,435,536 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 527 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 98.3%\n",
      "  C: 1.1%\n",
      "  G: 0.2%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t856.6\t0\t5\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t373\t0.0\t2\t0 4 369\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "137\t59\t0.0\t2\t0 0 59\n",
      "145\t55\t0.0\t2\t0 0 55\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 1\n",
      "160\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_274-LP3-ITS-0274_583_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_274-LP3-ITS-0274_583_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_274-LP3-ITS-0274_583_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_274-LP3-ITS-0274_583_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.287 s (10.286 ¬µs/read; 5.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,899\n",
      "Reads with adapters:                       316 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,899 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,397,599 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,355,644 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 316 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.0%\n",
      "  C: 6.6%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "103\t3\t0.0\t2\t0 1 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t143\t0.0\t2\t0 0 143\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t35\t0.0\t2\t0 0 35\n",
      "145\t24\t0.0\t2\t0 0 24\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t21\t0.0\t2\t0 0 21\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "160\t58\t0.0\t2\t0 0 58\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_275-LP3-ITS-0275_409_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_275-LP3-ITS-0275_409_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_275-LP3-ITS-0275_409_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_275-LP3-ITS-0275_409_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.355 s (10.516 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,767\n",
      "Reads with adapters:                       378 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,767 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,163,867 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,115,799 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 378 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.9%\n",
      "  C: 23.8%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t527.6\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t258\t0.0\t2\t0 9 249\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t62\t0.0\t2\t0 1 61\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_276-LP3-ITS-0276_47_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_276-LP3-ITS-0276_47_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_276-LP3-ITS-0276_47_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_276-LP3-ITS-0276_47_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.347 s (10.345 ¬µs/read; 5.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,527\n",
      "Reads with adapters:                       293 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,527 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,091,627 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,054,525 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 293 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.3%\n",
      "  C: 28.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "49\t5\t0.0\t2\t0 0 5\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t175\t0.0\t2\t0 3 172\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t25\t0.0\t2\t0 0 25\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t14\t0.0\t2\t0 1 13\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "155\t48\t0.0\t2\t0 0 48\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_277-LP3-ITS-0277_503_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_277-LP3-ITS-0277_503_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_277-LP3-ITS-0277_503_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_277-LP3-ITS-0277_503_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.362 s (10.658 ¬µs/read; 5.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,993\n",
      "Reads with adapters:                       338 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,993 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,231,893 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,190,992 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 338 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.3%\n",
      "  C: 23.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t531.1\t0\t3\n",
      "6\t1\t8.3\t0\t1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "51\t25\t0.0\t2\t0 1 24\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "118\t186\t0.0\t2\t0 4 182\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "143\t12\t0.0\t2\t0 0 12\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t23\t0.0\t2\t0 0 23\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_278-LP3-ITS-0278_866_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_278-LP3-ITS-0278_866_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_278-LP3-ITS-0278_866_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_278-LP3-ITS-0278_866_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.391 s (10.617 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,794\n",
      "Reads with adapters:                       973 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,074,994 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,033,606 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 973 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.7%\n",
      "  C: 37.5%\n",
      "  G: 2.0%\n",
      "  T: 15.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t493\t574.9\t0\t493\n",
      "8\t1\t0.6\t0\t1\n",
      "9\t34\t0.1\t0\t32 2\n",
      "10\t142\t0.0\t1\t6 136\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "80\t2\t0.0\t2\t0 0 2\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t181\t0.0\t2\t0 4 177\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "216\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_279-LP3-ITS-0279_347_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_279-LP3-ITS-0279_347_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_279-LP3-ITS-0279_347_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_279-LP3-ITS-0279_347_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.303 s (9.972 ¬µs/read; 6.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,389\n",
      "Reads with adapters:                       329 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,389 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,147,089 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,103,509 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 329 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.2%\n",
      "  C: 50.8%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t474.8\t0\t2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "100\t7\t0.0\t2\t0 0 7\n",
      "101\t15\t0.0\t2\t0 0 15\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "106\t7\t0.0\t2\t0 0 7\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "129\t124\t0.0\t2\t0 2 122\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t75\t0.0\t2\t0 0 75\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t27\t0.0\t2\t0 0 27\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "155\t57\t0.0\t2\t0 0 57\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_280-LP3-ITS-0280_696_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_280-LP3-ITS-0280_696_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_280-LP3-ITS-0280_696_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_280-LP3-ITS-0280_696_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (42.240 ¬µs/read; 1.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     179\n",
      "Reads with adapters:                         8 (4.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           179 (100.0%)\n",
      "\n",
      "Total basepairs processed:        53,879 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         53,147 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.0%\n",
      "  C: 25.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t2.8\t0\t3\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_281-LP3-ITS-0281_1035_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_281-LP3-ITS-0281_1035_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_281-LP3-ITS-0281_1035_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_281-LP3-ITS-0281_1035_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.675 s (9.834 ¬µs/read; 6.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  68,670\n",
      "Reads with adapters:                       749 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        68,670 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,669,670 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     20,572,626 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 749 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.0%\n",
      "  C: 11.9%\n",
      "  G: 0.0%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1073.0\t0\t4\n",
      "4\t1\t268.2\t0\t1\n",
      "10\t2\t0.1\t1\t0 2\n",
      "118\t385\t0.0\t2\t0 3 382\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "133\t77\t0.0\t2\t0 1 76\n",
      "137\t93\t0.0\t2\t0 0 93\n",
      "145\t77\t0.0\t2\t0 0 77\n",
      "151\t14\t0.0\t2\t0 0 14\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "160\t82\t0.0\t2\t0 1 81\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_282-LP3-ITS-0282_235_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_282-LP3-ITS-0282_235_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_282-LP3-ITS-0282_235_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_282-LP3-ITS-0282_235_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.473 s (10.491 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,048\n",
      "Reads with adapters:                       464 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,048 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,559,448 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,503,213 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 464 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.6%\n",
      "  C: 14.4%\n",
      "  G: 0.0%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t703.9\t0\t22\n",
      "8\t1\t0.7\t0\t1\n",
      "9\t4\t0.2\t0\t4\n",
      "10\t4\t0.0\t1\t1 3\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "118\t151\t0.0\t2\t0 3 148\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t15\t0.0\t2\t0 0 15\n",
      "134\t43\t0.0\t2\t0 0 43\n",
      "137\t22\t0.0\t2\t0 0 22\n",
      "142\t45\t0.0\t2\t0 0 45\n",
      "143\t39\t0.0\t2\t0 0 39\n",
      "145\t31\t0.0\t2\t0 0 31\n",
      "146\t4\t0.0\t2\t0 0 4\n",
      "150\t30\t0.0\t2\t0 0 30\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t14\t0.0\t2\t0 1 13\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_283-LP3-ITS-0283_513_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_283-LP3-ITS-0283_513_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_283-LP3-ITS-0283_513_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_283-LP3-ITS-0283_513_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.408 s (10.078 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,474\n",
      "Reads with adapters:                       375 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,474 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,182,674 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,138,601 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 375 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.0%\n",
      "  C: 79.5%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t632.4\t0\t4\n",
      "10\t1\t0.0\t1\t0 1\n",
      "45\t78\t0.0\t2\t0 1 77\n",
      "102\t12\t0.0\t2\t0 0 12\n",
      "118\t16\t0.0\t2\t0 1 15\n",
      "133\t143\t0.0\t2\t1 0 142\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "143\t11\t0.0\t2\t0 0 11\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "146\t9\t0.0\t2\t0 0 9\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t79\t0.0\t2\t0 0 79\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_284-LP3-ITS-0284_608_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_284-LP3-ITS-0284_608_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_284-LP3-ITS-0284_608_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_284-LP3-ITS-0284_608_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.297 s (10.515 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,276\n",
      "Reads with adapters:                       228 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,276 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,511,076 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,484,632 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 228 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.2%\n",
      "  C: 22.4%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t441.8\t0\t1\n",
      "4\t1\t110.5\t0\t1\n",
      "12\t11\t0.0\t1\t0 11\n",
      "26\t6\t0.0\t2\t0 0 6\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t152\t0.0\t2\t0 1 151\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "150\t3\t0.0\t2\t0 1 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t23\t0.0\t2\t0 0 23\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_285-LP3-ITS-0285_157_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_285-LP3-ITS-0285_157_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_285-LP3-ITS-0285_157_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_285-LP3-ITS-0285_157_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.467 s (10.515 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,384\n",
      "Reads with adapters:                       431 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,384 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,359,584 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,305,798 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 431 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.6%\n",
      "  C: 30.2%\n",
      "  G: 0.2%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t4\t0.0\t2\t0 0 4\n",
      "42\t4\t0.0\t2\t0 0 4\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "79\t4\t0.0\t2\t0 0 4\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t197\t0.0\t2\t0 2 195\n",
      "123\t40\t0.0\t2\t0 0 40\n",
      "133\t50\t0.0\t2\t0 0 50\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "160\t30\t0.0\t2\t0 0 30\n",
      "164\t9\t0.0\t2\t0 0 9\n",
      "273\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_286-LP3-ITS-0286_947_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_286-LP3-ITS-0286_947_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_286-LP3-ITS-0286_947_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_286-LP3-ITS-0286_947_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.481 s (10.358 ¬µs/read; 5.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,471\n",
      "Reads with adapters:                       458 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,987,771 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,932,970 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 458 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 61.6%\n",
      "  C: 37.3%\n",
      "  G: 0.0%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t726.1\t0\t5\n",
      "4\t3\t181.5\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "48\t5\t0.0\t2\t0 0 5\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t23\t0.0\t2\t0 0 23\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "67\t4\t0.0\t2\t0 0 4\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t11\t0.0\t2\t0 0 11\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t176\t0.0\t2\t0 2 174\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "122\t7\t0.0\t2\t0 0 7\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t59\t0.0\t2\t0 0 59\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t10\t0.0\t2\t0 0 10\n",
      "145\t37\t0.0\t2\t0 2 35\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t32\t0.0\t2\t0 0 32\n",
      "160\t28\t0.0\t2\t0 1 27\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_287-LP3-ITS-0287_344_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_287-LP3-ITS-0287_344_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_287-LP3-ITS-0287_344_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_287-LP3-ITS-0287_344_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.507 s (10.232 ¬µs/read; 5.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  49,506\n",
      "Reads with adapters:                       452 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        49,506 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,901,306 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,843,301 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 452 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.7%\n",
      "  C: 60.2%\n",
      "  G: 0.0%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t773.5\t0\t1\n",
      "26\t5\t0.0\t2\t0 1 4\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t3\t0.0\t2\t0 0 3\n",
      "83\t7\t0.0\t2\t0 0 7\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "93\t18\t0.0\t2\t0 0 18\n",
      "96\t4\t0.0\t2\t0 1 3\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "118\t83\t0.0\t2\t0 2 81\n",
      "123\t12\t0.0\t2\t0 0 12\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t148\t0.0\t2\t0 2 146\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t34\t0.0\t2\t0 0 34\n",
      "143\t18\t0.0\t2\t0 0 18\n",
      "144\t12\t0.0\t2\t0 0 12\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t61\t0.0\t2\t0 0 61\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_288-LP3-ITS-0288_41_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_288-LP3-ITS-0288_41_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_288-LP3-ITS-0288_41_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_288-LP3-ITS-0288_41_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (38.342 ¬µs/read; 1.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     193\n",
      "Reads with adapters:                         9 (4.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           193 (100.0%)\n",
      "\n",
      "Total basepairs processed:        58,093 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         57,835 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.9%\n",
      "  C: 11.1%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t3.0\t0\t7\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_289-LP3-ITS-0289_102_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_289-LP3-ITS-0289_102_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_289-LP3-ITS-0289_102_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_289-LP3-ITS-0289_102_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.407 s (11.755 ¬µs/read; 5.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,655\n",
      "Reads with adapters:                       619 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,655 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,431,155 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,383,132 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 619 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.7%\n",
      "  C: 56.1%\n",
      "  G: 1.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t82\t541.5\t0\t82\n",
      "4\t8\t135.4\t0\t8\n",
      "5\t20\t33.8\t0\t20\n",
      "6\t53\t8.5\t0\t53\n",
      "7\t1\t2.1\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t7\t0.0\t2\t0 0 7\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t26\t0.0\t2\t0 3 23\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "37\t6\t0.0\t2\t0 0 6\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t20\t0.0\t2\t0 0 20\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t6\t0.0\t2\t0 0 6\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "66\t16\t0.0\t2\t0 1 15\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "82\t3\t0.0\t2\t0 0 3\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t3\t0.0\t2\t0 0 3\n",
      "89\t8\t0.0\t2\t0 0 8\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t7\t0.0\t2\t0 0 7\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t5\t0.0\t2\t0 0 5\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t3\t0.0\t2\t0 1 2\n",
      "107\t19\t0.0\t2\t0 0 19\n",
      "108\t6\t0.0\t2\t0 0 6\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "119\t11\t0.0\t2\t0 0 11\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "125\t8\t0.0\t2\t0 0 8\n",
      "126\t30\t0.0\t2\t0 0 30\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t41\t0.0\t2\t0 0 41\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t15\t0.0\t2\t0 0 15\n",
      "148\t10\t0.0\t2\t0 0 10\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t11\t0.0\t2\t0 0 11\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t43\t0.0\t2\t0 0 43\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "176\t2\t0.0\t2\t0 0 2\n",
      "178\t3\t0.0\t2\t0 0 3\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "211\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_290-LP3-ITS-0290_694_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_290-LP3-ITS-0290_694_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_290-LP3-ITS-0290_694_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_290-LP3-ITS-0290_694_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.363 s (11.940 ¬µs/read; 5.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,386\n",
      "Reads with adapters:                       482 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,146,186 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,097,137 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 482 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.9%\n",
      "  C: 37.8%\n",
      "  G: 0.8%\n",
      "  T: 2.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t474.8\t0\t32\n",
      "4\t11\t118.7\t0\t11\n",
      "5\t7\t29.7\t0\t7\n",
      "6\t7\t7.4\t0\t7\n",
      "7\t5\t1.9\t0\t5\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t5\t0.0\t2\t0 1 4\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "31\t2\t0.0\t2\t0 0 2\n",
      "34\t2\t0.0\t2\t0 1 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "40\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "66\t12\t0.0\t2\t0 0 12\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "82\t5\t0.0\t2\t0 0 5\n",
      "83\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t5\t0.0\t2\t0 0 5\n",
      "91\t2\t0.0\t2\t0 1 1\n",
      "92\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "97\t4\t0.0\t2\t0 1 3\n",
      "98\t4\t0.0\t2\t0 2 2\n",
      "99\t11\t0.0\t2\t0 0 11\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t7\t0.0\t2\t0 0 7\n",
      "104\t16\t0.0\t2\t0 0 16\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t6\t0.0\t2\t0 0 6\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "113\t4\t0.0\t2\t0 0 4\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t23\t0.0\t2\t0 0 23\n",
      "119\t12\t0.0\t2\t0 1 11\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "125\t16\t0.0\t2\t0 0 16\n",
      "126\t34\t0.0\t2\t0 0 34\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t21\t0.0\t2\t0 0 21\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t46\t0.0\t2\t0 0 46\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t5\t0.0\t2\t0 0 5\n",
      "150\t14\t0.0\t2\t0 0 14\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "154\t5\t0.0\t2\t0 0 5\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t7\t0.0\t2\t0 0 7\n",
      "159\t11\t0.0\t2\t0 1 10\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_291-LP3-ITS-0291_633_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_291-LP3-ITS-0291_633_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_291-LP3-ITS-0291_633_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_291-LP3-ITS-0291_633_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.180 s (11.913 ¬µs/read; 5.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,088\n",
      "Reads with adapters:                       256 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,088 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,541,488 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,515,497 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 256 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 51.2%\n",
      "  C: 46.9%\n",
      "  G: 0.4%\n",
      "  T: 1.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t235.8\t0\t4\n",
      "4\t3\t58.9\t0\t3\n",
      "5\t4\t14.7\t0\t4\n",
      "6\t22\t3.7\t0\t22\n",
      "9\t1\t0.1\t0\t1\n",
      "21\t3\t0.0\t2\t0 0 3\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t8\t0.0\t2\t0 0 8\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t7\t0.0\t2\t0 0 7\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t7\t0.0\t2\t0 0 7\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "138\t6\t0.0\t2\t0 0 6\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t5\t0.0\t2\t0 0 5\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "165\t5\t0.0\t2\t0 0 5\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "181\t2\t0.0\t2\t0 1 1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_292-LP3-ITS-0292_287_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_292-LP3-ITS-0292_287_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_292-LP3-ITS-0292_287_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_292-LP3-ITS-0292_287_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.072 s (12.415 ¬µs/read; 4.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   5,774\n",
      "Reads with adapters:                       107 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         5,774 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,737,974 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,729,690 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 107 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.5%\n",
      "  C: 48.6%\n",
      "  G: 0.9%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t90.2\t0\t6\n",
      "4\t11\t22.6\t0\t11\n",
      "5\t4\t5.6\t0\t4\n",
      "6\t3\t1.4\t0\t3\n",
      "7\t6\t0.4\t0\t6\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "80\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t4\t0.0\t2\t0 0 4\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_293-LP3-ITS-0293_802_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_293-LP3-ITS-0293_802_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_293-LP3-ITS-0293_802_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_293-LP3-ITS-0293_802_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.241 s (11.780 ¬µs/read; 5.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,428\n",
      "Reads with adapters:                       371 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,148,828 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,112,180 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 371 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.8%\n",
      "  C: 60.1%\n",
      "  G: 1.6%\n",
      "  T: 3.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t319.2\t0\t16\n",
      "4\t12\t79.8\t0\t12\n",
      "5\t4\t19.9\t0\t4\n",
      "6\t5\t5.0\t0\t5\n",
      "8\t1\t0.3\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "34\t7\t0.0\t2\t0 0 7\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "46\t9\t0.0\t2\t0 0 9\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t6\t0.0\t2\t0 0 6\n",
      "50\t8\t0.0\t2\t0 2 6\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "65\t3\t0.0\t2\t0 1 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t3\t0.0\t2\t0 1 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t3\t0.0\t2\t0 1 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t7\t0.0\t2\t0 0 7\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t6\t0.0\t2\t0 0 6\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t7\t0.0\t2\t0 0 7\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t10\t0.0\t2\t0 0 10\n",
      "119\t12\t0.0\t2\t0 0 12\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "124\t15\t0.0\t2\t0 0 15\n",
      "125\t61\t0.0\t2\t0 1 60\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t6\t0.0\t2\t0 0 6\n",
      "128\t7\t0.0\t2\t0 0 7\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "135\t6\t0.0\t2\t0 0 6\n",
      "136\t2\t0.0\t2\t0 1 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "181\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_294-LP3-ITS-0294_770_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_294-LP3-ITS-0294_770_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_294-LP3-ITS-0294_770_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_294-LP3-ITS-0294_770_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.247 s (12.263 ¬µs/read; 4.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,106\n",
      "Reads with adapters:                       248 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,106 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,051,906 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,028,277 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 248 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.2%\n",
      "  C: 45.2%\n",
      "  G: 0.4%\n",
      "  T: 7.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t314.2\t0\t26\n",
      "4\t2\t78.5\t0\t2\n",
      "5\t1\t19.6\t0\t1\n",
      "6\t15\t4.9\t0\t15\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t3\t0.0\t1\t1 2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t10\t0.0\t2\t0 0 10\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 1 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t5\t0.0\t2\t0 0 5\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "110\t8\t0.0\t2\t0 0 8\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t12\t0.0\t2\t0 0 12\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t22\t0.0\t2\t0 0 22\n",
      "160\t11\t0.0\t2\t0 0 11\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "185\t2\t0.0\t2\t0 0 2\n",
      "196\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_295-LP3-ITS-0295_720_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_295-LP3-ITS-0295_720_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_295-LP3-ITS-0295_720_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_295-LP3-ITS-0295_720_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.346 s (12.165 ¬µs/read; 4.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,474\n",
      "Reads with adapters:                       752 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,474 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,570,674 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,503,905 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 752 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.5%\n",
      "  C: 64.6%\n",
      "  G: 1.5%\n",
      "  T: 4.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t444.9\t0\t48\n",
      "4\t28\t111.2\t0\t28\n",
      "5\t5\t27.8\t0\t5\n",
      "6\t2\t7.0\t0\t2\n",
      "7\t1\t1.7\t0\t1\n",
      "9\t2\t0.1\t0\t2\n",
      "10\t4\t0.0\t1\t0 4\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t3\t0.0\t2\t0 0 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "40\t2\t0.0\t2\t0 0 2\n",
      "41\t4\t0.0\t2\t0 0 4\n",
      "46\t24\t0.0\t2\t0 1 23\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t6\t0.0\t2\t0 0 6\n",
      "51\t148\t0.0\t2\t0 2 146\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "61\t4\t0.0\t2\t0 0 4\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "73\t8\t0.0\t2\t0 0 8\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "89\t14\t0.0\t2\t0 0 14\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t11\t0.0\t2\t0 0 11\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t6\t0.0\t2\t0 0 6\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "111\t7\t0.0\t2\t0 0 7\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "119\t3\t0.0\t2\t0 1 2\n",
      "120\t16\t0.0\t2\t0 1 15\n",
      "121\t6\t0.0\t2\t0 0 6\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "124\t18\t0.0\t2\t0 0 18\n",
      "125\t30\t0.0\t2\t0 0 30\n",
      "126\t13\t0.0\t2\t0 0 13\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t42\t0.0\t2\t0 0 42\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t4\t0.0\t2\t0 0 4\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t80\t0.0\t2\t0 1 79\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t15\t0.0\t2\t0 0 15\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t5\t0.0\t2\t0 0 5\n",
      "142\t21\t0.0\t2\t0 0 21\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t11\t0.0\t2\t0 0 11\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t8\t0.0\t2\t0 1 7\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 1 5\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "177\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "189\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_296-LP3-ITS-0296_282_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_296-LP3-ITS-0296_282_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_296-LP3-ITS-0296_282_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_296-LP3-ITS-0296_282_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.175 s (13.348 ¬µs/read; 4.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,101\n",
      "Reads with adapters:                       176 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,101 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,943,401 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,930,978 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 176 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 51.7%\n",
      "  G: 0.0%\n",
      "  T: 16.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t204.7\t0\t26\n",
      "4\t10\t51.2\t0\t10\n",
      "5\t6\t12.8\t0\t6\n",
      "10\t1\t0.0\t1\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t28\t0.0\t2\t0 0 28\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_297-LP3-ITS-0297_378_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_297-LP3-ITS-0297_378_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_297-LP3-ITS-0297_378_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_297-LP3-ITS-0297_378_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (41.575 ¬µs/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     167\n",
      "Reads with adapters:                         2 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           167 (100.0%)\n",
      "\n",
      "Total basepairs processed:        50,267 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         50,128 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t0.0\t0\t1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_298-LP3-ITS-0298_980_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_298-LP3-ITS-0298_980_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_298-LP3-ITS-0298_980_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_298-LP3-ITS-0298_980_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (34.313 ¬µs/read; 1.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     227\n",
      "Reads with adapters:                         2 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           227 (100.0%)\n",
      "\n",
      "Total basepairs processed:        68,327 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         68,039 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_299-LP3-ITS-0299_874_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_299-LP3-ITS-0299_874_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_299-LP3-ITS-0299_874_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_299-LP3-ITS-0299_874_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (72.667 ¬µs/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      75\n",
      "Reads with adapters:                         3 (4.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            75 (100.0%)\n",
      "\n",
      "Total basepairs processed:        22,575 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         22,294 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_300-LP3-ITS-0300_751_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_300-LP3-ITS-0300_751_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_300-LP3-ITS-0300_751_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_300-LP3-ITS-0300_751_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (53.724 ¬µs/read; 1.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     141\n",
      "Reads with adapters:                         4 (2.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           141 (100.0%)\n",
      "\n",
      "Total basepairs processed:        42,441 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         42,278 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 0.0%\n",
      "  G: 25.0%\n",
      "  T: 25.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.2\t0\t1\n",
      "4\t1\t0.6\t0\t1\n",
      "7\t1\t0.0\t0\t1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_301-LP3-ITS-0301_719_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_301-LP3-ITS-0301_719_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_301-LP3-ITS-0301_719_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_301-LP3-ITS-0301_719_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (56.141 ¬µs/read; 1.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     136\n",
      "Reads with adapters:                         2 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           136 (100.0%)\n",
      "\n",
      "Total basepairs processed:        40,936 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         40,647 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_302-LP3-ITS-0302_373_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_302-LP3-ITS-0302_373_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_302-LP3-ITS-0302_373_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_302-LP3-ITS-0302_373_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (39.897 ¬µs/read; 1.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     174\n",
      "Reads with adapters:                         1 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           174 (100.0%)\n",
      "\n",
      "Total basepairs processed:        52,374 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         52,224 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_303-LP3-ITS-0303_231_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_303-LP3-ITS-0303_231_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_303-LP3-ITS-0303_231_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_303-LP3-ITS-0303_231_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (44.067 ¬µs/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     161\n",
      "Reads with adapters:                         1 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           161 (100.0%)\n",
      "\n",
      "Total basepairs processed:        48,461 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         48,412 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_304-LP3-ITS-0304_155_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_304-LP3-ITS-0304_155_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_304-LP3-ITS-0304_155_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_304-LP3-ITS-0304_155_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (81.937 ¬µs/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      66\n",
      "Reads with adapters:                         2 (3.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            66 (100.0%)\n",
      "\n",
      "Total basepairs processed:        19,866 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         19,613 bp (98.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_305-LP3-ITS-0305_170_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_305-LP3-ITS-0305_170_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_305-LP3-ITS-0305_170_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_305-LP3-ITS-0305_170_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.275 s (11.728 ¬µs/read; 5.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,420\n",
      "Reads with adapters:                       499 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,420 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,049,420 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,011,353 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 499 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.5%\n",
      "  C: 47.7%\n",
      "  G: 1.2%\n",
      "  T: 3.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t365.9\t0\t43\n",
      "4\t77\t91.5\t0\t77\n",
      "5\t2\t22.9\t0\t2\n",
      "6\t5\t5.7\t0\t5\n",
      "7\t2\t1.4\t0\t2\n",
      "8\t2\t0.4\t0\t2\n",
      "10\t3\t0.0\t1\t0 3\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 1 1\n",
      "27\t21\t0.0\t2\t0 0 21\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "34\t5\t0.0\t2\t0 0 5\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "58\t14\t0.0\t2\t0 1 13\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "64\t7\t0.0\t2\t0 0 7\n",
      "66\t7\t0.0\t2\t0 0 7\n",
      "67\t5\t0.0\t2\t0 0 5\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t4\t0.0\t2\t0 0 4\n",
      "89\t18\t0.0\t2\t0 0 18\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t4\t0.0\t2\t0 0 4\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t5\t0.0\t2\t0 0 5\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "119\t13\t0.0\t2\t0 0 13\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t5\t0.0\t2\t0 0 5\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t32\t0.0\t2\t0 0 32\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 1\n",
      "131\t13\t0.0\t2\t0 0 13\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t50\t0.0\t2\t0 1 49\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 1\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "178\t5\t0.0\t2\t0 0 5\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "196\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_306-LP3-ITS-0306_443_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_306-LP3-ITS-0306_443_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_306-LP3-ITS-0306_443_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_306-LP3-ITS-0306_443_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.211 s (12.565 ¬µs/read; 4.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,776\n",
      "Reads with adapters:                       278 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,049,576 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,022,606 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 278 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 56.8%\n",
      "  C: 40.3%\n",
      "  G: 1.4%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t262.1\t0\t23\n",
      "4\t4\t65.5\t0\t4\n",
      "5\t1\t16.4\t0\t1\n",
      "7\t4\t1.0\t0\t4\n",
      "8\t1\t0.3\t0\t1\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t8\t0.0\t2\t0 0 8\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t14\t0.0\t2\t0 0 14\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "99\t8\t0.0\t2\t0 0 8\n",
      "100\t22\t0.0\t2\t0 0 22\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "104\t23\t0.0\t2\t0 0 23\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t21\t0.0\t2\t0 1 20\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t10\t0.0\t2\t0 0 10\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t17\t0.0\t2\t0 1 16\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 1 4\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_307-LP3-ITS-0307_1003_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_307-LP3-ITS-0307_1003_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_307-LP3-ITS-0307_1003_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_307-LP3-ITS-0307_1003_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.080 s (13.092 ¬µs/read; 4.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,092\n",
      "Reads with adapters:                       303 (5.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         6,092 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,833,692 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,826,673 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 303 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 10.9%\n",
      "  C: 80.9%\n",
      "  G: 2.3%\n",
      "  T: 5.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t170\t95.2\t0\t170\n",
      "4\t1\t23.8\t0\t1\n",
      "5\t6\t5.9\t0\t6\n",
      "6\t3\t1.5\t0\t3\n",
      "7\t33\t0.4\t0\t33\n",
      "9\t2\t0.0\t0\t2\n",
      "10\t15\t0.0\t1\t1 14\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t8\t0.0\t2\t0 1 7\n",
      "28\t8\t0.0\t2\t0 1 7\n",
      "34\t4\t0.0\t2\t0 1 3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "80\t5\t0.0\t2\t0 1 4\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "99\t5\t0.0\t2\t0 0 5\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "121\t3\t0.0\t2\t0 0 3\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_308-LP3-ITS-0308_678_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_308-LP3-ITS-0308_678_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_308-LP3-ITS-0308_678_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_308-LP3-ITS-0308_678_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (47.701 ¬µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     141\n",
      "Reads with adapters:                         3 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           141 (100.0%)\n",
      "\n",
      "Total basepairs processed:        42,441 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         42,039 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_309-LP3-ITS-0309_952_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_309-LP3-ITS-0309_952_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_309-LP3-ITS-0309_952_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_309-LP3-ITS-0309_952_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.380 s (12.096 ¬µs/read; 4.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,434\n",
      "Reads with adapters:                       638 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,434 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,461,634 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,412,574 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 638 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.8%\n",
      "  C: 54.7%\n",
      "  G: 2.5%\n",
      "  T: 2.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t491.2\t0\t73\n",
      "4\t57\t122.8\t0\t57\n",
      "5\t19\t30.7\t0\t19\n",
      "6\t5\t7.7\t0\t5\n",
      "7\t28\t1.9\t0\t28\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t1\t0.1\t0\t0 1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "25\t4\t0.0\t2\t0 0 4\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t3\t0.0\t2\t0 0 3\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t3\t0.0\t2\t0 0 3\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t21\t0.0\t2\t0 1 20\n",
      "51\t2\t0.0\t2\t0 1 1\n",
      "52\t17\t0.0\t2\t0 0 17\n",
      "53\t4\t0.0\t2\t0 0 4\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t21\t0.0\t2\t0 0 21\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 1 2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "86\t10\t0.0\t2\t0 0 10\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t8\t0.0\t2\t0 1 7\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t5\t0.0\t2\t0 0 5\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t11\t0.0\t2\t0 0 11\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t34\t0.0\t2\t0 1 33\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t17\t0.0\t2\t0 1 16\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t12\t0.0\t2\t0 0 12\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t15\t0.0\t2\t0 0 15\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t14\t0.0\t2\t0 1 13\n",
      "126\t10\t0.0\t2\t0 0 10\n",
      "127\t17\t0.0\t2\t0 0 17\n",
      "128\t12\t0.0\t2\t0 0 12\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t4\t0.0\t2\t0 0 4\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "135\t18\t0.0\t2\t0 0 18\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t19\t0.0\t2\t0 0 19\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "147\t11\t0.0\t2\t0 0 11\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t19\t0.0\t2\t0 0 19\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_310-LP3-ITS-0310_1006_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_310-LP3-ITS-0310_1006_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_310-LP3-ITS-0310_1006_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_310-LP3-ITS-0310_1006_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (62.611 ¬µs/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     102\n",
      "Reads with adapters:                         1 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           102 (100.0%)\n",
      "\n",
      "Total basepairs processed:        30,702 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         30,569 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_311-LP3-ITS-0311_457_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_311-LP3-ITS-0311_457_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_311-LP3-ITS-0311_457_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_311-LP3-ITS-0311_457_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.168 s (11.677 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,397\n",
      "Reads with adapters:                       280 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,333,497 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,311,668 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 280 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.7%\n",
      "  C: 55.0%\n",
      "  G: 4.3%\n",
      "  T: 5.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t225.0\t0\t16\n",
      "4\t13\t56.2\t0\t13\n",
      "5\t12\t14.1\t0\t12\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "38\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t86\t0.0\t2\t0 2 84\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t10\t0.0\t2\t0 0 10\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t4\t0.0\t2\t0 1 3\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t11\t0.0\t2\t0 0 11\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t6\t0.0\t2\t0 0 6\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_312-LP3-ITS-0312_96_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_312-LP3-ITS-0312_96_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_312-LP3-ITS-0312_96_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_312-LP3-ITS-0312_96_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.265 s (12.604 ¬µs/read; 4.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,013\n",
      "Reads with adapters:                       328 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,013 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,324,913 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,295,535 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 328 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.3%\n",
      "  C: 50.0%\n",
      "  G: 0.9%\n",
      "  T: 5.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t328.3\t0\t29\n",
      "4\t10\t82.1\t0\t10\n",
      "5\t7\t20.5\t0\t7\n",
      "6\t8\t5.1\t0\t8\n",
      "7\t1\t1.3\t0\t1\n",
      "21\t4\t0.0\t2\t0 1 3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 1\n",
      "27\t10\t0.0\t2\t0 0 10\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "33\t1\t0.0\t2\t0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "40\t25\t0.0\t2\t0 1 24\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "76\t4\t0.0\t2\t0 0 4\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "83\t3\t0.0\t2\t0 0 3\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "92\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t13\t0.0\t2\t0 1 12\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t10\t0.0\t2\t0 0 10\n",
      "107\t3\t0.0\t2\t0 1 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "119\t6\t0.0\t2\t0 0 6\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "125\t4\t0.0\t2\t0 1 3\n",
      "126\t9\t0.0\t2\t0 0 9\n",
      "128\t8\t0.0\t2\t0 0 8\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t9\t0.0\t2\t0 0 9\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "192\t2\t0.0\t2\t0 0 2\n",
      "194\t2\t0.0\t2\t0 0 2\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_313-LP3-ITS-0313_441_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_313-LP3-ITS-0313_441_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_313-LP3-ITS-0313_441_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_313-LP3-ITS-0313_441_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.328 s (12.134 ¬µs/read; 4.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,005\n",
      "Reads with adapters:                       428 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,005 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,128,505 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,088,106 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 428 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.4%\n",
      "  C: 53.5%\n",
      "  G: 1.4%\n",
      "  T: 4.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t422.0\t0\t45\n",
      "4\t22\t105.5\t0\t22\n",
      "5\t9\t26.4\t0\t9\n",
      "6\t6\t6.6\t0\t6\n",
      "7\t1\t1.6\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "27\t16\t0.0\t2\t0 0 16\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "47\t5\t0.0\t2\t0 0 5\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "73\t6\t0.0\t2\t0 0 6\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t8\t0.0\t2\t0 1 7\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t4\t0.0\t2\t0 0 4\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "119\t5\t0.0\t2\t0 0 5\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t4\t0.0\t2\t0 0 4\n",
      "123\t9\t0.0\t2\t0 0 9\n",
      "124\t14\t0.0\t2\t0 0 14\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t6\t0.0\t2\t0 0 6\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "133\t23\t0.0\t2\t0 1 22\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t13\t0.0\t2\t0 0 13\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t11\t0.0\t2\t0 0 11\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "161\t6\t0.0\t2\t0 0 6\n",
      "163\t36\t0.0\t2\t0 0 36\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "168\t2\t0.0\t2\t0 0 2\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "178\t3\t0.0\t2\t0 0 3\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "202\t1\t0.0\t2\t0 0 1\n",
      "227\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_314-LP3-ITS-0314_856_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_314-LP3-ITS-0314_856_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_314-LP3-ITS-0314_856_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_314-LP3-ITS-0314_856_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.413 s (12.067 ¬µs/read; 4.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,262\n",
      "Reads with adapters:                       638 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,312,862 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,257,898 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 638 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.4%\n",
      "  C: 60.5%\n",
      "  G: 0.8%\n",
      "  T: 2.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t535.3\t0\t58\n",
      "4\t49\t133.8\t0\t49\n",
      "5\t26\t33.5\t0\t26\n",
      "6\t4\t8.4\t0\t4\n",
      "7\t3\t2.1\t0\t3\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "18\t1\t0.0\t1\t0 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "23\t8\t0.0\t2\t0 0 8\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "27\t20\t0.0\t2\t0 1 19\n",
      "28\t15\t0.0\t2\t0 1 14\n",
      "31\t3\t0.0\t2\t0 0 3\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "34\t5\t0.0\t2\t0 0 5\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t3\t0.0\t2\t0 1 2\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "85\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "89\t11\t0.0\t2\t0 0 11\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t22\t0.0\t2\t0 1 21\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t4\t0.0\t2\t0 0 4\n",
      "99\t9\t0.0\t2\t0 0 9\n",
      "100\t21\t0.0\t2\t0 0 21\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t10\t0.0\t2\t0 0 10\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t9\t0.0\t2\t0 1 8\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t8\t0.0\t2\t0 0 8\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t12\t0.0\t2\t0 1 11\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t18\t0.0\t2\t0 0 18\n",
      "124\t27\t0.0\t2\t0 0 27\n",
      "125\t9\t0.0\t2\t0 0 9\n",
      "126\t8\t0.0\t2\t0 0 8\n",
      "127\t7\t0.0\t2\t0 0 7\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t6\t0.0\t2\t0 0 6\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t10\t0.0\t2\t0 0 10\n",
      "135\t8\t0.0\t2\t0 0 8\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t13\t0.0\t2\t0 0 13\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t12\t0.0\t2\t0 0 12\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t15\t0.0\t2\t0 0 15\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t8\t0.0\t2\t0 0 8\n",
      "158\t9\t0.0\t2\t0 0 9\n",
      "159\t21\t0.0\t2\t0 0 21\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "166\t2\t0.0\t2\t0 0 2\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "196\t1\t0.0\t2\t0 0 1\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "218\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_315-LP3-ITS-0315_1039_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_315-LP3-ITS-0315_1039_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_315-LP3-ITS-0315_1039_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_315-LP3-ITS-0315_1039_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (33.824 ¬µs/read; 1.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     267\n",
      "Reads with adapters:                         9 (3.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           267 (100.0%)\n",
      "\n",
      "Total basepairs processed:        80,367 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         79,615 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.8%\n",
      "  C: 22.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t1.0\t0\t2\n",
      "7\t1\t0.0\t0\t1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_316-LP3-ITS-0316_448_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_316-LP3-ITS-0316_448_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_316-LP3-ITS-0316_448_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_316-LP3-ITS-0316_448_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.210 s (13.187 ¬µs/read; 4.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,890\n",
      "Reads with adapters:                       377 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,782,890 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,752,738 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 377 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.9%\n",
      "  C: 59.2%\n",
      "  G: 0.8%\n",
      "  T: 2.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t248.3\t0\t30\n",
      "4\t14\t62.1\t0\t14\n",
      "5\t34\t15.5\t0\t34\n",
      "6\t2\t3.9\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t3\t0.0\t1\t0 3\n",
      "13\t3\t0.0\t1\t0 3\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "26\t5\t0.0\t2\t0 0 5\n",
      "27\t7\t0.0\t2\t0 1 6\n",
      "28\t23\t0.0\t2\t0 2 21\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "66\t11\t0.0\t2\t0 1 10\n",
      "68\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t9\t0.0\t2\t0 0 9\n",
      "90\t6\t0.0\t2\t0 0 6\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t5\t0.0\t2\t0 0 5\n",
      "98\t10\t0.0\t2\t0 0 10\n",
      "99\t6\t0.0\t2\t0 0 6\n",
      "100\t4\t0.0\t2\t0 0 4\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t12\t0.0\t2\t0 0 12\n",
      "105\t5\t0.0\t2\t0 1 4\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t18\t0.0\t2\t0 0 18\n",
      "124\t11\t0.0\t2\t0 0 11\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t6\t0.0\t2\t0 1 5\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t2\t0.0\t2\t0 1 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 1 6\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "248\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_317-LP3-ITS-0317_201_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_317-LP3-ITS-0317_201_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_317-LP3-ITS-0317_201_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_317-LP3-ITS-0317_201_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.010 s (65.979 ¬µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     144\n",
      "Reads with adapters:                         6 (4.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           144 (100.0%)\n",
      "\n",
      "Total basepairs processed:        43,344 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         42,639 bp (98.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_318-LP3-ITS-0318_302_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_318-LP3-ITS-0318_302_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_318-LP3-ITS-0318_302_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_318-LP3-ITS-0318_302_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.415 s (12.521 ¬µs/read; 4.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,111\n",
      "Reads with adapters:                       634 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,111 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,966,411 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,909,539 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 634 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 46.4%\n",
      "  C: 49.4%\n",
      "  G: 2.4%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t517.4\t0\t46\n",
      "4\t17\t129.3\t0\t17\n",
      "5\t14\t32.3\t0\t14\n",
      "6\t6\t8.1\t0\t6\n",
      "7\t7\t2.0\t0\t7\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "22\t17\t0.0\t2\t0 0 17\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t17\t0.0\t2\t0 0 17\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "35\t3\t0.0\t2\t0 0 3\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t45\t0.0\t2\t0 1 44\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t6\t0.0\t2\t0 0 6\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t19\t0.0\t2\t0 0 19\n",
      "63\t17\t0.0\t2\t0 1 16\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t4\t0.0\t2\t0 1 3\n",
      "80\t5\t0.0\t2\t0 0 5\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t3\t0.0\t2\t0 0 3\n",
      "88\t3\t0.0\t2\t0 0 3\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 1 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t45\t0.0\t2\t0 1 44\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t7\t0.0\t2\t0 0 7\n",
      "110\t4\t0.0\t2\t0 0 4\n",
      "111\t9\t0.0\t2\t0 0 9\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t3\t0.0\t2\t0 0 3\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t10\t0.0\t2\t0 0 10\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t8\t0.0\t2\t0 0 8\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t22\t0.0\t2\t0 0 22\n",
      "127\t8\t0.0\t2\t0 0 8\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t30\t0.0\t2\t0 1 29\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t18\t0.0\t2\t0 0 18\n",
      "138\t8\t0.0\t2\t0 0 8\n",
      "139\t16\t0.0\t2\t0 0 16\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "144\t7\t0.0\t2\t0 0 7\n",
      "147\t9\t0.0\t2\t0 0 9\n",
      "148\t5\t0.0\t2\t0 0 5\n",
      "149\t12\t0.0\t2\t0 0 12\n",
      "150\t41\t0.0\t2\t0 2 39\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t4\t0.0\t2\t0 0 4\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t17\t0.0\t2\t0 0 17\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "178\t4\t0.0\t2\t0 0 4\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_319-LP3-ITS-0319_1118_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_319-LP3-ITS-0319_1118_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_319-LP3-ITS-0319_1118_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_319-LP3-ITS-0319_1118_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (32.467 ¬µs/read; 1.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     251\n",
      "Reads with adapters:                        10 (4.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           251 (100.0%)\n",
      "\n",
      "Total basepairs processed:        75,551 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         74,661 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.0%\n",
      "  C: 40.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t3.9\t0\t2\n",
      "4\t1\t1.0\t0\t1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_320-LP3-ITS-0320_821_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_320-LP3-ITS-0320_821_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_320-LP3-ITS-0320_821_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_320-LP3-ITS-0320_821_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (70.802 ¬µs/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      91\n",
      "Reads with adapters:                         1 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            91 (100.0%)\n",
      "\n",
      "Total basepairs processed:        27,391 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         27,272 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_321-LP3-ITS-0321_704_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_321-LP3-ITS-0321_704_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_321-LP3-ITS-0321_704_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_321-LP3-ITS-0321_704_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.369 s (11.933 ¬µs/read; 5.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,955\n",
      "Reads with adapters:                       782 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,955 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,317,455 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,259,490 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 782 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.1%\n",
      "  C: 59.8%\n",
      "  G: 1.5%\n",
      "  T: 5.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t483.7\t0\t57\n",
      "4\t5\t120.9\t0\t5\n",
      "5\t8\t30.2\t0\t8\n",
      "6\t1\t7.6\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "26\t3\t0.0\t2\t0 0 3\n",
      "27\t19\t0.0\t2\t0 0 19\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "29\t4\t0.0\t2\t0 0 4\n",
      "32\t1\t0.0\t2\t0 1\n",
      "34\t5\t0.0\t2\t0 0 5\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "38\t7\t0.0\t2\t0 0 7\n",
      "44\t6\t0.0\t2\t0 0 6\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "47\t302\t0.0\t2\t0 4 298\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t8\t0.0\t2\t0 0 8\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 1 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t11\t0.0\t2\t0 0 11\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "67\t3\t0.0\t2\t0 1 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t2\t0.0\t2\t0 1 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t10\t0.0\t2\t0 0 10\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "102\t11\t0.0\t2\t0 0 11\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t22\t0.0\t2\t0 0 22\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "111\t3\t0.0\t2\t0 1 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "118\t16\t0.0\t2\t0 0 16\n",
      "119\t13\t0.0\t2\t0 0 13\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t16\t0.0\t2\t0 0 16\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t28\t0.0\t2\t0 0 28\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t40\t0.0\t2\t0 0 40\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "147\t14\t0.0\t2\t0 1 13\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "161\t2\t0.0\t2\t0 0 2\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "188\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_322-LP3-ITS-0322_414_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_322-LP3-ITS-0322_414_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_322-LP3-ITS-0322_414_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_322-LP3-ITS-0322_414_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.309 s (10.805 ¬µs/read; 5.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,571\n",
      "Reads with adapters:                       399 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,571 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,599,871 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,555,324 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 399 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 54.4%\n",
      "  C: 43.9%\n",
      "  G: 1.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t446.4\t0\t9\n",
      "4\t5\t111.6\t0\t5\n",
      "5\t5\t27.9\t0\t5\n",
      "6\t3\t7.0\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t3\t0.0\t2\t0 0 3\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "25\t2\t0.0\t2\t0 1 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t6\t0.0\t2\t0 1 5\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "88\t3\t0.0\t2\t0 0 3\n",
      "89\t6\t0.0\t2\t0 0 6\n",
      "90\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 1\n",
      "96\t12\t0.0\t2\t0 0 12\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "104\t11\t0.0\t2\t0 0 11\n",
      "105\t2\t0.0\t2\t0 1 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "108\t5\t0.0\t2\t0 0 5\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 1 2\n",
      "124\t12\t0.0\t2\t0 0 12\n",
      "125\t26\t0.0\t2\t0 0 26\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t20\t0.0\t2\t0 0 20\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "147\t39\t0.0\t2\t0 0 39\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "154\t11\t0.0\t2\t0 0 11\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t15\t0.0\t2\t0 0 15\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "172\t2\t0.0\t2\t0 0 2\n",
      "178\t4\t0.0\t2\t0 0 4\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "257\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_323-LP3-ITS-0323_117_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_323-LP3-ITS-0323_117_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_323-LP3-ITS-0323_117_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_323-LP3-ITS-0323_117_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.578 s (11.680 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  49,468\n",
      "Reads with adapters:                       841 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        49,468 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,889,868 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,811,642 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 841 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.2%\n",
      "  C: 47.3%\n",
      "  G: 1.3%\n",
      "  T: 2.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t61\t772.9\t0\t61\n",
      "4\t15\t193.2\t0\t15\n",
      "5\t10\t48.3\t0\t10\n",
      "6\t10\t12.1\t0\t10\n",
      "7\t4\t3.0\t0\t4\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t40\t0.0\t2\t0 0 40\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t24\t0.0\t2\t0 0 24\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "44\t5\t0.0\t2\t0 0 5\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t4\t0.0\t2\t0 0 4\n",
      "51\t10\t0.0\t2\t0 0 10\n",
      "53\t11\t0.0\t2\t0 0 11\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t5\t0.0\t2\t0 0 5\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t4\t0.0\t2\t0 0 4\n",
      "80\t9\t0.0\t2\t0 1 8\n",
      "82\t3\t0.0\t2\t0 0 3\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t3\t0.0\t2\t0 0 3\n",
      "87\t17\t0.0\t2\t0 0 17\n",
      "88\t4\t0.0\t2\t0 0 4\n",
      "89\t15\t0.0\t2\t0 0 15\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "91\t6\t0.0\t2\t0 0 6\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t16\t0.0\t2\t0 0 16\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "100\t24\t0.0\t2\t0 0 24\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t23\t0.0\t2\t0 0 23\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t6\t0.0\t2\t0 1 5\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "110\t16\t0.0\t2\t0 0 16\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "112\t5\t0.0\t2\t0 1 4\n",
      "114\t5\t0.0\t2\t0 0 5\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t9\t0.0\t2\t0 0 9\n",
      "118\t8\t0.0\t2\t0 0 8\n",
      "119\t7\t0.0\t2\t0 0 7\n",
      "120\t6\t0.0\t2\t0 0 6\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t13\t0.0\t2\t0 0 13\n",
      "124\t15\t0.0\t2\t0 0 15\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t59\t0.0\t2\t0 1 58\n",
      "127\t6\t0.0\t2\t0 0 6\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t27\t0.0\t2\t0 1 26\n",
      "133\t97\t0.0\t2\t0 1 96\n",
      "137\t23\t0.0\t2\t0 0 23\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t5\t0.0\t2\t0 0 5\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "146\t5\t0.0\t2\t0 0 5\n",
      "147\t14\t0.0\t2\t0 0 14\n",
      "148\t22\t0.0\t2\t0 2 20\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "173\t2\t0.0\t2\t0 0 2\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "178\t8\t0.0\t2\t0 0 8\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "192\t1\t0.0\t2\t0 0 1\n",
      "196\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_324-LP3-ITS-0324_884_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_324-LP3-ITS-0324_884_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_324-LP3-ITS-0324_884_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_324-LP3-ITS-0324_884_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.365 s (11.258 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,453\n",
      "Reads with adapters:                       589 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,453 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,768,353 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,706,963 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 589 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 59.1%\n",
      "  C: 37.9%\n",
      "  G: 0.3%\n",
      "  T: 2.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t507.1\t0\t18\n",
      "4\t11\t126.8\t0\t11\n",
      "5\t6\t31.7\t0\t6\n",
      "6\t30\t7.9\t0\t30\n",
      "8\t1\t0.5\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "22\t7\t0.0\t2\t0 1 6\n",
      "27\t13\t0.0\t2\t0 0 13\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t10\t0.0\t2\t0 0 10\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t11\t0.0\t2\t0 0 11\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "87\t5\t0.0\t2\t0 0 5\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t4\t0.0\t2\t0 0 4\n",
      "90\t4\t0.0\t2\t0 0 4\n",
      "92\t3\t0.0\t2\t0 0 3\n",
      "93\t12\t0.0\t2\t0 0 12\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t14\t0.0\t2\t0 0 14\n",
      "97\t7\t0.0\t2\t0 0 7\n",
      "98\t14\t0.0\t2\t0 0 14\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t30\t0.0\t2\t0 1 29\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t7\t0.0\t2\t0 0 7\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t14\t0.0\t2\t0 0 14\n",
      "119\t8\t0.0\t2\t0 0 8\n",
      "120\t9\t0.0\t2\t0 0 9\n",
      "121\t4\t0.0\t2\t0 0 4\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t10\t0.0\t2\t0 0 10\n",
      "126\t15\t0.0\t2\t0 0 15\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t41\t0.0\t2\t0 0 41\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "135\t7\t0.0\t2\t0 0 7\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "138\t5\t0.0\t2\t0 1 4\n",
      "139\t27\t0.0\t2\t0 0 27\n",
      "140\t4\t0.0\t2\t0 0 4\n",
      "141\t1\t0.0\t2\t0 1\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t8\t0.0\t2\t0 0 8\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "147\t9\t0.0\t2\t0 0 9\n",
      "148\t10\t0.0\t2\t0 0 10\n",
      "149\t9\t0.0\t2\t0 0 9\n",
      "150\t18\t0.0\t2\t0 0 18\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "154\t4\t0.0\t2\t0 0 4\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t7\t0.0\t2\t0 0 7\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t10\t0.0\t2\t0 0 10\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t3\t0.0\t2\t0 0 3\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "171\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "188\t2\t0.0\t2\t0 0 2\n",
      "189\t3\t0.0\t2\t0 0 3\n",
      "209\t1\t0.0\t2\t0 1\n",
      "238\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_325-LP3-ITS-0325_305_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_325-LP3-ITS-0325_305_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_325-LP3-ITS-0325_305_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_325-LP3-ITS-0325_305_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.193 s (12.663 ¬µs/read; 4.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,264\n",
      "Reads with adapters:                       288 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,264 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,594,464 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,567,187 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 288 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.1%\n",
      "  C: 53.5%\n",
      "  G: 0.7%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t238.5\t0\t20\n",
      "4\t7\t59.6\t0\t7\n",
      "5\t10\t14.9\t0\t10\n",
      "6\t2\t3.7\t0\t2\n",
      "11\t2\t0.0\t1\t0 2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "23\t4\t0.0\t2\t0 0 4\n",
      "27\t16\t0.0\t2\t0 0 16\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t7\t0.0\t2\t0 0 7\n",
      "100\t12\t0.0\t2\t0 0 12\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t17\t0.0\t2\t0 1 16\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t8\t0.0\t2\t0 0 8\n",
      "125\t67\t0.0\t2\t0 1 66\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t8\t0.0\t2\t0 0 8\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "154\t4\t0.0\t2\t0 0 4\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_326-LP3-ITS-0326_978_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_326-LP3-ITS-0326_978_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_326-LP3-ITS-0326_978_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_326-LP3-ITS-0326_978_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.573 s (10.705 ¬µs/read; 5.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,571\n",
      "Reads with adapters:                       781 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,571 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,124,871 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,037,974 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 781 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.0%\n",
      "  C: 18.1%\n",
      "  G: 0.6%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t837.0\t0\t1\n",
      "25\t20\t0.0\t2\t0 0 20\n",
      "26\t22\t0.0\t2\t0 2 20\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "58\t9\t0.0\t2\t0 0 9\n",
      "63\t30\t0.0\t2\t0 0 30\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t182\t0.0\t2\t0 2 180\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t24\t0.0\t2\t0 0 24\n",
      "109\t85\t0.0\t2\t0 0 85\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t5\t0.0\t2\t0 0 5\n",
      "114\t8\t0.0\t2\t0 0 8\n",
      "115\t8\t0.0\t2\t0 0 8\n",
      "116\t5\t0.0\t2\t0 0 5\n",
      "118\t21\t0.0\t2\t0 0 21\n",
      "120\t7\t0.0\t2\t0 0 7\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "123\t196\t0.0\t2\t0 3 193\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t7\t0.0\t2\t0 0 7\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t10\t0.0\t2\t0 0 10\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "163\t57\t0.0\t2\t0 0 57\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "198\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_327-LP3-ITS-0327_45_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_327-LP3-ITS-0327_45_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_327-LP3-ITS-0327_45_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_327-LP3-ITS-0327_45_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.444 s (11.616 ¬µs/read; 5.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,262\n",
      "Reads with adapters:                       594 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,516,862 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,453,297 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 594 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.5%\n",
      "  C: 56.6%\n",
      "  G: 1.2%\n",
      "  T: 6.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t597.8\t0\t27\n",
      "4\t17\t149.5\t0\t17\n",
      "5\t5\t37.4\t0\t5\n",
      "6\t2\t9.3\t0\t2\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "21\t3\t0.0\t2\t0 0 3\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t10\t0.0\t2\t0 0 10\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "63\t13\t0.0\t2\t0 1 12\n",
      "65\t5\t0.0\t2\t0 0 5\n",
      "66\t18\t0.0\t2\t0 0 18\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t12\t0.0\t2\t0 0 12\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t56\t0.0\t2\t0 0 56\n",
      "94\t10\t0.0\t2\t0 0 10\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 1\n",
      "97\t4\t0.0\t2\t0 0 4\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t8\t0.0\t2\t0 0 8\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t8\t0.0\t2\t0 0 8\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "116\t10\t0.0\t2\t0 0 10\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t10\t0.0\t2\t0 0 10\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t43\t0.0\t2\t0 0 43\n",
      "134\t33\t0.0\t2\t0 0 33\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t9\t0.0\t2\t0 0 9\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "142\t31\t0.0\t2\t0 0 31\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "151\t15\t0.0\t2\t0 0 15\n",
      "152\t14\t0.0\t2\t0 0 14\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "156\t11\t0.0\t2\t0 0 11\n",
      "157\t23\t0.0\t2\t0 0 23\n",
      "158\t7\t0.0\t2\t0 0 7\n",
      "159\t23\t0.0\t2\t0 0 23\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "207\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_328-LP3-ITS-0328_328_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_328-LP3-ITS-0328_328_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_328-LP3-ITS-0328_328_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_328-LP3-ITS-0328_328_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.094 s (12.465 ¬µs/read; 4.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,574\n",
      "Reads with adapters:                       169 (2.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,574 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,279,774 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,266,164 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 169 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.1%\n",
      "  C: 66.3%\n",
      "  G: 2.4%\n",
      "  T: 8.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t118.3\t0\t14\n",
      "4\t3\t29.6\t0\t3\n",
      "5\t34\t7.4\t0\t34\n",
      "6\t3\t1.8\t0\t3\n",
      "7\t1\t0.5\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t1 0 3\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "56\t11\t0.0\t2\t0 0 11\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_329-LP3-ITS-0329_737_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_329-LP3-ITS-0329_737_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_329-LP3-ITS-0329_737_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_329-LP3-ITS-0329_737_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.046 s (11.766 ¬µs/read; 5.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   3,905\n",
      "Reads with adapters:                        67 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         3,905 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,175,405 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,170,085 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 67 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 7.5%\n",
      "  C: 91.0%\n",
      "  G: 0.0%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t37\t0.0\t2\t0 0 37\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "100\t19\t0.0\t2\t0 0 19\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_330-LP3-ITS-0330_330_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_330-LP3-ITS-0330_330_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_330-LP3-ITS-0330_330_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_330-LP3-ITS-0330_330_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.292 s (12.017 ¬µs/read; 4.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,285\n",
      "Reads with adapters:                       421 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,309,785 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,263,830 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 421 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.6%\n",
      "  C: 46.3%\n",
      "  G: 0.7%\n",
      "  T: 3.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t379.5\t0\t17\n",
      "4\t1\t94.9\t0\t1\n",
      "5\t2\t23.7\t0\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t3\t0.0\t2\t0 0 3\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t9\t0.0\t2\t0 0 9\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "58\t31\t0.0\t2\t0 0 31\n",
      "59\t9\t0.0\t2\t0 0 9\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t11\t0.0\t2\t0 1 10\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "83\t3\t0.0\t2\t0 0 3\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t8\t0.0\t2\t0 1 7\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t10\t0.0\t2\t0 0 10\n",
      "120\t7\t0.0\t2\t0 0 7\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t9\t0.0\t2\t0 0 9\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t11\t0.0\t2\t0 0 11\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t27\t0.0\t2\t0 2 25\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t5\t0.0\t2\t0 0 5\n",
      "137\t17\t0.0\t2\t0 1 16\n",
      "138\t9\t0.0\t2\t0 0 9\n",
      "139\t12\t0.0\t2\t0 0 12\n",
      "140\t3\t0.0\t2\t0 0 3\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "150\t14\t0.0\t2\t0 0 14\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "156\t22\t0.0\t2\t0 0 22\n",
      "157\t12\t0.0\t2\t0 0 12\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t11\t0.0\t2\t0 0 11\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 1\n",
      "191\t2\t0.0\t2\t0 0 2\n",
      "196\t1\t0.0\t2\t0 0 1\n",
      "198\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_331-LP3-ITS-0331_307_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_331-LP3-ITS-0331_307_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_331-LP3-ITS-0331_307_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_331-LP3-ITS-0331_307_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.238 s (11.801 ¬µs/read; 5.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,156\n",
      "Reads with adapters:                       507 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,156 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,066,956 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,022,735 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 507 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.4%\n",
      "  C: 64.9%\n",
      "  G: 0.2%\n",
      "  T: 7.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t314.9\t0\t5\n",
      "4\t1\t78.7\t0\t1\n",
      "5\t56\t19.7\t0\t56\n",
      "6\t1\t4.9\t0\t1\n",
      "8\t1\t0.3\t0\t1\n",
      "27\t20\t0.0\t2\t0 2 18\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "38\t11\t0.0\t2\t0 1 10\n",
      "43\t19\t0.0\t2\t0 0 19\n",
      "44\t21\t0.0\t2\t0 1 20\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t12\t0.0\t2\t0 0 12\n",
      "62\t9\t0.0\t2\t0 0 9\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "64\t9\t0.0\t2\t0 0 9\n",
      "66\t29\t0.0\t2\t0 1 28\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "73\t9\t0.0\t2\t0 0 9\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "78\t5\t0.0\t2\t0 0 5\n",
      "80\t9\t0.0\t2\t0 0 9\n",
      "81\t4\t0.0\t2\t0 0 4\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t5\t0.0\t2\t0 0 5\n",
      "91\t5\t0.0\t2\t0 0 5\n",
      "94\t8\t0.0\t2\t0 1 7\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t7\t0.0\t2\t0 0 7\n",
      "99\t1\t0.0\t2\t0 1\n",
      "104\t23\t0.0\t2\t0 0 23\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t3\t0.0\t2\t0 1 2\n",
      "107\t12\t0.0\t2\t0 1 11\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "119\t5\t0.0\t2\t0 0 5\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t16\t0.0\t2\t0 1 15\n",
      "127\t6\t0.0\t2\t0 0 6\n",
      "128\t7\t0.0\t2\t0 0 7\n",
      "129\t4\t0.0\t2\t0 0 4\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t7\t0.0\t2\t0 0 7\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "142\t33\t0.0\t2\t0 0 33\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t10\t0.0\t2\t0 1 9\n",
      "152\t6\t0.0\t2\t0 1 5\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t11\t0.0\t2\t0 0 11\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "170\t4\t0.0\t2\t0 0 4\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "203\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_332-LP3-ITS-0332_189_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_332-LP3-ITS-0332_189_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_332-LP3-ITS-0332_189_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_332-LP3-ITS-0332_189_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.150 s (9.099 ¬µs/read; 6.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,500\n",
      "Reads with adapters:                     1,250 (7.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,500 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,966,500 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,958,430 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1250 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.8%\n",
      "  C: 8.7%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1216\t257.8\t0\t1216\n",
      "4\t8\t64.5\t0\t8\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "176\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_333-LP3-ITS-0333_955_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_333-LP3-ITS-0333_955_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_333-LP3-ITS-0333_955_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_333-LP3-ITS-0333_955_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.182 s (11.451 ¬µs/read; 5.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,914\n",
      "Reads with adapters:                       285 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,914 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,790,114 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,768,251 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 285 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.2%\n",
      "  C: 51.6%\n",
      "  G: 4.2%\n",
      "  T: 6.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t248.7\t0\t50\n",
      "4\t6\t62.2\t0\t6\n",
      "5\t7\t15.5\t0\t7\n",
      "6\t3\t3.9\t0\t3\n",
      "8\t2\t0.2\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "45\t31\t0.0\t2\t0 1 30\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t6\t0.0\t2\t0 0 6\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t6\t0.0\t2\t0 1 5\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "66\t6\t0.0\t2\t0 0 6\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "105\t8\t0.0\t2\t0 0 8\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_334-LP3-ITS-0334_472_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_334-LP3-ITS-0334_472_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_334-LP3-ITS-0334_472_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_334-LP3-ITS-0334_472_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.100 s (14.419 ¬µs/read; 4.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,953\n",
      "Reads with adapters:                       157 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         6,953 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,092,853 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,080,638 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 157 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.2%\n",
      "  C: 57.3%\n",
      "  G: 0.6%\n",
      "  T: 10.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t108.6\t0\t15\n",
      "4\t1\t27.2\t0\t1\n",
      "5\t24\t6.8\t0\t24\n",
      "7\t1\t0.4\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "30\t7\t0.0\t2\t0 0 7\n",
      "33\t3\t0.0\t2\t0 0 3\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t5\t0.0\t2\t0 0 5\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "94\t5\t0.0\t2\t0 0 5\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t9\t0.0\t2\t0 0 9\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_335-LP3-ITS-0335_292_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_335-LP3-ITS-0335_292_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_335-LP3-ITS-0335_292_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_335-LP3-ITS-0335_292_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.349 s (11.785 ¬µs/read; 5.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,594\n",
      "Reads with adapters:                       438 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,594 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,907,794 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,861,691 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 438 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.0%\n",
      "  C: 57.8%\n",
      "  G: 0.9%\n",
      "  T: 7.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t462.4\t0\t27\n",
      "4\t4\t115.6\t0\t4\n",
      "5\t7\t28.9\t0\t7\n",
      "7\t1\t1.8\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "36\t2\t0.0\t2\t0 0 2\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t7\t0.0\t2\t0 1 6\n",
      "56\t6\t0.0\t2\t0 0 6\n",
      "58\t5\t0.0\t2\t0 1 4\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "62\t5\t0.0\t2\t0 0 5\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t8\t0.0\t2\t0 0 8\n",
      "69\t3\t0.0\t2\t0 0 3\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "80\t2\t0.0\t2\t0 0 2\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t6\t0.0\t2\t0 0 6\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t2\t0.0\t2\t0 1 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t8\t0.0\t2\t0 0 8\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "119\t5\t0.0\t2\t0 0 5\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "127\t9\t0.0\t2\t0 0 9\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t49\t0.0\t2\t0 0 49\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t6\t0.0\t2\t0 0 6\n",
      "137\t18\t0.0\t2\t0 0 18\n",
      "138\t6\t0.0\t2\t0 0 6\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t26\t0.0\t2\t0 0 26\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t5\t0.0\t2\t0 0 5\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "156\t15\t0.0\t2\t0 0 15\n",
      "157\t6\t0.0\t2\t0 0 6\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "166\t3\t0.0\t2\t0 0 3\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_336-LP3-ITS-0336_1030_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_336-LP3-ITS-0336_1030_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_336-LP3-ITS-0336_1030_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_336-LP3-ITS-0336_1030_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.166 s (11.652 ¬µs/read; 5.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,268\n",
      "Reads with adapters:                       215 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,294,668 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,279,029 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 215 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.4%\n",
      "  C: 57.2%\n",
      "  G: 2.8%\n",
      "  T: 11.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t222.9\t0\t35\n",
      "4\t6\t55.7\t0\t6\n",
      "5\t13\t13.9\t0\t13\n",
      "6\t1\t3.5\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t9\t0.0\t2\t0 1 8\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "58\t3\t0.0\t2\t0 1 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "66\t13\t0.0\t2\t0 1 12\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t6\t0.0\t2\t0 0 6\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "170\t3\t0.0\t2\t0 0 3\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_337-LP3-ITS-0337_15_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_337-LP3-ITS-0337_15_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_337-LP3-ITS-0337_15_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_337-LP3-ITS-0337_15_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.164 s (12.002 ¬µs/read; 5.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,663\n",
      "Reads with adapters:                       282 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,663 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,112,563 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,093,015 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 282 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.7%\n",
      "  C: 64.2%\n",
      "  G: 1.8%\n",
      "  T: 11.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t213.5\t0\t18\n",
      "4\t10\t53.4\t0\t10\n",
      "5\t61\t13.3\t0\t61\n",
      "6\t1\t3.3\t0\t1\n",
      "7\t1\t0.8\t0\t1\n",
      "8\t5\t0.2\t0\t5\n",
      "11\t1\t0.0\t1\t0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "32\t4\t0.0\t2\t0 1 3\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "45\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t4\t0.0\t2\t0 0 4\n",
      "57\t1\t0.0\t2\t0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t6\t0.0\t2\t0 0 6\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t12\t0.0\t2\t0 0 12\n",
      "73\t8\t0.0\t2\t0 0 8\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t9\t0.0\t2\t0 0 9\n",
      "106\t11\t0.0\t2\t0 1 10\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "142\t26\t0.0\t2\t0 0 26\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "170\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_338-LP3-ITS-0338_496_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_338-LP3-ITS-0338_496_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_338-LP3-ITS-0338_496_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_338-LP3-ITS-0338_496_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.220 s (10.430 ¬µs/read; 5.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,066\n",
      "Reads with adapters:                       203 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,066 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,340,866 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,315,254 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 203 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.9%\n",
      "  C: 20.7%\n",
      "  G: 1.0%\n",
      "  T: 6.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t329.2\t0\t2\n",
      "4\t1\t82.3\t0\t1\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "66\t5\t0.0\t2\t0 0 5\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t4\t0.0\t2\t0 0 4\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t18\t0.0\t2\t0 0 18\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t92\t0.0\t2\t0 1 91\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_339-LP3-ITS-0339_266_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_339-LP3-ITS-0339_266_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_339-LP3-ITS-0339_266_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_339-LP3-ITS-0339_266_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.117 s (11.735 ¬µs/read; 5.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,009\n",
      "Reads with adapters:                       134 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        10,009 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,012,709 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,998,796 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 134 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.3%\n",
      "  C: 53.7%\n",
      "  G: 1.5%\n",
      "  T: 7.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t156.4\t0\t11\n",
      "4\t1\t39.1\t0\t1\n",
      "5\t1\t9.8\t0\t1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t2\t0.0\t2\t0 1 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t7\t0.0\t2\t0 0 7\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_340-LP3-ITS-0340_387_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_340-LP3-ITS-0340_387_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_340-LP3-ITS-0340_387_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_340-LP3-ITS-0340_387_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.230 s (13.536 ¬µs/read; 4.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,996\n",
      "Reads with adapters:                       331 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,996 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,115,796 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,082,813 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 331 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.9%\n",
      "  C: 49.5%\n",
      "  G: 1.8%\n",
      "  T: 5.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t265.6\t0\t16\n",
      "4\t7\t66.4\t0\t7\n",
      "5\t5\t16.6\t0\t5\n",
      "6\t2\t4.1\t0\t2\n",
      "7\t1\t1.0\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t6\t0.0\t2\t0 0 6\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "56\t14\t0.0\t2\t0 0 14\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t5\t0.0\t2\t0 0 5\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t6\t0.0\t2\t0 1 5\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t4\t0.0\t2\t0 0 4\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "75\t6\t0.0\t2\t0 0 6\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t5\t0.0\t2\t0 0 5\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "94\t7\t0.0\t2\t0 0 7\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t7\t0.0\t2\t0 0 7\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t13\t0.0\t2\t0 0 13\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t11\t0.0\t2\t0 1 10\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "140\t3\t0.0\t2\t0 0 3\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "153\t2\t0.0\t2\t0 1 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "156\t20\t0.0\t2\t0 0 20\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_341-LP3-ITS-0341_562_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_341-LP3-ITS-0341_562_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_341-LP3-ITS-0341_562_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_341-LP3-ITS-0341_562_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.084 s (12.478 ¬µs/read; 4.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,717\n",
      "Reads with adapters:                       100 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         6,717 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,021,817 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,012,770 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 100 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.0%\n",
      "  C: 60.0%\n",
      "  G: 3.0%\n",
      "  T: 9.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t105.0\t0\t3\n",
      "5\t1\t6.6\t0\t1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "33\t6\t0.0\t2\t0 0 6\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "50\t8\t0.0\t2\t0 0 8\n",
      "53\t6\t0.0\t2\t0 0 6\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "66\t13\t0.0\t2\t0 0 13\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "109\t12\t0.0\t2\t0 0 12\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_342-LP3-ITS-0342_491_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_342-LP3-ITS-0342_491_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_342-LP3-ITS-0342_491_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_342-LP3-ITS-0342_491_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.222 s (13.270 ¬µs/read; 4.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,745\n",
      "Reads with adapters:                       327 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,745 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,040,245 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,010,851 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 327 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.5%\n",
      "  C: 55.0%\n",
      "  G: 2.4%\n",
      "  T: 4.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t261.6\t0\t32\n",
      "4\t4\t65.4\t0\t4\n",
      "5\t7\t16.4\t0\t7\n",
      "6\t3\t4.1\t0\t3\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "27\t10\t0.0\t2\t0 0 10\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t2\t0.0\t2\t0 1 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "45\t8\t0.0\t2\t0 1 7\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "56\t7\t0.0\t2\t0 1 6\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t7\t0.0\t2\t0 0 7\n",
      "60\t5\t0.0\t2\t0 0 5\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t5\t0.0\t2\t0 0 5\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t5\t0.0\t2\t0 0 5\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t10\t0.0\t2\t0 0 10\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t7\t0.0\t2\t0 0 7\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t4\t0.0\t2\t0 0 4\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "124\t9\t0.0\t2\t0 0 9\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t6\t0.0\t2\t0 0 6\n",
      "139\t13\t0.0\t2\t0 0 13\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t13\t0.0\t2\t0 0 13\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t14\t0.0\t2\t0 0 14\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t13\t0.0\t2\t0 0 13\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_343-LP3-ITS-0343_436_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_343-LP3-ITS-0343_436_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_343-LP3-ITS-0343_436_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_343-LP3-ITS-0343_436_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.382 s (12.432 ¬µs/read; 4.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,740\n",
      "Reads with adapters:                       561 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,740 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,252,740 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,196,307 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 561 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.0%\n",
      "  C: 48.8%\n",
      "  G: 1.4%\n",
      "  T: 10.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t480.3\t0\t43\n",
      "4\t8\t120.1\t0\t8\n",
      "5\t6\t30.0\t0\t6\n",
      "6\t2\t7.5\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t7\t0.0\t2\t0 0 7\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t6\t0.0\t2\t0 1 5\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "30\t3\t0.0\t2\t0 0 3\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "38\t5\t0.0\t2\t0 0 5\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t8\t0.0\t2\t0 0 8\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t6\t0.0\t2\t0 0 6\n",
      "56\t14\t0.0\t2\t0 0 14\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t8\t0.0\t2\t0 0 8\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t4\t0.0\t2\t0 1 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "64\t4\t0.0\t2\t0 0 4\n",
      "66\t13\t0.0\t2\t0 1 12\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t5\t0.0\t2\t0 0 5\n",
      "75\t3\t0.0\t2\t0 0 3\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t4\t0.0\t2\t0 0 4\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t4\t0.0\t2\t0 0 4\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t8\t0.0\t2\t0 0 8\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t7\t0.0\t2\t0 0 7\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t5\t0.0\t2\t0 0 5\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t4\t0.0\t2\t0 0 4\n",
      "114\t4\t0.0\t2\t0 0 4\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "127\t6\t0.0\t2\t0 0 6\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t8\t0.0\t2\t0 0 8\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t7\t0.0\t2\t0 0 7\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "138\t18\t0.0\t2\t0 0 18\n",
      "139\t12\t0.0\t2\t0 0 12\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t45\t0.0\t2\t0 0 45\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "144\t9\t0.0\t2\t0 0 9\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t16\t0.0\t2\t0 1 15\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "156\t19\t0.0\t2\t0 0 19\n",
      "157\t9\t0.0\t2\t0 0 9\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t10\t0.0\t2\t0 1 9\n",
      "162\t4\t0.0\t2\t0 0 4\n",
      "163\t4\t0.0\t2\t0 0 4\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "172\t4\t0.0\t2\t0 0 4\n",
      "177\t1\t0.0\t2\t0 1\n",
      "181\t2\t0.0\t2\t0 0 2\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_344-LP3-ITS-0344_377_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_344-LP3-ITS-0344_377_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_344-LP3-ITS-0344_377_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_344-LP3-ITS-0344_377_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (56.561 ¬µs/read; 1.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     116\n",
      "Reads with adapters:                         3 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           116 (100.0%)\n",
      "\n",
      "Total basepairs processed:        34,916 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         34,694 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_345-LP3-ITS-0345_112_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_345-LP3-ITS-0345_112_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_345-LP3-ITS-0345_112_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_345-LP3-ITS-0345_112_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.402 s (11.620 ¬µs/read; 5.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,635\n",
      "Reads with adapters:                       578 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,635 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,425,135 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,371,006 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 578 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.3%\n",
      "  C: 60.0%\n",
      "  G: 2.4%\n",
      "  T: 7.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t541.2\t0\t46\n",
      "4\t21\t135.3\t0\t21\n",
      "5\t25\t33.8\t0\t25\n",
      "6\t3\t8.5\t0\t3\n",
      "7\t1\t2.1\t0\t1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t6\t0.0\t2\t0 0 6\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t3\t0.0\t2\t0 1 2\n",
      "64\t1\t0.0\t2\t0 1\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "73\t45\t0.0\t2\t0 1 44\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "75\t57\t0.0\t2\t0 1 56\n",
      "78\t4\t0.0\t2\t0 0 4\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "98\t10\t0.0\t2\t0 0 10\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "104\t15\t0.0\t2\t0 0 15\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "108\t9\t0.0\t2\t0 1 8\n",
      "109\t21\t0.0\t2\t0 0 21\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t7\t0.0\t2\t0 0 7\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t9\t0.0\t2\t0 0 9\n",
      "124\t24\t0.0\t2\t0 0 24\n",
      "125\t2\t0.0\t2\t0 1 1\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t9\t0.0\t2\t0 0 9\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "135\t4\t0.0\t2\t0 0 4\n",
      "136\t5\t0.0\t2\t0 0 5\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t29\t0.0\t2\t0 0 29\n",
      "144\t33\t0.0\t2\t0 1 32\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t12\t0.0\t2\t0 0 12\n",
      "152\t7\t0.0\t2\t0 0 7\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t7\t0.0\t2\t0 0 7\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t11\t0.0\t2\t0 0 11\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t5\t0.0\t2\t0 0 5\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "193\t2\t0.0\t2\t0 0 2\n",
      "207\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_346-LP3-ITS-0346_982_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_346-LP3-ITS-0346_982_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_346-LP3-ITS-0346_982_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_346-LP3-ITS-0346_982_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.294 s (11.031 ¬µs/read; 5.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,632\n",
      "Reads with adapters:                       569 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,632 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,016,232 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,978,901 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 569 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 59.1%\n",
      "  C: 39.4%\n",
      "  G: 0.4%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t150\t416.1\t0\t150\n",
      "4\t24\t104.0\t0\t24\n",
      "5\t30\t26.0\t0\t30\n",
      "6\t3\t6.5\t0\t3\n",
      "7\t3\t1.6\t0\t3\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t35\t0.0\t2\t0 1 34\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t73\t0.0\t2\t0 1 72\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t9\t0.0\t2\t0 0 9\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t8\t0.0\t2\t0 0 8\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 1 3\n",
      "124\t16\t0.0\t2\t0 0 16\n",
      "125\t5\t0.0\t2\t0 0 5\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t20\t0.0\t2\t0 0 20\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t12\t0.0\t2\t0 0 12\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "138\t6\t0.0\t2\t0 0 6\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t8\t0.0\t2\t0 0 8\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "161\t3\t0.0\t2\t0 0 3\n",
      "162\t4\t0.0\t2\t0 0 4\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "238\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_347-LP3-ITS-0347_913_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_347-LP3-ITS-0347_913_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_347-LP3-ITS-0347_913_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_347-LP3-ITS-0347_913_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.096 s (12.768 ¬µs/read; 4.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,557\n",
      "Reads with adapters:                       133 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,557 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,274,657 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,261,841 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 133 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.1%\n",
      "  C: 55.6%\n",
      "  G: 1.5%\n",
      "  T: 6.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t118.1\t0\t7\n",
      "4\t2\t29.5\t0\t2\n",
      "5\t2\t7.4\t0\t2\n",
      "6\t4\t1.8\t0\t4\n",
      "7\t1\t0.5\t0\t1\n",
      "20\t3\t0.0\t2\t0 1 2\n",
      "25\t2\t0.0\t2\t0 1 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t7\t0.0\t2\t0 0 7\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_348-LP3-ITS-0348_909_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_348-LP3-ITS-0348_909_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_348-LP3-ITS-0348_909_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_348-LP3-ITS-0348_909_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.219 s (11.138 ¬µs/read; 5.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,667\n",
      "Reads with adapters:                       277 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,667 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,919,767 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,889,050 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 277 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.9%\n",
      "  C: 53.4%\n",
      "  G: 1.4%\n",
      "  T: 7.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t307.3\t0\t16\n",
      "4\t4\t76.8\t0\t4\n",
      "5\t1\t19.2\t0\t1\n",
      "6\t1\t4.8\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "27\t4\t0.0\t2\t0 1 3\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 1 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t8\t0.0\t2\t0 2 6\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "98\t1\t0.0\t2\t0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "124\t11\t0.0\t2\t0 0 11\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t7\t0.0\t2\t0 0 7\n",
      "139\t10\t0.0\t2\t0 1 9\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t9\t0.0\t2\t0 0 9\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "156\t10\t0.0\t2\t0 0 10\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t18\t0.0\t2\t0 0 18\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "177\t2\t0.0\t2\t0 0 2\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_349-LP3-ITS-0349_846_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_349-LP3-ITS-0349_846_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_349-LP3-ITS-0349_846_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_349-LP3-ITS-0349_846_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.156 s (14.218 ¬µs/read; 4.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,968\n",
      "Reads with adapters:                       217 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        10,968 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,301,368 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,280,768 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 217 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.9%\n",
      "  C: 41.9%\n",
      "  G: 4.1%\n",
      "  T: 11.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t171.4\t0\t22\n",
      "4\t6\t42.8\t0\t6\n",
      "5\t6\t10.7\t0\t6\n",
      "6\t3\t2.7\t0\t3\n",
      "10\t3\t0.0\t1\t0 3\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t3\t0.0\t2\t0 1 2\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t4\t0.0\t2\t0 0 4\n",
      "57\t5\t0.0\t2\t0 0 5\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "71\t2\t0.0\t2\t0 1 1\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t5\t0.0\t2\t0 0 5\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t11\t0.0\t2\t0 0 11\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t7\t0.0\t2\t0 0 7\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t11\t0.0\t2\t0 1 10\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_350-LP3-ITS-0350_264_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_350-LP3-ITS-0350_264_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_350-LP3-ITS-0350_264_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_350-LP3-ITS-0350_264_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.177 s (12.170 ¬µs/read; 4.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,560\n",
      "Reads with adapters:                       251 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,560 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,382,560 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,359,230 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 251 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.1%\n",
      "  C: 59.4%\n",
      "  G: 2.4%\n",
      "  T: 5.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t227.5\t0\t22\n",
      "4\t3\t56.9\t0\t3\n",
      "5\t3\t14.2\t0\t3\n",
      "6\t5\t3.6\t0\t5\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "45\t12\t0.0\t2\t0 0 12\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t7\t0.0\t2\t0 0 7\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t5\t0.0\t2\t0 1 4\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "124\t14\t0.0\t2\t0 0 14\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t6\t0.0\t2\t0 0 6\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "182\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_351-LP3-ITS-0351_306_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_351-LP3-ITS-0351_306_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_351-LP3-ITS-0351_306_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_351-LP3-ITS-0351_306_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (65.160 ¬µs/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     100\n",
      "Reads with adapters:                         1 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           100 (100.0%)\n",
      "\n",
      "Total basepairs processed:        30,100 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         29,963 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_352-LP3-ITS-0352_498_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_352-LP3-ITS-0352_498_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_352-LP3-ITS-0352_498_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_352-LP3-ITS-0352_498_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.373 s (11.976 ¬µs/read; 5.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,185\n",
      "Reads with adapters:                       463 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,185 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,386,685 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,344,523 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 463 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.4%\n",
      "  C: 52.5%\n",
      "  G: 1.9%\n",
      "  T: 7.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t487.3\t0\t59\n",
      "4\t14\t121.8\t0\t14\n",
      "5\t13\t30.5\t0\t13\n",
      "6\t6\t7.6\t0\t6\n",
      "7\t2\t1.9\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t8\t0.0\t2\t0 1 7\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "45\t3\t0.0\t2\t0 0 3\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t3\t0.0\t2\t0 1 2\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t4\t0.0\t2\t0 1 3\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t7\t0.0\t2\t0 0 7\n",
      "66\t10\t0.0\t2\t0 0 10\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "81\t5\t0.0\t2\t0 0 5\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t5\t0.0\t2\t0 0 5\n",
      "93\t25\t0.0\t2\t0 0 25\n",
      "94\t5\t0.0\t2\t0 0 5\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t4\t0.0\t2\t0 0 4\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t6\t0.0\t2\t0 1 5\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t13\t0.0\t2\t0 0 13\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t20\t0.0\t2\t0 0 20\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t18\t0.0\t2\t0 0 18\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t17\t0.0\t2\t0 1 16\n",
      "158\t7\t0.0\t2\t0 0 7\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "170\t2\t0.0\t2\t0 0 2\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_353-LP3-ITS-0353_1092_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_353-LP3-ITS-0353_1092_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_353-LP3-ITS-0353_1092_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_353-LP3-ITS-0353_1092_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.233 s (13.146 ¬µs/read; 4.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,749\n",
      "Reads with adapters:                       300 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,749 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,342,449 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,310,995 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 300 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.0%\n",
      "  C: 53.3%\n",
      "  G: 1.7%\n",
      "  T: 7.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t277.3\t0\t18\n",
      "4\t6\t69.3\t0\t6\n",
      "5\t5\t17.3\t0\t5\n",
      "20\t5\t0.0\t2\t0 0 5\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "36\t2\t0.0\t2\t0 1 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "45\t5\t0.0\t2\t0 0 5\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t6\t0.0\t2\t0 0 6\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t9\t0.0\t2\t0 0 9\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "75\t5\t0.0\t2\t0 0 5\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t4\t0.0\t2\t0 0 4\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t9\t0.0\t2\t0 0 9\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "144\t7\t0.0\t2\t0 0 7\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 1 3\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t11\t0.0\t2\t0 0 11\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t14\t0.0\t2\t0 1 13\n",
      "156\t21\t0.0\t2\t0 0 21\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_354-LP3-ITS-0354_355_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_354-LP3-ITS-0354_355_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_354-LP3-ITS-0354_355_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_354-LP3-ITS-0354_355_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.214 s (11.684 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,294\n",
      "Reads with adapters:                       316 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,294 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,506,494 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,478,559 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 316 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.2%\n",
      "  C: 46.5%\n",
      "  G: 6.0%\n",
      "  T: 8.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t285.8\t0\t43\n",
      "4\t23\t71.5\t0\t23\n",
      "5\t3\t17.9\t0\t3\n",
      "6\t2\t4.5\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "38\t3\t0.0\t2\t0 0 3\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "56\t7\t0.0\t2\t0 0 7\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "66\t8\t0.0\t2\t0 1 7\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t11\t0.0\t2\t0 0 11\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t8\t0.0\t2\t0 0 8\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t4\t0.0\t2\t0 0 4\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t4\t0.0\t2\t0 1 3\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t10\t0.0\t2\t0 0 10\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t17\t0.0\t2\t0 0 17\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "181\t1\t0.0\t2\t0 0 1\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_355-LP3-ITS-0355_207_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_355-LP3-ITS-0355_207_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_355-LP3-ITS-0355_207_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_355-LP3-ITS-0355_207_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.281 s (11.326 ¬µs/read; 5.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,827\n",
      "Reads with adapters:                       400 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,827 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,472,927 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,434,543 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 400 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.0%\n",
      "  C: 61.5%\n",
      "  G: 3.0%\n",
      "  T: 8.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t387.9\t0\t44\n",
      "4\t10\t97.0\t0\t10\n",
      "5\t2\t24.2\t0\t2\n",
      "6\t30\t6.1\t0\t30\n",
      "10\t3\t0.0\t1\t0 3\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t15\t0.0\t2\t0 0 15\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t6\t0.0\t2\t0 0 6\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t12\t0.0\t2\t0 1 11\n",
      "107\t8\t0.0\t2\t0 0 8\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t8\t0.0\t2\t0 0 8\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t8\t0.0\t2\t0 0 8\n",
      "125\t8\t0.0\t2\t0 0 8\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t5\t0.0\t2\t0 0 5\n",
      "131\t18\t0.0\t2\t0 1 17\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "135\t5\t0.0\t2\t0 0 5\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t2\t0.0\t2\t0 1 1\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "156\t6\t0.0\t2\t0 0 6\n",
      "158\t20\t0.0\t2\t0 0 20\n",
      "159\t12\t0.0\t2\t0 0 12\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_356-LP3-ITS-0356_79_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_356-LP3-ITS-0356_79_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_356-LP3-ITS-0356_79_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_356-LP3-ITS-0356_79_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (55.556 ¬µs/read; 1.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     144\n",
      "Reads with adapters:                         2 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           144 (100.0%)\n",
      "\n",
      "Total basepairs processed:        43,344 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         43,194 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 50.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.2\t0\t1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_357-LP3-ITS-0357_573_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_357-LP3-ITS-0357_573_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_357-LP3-ITS-0357_573_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_357-LP3-ITS-0357_573_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.146 s (11.836 ¬µs/read; 5.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,376\n",
      "Reads with adapters:                       244 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,376 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,725,176 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,706,134 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 244 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.3%\n",
      "  C: 50.8%\n",
      "  G: 6.6%\n",
      "  T: 14.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t193.4\t0\t31\n",
      "4\t40\t48.3\t0\t40\n",
      "5\t7\t12.1\t0\t7\n",
      "6\t4\t3.0\t0\t4\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t8\t0.0\t2\t0 0 8\n",
      "125\t4\t0.0\t2\t0 1 3\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t7\t0.0\t2\t0 0 7\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t5\t0.0\t2\t0 1 4\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "218\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_358-LP3-ITS-0358_119_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_358-LP3-ITS-0358_119_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_358-LP3-ITS-0358_119_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_358-LP3-ITS-0358_119_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.180 s (13.181 ¬µs/read; 4.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,653\n",
      "Reads with adapters:                     1,148 (8.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,109,553 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,082,988 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1148 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 6.9%\n",
      "  C: 90.0%\n",
      "  G: 1.7%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t213.3\t0\t7\n",
      "4\t10\t53.3\t0\t10\n",
      "5\t3\t13.3\t0\t3\n",
      "6\t947\t3.3\t0\t947\n",
      "10\t2\t0.0\t1\t0 2\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "33\t2\t0.0\t2\t0 0 2\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t6\t0.0\t2\t0 0 6\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t4\t0.0\t2\t0 0 4\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t12\t0.0\t2\t0 0 12\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t23\t0.0\t2\t0 0 23\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t9\t0.0\t2\t0 1 8\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t6\t0.0\t2\t0 0 6\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_359-LP3-ITS-0359_859_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_359-LP3-ITS-0359_859_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_359-LP3-ITS-0359_859_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_359-LP3-ITS-0359_859_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.115 s (11.908 ¬µs/read; 5.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   9,640\n",
      "Reads with adapters:                       238 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         9,640 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,901,640 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,884,536 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 238 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.5%\n",
      "  C: 59.2%\n",
      "  G: 0.8%\n",
      "  T: 5.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t150.6\t0\t29\n",
      "4\t3\t37.7\t0\t3\n",
      "5\t37\t9.4\t0\t37\n",
      "6\t3\t2.4\t0\t3\n",
      "7\t5\t0.6\t0\t5\n",
      "9\t1\t0.0\t0\t1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t16\t0.0\t2\t0 0 16\n",
      "66\t10\t0.0\t2\t0 0 10\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "129\t16\t0.0\t2\t0 0 16\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "136\t6\t0.0\t2\t0 0 6\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t6\t0.0\t2\t0 0 6\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_360-LP3-ITS-0360_410_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_360-LP3-ITS-0360_410_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_360-LP3-ITS-0360_410_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_360-LP3-ITS-0360_410_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (72.564 ¬µs/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      76\n",
      "Reads with adapters:                         3 (3.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            76 (100.0%)\n",
      "\n",
      "Total basepairs processed:        22,876 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         22,620 bp (98.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 33.3%\n",
      "  G: 33.3%\n",
      "  T: 33.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.2\t0\t2\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_361-LP3-ITS-0361_285_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_361-LP3-ITS-0361_285_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_361-LP3-ITS-0361_285_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_361-LP3-ITS-0361_285_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.489 s (11.355 ¬µs/read; 5.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,033\n",
      "Reads with adapters:                       759 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,033 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,952,933 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,886,821 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 759 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.9%\n",
      "  C: 63.5%\n",
      "  G: 0.7%\n",
      "  T: 5.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t672.4\t0\t53\n",
      "4\t20\t168.1\t0\t20\n",
      "5\t5\t42.0\t0\t5\n",
      "6\t10\t10.5\t0\t10\n",
      "7\t2\t2.6\t0\t2\n",
      "10\t2\t0.0\t1\t0 2\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "45\t118\t0.0\t2\t0 3 115\n",
      "46\t9\t0.0\t2\t0 0 9\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t29\t0.0\t2\t0 2 27\n",
      "53\t6\t0.0\t2\t0 0 6\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t11\t0.0\t2\t0 0 11\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t4\t0.0\t2\t0 0 4\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "63\t13\t0.0\t2\t0 0 13\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t40\t0.0\t2\t0 1 39\n",
      "74\t4\t0.0\t2\t0 0 4\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "78\t5\t0.0\t2\t0 0 5\n",
      "81\t8\t0.0\t2\t0 1 7\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t5\t0.0\t2\t0 0 5\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "96\t5\t0.0\t2\t0 0 5\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "104\t12\t0.0\t2\t0 0 12\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t16\t0.0\t2\t0 1 15\n",
      "120\t11\t0.0\t2\t0 0 11\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t9\t0.0\t2\t0 0 9\n",
      "124\t41\t0.0\t2\t0 1 40\n",
      "125\t7\t0.0\t2\t0 0 7\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t7\t0.0\t2\t0 1 6\n",
      "135\t7\t0.0\t2\t0 0 7\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t28\t0.0\t2\t0 1 27\n",
      "138\t11\t0.0\t2\t0 0 11\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t29\t0.0\t2\t0 0 29\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t12\t0.0\t2\t0 0 12\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t12\t0.0\t2\t0 0 12\n",
      "152\t7\t0.0\t2\t0 0 7\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "154\t14\t0.0\t2\t0 1 13\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "157\t6\t0.0\t2\t0 0 6\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t11\t0.0\t2\t0 1 10\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t5\t0.0\t2\t0 0 5\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "182\t2\t0.0\t2\t0 0 2\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "193\t1\t0.0\t2\t0 0 1\n",
      "202\t1\t0.0\t2\t0 0 1\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_362-LP3-ITS-0362_40_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_362-LP3-ITS-0362_40_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_362-LP3-ITS-0362_40_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_362-LP3-ITS-0362_40_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.350 s (11.384 ¬µs/read; 5.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,770\n",
      "Reads with adapters:                       482 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,261,770 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,213,127 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 482 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.1%\n",
      "  C: 58.9%\n",
      "  G: 1.0%\n",
      "  T: 5.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t480.8\t0\t29\n",
      "4\t8\t120.2\t0\t8\n",
      "5\t4\t30.0\t0\t4\n",
      "6\t3\t7.5\t0\t3\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "45\t16\t0.0\t2\t0 1 15\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t7\t0.0\t2\t0 0 7\n",
      "57\t12\t0.0\t2\t0 0 12\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t14\t0.0\t2\t0 1 13\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "73\t24\t0.0\t2\t0 0 24\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t4\t0.0\t2\t0 1 3\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t27\t0.0\t2\t0 0 27\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t8\t0.0\t2\t0 1 7\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "114\t4\t0.0\t2\t0 0 4\n",
      "117\t8\t0.0\t2\t0 0 8\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t14\t0.0\t2\t0 0 14\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "124\t27\t0.0\t2\t0 0 27\n",
      "125\t16\t0.0\t2\t0 0 16\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t23\t0.0\t2\t0 0 23\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "138\t5\t0.0\t2\t0 0 5\n",
      "139\t13\t0.0\t2\t0 0 13\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t2\t0.0\t2\t0 1 1\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "157\t8\t0.0\t2\t0 0 8\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t11\t0.0\t2\t0 0 11\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "163\t10\t0.0\t2\t0 0 10\n",
      "165\t3\t0.0\t2\t0 0 3\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "200\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_363-LP3-ITS-0363_549_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_363-LP3-ITS-0363_549_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_363-LP3-ITS-0363_549_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_363-LP3-ITS-0363_549_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.214 s (11.359 ¬µs/read; 5.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,822\n",
      "Reads with adapters:                       350 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,822 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,665,422 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,630,983 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 350 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.1%\n",
      "  C: 74.0%\n",
      "  G: 0.9%\n",
      "  T: 4.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t294.1\t0\t12\n",
      "4\t6\t73.5\t0\t6\n",
      "5\t4\t18.4\t0\t4\n",
      "6\t10\t4.6\t0\t10\n",
      "7\t3\t1.1\t0\t3\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "54\t4\t0.0\t2\t0 0 4\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t16\t0.0\t2\t0 0 16\n",
      "57\t20\t0.0\t2\t0 0 20\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t11\t0.0\t2\t0 0 11\n",
      "65\t5\t0.0\t2\t0 0 5\n",
      "69\t3\t0.0\t2\t0 0 3\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "78\t4\t0.0\t2\t0 0 4\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "85\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t6\t0.0\t2\t0 0 6\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t4\t0.0\t2\t0 1 3\n",
      "106\t4\t0.0\t2\t0 1 3\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t8\t0.0\t2\t0 0 8\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "124\t22\t0.0\t2\t0 0 22\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t20\t0.0\t2\t0 0 20\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "135\t4\t0.0\t2\t0 0 4\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t17\t0.0\t2\t0 0 17\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "145\t1\t0.0\t2\t0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t23\t0.0\t2\t0 0 23\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_364-LP3-ITS-0364_100_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_364-LP3-ITS-0364_100_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_364-LP3-ITS-0364_100_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_364-LP3-ITS-0364_100_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (68.035 ¬µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      92\n",
      "Reads with adapters:                         3 (3.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            92 (100.0%)\n",
      "\n",
      "Total basepairs processed:        27,692 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         27,568 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 33.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.4\t0\t2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_365-LP3-ITS-0365_1045_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_365-LP3-ITS-0365_1045_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_365-LP3-ITS-0365_1045_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_365-LP3-ITS-0365_1045_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (37.805 ¬µs/read; 1.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     200\n",
      "Reads with adapters:                         9 (4.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           200 (100.0%)\n",
      "\n",
      "Total basepairs processed:        60,200 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         59,767 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 55.6%\n",
      "  G: 0.0%\n",
      "  T: 11.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t3.1\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_366-LP3-ITS-0366_130_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_366-LP3-ITS-0366_130_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_366-LP3-ITS-0366_130_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_366-LP3-ITS-0366_130_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.314 s (11.707 ¬µs/read; 5.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,801\n",
      "Reads with adapters:                       436 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,801 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,067,101 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,021,373 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 436 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.1%\n",
      "  C: 52.8%\n",
      "  G: 0.7%\n",
      "  T: 5.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t418.8\t0\t27\n",
      "4\t10\t104.7\t0\t10\n",
      "5\t9\t26.2\t0\t9\n",
      "6\t12\t6.5\t0\t12\n",
      "7\t1\t1.6\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t5\t0.0\t2\t0 0 5\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t5\t0.0\t2\t0 0 5\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t8\t0.0\t2\t0 0 8\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "104\t29\t0.0\t2\t0 0 29\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t5\t0.0\t2\t0 0 5\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t4\t0.0\t2\t0 0 4\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t5\t0.0\t2\t0 0 5\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t16\t0.0\t2\t0 1 15\n",
      "120\t8\t0.0\t2\t0 0 8\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t21\t0.0\t2\t0 0 21\n",
      "125\t24\t0.0\t2\t0 0 24\n",
      "126\t10\t0.0\t2\t0 0 10\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t16\t0.0\t2\t0 0 16\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "135\t6\t0.0\t2\t0 0 6\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t21\t0.0\t2\t0 0 21\n",
      "140\t1\t0.0\t2\t0 1\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t19\t0.0\t2\t0 0 19\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "152\t9\t0.0\t2\t0 0 9\n",
      "153\t9\t0.0\t2\t0 0 9\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "156\t1\t0.0\t2\t0 1\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "272\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_367-LP3-ITS-0367_1125_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_367-LP3-ITS-0367_1125_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_367-LP3-ITS-0367_1125_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_367-LP3-ITS-0367_1125_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.334 s (12.109 ¬µs/read; 4.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,600\n",
      "Reads with adapters:                       454 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,600 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,307,600 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,264,328 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 454 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.9%\n",
      "  C: 59.3%\n",
      "  G: 2.2%\n",
      "  T: 6.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t431.2\t0\t22\n",
      "4\t21\t107.8\t0\t21\n",
      "5\t3\t27.0\t0\t3\n",
      "6\t9\t6.7\t0\t9\n",
      "7\t1\t1.7\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t6\t0.0\t2\t0 0 6\n",
      "58\t61\t0.0\t2\t0 2 59\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t24\t0.0\t2\t0 0 24\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "85\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t5\t0.0\t2\t0 0 5\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "104\t28\t0.0\t2\t0 0 28\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "114\t6\t0.0\t2\t0 0 6\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "119\t5\t0.0\t2\t0 1 4\n",
      "120\t7\t0.0\t2\t0 0 7\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "124\t15\t0.0\t2\t0 0 15\n",
      "125\t15\t0.0\t2\t0 0 15\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "135\t4\t0.0\t2\t0 0 4\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t14\t0.0\t2\t0 0 14\n",
      "138\t5\t0.0\t2\t0 0 5\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "142\t22\t0.0\t2\t0 0 22\n",
      "144\t8\t0.0\t2\t0 0 8\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t13\t0.0\t2\t0 0 13\n",
      "153\t7\t0.0\t2\t0 1 6\n",
      "154\t6\t0.0\t2\t0 0 6\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t21\t0.0\t2\t0 0 21\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_368-LP3-ITS-0368_312_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_368-LP3-ITS-0368_312_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_368-LP3-ITS-0368_312_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_368-LP3-ITS-0368_312_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.378 s (11.649 ¬µs/read; 5.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,408\n",
      "Reads with adapters:                       554 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,408 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,754,808 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,704,738 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 554 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.3%\n",
      "  C: 51.8%\n",
      "  G: 2.7%\n",
      "  T: 9.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t506.4\t0\t53\n",
      "4\t27\t126.6\t0\t27\n",
      "5\t5\t31.6\t0\t5\n",
      "6\t2\t7.9\t0\t2\n",
      "7\t2\t2.0\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t17\t0.0\t2\t0 0 17\n",
      "35\t2\t0.0\t2\t0 1 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "38\t3\t0.0\t2\t0 0 3\n",
      "39\t4\t0.0\t2\t0 1 3\n",
      "40\t4\t0.0\t2\t0 0 4\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "45\t21\t0.0\t2\t0 1 20\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t6\t0.0\t2\t0 0 6\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "63\t4\t0.0\t2\t0 1 3\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t5\t0.0\t2\t0 0 5\n",
      "68\t2\t0.0\t2\t0 0 2\n",
      "70\t2\t0.0\t2\t0 0 2\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "73\t29\t0.0\t2\t0 0 29\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t7\t0.0\t2\t0 0 7\n",
      "89\t11\t0.0\t2\t0 0 11\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t6\t0.0\t2\t0 0 6\n",
      "95\t6\t0.0\t2\t0 0 6\n",
      "96\t1\t0.0\t2\t0 1\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t6\t0.0\t2\t0 0 6\n",
      "124\t8\t0.0\t2\t0 0 8\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t10\t0.0\t2\t0 0 10\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t26\t0.0\t2\t0 0 26\n",
      "138\t9\t0.0\t2\t0 0 9\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t37\t0.0\t2\t0 0 37\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t22\t0.0\t2\t0 1 21\n",
      "151\t11\t0.0\t2\t0 2 9\n",
      "152\t16\t0.0\t2\t0 1 15\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t14\t0.0\t2\t0 0 14\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t12\t0.0\t2\t0 1 11\n",
      "160\t3\t0.0\t2\t0 1 2\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "180\t2\t0.0\t2\t0 0 2\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_369-LP3-ITS-0369_289_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_369-LP3-ITS-0369_289_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_369-LP3-ITS-0369_289_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_369-LP3-ITS-0369_289_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.508 s (11.300 ¬µs/read; 5.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,953\n",
      "Reads with adapters:                       659 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,953 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,530,853 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,465,909 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 659 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.0%\n",
      "  C: 48.6%\n",
      "  G: 0.9%\n",
      "  T: 8.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t702.4\t0\t46\n",
      "4\t17\t175.6\t0\t17\n",
      "5\t22\t43.9\t0\t22\n",
      "6\t8\t11.0\t0\t8\n",
      "9\t1\t0.2\t0\t1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t9\t0.0\t2\t0 0 9\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "50\t8\t0.0\t2\t0 0 8\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t9\t0.0\t2\t0 0 9\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t17\t0.0\t2\t0 0 17\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t5\t0.0\t2\t0 0 5\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "73\t11\t0.0\t2\t0 0 11\n",
      "74\t3\t0.0\t2\t0 1 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "98\t5\t0.0\t2\t0 0 5\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "105\t7\t0.0\t2\t0 0 7\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t8\t0.0\t2\t0 0 8\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t67\t0.0\t2\t0 0 67\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "118\t21\t0.0\t2\t0 0 21\n",
      "119\t16\t0.0\t2\t0 0 16\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t13\t0.0\t2\t0 0 13\n",
      "124\t31\t0.0\t2\t0 0 31\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t5\t0.0\t2\t0 0 5\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t7\t0.0\t2\t0 0 7\n",
      "139\t13\t0.0\t2\t0 0 13\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t45\t0.0\t2\t0 0 45\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t6\t0.0\t2\t0 0 6\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t20\t0.0\t2\t0 0 20\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t19\t0.0\t2\t0 1 18\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t29\t0.0\t2\t0 0 29\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "207\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_370-LP3-ITS-0370_851_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_370-LP3-ITS-0370_851_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_370-LP3-ITS-0370_851_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_370-LP3-ITS-0370_851_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.505 s (11.046 ¬µs/read; 5.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,717\n",
      "Reads with adapters:                       687 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,717 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,760,817 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,687,139 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 687 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 63.0%\n",
      "  G: 2.2%\n",
      "  T: 3.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t714.3\t0\t53\n",
      "4\t12\t178.6\t0\t12\n",
      "5\t18\t44.6\t0\t18\n",
      "6\t6\t11.2\t0\t6\n",
      "7\t1\t2.8\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 1 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 1 2\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "40\t4\t0.0\t2\t0 0 4\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "56\t4\t0.0\t2\t0 0 4\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t16\t0.0\t2\t0 0 16\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "73\t10\t0.0\t2\t0 1 9\n",
      "74\t7\t0.0\t2\t0 0 7\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "77\t3\t0.0\t2\t0 0 3\n",
      "78\t6\t0.0\t2\t0 1 5\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t18\t0.0\t2\t0 0 18\n",
      "105\t8\t0.0\t2\t0 0 8\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t9\t0.0\t2\t0 0 9\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t6\t0.0\t2\t0 1 5\n",
      "118\t2\t0.0\t2\t0 1 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t8\t0.0\t2\t0 0 8\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t18\t0.0\t2\t0 0 18\n",
      "125\t2\t0.0\t2\t0 1 1\n",
      "126\t40\t0.0\t2\t0 1 39\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "134\t60\t0.0\t2\t0 2 58\n",
      "135\t15\t0.0\t2\t0 2 13\n",
      "136\t5\t0.0\t2\t0 1 4\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t20\t0.0\t2\t0 1 19\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t60\t0.0\t2\t0 1 59\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "147\t16\t0.0\t2\t0 0 16\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t13\t0.0\t2\t0 0 13\n",
      "151\t19\t0.0\t2\t0 0 19\n",
      "152\t11\t0.0\t2\t0 0 11\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "156\t3\t0.0\t2\t0 1 2\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t19\t0.0\t2\t0 0 19\n",
      "159\t27\t0.0\t2\t0 0 27\n",
      "160\t12\t0.0\t2\t0 1 11\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "176\t2\t0.0\t2\t0 0 2\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_371-LP3-ITS-0371_358_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_371-LP3-ITS-0371_358_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_371-LP3-ITS-0371_358_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_371-LP3-ITS-0371_358_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.526 s (11.969 ¬µs/read; 5.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,962\n",
      "Reads with adapters:                       821 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,962 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,232,562 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,152,022 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 821 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.2%\n",
      "  C: 57.0%\n",
      "  G: 1.3%\n",
      "  T: 3.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t686.9\t0\t42\n",
      "4\t36\t171.7\t0\t36\n",
      "5\t16\t42.9\t0\t16\n",
      "6\t9\t10.7\t0\t9\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 1 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "31\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t2\t0.0\t2\t0 0 2\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t5\t0.0\t2\t0 0 5\n",
      "56\t55\t0.0\t2\t0 1 54\n",
      "57\t7\t0.0\t2\t0 0 7\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t44\t0.0\t2\t0 1 43\n",
      "64\t5\t0.0\t2\t0 0 5\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "68\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t6\t0.0\t2\t0 0 6\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "80\t3\t0.0\t2\t0 0 3\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "94\t6\t0.0\t2\t0 0 6\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t4\t0.0\t2\t0 1 3\n",
      "104\t56\t0.0\t2\t0 0 56\n",
      "105\t6\t0.0\t2\t0 0 6\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t9\t0.0\t2\t0 0 9\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t5\t0.0\t2\t0 0 5\n",
      "117\t8\t0.0\t2\t0 0 8\n",
      "119\t16\t0.0\t2\t0 0 16\n",
      "120\t16\t0.0\t2\t0 1 15\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t32\t0.0\t2\t0 3 29\n",
      "125\t16\t0.0\t2\t0 2 14\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t12\t0.0\t2\t0 0 12\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "135\t10\t0.0\t2\t0 0 10\n",
      "136\t11\t0.0\t2\t0 0 11\n",
      "137\t21\t0.0\t2\t0 0 21\n",
      "138\t12\t0.0\t2\t0 0 12\n",
      "139\t36\t0.0\t2\t0 1 35\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t23\t0.0\t2\t0 0 23\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t17\t0.0\t2\t0 1 16\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t8\t0.0\t2\t0 0 8\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t14\t0.0\t2\t0 0 14\n",
      "151\t15\t0.0\t2\t0 0 15\n",
      "152\t11\t0.0\t2\t0 0 11\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "154\t9\t0.0\t2\t0 1 8\n",
      "155\t10\t0.0\t2\t0 1 9\n",
      "156\t11\t0.0\t2\t0 0 11\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "162\t5\t0.0\t2\t0 1 4\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "182\t2\t0.0\t2\t0 0 2\n",
      "190\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_372-LP3-ITS-0372_1057_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_372-LP3-ITS-0372_1057_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_372-LP3-ITS-0372_1057_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_372-LP3-ITS-0372_1057_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.307 s (11.500 ¬µs/read; 5.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,696\n",
      "Reads with adapters:                       391 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,696 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,035,496 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,993,805 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 391 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.9%\n",
      "  C: 63.7%\n",
      "  G: 1.8%\n",
      "  T: 5.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t417.1\t0\t29\n",
      "4\t2\t104.3\t0\t2\n",
      "5\t8\t26.1\t0\t8\n",
      "6\t8\t6.5\t0\t8\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t10\t0.0\t2\t0 0 10\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "47\t5\t0.0\t2\t0 0 5\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "52\t20\t0.0\t2\t0 0 20\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "74\t5\t0.0\t2\t0 0 5\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t15\t0.0\t2\t0 0 15\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "124\t19\t0.0\t2\t0 0 19\n",
      "125\t7\t0.0\t2\t0 0 7\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t61\t0.0\t2\t0 0 61\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t29\t0.0\t2\t0 0 29\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t5\t0.0\t2\t0 0 5\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t4\t0.0\t2\t0 1 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t6\t0.0\t2\t0 0 6\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "180\t2\t0.0\t2\t0 0 2\n",
      "182\t2\t0.0\t2\t0 0 2\n",
      "201\t1\t0.0\t2\t0 0 1\n",
      "248\t1\t0.0\t2\t0 0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_373-LP3-ITS-0373_922_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_373-LP3-ITS-0373_922_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_373-LP3-ITS-0373_922_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_373-LP3-ITS-0373_922_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.525 s (11.527 ¬µs/read; 5.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,537\n",
      "Reads with adapters:                       868 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,537 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,706,637 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,624,842 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 868 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.1%\n",
      "  C: 54.5%\n",
      "  G: 2.8%\n",
      "  T: 8.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t711.5\t0\t62\n",
      "4\t28\t177.9\t0\t28\n",
      "5\t10\t44.5\t0\t10\n",
      "6\t8\t11.1\t0\t8\n",
      "7\t3\t2.8\t0\t3\n",
      "8\t1\t0.7\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "35\t6\t0.0\t2\t0 0 6\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t2\t0.0\t2\t0 0 2\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "45\t31\t0.0\t2\t0 0 31\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t65\t0.0\t2\t0 2 63\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t8\t0.0\t2\t0 0 8\n",
      "60\t8\t0.0\t2\t0 0 8\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t10\t0.0\t2\t0 1 9\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "65\t15\t0.0\t2\t0 0 15\n",
      "66\t5\t0.0\t2\t0 0 5\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t37\t0.0\t2\t0 3 34\n",
      "74\t4\t0.0\t2\t0 0 4\n",
      "75\t6\t0.0\t2\t0 0 6\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t4\t0.0\t2\t0 0 4\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "83\t3\t0.0\t2\t0 0 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t10\t0.0\t2\t0 1 9\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t8\t0.0\t2\t0 0 8\n",
      "95\t6\t0.0\t2\t0 0 6\n",
      "96\t9\t0.0\t2\t0 0 9\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "102\t4\t0.0\t2\t0 1 3\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t15\t0.0\t2\t0 0 15\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t7\t0.0\t2\t0 0 7\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "117\t8\t0.0\t2\t0 0 8\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t14\t0.0\t2\t0 0 14\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t17\t0.0\t2\t0 0 17\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "135\t6\t0.0\t2\t0 0 6\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t20\t0.0\t2\t0 0 20\n",
      "138\t14\t0.0\t2\t0 0 14\n",
      "139\t20\t0.0\t2\t0 0 20\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t55\t0.0\t2\t0 1 54\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t15\t0.0\t2\t0 0 15\n",
      "146\t7\t0.0\t2\t0 0 7\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t9\t0.0\t2\t0 0 9\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t27\t0.0\t2\t0 0 27\n",
      "151\t14\t0.0\t2\t0 0 14\n",
      "152\t24\t0.0\t2\t0 0 24\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t34\t0.0\t2\t0 0 34\n",
      "157\t6\t0.0\t2\t0 0 6\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t32\t0.0\t2\t0 0 32\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "169\t2\t0.0\t2\t0 0 2\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "176\t2\t0.0\t2\t0 0 2\n",
      "180\t2\t0.0\t2\t0 0 2\n",
      "193\t3\t0.0\t2\t0 0 3\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_374-LP3-ITS-0374_88_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_374-LP3-ITS-0374_88_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_374-LP3-ITS-0374_88_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_374-LP3-ITS-0374_88_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.595 s (11.620 ¬µs/read; 5.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,230\n",
      "Reads with adapters:                       824 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,230 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,420,230 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,345,555 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 824 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.1%\n",
      "  C: 69.8%\n",
      "  G: 1.6%\n",
      "  T: 3.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t800.5\t0\t48\n",
      "4\t32\t200.1\t0\t32\n",
      "5\t18\t50.0\t0\t18\n",
      "6\t2\t12.5\t0\t2\n",
      "7\t2\t3.1\t0\t2\n",
      "10\t2\t0.0\t1\t0 2\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "23\t13\t0.0\t2\t0 1 12\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t5\t0.0\t2\t0 0 5\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "52\t113\t0.0\t2\t0 4 109\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t27\t0.0\t2\t0 0 27\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t24\t0.0\t2\t0 0 24\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t6\t0.0\t2\t0 1 5\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t34\t0.0\t2\t0 1 33\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t14\t0.0\t2\t0 0 14\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t9\t0.0\t2\t0 0 9\n",
      "94\t10\t0.0\t2\t0 0 10\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t8\t0.0\t2\t0 0 8\n",
      "98\t4\t0.0\t2\t0 0 4\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t13\t0.0\t2\t0 0 13\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t7\t0.0\t2\t0 0 7\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t3\t0.0\t2\t0 0 3\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t3\t0.0\t2\t0 0 3\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "119\t5\t0.0\t2\t0 1 4\n",
      "120\t7\t0.0\t2\t0 0 7\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t12\t0.0\t2\t0 0 12\n",
      "124\t52\t0.0\t2\t0 1 51\n",
      "125\t10\t0.0\t2\t0 0 10\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t4\t0.0\t2\t0 0 4\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "135\t16\t0.0\t2\t0 0 16\n",
      "136\t10\t0.0\t2\t0 0 10\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "138\t6\t0.0\t2\t0 0 6\n",
      "139\t26\t0.0\t2\t0 0 26\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t8\t0.0\t2\t0 0 8\n",
      "142\t20\t0.0\t2\t0 0 20\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t15\t0.0\t2\t0 0 15\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t8\t0.0\t2\t0 0 8\n",
      "150\t7\t0.0\t2\t0 1 6\n",
      "151\t37\t0.0\t2\t0 1 36\n",
      "152\t9\t0.0\t2\t0 0 9\n",
      "153\t6\t0.0\t2\t0 0 6\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t24\t0.0\t2\t0 0 24\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "161\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t4\t0.0\t2\t0 0 4\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "243\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_375-LP3-ITS-0375_985_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_375-LP3-ITS-0375_985_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_375-LP3-ITS-0375_985_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_375-LP3-ITS-0375_985_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.613 s (11.840 ¬µs/read; 5.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,814\n",
      "Reads with adapters:                       916 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,814 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,596,014 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,504,366 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 916 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.7%\n",
      "  C: 54.7%\n",
      "  G: 1.1%\n",
      "  T: 7.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t809.6\t0\t50\n",
      "4\t17\t202.4\t0\t17\n",
      "5\t22\t50.6\t0\t22\n",
      "6\t8\t12.6\t0\t8\n",
      "7\t1\t3.2\t0\t1\n",
      "9\t2\t0.2\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t11\t0.0\t2\t0 0 11\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "35\t5\t0.0\t2\t0 1 4\n",
      "36\t3\t0.0\t2\t0 0 3\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t7\t0.0\t2\t0 0 7\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "45\t25\t0.0\t2\t0 2 23\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "59\t6\t0.0\t2\t0 0 6\n",
      "60\t7\t0.0\t2\t0 0 7\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t5\t0.0\t2\t0 0 5\n",
      "63\t9\t0.0\t2\t0 2 7\n",
      "64\t7\t0.0\t2\t0 0 7\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t5\t0.0\t2\t0 0 5\n",
      "68\t30\t0.0\t2\t0 0 30\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t47\t0.0\t2\t0 0 47\n",
      "74\t11\t0.0\t2\t0 0 11\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t19\t0.0\t2\t0 0 19\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t5\t0.0\t2\t0 0 5\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t10\t0.0\t2\t0 0 10\n",
      "95\t7\t0.0\t2\t0 0 7\n",
      "96\t7\t0.0\t2\t0 0 7\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "105\t13\t0.0\t2\t0 0 13\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t4\t0.0\t2\t0 1 3\n",
      "109\t8\t0.0\t2\t0 0 8\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t5\t0.0\t2\t0 0 5\n",
      "116\t7\t0.0\t2\t0 0 7\n",
      "117\t11\t0.0\t2\t0 0 11\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t5\t0.0\t2\t0 0 5\n",
      "120\t11\t0.0\t2\t0 0 11\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t12\t0.0\t2\t0 0 12\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t16\t0.0\t2\t0 0 16\n",
      "128\t7\t0.0\t2\t0 0 7\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t6\t0.0\t2\t0 1 5\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "135\t4\t0.0\t2\t0 0 4\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t25\t0.0\t2\t1 0 24\n",
      "138\t19\t0.0\t2\t0 0 19\n",
      "139\t15\t0.0\t2\t0 0 15\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t6\t0.0\t2\t0 0 6\n",
      "142\t47\t0.0\t2\t0 0 47\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "146\t7\t0.0\t2\t0 0 7\n",
      "147\t10\t0.0\t2\t0 0 10\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t41\t0.0\t2\t0 0 41\n",
      "151\t16\t0.0\t2\t0 1 15\n",
      "152\t33\t0.0\t2\t0 0 33\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t33\t0.0\t2\t0 0 33\n",
      "157\t10\t0.0\t2\t0 0 10\n",
      "158\t9\t0.0\t2\t0 0 9\n",
      "159\t35\t0.0\t2\t0 0 35\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "182\t2\t0.0\t2\t0 0 2\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "190\t1\t0.0\t2\t0 0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_376-LP3-ITS-0376_82_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_376-LP3-ITS-0376_82_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_376-LP3-ITS-0376_82_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_376-LP3-ITS-0376_82_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.350 s (12.688 ¬µs/read; 4.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,547\n",
      "Reads with adapters:                       326 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,547 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,291,647 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,260,092 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 326 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.9%\n",
      "  C: 55.5%\n",
      "  G: 1.8%\n",
      "  T: 10.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t430.4\t0\t24\n",
      "4\t8\t107.6\t0\t8\n",
      "5\t6\t26.9\t0\t6\n",
      "7\t1\t1.7\t0\t1\n",
      "8\t1\t0.4\t0\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t3\t0.0\t2\t0 1 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "29\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "35\t3\t0.0\t2\t0 0 3\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t4\t0.0\t2\t0 1 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t4\t0.0\t2\t0 0 4\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t5\t0.0\t2\t0 0 5\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "83\t4\t0.0\t2\t0 0 4\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "94\t6\t0.0\t2\t0 0 6\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t5\t0.0\t2\t0 0 5\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "116\t5\t0.0\t2\t0 1 4\n",
      "117\t5\t0.0\t2\t0 1 4\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 1 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t11\t0.0\t2\t0 1 10\n",
      "138\t12\t0.0\t2\t0 0 12\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t17\t0.0\t2\t0 1 16\n",
      "144\t9\t0.0\t2\t0 0 9\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t14\t0.0\t2\t0 0 14\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t7\t0.0\t2\t0 1 6\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t15\t0.0\t2\t0 0 15\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "161\t2\t0.0\t2\t0 0 2\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "182\t3\t0.0\t2\t0 0 3\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_377-LP3-ITS-0377_818_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_377-LP3-ITS-0377_818_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_377-LP3-ITS-0377_818_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_377-LP3-ITS-0377_818_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.514 s (11.238 ¬µs/read; 5.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,698\n",
      "Reads with adapters:                       568 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,698 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,755,098 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,683,353 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 568 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 51.9%\n",
      "  C: 44.4%\n",
      "  G: 0.0%\n",
      "  T: 3.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t714.0\t0\t5\n",
      "4\t1\t178.5\t0\t1\n",
      "5\t2\t44.6\t0\t2\n",
      "7\t1\t2.8\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "29\t5\t0.0\t2\t0 0 5\n",
      "35\t4\t0.0\t2\t0 0 4\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t8\t0.0\t2\t0 1 7\n",
      "56\t10\t0.0\t2\t0 0 10\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "81\t6\t0.0\t2\t0 0 6\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "85\t3\t0.0\t2\t0 0 3\n",
      "86\t3\t0.0\t2\t0 0 3\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t8\t0.0\t2\t0 0 8\n",
      "110\t40\t0.0\t2\t0 4 36\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t8\t0.0\t2\t0 0 8\n",
      "127\t24\t0.0\t2\t0 0 24\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "133\t40\t0.0\t2\t0 1 39\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t10\t0.0\t2\t0 0 10\n",
      "139\t30\t0.0\t2\t0 0 30\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t20\t0.0\t2\t0 0 20\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "147\t14\t0.0\t2\t0 0 14\n",
      "148\t8\t0.0\t2\t0 0 8\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t21\t0.0\t2\t0 0 21\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t14\t0.0\t2\t0 0 14\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "156\t59\t0.0\t2\t0 1 58\n",
      "157\t13\t0.0\t2\t0 0 13\n",
      "158\t19\t0.0\t2\t0 0 19\n",
      "159\t25\t0.0\t2\t0 0 25\n",
      "160\t8\t0.0\t2\t0 1 7\n",
      "161\t4\t0.0\t2\t0 0 4\n",
      "162\t4\t0.0\t2\t0 0 4\n",
      "163\t4\t0.0\t2\t0 0 4\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "169\t2\t0.0\t2\t0 0 2\n",
      "170\t3\t0.0\t2\t0 0 3\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "181\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "257\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_378-LP3-ITS-0378_1043_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_378-LP3-ITS-0378_1043_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_378-LP3-ITS-0378_1043_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_378-LP3-ITS-0378_1043_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.521 s (11.707 ¬µs/read; 5.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,483\n",
      "Reads with adapters:                       714 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,389,383 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,320,023 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 714 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.0%\n",
      "  C: 49.3%\n",
      "  G: 0.8%\n",
      "  T: 8.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t695.0\t0\t59\n",
      "4\t49\t173.8\t0\t49\n",
      "5\t5\t43.4\t0\t5\n",
      "6\t9\t10.9\t0\t9\n",
      "7\t2\t2.7\t0\t2\n",
      "20\t1\t0.0\t2\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "40\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t10\t0.0\t2\t0 0 10\n",
      "53\t28\t0.0\t2\t0 1 27\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t24\t0.0\t2\t0 0 24\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t4\t0.0\t2\t0 0 4\n",
      "66\t4\t0.0\t2\t0 0 4\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t2\t0.0\t2\t0 0 2\n",
      "73\t13\t0.0\t2\t0 0 13\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "77\t4\t0.0\t2\t0 0 4\n",
      "78\t7\t0.0\t2\t0 0 7\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "85\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t20\t0.0\t2\t0 0 20\n",
      "105\t6\t0.0\t2\t0 0 6\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t11\t0.0\t2\t0 0 11\n",
      "108\t3\t0.0\t2\t0 1 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "113\t6\t0.0\t2\t0 0 6\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "119\t10\t0.0\t2\t0 0 10\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "124\t31\t0.0\t2\t0 0 31\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t4\t0.0\t2\t0 0 4\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t3\t0.0\t2\t0 1 2\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "135\t5\t0.0\t2\t0 0 5\n",
      "136\t10\t0.0\t2\t0 0 10\n",
      "137\t14\t0.0\t2\t0 0 14\n",
      "138\t9\t0.0\t2\t0 0 9\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t58\t0.0\t2\t0 0 58\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "144\t15\t0.0\t2\t0 0 15\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t13\t0.0\t2\t0 0 13\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t22\t0.0\t2\t0 0 22\n",
      "151\t16\t0.0\t2\t0 0 16\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t4\t0.0\t2\t0 0 4\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t9\t0.0\t2\t0 0 9\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t16\t0.0\t2\t0 0 16\n",
      "159\t33\t0.0\t2\t0 0 33\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "168\t2\t0.0\t2\t0 0 2\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "243\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_379-LP3-ITS-0379_460_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_379-LP3-ITS-0379_460_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_379-LP3-ITS-0379_460_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_379-LP3-ITS-0379_460_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.307 s (11.540 ¬µs/read; 5.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,630\n",
      "Reads with adapters:                       428 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,015,630 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,971,594 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 428 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.2%\n",
      "  C: 45.1%\n",
      "  G: 1.9%\n",
      "  T: 9.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t416.1\t0\t32\n",
      "4\t6\t104.0\t0\t6\n",
      "5\t7\t26.0\t0\t7\n",
      "6\t1\t6.5\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "38\t4\t0.0\t2\t0 0 4\n",
      "40\t2\t0.0\t2\t0 0 2\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t5\t0.0\t2\t0 0 5\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "56\t7\t0.0\t2\t0 0 7\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "65\t5\t0.0\t2\t0 0 5\n",
      "66\t10\t0.0\t2\t0 0 10\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "73\t8\t0.0\t2\t0 0 8\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t3\t0.0\t2\t0 1 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "94\t7\t0.0\t2\t0 0 7\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t4\t0.0\t2\t0 0 4\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t4\t0.0\t2\t0 1 3\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t4\t0.0\t2\t0 0 4\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t7\t0.0\t2\t0 0 7\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t8\t0.0\t2\t0 0 8\n",
      "139\t14\t0.0\t2\t0 0 14\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t30\t0.0\t2\t0 0 30\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "144\t8\t0.0\t2\t0 0 8\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t11\t0.0\t2\t0 0 11\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t19\t0.0\t2\t0 0 19\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t5\t0.0\t2\t0 0 5\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "181\t2\t0.0\t2\t0 0 2\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_380-LP3-ITS-0380_885_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_380-LP3-ITS-0380_885_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_380-LP3-ITS-0380_885_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_380-LP3-ITS-0380_885_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (60.008 ¬µs/read; 1.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     136\n",
      "Reads with adapters:                         1 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           136 (100.0%)\n",
      "\n",
      "Total basepairs processed:        40,936 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         40,933 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 100.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_381-LP3-ITS-0381_1029_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_381-LP3-ITS-0381_1029_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_381-LP3-ITS-0381_1029_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_381-LP3-ITS-0381_1029_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.421 s (12.012 ¬µs/read; 5.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,075\n",
      "Reads with adapters:                       569 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,557,575 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,500,842 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 569 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.8%\n",
      "  C: 50.3%\n",
      "  G: 1.9%\n",
      "  T: 7.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t548.0\t0\t50\n",
      "4\t31\t137.0\t0\t31\n",
      "5\t7\t34.3\t0\t7\n",
      "6\t2\t8.6\t0\t2\n",
      "7\t1\t2.1\t0\t1\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t5\t0.0\t2\t0 0 5\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t8\t0.0\t2\t0 0 8\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t6\t0.0\t2\t0 0 6\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t6\t0.0\t2\t0 0 6\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "80\t5\t0.0\t2\t0 1 4\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t8\t0.0\t2\t0 0 8\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t9\t0.0\t2\t0 0 9\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t17\t0.0\t2\t0 0 17\n",
      "94\t12\t0.0\t2\t0 0 12\n",
      "96\t5\t0.0\t2\t0 1 4\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t5\t0.0\t2\t0 0 5\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "105\t6\t0.0\t2\t0 0 6\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t4\t0.0\t2\t0 0 4\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "127\t17\t0.0\t2\t0 0 17\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t20\t0.0\t2\t0 1 19\n",
      "134\t17\t0.0\t2\t0 0 17\n",
      "136\t8\t0.0\t2\t0 0 8\n",
      "137\t16\t0.0\t2\t0 0 16\n",
      "138\t5\t0.0\t2\t0 0 5\n",
      "139\t20\t0.0\t2\t0 0 20\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t37\t0.0\t2\t0 1 36\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t24\t0.0\t2\t0 0 24\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "156\t22\t0.0\t2\t0 0 22\n",
      "157\t18\t0.0\t2\t0 1 17\n",
      "158\t12\t0.0\t2\t0 0 12\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "194\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_382-LP3-ITS-0382_873_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_382-LP3-ITS-0382_873_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_382-LP3-ITS-0382_873_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_382-LP3-ITS-0382_873_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.429 s (11.403 ¬µs/read; 5.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,580\n",
      "Reads with adapters:                       607 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,311,580 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,249,193 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 607 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.4%\n",
      "  C: 48.1%\n",
      "  G: 1.8%\n",
      "  T: 8.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t587.2\t0\t36\n",
      "4\t12\t146.8\t0\t12\n",
      "5\t5\t36.7\t0\t5\n",
      "6\t7\t9.2\t0\t7\n",
      "7\t2\t2.3\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t3\t0.0\t2\t0 0 3\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "39\t2\t0.0\t2\t0 0 2\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t9\t0.0\t2\t0 0 9\n",
      "46\t11\t0.0\t2\t0 0 11\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t5\t0.0\t2\t0 0 5\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t5\t0.0\t2\t0 1 4\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t9\t0.0\t2\t0 1 8\n",
      "66\t5\t0.0\t2\t0 0 5\n",
      "68\t3\t0.0\t2\t0 0 3\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t35\t0.0\t2\t0 0 35\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "79\t4\t0.0\t2\t0 0 4\n",
      "83\t5\t0.0\t2\t0 0 5\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t18\t0.0\t2\t0 0 18\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t9\t0.0\t2\t0 0 9\n",
      "95\t5\t0.0\t2\t0 0 5\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t5\t0.0\t2\t0 0 5\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t8\t0.0\t2\t0 0 8\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t5\t0.0\t2\t0 0 5\n",
      "116\t6\t0.0\t2\t0 0 6\n",
      "117\t3\t0.0\t2\t0 1 2\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t8\t0.0\t2\t0 0 8\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t8\t0.0\t2\t0 0 8\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t5\t0.0\t2\t0 0 5\n",
      "130\t2\t0.0\t2\t1 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t7\t0.0\t2\t0 0 7\n",
      "137\t17\t0.0\t2\t0 1 16\n",
      "138\t13\t0.0\t2\t0 0 13\n",
      "139\t16\t0.0\t2\t0 0 16\n",
      "142\t42\t0.0\t2\t0 0 42\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t5\t0.0\t2\t0 0 5\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t35\t0.0\t2\t0 0 35\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "152\t11\t0.0\t2\t0 0 11\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t31\t0.0\t2\t0 0 31\n",
      "157\t9\t0.0\t2\t0 0 9\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t14\t0.0\t2\t0 0 14\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_383-LP3-ITS-0383_263_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_383-LP3-ITS-0383_263_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_383-LP3-ITS-0383_263_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_383-LP3-ITS-0383_263_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.520 s (11.681 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,542\n",
      "Reads with adapters:                       756 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,542 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,407,142 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,326,132 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 756 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.5%\n",
      "  C: 61.5%\n",
      "  G: 1.3%\n",
      "  T: 7.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t696.0\t0\t45\n",
      "4\t18\t174.0\t0\t18\n",
      "5\t6\t43.5\t0\t6\n",
      "6\t5\t10.9\t0\t5\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t14\t0.0\t2\t0 1 13\n",
      "47\t12\t0.0\t2\t0 0 12\n",
      "49\t5\t0.0\t2\t0 0 5\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t4\t0.0\t2\t0 0 4\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t6\t0.0\t2\t0 0 6\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "65\t6\t0.0\t2\t1 0 5\n",
      "66\t5\t0.0\t2\t0 0 5\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t2\t0.0\t2\t0 0 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t25\t0.0\t2\t0 0 25\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t5\t0.0\t2\t0 0 5\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t25\t0.0\t2\t0 0 25\n",
      "105\t6\t0.0\t2\t0 0 6\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t5\t0.0\t2\t0 0 5\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "117\t5\t0.0\t2\t0 0 5\n",
      "119\t11\t0.0\t2\t0 0 11\n",
      "120\t25\t0.0\t2\t0 0 25\n",
      "121\t3\t0.0\t2\t0 0 3\n",
      "122\t4\t0.0\t2\t0 0 4\n",
      "123\t16\t0.0\t2\t0 0 16\n",
      "124\t54\t0.0\t2\t0 0 54\n",
      "125\t7\t0.0\t2\t0 0 7\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t15\t0.0\t2\t0 0 15\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t46\t0.0\t2\t0 0 46\n",
      "135\t13\t0.0\t2\t0 0 13\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t35\t0.0\t2\t0 0 35\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "144\t27\t0.0\t2\t0 0 27\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t17\t0.0\t2\t0 0 17\n",
      "150\t17\t0.0\t2\t0 0 17\n",
      "151\t27\t0.0\t2\t0 0 27\n",
      "152\t15\t0.0\t2\t0 0 15\n",
      "153\t10\t0.0\t2\t0 0 10\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "156\t12\t0.0\t2\t0 0 12\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t18\t0.0\t2\t0 0 18\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "176\t4\t0.0\t2\t0 0 4\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_384-LP3-ITS-0384_362_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_384-LP3-ITS-0384_362_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_384-LP3-ITS-0384_362_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_384-LP3-ITS-0384_362_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.594 s (11.645 ¬µs/read; 5.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,027\n",
      "Reads with adapters:                       698 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,027 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,359,127 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,309,736 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 698 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.5%\n",
      "  C: 57.4%\n",
      "  G: 4.0%\n",
      "  T: 7.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t93\t797.3\t0\t93\n",
      "4\t32\t199.3\t0\t32\n",
      "5\t28\t49.8\t0\t28\n",
      "6\t5\t12.5\t0\t5\n",
      "7\t4\t3.1\t0\t4\n",
      "8\t1\t0.8\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "12\t3\t0.0\t1\t0 3\n",
      "17\t1\t0.0\t1\t0 1\n",
      "18\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "27\t8\t0.0\t2\t0 0 8\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "31\t2\t0.0\t2\t0 0 2\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t3\t0.0\t2\t0 1 2\n",
      "35\t3\t0.0\t2\t0 1 2\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "45\t127\t0.0\t2\t0 3 124\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t4\t0.0\t2\t0 0 4\n",
      "56\t16\t0.0\t2\t0 0 16\n",
      "57\t6\t0.0\t2\t0 0 6\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t20\t0.0\t2\t0 1 19\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t3\t0.0\t2\t0 0 3\n",
      "73\t9\t0.0\t2\t0 0 9\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t20\t0.0\t2\t0 2 18\n",
      "78\t8\t0.0\t2\t0 0 8\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 1 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t5\t0.0\t2\t0 0 5\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t8\t0.0\t2\t0 0 8\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t6\t0.0\t2\t0 0 6\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t8\t0.0\t2\t0 0 8\n",
      "139\t15\t0.0\t2\t0 1 14\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t29\t0.0\t2\t0 0 29\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "144\t7\t0.0\t2\t0 0 7\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t13\t0.0\t2\t0 0 13\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t11\t0.0\t2\t0 0 11\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t14\t0.0\t2\t0 1 13\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "181\t1\t0.0\t2\t0 0 1\n",
      "193\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "220\t1\t0.0\t2\t0 0 1\n",
      "222\t1\t0.0\t2\t0 0 1\n",
      "259\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_385-LP3-ITS-0385_323_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_385-LP3-ITS-0385_323_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_385-LP3-ITS-0385_323_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_385-LP3-ITS-0385_323_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.238 s (11.396 ¬µs/read; 5.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,921\n",
      "Reads with adapters:                       253 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,297,221 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,263,767 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 253 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.7%\n",
      "  C: 10.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t326.9\t0\t2\n",
      "4\t3\t81.7\t0\t3\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t83\t0.0\t2\t0 1 82\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t10\t0.0\t2\t0 0 10\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t7\t0.0\t2\t0 1 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t90\t0.0\t2\t0 0 90\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_386-LP3-ITS-0386_281_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_386-LP3-ITS-0386_281_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_386-LP3-ITS-0386_281_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_386-LP3-ITS-0386_281_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.285 s (10.299 ¬µs/read; 5.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,668\n",
      "Reads with adapters:                       288 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,668 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,328,068 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,291,178 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 288 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.2%\n",
      "  C: 34.4%\n",
      "  G: 1.4%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t432.3\t0\t5\n",
      "4\t1\t108.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t63\t0.0\t2\t0 2 61\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "133\t68\t0.0\t2\t0 0 68\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t5\t0.0\t2\t0 1 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t29\t0.0\t2\t0 1 28\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "151\t21\t0.0\t2\t0 0 21\n",
      "153\t8\t0.0\t2\t0 0 8\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "164\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_387-LP3-ITS-0387_432_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_387-LP3-ITS-0387_432_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_387-LP3-ITS-0387_432_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_387-LP3-ITS-0387_432_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.329 s (10.969 ¬µs/read; 5.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,028\n",
      "Reads with adapters:                       373 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,028 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,038,428 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,989,091 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 373 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.2%\n",
      "  C: 19.6%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t8\t117.3\t0\t8\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "58\t9\t0.0\t2\t0 0 9\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t109\t0.0\t2\t0 1 108\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t39\t0.0\t2\t0 0 39\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "145\t21\t0.0\t2\t0 0 21\n",
      "146\t10\t0.0\t2\t0 0 10\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t16\t0.0\t2\t0 0 16\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t34\t0.0\t2\t0 0 34\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t42\t0.0\t2\t0 0 42\n",
      "188\t5\t0.0\t2\t0 1 4\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_388-LP3-ITS-0388_1037_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_388-LP3-ITS-0388_1037_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_388-LP3-ITS-0388_1037_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_388-LP3-ITS-0388_1037_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.300 s (9.954 ¬µs/read; 6.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,118\n",
      "Reads with adapters:                       326 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,118 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,065,518 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,028,875 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 326 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.8%\n",
      "  C: 36.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t470.6\t0\t4\n",
      "4\t9\t117.6\t0\t9\n",
      "6\t1\t7.4\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "58\t31\t0.0\t2\t0 1 30\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t6\t0.0\t2\t0 0 6\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t114\t0.0\t2\t1 1 112\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t97\t0.0\t2\t0 1 96\n",
      "134\t12\t0.0\t2\t0 0 12\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_389-LP3-ITS-0389_1017_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_389-LP3-ITS-0389_1017_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_389-LP3-ITS-0389_1017_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_389-LP3-ITS-0389_1017_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.384 s (10.922 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,149\n",
      "Reads with adapters:                       517 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,149 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,579,849 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,508,246 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 517 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 94.2%\n",
      "  C: 5.0%\n",
      "  G: 0.4%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t137.3\t0\t2\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "118\t132\t0.0\t2\t0 1 131\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "134\t27\t0.0\t2\t0 0 27\n",
      "137\t41\t0.0\t2\t0 0 41\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t95\t0.0\t2\t0 1 94\n",
      "146\t18\t0.0\t2\t0 0 18\n",
      "151\t99\t0.0\t2\t0 0 99\n",
      "153\t20\t0.0\t2\t0 0 20\n",
      "156\t8\t0.0\t2\t0 0 8\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_390-LP3-ITS-0390_880_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_390-LP3-ITS-0390_880_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_390-LP3-ITS-0390_880_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_390-LP3-ITS-0390_880_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.382 s (10.738 ¬µs/read; 5.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,553\n",
      "Reads with adapters:                       426 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,553 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,701,453 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,644,549 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 426 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.3%\n",
      "  C: 15.5%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t555.5\t0\t3\n",
      "4\t2\t138.9\t0\t2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "118\t105\t0.0\t2\t0 0 105\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "133\t44\t0.0\t2\t0 1 43\n",
      "134\t41\t0.0\t2\t0 0 41\n",
      "137\t19\t0.0\t2\t0 0 19\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "145\t29\t0.0\t2\t0 0 29\n",
      "146\t34\t0.0\t2\t0 0 34\n",
      "151\t85\t0.0\t2\t0 0 85\n",
      "153\t13\t0.0\t2\t0 0 13\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_391-LP3-ITS-0391_861_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_391-LP3-ITS-0391_861_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_391-LP3-ITS-0391_861_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_391-LP3-ITS-0391_861_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.226 s (10.900 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,742\n",
      "Reads with adapters:                       211 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,742 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,243,342 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,218,150 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 211 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.9%\n",
      "  C: 16.1%\n",
      "  G: 0.5%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t324.1\t0\t5\n",
      "4\t3\t81.0\t0\t3\n",
      "6\t1\t5.1\t0\t1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "118\t139\t0.0\t2\t0 3 136\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t25\t0.0\t2\t0 0 25\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_392-LP3-ITS-0392_769_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_392-LP3-ITS-0392_769_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_392-LP3-ITS-0392_769_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_392-LP3-ITS-0392_769_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.211 s (10.446 ¬µs/read; 5.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,243\n",
      "Reads with adapters:                       236 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,243 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,093,143 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,062,016 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 236 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.6%\n",
      "  C: 16.5%\n",
      "  G: 0.4%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t316.3\t0\t1\n",
      "4\t2\t79.1\t0\t2\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t79\t0.0\t2\t0 7 72\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t31\t0.0\t2\t0 1 30\n",
      "146\t5\t0.0\t2\t0 0 5\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "153\t25\t0.0\t2\t0 0 25\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_393-LP3-ITS-0393_657_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_393-LP3-ITS-0393_657_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_393-LP3-ITS-0393_657_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_393-LP3-ITS-0393_657_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.285 s (10.920 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,071\n",
      "Reads with adapters:                       404 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,847,371 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,794,188 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 404 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.9%\n",
      "  C: 18.8%\n",
      "  G: 0.2%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "58\t14\t0.0\t2\t0 0 14\n",
      "59\t9\t0.0\t2\t0 0 9\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t115\t0.0\t2\t0 2 113\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "126\t4\t0.0\t2\t0 0 4\n",
      "128\t8\t0.0\t2\t0 0 8\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t37\t0.0\t2\t0 1 36\n",
      "134\t19\t0.0\t2\t0 0 19\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "145\t54\t0.0\t2\t0 1 53\n",
      "146\t7\t0.0\t2\t0 0 7\n",
      "151\t16\t0.0\t2\t0 0 16\n",
      "153\t40\t0.0\t2\t0 0 40\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t28\t0.0\t2\t0 0 28\n",
      "188\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_394-LP3-ITS-0394_650_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_394-LP3-ITS-0394_650_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_394-LP3-ITS-0394_650_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_394-LP3-ITS-0394_650_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.347 s (10.579 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,811\n",
      "Reads with adapters:                       397 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,876,111 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,827,710 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 397 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.9%\n",
      "  C: 15.4%\n",
      "  G: 0.3%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t512.7\t0\t5\n",
      "4\t1\t128.2\t0\t1\n",
      "5\t1\t32.0\t0\t1\n",
      "6\t1\t8.0\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "58\t42\t0.0\t2\t0 1 41\n",
      "59\t7\t0.0\t2\t0 0 7\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t88\t0.0\t2\t0 1 87\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t35\t0.0\t2\t0 0 35\n",
      "134\t16\t0.0\t2\t0 0 16\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t98\t0.0\t2\t0 1 97\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t11\t0.0\t2\t0 0 11\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "164\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_395-LP3-ITS-0395_297_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_395-LP3-ITS-0395_297_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_395-LP3-ITS-0395_297_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_395-LP3-ITS-0395_297_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.277 s (10.512 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,306\n",
      "Reads with adapters:                       437 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,306 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,918,106 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,860,369 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 437 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.4%\n",
      "  C: 11.7%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t411.0\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t14\t0.0\t2\t0 0 14\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t110\t0.0\t2\t0 2 108\n",
      "128\t11\t0.0\t2\t0 0 11\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t67\t0.0\t2\t0 1 66\n",
      "137\t26\t0.0\t2\t0 1 25\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t48\t0.0\t2\t0 2 46\n",
      "146\t9\t0.0\t2\t0 0 9\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "153\t44\t0.0\t2\t0 0 44\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t2\t0.0\t2\t0 1 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_396-LP3-ITS-0396_394_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_396-LP3-ITS-0396_394_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_396-LP3-ITS-0396_394_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_396-LP3-ITS-0396_394_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.283 s (10.502 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,919\n",
      "Reads with adapters:                       353 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,919 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,102,619 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,060,533 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 353 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.1%\n",
      "  C: 25.2%\n",
      "  G: 0.6%\n",
      "  T: 3.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t420.6\t0\t23\n",
      "4\t10\t105.2\t0\t10\n",
      "6\t2\t6.6\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t8\t0.0\t2\t0 0 8\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t91\t0.0\t2\t0 0 91\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "128\t8\t0.0\t2\t0 0 8\n",
      "133\t27\t0.0\t2\t0 1 26\n",
      "134\t12\t0.0\t2\t0 1 11\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t22\t0.0\t2\t0 0 22\n",
      "146\t6\t0.0\t2\t0 0 6\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "153\t43\t0.0\t2\t0 0 43\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t25\t0.0\t2\t0 0 25\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t21\t0.0\t2\t0 1 20\n",
      "188\t2\t0.0\t2\t0 0 2\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_397-LP3-ITS-0397_728_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_397-LP3-ITS-0397_728_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_397-LP3-ITS-0397_728_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_397-LP3-ITS-0397_728_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.249 s (10.279 ¬µs/read; 5.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,256\n",
      "Reads with adapters:                       306 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,256 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,301,056 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,261,123 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 306 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.7%\n",
      "  C: 16.3%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t379.0\t0\t2\n",
      "4\t1\t94.8\t0\t1\n",
      "7\t2\t1.5\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "118\t76\t0.0\t2\t0 2 74\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "133\t37\t0.0\t2\t0 0 37\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "137\t35\t0.0\t2\t0 1 34\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "145\t15\t0.0\t2\t0 0 15\n",
      "146\t4\t0.0\t2\t0 0 4\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t42\t0.0\t2\t0 0 42\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t25\t0.0\t2\t0 1 24\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_398-LP3-ITS-0398_272_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_398-LP3-ITS-0398_272_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_398-LP3-ITS-0398_272_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_398-LP3-ITS-0398_272_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.230 s (10.243 ¬µs/read; 5.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,485\n",
      "Reads with adapters:                       251 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,485 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,767,985 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,734,948 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 251 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.1%\n",
      "  C: 21.9%\n",
      "  G: 0.0%\n",
      "  T: 2.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t351.3\t0\t4\n",
      "4\t1\t87.8\t0\t1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t69\t0.0\t2\t0 1 68\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t8\t0.0\t2\t0 1 7\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t51\t0.0\t2\t0 2 49\n",
      "146\t9\t0.0\t2\t0 0 9\n",
      "151\t34\t0.0\t2\t0 0 34\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_399-LP3-ITS-0399_215_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_399-LP3-ITS-0399_215_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_399-LP3-ITS-0399_215_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_399-LP3-ITS-0399_215_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.293 s (10.224 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,666\n",
      "Reads with adapters:                       321 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,628,466 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,588,473 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 321 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.5%\n",
      "  C: 25.2%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t447.9\t0\t6\n",
      "10\t2\t0.0\t1\t0 2\n",
      "45\t4\t0.0\t2\t0 0 4\n",
      "51\t10\t0.0\t2\t0 1 9\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "58\t5\t0.0\t2\t0 0 5\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t5\t0.0\t2\t0 0 5\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "118\t87\t0.0\t2\t0 1 86\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t39\t0.0\t2\t0 1 38\n",
      "134\t18\t0.0\t2\t0 0 18\n",
      "137\t31\t0.0\t2\t0 0 31\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t14\t0.0\t2\t0 0 14\n",
      "146\t10\t0.0\t2\t0 0 10\n",
      "151\t28\t0.0\t2\t0 0 28\n",
      "153\t8\t0.0\t2\t0 1 7\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "164\t12\t0.0\t2\t0 0 12\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_400-LP3-ITS-0400_569_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_400-LP3-ITS-0400_569_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_400-LP3-ITS-0400_569_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_400-LP3-ITS-0400_569_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.193 s (11.474 ¬µs/read; 5.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,861\n",
      "Reads with adapters:                       207 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,861 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,075,161 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,051,212 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 207 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.8%\n",
      "  C: 23.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "58\t23\t0.0\t2\t0 0 23\n",
      "59\t14\t0.0\t2\t0 0 14\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "118\t73\t0.0\t2\t0 0 73\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t22\t0.0\t2\t0 1 21\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t52\t0.0\t2\t0 0 52\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_401-LP3-ITS-0401_708_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_401-LP3-ITS-0401_708_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_401-LP3-ITS-0401_708_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_401-LP3-ITS-0401_708_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.183 s (10.289 ¬µs/read; 5.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,790\n",
      "Reads with adapters:                       255 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,354,790 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,324,564 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 255 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.5%\n",
      "  C: 23.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t278.0\t0\t1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t16\t0.0\t2\t0 0 16\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t145\t0.0\t2\t0 1 144\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t35\t0.0\t2\t0 0 35\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "153\t8\t0.0\t2\t0 0 8\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_402-LP3-ITS-0402_3_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_402-LP3-ITS-0402_3_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_402-LP3-ITS-0402_3_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_402-LP3-ITS-0402_3_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.240 s (10.305 ¬µs/read; 5.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,274\n",
      "Reads with adapters:                       245 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,274 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,005,474 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,973,679 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 245 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.9%\n",
      "  C: 15.9%\n",
      "  G: 0.8%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t90.9\t0\t1\n",
      "5\t2\t22.7\t0\t2\n",
      "6\t1\t5.7\t0\t1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t92\t0.0\t2\t0 4 88\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "133\t31\t0.0\t2\t0 0 31\n",
      "134\t12\t0.0\t2\t0 0 12\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t18\t0.0\t2\t0 2 16\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t21\t0.0\t2\t0 0 21\n",
      "153\t14\t0.0\t2\t0 1 13\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t12\t0.0\t2\t0 0 12\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_403-LP3-ITS-0403_790_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_403-LP3-ITS-0403_790_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_403-LP3-ITS-0403_790_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_403-LP3-ITS-0403_790_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.225 s (11.200 ¬µs/read; 5.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,077\n",
      "Reads with adapters:                       261 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,077 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,043,177 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,012,619 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 261 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.8%\n",
      "  C: 34.9%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t313.7\t0\t6\n",
      "10\t3\t0.0\t1\t0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t16\t0.0\t2\t0 0 16\n",
      "60\t5\t0.0\t2\t0 0 5\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t122\t0.0\t2\t0 1 121\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t40\t0.0\t2\t0 0 40\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_404-LP3-ITS-0404_162_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_404-LP3-ITS-0404_162_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_404-LP3-ITS-0404_162_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_404-LP3-ITS-0404_162_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.220 s (9.937 ¬µs/read; 6.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,134\n",
      "Reads with adapters:                       235 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,134 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,662,334 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,634,142 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 235 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.9%\n",
      "  C: 30.6%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t345.8\t0\t2\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t6\t0.0\t2\t0 1 5\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t128\t0.0\t2\t0 3 125\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t43\t0.0\t2\t0 0 43\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t10\t0.0\t2\t0 0 10\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_405-LP3-ITS-0405_295_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_405-LP3-ITS-0405_295_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_405-LP3-ITS-0405_295_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_405-LP3-ITS-0405_295_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.205 s (10.418 ¬µs/read; 5.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,719\n",
      "Reads with adapters:                       232 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,719 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,935,419 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,908,823 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 232 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.0%\n",
      "  C: 30.6%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t77.0\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "58\t26\t0.0\t2\t0 0 26\n",
      "59\t16\t0.0\t2\t0 0 16\n",
      "60\t5\t0.0\t2\t0 0 5\n",
      "95\t1\t0.0\t2\t0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t77\t0.0\t2\t0 0 77\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t40\t0.0\t2\t0 0 40\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t5\t0.0\t2\t0 1 4\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "153\t14\t0.0\t2\t0 0 14\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_406-LP3-ITS-0406_396_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_406-LP3-ITS-0406_396_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_406-LP3-ITS-0406_396_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_406-LP3-ITS-0406_396_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.210 s (10.352 ¬µs/read; 5.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,316\n",
      "Reads with adapters:                       252 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,316 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,115,116 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,085,716 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 252 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.1%\n",
      "  C: 15.5%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t317.4\t0\t2\n",
      "4\t5\t79.4\t0\t5\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "58\t23\t0.0\t2\t0 0 23\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "118\t103\t0.0\t2\t0 3 100\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "134\t10\t0.0\t2\t0 0 10\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t25\t0.0\t2\t0 1 24\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t17\t0.0\t2\t0 0 17\n",
      "153\t6\t0.0\t2\t0 0 6\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t10\t0.0\t2\t0 0 10\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_407-LP3-ITS-0407_660_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_407-LP3-ITS-0407_660_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_407-LP3-ITS-0407_660_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_407-LP3-ITS-0407_660_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.318 s (10.583 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,045\n",
      "Reads with adapters:                       285 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,045 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,043,545 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,005,395 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 285 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.4%\n",
      "  C: 19.3%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t469.5\t0\t2\n",
      "4\t3\t117.4\t0\t3\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "118\t55\t0.0\t2\t0 0 55\n",
      "133\t44\t0.0\t2\t0 0 44\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "145\t17\t0.0\t2\t0 1 16\n",
      "151\t108\t0.0\t2\t0 1 107\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t13\t0.0\t2\t0 0 13\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_408-LP3-ITS-0408_622_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_408-LP3-ITS-0408_622_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_408-LP3-ITS-0408_622_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_408-LP3-ITS-0408_622_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.209 s (10.217 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,484\n",
      "Reads with adapters:                     1,210 (5.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,484 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,165,684 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,148,341 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1210 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.4%\n",
      "  C: 8.2%\n",
      "  G: 0.1%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1094\t320.1\t0\t1094\n",
      "4\t1\t80.0\t0\t1\n",
      "5\t1\t20.0\t0\t1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "59\t10\t0.0\t2\t0 0 10\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "118\t36\t0.0\t2\t0 0 36\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "176\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_409-LP3-ITS-0409_597_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_409-LP3-ITS-0409_597_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_409-LP3-ITS-0409_597_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_409-LP3-ITS-0409_597_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.178 s (11.263 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,844\n",
      "Reads with adapters:                       188 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,844 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,769,044 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,747,799 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 188 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.6%\n",
      "  C: 31.9%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t247.6\t0\t4\n",
      "10\t1\t0.0\t1\t0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "59\t8\t0.0\t2\t0 1 7\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t108\t0.0\t2\t0 2 106\n",
      "133\t38\t0.0\t2\t0 1 37\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_410-LP3-ITS-0410_128_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_410-LP3-ITS-0410_128_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_410-LP3-ITS-0410_128_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_410-LP3-ITS-0410_128_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.254 s (10.859 ¬µs/read; 5.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,358\n",
      "Reads with adapters:                       244 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,358 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,030,758 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,000,399 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 244 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.4%\n",
      "  C: 24.6%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t365.0\t0\t1\n",
      "4\t4\t91.2\t0\t4\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "118\t113\t0.0\t2\t0 2 111\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t48\t0.0\t2\t0 0 48\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "145\t12\t0.0\t2\t0 1 11\n",
      "151\t20\t0.0\t2\t0 0 20\n",
      "153\t11\t0.0\t2\t0 0 11\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_411-LP3-ITS-0411_7_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_411-LP3-ITS-0411_7_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_411-LP3-ITS-0411_7_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_411-LP3-ITS-0411_7_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.324 s (10.022 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,372\n",
      "Reads with adapters:                       409 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,372 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,743,972 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,696,828 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 409 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.3%\n",
      "  C: 26.4%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t505.8\t0\t3\n",
      "4\t8\t126.5\t0\t8\n",
      "10\t1\t0.0\t1\t0 1\n",
      "47\t3\t0.0\t2\t0 1 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "58\t28\t0.0\t2\t0 1 27\n",
      "59\t14\t0.0\t2\t0 0 14\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t185\t0.0\t2\t0 3 182\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "133\t74\t0.0\t2\t0 1 73\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "142\t17\t0.0\t2\t0 0 17\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "146\t6\t0.0\t2\t0 0 6\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t11\t0.0\t2\t0 0 11\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_412-LP3-ITS-0412_397_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_412-LP3-ITS-0412_397_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_412-LP3-ITS-0412_397_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_412-LP3-ITS-0412_397_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.162 s (10.317 ¬µs/read; 5.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,678\n",
      "Reads with adapters:                       196 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,678 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,719,078 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,696,597 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 196 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.4%\n",
      "  C: 28.6%\n",
      "  G: 0.5%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "59\t14\t0.0\t2\t0 1 13\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t100\t0.0\t2\t0 1 99\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t2\t0.0\t2\t0 1 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_413-LP3-ITS-0413_325_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_413-LP3-ITS-0413_325_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_413-LP3-ITS-0413_325_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_413-LP3-ITS-0413_325_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.154 s (11.707 ¬µs/read; 5.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,188\n",
      "Reads with adapters:                       151 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,188 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,969,588 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,950,396 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 151 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.4%\n",
      "  C: 15.9%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t206.1\t0\t3\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "118\t66\t0.0\t2\t0 1 65\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t18\t0.0\t2\t0 0 18\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t16\t0.0\t2\t0 0 16\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_414-LP3-ITS-0414_427_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_414-LP3-ITS-0414_427_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_414-LP3-ITS-0414_427_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_414-LP3-ITS-0414_427_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.197 s (9.933 ¬µs/read; 6.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,785\n",
      "Reads with adapters:                       408 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,955,285 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,931,334 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 408 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.3%\n",
      "  C: 12.7%\n",
      "  G: 0.5%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t208\t309.1\t0\t208\n",
      "4\t10\t77.3\t0\t10\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t103\t0.0\t2\t0 3 100\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "146\t4\t0.0\t2\t0 0 4\n",
      "151\t25\t0.0\t2\t0 0 25\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_415-LP3-ITS-0415_699_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_415-LP3-ITS-0415_699_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_415-LP3-ITS-0415_699_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_415-LP3-ITS-0415_699_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.235 s (9.070 ¬µs/read; 6.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,890\n",
      "Reads with adapters:                     2,313 (8.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,792,890 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,779,379 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2313 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.5%\n",
      "  C: 6.2%\n",
      "  G: 0.1%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2265\t404.5\t0\t2265\n",
      "4\t8\t101.1\t0\t8\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 1\n",
      "176\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_416-LP3-ITS-0416_606_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_416-LP3-ITS-0416_606_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_416-LP3-ITS-0416_606_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_416-LP3-ITS-0416_606_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (34.985 ¬µs/read; 1.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     209\n",
      "Reads with adapters:                        11 (5.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           209 (100.0%)\n",
      "\n",
      "Total basepairs processed:        62,909 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         61,884 bp (98.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.9%\n",
      "  C: 9.1%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t3\t0.8\t0\t3\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_417-LP3-ITS-0417_349_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_417-LP3-ITS-0417_349_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_417-LP3-ITS-0417_349_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_417-LP3-ITS-0417_349_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.242 s (11.357 ¬µs/read; 5.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,268\n",
      "Reads with adapters:                       367 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,268 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,401,668 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,354,602 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 367 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.6%\n",
      "  C: 25.6%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "58\t9\t0.0\t2\t0 0 9\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t86\t0.0\t2\t0 1 85\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t51\t0.0\t2\t0 1 50\n",
      "134\t54\t0.0\t2\t0 0 54\n",
      "137\t79\t0.0\t2\t0 1 78\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "145\t20\t0.0\t2\t0 0 20\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t12\t0.0\t2\t0 0 12\n",
      "155\t19\t0.0\t2\t0 0 19\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_418-LP3-ITS-0418_1067_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_418-LP3-ITS-0418_1067_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_418-LP3-ITS-0418_1067_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_418-LP3-ITS-0418_1067_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.146 s (10.574 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,854\n",
      "Reads with adapters:                       164 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,170,054 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,149,104 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 164 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.0%\n",
      "  C: 20.7%\n",
      "  G: 0.6%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t216.5\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "118\t44\t0.0\t2\t0 2 42\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t21\t0.0\t2\t0 1 20\n",
      "134\t24\t0.0\t2\t0 0 24\n",
      "137\t21\t0.0\t2\t0 0 21\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t27\t0.0\t2\t0 2 25\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 1 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_419-LP3-ITS-0419_270_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_419-LP3-ITS-0419_270_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_419-LP3-ITS-0419_270_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_419-LP3-ITS-0419_270_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.538 s (10.482 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,344\n",
      "Reads with adapters:                       698 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,344 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,454,544 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,362,081 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 698 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.2%\n",
      "  C: 23.9%\n",
      "  G: 0.3%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t802.2\t0\t2\n",
      "5\t5\t50.1\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t11\t0.0\t2\t0 0 11\n",
      "118\t109\t0.0\t2\t0 3 106\n",
      "127\t7\t0.0\t2\t0 0 7\n",
      "128\t16\t0.0\t2\t0 0 16\n",
      "132\t31\t0.0\t2\t0 0 31\n",
      "133\t81\t0.0\t2\t0 2 79\n",
      "134\t56\t0.0\t2\t0 0 56\n",
      "137\t136\t0.0\t2\t0 5 131\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t145\t0.0\t2\t0 1 144\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t20\t0.0\t2\t0 0 20\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_420-LP3-ITS-0420_283_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_420-LP3-ITS-0420_283_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_420-LP3-ITS-0420_283_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_420-LP3-ITS-0420_283_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.288 s (10.384 ¬µs/read; 5.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,708\n",
      "Reads with adapters:                       407 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,708 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,340,108 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,287,360 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 407 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.9%\n",
      "  C: 21.4%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t432.9\t0\t2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t6\t0.0\t2\t0 0 6\n",
      "51\t11\t0.0\t2\t0 0 11\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t62\t0.0\t2\t0 2 60\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "128\t20\t0.0\t2\t0 0 20\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t45\t0.0\t2\t0 0 45\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "137\t159\t0.0\t2\t0 2 157\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t22\t0.0\t2\t0 0 22\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t21\t0.0\t2\t0 0 21\n",
      "155\t6\t0.0\t2\t0 1 5\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_421-LP3-ITS-0421_834_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_421-LP3-ITS-0421_834_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_421-LP3-ITS-0421_834_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_421-LP3-ITS-0421_834_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.132 s (11.016 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,983\n",
      "Reads with adapters:                       226 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,983 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,606,883 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,584,726 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 226 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.9%\n",
      "  C: 22.1%\n",
      "  G: 0.4%\n",
      "  T: 3.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t41\t187.2\t0\t41\n",
      "9\t4\t0.0\t0\t4\n",
      "10\t16\t0.0\t1\t0 16\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "118\t42\t0.0\t2\t0 0 42\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "134\t29\t0.0\t2\t0 2 27\n",
      "137\t15\t0.0\t2\t0 1 14\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t53\t0.0\t2\t0 1 52\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_422-LP3-ITS-0422_519_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_422-LP3-ITS-0422_519_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_422-LP3-ITS-0422_519_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_422-LP3-ITS-0422_519_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.598 s (10.015 ¬µs/read; 5.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  59,686\n",
      "Reads with adapters:                       648 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        59,686 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,965,486 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,883,310 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 648 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.1%\n",
      "  C: 31.2%\n",
      "  G: 0.3%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t932.6\t0\t19\n",
      "10\t3\t0.1\t1\t1 2\n",
      "47\t7\t0.0\t2\t0 1 6\n",
      "51\t9\t0.0\t2\t0 0 9\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t4\t0.0\t2\t0 0 4\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "118\t114\t0.0\t2\t0 1 113\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t6\t0.0\t2\t0 0 6\n",
      "133\t146\t0.0\t2\t0 0 146\n",
      "134\t27\t0.0\t2\t0 2 25\n",
      "137\t167\t0.0\t2\t0 3 164\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t94\t0.0\t2\t0 0 94\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t13\t0.0\t2\t0 0 13\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_423-LP3-ITS-0423_324_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_423-LP3-ITS-0423_324_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_423-LP3-ITS-0423_324_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_423-LP3-ITS-0423_324_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.341 s (10.987 ¬µs/read; 5.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,048\n",
      "Reads with adapters:                       394 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,048 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,345,448 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,294,745 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 394 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.5%\n",
      "  C: 18.0%\n",
      "  G: 0.3%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t485.1\t0\t5\n",
      "47\t2\t0.0\t2\t0 1 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "58\t27\t0.0\t2\t0 2 25\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 1 2\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t42\t0.0\t2\t0 2 40\n",
      "128\t15\t0.0\t2\t0 0 15\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t52\t0.0\t2\t0 2 50\n",
      "134\t13\t0.0\t2\t0 1 12\n",
      "137\t55\t0.0\t2\t0 0 55\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t134\t0.0\t2\t0 0 134\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "153\t9\t0.0\t2\t0 0 9\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_424-LP3-ITS-0424_434_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_424-LP3-ITS-0424_434_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_424-LP3-ITS-0424_434_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_424-LP3-ITS-0424_434_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.177 s (10.733 ¬µs/read; 5.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,476\n",
      "Reads with adapters:                       237 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,476 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,959,276 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,927,793 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 237 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.5%\n",
      "  C: 15.2%\n",
      "  G: 0.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t257.4\t0\t2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "58\t8\t0.0\t2\t0 0 8\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t43\t0.0\t2\t0 1 42\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "128\t9\t0.0\t2\t0 1 8\n",
      "133\t25\t0.0\t2\t0 0 25\n",
      "134\t10\t0.0\t2\t0 1 9\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t24\t0.0\t2\t0 0 24\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "145\t84\t0.0\t2\t0 2 82\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t6\t0.0\t2\t0 1 5\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_425-LP3-ITS-0425_114_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_425-LP3-ITS-0425_114_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_425-LP3-ITS-0425_114_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_425-LP3-ITS-0425_114_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.324 s (11.362 ¬µs/read; 5.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,520\n",
      "Reads with adapters:                       422 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,520 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,584,520 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,528,573 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 422 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.8%\n",
      "  C: 18.5%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t445.6\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t11\t0.0\t2\t0 0 11\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "118\t71\t0.0\t2\t0 3 68\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t44\t0.0\t2\t0 3 41\n",
      "134\t46\t0.0\t2\t0 0 46\n",
      "137\t97\t0.0\t2\t0 1 96\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "145\t54\t0.0\t2\t0 1 53\n",
      "151\t4\t0.0\t2\t0 1 3\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t44\t0.0\t2\t0 0 44\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_426-LP3-ITS-0426_691_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_426-LP3-ITS-0426_691_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_426-LP3-ITS-0426_691_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_426-LP3-ITS-0426_691_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (30.931 ¬µs/read; 1.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     250\n",
      "Reads with adapters:                        11 (4.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           250 (100.0%)\n",
      "\n",
      "Total basepairs processed:        75,250 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         74,309 bp (98.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.6%\n",
      "  C: 27.3%\n",
      "  G: 0.0%\n",
      "  T: 9.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t3.9\t0\t4\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_427-LP3-ITS-0427_1055_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_427-LP3-ITS-0427_1055_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_427-LP3-ITS-0427_1055_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_427-LP3-ITS-0427_1055_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (40.625 ¬µs/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     189\n",
      "Reads with adapters:                         6 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           189 (100.0%)\n",
      "\n",
      "Total basepairs processed:        56,889 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         56,324 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t3.0\t0\t1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_428-LP3-ITS-0428_1068_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_428-LP3-ITS-0428_1068_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_428-LP3-ITS-0428_1068_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_428-LP3-ITS-0428_1068_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (29.217 ¬µs/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     267\n",
      "Reads with adapters:                         6 (2.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           267 (100.0%)\n",
      "\n",
      "Total basepairs processed:        80,367 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         79,874 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.3%\n",
      "  C: 16.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t4.2\t0\t2\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_429-LP3-ITS-0429_172_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_429-LP3-ITS-0429_172_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_429-LP3-ITS-0429_172_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_429-LP3-ITS-0429_172_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.198 s (10.294 ¬µs/read; 5.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,211\n",
      "Reads with adapters:                       224 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,211 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,782,511 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,754,116 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 224 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.7%\n",
      "  C: 21.4%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t300.2\t0\t1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t12\t0.0\t2\t0 0 12\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "118\t50\t0.0\t2\t0 1 49\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "134\t20\t0.0\t2\t0 0 20\n",
      "137\t48\t0.0\t2\t0 1 47\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t28\t0.0\t2\t0 0 28\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "164\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_430-LP3-ITS-0430_94_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_430-LP3-ITS-0430_94_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_430-LP3-ITS-0430_94_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_430-LP3-ITS-0430_94_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.229 s (10.555 ¬µs/read; 5.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,680\n",
      "Reads with adapters:                     1,184 (5.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,525,680 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,489,021 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1184 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.3%\n",
      "  C: 43.3%\n",
      "  G: 1.8%\n",
      "  T: 15.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t723\t338.8\t0\t723\n",
      "4\t1\t84.7\t0\t1\n",
      "9\t24\t0.1\t0\t24\n",
      "10\t185\t0.0\t1\t14 171\n",
      "58\t27\t0.0\t2\t0 0 27\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t39\t0.0\t2\t0 0 39\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t36\t0.0\t2\t0 0 36\n",
      "137\t23\t0.0\t2\t0 0 23\n",
      "145\t67\t0.0\t2\t0 1 66\n",
      "151\t29\t0.0\t2\t0 1 28\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_431-LP3-ITS-0431_800_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_431-LP3-ITS-0431_800_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_431-LP3-ITS-0431_800_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_431-LP3-ITS-0431_800_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.476 s (10.498 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,319\n",
      "Reads with adapters:                     1,670 (3.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,319 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,641,019 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,579,351 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1670 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.7%\n",
      "  C: 48.9%\n",
      "  G: 0.7%\n",
      "  T: 12.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t984\t708.1\t0\t984\n",
      "4\t2\t177.0\t0\t2\n",
      "9\t25\t0.2\t0\t23 2\n",
      "10\t237\t0.0\t1\t13 224\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t10\t0.0\t2\t0 0 10\n",
      "65\t1\t0.0\t2\t0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t127\t0.0\t2\t0 0 127\n",
      "128\t9\t0.0\t2\t0 0 9\n",
      "132\t7\t0.0\t2\t0 0 7\n",
      "133\t51\t0.0\t2\t0 0 51\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t50\t0.0\t2\t0 0 50\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t35\t0.0\t2\t0 0 35\n",
      "151\t47\t0.0\t2\t0 1 46\n",
      "153\t36\t0.0\t2\t0 1 35\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t12\t0.0\t2\t0 0 12\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "188\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_432-LP3-ITS-0432_428_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_432-LP3-ITS-0432_428_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_432-LP3-ITS-0432_428_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_432-LP3-ITS-0432_428_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.316 s (10.279 ¬µs/read; 5.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,735\n",
      "Reads with adapters:                       340 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,735 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,251,235 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,207,842 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 340 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.6%\n",
      "  C: 17.9%\n",
      "  G: 0.0%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t480.2\t0\t2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "58\t17\t0.0\t2\t0 0 17\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t54\t0.0\t2\t0 0 54\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t40\t0.0\t2\t0 1 39\n",
      "134\t40\t0.0\t2\t0 0 40\n",
      "137\t61\t0.0\t2\t0 0 61\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t82\t0.0\t2\t0 1 81\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t8\t0.0\t2\t0 0 8\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_433-LP3-ITS-0433_560_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_433-LP3-ITS-0433_560_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_433-LP3-ITS-0433_560_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_433-LP3-ITS-0433_560_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.200 s (10.588 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,877\n",
      "Reads with adapters:                       917 (4.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,877 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,681,977 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,661,212 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 917 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.9%\n",
      "  C: 44.2%\n",
      "  G: 2.4%\n",
      "  T: 18.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t592\t295.0\t0\t592\n",
      "8\t24\t0.3\t0\t24\n",
      "9\t38\t0.1\t0\t36 2\n",
      "10\t130\t0.0\t1\t4 126\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "118\t56\t0.0\t2\t0 2 54\n",
      "133\t15\t0.0\t2\t0 0 15\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t22\t0.0\t2\t0 0 22\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_434-LP3-ITS-0434_1058_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_434-LP3-ITS-0434_1058_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_434-LP3-ITS-0434_1058_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_434-LP3-ITS-0434_1058_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.220 s (10.319 ¬µs/read; 5.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,334\n",
      "Reads with adapters:                       278 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,334 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,421,534 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,388,626 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 278 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.2%\n",
      "  C: 21.6%\n",
      "  G: 1.4%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t333.3\t0\t21\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "58\t16\t0.0\t2\t0 0 16\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t49\t0.0\t2\t0 1 48\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "128\t18\t0.0\t2\t0 0 18\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "134\t6\t0.0\t2\t0 1 5\n",
      "137\t35\t0.0\t2\t0 0 35\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t56\t0.0\t2\t0 1 55\n",
      "153\t13\t0.0\t2\t0 0 13\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_435-LP3-ITS-0435_701_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_435-LP3-ITS-0435_701_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_435-LP3-ITS-0435_701_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_435-LP3-ITS-0435_701_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.218 s (10.445 ¬µs/read; 5.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,846\n",
      "Reads with adapters:                       553 (2.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,274,646 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,233,931 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 553 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 56.6%\n",
      "  C: 30.0%\n",
      "  G: 0.9%\n",
      "  T: 12.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t217\t325.7\t0\t217\n",
      "10\t43\t0.0\t1\t3 40\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "118\t78\t0.0\t2\t0 4 74\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t13\t0.0\t2\t0 0 13\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "134\t16\t0.0\t2\t0 0 16\n",
      "137\t54\t0.0\t2\t0 0 54\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "145\t57\t0.0\t2\t0 2 55\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t25\t0.0\t2\t0 0 25\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "164\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_436-LP3-ITS-0436_996_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_436-LP3-ITS-0436_996_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_436-LP3-ITS-0436_996_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_436-LP3-ITS-0436_996_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.173 s (10.635 ¬µs/read; 5.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,227\n",
      "Reads with adapters:                       215 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,884,327 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,857,679 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 215 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.3%\n",
      "  C: 16.3%\n",
      "  G: 0.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t253.5\t0\t7\n",
      "10\t1\t0.0\t1\t1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t12\t0.0\t2\t0 0 12\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t42\t0.0\t2\t0 1 41\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t9\t0.0\t2\t0 0 9\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "134\t44\t0.0\t2\t0 1 43\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "145\t28\t0.0\t2\t0 0 28\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t8\t0.0\t2\t0 0 8\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_437-LP3-ITS-0437_785_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_437-LP3-ITS-0437_785_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_437-LP3-ITS-0437_785_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_437-LP3-ITS-0437_785_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.180 s (10.418 ¬µs/read; 5.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,232\n",
      "Reads with adapters:                       304 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,186,832 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,159,951 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 304 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 62.2%\n",
      "  C: 29.3%\n",
      "  G: 0.7%\n",
      "  T: 7.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t71\t269.2\t0\t71\n",
      "4\t1\t67.3\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t20\t0.0\t1\t1 19\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "58\t24\t0.0\t2\t0 0 24\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t54\t0.0\t2\t0 0 54\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t34\t0.0\t2\t0 0 34\n",
      "134\t13\t0.0\t2\t0 0 13\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "145\t44\t0.0\t2\t0 0 44\n",
      "153\t8\t0.0\t2\t0 0 8\n",
      "154\t1\t0.0\t2\t0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_438-LP3-ITS-0438_1073_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_438-LP3-ITS-0438_1073_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_438-LP3-ITS-0438_1073_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_438-LP3-ITS-0438_1073_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.375 s (10.752 ¬µs/read; 5.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,851\n",
      "Reads with adapters:                       476 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,851 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,490,151 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,427,609 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 476 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.8%\n",
      "  C: 13.0%\n",
      "  G: 0.0%\n",
      "  T: 4.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t544.5\t0\t10\n",
      "10\t2\t0.0\t1\t0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "70\t3\t0.0\t2\t0 0 3\n",
      "118\t111\t0.0\t2\t0 2 109\n",
      "128\t10\t0.0\t2\t0 1 9\n",
      "133\t42\t0.0\t2\t0 0 42\n",
      "134\t25\t0.0\t2\t0 1 24\n",
      "137\t96\t0.0\t2\t0 0 96\n",
      "145\t127\t0.0\t2\t0 1 126\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t19\t0.0\t2\t0 0 19\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "164\t15\t0.0\t2\t0 1 14\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_439-LP3-ITS-0439_1133_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_439-LP3-ITS-0439_1133_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_439-LP3-ITS-0439_1133_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_439-LP3-ITS-0439_1133_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.293 s (11.146 ¬µs/read; 5.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,313\n",
      "Reads with adapters:                       331 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,313 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,920,213 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,878,510 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 331 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.5%\n",
      "  C: 20.8%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t411.1\t0\t6\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t23\t0.0\t2\t0 0 23\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t73\t0.0\t2\t0 2 71\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t50\t0.0\t2\t0 1 49\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t25\t0.0\t2\t0 1 24\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t84\t0.0\t2\t0 0 84\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 1 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t12\t0.0\t2\t0 0 12\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_440-LP3-ITS-0440_976_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_440-LP3-ITS-0440_976_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_440-LP3-ITS-0440_976_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_440-LP3-ITS-0440_976_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.252 s (10.250 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,538\n",
      "Reads with adapters:                       441 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,538 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,385,938 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,353,662 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 441 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 57.1%\n",
      "  C: 34.0%\n",
      "  G: 0.9%\n",
      "  T: 7.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t165\t383.4\t0\t165\n",
      "9\t6\t0.1\t0\t6\n",
      "10\t34\t0.0\t1\t1 33\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t59\t0.0\t2\t0 1 58\n",
      "128\t11\t0.0\t2\t0 0 11\n",
      "133\t22\t0.0\t2\t0 1 21\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t34\t0.0\t2\t0 0 34\n",
      "145\t66\t0.0\t2\t0 1 65\n",
      "151\t21\t0.0\t2\t0 1 20\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_441-LP3-ITS-0441_425_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_441-LP3-ITS-0441_425_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_441-LP3-ITS-0441_425_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_441-LP3-ITS-0441_425_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.205 s (11.485 ¬µs/read; 5.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,875\n",
      "Reads with adapters:                       209 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,875 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,380,375 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,361,142 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 209 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.3%\n",
      "  C: 12.0%\n",
      "  G: 0.0%\n",
      "  T: 6.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t279.3\t0\t27\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t39\t0.1\t0\t35 4\n",
      "10\t2\t0.0\t1\t0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "118\t36\t0.0\t2\t0 1 35\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t15\t0.0\t2\t0 1 14\n",
      "134\t12\t0.0\t2\t0 0 12\n",
      "135\t1\t0.0\t2\t0 1\n",
      "137\t25\t0.0\t2\t0 0 25\n",
      "145\t33\t0.0\t2\t0 0 33\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 1 3\n",
      "165\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_442-LP3-ITS-0442_321_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_442-LP3-ITS-0442_321_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_442-LP3-ITS-0442_321_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_442-LP3-ITS-0442_321_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.298 s (10.830 ¬µs/read; 5.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,557\n",
      "Reads with adapters:                       315 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,557 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,294,657 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,253,832 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 315 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.8%\n",
      "  C: 28.9%\n",
      "  G: 1.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t430.6\t0\t10\n",
      "5\t1\t26.9\t0\t1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t69\t0.0\t2\t0 1 68\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t65\t0.0\t2\t0 1 64\n",
      "134\t19\t0.0\t2\t0 0 19\n",
      "137\t53\t0.0\t2\t0 0 53\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t33\t0.0\t2\t0 0 33\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t32\t0.0\t2\t0 0 32\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t9\t0.0\t2\t0 1 8\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_443-LP3-ITS-0443_763_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_443-LP3-ITS-0443_763_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_443-LP3-ITS-0443_763_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_443-LP3-ITS-0443_763_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.353 s (10.295 ¬µs/read; 5.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,281\n",
      "Reads with adapters:                       516 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,281 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,318,581 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,255,509 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 516 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.0%\n",
      "  C: 17.4%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t535.6\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t79\t0.0\t2\t0 0 79\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "118\t86\t0.0\t2\t0 1 85\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "127\t9\t0.0\t2\t0 0 9\n",
      "128\t15\t0.0\t2\t0 0 15\n",
      "133\t72\t0.0\t2\t0 0 72\n",
      "134\t47\t0.0\t2\t0 2 45\n",
      "137\t63\t0.0\t2\t0 0 63\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t65\t0.0\t2\t0 1 64\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t5\t0.0\t2\t0 0 5\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "164\t42\t0.0\t2\t0 0 42\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_444-LP3-ITS-0444_48_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_444-LP3-ITS-0444_48_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_444-LP3-ITS-0444_48_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_444-LP3-ITS-0444_48_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.343 s (10.875 ¬µs/read; 5.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,539\n",
      "Reads with adapters:                       471 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,539 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,493,239 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,431,203 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 471 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.5%\n",
      "  C: 28.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t492.8\t0\t6\n",
      "10\t2\t0.0\t1\t0 2\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t53\t0.0\t2\t0 0 53\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t15\t0.0\t2\t0 0 15\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t76\t0.0\t2\t0 1 75\n",
      "134\t113\t0.0\t2\t0 1 112\n",
      "137\t48\t0.0\t2\t0 1 47\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t72\t0.0\t2\t0 0 72\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t13\t0.0\t2\t0 0 13\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_445-LP3-ITS-0445_807_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_445-LP3-ITS-0445_807_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_445-LP3-ITS-0445_807_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_445-LP3-ITS-0445_807_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.210 s (12.099 ¬µs/read; 4.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,371\n",
      "Reads with adapters:                     3,425 (19.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,371 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,228,671 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,200,660 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3425 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.8%\n",
      "  C: 58.4%\n",
      "  G: 2.3%\n",
      "  T: 16.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2659\t271.4\t0\t2659\n",
      "4\t1\t67.9\t0\t1\n",
      "9\t51\t0.1\t0\t41 10\n",
      "10\t614\t0.0\t1\t31 583\n",
      "11\t1\t0.0\t1\t0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "118\t25\t0.0\t2\t0 0 25\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t44\t0.0\t2\t0 0 44\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "158\t3\t0.0\t2\t0 1 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_446-LP3-ITS-0446_1077_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_446-LP3-ITS-0446_1077_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_446-LP3-ITS-0446_1077_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_446-LP3-ITS-0446_1077_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.336 s (11.138 ¬µs/read; 5.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,171\n",
      "Reads with adapters:                     6,148 (20.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,081,471 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,037,893 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6148 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.2%\n",
      "  C: 59.6%\n",
      "  G: 1.5%\n",
      "  T: 17.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4892\t471.4\t0\t4892\n",
      "4\t1\t117.9\t0\t1\n",
      "6\t1\t7.4\t0\t1\n",
      "9\t94\t0.1\t0\t88 6\n",
      "10\t1029\t0.0\t1\t52 977\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t38\t0.0\t2\t0 2 36\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "145\t32\t0.0\t2\t0 0 32\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "234\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_447-LP3-ITS-0447_90_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_447-LP3-ITS-0447_90_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_447-LP3-ITS-0447_90_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_447-LP3-ITS-0447_90_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.237 s (10.556 ¬µs/read; 5.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,491\n",
      "Reads with adapters:                       299 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,491 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,769,791 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,730,744 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 299 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.0%\n",
      "  C: 12.7%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t351.4\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t15\t0.0\t2\t0 1 14\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t41\t0.0\t2\t0 1 40\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t69\t0.0\t2\t0 0 69\n",
      "137\t28\t0.0\t2\t0 0 28\n",
      "145\t74\t0.0\t2\t0 2 72\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_448-LP3-ITS-0448_476_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_448-LP3-ITS-0448_476_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_448-LP3-ITS-0448_476_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_448-LP3-ITS-0448_476_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.223 s (10.515 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,164\n",
      "Reads with adapters:                       342 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,164 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,370,364 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,323,978 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 342 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.6%\n",
      "  C: 15.8%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t330.7\t0\t1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "70\t2\t0.0\t2\t0 1 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t45\t0.0\t2\t0 1 44\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t40\t0.0\t2\t0 0 40\n",
      "134\t46\t0.0\t2\t0 1 45\n",
      "137\t66\t0.0\t2\t0 1 65\n",
      "145\t94\t0.0\t2\t0 1 93\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t8\t0.0\t2\t0 0 8\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t13\t0.0\t2\t0 0 13\n",
      "180\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_449-LP3-ITS-0449_233_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_449-LP3-ITS-0449_233_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_449-LP3-ITS-0449_233_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_449-LP3-ITS-0449_233_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.197 s (11.474 ¬µs/read; 5.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,177\n",
      "Reads with adapters:                       271 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,177 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,170,277 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,136,370 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 271 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.7%\n",
      "  C: 17.0%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t268.4\t0\t1\n",
      "4\t2\t67.1\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "58\t26\t0.0\t2\t0 0 26\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "118\t46\t0.0\t2\t0 0 46\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t25\t0.0\t2\t0 0 25\n",
      "134\t39\t0.0\t2\t0 0 39\n",
      "137\t35\t0.0\t2\t0 1 34\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t25\t0.0\t2\t0 0 25\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t39\t0.0\t2\t0 1 38\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_450-LP3-ITS-0450_711_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_450-LP3-ITS-0450_711_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_450-LP3-ITS-0450_711_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_450-LP3-ITS-0450_711_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.203 s (11.379 ¬µs/read; 5.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,807\n",
      "Reads with adapters:                     1,970 (11.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,807 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,359,907 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,334,293 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1970 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.6%\n",
      "  C: 53.1%\n",
      "  G: 1.6%\n",
      "  T: 22.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1529\t278.2\t0\t1529\n",
      "4\t1\t69.6\t0\t1\n",
      "8\t3\t0.3\t0\t3\n",
      "9\t16\t0.1\t0\t16\n",
      "10\t293\t0.0\t1\t19 274\n",
      "11\t1\t0.0\t1\t0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t23\t0.0\t2\t0 1 22\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t15\t0.0\t2\t0 3 12\n",
      "151\t31\t0.0\t2\t0 0 31\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "234\t4\t0.0\t2\t0 0 4\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_451-LP3-ITS-0451_300_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_451-LP3-ITS-0451_300_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_451-LP3-ITS-0451_300_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_451-LP3-ITS-0451_300_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.343 s (10.288 ¬µs/read; 5.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,299\n",
      "Reads with adapters:                     2,020 (6.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,299 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,022,999 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,972,309 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2020 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.6%\n",
      "  C: 45.9%\n",
      "  G: 2.1%\n",
      "  T: 19.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1314\t520.3\t0\t1314\n",
      "4\t2\t130.1\t0\t2\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t71\t0.1\t0\t68 3\n",
      "10\t320\t0.0\t1\t20 300\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "81\t5\t0.0\t2\t0 0 5\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "118\t51\t0.0\t2\t0 1 50\n",
      "132\t9\t0.0\t2\t0 0 9\n",
      "133\t49\t0.0\t2\t0 0 49\n",
      "134\t19\t0.0\t2\t0 0 19\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t83\t0.0\t2\t0 0 83\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t32\t0.0\t2\t0 1 31\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t28\t0.0\t2\t0 0 28\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_452-LP3-ITS-0452_74_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_452-LP3-ITS-0452_74_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_452-LP3-ITS-0452_74_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_452-LP3-ITS-0452_74_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.318 s (10.795 ¬µs/read; 5.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,468\n",
      "Reads with adapters:                       487 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,468 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,869,868 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,813,437 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 487 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.1%\n",
      "  C: 29.4%\n",
      "  G: 0.4%\n",
      "  T: 3.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t60\t460.4\t0\t60\n",
      "9\t2\t0.1\t0\t2\n",
      "10\t14\t0.0\t1\t2 12\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t67\t0.0\t2\t0 1 66\n",
      "128\t7\t0.0\t2\t0 0 7\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t77\t0.0\t2\t0 2 75\n",
      "134\t17\t0.0\t2\t0 0 17\n",
      "137\t38\t0.0\t2\t0 0 38\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t146\t0.0\t2\t0 4 142\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "164\t19\t0.0\t2\t0 0 19\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_453-LP3-ITS-0453_984_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_453-LP3-ITS-0453_984_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_453-LP3-ITS-0453_984_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_453-LP3-ITS-0453_984_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.195 s (10.999 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,750\n",
      "Reads with adapters:                       236 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,750 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,342,750 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,316,377 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 236 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.5%\n",
      "  C: 39.8%\n",
      "  G: 0.0%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t277.3\t0\t15\n",
      "10\t5\t0.0\t1\t0 5\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t110\t0.0\t2\t0 1 109\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t53\t0.0\t2\t0 1 52\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_454-LP3-ITS-0454_724_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_454-LP3-ITS-0454_724_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_454-LP3-ITS-0454_724_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_454-LP3-ITS-0454_724_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.213 s (10.090 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,140\n",
      "Reads with adapters:                       215 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,140 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,363,140 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,336,841 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 215 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.4%\n",
      "  C: 31.2%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t330.3\t0\t8\n",
      "6\t1\t5.2\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t74\t0.0\t2\t0 4 70\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t41\t0.0\t2\t0 0 41\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t20\t0.0\t2\t0 1 19\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t21\t0.0\t2\t0 0 21\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_455-LP3-ITS-0455_917_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_455-LP3-ITS-0455_917_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_455-LP3-ITS-0455_917_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_455-LP3-ITS-0455_917_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.980 s (10.488 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  93,467\n",
      "Reads with adapters:                       948 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        93,467 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,133,567 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     28,013,223 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 948 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.5%\n",
      "  C: 27.2%\n",
      "  G: 0.1%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1460.4\t0\t5\n",
      "10\t1\t0.1\t1\t0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "58\t19\t0.0\t2\t0 0 19\n",
      "59\t17\t0.0\t2\t0 0 17\n",
      "60\t12\t0.0\t2\t0 0 12\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t3\t0.0\t2\t0 1 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t322\t0.0\t2\t0 6 316\n",
      "128\t57\t0.0\t2\t0 1 56\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "133\t137\t0.0\t2\t0 1 136\n",
      "134\t47\t0.0\t2\t0 0 47\n",
      "137\t29\t0.0\t2\t0 0 29\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t72\t0.0\t2\t0 0 72\n",
      "146\t5\t0.0\t2\t0 0 5\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t22\t0.0\t2\t0 0 22\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t59\t0.0\t2\t0 0 59\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t31\t0.0\t2\t0 0 31\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t12\t0.0\t2\t0 0 12\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "162\t3\t0.0\t2\t0 0 3\n",
      "164\t35\t0.0\t2\t0 0 35\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_456-LP3-ITS-0456_718_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_456-LP3-ITS-0456_718_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_456-LP3-ITS-0456_718_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_456-LP3-ITS-0456_718_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (57.575 ¬µs/read; 1.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     101\n",
      "Reads with adapters:                         5 (5.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           101 (100.0%)\n",
      "\n",
      "Total basepairs processed:        30,401 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         30,034 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.0%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.6\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_457-LP3-ITS-0457_17_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_457-LP3-ITS-0457_17_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_457-LP3-ITS-0457_17_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_457-LP3-ITS-0457_17_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.353 s (10.589 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,311\n",
      "Reads with adapters:                       518 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,311 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,026,611 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,962,749 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 518 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.0%\n",
      "  C: 15.3%\n",
      "  G: 0.0%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t520.5\t0\t14\n",
      "4\t1\t130.1\t0\t1\n",
      "6\t1\t8.1\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "58\t61\t0.0\t2\t0 2 59\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "118\t54\t0.0\t2\t0 2 52\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t22\t0.0\t2\t0 0 22\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t50\t0.0\t2\t0 1 49\n",
      "134\t25\t0.0\t2\t0 0 25\n",
      "137\t58\t0.0\t2\t0 1 57\n",
      "145\t148\t0.0\t2\t0 3 145\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t24\t0.0\t2\t0 0 24\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t22\t0.0\t2\t0 1 21\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_458-LP3-ITS-0458_905_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_458-LP3-ITS-0458_905_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_458-LP3-ITS-0458_905_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_458-LP3-ITS-0458_905_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.280 s (11.144 ¬µs/read; 5.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,081\n",
      "Reads with adapters:                       287 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,081 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,549,381 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,510,950 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 287 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.6%\n",
      "  C: 21.3%\n",
      "  G: 0.0%\n",
      "  T: 3.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t391.9\t0\t5\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t36\t0.0\t2\t0 0 36\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t13\t0.0\t2\t0 0 13\n",
      "133\t52\t0.0\t2\t0 0 52\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t41\t0.0\t2\t0 0 41\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t75\t0.0\t2\t0 0 75\n",
      "152\t7\t0.0\t2\t0 0 7\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "164\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_459-LP3-ITS-0459_1008_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_459-LP3-ITS-0459_1008_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_459-LP3-ITS-0459_1008_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_459-LP3-ITS-0459_1008_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.395 s (10.875 ¬µs/read; 5.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,305\n",
      "Reads with adapters:                       499 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,927,805 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,863,804 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 499 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.5%\n",
      "  C: 26.5%\n",
      "  G: 0.6%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t567.3\t0\t7\n",
      "4\t1\t141.8\t0\t1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "51\t13\t0.0\t2\t0 0 13\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t9\t0.0\t2\t0 0 9\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t64\t0.0\t2\t0 2 62\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t8\t0.0\t2\t0 0 8\n",
      "133\t72\t0.0\t2\t0 1 71\n",
      "134\t49\t0.0\t2\t0 1 48\n",
      "137\t117\t0.0\t2\t0 2 115\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t66\t0.0\t2\t0 1 65\n",
      "153\t37\t0.0\t2\t0 0 37\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_460-LP3-ITS-0460_56_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_460-LP3-ITS-0460_56_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_460-LP3-ITS-0460_56_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_460-LP3-ITS-0460_56_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.228 s (11.528 ¬µs/read; 5.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,785\n",
      "Reads with adapters:                       354 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,955,285 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,917,232 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 354 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.9%\n",
      "  C: 26.6%\n",
      "  G: 0.6%\n",
      "  T: 4.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t309.1\t0\t67\n",
      "10\t11\t0.0\t1\t1 10\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "118\t48\t0.0\t2\t0 0 48\n",
      "128\t12\t0.0\t2\t0 0 12\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "134\t12\t0.0\t2\t0 0 12\n",
      "137\t58\t0.0\t2\t0 0 58\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t58\t0.0\t2\t0 1 57\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t20\t0.0\t2\t0 0 20\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "164\t15\t0.0\t2\t0 0 15\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_461-LP3-ITS-0461_279_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_461-LP3-ITS-0461_279_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_461-LP3-ITS-0461_279_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_461-LP3-ITS-0461_279_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.390 s (10.511 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,147\n",
      "Reads with adapters:                     1,689 (4.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,147 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,181,247 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,113,749 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1689 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.0%\n",
      "  C: 40.7%\n",
      "  G: 1.4%\n",
      "  T: 18.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t966\t580.4\t0\t966\n",
      "4\t1\t145.1\t0\t1\n",
      "6\t1\t9.1\t0\t1\n",
      "8\t1\t0.6\t0\t1\n",
      "9\t5\t0.1\t0\t4 1\n",
      "10\t237\t0.0\t1\t8 229\n",
      "11\t1\t0.0\t1\t0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t39\t0.0\t2\t0 0 39\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t83\t0.0\t2\t0 2 81\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t9\t0.0\t2\t0 0 9\n",
      "133\t48\t0.0\t2\t0 0 48\n",
      "134\t56\t0.0\t2\t0 1 55\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "145\t178\t0.0\t2\t0 1 177\n",
      "153\t17\t0.0\t2\t0 0 17\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_462-LP3-ITS-0462_163_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_462-LP3-ITS-0462_163_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_462-LP3-ITS-0462_163_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_462-LP3-ITS-0462_163_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.422 s (10.812 ¬µs/read; 5.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,029\n",
      "Reads with adapters:                       534 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,029 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,747,729 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,676,690 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 534 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.3%\n",
      "  C: 16.9%\n",
      "  G: 0.4%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t609.8\t0\t6\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "58\t7\t0.0\t2\t0 1 6\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t95\t0.0\t2\t0 2 93\n",
      "127\t5\t0.0\t2\t0 1 4\n",
      "128\t19\t0.0\t2\t0 1 18\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t64\t0.0\t2\t0 0 64\n",
      "134\t58\t0.0\t2\t0 0 58\n",
      "137\t51\t0.0\t2\t0 0 51\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t121\t0.0\t2\t0 1 120\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t19\t0.0\t2\t0 0 19\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "153\t35\t0.0\t2\t0 0 35\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t11\t0.0\t2\t0 0 11\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_463-LP3-ITS-0463_810_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_463-LP3-ITS-0463_810_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_463-LP3-ITS-0463_810_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_463-LP3-ITS-0463_810_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.033 s (12.094 ¬µs/read; 4.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   2,758\n",
      "Reads with adapters:                        56 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         2,758 (100.0%)\n",
      "\n",
      "Total basepairs processed:       830,158 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        825,468 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 56 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.1%\n",
      "  C: 58.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t43.1\t0\t3\n",
      "8\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "66\t30\t0.0\t2\t0 1 29\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "128\t7\t0.0\t2\t0 0 7\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_464-LP3-ITS-0464_916_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_464-LP3-ITS-0464_916_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_464-LP3-ITS-0464_916_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_464-LP3-ITS-0464_916_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (45.148 ¬µs/read; 1.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     126\n",
      "Reads with adapters:                         6 (4.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           126 (100.0%)\n",
      "\n",
      "Total basepairs processed:        37,926 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         37,407 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2.0\t0\t2\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_465-LP3-ITS-0465_467_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_465-LP3-ITS-0465_467_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_465-LP3-ITS-0465_467_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_465-LP3-ITS-0465_467_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.020 s (14.051 ¬µs/read; 4.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   1,455\n",
      "Reads with adapters:                        57 (3.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         1,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:       437,955 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        434,221 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 57 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.3%\n",
      "  C: 78.9%\n",
      "  G: 1.8%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t22.7\t0\t2\n",
      "10\t3\t0.0\t1\t0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "67\t47\t0.0\t2\t0 1 46\n",
      "118\t3\t0.0\t2\t0 1 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_466-LP3-ITS-0466_83_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_466-LP3-ITS-0466_83_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_466-LP3-ITS-0466_83_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_466-LP3-ITS-0466_83_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.396 s (10.902 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,329\n",
      "Reads with adapters:                       526 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,935,029 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,870,108 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 526 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.5%\n",
      "  C: 16.2%\n",
      "  G: 0.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t567.6\t0\t8\n",
      "10\t2\t0.0\t1\t1 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t52\t0.0\t2\t0 0 52\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t5\t0.0\t2\t0 0 5\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t84\t0.0\t2\t0 3 81\n",
      "128\t15\t0.0\t2\t0 0 15\n",
      "132\t8\t0.0\t2\t0 0 8\n",
      "133\t54\t0.0\t2\t0 0 54\n",
      "134\t115\t0.0\t2\t0 1 114\n",
      "137\t39\t0.0\t2\t0 0 39\n",
      "145\t104\t0.0\t2\t0 0 104\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "164\t13\t0.0\t2\t0 0 13\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_467-LP3-ITS-0467_945_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_467-LP3-ITS-0467_945_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_467-LP3-ITS-0467_945_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_467-LP3-ITS-0467_945_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.638 s (10.803 ¬µs/read; 5.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  59,067\n",
      "Reads with adapters:                     2,773 (4.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        59,067 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,779,167 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,658,279 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2773 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.1%\n",
      "  C: 42.3%\n",
      "  G: 1.0%\n",
      "  T: 17.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1530\t922.9\t0\t1530\n",
      "4\t1\t230.7\t0\t1\n",
      "8\t2\t0.9\t0\t2\n",
      "9\t55\t0.2\t0\t50 5\n",
      "10\t344\t0.1\t1\t17 327\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "58\t22\t0.0\t2\t0 1 21\n",
      "59\t6\t0.0\t2\t0 0 6\n",
      "60\t8\t0.0\t2\t0 1 7\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "117\t6\t0.0\t2\t0 0 6\n",
      "118\t20\t0.0\t2\t0 0 20\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t97\t0.0\t2\t0 2 95\n",
      "134\t231\t0.0\t2\t0 6 225\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t29\t0.0\t2\t0 0 29\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t293\t0.0\t2\t0 4 289\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t43\t0.0\t2\t0 1 42\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 1\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_468-LP3-ITS-0468_951_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_468-LP3-ITS-0468_951_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_468-LP3-ITS-0468_951_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_468-LP3-ITS-0468_951_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.372 s (10.256 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,310\n",
      "Reads with adapters:                       514 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,310 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,929,310 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,862,773 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 514 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.4%\n",
      "  C: 19.6%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t567.3\t0\t12\n",
      "10\t4\t0.0\t1\t0 4\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t13\t0.0\t2\t0 0 13\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t61\t0.0\t2\t0 2 59\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t19\t0.0\t2\t0 0 19\n",
      "133\t51\t0.0\t2\t0 2 49\n",
      "134\t56\t0.0\t2\t0 1 55\n",
      "137\t135\t0.0\t2\t0 1 134\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "145\t51\t0.0\t2\t0 3 48\n",
      "153\t35\t0.0\t2\t0 2 33\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "164\t38\t0.0\t2\t0 0 38\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_469-LP3-ITS-0469_226_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_469-LP3-ITS-0469_226_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_469-LP3-ITS-0469_226_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_469-LP3-ITS-0469_226_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.422 s (10.855 ¬µs/read; 5.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,875\n",
      "Reads with adapters:                       662 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,875 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,701,375 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,614,519 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 662 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.5%\n",
      "  C: 18.1%\n",
      "  G: 0.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t607.4\t0\t15\n",
      "10\t5\t0.0\t1\t0 5\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 1 1\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t79\t0.0\t2\t0 3 76\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t52\t0.0\t2\t0 0 52\n",
      "134\t162\t0.0\t2\t0 0 162\n",
      "137\t99\t0.0\t2\t0 0 99\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t104\t0.0\t2\t0 0 104\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t12\t0.0\t2\t0 0 12\n",
      "155\t22\t0.0\t2\t0 0 22\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t51\t0.0\t2\t0 0 51\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_470-LP3-ITS-0470_177_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_470-LP3-ITS-0470_177_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_470-LP3-ITS-0470_177_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_470-LP3-ITS-0470_177_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.528 s (10.312 ¬µs/read; 5.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,201\n",
      "Reads with adapters:                       732 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,201 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,411,501 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,316,461 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 732 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.4%\n",
      "  C: 23.4%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t800.0\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t12\t0.0\t2\t0 0 12\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "117\t5\t0.0\t2\t0 0 5\n",
      "118\t158\t0.0\t2\t0 1 157\n",
      "128\t7\t0.0\t2\t0 0 7\n",
      "132\t9\t0.0\t2\t0 0 9\n",
      "133\t97\t0.0\t2\t0 1 96\n",
      "134\t122\t0.0\t2\t0 0 122\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "137\t98\t0.0\t2\t0 1 97\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t113\t0.0\t2\t0 0 113\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t36\t0.0\t2\t0 1 35\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t23\t0.0\t2\t0 0 23\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_471-LP3-ITS-0471_450_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_471-LP3-ITS-0471_450_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_471-LP3-ITS-0471_450_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_471-LP3-ITS-0471_450_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.444 s (10.736 ¬µs/read; 5.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,312\n",
      "Reads with adapters:                       490 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,312 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,434,912 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,369,476 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 490 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.5%\n",
      "  C: 22.7%\n",
      "  G: 0.2%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t645.5\t0\t3\n",
      "4\t1\t161.4\t0\t1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "61\t7\t0.0\t2\t0 0 7\n",
      "118\t57\t0.0\t2\t0 2 55\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t79\t0.0\t2\t0 1 78\n",
      "134\t63\t0.0\t2\t0 0 63\n",
      "137\t72\t0.0\t2\t0 0 72\n",
      "142\t13\t0.0\t2\t0 1 12\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t102\t0.0\t2\t0 2 100\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t41\t0.0\t2\t0 0 41\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_472-LP3-ITS-0472_516_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_472-LP3-ITS-0472_516_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_472-LP3-ITS-0472_516_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_472-LP3-ITS-0472_516_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.012 s (16.941 ¬µs/read; 3.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     730\n",
      "Reads with adapters:                         7 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           730 (100.0%)\n",
      "\n",
      "Total basepairs processed:       219,730 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        219,055 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.9%\n",
      "  C: 57.1%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t11.4\t0\t2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_473-LP3-ITS-0473_386_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_473-LP3-ITS-0473_386_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_473-LP3-ITS-0473_386_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_473-LP3-ITS-0473_386_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.015 s (15.014 ¬µs/read; 4.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   1,021\n",
      "Reads with adapters:                        26 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         1,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:       307,321 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        306,230 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 26 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.5%\n",
      "  C: 53.8%\n",
      "  G: 0.0%\n",
      "  T: 7.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t16.0\t0\t6\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "31\t10\t0.0\t2\t0 0 10\n",
      "75\t5\t0.0\t2\t0 0 5\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_474-LP3-ITS-0474_87_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_474-LP3-ITS-0474_87_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_474-LP3-ITS-0474_87_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_474-LP3-ITS-0474_87_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.011 s (18.000 ¬µs/read; 3.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     587\n",
      "Reads with adapters:                        21 (3.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           587 (100.0%)\n",
      "\n",
      "Total basepairs processed:       176,687 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        176,007 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 21 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.1%\n",
      "  C: 57.1%\n",
      "  G: 0.0%\n",
      "  T: 4.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t9.2\t0\t16\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_475-LP3-ITS-0475_727_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_475-LP3-ITS-0475_727_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_475-LP3-ITS-0475_727_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_475-LP3-ITS-0475_727_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.523 s (11.806 ¬µs/read; 5.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,333\n",
      "Reads with adapters:                       683 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,333 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,344,233 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,271,426 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 683 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.7%\n",
      "  C: 51.8%\n",
      "  G: 2.0%\n",
      "  T: 7.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t692.7\t0\t34\n",
      "4\t5\t173.2\t0\t5\n",
      "5\t3\t43.3\t0\t3\n",
      "6\t2\t10.8\t0\t2\n",
      "10\t3\t0.0\t1\t0 3\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "45\t15\t0.0\t2\t0 0 15\n",
      "46\t6\t0.0\t2\t0 0 6\n",
      "47\t7\t0.0\t2\t0 0 7\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t4\t0.0\t2\t0 0 4\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t9\t0.0\t2\t0 0 9\n",
      "58\t11\t0.0\t2\t0 0 11\n",
      "59\t7\t0.0\t2\t0 0 7\n",
      "60\t10\t0.0\t2\t0 1 9\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "63\t7\t0.0\t2\t0 0 7\n",
      "64\t7\t0.0\t2\t0 0 7\n",
      "65\t5\t0.0\t2\t0 0 5\n",
      "66\t7\t0.0\t2\t0 1 6\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t16\t0.0\t2\t0 0 16\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "75\t11\t0.0\t2\t0 0 11\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "79\t6\t0.0\t2\t0 0 6\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "83\t7\t0.0\t2\t0 0 7\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t6\t0.0\t2\t0 0 6\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t12\t0.0\t2\t0 0 12\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t6\t0.0\t2\t0 0 6\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t8\t0.0\t2\t0 0 8\n",
      "116\t6\t0.0\t2\t0 0 6\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t8\t0.0\t2\t0 1 7\n",
      "119\t3\t0.0\t2\t0 1 2\n",
      "120\t21\t0.0\t2\t0 0 21\n",
      "124\t36\t0.0\t2\t0 0 36\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t5\t0.0\t2\t0 1 4\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "135\t10\t0.0\t2\t0 0 10\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "138\t21\t0.0\t2\t0 0 21\n",
      "139\t12\t0.0\t2\t0 1 11\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t39\t0.0\t2\t0 0 39\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t5\t0.0\t2\t0 0 5\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t8\t0.0\t2\t0 0 8\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t38\t0.0\t2\t0 1 37\n",
      "151\t13\t0.0\t2\t0 0 13\n",
      "152\t24\t0.0\t2\t0 0 24\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t51\t0.0\t2\t0 0 51\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "180\t2\t0.0\t2\t0 0 2\n",
      "182\t2\t0.0\t2\t0 0 2\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_476-LP3-ITS-0476_179_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_476-LP3-ITS-0476_179_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_476-LP3-ITS-0476_179_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_476-LP3-ITS-0476_179_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.311 s (12.468 ¬µs/read; 4.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,965\n",
      "Reads with adapters:                       406 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,965 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,514,465 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,473,865 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 406 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.2%\n",
      "  C: 51.7%\n",
      "  G: 2.0%\n",
      "  T: 8.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t390.1\t0\t34\n",
      "4\t11\t97.5\t0\t11\n",
      "5\t5\t24.4\t0\t5\n",
      "6\t3\t6.1\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "30\t2\t0.0\t2\t0 1 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t4\t0.0\t2\t0 0 4\n",
      "37\t5\t0.0\t2\t0 0 5\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "44\t4\t0.0\t2\t0 0 4\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t4\t0.0\t2\t0 0 4\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t7\t0.0\t2\t0 0 7\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t6\t0.0\t2\t0 0 6\n",
      "66\t7\t0.0\t2\t0 0 7\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "83\t4\t0.0\t2\t0 0 4\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "94\t8\t0.0\t2\t0 0 8\n",
      "96\t12\t0.0\t2\t0 0 12\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t14\t0.0\t2\t0 0 14\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "109\t11\t0.0\t2\t0 0 11\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t10\t0.0\t2\t0 0 10\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t9\t0.0\t2\t0 0 9\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t8\t0.0\t2\t0 0 8\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t14\t0.0\t2\t0 0 14\n",
      "138\t11\t0.0\t2\t0 0 11\n",
      "139\t22\t0.0\t2\t0 0 22\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "150\t20\t0.0\t2\t0 0 20\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t9\t0.0\t2\t0 0 9\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t20\t0.0\t2\t0 0 20\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "182\t2\t0.0\t2\t0 0 2\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "269\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_477-LP3-ITS-0477_122_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_477-LP3-ITS-0477_122_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_477-LP3-ITS-0477_122_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_477-LP3-ITS-0477_122_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.129 s (15.843 ¬µs/read; 3.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   8,120\n",
      "Reads with adapters:                       141 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         8,120 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,444,120 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,431,184 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 141 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.4%\n",
      "  C: 59.6%\n",
      "  G: 2.1%\n",
      "  T: 9.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t126.9\t0\t18\n",
      "4\t4\t31.7\t0\t4\n",
      "5\t1\t7.9\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "63\t10\t0.0\t2\t0 0 10\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "74\t4\t0.0\t2\t0 0 4\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "132\t2\t0.0\t2\t0 1 1\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_478-LP3-ITS-0478_956_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_478-LP3-ITS-0478_956_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_478-LP3-ITS-0478_956_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_478-LP3-ITS-0478_956_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (32.211 ¬µs/read; 1.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     270\n",
      "Reads with adapters:                        10 (3.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           270 (100.0%)\n",
      "\n",
      "Total basepairs processed:        81,270 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         80,920 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.0%\n",
      "  C: 70.0%\n",
      "  G: 0.0%\n",
      "  T: 10.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t4.2\t0\t6\n",
      "10\t1\t0.0\t1\t0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_479-LP3-ITS-0479_891_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_479-LP3-ITS-0479_891_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_479-LP3-ITS-0479_891_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_479-LP3-ITS-0479_891_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.332 s (11.855 ¬µs/read; 5.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,043\n",
      "Reads with adapters:                       470 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,043 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,440,943 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,402,798 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 470 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.3%\n",
      "  C: 57.9%\n",
      "  G: 0.6%\n",
      "  T: 7.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t438.2\t0\t72\n",
      "4\t8\t109.5\t0\t8\n",
      "5\t4\t27.4\t0\t4\n",
      "6\t39\t6.8\t0\t39\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "39\t3\t0.0\t2\t0 0 3\n",
      "40\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t19\t0.0\t2\t0 1 18\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t17\t0.0\t2\t0 1 16\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t13\t0.0\t2\t0 0 13\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t1\t0.0\t2\t0 1\n",
      "63\t9\t0.0\t2\t0 0 9\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t3\t0.0\t2\t0 0 3\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t17\t0.0\t2\t0 0 17\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 1 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t4\t0.0\t2\t0 0 4\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "123\t9\t0.0\t2\t0 0 9\n",
      "124\t14\t0.0\t2\t0 0 14\n",
      "125\t7\t0.0\t2\t0 0 7\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t4\t0.0\t2\t0 1 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "135\t6\t0.0\t2\t0 0 6\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t4\t0.0\t2\t0 1 3\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "141\t5\t0.0\t2\t0 0 5\n",
      "142\t25\t0.0\t2\t0 0 25\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t6\t0.0\t2\t0 0 6\n",
      "150\t21\t0.0\t2\t0 0 21\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t12\t0.0\t2\t0 0 12\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_480-LP3-ITS-0480_629_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_480-LP3-ITS-0480_629_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_480-LP3-ITS-0480_629_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_480-LP3-ITS-0480_629_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.354 s (11.044 ¬µs/read; 5.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,018\n",
      "Reads with adapters:                       399 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,018 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,637,418 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,604,220 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 399 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.1%\n",
      "  C: 50.9%\n",
      "  G: 0.3%\n",
      "  T: 11.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t500.3\t0\t7\n",
      "4\t11\t125.1\t0\t11\n",
      "5\t5\t31.3\t0\t5\n",
      "6\t88\t7.8\t0\t88\n",
      "7\t1\t2.0\t0\t1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "26\t6\t0.0\t2\t0 0 6\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t8\t0.0\t2\t0 0 8\n",
      "56\t8\t0.0\t2\t0 0 8\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t7\t0.0\t2\t0 0 7\n",
      "66\t4\t0.0\t2\t0 0 4\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t36\t0.0\t2\t0 1 35\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t5\t0.0\t2\t0 0 5\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t15\t0.0\t2\t0 0 15\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t50\t0.0\t2\t0 0 50\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_481-LP3-ITS-0481_765_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_481-LP3-ITS-0481_765_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_481-LP3-ITS-0481_765_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_481-LP3-ITS-0481_765_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.263 s (10.548 ¬µs/read; 5.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,927\n",
      "Reads with adapters:                     2,533 (10.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,927 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,503,027 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,463,572 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2533 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.3%\n",
      "  C: 10.1%\n",
      "  G: 0.0%\n",
      "  T: 3.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t250\t389.5\t0\t250\n",
      "4\t1983\t97.4\t0\t1983\n",
      "5\t10\t24.3\t0\t10\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t9\t0.1\t0\t9\n",
      "10\t52\t0.0\t1\t4 48\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "118\t66\t0.0\t2\t0 0 66\n",
      "123\t13\t0.0\t2\t0 0 13\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t25\t0.0\t2\t0 0 25\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t16\t0.0\t2\t0 0 16\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t16\t0.0\t2\t0 0 16\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "160\t13\t0.0\t2\t0 0 13\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_482-LP3-ITS-0482_647_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_482-LP3-ITS-0482_647_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_482-LP3-ITS-0482_647_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_482-LP3-ITS-0482_647_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.247 s (11.583 ¬µs/read; 5.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,294\n",
      "Reads with adapters:                     2,610 (12.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,294 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,409,494 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,382,441 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2610 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.9%\n",
      "  C: 46.6%\n",
      "  G: 1.8%\n",
      "  T: 15.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1679\t332.7\t0\t1679\n",
      "4\t280\t83.2\t0\t280\n",
      "5\t1\t20.8\t0\t1\n",
      "7\t3\t1.3\t0\t3\n",
      "8\t20\t0.3\t0\t20\n",
      "9\t55\t0.1\t0\t51 4\n",
      "10\t451\t0.0\t1\t19 432\n",
      "11\t1\t0.0\t1\t0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "118\t61\t0.0\t2\t0 0 61\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t22\t0.0\t2\t0 1 21\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "197\t2\t0.0\t2\t0 0 2\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_483-LP3-ITS-0483_661_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_483-LP3-ITS-0483_661_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_483-LP3-ITS-0483_661_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_483-LP3-ITS-0483_661_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.324 s (11.449 ¬µs/read; 5.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,335\n",
      "Reads with adapters:                     5,855 (20.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,335 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,528,835 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,478,681 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5855 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.2%\n",
      "  C: 10.0%\n",
      "  G: 0.2%\n",
      "  T: 2.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t620\t442.7\t0\t620\n",
      "4\t4801\t110.7\t0\t4801\n",
      "5\t22\t27.7\t0\t22\n",
      "8\t4\t0.4\t0\t4\n",
      "9\t14\t0.1\t0\t14\n",
      "10\t172\t0.0\t1\t14 158\n",
      "12\t2\t0.0\t1\t0 2\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "46\t5\t0.0\t2\t0 0 5\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t4\t0.0\t2\t0 0 4\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t6\t0.0\t2\t0 0 6\n",
      "52\t5\t0.0\t2\t0 0 5\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t7\t0.0\t2\t0 0 7\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t36\t0.0\t2\t0 0 36\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t12\t0.0\t2\t0 0 12\n",
      "133\t15\t0.0\t2\t0 0 15\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 1 4\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t19\t0.0\t2\t0 0 19\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "148\t5\t0.0\t2\t0 0 5\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t13\t0.0\t2\t0 0 13\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "197\t12\t0.0\t2\t0 1 11\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_484-LP3-ITS-0484_486_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_484-LP3-ITS-0484_486_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_484-LP3-ITS-0484_486_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_484-LP3-ITS-0484_486_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.273 s (11.266 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,195\n",
      "Reads with adapters:                     3,956 (16.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,195 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,282,695 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,253,865 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3956 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.6%\n",
      "  C: 44.0%\n",
      "  G: 0.8%\n",
      "  T: 15.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2387\t378.0\t0\t2387\n",
      "4\t921\t94.5\t0\t921\n",
      "5\t7\t23.6\t0\t7\n",
      "6\t23\t5.9\t0\t23\n",
      "7\t2\t1.5\t0\t2\n",
      "8\t8\t0.4\t0\t8\n",
      "9\t65\t0.1\t0\t61 4\n",
      "10\t428\t0.0\t1\t21 407\n",
      "11\t4\t0.0\t1\t0 4\n",
      "32\t28\t0.0\t2\t0 1 27\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "118\t39\t0.0\t2\t0 0 39\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "232\t7\t0.0\t2\t0 0 7\n",
      "234\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_485-LP3-ITS-0485_480_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_485-LP3-ITS-0485_480_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_485-LP3-ITS-0485_480_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_485-LP3-ITS-0485_480_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.305 s (11.180 ¬µs/read; 5.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,272\n",
      "Reads with adapters:                     2,990 (11.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,272 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,208,872 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,157,538 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2990 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.1%\n",
      "  C: 19.7%\n",
      "  G: 0.6%\n",
      "  T: 8.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t744\t426.1\t0\t744\n",
      "4\t1728\t106.5\t0\t1728\n",
      "5\t11\t26.6\t0\t11\n",
      "7\t1\t1.7\t0\t1\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t29\t0.1\t0\t25 4\n",
      "10\t175\t0.0\t1\t11 164\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t18\t0.0\t2\t0 0 18\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t93\t0.0\t2\t0 3 90\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t42\t0.0\t2\t0 1 41\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "142\t19\t0.0\t2\t0 0 19\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "232\t11\t0.0\t2\t0 0 11\n",
      "234\t9\t0.0\t2\t0 0 9\n",
      "235\t9\t0.0\t2\t0 0 9\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_486-LP3-ITS-0486_1069_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_486-LP3-ITS-0486_1069_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_486-LP3-ITS-0486_1069_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_486-LP3-ITS-0486_1069_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.311 s (10.497 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,588\n",
      "Reads with adapters:                     2,246 (7.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,588 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,905,988 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,851,390 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2246 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.9%\n",
      "  C: 13.8%\n",
      "  G: 0.5%\n",
      "  T: 3.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t318\t462.3\t0\t318\n",
      "4\t1490\t115.6\t0\t1490\n",
      "5\t8\t28.9\t0\t8\n",
      "6\t2\t7.2\t0\t2\n",
      "9\t15\t0.1\t0\t15\n",
      "10\t82\t0.0\t1\t1 81\n",
      "32\t3\t0.0\t2\t0 1 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "79\t6\t0.0\t2\t0 0 6\n",
      "84\t7\t0.0\t2\t0 0 7\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t119\t0.0\t2\t0 1 118\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t38\t0.0\t2\t0 0 38\n",
      "134\t13\t0.0\t2\t0 0 13\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t16\t0.0\t2\t0 1 15\n",
      "197\t4\t0.0\t2\t0 0 4\n",
      "232\t8\t0.0\t2\t0 1 7\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t36\t0.0\t2\t0 0 36\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_487-LP3-ITS-0487_276_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_487-LP3-ITS-0487_276_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_487-LP3-ITS-0487_276_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_487-LP3-ITS-0487_276_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.192 s (11.517 ¬µs/read; 5.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,693\n",
      "Reads with adapters:                     4,650 (27.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,693 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,024,593 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,994,786 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4650 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 95.5%\n",
      "  C: 3.6%\n",
      "  G: 0.1%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t149\t260.8\t0\t149\n",
      "4\t4351\t65.2\t0\t4351\n",
      "5\t35\t16.3\t0\t35\n",
      "9\t3\t0.1\t0\t3\n",
      "10\t39\t0.0\t1\t3 36\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t17\t0.0\t2\t0 0 17\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "148\t5\t0.0\t2\t0 0 5\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "197\t7\t0.0\t2\t0 0 7\n",
      "231\t7\t0.0\t2\t0 0 7\n",
      "234\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_488-LP3-ITS-0488_267_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_488-LP3-ITS-0488_267_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_488-LP3-ITS-0488_267_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_488-LP3-ITS-0488_267_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.224 s (10.969 ¬µs/read; 5.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,437\n",
      "Reads with adapters:                     1,987 (9.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,437 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,151,537 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,121,706 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1987 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.4%\n",
      "  C: 23.3%\n",
      "  G: 1.0%\n",
      "  T: 7.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t573\t319.3\t0\t573\n",
      "4\t1085\t79.8\t0\t1085\n",
      "5\t4\t20.0\t0\t4\n",
      "7\t1\t1.2\t0\t1\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t14\t0.1\t0\t13 1\n",
      "10\t133\t0.0\t1\t8 125\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t15\t0.0\t2\t0 0 15\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "118\t48\t0.0\t2\t0 1 47\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t39\t0.0\t2\t0 1 38\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "197\t3\t0.0\t2\t0 0 3\n",
      "232\t3\t0.0\t2\t0 1 2\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_489-LP3-ITS-0489_752_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_489-LP3-ITS-0489_752_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_489-LP3-ITS-0489_752_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_489-LP3-ITS-0489_752_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.144 s (11.868 ¬µs/read; 5.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,139\n",
      "Reads with adapters:                       609 (5.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,139 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,653,839 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,639,278 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 609 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 50.1%\n",
      "  G: 1.8%\n",
      "  T: 17.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t390\t189.7\t0\t390\n",
      "4\t4\t47.4\t0\t4\n",
      "6\t4\t3.0\t0\t4\n",
      "9\t8\t0.0\t0\t8\n",
      "10\t108\t0.0\t1\t7 101\n",
      "46\t8\t0.0\t2\t0 0 8\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "118\t30\t0.0\t2\t0 1 29\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_490-LP3-ITS-0490_837_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_490-LP3-ITS-0490_837_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_490-LP3-ITS-0490_837_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_490-LP3-ITS-0490_837_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.180 s (10.271 ¬µs/read; 5.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,573\n",
      "Reads with adapters:                     1,124 (6.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,573 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,289,473 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,264,960 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1124 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.3%\n",
      "  C: 18.9%\n",
      "  G: 0.4%\n",
      "  T: 7.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t291\t274.6\t0\t291\n",
      "4\t604\t68.6\t0\t604\n",
      "5\t2\t17.2\t0\t2\n",
      "9\t3\t0.1\t0\t2 1\n",
      "10\t61\t0.0\t1\t5 56\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t113\t0.0\t2\t0 6 107\n",
      "133\t8\t0.0\t2\t0 0 8\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "148\t8\t0.0\t2\t0 0 8\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t21\t0.0\t2\t0 0 21\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_491-LP3-ITS-0491_219_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_491-LP3-ITS-0491_219_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_491-LP3-ITS-0491_219_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_491-LP3-ITS-0491_219_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.244 s (10.900 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,379\n",
      "Reads with adapters:                     2,912 (13.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,379 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,736,079 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,707,815 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2912 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.5%\n",
      "  C: 48.3%\n",
      "  G: 2.5%\n",
      "  T: 17.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2028\t349.7\t0\t2028\n",
      "4\t10\t87.4\t0\t10\n",
      "9\t78\t0.1\t0\t75 3\n",
      "10\t680\t0.0\t1\t51 629\n",
      "12\t1\t0.0\t1\t0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t70\t0.0\t2\t0 3 67\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_492-LP3-ITS-0492_746_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_492-LP3-ITS-0492_746_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_492-LP3-ITS-0492_746_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_492-LP3-ITS-0492_746_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.231 s (10.506 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,010\n",
      "Reads with adapters:                     5,903 (26.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,010 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,625,010 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,591,309 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5903 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.8%\n",
      "  C: 44.9%\n",
      "  G: 0.8%\n",
      "  T: 15.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3622\t343.9\t0\t3622\n",
      "4\t1506\t86.0\t0\t1506\n",
      "5\t15\t21.5\t0\t15\n",
      "9\t43\t0.1\t0\t39 4\n",
      "10\t640\t0.0\t1\t41 599\n",
      "11\t2\t0.0\t1\t0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t27\t0.0\t2\t0 0 27\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "197\t3\t0.0\t2\t0 0 3\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_493-LP3-ITS-0493_1070_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_493-LP3-ITS-0493_1070_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_493-LP3-ITS-0493_1070_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_493-LP3-ITS-0493_1070_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.271 s (10.381 ¬µs/read; 5.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,109\n",
      "Reads with adapters:                     1,065 (4.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,109 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,858,809 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,807,010 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1065 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.3%\n",
      "  C: 43.8%\n",
      "  G: 1.4%\n",
      "  T: 15.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t531\t408.0\t0\t531\n",
      "5\t4\t25.5\t0\t4\n",
      "8\t10\t0.4\t0\t10\n",
      "9\t21\t0.1\t0\t17 4\n",
      "10\t153\t0.0\t1\t9 144\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "76\t6\t0.0\t2\t0 0 6\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t108\t0.0\t2\t0 2 106\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t64\t0.0\t2\t0 0 64\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t25\t0.0\t2\t0 0 25\n",
      "155\t34\t0.0\t2\t0 0 34\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "234\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_494-LP3-ITS-0494_352_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_494-LP3-ITS-0494_352_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_494-LP3-ITS-0494_352_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_494-LP3-ITS-0494_352_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.400 s (10.646 ¬µs/read; 5.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,588\n",
      "Reads with adapters:                       713 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,588 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,313,988 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,257,607 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 713 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.4%\n",
      "  C: 27.9%\n",
      "  G: 0.3%\n",
      "  T: 3.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t587.3\t0\t109\n",
      "4\t173\t146.8\t0\t173\n",
      "5\t2\t36.7\t0\t2\n",
      "6\t2\t9.2\t0\t2\n",
      "9\t6\t0.1\t0\t5 1\n",
      "10\t24\t0.0\t1\t2 22\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t7\t0.0\t2\t0 0 7\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t128\t0.0\t2\t1 2 125\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t63\t0.0\t2\t0 1 62\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t53\t0.0\t2\t0 0 53\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t12\t0.0\t2\t0 0 12\n",
      "160\t39\t0.0\t2\t0 0 39\n",
      "162\t6\t0.0\t2\t0 0 6\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t19\t0.0\t2\t0 0 19\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_495-LP3-ITS-0495_502_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_495-LP3-ITS-0495_502_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_495-LP3-ITS-0495_502_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_495-LP3-ITS-0495_502_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.241 s (11.224 ¬µs/read; 5.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,482\n",
      "Reads with adapters:                     1,877 (8.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,482 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,466,082 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,442,067 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1877 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.4%\n",
      "  C: 57.2%\n",
      "  G: 1.6%\n",
      "  T: 16.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1415\t335.7\t0\t1415\n",
      "4\t5\t83.9\t0\t5\n",
      "6\t2\t5.2\t0\t2\n",
      "7\t2\t1.3\t0\t2\n",
      "8\t36\t0.3\t0\t36\n",
      "9\t41\t0.1\t0\t39 2\n",
      "10\t252\t0.0\t1\t18 234\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "76\t27\t0.0\t2\t0 1 26\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t33\t0.0\t2\t0 2 31\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t8\t0.0\t2\t0 1 7\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t7\t0.0\t2\t0 1 6\n",
      "232\t6\t0.0\t2\t0 0 6\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_496-LP3-ITS-0496_929_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_496-LP3-ITS-0496_929_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_496-LP3-ITS-0496_929_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_496-LP3-ITS-0496_929_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.167 s (11.011 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,186\n",
      "Reads with adapters:                     1,248 (8.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,570,986 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,546,125 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1248 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.8%\n",
      "  C: 42.8%\n",
      "  G: 1.6%\n",
      "  T: 13.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t686\t237.3\t0\t686\n",
      "4\t186\t59.3\t0\t186\n",
      "5\t2\t14.8\t0\t2\n",
      "9\t17\t0.1\t0\t15 2\n",
      "10\t210\t0.0\t1\t9 201\n",
      "11\t1\t0.0\t1\t0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t42\t0.0\t2\t0 0 42\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t17\t0.0\t2\t0 0 17\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t17\t0.0\t2\t0 1 16\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "234\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_497-LP3-ITS-0497_912_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_497-LP3-ITS-0497_912_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_497-LP3-ITS-0497_912_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_497-LP3-ITS-0497_912_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.288 s (10.518 ¬µs/read; 5.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,428\n",
      "Reads with adapters:                     1,078 (3.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,428 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,255,828 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,215,844 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1078 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.0%\n",
      "  C: 43.8%\n",
      "  G: 2.1%\n",
      "  T: 19.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t632\t428.6\t0\t632\n",
      "4\t1\t107.1\t0\t1\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t32\t0.1\t0\t31 1\n",
      "10\t154\t0.0\t1\t7 147\n",
      "11\t1\t0.0\t1\t1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t8\t0.0\t2\t0 0 8\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t86\t0.0\t2\t0 1 85\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "127\t6\t0.0\t2\t0 0 6\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t24\t0.0\t2\t0 0 24\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "232\t5\t0.0\t2\t0 0 5\n",
      "234\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_498-LP3-ITS-0498_141_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_498-LP3-ITS-0498_141_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_498-LP3-ITS-0498_141_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_498-LP3-ITS-0498_141_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.176 s (10.692 ¬µs/read; 5.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,461\n",
      "Reads with adapters:                     2,969 (18.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,461 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,954,761 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,932,529 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2969 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.7%\n",
      "  C: 11.2%\n",
      "  G: 0.3%\n",
      "  T: 3.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t401\t257.2\t0\t401\n",
      "4\t2327\t64.3\t0\t2327\n",
      "5\t15\t16.1\t0\t15\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t48\t0.1\t0\t46 2\n",
      "10\t101\t0.0\t1\t4 97\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t30\t0.0\t2\t0 0 30\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "133\t10\t0.0\t2\t0 0 10\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "197\t5\t0.0\t2\t0 0 5\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_499-LP3-ITS-0499_418_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_499-LP3-ITS-0499_418_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_499-LP3-ITS-0499_418_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_499-LP3-ITS-0499_418_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.312 s (11.442 ¬µs/read; 5.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,285\n",
      "Reads with adapters:                     6,452 (23.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,212,785 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,177,651 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6452 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.7%\n",
      "  C: 56.1%\n",
      "  G: 1.8%\n",
      "  T: 16.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4841\t426.3\t0\t4841\n",
      "4\t41\t106.6\t0\t41\n",
      "9\t145\t0.1\t0\t135 10\n",
      "10\t1381\t0.0\t1\t85 1296\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t32\t0.0\t2\t0 1 31\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_500-LP3-ITS-0500_416_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_500-LP3-ITS-0500_416_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_500-LP3-ITS-0500_416_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_500-LP3-ITS-0500_416_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.279 s (10.054 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,776\n",
      "Reads with adapters:                       901 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,360,576 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,314,907 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 901 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.4%\n",
      "  C: 43.2%\n",
      "  G: 1.1%\n",
      "  T: 14.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t483\t434.0\t0\t483\n",
      "4\t3\t108.5\t0\t3\n",
      "8\t5\t0.4\t0\t5\n",
      "9\t20\t0.1\t0\t19 1\n",
      "10\t128\t0.0\t1\t6 122\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "76\t5\t0.0\t2\t0 0 5\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t74\t0.0\t2\t0 1 73\n",
      "122\t8\t0.0\t2\t0 0 8\n",
      "133\t35\t0.0\t2\t0 1 34\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "160\t16\t0.0\t2\t0 0 16\n",
      "231\t2\t0.0\t2\t0 0 2\n",
      "232\t16\t0.0\t2\t0 0 16\n",
      "234\t66\t0.0\t2\t0 0 66\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_501-LP3-ITS-0501_508_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_501-LP3-ITS-0501_508_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_501-LP3-ITS-0501_508_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_501-LP3-ITS-0501_508_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.281 s (11.094 ¬µs/read; 5.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,287\n",
      "Reads with adapters:                     4,406 (17.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,287 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,611,387 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,570,548 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4406 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.5%\n",
      "  C: 17.1%\n",
      "  G: 0.4%\n",
      "  T: 7.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1043\t395.1\t0\t1043\n",
      "4\t2911\t98.8\t0\t2911\n",
      "5\t12\t24.7\t0\t12\n",
      "7\t2\t1.5\t0\t2\n",
      "8\t4\t0.4\t0\t4\n",
      "9\t24\t0.1\t0\t22 2\n",
      "10\t235\t0.0\t1\t22 213\n",
      "11\t2\t0.0\t1\t0 2\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t84\t0.0\t2\t0 2 82\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "234\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_502-LP3-ITS-0502_147_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_502-LP3-ITS-0502_147_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_502-LP3-ITS-0502_147_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_502-LP3-ITS-0502_147_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.218 s (10.611 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,559\n",
      "Reads with adapters:                     2,409 (11.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,559 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,188,259 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,162,163 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2409 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 86.0%\n",
      "  C: 11.3%\n",
      "  G: 0.2%\n",
      "  T: 2.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t280\t321.2\t0\t280\n",
      "4\t1881\t80.3\t0\t1881\n",
      "5\t7\t20.1\t0\t7\n",
      "6\t7\t5.0\t0\t7\n",
      "9\t48\t0.1\t0\t42 6\n",
      "10\t57\t0.0\t1\t3 54\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t54\t0.0\t2\t0 0 54\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t12\t0.0\t2\t0 0 12\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "234\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_503-LP3-ITS-0503_52_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_503-LP3-ITS-0503_52_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_503-LP3-ITS-0503_52_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_503-LP3-ITS-0503_52_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.331 s (10.698 ¬µs/read; 5.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,974\n",
      "Reads with adapters:                     3,099 (10.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,974 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,323,174 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,286,959 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3099 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.4%\n",
      "  C: 15.5%\n",
      "  G: 0.3%\n",
      "  T: 3.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t548\t484.0\t0\t548\n",
      "4\t2250\t121.0\t0\t2250\n",
      "5\t16\t30.2\t0\t16\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t10\t0.1\t0\t10\n",
      "10\t97\t0.0\t1\t5 92\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t60\t0.0\t2\t0 1 59\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "234\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_504-LP3-ITS-0504_268_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_504-LP3-ITS-0504_268_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_504-LP3-ITS-0504_268_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_504-LP3-ITS-0504_268_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.245 s (10.692 ¬µs/read; 5.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,924\n",
      "Reads with adapters:                     2,957 (12.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,924 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,900,124 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,856,867 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2957 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.3%\n",
      "  C: 53.7%\n",
      "  G: 2.1%\n",
      "  T: 17.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2185\t358.2\t0\t2185\n",
      "4\t5\t89.5\t0\t5\n",
      "8\t16\t0.3\t0\t16\n",
      "9\t38\t0.1\t0\t37 1\n",
      "10\t506\t0.0\t1\t26 480\n",
      "11\t1\t0.0\t1\t0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "76\t69\t0.0\t2\t0 3 66\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "118\t28\t0.0\t2\t0 0 28\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "232\t19\t0.0\t2\t0 0 19\n",
      "234\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_505-LP3-ITS-0505_803_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_505-LP3-ITS-0505_803_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_505-LP3-ITS-0505_803_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_505-LP3-ITS-0505_803_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.243 s (10.826 ¬µs/read; 5.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,401\n",
      "Reads with adapters:                     4,809 (21.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,401 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,742,701 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,709,170 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4809 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.9%\n",
      "  C: 57.8%\n",
      "  G: 1.8%\n",
      "  T: 18.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3756\t350.0\t0\t3756\n",
      "4\t75\t87.5\t0\t75\n",
      "9\t55\t0.1\t0\t53 2\n",
      "10\t831\t0.0\t1\t38 793\n",
      "11\t2\t0.0\t1\t0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t27\t0.0\t2\t0 0 27\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "234\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_506-LP3-ITS-0506_468_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_506-LP3-ITS-0506_468_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_506-LP3-ITS-0506_468_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_506-LP3-ITS-0506_468_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.225 s (10.409 ¬µs/read; 5.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,600\n",
      "Reads with adapters:                     3,078 (14.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,600 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,501,600 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,469,255 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3078 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 94.6%\n",
      "  C: 4.9%\n",
      "  G: 0.1%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t337.5\t0\t85\n",
      "4\t2784\t84.4\t0\t2784\n",
      "5\t14\t21.1\t0\t14\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t3\t0.1\t0\t2 1\n",
      "10\t21\t0.0\t1\t0 21\n",
      "51\t18\t0.0\t2\t0 0 18\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t80\t0.0\t2\t0 2 78\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "197\t6\t0.0\t2\t0 0 6\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_507-LP3-ITS-0507_204_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_507-LP3-ITS-0507_204_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_507-LP3-ITS-0507_204_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_507-LP3-ITS-0507_204_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.307 s (10.355 ¬µs/read; 5.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,633\n",
      "Reads with adapters:                     5,191 (17.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,633 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,919,533 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,869,124 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5191 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.9%\n",
      "  C: 5.0%\n",
      "  G: 0.1%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t153\t463.0\t0\t153\n",
      "4\t4743\t115.8\t0\t4743\n",
      "5\t17\t28.9\t0\t17\n",
      "6\t6\t7.2\t0\t6\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t5\t0.1\t0\t5\n",
      "10\t36\t0.0\t1\t5 31\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "118\t62\t0.0\t2\t0 1 61\n",
      "123\t9\t0.0\t2\t0 1 8\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t45\t0.0\t2\t0 0 45\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t19\t0.0\t2\t0 0 19\n",
      "143\t14\t0.0\t2\t0 0 14\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "197\t8\t0.0\t2\t0 0 8\n",
      "231\t2\t0.0\t2\t0 0 2\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_508-LP3-ITS-0508_576_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_508-LP3-ITS-0508_576_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_508-LP3-ITS-0508_576_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_508-LP3-ITS-0508_576_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.140 s (10.520 ¬µs/read; 5.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,294\n",
      "Reads with adapters:                       192 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,294 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,001,494 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,985,307 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 192 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.1%\n",
      "  C: 37.5%\n",
      "  G: 0.0%\n",
      "  T: 10.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t207.7\t0\t49\n",
      "6\t1\t3.2\t0\t1\n",
      "8\t1\t0.2\t0\t1\n",
      "10\t16\t0.0\t1\t0 16\n",
      "12\t1\t0.0\t1\t0 1\n",
      "29\t1\t0.0\t2\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t42\t0.0\t2\t0 0 42\n",
      "133\t25\t0.0\t2\t0 1 24\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t9\t0.0\t2\t0 0 9\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_509-LP3-ITS-0509_537_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_509-LP3-ITS-0509_537_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_509-LP3-ITS-0509_537_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_509-LP3-ITS-0509_537_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.148 s (10.649 ¬µs/read; 5.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,874\n",
      "Reads with adapters:                     1,045 (7.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,874 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,176,074 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,154,063 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1045 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.8%\n",
      "  C: 47.8%\n",
      "  G: 1.7%\n",
      "  T: 17.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t672\t216.8\t0\t672\n",
      "4\t20\t54.2\t0\t20\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t23\t0.1\t0\t21 2\n",
      "10\t186\t0.0\t1\t14 172\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "118\t71\t0.0\t2\t0 3 68\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "160\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_510-LP3-ITS-0510_431_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_510-LP3-ITS-0510_431_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_510-LP3-ITS-0510_431_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_510-LP3-ITS-0510_431_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.308 s (11.001 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,043\n",
      "Reads with adapters:                     8,659 (30.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,043 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,440,943 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,383,424 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8659 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 96.1%\n",
      "  C: 3.1%\n",
      "  G: 0.1%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t205\t438.2\t0\t205\n",
      "4\t8190\t109.5\t0\t8190\n",
      "5\t38\t27.4\t0\t38\n",
      "6\t1\t6.8\t0\t1\n",
      "7\t1\t1.7\t0\t1\n",
      "9\t2\t0.1\t0\t1 1\n",
      "10\t42\t0.0\t1\t2 40\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t94\t0.0\t2\t0 3 91\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t2\t0.0\t2\t0 1 1\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 1 1\n",
      "160\t9\t0.0\t2\t0 1 8\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "197\t2\t0.0\t2\t0 0 2\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_511-LP3-ITS-0511_556_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_511-LP3-ITS-0511_556_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_511-LP3-ITS-0511_556_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_511-LP3-ITS-0511_556_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.383 s (10.662 ¬µs/read; 5.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,921\n",
      "Reads with adapters:                     7,302 (20.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,921 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,812,221 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,756,961 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 7302 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.3%\n",
      "  C: 8.7%\n",
      "  G: 0.1%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t662\t561.3\t0\t662\n",
      "4\t6249\t140.3\t0\t6249\n",
      "5\t49\t35.1\t0\t49\n",
      "8\t4\t0.5\t0\t4\n",
      "9\t15\t0.1\t0\t15\n",
      "10\t112\t0.0\t1\t3 109\n",
      "11\t3\t0.0\t1\t0 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t64\t0.0\t2\t0 2 62\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t45\t0.0\t2\t0 0 45\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t13\t0.0\t2\t0 0 13\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "197\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_512-LP3-ITS-0512_202_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_512-LP3-ITS-0512_202_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_512-LP3-ITS-0512_202_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_512-LP3-ITS-0512_202_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.246 s (11.157 ¬µs/read; 5.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,024\n",
      "Reads with adapters:                     2,720 (12.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,024 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,629,224 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,589,871 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2720 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.0%\n",
      "  C: 8.0%\n",
      "  G: 0.1%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t163\t344.1\t0\t163\n",
      "4\t2292\t86.0\t0\t2292\n",
      "5\t10\t21.5\t0\t10\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t34\t0.0\t1\t4 30\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t90\t0.0\t2\t0 2 88\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t34\t0.0\t2\t0 0 34\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t25\t0.0\t2\t0 0 25\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "197\t2\t0.0\t2\t0 0 2\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_513-LP3-ITS-0513_653_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_513-LP3-ITS-0513_653_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_513-LP3-ITS-0513_653_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_513-LP3-ITS-0513_653_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.135 s (10.934 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,377\n",
      "Reads with adapters:                     1,231 (9.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,377 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,725,477 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,713,666 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1231 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.4%\n",
      "  C: 19.8%\n",
      "  G: 0.6%\n",
      "  T: 8.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t342\t193.4\t0\t342\n",
      "4\t697\t48.3\t0\t697\n",
      "5\t6\t12.1\t0\t6\n",
      "6\t4\t3.0\t0\t4\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t53\t0.0\t0\t46 7\n",
      "10\t77\t0.0\t1\t5 72\n",
      "118\t29\t0.0\t2\t0 0 29\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_514-LP3-ITS-0514_1054_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_514-LP3-ITS-0514_1054_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_514-LP3-ITS-0514_1054_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_514-LP3-ITS-0514_1054_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.263 s (10.175 ¬µs/read; 5.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,846\n",
      "Reads with adapters:                     1,173 (4.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,846 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,779,646 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,738,658 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1173 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.4%\n",
      "  C: 50.0%\n",
      "  G: 1.3%\n",
      "  T: 11.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t708\t403.8\t0\t708\n",
      "6\t1\t6.3\t0\t1\n",
      "8\t3\t0.4\t0\t3\n",
      "9\t5\t0.1\t0\t5\n",
      "10\t176\t0.0\t1\t8 168\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t107\t0.0\t2\t0 2 105\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t79\t0.0\t2\t0 0 79\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t35\t0.0\t2\t0 0 35\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_515-LP3-ITS-0515_1023_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_515-LP3-ITS-0515_1023_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_515-LP3-ITS-0515_1023_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_515-LP3-ITS-0515_1023_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.358 s (10.098 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,458\n",
      "Reads with adapters:                     1,858 (5.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,458 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,672,858 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,625,929 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1858 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.8%\n",
      "  C: 11.6%\n",
      "  G: 0.1%\n",
      "  T: 2.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t161\t554.0\t0\t161\n",
      "4\t1326\t138.5\t0\t1326\n",
      "5\t7\t34.6\t0\t7\n",
      "6\t1\t8.7\t0\t1\n",
      "9\t6\t0.1\t0\t6\n",
      "10\t55\t0.0\t1\t2 53\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t131\t0.0\t2\t0 3 128\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t12\t0.0\t2\t0 0 12\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t36\t0.0\t2\t0 0 36\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t17\t0.0\t2\t0 0 17\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t8\t0.0\t2\t0 0 8\n",
      "235\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_516-LP3-ITS-0516_658_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_516-LP3-ITS-0516_658_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_516-LP3-ITS-0516_658_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_516-LP3-ITS-0516_658_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.203 s (9.978 ¬µs/read; 6.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,298\n",
      "Reads with adapters:                       298 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,298 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,109,698 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,085,003 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 298 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.3%\n",
      "  C: 47.0%\n",
      "  G: 0.0%\n",
      "  T: 8.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t83\t317.2\t0\t83\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t5\t0.1\t0\t5\n",
      "10\t18\t0.0\t1\t1 17\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "118\t73\t0.0\t2\t0 0 73\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t10\t0.0\t2\t0 0 10\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t40\t0.0\t2\t0 0 40\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_517-LP3-ITS-0517_815_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_517-LP3-ITS-0517_815_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_517-LP3-ITS-0517_815_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_517-LP3-ITS-0517_815_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.139 s (10.420 ¬µs/read; 5.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,309\n",
      "Reads with adapters:                       198 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,006,009 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,988,859 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 198 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.5%\n",
      "  C: 46.0%\n",
      "  G: 0.5%\n",
      "  T: 6.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t208.0\t0\t56\n",
      "10\t12\t0.0\t1\t1 11\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t64\t0.0\t2\t0 1 63\n",
      "133\t38\t0.0\t2\t0 0 38\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_518-LP3-ITS-0518_320_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_518-LP3-ITS-0518_320_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_518-LP3-ITS-0518_320_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_518-LP3-ITS-0518_320_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.352 s (10.228 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,435\n",
      "Reads with adapters:                     5,722 (16.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,435 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,364,935 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,325,235 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5722 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.0%\n",
      "  C: 63.7%\n",
      "  G: 1.6%\n",
      "  T: 12.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4487\t538.0\t0\t4487\n",
      "4\t3\t134.5\t0\t3\n",
      "7\t5\t2.1\t0\t5\n",
      "8\t45\t0.5\t0\t45\n",
      "9\t82\t0.1\t0\t79 3\n",
      "10\t989\t0.0\t1\t36 953\n",
      "11\t2\t0.0\t1\t0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "118\t47\t0.0\t2\t0 0 47\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "219\t2\t0.0\t2\t0 0 2\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "234\t12\t0.0\t2\t0 1 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_519-LP3-ITS-0519_63_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_519-LP3-ITS-0519_63_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_519-LP3-ITS-0519_63_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_519-LP3-ITS-0519_63_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.261 s (11.109 ¬µs/read; 5.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,485\n",
      "Reads with adapters:                       755 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,485 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,068,985 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,045,900 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 755 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.1%\n",
      "  C: 45.2%\n",
      "  G: 0.7%\n",
      "  T: 14.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t442\t367.0\t0\t442\n",
      "4\t7\t91.7\t0\t7\n",
      "8\t3\t0.4\t0\t3\n",
      "9\t19\t0.1\t0\t18 1\n",
      "10\t81\t0.0\t1\t5 76\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t66\t0.0\t2\t0 4 62\n",
      "40\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "118\t50\t0.0\t2\t0 0 50\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t49\t0.0\t2\t0 1 48\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_520-LP3-ITS-0520_400_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_520-LP3-ITS-0520_400_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_520-LP3-ITS-0520_400_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_520-LP3-ITS-0520_400_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.082 s (10.862 ¬µs/read; 5.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,538\n",
      "Reads with adapters:                       110 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,538 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,268,938 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,257,168 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 110 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.1%\n",
      "  C: 28.2%\n",
      "  G: 0.0%\n",
      "  T: 2.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t117.8\t0\t9\n",
      "4\t9\t29.4\t0\t9\n",
      "10\t2\t0.0\t1\t0 2\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "109\t5\t0.0\t2\t0 0 5\n",
      "118\t25\t0.0\t2\t0 1 24\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t2\t0.0\t2\t0 1 1\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "160\t9\t0.0\t2\t0 1 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_521-LP3-ITS-0521_478_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_521-LP3-ITS-0521_478_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_521-LP3-ITS-0521_478_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_521-LP3-ITS-0521_478_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.248 s (11.654 ¬µs/read; 5.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,303\n",
      "Reads with adapters:                     2,438 (11.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,303 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,412,203 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,393,482 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2438 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.5%\n",
      "  C: 53.9%\n",
      "  G: 1.8%\n",
      "  T: 19.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1860\t332.9\t0\t1860\n",
      "4\t12\t83.2\t0\t12\n",
      "5\t2\t20.8\t0\t2\n",
      "6\t6\t5.2\t0\t6\n",
      "7\t1\t1.3\t0\t1\n",
      "8\t21\t0.3\t0\t21\n",
      "9\t79\t0.1\t0\t75 4\n",
      "10\t371\t0.0\t1\t19 352\n",
      "11\t1\t0.0\t1\t0 1\n",
      "21\t3\t0.0\t2\t0 0 3\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "40\t24\t0.0\t2\t0 0 24\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t37\t0.0\t2\t0 1 36\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_522-LP3-ITS-0522_542_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_522-LP3-ITS-0522_542_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_522-LP3-ITS-0522_542_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_522-LP3-ITS-0522_542_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.194 s (10.609 ¬µs/read; 5.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,262\n",
      "Reads with adapters:                     2,180 (11.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,496,862 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,457,209 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2180 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.8%\n",
      "  C: 55.7%\n",
      "  G: 2.1%\n",
      "  T: 14.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1590\t285.3\t0\t1590\n",
      "4\t3\t71.3\t0\t3\n",
      "8\t8\t0.3\t0\t8\n",
      "9\t34\t0.1\t0\t33 1\n",
      "10\t388\t0.0\t1\t18 370\n",
      "11\t1\t0.0\t1\t0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "118\t23\t0.0\t2\t0 0 23\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t31\t0.0\t2\t0 0 31\n",
      "234\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_523-LP3-ITS-0523_651_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_523-LP3-ITS-0523_651_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_523-LP3-ITS-0523_651_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_523-LP3-ITS-0523_651_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.272 s (10.430 ¬µs/read; 5.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,046\n",
      "Reads with adapters:                       417 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,046 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,839,846 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,803,390 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 417 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.8%\n",
      "  C: 22.8%\n",
      "  G: 0.5%\n",
      "  T: 7.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t96\t407.0\t0\t96\n",
      "4\t2\t101.7\t0\t2\n",
      "9\t2\t0.1\t0\t2\n",
      "10\t43\t0.0\t1\t3 40\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t147\t0.0\t2\t0 2 145\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t50\t0.0\t2\t0 0 50\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "160\t15\t0.0\t2\t0 0 15\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_524-LP3-ITS-0524_143_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_524-LP3-ITS-0524_143_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_524-LP3-ITS-0524_143_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_524-LP3-ITS-0524_143_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.164 s (11.675 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,080\n",
      "Reads with adapters:                       470 (3.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,238,080 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,221,029 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 470 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.6%\n",
      "  C: 43.4%\n",
      "  G: 1.5%\n",
      "  T: 14.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t236\t220.0\t0\t236\n",
      "4\t1\t55.0\t0\t1\n",
      "7\t3\t0.9\t0\t3\n",
      "8\t26\t0.2\t0\t26\n",
      "9\t23\t0.1\t0\t22 1\n",
      "10\t48\t0.0\t1\t3 45\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "53\t7\t0.0\t2\t0 0 7\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "79\t3\t0.0\t2\t0 1 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "109\t24\t0.0\t2\t0 0 24\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t32\t0.0\t2\t0 3 29\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "148\t17\t0.0\t2\t0 0 17\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_525-LP3-ITS-0525_343_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_525-LP3-ITS-0525_343_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_525-LP3-ITS-0525_343_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_525-LP3-ITS-0525_343_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.128 s (10.659 ¬µs/read; 5.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,055\n",
      "Reads with adapters:                       382 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,055 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,628,555 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,613,092 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 382 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.3%\n",
      "  C: 40.1%\n",
      "  G: 0.5%\n",
      "  T: 14.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t193\t188.4\t0\t193\n",
      "5\t2\t11.8\t0\t2\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t20\t0.0\t0\t20\n",
      "10\t50\t0.0\t1\t7 43\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "118\t80\t0.0\t2\t0 2 78\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_526-LP3-ITS-0526_676_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_526-LP3-ITS-0526_676_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_526-LP3-ITS-0526_676_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_526-LP3-ITS-0526_676_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.085 s (11.259 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,559\n",
      "Reads with adapters:                       870 (11.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,559 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,275,259 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,262,422 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 870 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.6%\n",
      "  C: 56.9%\n",
      "  G: 2.4%\n",
      "  T: 12.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t606\t118.1\t0\t606\n",
      "4\t2\t29.5\t0\t2\n",
      "9\t24\t0.0\t0\t21 3\n",
      "10\t180\t0.0\t1\t8 172\n",
      "11\t1\t0.0\t1\t0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "232\t6\t0.0\t2\t0 0 6\n",
      "234\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_527-LP3-ITS-0527_1126_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_527-LP3-ITS-0527_1126_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_527-LP3-ITS-0527_1126_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_527-LP3-ITS-0527_1126_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.278 s (10.291 ¬µs/read; 5.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,034\n",
      "Reads with adapters:                     3,281 (12.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,034 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,137,234 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,107,783 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3281 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.7%\n",
      "  C: 61.7%\n",
      "  G: 2.0%\n",
      "  T: 16.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2644\t422.4\t0\t2644\n",
      "4\t9\t105.6\t0\t9\n",
      "7\t1\t1.7\t0\t1\n",
      "8\t12\t0.4\t0\t12\n",
      "9\t38\t0.1\t0\t34 4\n",
      "10\t442\t0.0\t1\t28 414\n",
      "11\t1\t0.0\t1\t0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t9\t0.0\t2\t0 0 9\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t58\t0.0\t2\t0 0 58\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t12\t0.0\t2\t0 0 12\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_528-LP3-ITS-0528_757_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_528-LP3-ITS-0528_757_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_528-LP3-ITS-0528_757_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_528-LP3-ITS-0528_757_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.203 s (10.729 ¬µs/read; 5.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,941\n",
      "Reads with adapters:                     4,465 (23.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,941 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,701,241 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,673,453 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4465 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.1%\n",
      "  C: 62.8%\n",
      "  G: 2.4%\n",
      "  T: 14.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3576\t296.0\t0\t3576\n",
      "4\t4\t74.0\t0\t4\n",
      "7\t3\t1.2\t0\t3\n",
      "8\t33\t0.3\t0\t33\n",
      "9\t31\t0.1\t0\t29 2\n",
      "10\t749\t0.0\t1\t27 722\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "118\t36\t0.0\t2\t0 0 36\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t13\t0.0\t2\t0 0 13\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_529-LP3-ITS-0529_53_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_529-LP3-ITS-0529_53_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_529-LP3-ITS-0529_53_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_529-LP3-ITS-0529_53_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.212 s (11.203 ¬µs/read; 5.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,880\n",
      "Reads with adapters:                       731 (3.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,880 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,682,880 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,670,230 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 731 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.3%\n",
      "  C: 38.4%\n",
      "  G: 2.6%\n",
      "  T: 11.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t367\t295.0\t0\t367\n",
      "4\t7\t73.8\t0\t7\n",
      "7\t2\t1.2\t0\t2\n",
      "8\t109\t0.3\t0\t109\n",
      "9\t38\t0.1\t0\t35 3\n",
      "10\t81\t0.0\t1\t1 80\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t69\t0.0\t2\t0 2 67\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "118\t25\t0.0\t2\t0 0 25\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_530-LP3-ITS-0530_816_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_530-LP3-ITS-0530_816_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_530-LP3-ITS-0530_816_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_530-LP3-ITS-0530_816_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.180 s (12.196 ¬µs/read; 4.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,741\n",
      "Reads with adapters:                     1,901 (12.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,741 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,437,041 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,414,050 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1901 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.7%\n",
      "  C: 55.5%\n",
      "  G: 1.9%\n",
      "  T: 14.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1386\t230.3\t0\t1386\n",
      "5\t1\t14.4\t0\t1\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t40\t0.1\t0\t39 1\n",
      "10\t381\t0.0\t1\t17 364\n",
      "118\t44\t0.0\t2\t0 0 44\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "232\t10\t0.0\t2\t0 0 10\n",
      "234\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_531-LP3-ITS-0531_32_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_531-LP3-ITS-0531_32_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_531-LP3-ITS-0531_32_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_531-LP3-ITS-0531_32_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.127 s (10.561 ¬µs/read; 5.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,980\n",
      "Reads with adapters:                       561 (4.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,980 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,605,980 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,589,281 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 561 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.7%\n",
      "  C: 51.2%\n",
      "  G: 2.1%\n",
      "  T: 18.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t379\t187.2\t0\t379\n",
      "9\t11\t0.0\t0\t11\n",
      "10\t54\t0.0\t1\t5 49\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t51\t0.0\t2\t0 0 51\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t22\t0.0\t2\t0 0 22\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_532-LP3-ITS-0532_357_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_532-LP3-ITS-0532_357_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_532-LP3-ITS-0532_357_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_532-LP3-ITS-0532_357_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.153 s (12.916 ¬µs/read; 4.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,884\n",
      "Reads with adapters:                       866 (7.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,884 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,577,084 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,564,457 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 866 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.6%\n",
      "  C: 44.5%\n",
      "  G: 0.7%\n",
      "  T: 14.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t540\t185.7\t0\t540\n",
      "4\t1\t46.4\t0\t1\n",
      "8\t66\t0.2\t0\t66\n",
      "9\t61\t0.0\t0\t58 3\n",
      "10\t140\t0.0\t1\t9 131\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "118\t26\t0.0\t2\t0 0 26\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_533-LP3-ITS-0533_11_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_533-LP3-ITS-0533_11_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_533-LP3-ITS-0533_11_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_533-LP3-ITS-0533_11_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.162 s (12.580 ¬µs/read; 4.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,900\n",
      "Reads with adapters:                     5,685 (44.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,900 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,882,900 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,855,941 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5685 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.4%\n",
      "  C: 59.0%\n",
      "  G: 1.5%\n",
      "  T: 19.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4585\t201.6\t0\t4585\n",
      "4\t3\t50.4\t0\t3\n",
      "6\t3\t3.1\t0\t3\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t15\t0.0\t0\t15\n",
      "10\t1057\t0.0\t1\t55 1002\n",
      "11\t2\t0.0\t1\t0 2\n",
      "118\t14\t0.0\t2\t0 0 14\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_534-LP3-ITS-0534_910_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_534-LP3-ITS-0534_910_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_534-LP3-ITS-0534_910_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_534-LP3-ITS-0534_910_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.117 s (10.680 ¬µs/read; 5.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,950\n",
      "Reads with adapters:                     1,356 (12.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        10,950 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,295,950 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,287,108 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1356 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.7%\n",
      "  C: 57.1%\n",
      "  G: 2.2%\n",
      "  T: 13.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t997\t171.1\t0\t997\n",
      "4\t10\t42.8\t0\t10\n",
      "7\t4\t0.7\t0\t4\n",
      "8\t15\t0.2\t0\t15\n",
      "9\t58\t0.0\t0\t48 10\n",
      "10\t250\t0.0\t1\t11 239\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "118\t16\t0.0\t2\t0 0 16\n",
      "219\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_535-LP3-ITS-0535_265_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_535-LP3-ITS-0535_265_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_535-LP3-ITS-0535_265_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_535-LP3-ITS-0535_265_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.265 s (11.068 ¬µs/read; 5.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,950\n",
      "Reads with adapters:                       605 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,950 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,208,950 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,182,412 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 605 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.8%\n",
      "  C: 45.0%\n",
      "  G: 1.3%\n",
      "  T: 10.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t325\t374.2\t0\t325\n",
      "4\t10\t93.6\t0\t10\n",
      "7\t3\t1.5\t0\t3\n",
      "8\t6\t0.4\t0\t6\n",
      "9\t16\t0.1\t0\t16\n",
      "10\t53\t0.0\t1\t3 50\n",
      "32\t10\t0.0\t2\t0 0 10\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "118\t76\t0.0\t2\t0 1 75\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t33\t0.0\t2\t0 0 33\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t11\t0.0\t2\t0 1 10\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_536-LP3-ITS-0536_995_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_536-LP3-ITS-0536_995_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_536-LP3-ITS-0536_995_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_536-LP3-ITS-0536_995_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.183 s (12.026 ¬µs/read; 4.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,241\n",
      "Reads with adapters:                     2,273 (14.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,241 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,587,541 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,572,155 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2273 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.0%\n",
      "  C: 61.2%\n",
      "  G: 2.6%\n",
      "  T: 16.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1827\t238.1\t0\t1827\n",
      "4\t1\t59.5\t0\t1\n",
      "7\t2\t0.9\t0\t2\n",
      "8\t3\t0.2\t0\t3\n",
      "9\t49\t0.1\t0\t46 3\n",
      "10\t328\t0.0\t1\t20 308\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "32\t7\t0.0\t2\t0 0 7\n",
      "40\t6\t0.0\t2\t0 2 4\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "118\t25\t0.0\t2\t0 0 25\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_537-LP3-ITS-0537_761_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_537-LP3-ITS-0537_761_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_537-LP3-ITS-0537_761_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_537-LP3-ITS-0537_761_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.253 s (11.157 ¬µs/read; 5.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,657\n",
      "Reads with adapters:                       452 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,657 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,819,757 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,779,627 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 452 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.6%\n",
      "  C: 29.6%\n",
      "  G: 0.9%\n",
      "  T: 10.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t175\t354.0\t0\t175\n",
      "4\t1\t88.5\t0\t1\n",
      "9\t9\t0.1\t0\t9\n",
      "10\t30\t0.0\t1\t2 28\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t86\t0.0\t2\t0 0 86\n",
      "133\t10\t0.0\t2\t0 0 10\n",
      "142\t17\t0.0\t2\t0 0 17\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t11\t0.0\t2\t0 0 11\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t85\t0.0\t2\t0 0 85\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_538-LP3-ITS-0538_252_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_538-LP3-ITS-0538_252_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_538-LP3-ITS-0538_252_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_538-LP3-ITS-0538_252_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.184 s (11.685 ¬µs/read; 5.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,758\n",
      "Reads with adapters:                     4,058 (25.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,758 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,743,158 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,723,902 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4058 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.6%\n",
      "  C: 62.7%\n",
      "  G: 2.0%\n",
      "  T: 16.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3324\t246.2\t0\t3324\n",
      "4\t2\t61.6\t0\t2\n",
      "8\t2\t0.2\t0\t2\n",
      "9\t67\t0.1\t0\t65 2\n",
      "10\t644\t0.0\t1\t29 615\n",
      "11\t3\t0.0\t1\t0 3\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "118\t8\t0.0\t2\t0 0 8\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_539-LP3-ITS-0539_860_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_539-LP3-ITS-0539_860_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_539-LP3-ITS-0539_860_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_539-LP3-ITS-0539_860_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.183 s (11.675 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,690\n",
      "Reads with adapters:                       192 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,690 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,722,690 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,701,255 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 192 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.4%\n",
      "  C: 25.5%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t245.2\t0\t5\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "84\t9\t0.0\t2\t0 0 9\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "118\t117\t0.0\t2\t0 1 116\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_540-LP3-ITS-0540_555_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_540-LP3-ITS-0540_555_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_540-LP3-ITS-0540_555_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_540-LP3-ITS-0540_555_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.109 s (11.013 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   9,885\n",
      "Reads with adapters:                       503 (5.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         9,885 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,975,385 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,966,370 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 503 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.7%\n",
      "  C: 40.4%\n",
      "  G: 0.8%\n",
      "  T: 14.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t265\t154.5\t0\t265\n",
      "4\t5\t38.6\t0\t5\n",
      "9\t6\t0.0\t0\t5 1\n",
      "10\t44\t0.0\t1\t0 44\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t161\t0.0\t2\t0 2 159\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_541-LP3-ITS-0541_70_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_541-LP3-ITS-0541_70_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_541-LP3-ITS-0541_70_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_541-LP3-ITS-0541_70_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.188 s (11.975 ¬µs/read; 5.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,693\n",
      "Reads with adapters:                     2,072 (13.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,693 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,723,593 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,707,846 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2072 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.1%\n",
      "  C: 54.1%\n",
      "  G: 0.9%\n",
      "  T: 20.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1582\t245.2\t0\t1582\n",
      "4\t6\t61.3\t0\t6\n",
      "5\t1\t15.3\t0\t1\n",
      "7\t5\t1.0\t0\t5\n",
      "8\t4\t0.2\t0\t4\n",
      "9\t72\t0.1\t0\t70 2\n",
      "10\t344\t0.0\t1\t20 324\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t23\t0.0\t2\t0 0 23\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_542-LP3-ITS-0542_786_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_542-LP3-ITS-0542_786_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_542-LP3-ITS-0542_786_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_542-LP3-ITS-0542_786_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.243 s (10.315 ¬µs/read; 5.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,590\n",
      "Reads with adapters:                     2,174 (9.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,590 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,100,590 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,069,253 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2174 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 56.0%\n",
      "  G: 1.6%\n",
      "  T: 13.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1570\t368.6\t0\t1570\n",
      "4\t2\t92.1\t0\t2\n",
      "7\t1\t1.4\t0\t1\n",
      "8\t6\t0.4\t0\t6\n",
      "9\t59\t0.1\t0\t55 4\n",
      "10\t372\t0.0\t1\t21 351\n",
      "118\t67\t0.0\t2\t0 0 67\n",
      "133\t8\t0.0\t2\t0 0 8\n",
      "145\t16\t0.0\t2\t0 0 16\n",
      "150\t13\t0.0\t2\t0 1 12\n",
      "151\t59\t0.0\t2\t0 0 59\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_543-LP3-ITS-0543_842_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_543-LP3-ITS-0543_842_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_543-LP3-ITS-0543_842_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_543-LP3-ITS-0543_842_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.200 s (11.663 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,189\n",
      "Reads with adapters:                     1,280 (7.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,173,889 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,152,534 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1280 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.7%\n",
      "  C: 48.8%\n",
      "  G: 1.6%\n",
      "  T: 13.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t839\t268.6\t0\t839\n",
      "4\t4\t67.1\t0\t4\n",
      "6\t1\t4.2\t0\t1\n",
      "7\t1\t1.0\t0\t1\n",
      "8\t73\t0.3\t0\t73\n",
      "9\t32\t0.1\t0\t28 4\n",
      "10\t213\t0.0\t1\t18 195\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t51\t0.0\t2\t0 3 48\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "145\t22\t0.0\t2\t0 0 22\n",
      "150\t31\t0.0\t2\t0 0 31\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_544-LP3-ITS-0544_1103_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_544-LP3-ITS-0544_1103_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_544-LP3-ITS-0544_1103_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_544-LP3-ITS-0544_1103_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.304 s (11.420 ¬µs/read; 5.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,587\n",
      "Reads with adapters:                       599 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,002,687 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,978,528 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 599 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 57.1%\n",
      "  C: 28.5%\n",
      "  G: 1.2%\n",
      "  T: 13.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t250\t415.4\t0\t250\n",
      "4\t3\t103.9\t0\t3\n",
      "5\t1\t26.0\t0\t1\n",
      "8\t12\t0.4\t0\t12\n",
      "9\t134\t0.1\t0\t125 9\n",
      "10\t46\t0.0\t1\t4 42\n",
      "11\t1\t0.0\t1\t0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t27\t0.0\t2\t0 0 27\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t16\t0.0\t2\t0 0 16\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t6\t0.0\t2\t0 1 5\n",
      "145\t39\t0.0\t2\t0 0 39\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t20\t0.0\t2\t0 0 20\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "175\t4\t0.0\t2\t0 0 4\n",
      "212\t3\t0.0\t2\t0 0 3\n",
      "229\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_545-LP3-ITS-0545_958_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_545-LP3-ITS-0545_958_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_545-LP3-ITS-0545_958_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_545-LP3-ITS-0545_958_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.165 s (11.008 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,001\n",
      "Reads with adapters:                     1,710 (11.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,001 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,515,301 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,500,445 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1710 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.2%\n",
      "  C: 58.0%\n",
      "  G: 2.0%\n",
      "  T: 17.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1350\t234.4\t0\t1350\n",
      "7\t2\t0.9\t0\t2\n",
      "8\t71\t0.2\t0\t71\n",
      "9\t33\t0.1\t0\t29 4\n",
      "10\t191\t0.0\t1\t8 183\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "118\t36\t0.0\t2\t0 1 35\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_546-LP3-ITS-0546_844_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_546-LP3-ITS-0546_844_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_546-LP3-ITS-0546_844_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_546-LP3-ITS-0546_844_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.326 s (11.872 ¬µs/read; 5.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,502\n",
      "Reads with adapters:                    12,495 (45.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,502 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,278,102 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,221,637 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 12495 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 16.9%\n",
      "  C: 65.3%\n",
      "  G: 1.9%\n",
      "  T: 15.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10492\t429.7\t0\t10492\n",
      "4\t4\t107.4\t0\t4\n",
      "5\t1\t26.9\t0\t1\n",
      "9\t21\t0.1\t0\t17 4\n",
      "10\t1915\t0.0\t1\t77 1838\n",
      "11\t5\t0.0\t1\t0 5\n",
      "32\t17\t0.0\t2\t0 0 17\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "118\t23\t0.0\t2\t0 0 23\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_547-LP3-ITS-0547_745_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_547-LP3-ITS-0547_745_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_547-LP3-ITS-0547_745_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_547-LP3-ITS-0547_745_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.287 s (10.940 ¬µs/read; 5.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,222\n",
      "Reads with adapters:                       269 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,892,822 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,863,273 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 269 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.3%\n",
      "  C: 17.8%\n",
      "  G: 0.0%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t409.7\t0\t17\n",
      "4\t13\t102.4\t0\t13\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t5\t0.0\t1\t0 5\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t140\t0.0\t2\t0 1 139\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t19\t0.0\t2\t0 0 19\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t17\t0.0\t2\t0 1 16\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_548-LP3-ITS-0548_992_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_548-LP3-ITS-0548_992_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_548-LP3-ITS-0548_992_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_548-LP3-ITS-0548_992_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.188 s (10.517 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,855\n",
      "Reads with adapters:                     1,576 (8.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,855 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,374,355 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,359,597 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1576 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.3%\n",
      "  C: 51.6%\n",
      "  G: 1.8%\n",
      "  T: 17.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1127\t279.0\t0\t1127\n",
      "4\t1\t69.7\t0\t1\n",
      "7\t1\t1.1\t0\t1\n",
      "8\t74\t0.3\t0\t74\n",
      "9\t60\t0.1\t0\t56 4\n",
      "10\t252\t0.0\t1\t15 237\n",
      "118\t39\t0.0\t2\t0 0 39\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "134\t1\t0.0\t2\t0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_549-LP3-ITS-0549_356_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_549-LP3-ITS-0549_356_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_549-LP3-ITS-0549_356_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_549-LP3-ITS-0549_356_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.325 s (10.719 ¬µs/read; 5.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,337\n",
      "Reads with adapters:                       502 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,337 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,131,437 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,061,001 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 502 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.1%\n",
      "  C: 17.3%\n",
      "  G: 1.0%\n",
      "  T: 4.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t474.0\t0\t77\n",
      "9\t3\t0.1\t0\t3\n",
      "10\t22\t0.0\t1\t0 22\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t142\t0.0\t2\t0 2 140\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "148\t2\t0.0\t2\t0 1 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "232\t9\t0.0\t2\t0 0 9\n",
      "233\t6\t0.0\t2\t0 0 6\n",
      "234\t122\t0.0\t2\t0 0 122\n",
      "235\t46\t0.0\t2\t0 1 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_550-LP3-ITS-0550_257_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_550-LP3-ITS-0550_257_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_550-LP3-ITS-0550_257_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_550-LP3-ITS-0550_257_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.164 s (10.458 ¬µs/read; 5.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,703\n",
      "Reads with adapters:                     1,305 (8.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,703 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,726,603 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,712,614 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1305 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 57.7%\n",
      "  G: 1.8%\n",
      "  T: 11.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t935\t245.4\t0\t935\n",
      "4\t1\t61.3\t0\t1\n",
      "5\t1\t15.3\t0\t1\n",
      "7\t1\t1.0\t0\t1\n",
      "8\t5\t0.2\t0\t5\n",
      "9\t97\t0.1\t0\t91 6\n",
      "10\t217\t0.0\t1\t10 207\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t19\t0.0\t2\t0 1 18\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t6\t0.0\t2\t0 0 6\n",
      "234\t12\t0.0\t2\t0 0 12\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_551-LP3-ITS-0551_475_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_551-LP3-ITS-0551_475_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_551-LP3-ITS-0551_475_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_551-LP3-ITS-0551_475_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.244 s (11.932 ¬µs/read; 5.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,479\n",
      "Reads with adapters:                     4,818 (23.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,479 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,164,179 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,136,462 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4818 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 17.1%\n",
      "  C: 64.3%\n",
      "  G: 2.1%\n",
      "  T: 16.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4057\t320.0\t0\t4057\n",
      "4\t3\t80.0\t0\t3\n",
      "7\t1\t1.2\t0\t1\n",
      "8\t4\t0.3\t0\t4\n",
      "9\t45\t0.1\t0\t44 1\n",
      "10\t660\t0.0\t1\t30 630\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "118\t19\t0.0\t2\t0 0 19\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "232\t6\t0.0\t2\t0 0 6\n",
      "234\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_552-LP3-ITS-0552_1024_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_552-LP3-ITS-0552_1024_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_552-LP3-ITS-0552_1024_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_552-LP3-ITS-0552_1024_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.214 s (11.019 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,409\n",
      "Reads with adapters:                     3,999 (20.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,409 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,842,109 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,816,421 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3999 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.8%\n",
      "  C: 60.7%\n",
      "  G: 1.9%\n",
      "  T: 16.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3220\t303.3\t0\t3220\n",
      "4\t4\t75.8\t0\t4\n",
      "7\t1\t1.2\t0\t1\n",
      "8\t28\t0.3\t0\t28\n",
      "9\t65\t0.1\t0\t57 8\n",
      "10\t622\t0.0\t1\t39 583\n",
      "11\t3\t0.0\t1\t0 3\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "118\t23\t0.0\t2\t0 0 23\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "232\t5\t0.0\t2\t0 0 5\n",
      "234\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_553-LP3-ITS-0553_200_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_553-LP3-ITS-0553_200_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_553-LP3-ITS-0553_200_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_553-LP3-ITS-0553_200_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.373 s (11.646 ¬µs/read; 5.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,025\n",
      "Reads with adapters:                     9,136 (28.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,025 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,639,525 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,585,572 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9136 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.7%\n",
      "  C: 55.6%\n",
      "  G: 2.0%\n",
      "  T: 18.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7062\t500.4\t0\t7062\n",
      "4\t2\t125.1\t0\t2\n",
      "7\t1\t2.0\t0\t1\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t101\t0.1\t0\t90 11\n",
      "10\t1903\t0.0\t1\t107 1796\n",
      "118\t17\t0.0\t2\t0 2 15\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t10\t0.0\t2\t0 0 10\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t28\t0.0\t2\t0 0 28\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_554-LP3-ITS-0554_479_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_554-LP3-ITS-0554_479_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_554-LP3-ITS-0554_479_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_554-LP3-ITS-0554_479_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.276 s (11.443 ¬µs/read; 5.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,107\n",
      "Reads with adapters:                       409 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,107 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,256,207 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,229,851 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 409 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.3%\n",
      "  C: 27.1%\n",
      "  G: 0.7%\n",
      "  T: 6.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t97\t376.7\t0\t97\n",
      "4\t1\t94.2\t0\t1\n",
      "8\t2\t0.4\t0\t2\n",
      "9\t6\t0.1\t0\t4 2\n",
      "10\t36\t0.0\t1\t0 36\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "32\t90\t0.0\t2\t0 2 88\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t65\t0.0\t2\t0 2 63\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t48\t0.0\t2\t0 0 48\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 1 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_555-LP3-ITS-0555_239_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_555-LP3-ITS-0555_239_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_555-LP3-ITS-0555_239_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_555-LP3-ITS-0555_239_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.379 s (11.148 ¬µs/read; 5.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,001\n",
      "Reads with adapters:                     2,499 (7.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,001 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,234,301 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,183,854 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2499 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.3%\n",
      "  C: 52.3%\n",
      "  G: 2.2%\n",
      "  T: 17.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1783\t531.3\t0\t1783\n",
      "4\t2\t132.8\t0\t2\n",
      "7\t1\t2.1\t0\t1\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t72\t0.1\t0\t65 7\n",
      "10\t404\t0.0\t1\t24 380\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "118\t58\t0.0\t2\t0 2 56\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t15\t0.0\t2\t0 0 15\n",
      "232\t5\t0.0\t2\t0 0 5\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t56\t0.0\t2\t0 0 56\n",
      "235\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_556-LP3-ITS-0556_592_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_556-LP3-ITS-0556_592_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_556-LP3-ITS-0556_592_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_556-LP3-ITS-0556_592_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.132 s (11.158 ¬µs/read; 5.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,795\n",
      "Reads with adapters:                     1,425 (12.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,795 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,550,295 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,532,477 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1425 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.6%\n",
      "  C: 52.2%\n",
      "  G: 1.1%\n",
      "  T: 21.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1081\t184.3\t0\t1081\n",
      "4\t1\t46.1\t0\t1\n",
      "8\t10\t0.2\t0\t10\n",
      "9\t40\t0.0\t0\t37 3\n",
      "10\t231\t0.0\t1\t16 215\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "118\t13\t0.0\t2\t0 0 13\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "232\t15\t0.0\t2\t0 1 14\n",
      "234\t21\t0.0\t2\t0 1 20\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_557-LP3-ITS-0557_1046_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_557-LP3-ITS-0557_1046_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_557-LP3-ITS-0557_1046_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_557-LP3-ITS-0557_1046_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.306 s (11.320 ¬µs/read; 5.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,008\n",
      "Reads with adapters:                     1,344 (5.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,008 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,129,408 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,101,156 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1344 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 46.9%\n",
      "  C: 35.1%\n",
      "  G: 1.2%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t721\t422.0\t0\t721\n",
      "4\t5\t105.5\t0\t5\n",
      "7\t6\t1.6\t0\t6\n",
      "8\t177\t0.4\t0\t177\n",
      "9\t103\t0.1\t0\t96 7\n",
      "10\t196\t0.0\t1\t13 183\n",
      "118\t78\t0.0\t2\t0 2 76\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t44\t0.0\t2\t0 0 44\n",
      "235\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_558-LP3-ITS-0558_1136_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_558-LP3-ITS-0558_1136_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_558-LP3-ITS-0558_1136_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_558-LP3-ITS-0558_1136_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.261 s (11.628 ¬µs/read; 5.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,481\n",
      "Reads with adapters:                     1,427 (6.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,481 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,766,781 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,739,077 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1427 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.4%\n",
      "  C: 50.8%\n",
      "  G: 2.7%\n",
      "  T: 14.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t948\t351.3\t0\t948\n",
      "4\t3\t87.8\t0\t3\n",
      "8\t4\t0.3\t0\t4\n",
      "9\t61\t0.1\t0\t55 6\n",
      "10\t232\t0.0\t1\t15 217\n",
      "32\t24\t0.0\t2\t0 2 22\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "118\t56\t0.0\t2\t0 2 54\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t40\t0.0\t2\t0 0 40\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "212\t3\t0.0\t2\t0 0 3\n",
      "232\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_559-LP3-ITS-0559_787_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_559-LP3-ITS-0559_787_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_559-LP3-ITS-0559_787_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_559-LP3-ITS-0559_787_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.369 s (10.539 ¬µs/read; 5.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,044\n",
      "Reads with adapters:                     1,910 (5.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,044 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,548,244 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,534,087 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1910 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.0%\n",
      "  C: 43.7%\n",
      "  G: 1.0%\n",
      "  T: 17.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1206\t547.6\t0\t1206\n",
      "4\t2\t136.9\t0\t2\n",
      "7\t7\t2.1\t0\t7\n",
      "8\t294\t0.5\t0\t294\n",
      "9\t140\t0.1\t0\t131 9\n",
      "10\t222\t0.0\t1\t12 210\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t35\t0.0\t2\t0 0 35\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_560-LP3-ITS-0560_194_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_560-LP3-ITS-0560_194_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_560-LP3-ITS-0560_194_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_560-LP3-ITS-0560_194_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.209 s (10.438 ¬µs/read; 5.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,980\n",
      "Reads with adapters:                       857 (4.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,980 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,013,980 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,995,861 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 857 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.3%\n",
      "  C: 36.3%\n",
      "  G: 1.8%\n",
      "  T: 14.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t441\t312.2\t0\t441\n",
      "4\t3\t78.0\t0\t3\n",
      "8\t142\t0.3\t0\t142\n",
      "9\t66\t0.1\t0\t65 1\n",
      "10\t95\t0.0\t1\t2 93\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t53\t0.0\t2\t0 1 52\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_561-LP3-ITS-0561_1080_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_561-LP3-ITS-0561_1080_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_561-LP3-ITS-0561_1080_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_561-LP3-ITS-0561_1080_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.214 s (12.043 ¬µs/read; 4.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,779\n",
      "Reads with adapters:                     3,426 (19.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,779 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,351,479 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,321,718 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3426 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.7%\n",
      "  C: 53.6%\n",
      "  G: 2.4%\n",
      "  T: 18.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2603\t277.8\t0\t2603\n",
      "4\t2\t69.4\t0\t2\n",
      "7\t2\t1.1\t0\t2\n",
      "8\t11\t0.3\t0\t11\n",
      "9\t65\t0.1\t0\t63 2\n",
      "10\t669\t0.0\t1\t34 635\n",
      "11\t3\t0.0\t1\t0 3\n",
      "118\t14\t0.0\t2\t0 0 14\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "232\t5\t0.0\t2\t0 0 5\n",
      "234\t46\t0.0\t2\t0 1 45\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_562-LP3-ITS-0562_280_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_562-LP3-ITS-0562_280_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_562-LP3-ITS-0562_280_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_562-LP3-ITS-0562_280_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.327 s (11.128 ¬µs/read; 5.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,343\n",
      "Reads with adapters:                     3,653 (12.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,343 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,832,243 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,777,094 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3653 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 52.9%\n",
      "  G: 1.8%\n",
      "  T: 17.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2676\t458.5\t0\t2676\n",
      "4\t3\t114.6\t0\t3\n",
      "7\t2\t1.8\t0\t2\n",
      "8\t6\t0.4\t0\t6\n",
      "9\t79\t0.1\t0\t75 4\n",
      "10\t687\t0.0\t1\t31 656\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "118\t45\t0.0\t2\t0 0 45\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t35\t0.0\t2\t0 1 34\n",
      "234\t99\t0.0\t2\t0 1 98\n",
      "235\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_563-LP3-ITS-0563_504_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_563-LP3-ITS-0563_504_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_563-LP3-ITS-0563_504_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_563-LP3-ITS-0563_504_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.242 s (11.402 ¬µs/read; 5.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,225\n",
      "Reads with adapters:                       525 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,225 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,388,725 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,372,432 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 525 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 46.1%\n",
      "  C: 36.0%\n",
      "  G: 0.6%\n",
      "  T: 17.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t253\t331.6\t0\t253\n",
      "4\t9\t82.9\t0\t9\n",
      "8\t11\t0.3\t0\t11\n",
      "9\t109\t0.1\t0\t102 7\n",
      "10\t36\t0.0\t1\t1 35\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t10\t0.0\t2\t0 0 10\n",
      "118\t20\t0.0\t2\t0 0 20\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t11\t0.0\t2\t0 0 11\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "234\t3\t0.0\t2\t0 1 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_564-LP3-ITS-0564_109_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_564-LP3-ITS-0564_109_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_564-LP3-ITS-0564_109_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_564-LP3-ITS-0564_109_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.196 s (11.898 ¬µs/read; 5.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,502\n",
      "Reads with adapters:                     2,643 (16.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,502 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,967,102 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,950,989 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2643 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.2%\n",
      "  C: 58.3%\n",
      "  G: 1.1%\n",
      "  T: 19.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2117\t257.8\t0\t2117\n",
      "8\t10\t0.3\t0\t10\n",
      "9\t73\t0.1\t0\t70 3\n",
      "10\t405\t0.0\t1\t22 383\n",
      "11\t1\t0.0\t1\t0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "118\t15\t0.0\t2\t0 0 15\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t8\t0.0\t2\t0 1 7\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_565-LP3-ITS-0565_944_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_565-LP3-ITS-0565_944_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_565-LP3-ITS-0565_944_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_565-LP3-ITS-0565_944_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.328 s (11.245 ¬µs/read; 5.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,170\n",
      "Reads with adapters:                     8,747 (30.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,170 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,780,170 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,725,052 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8747 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.7%\n",
      "  C: 53.4%\n",
      "  G: 1.8%\n",
      "  T: 22.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6901\t455.8\t0\t6901\n",
      "4\t3\t113.9\t0\t3\n",
      "8\t4\t0.4\t0\t4\n",
      "9\t89\t0.1\t0\t84 5\n",
      "10\t1672\t0.0\t1\t104 1568\n",
      "11\t2\t0.0\t1\t0 2\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "231\t4\t0.0\t2\t0 0 4\n",
      "232\t20\t0.0\t2\t0 1 19\n",
      "234\t44\t0.0\t2\t0 2 42\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_566-LP3-ITS-0566_635_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_566-LP3-ITS-0566_635_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_566-LP3-ITS-0566_635_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_566-LP3-ITS-0566_635_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.196 s (11.903 ¬µs/read; 5.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,499\n",
      "Reads with adapters:                     4,707 (28.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,499 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,966,199 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,935,672 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4707 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.6%\n",
      "  C: 56.5%\n",
      "  G: 2.1%\n",
      "  T: 15.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3551\t257.8\t0\t3551\n",
      "4\t6\t64.4\t0\t6\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t78\t0.1\t0\t70 8\n",
      "10\t1025\t0.0\t1\t72 953\n",
      "11\t1\t0.0\t1\t0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t6\t0.0\t2\t0 0 6\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_567-LP3-ITS-0567_832_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_567-LP3-ITS-0567_832_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_567-LP3-ITS-0567_832_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_567-LP3-ITS-0567_832_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.297 s (10.142 ¬µs/read; 5.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,312\n",
      "Reads with adapters:                       956 (3.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,312 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,822,912 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,780,338 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 956 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.3%\n",
      "  C: 44.8%\n",
      "  G: 2.1%\n",
      "  T: 15.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t604\t458.0\t0\t604\n",
      "4\t3\t114.5\t0\t3\n",
      "5\t1\t28.6\t0\t1\n",
      "8\t5\t0.4\t0\t5\n",
      "9\t33\t0.1\t0\t30 3\n",
      "10\t121\t0.0\t1\t7 114\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "209\t185\t0.0\t2\t0 3 182\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_568-LP3-ITS-0568_801_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_568-LP3-ITS-0568_801_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_568-LP3-ITS-0568_801_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_568-LP3-ITS-0568_801_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.319 s (9.515 ¬µs/read; 6.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,560\n",
      "Reads with adapters:                     4,832 (14.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,560 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,101,560 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,074,906 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4832 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.4%\n",
      "  C: 61.4%\n",
      "  G: 1.5%\n",
      "  T: 18.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3948\t524.4\t0\t3948\n",
      "4\t13\t131.1\t0\t13\n",
      "5\t2\t32.8\t0\t2\n",
      "6\t6\t8.2\t0\t6\n",
      "7\t1\t2.0\t0\t1\n",
      "9\t74\t0.1\t0\t71 3\n",
      "10\t748\t0.0\t1\t30 718\n",
      "11\t6\t0.0\t1\t0 6\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "209\t24\t0.0\t2\t0 1 23\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_569-LP3-ITS-0569_668_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_569-LP3-ITS-0569_668_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_569-LP3-ITS-0569_668_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_569-LP3-ITS-0569_668_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.339 s (11.240 ¬µs/read; 5.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,158\n",
      "Reads with adapters:                     5,007 (16.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,077,558 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,034,624 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5007 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.2%\n",
      "  C: 52.1%\n",
      "  G: 1.4%\n",
      "  T: 19.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3787\t471.2\t0\t3787\n",
      "4\t2\t117.8\t0\t2\n",
      "6\t1\t7.4\t0\t1\n",
      "8\t9\t0.5\t0\t9\n",
      "9\t137\t0.1\t0\t135 2\n",
      "10\t970\t0.0\t1\t68 902\n",
      "11\t1\t0.0\t1\t1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "118\t12\t0.0\t2\t0 0 12\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "175\t13\t0.0\t2\t0 0 13\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "232\t14\t0.0\t2\t0 0 14\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t43\t0.0\t2\t0 0 43\n",
      "235\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_570-LP3-ITS-0570_197_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_570-LP3-ITS-0570_197_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_570-LP3-ITS-0570_197_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_570-LP3-ITS-0570_197_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.252 s (9.484 ¬µs/read; 6.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,531\n",
      "Reads with adapters:                     2,420 (9.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,531 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,985,831 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,965,722 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2420 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.0%\n",
      "  C: 17.5%\n",
      "  G: 0.5%\n",
      "  T: 5.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2232\t414.5\t0\t2232\n",
      "4\t9\t103.6\t0\t9\n",
      "6\t1\t6.5\t0\t1\n",
      "9\t13\t0.1\t0\t12 1\n",
      "10\t104\t0.0\t1\t7 97\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "176\t8\t0.0\t2\t0 1 7\n",
      "209\t49\t0.0\t2\t0 0 49\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_571-LP3-ITS-0571_582_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_571-LP3-ITS-0571_582_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_571-LP3-ITS-0571_582_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_571-LP3-ITS-0571_582_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.534 s (10.175 ¬µs/read; 5.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,523\n",
      "Reads with adapters:                     4,371 (8.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,523 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,809,423 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,720,415 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4371 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.8%\n",
      "  C: 48.5%\n",
      "  G: 1.6%\n",
      "  T: 22.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3186\t820.7\t0\t3186\n",
      "4\t6\t205.2\t0\t6\n",
      "8\t2\t0.8\t0\t2\n",
      "9\t57\t0.2\t0\t53 4\n",
      "10\t778\t0.1\t1\t35 743\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "209\t339\t0.0\t2\t0 3 336\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_572-LP3-ITS-0572_876_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_572-LP3-ITS-0572_876_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_572-LP3-ITS-0572_876_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_572-LP3-ITS-0572_876_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.173 s (10.949 ¬µs/read; 5.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,835\n",
      "Reads with adapters:                     2,360 (14.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,835 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,766,335 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,742,439 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2360 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.4%\n",
      "  C: 55.8%\n",
      "  G: 0.9%\n",
      "  T: 23.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1924\t247.4\t0\t1924\n",
      "4\t1\t61.9\t0\t1\n",
      "8\t3\t0.2\t0\t3\n",
      "9\t20\t0.1\t0\t19 1\n",
      "10\t340\t0.0\t1\t15 325\n",
      "11\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "209\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_573-LP3-ITS-0573_641_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_573-LP3-ITS-0573_641_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_573-LP3-ITS-0573_641_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_573-LP3-ITS-0573_641_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.335 s (10.917 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,677\n",
      "Reads with adapters:                     3,075 (10.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,677 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,233,777 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,202,570 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3075 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.7%\n",
      "  C: 47.8%\n",
      "  G: 1.2%\n",
      "  T: 21.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2206\t479.3\t0\t2206\n",
      "4\t1\t119.8\t0\t1\n",
      "8\t2\t0.5\t0\t2\n",
      "9\t211\t0.1\t0\t203 8\n",
      "10\t537\t0.0\t1\t39 498\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "118\t55\t0.0\t2\t0 1 54\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "175\t6\t0.0\t2\t0 0 6\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "212\t4\t0.0\t2\t0 0 4\n",
      "216\t2\t0.0\t2\t0 0 2\n",
      "219\t3\t0.0\t2\t0 0 3\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_574-LP3-ITS-0574_1000_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_574-LP3-ITS-0574_1000_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_574-LP3-ITS-0574_1000_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_574-LP3-ITS-0574_1000_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.209 s (8.460 ¬µs/read; 7.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,742\n",
      "Reads with adapters:                     1,536 (6.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,742 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,447,342 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,438,152 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1536 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.4%\n",
      "  C: 61.8%\n",
      "  G: 2.0%\n",
      "  T: 17.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1242\t386.6\t0\t1242\n",
      "4\t21\t96.6\t0\t21\n",
      "5\t2\t24.2\t0\t2\n",
      "6\t7\t6.0\t0\t7\n",
      "9\t19\t0.1\t0\t18 1\n",
      "10\t228\t0.0\t1\t9 219\n",
      "11\t1\t0.0\t1\t0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "118\t2\t0.0\t2\t0 1 1\n",
      "133\t1\t0.0\t2\t0 1\n",
      "209\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_575-LP3-ITS-0575_575_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_575-LP3-ITS-0575_575_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_575-LP3-ITS-0575_575_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_575-LP3-ITS-0575_575_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.466 s (10.959 ¬µs/read; 5.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,562\n",
      "Reads with adapters:                     5,483 (12.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,562 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,811,162 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,755,184 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5483 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.6%\n",
      "  C: 54.6%\n",
      "  G: 2.0%\n",
      "  T: 17.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4189\t665.0\t0\t4189\n",
      "4\t5\t166.3\t0\t5\n",
      "6\t1\t10.4\t0\t1\n",
      "8\t5\t0.6\t0\t5\n",
      "9\t196\t0.2\t0\t187 9\n",
      "10\t933\t0.0\t1\t61 872\n",
      "118\t24\t0.0\t2\t0 0 24\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "175\t4\t0.0\t2\t0 0 4\n",
      "219\t3\t0.0\t2\t0 0 3\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t22\t0.0\t2\t0 2 20\n",
      "234\t81\t0.0\t2\t0 1 80\n",
      "235\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_576-LP3-ITS-0576_99_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_576-LP3-ITS-0576_99_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364525_576-LP3-ITS-0576_99_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364525_576-LP3-ITS-0576_99_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.379 s (8.249 ¬µs/read; 7.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,899\n",
      "Reads with adapters:                     4,132 (9.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,899 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,815,599 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,797,107 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4132 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 94.7%\n",
      "  C: 4.8%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4083\t717.2\t0\t4083\n",
      "4\t8\t179.3\t0\t8\n",
      "5\t1\t44.8\t0\t1\n",
      "6\t1\t11.2\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "176\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_001-LP3-ITS-0577_244_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_001-LP3-ITS-0577_244_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_001-LP3-ITS-0577_244_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_001-LP3-ITS-0577_244_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.236 s (12.484 ¬µs/read; 4.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,905\n",
      "Reads with adapters:                       355 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,905 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,690,405 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,661,878 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 355 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 64.2%\n",
      "  G: 2.0%\n",
      "  T: 5.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t295.4\t0\t44\n",
      "4\t8\t73.8\t0\t8\n",
      "5\t6\t18.5\t0\t6\n",
      "6\t13\t4.6\t0\t13\n",
      "7\t1\t1.2\t0\t1\n",
      "10\t4\t0.0\t1\t2 2\n",
      "12\t4\t0.0\t1\t0 4\n",
      "18\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "30\t11\t0.0\t2\t0 0 11\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "39\t4\t0.0\t2\t0 0 4\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t5\t0.0\t2\t0 1 4\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "55\t5\t0.0\t2\t0 0 5\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "59\t40\t0.0\t2\t0 1 39\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "82\t3\t0.0\t2\t0 0 3\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "89\t4\t0.0\t2\t0 0 4\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "94\t6\t0.0\t2\t0 0 6\n",
      "95\t6\t0.0\t2\t0 0 6\n",
      "96\t11\t0.0\t2\t0 0 11\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t6\t0.0\t2\t0 0 6\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t5\t0.0\t2\t0 0 5\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t16\t0.0\t2\t0 0 16\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t16\t0.0\t2\t0 1 15\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t7\t0.0\t2\t0 0 7\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t9\t0.0\t2\t0 0 9\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_002-LP3-ITS-0578_796_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_002-LP3-ITS-0578_796_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_002-LP3-ITS-0578_796_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_002-LP3-ITS-0578_796_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.325 s (12.870 ¬µs/read; 4.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,289\n",
      "Reads with adapters:                       593 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,289 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,611,989 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,569,533 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 593 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.8%\n",
      "  C: 70.2%\n",
      "  G: 1.3%\n",
      "  T: 5.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t395.1\t0\t39\n",
      "4\t34\t98.8\t0\t34\n",
      "5\t73\t24.7\t0\t73\n",
      "6\t27\t6.2\t0\t27\n",
      "7\t7\t1.5\t0\t7\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t13\t0.0\t1\t0 13\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "33\t46\t0.0\t2\t0 2 44\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "47\t6\t0.0\t2\t0 1 5\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t15\t0.0\t2\t0 0 15\n",
      "54\t2\t0.0\t2\t0 1 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t4\t0.0\t2\t0 0 4\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "76\t10\t0.0\t2\t0 0 10\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t5\t0.0\t2\t0 0 5\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "93\t9\t0.0\t2\t0 1 8\n",
      "94\t16\t0.0\t2\t0 0 16\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t9\t0.0\t2\t0 0 9\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "112\t3\t0.0\t2\t0 0 3\n",
      "113\t3\t0.0\t2\t0 0 3\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t15\t0.0\t2\t0 0 15\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t19\t0.0\t2\t0 0 19\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t16\t0.0\t2\t0 1 15\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t9\t0.0\t2\t0 0 9\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "156\t10\t0.0\t2\t0 0 10\n",
      "157\t9\t0.0\t2\t0 0 9\n",
      "158\t26\t0.0\t2\t0 0 26\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "170\t3\t0.0\t2\t0 0 3\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_003-LP3-ITS-0579_1091_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_003-LP3-ITS-0579_1091_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_003-LP3-ITS-0579_1091_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_003-LP3-ITS-0579_1091_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.259 s (13.569 ¬µs/read; 4.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,071\n",
      "Reads with adapters:                       336 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,740,371 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,710,067 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 336 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 58.0%\n",
      "  G: 1.5%\n",
      "  T: 8.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t298.0\t0\t13\n",
      "4\t10\t74.5\t0\t10\n",
      "5\t2\t18.6\t0\t2\n",
      "6\t2\t4.7\t0\t2\n",
      "7\t1\t1.2\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "20\t7\t0.0\t2\t0 1 6\n",
      "21\t4\t0.0\t2\t0 0 4\n",
      "26\t11\t0.0\t2\t0 1 10\n",
      "27\t9\t0.0\t2\t0 0 9\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t2\t0.0\t2\t0 1 1\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "38\t5\t0.0\t2\t0 0 5\n",
      "40\t2\t0.0\t2\t0 0 2\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t8\t0.0\t2\t0 0 8\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "63\t8\t0.0\t2\t0 1 7\n",
      "64\t4\t0.0\t2\t0 0 4\n",
      "65\t11\t0.0\t2\t0 0 11\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "71\t3\t0.0\t2\t0 0 3\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t12\t0.0\t2\t0 0 12\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t4\t0.0\t2\t0 1 3\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t5\t0.0\t2\t0 0 5\n",
      "116\t9\t0.0\t2\t0 0 9\n",
      "117\t5\t0.0\t2\t0 0 5\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t4\t0.0\t2\t0 0 4\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "142\t15\t0.0\t2\t0 0 15\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t11\t0.0\t2\t0 1 10\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_004-LP3-ITS-0580_392_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_004-LP3-ITS-0580_392_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_004-LP3-ITS-0580_392_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_004-LP3-ITS-0580_392_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.330 s (12.286 ¬µs/read; 4.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,859\n",
      "Reads with adapters:                       419 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,859 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,084,559 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,046,703 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 419 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.6%\n",
      "  C: 57.5%\n",
      "  G: 2.6%\n",
      "  T: 5.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t419.7\t0\t26\n",
      "4\t18\t104.9\t0\t18\n",
      "5\t5\t26.2\t0\t5\n",
      "6\t14\t6.6\t0\t14\n",
      "7\t2\t1.6\t0\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t4\t0.0\t2\t0 0 4\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t59\t0.0\t2\t0 5 54\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t8\t0.0\t2\t0 0 8\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "89\t7\t0.0\t2\t0 0 7\n",
      "90\t4\t0.0\t2\t0 0 4\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t8\t0.0\t2\t0 0 8\n",
      "95\t5\t0.0\t2\t0 0 5\n",
      "96\t29\t0.0\t2\t0 0 29\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t4\t0.0\t2\t0 1 3\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t5\t0.0\t2\t0 0 5\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t7\t0.0\t2\t0 0 7\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t6\t0.0\t2\t0 0 6\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "149\t8\t0.0\t2\t0 0 8\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t8\t0.0\t2\t0 1 7\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t14\t0.0\t2\t0 0 14\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "178\t6\t0.0\t2\t0 0 6\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "185\t2\t0.0\t2\t0 0 2\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_005-LP3-ITS-0581_540_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_005-LP3-ITS-0581_540_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_005-LP3-ITS-0581_540_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_005-LP3-ITS-0581_540_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.424 s (12.558 ¬µs/read; 4.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,790\n",
      "Reads with adapters:                       537 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,170,790 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,113,316 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 537 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.8%\n",
      "  C: 39.3%\n",
      "  G: 0.7%\n",
      "  T: 11.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t528.0\t0\t24\n",
      "4\t5\t132.0\t0\t5\n",
      "5\t6\t33.0\t0\t6\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t9\t0.0\t2\t0 0 9\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t8\t0.0\t2\t0 0 8\n",
      "37\t5\t0.0\t2\t0 0 5\n",
      "38\t3\t0.0\t2\t0 0 3\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "44\t5\t0.0\t2\t0 0 5\n",
      "45\t3\t0.0\t2\t0 0 3\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t5\t0.0\t2\t0 0 5\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t9\t0.0\t2\t0 0 9\n",
      "66\t6\t0.0\t2\t0 0 6\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t17\t0.0\t2\t0 1 16\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t10\t0.0\t2\t0 0 10\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t12\t0.0\t2\t0 0 12\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t24\t0.0\t2\t0 2 22\n",
      "101\t9\t0.0\t2\t0 0 9\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t9\t0.0\t2\t0 1 8\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "114\t7\t0.0\t2\t0 0 7\n",
      "116\t20\t0.0\t2\t0 0 20\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t6\t0.0\t2\t0 0 6\n",
      "124\t9\t0.0\t2\t0 0 9\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t7\t0.0\t2\t0 0 7\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t24\t0.0\t2\t0 0 24\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t80\t0.0\t2\t0 1 79\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t8\t0.0\t2\t0 1 7\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t17\t0.0\t2\t0 0 17\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "182\t3\t0.0\t2\t0 0 3\n",
      "191\t2\t0.0\t2\t0 0 2\n",
      "193\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t0 0 1\n",
      "201\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_006-LP3-ITS-0582_1082_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_006-LP3-ITS-0582_1082_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_006-LP3-ITS-0582_1082_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_006-LP3-ITS-0582_1082_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.264 s (15.620 ¬µs/read; 3.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,915\n",
      "Reads with adapters:                       284 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,915 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,091,415 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,064,945 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 284 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.6%\n",
      "  C: 67.6%\n",
      "  G: 1.1%\n",
      "  T: 6.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t264.3\t0\t16\n",
      "4\t10\t66.1\t0\t10\n",
      "5\t12\t16.5\t0\t12\n",
      "6\t6\t4.1\t0\t6\n",
      "7\t1\t1.0\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "33\t1\t0.0\t2\t0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "66\t7\t0.0\t2\t0 0 7\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t4\t0.0\t2\t0 0 4\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t4\t0.0\t2\t0 0 4\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "92\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t10\t0.0\t2\t0 0 10\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t43\t0.0\t2\t0 0 43\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t17\t0.0\t2\t0 0 17\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t7\t0.0\t2\t0 0 7\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "156\t6\t0.0\t2\t0 1 5\n",
      "157\t8\t0.0\t2\t0 0 8\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_007-LP3-ITS-0583_681_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_007-LP3-ITS-0583_681_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_007-LP3-ITS-0583_681_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_007-LP3-ITS-0583_681_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.242 s (12.836 ¬µs/read; 4.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,889\n",
      "Reads with adapters:                       381 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,889 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,685,589 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,648,693 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 381 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.7%\n",
      "  C: 57.7%\n",
      "  G: 1.3%\n",
      "  T: 10.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t295.1\t0\t15\n",
      "4\t10\t73.8\t0\t10\n",
      "5\t5\t18.4\t0\t5\n",
      "6\t31\t4.6\t0\t31\n",
      "7\t1\t1.2\t0\t1\n",
      "10\t2\t0.0\t1\t1 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t11\t0.0\t2\t0 0 11\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t5\t0.0\t2\t0 0 5\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t11\t0.0\t2\t0 0 11\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t16\t0.0\t2\t0 0 16\n",
      "108\t6\t0.0\t2\t0 1 5\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t6\t0.0\t2\t0 0 6\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t18\t0.0\t2\t0 0 18\n",
      "125\t11\t0.0\t2\t0 0 11\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t5\t0.0\t2\t0 0 5\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "142\t16\t0.0\t2\t0 0 16\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t12\t0.0\t2\t0 0 12\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "156\t6\t0.0\t2\t0 0 6\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "168\t2\t0.0\t2\t0 0 2\n",
      "170\t2\t0.0\t2\t0 0 2\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_008-LP3-ITS-0584_585_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_008-LP3-ITS-0584_585_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_008-LP3-ITS-0584_585_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_008-LP3-ITS-0584_585_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.172 s (15.191 ¬µs/read; 3.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,311\n",
      "Reads with adapters:                       280 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,311 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,404,611 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,385,909 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 280 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.3%\n",
      "  C: 69.3%\n",
      "  G: 1.8%\n",
      "  T: 4.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t176.7\t0\t20\n",
      "4\t31\t44.2\t0\t31\n",
      "5\t45\t11.0\t0\t45\n",
      "6\t19\t2.8\t0\t19\n",
      "7\t5\t0.7\t0\t5\n",
      "11\t5\t0.0\t1\t0 5\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t9\t0.0\t2\t0 0 9\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t4\t0.0\t2\t0 0 4\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t6\t0.0\t2\t0 0 6\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t11\t0.0\t2\t0 1 10\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t8\t0.0\t2\t0 1 7\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t1\t0.0\t2\t0 1\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "191\t2\t0.0\t2\t0 0 2\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_009-LP3-ITS-0585_588_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_009-LP3-ITS-0585_588_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_009-LP3-ITS-0585_588_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_009-LP3-ITS-0585_588_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.012 s (119.969 ¬µs/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     103\n",
      "Reads with adapters:                         2 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           103 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,003 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         30,882 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.6\t0\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_010-LP3-ITS-0586_126_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_010-LP3-ITS-0586_126_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_010-LP3-ITS-0586_126_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_010-LP3-ITS-0586_126_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.240 s (12.619 ¬µs/read; 4.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,054\n",
      "Reads with adapters:                     1,015 (5.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,054 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,735,254 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,701,260 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1015 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 9.5%\n",
      "  C: 85.0%\n",
      "  G: 1.7%\n",
      "  T: 3.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t297.7\t0\t56\n",
      "4\t16\t74.4\t0\t16\n",
      "5\t3\t18.6\t0\t3\n",
      "6\t679\t4.7\t0\t679\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t24\t0.0\t2\t0 0 24\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 1 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "116\t13\t0.0\t2\t0 0 13\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t9\t0.0\t2\t0 0 9\n",
      "124\t11\t0.0\t2\t0 0 11\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t20\t0.0\t2\t0 2 18\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t14\t0.0\t2\t0 0 14\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t6\t0.0\t2\t0 1 5\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t6\t0.0\t2\t0 1 5\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t8\t0.0\t2\t0 1 7\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_011-LP3-ITS-0587_740_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_011-LP3-ITS-0587_740_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_011-LP3-ITS-0587_740_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_011-LP3-ITS-0587_740_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.330 s (12.338 ¬µs/read; 4.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,744\n",
      "Reads with adapters:                       468 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,744 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,049,944 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,008,925 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 468 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.1%\n",
      "  C: 63.9%\n",
      "  G: 2.4%\n",
      "  T: 3.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t417.9\t0\t62\n",
      "4\t11\t104.5\t0\t11\n",
      "5\t7\t26.1\t0\t7\n",
      "6\t10\t6.5\t0\t10\n",
      "7\t2\t1.6\t0\t2\n",
      "8\t1\t0.4\t0\t1\n",
      "10\t17\t0.0\t1\t0 17\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t5\t0.0\t1\t0 5\n",
      "19\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "30\t4\t0.0\t2\t0 0 4\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "41\t3\t0.0\t2\t0 0 3\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t10\t0.0\t2\t0 0 10\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t13\t0.0\t2\t0 0 13\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "73\t6\t0.0\t2\t0 0 6\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 1\n",
      "81\t3\t0.0\t2\t0 1 2\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t5\t0.0\t2\t0 0 5\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t29\t0.0\t2\t0 0 29\n",
      "94\t11\t0.0\t2\t0 0 11\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t7\t0.0\t2\t0 0 7\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t21\t0.0\t2\t0 0 21\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t18\t0.0\t2\t0 0 18\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "157\t6\t0.0\t2\t0 0 6\n",
      "158\t27\t0.0\t2\t0 0 27\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "177\t7\t0.0\t2\t0 0 7\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_012-LP3-ITS-0588_116_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_012-LP3-ITS-0588_116_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_012-LP3-ITS-0588_116_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_012-LP3-ITS-0588_116_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.225 s (13.743 ¬µs/read; 4.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,361\n",
      "Reads with adapters:                       211 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,361 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,924,661 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,904,997 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 211 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.2%\n",
      "  C: 48.8%\n",
      "  G: 0.0%\n",
      "  T: 9.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t255.6\t0\t10\n",
      "4\t5\t63.9\t0\t5\n",
      "5\t1\t16.0\t0\t1\n",
      "6\t2\t4.0\t0\t2\n",
      "7\t1\t1.0\t0\t1\n",
      "8\t1\t0.2\t0\t1\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "21\t5\t0.0\t2\t0 0 5\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "50\t5\t0.0\t2\t0 1 4\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 1 2\n",
      "60\t7\t0.0\t2\t0 1 6\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "65\t3\t0.0\t2\t0 1 2\n",
      "66\t4\t0.0\t2\t0 0 4\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t7\t0.0\t2\t0 0 7\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t7\t0.0\t2\t0 0 7\n",
      "117\t11\t0.0\t2\t0 0 11\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t15\t0.0\t2\t0 1 14\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_013-LP3-ITS-0589_870_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_013-LP3-ITS-0589_870_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_013-LP3-ITS-0589_870_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_013-LP3-ITS-0589_870_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (118.611 ¬µs/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      59\n",
      "Reads with adapters:                         1 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            59 (100.0%)\n",
      "\n",
      "Total basepairs processed:        17,759 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         17,672 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_014-LP3-ITS-0590_533_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_014-LP3-ITS-0590_533_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_014-LP3-ITS-0590_533_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_014-LP3-ITS-0590_533_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.310 s (12.306 ¬µs/read; 4.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,199\n",
      "Reads with adapters:                       406 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,199 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,584,899 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,546,174 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 406 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.8%\n",
      "  C: 59.1%\n",
      "  G: 0.7%\n",
      "  T: 10.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t393.7\t0\t27\n",
      "4\t11\t98.4\t0\t11\n",
      "5\t5\t24.6\t0\t5\n",
      "6\t4\t6.2\t0\t4\n",
      "8\t1\t0.4\t0\t1\n",
      "11\t3\t0.0\t1\t0 3\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 1 2\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t4\t0.0\t2\t0 0 4\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t10\t0.0\t2\t0 0 10\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t12\t0.0\t2\t0 0 12\n",
      "59\t6\t0.0\t2\t0 0 6\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t30\t0.0\t2\t0 1 29\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t1\t0.0\t2\t0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t10\t0.0\t2\t0 0 10\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "106\t5\t0.0\t2\t0 1 4\n",
      "107\t16\t0.0\t2\t0 0 16\n",
      "108\t5\t0.0\t2\t0 0 5\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "116\t7\t0.0\t2\t0 0 7\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t25\t0.0\t2\t0 0 25\n",
      "125\t8\t0.0\t2\t0 0 8\n",
      "126\t9\t0.0\t2\t0 0 9\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t13\t0.0\t2\t0 1 12\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t17\t0.0\t2\t0 0 17\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t9\t0.0\t2\t0 0 9\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t0 0 1\n",
      "201\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_015-LP3-ITS-0591_831_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_015-LP3-ITS-0591_831_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_015-LP3-ITS-0591_831_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_015-LP3-ITS-0591_831_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.201 s (9.736 ¬µs/read; 6.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,642\n",
      "Reads with adapters:                     1,878 (9.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,213,242 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,202,002 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1878 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.4%\n",
      "  C: 6.9%\n",
      "  G: 0.1%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1842\t322.5\t0\t1842\n",
      "4\t3\t80.6\t0\t3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "176\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_016-LP3-ITS-0592_205_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_016-LP3-ITS-0592_205_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_016-LP3-ITS-0592_205_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_016-LP3-ITS-0592_205_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (72.575 ¬µs/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      75\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            75 (100.0%)\n",
      "\n",
      "Total basepairs processed:        22,575 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         22,575 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_017-LP3-ITS-0593_772_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_017-LP3-ITS-0593_772_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_017-LP3-ITS-0593_772_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_017-LP3-ITS-0593_772_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.252 s (12.535 ¬µs/read; 4.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,079\n",
      "Reads with adapters:                       372 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,079 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,043,779 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,006,406 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 372 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.6%\n",
      "  C: 50.3%\n",
      "  G: 1.3%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t313.7\t0\t28\n",
      "4\t8\t78.4\t0\t8\n",
      "5\t16\t19.6\t0\t16\n",
      "6\t9\t4.9\t0\t9\n",
      "7\t1\t1.2\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "27\t5\t0.0\t2\t0 2 3\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "82\t4\t0.0\t2\t0 0 4\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t4\t0.0\t2\t0 0 4\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t4\t0.0\t2\t0 0 4\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t11\t0.0\t2\t0 1 10\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t9\t0.0\t2\t0 0 9\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "109\t5\t0.0\t2\t0 0 5\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "113\t4\t0.0\t2\t0 0 4\n",
      "115\t5\t0.0\t2\t0 1 4\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "121\t3\t0.0\t2\t0 0 3\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t19\t0.0\t2\t0 1 18\n",
      "127\t14\t0.0\t2\t0 0 14\n",
      "128\t8\t0.0\t2\t0 0 8\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t8\t0.0\t2\t0 0 8\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t15\t0.0\t2\t0 0 15\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "135\t4\t0.0\t2\t0 0 4\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t14\t0.0\t2\t0 0 14\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t13\t0.0\t2\t0 0 13\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_018-LP3-ITS-0594_731_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_018-LP3-ITS-0594_731_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_018-LP3-ITS-0594_731_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_018-LP3-ITS-0594_731_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.212 s (13.198 ¬µs/read; 4.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,057\n",
      "Reads with adapters:                       307 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,057 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,833,157 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,803,503 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 307 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 46.3%\n",
      "  C: 46.6%\n",
      "  G: 0.7%\n",
      "  T: 6.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t250.9\t0\t35\n",
      "4\t11\t62.7\t0\t11\n",
      "5\t8\t15.7\t0\t8\n",
      "6\t4\t3.9\t0\t4\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "43\t6\t0.0\t2\t0 0 6\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t9\t0.0\t2\t0 0 9\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t14\t0.0\t2\t0 0 14\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "115\t5\t0.0\t2\t0 0 5\n",
      "117\t9\t0.0\t2\t0 0 9\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t6\t0.0\t2\t0 0 6\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t9\t0.0\t2\t0 0 9\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t4\t0.0\t2\t0 0 4\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t4\t0.0\t2\t0 0 4\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t9\t0.0\t2\t0 0 9\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t7\t0.0\t2\t0 0 7\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "181\t2\t0.0\t2\t0 0 2\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "191\t3\t0.0\t2\t0 0 3\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "198\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_019-LP3-ITS-0595_1128_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_019-LP3-ITS-0595_1128_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_019-LP3-ITS-0595_1128_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_019-LP3-ITS-0595_1128_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.269 s (12.917 ¬µs/read; 4.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,832\n",
      "Reads with adapters:                       320 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,832 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,270,432 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,236,853 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 320 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.4%\n",
      "  C: 43.1%\n",
      "  G: 1.9%\n",
      "  T: 5.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t325.5\t0\t15\n",
      "4\t10\t81.4\t0\t10\n",
      "5\t2\t20.3\t0\t2\n",
      "6\t8\t5.1\t0\t8\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t6\t0.0\t2\t0 1 5\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t4\t0.0\t2\t0 0 4\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "48\t6\t0.0\t2\t0 0 6\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "89\t4\t0.0\t2\t0 1 3\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t16\t0.0\t2\t0 0 16\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t16\t0.0\t2\t0 0 16\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "125\t14\t0.0\t2\t0 0 14\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t13\t0.0\t2\t0 0 13\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t11\t0.0\t2\t0 0 11\n",
      "145\t13\t0.0\t2\t0 0 13\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t6\t0.0\t2\t0 1 5\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t20\t0.0\t2\t0 0 20\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "194\t2\t0.0\t2\t0 0 2\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_020-LP3-ITS-0596_603_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_020-LP3-ITS-0596_603_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_020-LP3-ITS-0596_603_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_020-LP3-ITS-0596_603_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.223 s (13.064 ¬µs/read; 4.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,039\n",
      "Reads with adapters:                       223 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,128,739 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,107,468 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 223 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.6%\n",
      "  C: 52.5%\n",
      "  G: 2.2%\n",
      "  T: 2.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t266.2\t0\t20\n",
      "4\t6\t66.6\t0\t6\n",
      "5\t13\t16.6\t0\t13\n",
      "6\t1\t4.2\t0\t1\n",
      "27\t11\t0.0\t2\t0 0 11\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 1 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "88\t3\t0.0\t2\t0 0 3\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t7\t0.0\t2\t0 0 7\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "125\t5\t0.0\t2\t0 0 5\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t4\t0.0\t2\t0 1 3\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t13\t0.0\t2\t0 0 13\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t13\t0.0\t2\t0 0 13\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "238\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_021-LP3-ITS-0597_979_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_021-LP3-ITS-0597_979_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_021-LP3-ITS-0597_979_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_021-LP3-ITS-0597_979_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.305 s (12.021 ¬µs/read; 4.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,359\n",
      "Reads with adapters:                       333 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,633,059 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,598,072 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 333 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.0%\n",
      "  C: 55.3%\n",
      "  G: 0.6%\n",
      "  T: 5.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t396.2\t0\t18\n",
      "4\t14\t99.1\t0\t14\n",
      "5\t22\t24.8\t0\t22\n",
      "7\t8\t1.5\t0\t8\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "111\t9\t0.0\t2\t0 0 9\n",
      "113\t18\t0.0\t2\t0 0 18\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "125\t5\t0.0\t2\t0 0 5\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "131\t9\t0.0\t2\t0 0 9\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t25\t0.0\t2\t0 0 25\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t9\t0.0\t2\t0 0 9\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t18\t0.0\t2\t0 0 18\n",
      "152\t9\t0.0\t2\t0 0 9\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t35\t0.0\t2\t0 0 35\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "191\t3\t0.0\t2\t0 0 3\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_022-LP3-ITS-0598_485_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_022-LP3-ITS-0598_485_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_022-LP3-ITS-0598_485_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_022-LP3-ITS-0598_485_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.292 s (12.366 ¬µs/read; 4.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,619\n",
      "Reads with adapters:                       378 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,619 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,109,319 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,069,012 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 378 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 54.5%\n",
      "  C: 41.3%\n",
      "  G: 0.8%\n",
      "  T: 3.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t369.0\t0\t13\n",
      "4\t11\t92.3\t0\t11\n",
      "5\t28\t23.1\t0\t28\n",
      "6\t1\t5.8\t0\t1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t9\t0.0\t2\t0 0 9\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "68\t2\t0.0\t2\t0 0 2\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "81\t4\t0.0\t2\t0 0 4\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "89\t6\t0.0\t2\t0 0 6\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 1 1\n",
      "100\t7\t0.0\t2\t0 0 7\n",
      "104\t22\t0.0\t2\t0 0 22\n",
      "106\t6\t0.0\t2\t0 0 6\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "111\t6\t0.0\t2\t0 0 6\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t6\t0.0\t2\t0 0 6\n",
      "118\t8\t0.0\t2\t0 0 8\n",
      "119\t10\t0.0\t2\t0 0 10\n",
      "120\t7\t0.0\t2\t0 0 7\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t14\t0.0\t2\t0 0 14\n",
      "125\t5\t0.0\t2\t0 0 5\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t15\t0.0\t2\t0 1 14\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "141\t5\t0.0\t2\t0 0 5\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "144\t10\t0.0\t2\t0 0 10\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t8\t0.0\t2\t0 0 8\n",
      "150\t14\t0.0\t2\t0 1 13\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "158\t9\t0.0\t2\t0 0 9\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "178\t33\t0.0\t2\t0 2 31\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "199\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_023-LP3-ITS-0599_703_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_023-LP3-ITS-0599_703_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_023-LP3-ITS-0599_703_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_023-LP3-ITS-0599_703_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.248 s (12.714 ¬µs/read; 4.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,514\n",
      "Reads with adapters:                       425 (2.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,514 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,873,714 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,839,386 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 425 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.1%\n",
      "  C: 54.6%\n",
      "  G: 0.5%\n",
      "  T: 2.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t304.9\t0\t43\n",
      "4\t9\t76.2\t0\t9\n",
      "5\t42\t19.1\t0\t42\n",
      "6\t4\t4.8\t0\t4\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "27\t16\t0.0\t2\t0 1 15\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t8\t0.0\t2\t0 0 8\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 1 1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t12\t0.0\t2\t0 0 12\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t17\t0.0\t2\t0 0 17\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t7\t0.0\t2\t0 0 7\n",
      "100\t11\t0.0\t2\t0 1 10\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "104\t24\t0.0\t2\t0 0 24\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "111\t6\t0.0\t2\t0 0 6\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t7\t0.0\t2\t0 0 7\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "147\t8\t0.0\t2\t0 0 8\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t13\t0.0\t2\t0 0 13\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t10\t0.0\t2\t0 0 10\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_024-LP3-ITS-0600_274_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_024-LP3-ITS-0600_274_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_024-LP3-ITS-0600_274_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_024-LP3-ITS-0600_274_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.223 s (13.363 ¬µs/read; 4.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,676\n",
      "Reads with adapters:                       311 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,676 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,019,476 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,987,641 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 311 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.2%\n",
      "  C: 38.3%\n",
      "  G: 1.0%\n",
      "  T: 2.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t260.6\t0\t10\n",
      "4\t9\t65.1\t0\t9\n",
      "5\t2\t16.3\t0\t2\n",
      "6\t6\t4.1\t0\t6\n",
      "7\t1\t1.0\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t10\t0.0\t2\t0 0 10\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t6\t0.0\t2\t0 0 6\n",
      "44\t5\t0.0\t2\t0 0 5\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t40\t0.0\t2\t0 1 39\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 1 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t12\t0.0\t2\t0 0 12\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t10\t0.0\t2\t0 0 10\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t4\t0.0\t2\t0 0 4\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t5\t0.0\t2\t0 0 5\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "161\t2\t0.0\t2\t0 1 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "166\t2\t0.0\t2\t0 0 2\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "191\t18\t0.0\t2\t0 0 18\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_025-LP3-ITS-0601_736_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_025-LP3-ITS-0601_736_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_025-LP3-ITS-0601_736_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_025-LP3-ITS-0601_736_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.227 s (15.383 ¬µs/read; 3.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,788\n",
      "Reads with adapters:                       341 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,788 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,451,188 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,426,523 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 341 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 46.0%\n",
      "  C: 49.3%\n",
      "  G: 0.9%\n",
      "  T: 3.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t231.1\t0\t54\n",
      "4\t9\t57.8\t0\t9\n",
      "5\t11\t14.4\t0\t11\n",
      "6\t36\t3.6\t0\t36\n",
      "7\t1\t0.9\t0\t1\n",
      "9\t2\t0.1\t0\t2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "28\t15\t0.0\t2\t0 2 13\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t8\t0.0\t2\t0 1 7\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "88\t3\t0.0\t2\t0 0 3\n",
      "89\t4\t0.0\t2\t0 0 4\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "92\t3\t0.0\t2\t0 1 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t6\t0.0\t2\t0 0 6\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "104\t11\t0.0\t2\t0 0 11\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t9\t0.0\t2\t0 0 9\n",
      "126\t15\t0.0\t2\t0 0 15\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "169\t2\t0.0\t2\t0 0 2\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_026-LP3-ITS-0602_636_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_026-LP3-ITS-0602_636_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_026-LP3-ITS-0602_636_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_026-LP3-ITS-0602_636_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.334 s (12.558 ¬µs/read; 4.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,570\n",
      "Reads with adapters:                       662 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,570 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,997,570 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,946,967 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 662 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.3%\n",
      "  C: 60.7%\n",
      "  G: 2.0%\n",
      "  T: 2.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t415.2\t0\t27\n",
      "4\t16\t103.8\t0\t16\n",
      "5\t20\t25.9\t0\t20\n",
      "6\t146\t6.5\t0\t146\n",
      "7\t3\t1.6\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t23\t0.0\t2\t0 1 22\n",
      "28\t18\t0.0\t2\t0 1 17\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t6\t0.0\t2\t0 0 6\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t7\t0.0\t2\t0 0 7\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "64\t5\t0.0\t2\t0 0 5\n",
      "65\t3\t0.0\t2\t0 1 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t2\t0.0\t2\t0 1 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t7\t0.0\t2\t0 0 7\n",
      "90\t5\t0.0\t2\t0 0 5\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t16\t0.0\t2\t0 0 16\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t17\t0.0\t2\t0 0 17\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t5\t0.0\t2\t0 0 5\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t4\t0.0\t2\t0 0 4\n",
      "118\t6\t0.0\t2\t0 0 6\n",
      "119\t7\t0.0\t2\t0 0 7\n",
      "120\t9\t0.0\t2\t0 0 9\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t11\t0.0\t2\t0 0 11\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t14\t0.0\t2\t0 0 14\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t35\t0.0\t2\t0 0 35\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "141\t7\t0.0\t2\t0 1 6\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t5\t0.0\t2\t0 2 3\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t16\t0.0\t2\t0 0 16\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t9\t0.0\t2\t0 0 9\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t15\t0.0\t2\t0 0 15\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t12\t0.0\t2\t0 0 12\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "161\t5\t0.0\t2\t0 0 5\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "211\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_027-LP3-ITS-0603_180_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_027-LP3-ITS-0603_180_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_027-LP3-ITS-0603_180_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_027-LP3-ITS-0603_180_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.419 s (12.519 ¬µs/read; 4.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,449\n",
      "Reads with adapters:                       637 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,449 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,068,149 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,034,681 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 637 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.7%\n",
      "  C: 59.2%\n",
      "  G: 3.3%\n",
      "  T: 2.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t522.6\t0\t21\n",
      "4\t51\t130.7\t0\t51\n",
      "5\t52\t32.7\t0\t52\n",
      "6\t3\t8.2\t0\t3\n",
      "7\t2\t2.0\t0\t2\n",
      "10\t2\t0.0\t1\t0 2\n",
      "15\t4\t0.0\t1\t0 4\n",
      "19\t2\t0.0\t1\t0 0 2\n",
      "20\t212\t0.0\t2\t0 9 203\n",
      "21\t10\t0.0\t2\t0 0 10\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "32\t24\t0.0\t2\t0 0 24\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t6\t0.0\t2\t0 1 5\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t19\t0.0\t2\t0 1 18\n",
      "64\t6\t0.0\t2\t0 0 6\n",
      "65\t10\t0.0\t2\t0 0 10\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "87\t5\t0.0\t2\t0 1 4\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "96\t12\t0.0\t2\t0 1 11\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "104\t12\t0.0\t2\t0 0 12\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "124\t9\t0.0\t2\t0 0 9\n",
      "125\t14\t0.0\t2\t0 0 14\n",
      "126\t18\t0.0\t2\t0 1 17\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t8\t0.0\t2\t0 0 8\n",
      "148\t5\t0.0\t2\t0 0 5\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_028-LP3-ITS-0604_774_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_028-LP3-ITS-0604_774_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_028-LP3-ITS-0604_774_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_028-LP3-ITS-0604_774_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.204 s (13.027 ¬µs/read; 4.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,646\n",
      "Reads with adapters:                       207 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,709,446 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,687,598 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 207 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.8%\n",
      "  C: 49.8%\n",
      "  G: 0.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t244.5\t0\t5\n",
      "4\t14\t61.1\t0\t14\n",
      "5\t3\t15.3\t0\t3\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "34\t5\t0.0\t2\t0 0 5\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t7\t0.0\t2\t0 0 7\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t10\t0.0\t2\t0 0 10\n",
      "125\t10\t0.0\t2\t0 0 10\n",
      "126\t13\t0.0\t2\t0 0 13\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "156\t10\t0.0\t2\t0 0 10\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "171\t2\t0.0\t2\t0 0 2\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "175\t4\t0.0\t2\t0 0 4\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_029-LP3-ITS-0605_667_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_029-LP3-ITS-0605_667_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_029-LP3-ITS-0605_667_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_029-LP3-ITS-0605_667_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (69.752 ¬µs/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      80\n",
      "Reads with adapters:                         2 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            80 (100.0%)\n",
      "\n",
      "Total basepairs processed:        24,080 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         23,971 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.2\t0\t1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_030-LP3-ITS-0606_1013_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_030-LP3-ITS-0606_1013_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_030-LP3-ITS-0606_1013_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_030-LP3-ITS-0606_1013_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.190 s (12.641 ¬µs/read; 4.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,021\n",
      "Reads with adapters:                       216 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,521,321 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,501,001 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 216 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 60.6%\n",
      "  G: 0.9%\n",
      "  T: 5.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t234.7\t0\t5\n",
      "4\t7\t58.7\t0\t7\n",
      "5\t1\t14.7\t0\t1\n",
      "6\t9\t3.7\t0\t9\n",
      "7\t1\t0.9\t0\t1\n",
      "22\t7\t0.0\t2\t0 0 7\n",
      "27\t9\t0.0\t2\t0 0 9\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "38\t5\t0.0\t2\t0 0 5\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t4\t0.0\t2\t0 0 4\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "68\t6\t0.0\t2\t0 0 6\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "84\t5\t0.0\t2\t0 0 5\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t5\t0.0\t2\t0 0 5\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t9\t0.0\t2\t0 0 9\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "110\t4\t0.0\t2\t0 0 4\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t5\t0.0\t2\t0 0 5\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 1\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t10\t0.0\t2\t0 0 10\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t4\t0.0\t2\t0 0 4\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_031-LP3-ITS-0607_799_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_031-LP3-ITS-0607_799_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_031-LP3-ITS-0607_799_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_031-LP3-ITS-0607_799_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.286 s (12.694 ¬µs/read; 4.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,510\n",
      "Reads with adapters:                       449 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,775,510 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,730,561 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 449 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.4%\n",
      "  C: 54.8%\n",
      "  G: 0.0%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t50\t351.7\t0\t50\n",
      "4\t10\t87.9\t0\t10\n",
      "5\t3\t22.0\t0\t3\n",
      "6\t1\t5.5\t0\t1\n",
      "7\t2\t1.4\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t10\t0.0\t2\t0 0 10\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t5\t0.0\t2\t0 1 4\n",
      "28\t3\t0.0\t2\t0 1 2\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t4\t0.0\t2\t0 0 4\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "57\t10\t0.0\t2\t0 0 10\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 1\n",
      "85\t2\t0.0\t2\t0 0 2\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t4\t0.0\t2\t0 0 4\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t4\t0.0\t2\t0 1 3\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "111\t2\t0.0\t2\t0 1 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t3\t0.0\t2\t0 0 3\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t56\t0.0\t2\t0 0 56\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t48\t0.0\t2\t0 0 48\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t21\t0.0\t2\t0 0 21\n",
      "139\t21\t0.0\t2\t0 0 21\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t8\t0.0\t2\t0 0 8\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t19\t0.0\t2\t0 0 19\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "161\t7\t0.0\t2\t0 0 7\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "189\t2\t0.0\t2\t0 0 2\n",
      "196\t1\t0.0\t2\t0 0 1\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "238\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_032-LP3-ITS-0608_240_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_032-LP3-ITS-0608_240_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_032-LP3-ITS-0608_240_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_032-LP3-ITS-0608_240_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (58.580 ¬µs/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     121\n",
      "Reads with adapters:                         1 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           121 (100.0%)\n",
      "\n",
      "Total basepairs processed:        36,421 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         36,417 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t0.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_033-LP3-ITS-0609_514_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_033-LP3-ITS-0609_514_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_033-LP3-ITS-0609_514_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_033-LP3-ITS-0609_514_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (57.731 ¬µs/read; 1.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     105\n",
      "Reads with adapters:                         3 (2.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           105 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,605 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         31,171 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_034-LP3-ITS-0610_563_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_034-LP3-ITS-0610_563_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_034-LP3-ITS-0610_563_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_034-LP3-ITS-0610_563_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.198 s (12.861 ¬µs/read; 4.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,382\n",
      "Reads with adapters:                       671 (4.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,382 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,629,982 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,602,631 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 671 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.4%\n",
      "  C: 79.0%\n",
      "  G: 1.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t240.3\t0\t15\n",
      "4\t8\t60.1\t0\t8\n",
      "5\t3\t15.0\t0\t3\n",
      "6\t422\t3.8\t0\t422\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "24\t5\t0.0\t2\t0 0 5\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t9\t0.0\t2\t0 0 9\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t3\t0.0\t2\t0 0 3\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "89\t5\t0.0\t2\t0 0 5\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t3\t0.0\t2\t0 0 3\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t6\t0.0\t2\t0 0 6\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t5\t0.0\t2\t0 0 5\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t9\t0.0\t2\t0 0 9\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "156\t7\t0.0\t2\t0 0 7\n",
      "158\t11\t0.0\t2\t0 0 11\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_035-LP3-ITS-0611_642_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_035-LP3-ITS-0611_642_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_035-LP3-ITS-0611_642_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_035-LP3-ITS-0611_642_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (55.572 ¬µs/read; 1.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     105\n",
      "Reads with adapters:                         2 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           105 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,605 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         31,339 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_036-LP3-ITS-0612_966_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_036-LP3-ITS-0612_966_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_036-LP3-ITS-0612_966_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_036-LP3-ITS-0612_966_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.150 s (12.965 ¬µs/read; 4.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,567\n",
      "Reads with adapters:                       205 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,567 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,481,667 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,468,357 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 205 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 69.8%\n",
      "  G: 0.5%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t180.7\t0\t9\n",
      "4\t10\t45.2\t0\t10\n",
      "5\t2\t11.3\t0\t2\n",
      "6\t41\t2.8\t0\t41\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "41\t2\t0.0\t2\t0 0 2\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t15\t0.0\t2\t0 1 14\n",
      "46\t13\t0.0\t2\t0 0 13\n",
      "52\t1\t0.0\t2\t0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "83\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "126\t4\t0.0\t2\t0 0 4\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_037-LP3-ITS-0613_484_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_037-LP3-ITS-0613_484_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_037-LP3-ITS-0613_484_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_037-LP3-ITS-0613_484_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.081 s (16.983 ¬µs/read; 3.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   4,770\n",
      "Reads with adapters:                        59 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         4,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,435,770 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,431,077 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 59 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 66.1%\n",
      "  G: 1.7%\n",
      "  T: 3.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t74.5\t0\t5\n",
      "4\t2\t18.6\t0\t2\n",
      "5\t6\t4.7\t0\t6\n",
      "6\t1\t1.2\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 1 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_038-LP3-ITS-0614_1142_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_038-LP3-ITS-0614_1142_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_038-LP3-ITS-0614_1142_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_038-LP3-ITS-0614_1142_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.357 s (11.856 ¬µs/read; 5.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,078\n",
      "Reads with adapters:                       363 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,078 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,053,478 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,016,994 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 363 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.0%\n",
      "  C: 54.0%\n",
      "  G: 2.2%\n",
      "  T: 5.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t470.0\t0\t30\n",
      "4\t17\t117.5\t0\t17\n",
      "5\t3\t29.4\t0\t3\n",
      "6\t5\t7.3\t0\t5\n",
      "7\t1\t1.8\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "36\t2\t0.0\t2\t0 0 2\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "65\t3\t0.0\t2\t0 1 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "69\t15\t0.0\t2\t0 1 14\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t10\t0.0\t2\t0 0 10\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t8\t0.0\t2\t0 0 8\n",
      "107\t7\t0.0\t2\t0 0 7\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t6\t0.0\t2\t0 0 6\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "119\t5\t0.0\t2\t0 0 5\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t23\t0.0\t2\t0 0 23\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t10\t0.0\t2\t0 0 10\n",
      "134\t5\t0.0\t2\t0 1 4\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t3\t0.0\t2\t0 1 2\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t11\t0.0\t2\t0 0 11\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "156\t25\t0.0\t2\t0 0 25\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t12\t0.0\t2\t0 0 12\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "194\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_039-LP3-ITS-0615_420_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_039-LP3-ITS-0615_420_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_039-LP3-ITS-0615_420_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_039-LP3-ITS-0615_420_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.315 s (12.959 ¬µs/read; 4.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,300\n",
      "Reads with adapters:                       470 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,300 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,314,300 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,269,599 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 470 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.4%\n",
      "  C: 63.4%\n",
      "  G: 0.2%\n",
      "  T: 6.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t379.7\t0\t29\n",
      "4\t12\t94.9\t0\t12\n",
      "5\t11\t23.7\t0\t11\n",
      "6\t15\t5.9\t0\t15\n",
      "7\t1\t1.5\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t29\t0.0\t2\t0 1 28\n",
      "53\t7\t0.0\t2\t0 0 7\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t11\t0.0\t2\t0 0 11\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "63\t6\t0.0\t2\t0 0 6\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t9\t0.0\t2\t0 0 9\n",
      "87\t9\t0.0\t2\t0 0 9\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t16\t0.0\t2\t0 1 15\n",
      "106\t6\t0.0\t2\t0 0 6\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t7\t0.0\t2\t0 0 7\n",
      "117\t6\t0.0\t2\t0 0 6\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t9\t0.0\t2\t0 0 9\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t15\t0.0\t2\t0 0 15\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t4\t0.0\t2\t0 0 4\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t7\t0.0\t2\t0 0 7\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "134\t24\t0.0\t2\t0 1 23\n",
      "135\t5\t0.0\t2\t0 0 5\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t5\t0.0\t2\t0 0 5\n",
      "149\t14\t0.0\t2\t0 0 14\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t10\t0.0\t2\t0 1 9\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t12\t0.0\t2\t0 0 12\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "166\t2\t0.0\t2\t0 0 2\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_040-LP3-ITS-0616_601_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_040-LP3-ITS-0616_601_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_040-LP3-ITS-0616_601_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_040-LP3-ITS-0616_601_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.205 s (16.218 ¬µs/read; 3.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,647\n",
      "Reads with adapters:                       282 (2.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,647 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,806,747 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,778,909 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 282 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.7%\n",
      "  C: 61.7%\n",
      "  G: 1.8%\n",
      "  T: 7.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t197.6\t0\t17\n",
      "4\t10\t49.4\t0\t10\n",
      "5\t11\t12.4\t0\t11\n",
      "6\t5\t3.1\t0\t5\n",
      "7\t1\t0.8\t0\t1\n",
      "11\t5\t0.0\t1\t0 5\n",
      "12\t2\t0.0\t1\t0 2\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "40\t7\t0.0\t2\t0 1 6\n",
      "41\t2\t0.0\t2\t0 0 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t3\t0.0\t2\t0 1 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "66\t4\t0.0\t2\t0 0 4\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "70\t2\t0.0\t2\t0 0 2\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "89\t1\t0.0\t2\t0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t5\t0.0\t2\t0 0 5\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t2\t0.0\t2\t0 1 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 1 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "140\t3\t0.0\t2\t0 0 3\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t16\t0.0\t2\t0 0 16\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t11\t0.0\t2\t0 0 11\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t10\t0.0\t2\t0 0 10\n",
      "157\t16\t0.0\t2\t0 1 15\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_041-LP3-ITS-0617_1036_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_041-LP3-ITS-0617_1036_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_041-LP3-ITS-0617_1036_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_041-LP3-ITS-0617_1036_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.204 s (13.713 ¬µs/read; 4.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,889\n",
      "Reads with adapters:                       364 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,889 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,481,589 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,454,517 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 364 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.4%\n",
      "  C: 68.7%\n",
      "  G: 2.2%\n",
      "  T: 5.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t232.6\t0\t30\n",
      "4\t27\t58.2\t0\t27\n",
      "5\t40\t14.5\t0\t40\n",
      "6\t7\t3.6\t0\t7\n",
      "7\t4\t0.9\t0\t4\n",
      "8\t1\t0.2\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t6\t0.0\t1\t0 6\n",
      "12\t1\t0.0\t1\t0 1\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t10\t0.0\t2\t0 0 10\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t9\t0.0\t2\t0 1 8\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "76\t4\t0.0\t2\t0 0 4\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "94\t16\t0.0\t2\t0 0 16\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t5\t0.0\t2\t0 0 5\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t7\t0.0\t2\t0 0 7\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t1\t0.0\t2\t0 1\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "148\t3\t0.0\t2\t0 1 2\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t11\t0.0\t2\t0 0 11\n",
      "158\t9\t0.0\t2\t0 0 9\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_042-LP3-ITS-0618_792_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_042-LP3-ITS-0618_792_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_042-LP3-ITS-0618_792_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_042-LP3-ITS-0618_792_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.214 s (14.933 ¬µs/read; 4.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,326\n",
      "Reads with adapters:                       267 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,312,126 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,286,831 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 267 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.6%\n",
      "  C: 53.2%\n",
      "  G: 1.9%\n",
      "  T: 12.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t223.8\t0\t23\n",
      "4\t2\t56.0\t0\t2\n",
      "5\t5\t14.0\t0\t5\n",
      "6\t4\t3.5\t0\t4\n",
      "7\t1\t0.9\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t3\t0.0\t2\t0 0 3\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "45\t17\t0.0\t2\t0 0 17\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "56\t4\t0.0\t2\t0 0 4\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t2\t0.0\t2\t0 1 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "62\t1\t0.0\t2\t0 1\n",
      "63\t6\t0.0\t2\t0 0 6\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t7\t0.0\t2\t0 0 7\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t5\t0.0\t2\t0 0 5\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "116\t8\t0.0\t2\t0 1 7\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t21\t0.0\t2\t0 0 21\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t21\t0.0\t2\t0 0 21\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "250\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_043-LP3-ITS-0619_135_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_043-LP3-ITS-0619_135_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_043-LP3-ITS-0619_135_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_043-LP3-ITS-0619_135_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (115.932 ¬µs/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      43\n",
      "Reads with adapters:                         3 (7.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            43 (100.0%)\n",
      "\n",
      "Total basepairs processed:        12,943 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         12,687 bp (98.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 33.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_044-LP3-ITS-0620_487_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_044-LP3-ITS-0620_487_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_044-LP3-ITS-0620_487_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_044-LP3-ITS-0620_487_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.155 s (12.964 ¬µs/read; 4.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,957\n",
      "Reads with adapters:                       217 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,957 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,599,057 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,581,436 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 217 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.8%\n",
      "  C: 62.2%\n",
      "  G: 1.4%\n",
      "  T: 10.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t186.8\t0\t26\n",
      "4\t8\t46.7\t0\t8\n",
      "5\t16\t11.7\t0\t16\n",
      "6\t5\t2.9\t0\t5\n",
      "7\t3\t0.7\t0\t3\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "60\t18\t0.0\t2\t0 1 17\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t4\t0.0\t2\t0 0 4\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t8\t0.0\t2\t0 0 8\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "170\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_045-LP3-ITS-0621_1050_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_045-LP3-ITS-0621_1050_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_045-LP3-ITS-0621_1050_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_045-LP3-ITS-0621_1050_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.267 s (12.566 ¬µs/read; 4.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,213\n",
      "Reads with adapters:                       356 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,213 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,385,113 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,350,730 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 356 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.1%\n",
      "  C: 77.0%\n",
      "  G: 1.1%\n",
      "  T: 2.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t331.5\t0\t17\n",
      "4\t14\t82.9\t0\t14\n",
      "5\t9\t20.7\t0\t9\n",
      "6\t13\t5.2\t0\t13\n",
      "7\t1\t1.3\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "12\t2\t0.0\t1\t0 2\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 1 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "64\t4\t0.0\t2\t0 0 4\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "81\t12\t0.0\t2\t0 0 12\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t15\t0.0\t2\t0 0 15\n",
      "87\t11\t0.0\t2\t0 0 11\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t6\t0.0\t2\t0 0 6\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t6\t0.0\t2\t0 1 5\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "128\t79\t0.0\t2\t0 3 76\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t24\t0.0\t2\t0 0 24\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "178\t5\t0.0\t2\t0 1 4\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_046-LP3-ITS-0622_447_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_046-LP3-ITS-0622_447_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_046-LP3-ITS-0622_447_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_046-LP3-ITS-0622_447_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.223 s (13.204 ¬µs/read; 4.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,859\n",
      "Reads with adapters:                       279 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,859 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,074,559 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,049,355 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 279 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.6%\n",
      "  C: 73.8%\n",
      "  G: 1.1%\n",
      "  T: 2.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t263.4\t0\t25\n",
      "4\t6\t65.9\t0\t6\n",
      "5\t5\t16.5\t0\t5\n",
      "6\t3\t4.1\t0\t3\n",
      "10\t3\t0.0\t1\t0 3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "30\t7\t0.0\t2\t0 0 7\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "36\t16\t0.0\t2\t0 0 16\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t15\t0.0\t2\t0 1 14\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t20\t0.0\t2\t0 0 20\n",
      "87\t8\t0.0\t2\t0 1 7\n",
      "93\t8\t0.0\t2\t0 0 8\n",
      "94\t16\t0.0\t2\t0 0 16\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "108\t5\t0.0\t2\t0 0 5\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "138\t5\t0.0\t2\t0 0 5\n",
      "139\t13\t0.0\t2\t0 0 13\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "156\t11\t0.0\t2\t0 0 11\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t11\t0.0\t2\t0 0 11\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_047-LP3-ITS-0623_707_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_047-LP3-ITS-0623_707_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_047-LP3-ITS-0623_707_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_047-LP3-ITS-0623_707_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (75.916 ¬µs/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      70\n",
      "Reads with adapters:                         1 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            70 (100.0%)\n",
      "\n",
      "Total basepairs processed:        21,070 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         21,067 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_048-LP3-ITS-0624_366_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_048-LP3-ITS-0624_366_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_048-LP3-ITS-0624_366_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_048-LP3-ITS-0624_366_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.182 s (11.414 ¬µs/read; 5.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,979\n",
      "Reads with adapters:                       266 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,979 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,809,679 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,782,125 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 266 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.2%\n",
      "  C: 50.8%\n",
      "  G: 0.0%\n",
      "  T: 9.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t249.7\t0\t10\n",
      "4\t6\t62.4\t0\t6\n",
      "5\t1\t15.6\t0\t1\n",
      "6\t1\t3.9\t0\t1\n",
      "7\t1\t1.0\t0\t1\n",
      "8\t1\t0.2\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "19\t1\t0.0\t1\t0 0 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t6\t0.0\t2\t0 0 6\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "55\t4\t0.0\t2\t0 0 4\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t14\t0.0\t2\t0 0 14\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t6\t0.0\t2\t0 0 6\n",
      "87\t8\t0.0\t2\t0 0 8\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t8\t0.0\t2\t0 0 8\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t4\t0.0\t2\t0 1 3\n",
      "138\t5\t0.0\t2\t0 0 5\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 1 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t43\t0.0\t2\t0 0 43\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "172\t2\t0.0\t2\t0 0 2\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_049-LP3-ITS-0625_296_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_049-LP3-ITS-0625_296_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_049-LP3-ITS-0625_296_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_049-LP3-ITS-0625_296_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.158 s (14.674 ¬µs/read; 4.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,794\n",
      "Reads with adapters:                       243 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        10,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,248,994 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,226,899 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 243 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.2%\n",
      "  C: 52.3%\n",
      "  G: 0.8%\n",
      "  T: 5.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t168.7\t0\t27\n",
      "4\t9\t42.2\t0\t9\n",
      "5\t14\t10.5\t0\t14\n",
      "6\t5\t2.6\t0\t5\n",
      "7\t1\t0.7\t0\t1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "83\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t1 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t21\t0.0\t2\t0 0 21\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t16\t0.0\t2\t0 2 14\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t8\t0.0\t2\t0 0 8\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_050-LP3-ITS-0626_1112_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_050-LP3-ITS-0626_1112_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_050-LP3-ITS-0626_1112_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_050-LP3-ITS-0626_1112_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.164 s (13.668 ¬µs/read; 4.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,025\n",
      "Reads with adapters:                       253 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,025 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,619,525 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,593,588 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 253 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.7%\n",
      "  C: 53.0%\n",
      "  G: 3.6%\n",
      "  T: 4.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t187.9\t0\t12\n",
      "4\t11\t47.0\t0\t11\n",
      "5\t4\t11.7\t0\t4\n",
      "6\t2\t2.9\t0\t2\n",
      "8\t1\t0.2\t0\t1\n",
      "19\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 1\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t5\t0.0\t2\t0 0 5\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t2\t0.0\t2\t0 0 2\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t7\t0.0\t2\t0 0 7\n",
      "94\t26\t0.0\t2\t0 0 26\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t9\t0.0\t2\t0 0 9\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t13\t0.0\t2\t0 0 13\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t9\t0.0\t2\t0 0 9\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t8\t0.0\t2\t0 0 8\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "187\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_051-LP3-ITS-0627_1061_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_051-LP3-ITS-0627_1061_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_051-LP3-ITS-0627_1061_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_051-LP3-ITS-0627_1061_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.171 s (14.841 ¬µs/read; 4.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,516\n",
      "Reads with adapters:                       208 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,516 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,466,316 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,448,487 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 208 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.9%\n",
      "  C: 61.5%\n",
      "  G: 1.0%\n",
      "  T: 10.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t179.9\t0\t12\n",
      "4\t6\t45.0\t0\t6\n",
      "6\t3\t2.8\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 1\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t6\t0.0\t2\t0 0 6\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "56\t4\t0.0\t2\t0 0 4\n",
      "60\t5\t0.0\t2\t0 1 4\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t4\t0.0\t2\t0 0 4\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t3\t0.0\t2\t0 0 3\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "116\t8\t0.0\t2\t0 0 8\n",
      "117\t6\t0.0\t2\t0 0 6\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 1 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "177\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_052-LP3-ITS-0628_525_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_052-LP3-ITS-0628_525_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_052-LP3-ITS-0628_525_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_052-LP3-ITS-0628_525_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.171 s (13.843 ¬µs/read; 4.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,336\n",
      "Reads with adapters:                       199 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,336 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,713,136 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,692,544 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 199 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.7%\n",
      "  C: 44.2%\n",
      "  G: 0.0%\n",
      "  T: 7.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t192.8\t0\t15\n",
      "4\t2\t48.2\t0\t2\n",
      "5\t7\t12.0\t0\t7\n",
      "6\t5\t3.0\t0\t5\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t6\t0.0\t2\t0 0 6\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t11\t0.0\t2\t0 1 10\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "116\t6\t0.0\t2\t0 0 6\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t16\t0.0\t2\t0 0 16\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "178\t19\t0.0\t2\t0 0 19\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_053-LP3-ITS-0629_442_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_053-LP3-ITS-0629_442_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_053-LP3-ITS-0629_442_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_053-LP3-ITS-0629_442_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.202 s (13.658 ¬µs/read; 4.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,817\n",
      "Reads with adapters:                       230 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,817 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,459,917 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,439,568 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 230 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 46.5%\n",
      "  C: 50.0%\n",
      "  G: 0.9%\n",
      "  T: 2.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t231.5\t0\t17\n",
      "4\t21\t57.9\t0\t21\n",
      "5\t7\t14.5\t0\t7\n",
      "6\t5\t3.6\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t2\t0.0\t2\t0 1 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t4\t0.0\t2\t0 0 4\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t5\t0.0\t2\t0 0 5\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "119\t7\t0.0\t2\t0 0 7\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "121\t5\t0.0\t2\t0 0 5\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 1\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_054-LP3-ITS-0630_682_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_054-LP3-ITS-0630_682_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_054-LP3-ITS-0630_682_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_054-LP3-ITS-0630_682_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.180 s (13.872 ¬µs/read; 4.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,977\n",
      "Reads with adapters:                       226 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,906,077 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,883,019 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 226 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.1%\n",
      "  C: 58.8%\n",
      "  G: 1.3%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t202.8\t0\t7\n",
      "5\t10\t12.7\t0\t10\n",
      "6\t2\t3.2\t0\t2\n",
      "11\t3\t0.0\t1\t0 3\n",
      "27\t8\t0.0\t2\t0 0 8\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "40\t2\t0.0\t2\t0 0 2\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 1 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t7\t0.0\t2\t0 0 7\n",
      "104\t8\t0.0\t2\t0 1 7\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t8\t0.0\t2\t0 0 8\n",
      "125\t11\t0.0\t2\t0 0 11\n",
      "126\t18\t0.0\t2\t0 0 18\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "147\t4\t0.0\t2\t0 1 3\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "176\t3\t0.0\t2\t0 0 3\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_055-LP3-ITS-0631_784_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_055-LP3-ITS-0631_784_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_055-LP3-ITS-0631_784_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_055-LP3-ITS-0631_784_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (26.868 ¬µs/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     303\n",
      "Reads with adapters:                        14 (4.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           303 (100.0%)\n",
      "\n",
      "Total basepairs processed:        91,203 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         90,291 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 42.9%\n",
      "  G: 0.0%\n",
      "  T: 7.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t4.7\t0\t6\n",
      "10\t1\t0.0\t1\t0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_056-LP3-ITS-0632_1097_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_056-LP3-ITS-0632_1097_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_056-LP3-ITS-0632_1097_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_056-LP3-ITS-0632_1097_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.143 s (11.673 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,227\n",
      "Reads with adapters:                       209 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,680,327 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,655,835 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 209 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 54.1%\n",
      "  C: 43.5%\n",
      "  G: 1.4%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t191.0\t0\t6\n",
      "4\t3\t47.8\t0\t3\n",
      "5\t1\t11.9\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t9\t0.0\t2\t0 0 9\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t7\t0.0\t2\t0 0 7\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t8\t0.0\t2\t0 0 8\n",
      "121\t5\t0.0\t2\t0 0 5\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t13\t0.0\t2\t0 0 13\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t6\t0.0\t2\t0 0 6\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t8\t0.0\t2\t0 0 8\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "130\t6\t0.0\t2\t0 0 6\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "161\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_057-LP3-ITS-0633_101_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_057-LP3-ITS-0633_101_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_057-LP3-ITS-0633_101_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_057-LP3-ITS-0633_101_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.258 s (12.804 ¬µs/read; 4.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,140\n",
      "Reads with adapters:                       487 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,140 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,062,140 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,021,027 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 487 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.9%\n",
      "  C: 53.6%\n",
      "  G: 1.0%\n",
      "  T: 4.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t314.7\t0\t27\n",
      "4\t29\t78.7\t0\t29\n",
      "5\t51\t19.7\t0\t51\n",
      "6\t14\t4.9\t0\t14\n",
      "7\t3\t1.2\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t9\t0.0\t1\t0 9\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "31\t2\t0.0\t2\t1 0 1\n",
      "33\t2\t0.0\t2\t0 1 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 1\n",
      "37\t2\t0.0\t2\t0 1 1\n",
      "38\t11\t0.0\t2\t0 0 11\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t9\t0.0\t2\t0 0 9\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "53\t5\t0.0\t2\t0 0 5\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "64\t23\t0.0\t2\t0 1 22\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t11\t0.0\t2\t0 0 11\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t6\t0.0\t2\t0 0 6\n",
      "124\t24\t0.0\t2\t0 1 23\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t20\t0.0\t2\t0 0 20\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t4\t0.0\t2\t0 0 4\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t29\t0.0\t2\t0 0 29\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t24\t0.0\t2\t0 1 23\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t14\t0.0\t2\t0 1 13\n",
      "157\t13\t0.0\t2\t0 0 13\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "161\t2\t0.0\t2\t0 0 2\n",
      "163\t4\t0.0\t2\t0 0 4\n",
      "179\t2\t0.0\t2\t0 0 2\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "218\t1\t0.0\t2\t0 0 1\n",
      "238\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_058-LP3-ITS-0634_753_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_058-LP3-ITS-0634_753_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_058-LP3-ITS-0634_753_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_058-LP3-ITS-0634_753_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (86.616 ¬µs/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      65\n",
      "Reads with adapters:                         5 (7.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            65 (100.0%)\n",
      "\n",
      "Total basepairs processed:        19,565 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         19,301 bp (98.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.0%\n",
      "  C: 60.0%\n",
      "  G: 0.0%\n",
      "  T: 20.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t0.3\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_059-LP3-ITS-0635_822_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_059-LP3-ITS-0635_822_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_059-LP3-ITS-0635_822_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_059-LP3-ITS-0635_822_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (71.220 ¬µs/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      78\n",
      "Reads with adapters:                         2 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            78 (100.0%)\n",
      "\n",
      "Total basepairs processed:        23,478 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         23,241 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_060-LP3-ITS-0636_656_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_060-LP3-ITS-0636_656_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_060-LP3-ITS-0636_656_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_060-LP3-ITS-0636_656_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (51.174 ¬µs/read; 1.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     114\n",
      "Reads with adapters:                         2 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           114 (100.0%)\n",
      "\n",
      "Total basepairs processed:        34,314 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         34,308 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_061-LP3-ITS-0637_666_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_061-LP3-ITS-0637_666_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_061-LP3-ITS-0637_666_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_061-LP3-ITS-0637_666_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (43.521 ¬µs/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     142\n",
      "Reads with adapters:                         6 (4.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           142 (100.0%)\n",
      "\n",
      "Total basepairs processed:        42,742 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         42,498 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 16.7%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t2.2\t0\t4\n",
      "23\t1\t0.0\t2\t0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_062-LP3-ITS-0638_875_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_062-LP3-ITS-0638_875_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_062-LP3-ITS-0638_875_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_062-LP3-ITS-0638_875_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (61.365 ¬µs/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     104\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           104 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,304 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         31,304 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_063-LP3-ITS-0639_439_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_063-LP3-ITS-0639_439_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_063-LP3-ITS-0639_439_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_063-LP3-ITS-0639_439_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (130.886 ¬µs/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      42\n",
      "Reads with adapters:                         3 (7.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            42 (100.0%)\n",
      "\n",
      "Total basepairs processed:        12,642 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         12,626 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t0.7\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_064-LP3-ITS-0640_845_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_064-LP3-ITS-0640_845_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_064-LP3-ITS-0640_845_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_064-LP3-ITS-0640_845_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (71.154 ¬µs/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      77\n",
      "Reads with adapters:                         3 (3.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            77 (100.0%)\n",
      "\n",
      "Total basepairs processed:        23,177 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         22,975 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.2\t0\t1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_065-LP3-ITS-0641_564_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_065-LP3-ITS-0641_564_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_065-LP3-ITS-0641_564_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_065-LP3-ITS-0641_564_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (74.148 ¬µs/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      74\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            74 (100.0%)\n",
      "\n",
      "Total basepairs processed:        22,274 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         22,274 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_066-LP3-ITS-0642_610_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_066-LP3-ITS-0642_610_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_066-LP3-ITS-0642_610_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_066-LP3-ITS-0642_610_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.119 s (12.155 ¬µs/read; 4.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   9,798\n",
      "Reads with adapters:                       170 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         9,798 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,949,198 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,936,920 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 170 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 57.6%\n",
      "  G: 0.6%\n",
      "  T: 10.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t153.1\t0\t30\n",
      "4\t1\t38.3\t0\t1\n",
      "5\t2\t9.6\t0\t2\n",
      "6\t9\t2.4\t0\t9\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "26\t1\t0.0\t2\t0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "43\t4\t0.0\t2\t0 0 4\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "58\t9\t0.0\t2\t0 0 9\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t7\t0.0\t2\t0 0 7\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t4\t0.0\t2\t0 0 4\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t3\t0.0\t2\t0 1 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "140\t3\t0.0\t2\t0 1 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t3\t0.0\t2\t0 1 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t7\t0.0\t2\t0 0 7\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "178\t4\t0.0\t2\t0 0 4\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_067-LP3-ITS-0643_148_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_067-LP3-ITS-0643_148_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_067-LP3-ITS-0643_148_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_067-LP3-ITS-0643_148_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.231 s (13.317 ¬µs/read; 4.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,378\n",
      "Reads with adapters:                       370 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,378 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,230,778 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,205,488 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 370 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.3%\n",
      "  C: 62.7%\n",
      "  G: 1.1%\n",
      "  T: 5.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t271.5\t0\t20\n",
      "4\t20\t67.9\t0\t20\n",
      "5\t14\t17.0\t0\t14\n",
      "6\t82\t4.2\t0\t82\n",
      "7\t2\t1.1\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t4\t0.0\t1\t1 3\n",
      "11\t6\t0.0\t1\t1 5\n",
      "12\t2\t0.0\t1\t0 2\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "34\t6\t0.0\t2\t0 0 6\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t5\t0.0\t2\t0 0 5\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t8\t0.0\t2\t0 0 8\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "88\t3\t0.0\t2\t0 0 3\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t28\t0.0\t2\t0 0 28\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t15\t0.0\t2\t0 0 15\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "172\t2\t0.0\t2\t0 0 2\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_068-LP3-ITS-0644_389_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_068-LP3-ITS-0644_389_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_068-LP3-ITS-0644_389_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_068-LP3-ITS-0644_389_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.238 s (13.299 ¬µs/read; 4.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,864\n",
      "Reads with adapters:                       252 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,864 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,377,064 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,349,477 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 252 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.3%\n",
      "  C: 34.9%\n",
      "  G: 0.4%\n",
      "  T: 4.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t279.1\t0\t11\n",
      "4\t1\t69.8\t0\t1\n",
      "5\t3\t17.4\t0\t3\n",
      "6\t5\t4.4\t0\t5\n",
      "10\t3\t0.0\t1\t1 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t17\t0.0\t2\t0 1 16\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "59\t5\t0.0\t2\t0 1 4\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t14\t0.0\t2\t0 0 14\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 1 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "116\t6\t0.0\t2\t0 0 6\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t6\t0.0\t2\t0 1 5\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t6\t0.0\t2\t0 0 6\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t7\t0.0\t2\t0 0 7\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t7\t0.0\t2\t0 0 7\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t17\t0.0\t2\t0 0 17\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "183\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_069-LP3-ITS-0645_269_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_069-LP3-ITS-0645_269_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_069-LP3-ITS-0645_269_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_069-LP3-ITS-0645_269_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.163 s (14.812 ¬µs/read; 4.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,976\n",
      "Reads with adapters:                       198 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        10,976 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,303,776 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,288,422 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 198 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.4%\n",
      "  C: 47.0%\n",
      "  G: 1.0%\n",
      "  T: 7.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t171.5\t0\t22\n",
      "4\t4\t42.9\t0\t4\n",
      "5\t4\t10.7\t0\t4\n",
      "6\t4\t2.7\t0\t4\n",
      "7\t1\t0.7\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t4\t0.0\t1\t0 4\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t9\t0.0\t2\t0 1 8\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t7\t0.0\t2\t0 1 6\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t15\t0.0\t2\t0 1 14\n",
      "86\t5\t0.0\t2\t0 0 5\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t11\t0.0\t2\t0 0 11\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "115\t3\t0.0\t2\t0 0 3\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 1 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t11\t0.0\t2\t0 0 11\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_070-LP3-ITS-0646_999_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_070-LP3-ITS-0646_999_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_070-LP3-ITS-0646_999_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_070-LP3-ITS-0646_999_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.144 s (15.393 ¬µs/read; 3.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   9,365\n",
      "Reads with adapters:                       174 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         9,365 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,818,865 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,804,817 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 174 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.5%\n",
      "  C: 48.3%\n",
      "  G: 0.6%\n",
      "  T: 8.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t146.3\t0\t7\n",
      "4\t6\t36.6\t0\t6\n",
      "6\t8\t2.3\t0\t8\n",
      "7\t1\t0.6\t0\t1\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t15\t0.0\t2\t0 2 13\n",
      "22\t11\t0.0\t2\t0 0 11\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "40\t3\t0.0\t2\t0 0 3\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t4\t0.0\t2\t0 1 3\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "126\t8\t0.0\t2\t0 0 8\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "146\t4\t0.0\t2\t0 0 4\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_071-LP3-ITS-0647_210_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_071-LP3-ITS-0647_210_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_071-LP3-ITS-0647_210_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_071-LP3-ITS-0647_210_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.125 s (15.540 ¬µs/read; 3.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   8,028\n",
      "Reads with adapters:                       171 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         8,028 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,416,428 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,402,009 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 171 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.0%\n",
      "  C: 45.0%\n",
      "  G: 1.2%\n",
      "  T: 15.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t125.4\t0\t24\n",
      "4\t7\t31.4\t0\t7\n",
      "5\t4\t7.8\t0\t4\n",
      "6\t12\t2.0\t0\t12\n",
      "7\t1\t0.5\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "122\t1\t0.0\t2\t0 1\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t14\t0.0\t2\t0 0 14\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "186\t2\t0.0\t2\t0 0 2\n",
      "192\t1\t0.0\t2\t0 0 1\n",
      "201\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_072-LP3-ITS-0648_920_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_072-LP3-ITS-0648_920_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_072-LP3-ITS-0648_920_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_072-LP3-ITS-0648_920_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.372 s (12.902 ¬µs/read; 4.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,841\n",
      "Reads with adapters:                       596 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,841 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,681,141 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,626,429 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 596 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.6%\n",
      "  C: 44.5%\n",
      "  G: 1.7%\n",
      "  T: 8.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t54\t450.6\t0\t54\n",
      "4\t25\t112.7\t0\t25\n",
      "5\t15\t28.2\t0\t15\n",
      "6\t14\t7.0\t0\t14\n",
      "7\t2\t1.8\t0\t2\n",
      "8\t1\t0.4\t0\t1\n",
      "10\t4\t0.0\t1\t0 4\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t14\t0.0\t1\t2 12\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t19\t0.0\t2\t0 1 18\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "29\t4\t0.0\t2\t0 0 4\n",
      "30\t1\t0.0\t2\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t4\t0.0\t2\t0 1 3\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t10\t0.0\t2\t0 1 9\n",
      "62\t8\t0.0\t2\t0 0 8\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "64\t3\t0.0\t2\t0 0 3\n",
      "65\t2\t0.0\t2\t0 1 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t9\t0.0\t2\t0 0 9\n",
      "105\t16\t0.0\t2\t0 0 16\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t26\t0.0\t2\t0 0 26\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t4\t0.0\t2\t0 0 4\n",
      "114\t10\t0.0\t2\t0 0 10\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t9\t0.0\t2\t0 0 9\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "122\t7\t0.0\t2\t0 0 7\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t7\t0.0\t2\t0 0 7\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t39\t0.0\t2\t0 0 39\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t21\t0.0\t2\t0 1 20\n",
      "138\t14\t0.0\t2\t0 0 14\n",
      "139\t13\t0.0\t2\t0 0 13\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "153\t12\t0.0\t2\t0 0 12\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t3\t0.0\t2\t0 0 3\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "171\t2\t0.0\t2\t0 0 2\n",
      "173\t2\t0.0\t2\t0 0 2\n",
      "174\t3\t0.0\t2\t0 0 3\n",
      "177\t2\t0.0\t2\t0 1 1\n",
      "178\t8\t0.0\t2\t0 0 8\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "190\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "199\t14\t0.0\t2\t0 0 14\n",
      "201\t2\t0.0\t2\t0 0 2\n",
      "227\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_073-LP3-ITS-0649_989_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_073-LP3-ITS-0649_989_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_073-LP3-ITS-0649_989_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_073-LP3-ITS-0649_989_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.580 s (13.310 ¬µs/read; 4.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,605\n",
      "Reads with adapters:                       849 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,125,105 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,041,654 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 849 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.5%\n",
      "  C: 46.6%\n",
      "  G: 0.6%\n",
      "  T: 13.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t681.3\t0\t45\n",
      "4\t10\t170.3\t0\t10\n",
      "5\t6\t42.6\t0\t6\n",
      "6\t98\t10.6\t0\t98\n",
      "7\t7\t2.7\t0\t7\n",
      "10\t3\t0.0\t1\t0 3\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "23\t4\t0.0\t2\t0 0 4\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "38\t13\t0.0\t2\t0 0 13\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t6\t0.0\t2\t0 0 6\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t3\t0.0\t2\t0 1 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "70\t2\t0.0\t2\t0 0 2\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "84\t8\t0.0\t2\t0 0 8\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t10\t0.0\t2\t0 0 10\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t22\t0.0\t2\t0 0 22\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "93\t30\t0.0\t2\t0 0 30\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t4\t0.0\t2\t0 1 3\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "104\t9\t0.0\t2\t0 0 9\n",
      "105\t8\t0.0\t2\t0 0 8\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t5\t0.0\t2\t0 0 5\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "120\t4\t0.0\t2\t0 1 3\n",
      "121\t3\t0.0\t2\t0 0 3\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t56\t0.0\t2\t0 1 55\n",
      "125\t31\t0.0\t2\t0 0 31\n",
      "126\t32\t0.0\t2\t0 0 32\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t65\t0.0\t2\t0 0 65\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t49\t0.0\t2\t0 0 49\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t13\t0.0\t2\t0 0 13\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t13\t0.0\t2\t0 1 12\n",
      "151\t13\t0.0\t2\t0 0 13\n",
      "152\t15\t0.0\t2\t0 0 15\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t16\t0.0\t2\t0 0 16\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t86\t0.0\t2\t0 2 84\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "196\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "294\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_074-LP3-ITS-0650_706_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_074-LP3-ITS-0650_706_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_074-LP3-ITS-0650_706_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_074-LP3-ITS-0650_706_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (60.836 ¬µs/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      92\n",
      "Reads with adapters:                         1 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            92 (100.0%)\n",
      "\n",
      "Total basepairs processed:        27,692 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         27,542 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_075-LP3-ITS-0651_65_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_075-LP3-ITS-0651_65_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_075-LP3-ITS-0651_65_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_075-LP3-ITS-0651_65_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (135.411 ¬µs/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      44\n",
      "Reads with adapters:                         1 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            44 (100.0%)\n",
      "\n",
      "Total basepairs processed:        13,244 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         13,241 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t0.7\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_076-LP3-ITS-0652_415_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_076-LP3-ITS-0652_415_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_076-LP3-ITS-0652_415_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_076-LP3-ITS-0652_415_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (63.030 ¬µs/read; 0.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      87\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            87 (100.0%)\n",
      "\n",
      "Total basepairs processed:        26,187 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         26,187 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_077-LP3-ITS-0653_466_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_077-LP3-ITS-0653_466_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_077-LP3-ITS-0653_466_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_077-LP3-ITS-0653_466_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (52.975 ¬µs/read; 1.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     115\n",
      "Reads with adapters:                         4 (3.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           115 (100.0%)\n",
      "\n",
      "Total basepairs processed:        34,615 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         34,344 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.8\t0\t2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_078-LP3-ITS-0654_971_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_078-LP3-ITS-0654_971_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_078-LP3-ITS-0654_971_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_078-LP3-ITS-0654_971_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (51.523 ¬µs/read; 1.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     116\n",
      "Reads with adapters:                         2 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           116 (100.0%)\n",
      "\n",
      "Total basepairs processed:        34,916 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         34,780 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 50.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_079-LP3-ITS-0655_887_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_079-LP3-ITS-0655_887_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_079-LP3-ITS-0655_887_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_079-LP3-ITS-0655_887_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (54.952 ¬µs/read; 1.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     103\n",
      "Reads with adapters:                         3 (2.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           103 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,003 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         30,860 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.6\t0\t2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_080-LP3-ITS-0656_729_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_080-LP3-ITS-0656_729_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_080-LP3-ITS-0656_729_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_080-LP3-ITS-0656_729_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (64.430 ¬µs/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      88\n",
      "Reads with adapters:                         2 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            88 (100.0%)\n",
      "\n",
      "Total basepairs processed:        26,488 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         26,183 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_081-LP3-ITS-0657_166_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_081-LP3-ITS-0657_166_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_081-LP3-ITS-0657_166_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_081-LP3-ITS-0657_166_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (55.928 ¬µs/read; 1.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     109\n",
      "Reads with adapters:                         2 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           109 (100.0%)\n",
      "\n",
      "Total basepairs processed:        32,809 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         32,586 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_082-LP3-ITS-0658_6_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_082-LP3-ITS-0658_6_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_082-LP3-ITS-0658_6_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_082-LP3-ITS-0658_6_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (46.294 ¬µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     130\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           130 (100.0%)\n",
      "\n",
      "Total basepairs processed:        39,130 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         39,130 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_083-LP3-ITS-0659_700_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_083-LP3-ITS-0659_700_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_083-LP3-ITS-0659_700_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_083-LP3-ITS-0659_700_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (62.767 ¬µs/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      91\n",
      "Reads with adapters:                         7 (7.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            91 (100.0%)\n",
      "\n",
      "Total basepairs processed:        27,391 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         27,117 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 14.3%\n",
      "  C: 42.9%\n",
      "  G: 0.0%\n",
      "  T: 42.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t1.4\t0\t4\n",
      "9\t1\t0.0\t0\t1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_084-LP3-ITS-0660_1096_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_084-LP3-ITS-0660_1096_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_084-LP3-ITS-0660_1096_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_084-LP3-ITS-0660_1096_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (41.325 ¬µs/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     148\n",
      "Reads with adapters:                         1 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           148 (100.0%)\n",
      "\n",
      "Total basepairs processed:        44,548 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         44,401 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_085-LP3-ITS-0661_30_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_085-LP3-ITS-0661_30_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_085-LP3-ITS-0661_30_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_085-LP3-ITS-0661_30_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (42.215 ¬µs/read; 1.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     158\n",
      "Reads with adapters:                         2 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           158 (100.0%)\n",
      "\n",
      "Total basepairs processed:        47,558 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         47,413 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.5\t0\t1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_086-LP3-ITS-0662_798_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_086-LP3-ITS-0662_798_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_086-LP3-ITS-0662_798_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_086-LP3-ITS-0662_798_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (42.020 ¬µs/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     152\n",
      "Reads with adapters:                         5 (3.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           152 (100.0%)\n",
      "\n",
      "Total basepairs processed:        45,752 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         45,359 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.0%\n",
      "  C: 60.0%\n",
      "  G: 0.0%\n",
      "  T: 20.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.4\t0\t1\n",
      "6\t1\t0.0\t0\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_087-LP3-ITS-0663_1138_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_087-LP3-ITS-0663_1138_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_087-LP3-ITS-0663_1138_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_087-LP3-ITS-0663_1138_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (55.037 ¬µs/read; 1.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     107\n",
      "Reads with adapters:                         2 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           107 (100.0%)\n",
      "\n",
      "Total basepairs processed:        32,207 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         32,201 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 50.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_088-LP3-ITS-0664_1114_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_088-LP3-ITS-0664_1114_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_088-LP3-ITS-0664_1114_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_088-LP3-ITS-0664_1114_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (51.908 ¬µs/read; 1.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     117\n",
      "Reads with adapters:                         3 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           117 (100.0%)\n",
      "\n",
      "Total basepairs processed:        35,217 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         35,071 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.8\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_089-LP3-ITS-0665_455_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_089-LP3-ITS-0665_455_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_089-LP3-ITS-0665_455_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_089-LP3-ITS-0665_455_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (54.178 ¬µs/read; 1.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     108\n",
      "Reads with adapters:                         4 (3.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           108 (100.0%)\n",
      "\n",
      "Total basepairs processed:        32,508 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         32,202 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.7\t0\t1\n",
      "6\t1\t0.0\t0\t1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_090-LP3-ITS-0666_812_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_090-LP3-ITS-0666_812_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_090-LP3-ITS-0666_812_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_090-LP3-ITS-0666_812_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.010 s (20.624 ¬µs/read; 2.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     468\n",
      "Reads with adapters:                        19 (4.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           468 (100.0%)\n",
      "\n",
      "Total basepairs processed:       140,868 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        138,782 bp (98.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.6%\n",
      "  C: 47.4%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t7.3\t0\t4\n",
      "5\t1\t0.5\t0\t1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "156\t7\t0.0\t2\t0 0 7\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "206\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_091-LP3-ITS-0667_789_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_091-LP3-ITS-0667_789_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_091-LP3-ITS-0667_789_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_091-LP3-ITS-0667_789_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (58.198 ¬µs/read; 1.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     101\n",
      "Reads with adapters:                         1 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           101 (100.0%)\n",
      "\n",
      "Total basepairs processed:        30,401 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         30,268 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_092-LP3-ITS-0668_617_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_092-LP3-ITS-0668_617_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_092-LP3-ITS-0668_617_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_092-LP3-ITS-0668_617_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (41.021 ¬µs/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     146\n",
      "Reads with adapters:                         1 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           146 (100.0%)\n",
      "\n",
      "Total basepairs processed:        43,946 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         43,943 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_093-LP3-ITS-0669_108_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_093-LP3-ITS-0669_108_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_093-LP3-ITS-0669_108_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_093-LP3-ITS-0669_108_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (58.558 ¬µs/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      97\n",
      "Reads with adapters:                         2 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            97 (100.0%)\n",
      "\n",
      "Total basepairs processed:        29,197 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         28,788 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_094-LP3-ITS-0670_161_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_094-LP3-ITS-0670_161_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_094-LP3-ITS-0670_161_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_094-LP3-ITS-0670_161_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (39.174 ¬µs/read; 1.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     160\n",
      "Reads with adapters:                         3 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           160 (100.0%)\n",
      "\n",
      "Total basepairs processed:        48,160 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         48,144 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2.5\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_095-LP3-ITS-0671_482_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_095-LP3-ITS-0671_482_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_095-LP3-ITS-0671_482_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_095-LP3-ITS-0671_482_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (72.169 ¬µs/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      76\n",
      "Reads with adapters:                         4 (5.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            76 (100.0%)\n",
      "\n",
      "Total basepairs processed:        22,876 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         22,605 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.2\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_096-LP3-ITS-0672_981_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_096-LP3-ITS-0672_981_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_096-LP3-ITS-0672_981_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_096-LP3-ITS-0672_981_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (44.879 ¬µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     126\n",
      "Reads with adapters:                         2 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           126 (100.0%)\n",
      "\n",
      "Total basepairs processed:        37,926 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         37,764 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t0.5\t0\t1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_097-LP3-ITS-0673_934_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_097-LP3-ITS-0673_934_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_097-LP3-ITS-0673_934_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_097-LP3-ITS-0673_934_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.190 s (12.572 ¬µs/read; 4.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,131\n",
      "Reads with adapters:                       328 (2.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,131 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,554,431 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,527,712 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 328 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.1%\n",
      "  C: 67.1%\n",
      "  G: 1.2%\n",
      "  T: 7.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t236.4\t0\t24\n",
      "4\t10\t59.1\t0\t10\n",
      "5\t38\t14.8\t0\t38\n",
      "6\t9\t3.7\t0\t9\n",
      "8\t1\t0.2\t0\t1\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "45\t3\t0.0\t2\t0 0 3\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "47\t4\t0.0\t2\t0 0 4\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t13\t0.0\t2\t0 2 11\n",
      "58\t16\t0.0\t2\t0 0 16\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t7\t0.0\t2\t0 0 7\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t6\t0.0\t2\t0 0 6\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t5\t0.0\t2\t0 0 5\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t6\t0.0\t2\t0 0 6\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t5\t0.0\t2\t0 1 4\n",
      "142\t20\t0.0\t2\t0 0 20\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t8\t0.0\t2\t0 0 8\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t5\t0.0\t2\t0 1 4\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "170\t2\t0.0\t2\t0 0 2\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_098-LP3-ITS-0674_1124_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_098-LP3-ITS-0674_1124_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_098-LP3-ITS-0674_1124_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_098-LP3-ITS-0674_1124_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.277 s (13.941 ¬µs/read; 4.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,876\n",
      "Reads with adapters:                       354 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,876 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,982,676 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,949,734 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 354 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.7%\n",
      "  C: 50.6%\n",
      "  G: 2.8%\n",
      "  T: 11.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t310.6\t0\t23\n",
      "4\t10\t77.6\t0\t10\n",
      "5\t22\t19.4\t0\t22\n",
      "6\t2\t4.9\t0\t2\n",
      "7\t1\t1.2\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "22\t4\t0.0\t2\t0 0 4\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t19\t0.0\t2\t0 1 18\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t26\t0.0\t2\t0 0 26\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t5\t0.0\t2\t0 0 5\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "66\t1\t0.0\t2\t0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t4\t0.0\t2\t0 1 3\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t12\t0.0\t2\t0 1 11\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t5\t0.0\t2\t0 0 5\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "102\t5\t0.0\t2\t0 0 5\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t5\t0.0\t2\t0 0 5\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t7\t0.0\t2\t0 1 6\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t15\t0.0\t2\t0 0 15\n",
      "157\t11\t0.0\t2\t0 0 11\n",
      "158\t7\t0.0\t2\t0 0 7\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "203\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_099-LP3-ITS-0675_675_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_099-LP3-ITS-0675_675_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_099-LP3-ITS-0675_675_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_099-LP3-ITS-0675_675_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.384 s (12.220 ¬µs/read; 4.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,401\n",
      "Reads with adapters:                       585 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,401 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,451,701 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,391,469 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 585 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.8%\n",
      "  C: 51.5%\n",
      "  G: 3.1%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t490.6\t0\t43\n",
      "4\t5\t122.7\t0\t5\n",
      "5\t3\t30.7\t0\t3\n",
      "6\t28\t7.7\t0\t28\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t14\t0.0\t2\t0 1 13\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "41\t3\t0.0\t2\t0 0 3\n",
      "44\t3\t0.0\t2\t0 0 3\n",
      "49\t9\t0.0\t2\t0 0 9\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "99\t7\t0.0\t2\t0 0 7\n",
      "100\t16\t0.0\t2\t0 0 16\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t11\t0.0\t2\t0 0 11\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t20\t0.0\t2\t0 0 20\n",
      "111\t5\t0.0\t2\t0 0 5\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t19\t0.0\t2\t0 0 19\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t9\t0.0\t2\t0 0 9\n",
      "119\t65\t0.0\t2\t0 1 64\n",
      "120\t9\t0.0\t2\t0 0 9\n",
      "121\t6\t0.0\t2\t0 1 5\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "125\t7\t0.0\t2\t0 1 6\n",
      "126\t9\t0.0\t2\t0 0 9\n",
      "127\t22\t0.0\t2\t0 0 22\n",
      "128\t17\t0.0\t2\t0 0 17\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t8\t0.0\t2\t0 0 8\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t53\t0.0\t2\t0 0 53\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t4\t0.0\t2\t0 0 4\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "141\t4\t0.0\t2\t0 1 3\n",
      "142\t20\t0.0\t2\t0 0 20\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t21\t0.0\t2\t0 0 21\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "162\t4\t0.0\t2\t0 0 4\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_100-LP3-ITS-0676_698_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_100-LP3-ITS-0676_698_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_100-LP3-ITS-0676_698_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_100-LP3-ITS-0676_698_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.284 s (12.274 ¬µs/read; 4.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,123\n",
      "Reads with adapters:                       454 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,960,023 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,908,768 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 454 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 56.6%\n",
      "  C: 40.5%\n",
      "  G: 1.1%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t361.3\t0\t19\n",
      "4\t7\t90.3\t0\t7\n",
      "6\t3\t5.6\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "23\t12\t0.0\t2\t0 0 12\n",
      "35\t10\t0.0\t2\t0 1 9\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "68\t2\t0.0\t2\t0 0 2\n",
      "70\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t7\t0.0\t2\t0 0 7\n",
      "99\t11\t0.0\t2\t0 0 11\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t33\t0.0\t2\t0 0 33\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t5\t0.0\t2\t0 0 5\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "123\t41\t0.0\t2\t0 1 40\n",
      "124\t20\t0.0\t2\t0 1 19\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t6\t0.0\t2\t0 0 6\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t5\t0.0\t2\t0 0 5\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "136\t6\t0.0\t2\t0 0 6\n",
      "137\t19\t0.0\t2\t0 0 19\n",
      "139\t58\t0.0\t2\t0 0 58\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t16\t0.0\t2\t0 0 16\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t21\t0.0\t2\t0 0 21\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t15\t0.0\t2\t0 0 15\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "188\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_101-LP3-ITS-0677_1094_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_101-LP3-ITS-0677_1094_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_101-LP3-ITS-0677_1094_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_101-LP3-ITS-0677_1094_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.175 s (13.803 ¬µs/read; 4.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,686\n",
      "Reads with adapters:                       247 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,686 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,818,486 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,794,337 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 247 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.0%\n",
      "  C: 48.2%\n",
      "  G: 1.2%\n",
      "  T: 1.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t198.2\t0\t21\n",
      "4\t10\t49.6\t0\t10\n",
      "5\t4\t12.4\t0\t4\n",
      "10\t3\t0.0\t1\t0 3\n",
      "12\t13\t0.0\t1\t0 13\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "89\t6\t0.0\t2\t0 0 6\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t14\t0.0\t2\t0 0 14\n",
      "124\t21\t0.0\t2\t0 0 21\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t36\t0.0\t2\t0 1 35\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t12\t0.0\t2\t0 0 12\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t15\t0.0\t2\t0 0 15\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "161\t7\t0.0\t2\t0 0 7\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_102-LP3-ITS-0678_84_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_102-LP3-ITS-0678_84_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_102-LP3-ITS-0678_84_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_102-LP3-ITS-0678_84_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.177 s (9.710 ¬µs/read; 6.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,277\n",
      "Reads with adapters:                     1,631 (8.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,277 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,501,377 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,494,090 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1631 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.3%\n",
      "  C: 6.6%\n",
      "  G: 0.1%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1614\t285.6\t0\t1614\n",
      "4\t2\t71.4\t0\t2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "176\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_103-LP3-ITS-0679_412_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_103-LP3-ITS-0679_412_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_103-LP3-ITS-0679_412_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_103-LP3-ITS-0679_412_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.216 s (12.711 ¬µs/read; 4.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,958\n",
      "Reads with adapters:                       516 (3.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,958 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,104,358 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,069,917 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 516 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.2%\n",
      "  C: 64.0%\n",
      "  G: 0.4%\n",
      "  T: 5.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t265.0\t0\t51\n",
      "4\t8\t66.2\t0\t8\n",
      "5\t9\t16.6\t0\t9\n",
      "6\t133\t4.1\t0\t133\n",
      "7\t2\t1.0\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 1 1\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t3\t0.0\t2\t0 0 3\n",
      "35\t5\t0.0\t2\t0 0 5\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t43\t0.0\t2\t0 2 41\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t10\t0.0\t2\t0 0 10\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t14\t0.0\t2\t0 0 14\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "109\t7\t0.0\t2\t0 0 7\n",
      "111\t7\t0.0\t2\t0 0 7\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t3\t0.0\t2\t0 1 2\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t22\t0.0\t2\t0 0 22\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t5\t0.0\t2\t0 0 5\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "135\t6\t0.0\t2\t0 0 6\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t32\t0.0\t2\t0 0 32\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t9\t0.0\t2\t0 0 9\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t8\t0.0\t2\t0 0 8\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "192\t1\t0.0\t2\t0 0 1\n",
      "228\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_104-LP3-ITS-0680_27_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_104-LP3-ITS-0680_27_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_104-LP3-ITS-0680_27_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_104-LP3-ITS-0680_27_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.281 s (12.344 ¬µs/read; 4.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,773\n",
      "Reads with adapters:                       375 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,773 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,854,673 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,813,346 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 375 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.3%\n",
      "  C: 46.1%\n",
      "  G: 0.5%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t355.8\t0\t20\n",
      "4\t13\t89.0\t0\t13\n",
      "5\t3\t22.2\t0\t3\n",
      "6\t3\t5.6\t0\t3\n",
      "7\t3\t1.4\t0\t3\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "39\t2\t0.0\t2\t0 0 2\n",
      "43\t5\t0.0\t2\t0 0 5\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "81\t5\t0.0\t2\t0 0 5\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "87\t8\t0.0\t2\t0 0 8\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "96\t7\t0.0\t2\t0 0 7\n",
      "99\t10\t0.0\t2\t0 0 10\n",
      "100\t8\t0.0\t2\t0 0 8\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "119\t7\t0.0\t2\t0 0 7\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t27\t0.0\t2\t0 0 27\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t3\t0.0\t2\t0 1 2\n",
      "133\t60\t0.0\t2\t0 0 60\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "147\t9\t0.0\t2\t0 0 9\n",
      "148\t10\t0.0\t2\t0 0 10\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t11\t0.0\t2\t0 1 10\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "161\t8\t0.0\t2\t0 0 8\n",
      "163\t19\t0.0\t2\t0 0 19\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_105-LP3-ITS-0681_643_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_105-LP3-ITS-0681_643_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_105-LP3-ITS-0681_643_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_105-LP3-ITS-0681_643_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.324 s (11.698 ¬µs/read; 5.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,703\n",
      "Reads with adapters:                       685 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,703 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,338,603 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,290,342 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 685 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.2%\n",
      "  C: 64.7%\n",
      "  G: 1.2%\n",
      "  T: 7.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t432.9\t0\t36\n",
      "4\t48\t108.2\t0\t48\n",
      "5\t91\t27.1\t0\t91\n",
      "6\t33\t6.8\t0\t33\n",
      "7\t12\t1.7\t0\t12\n",
      "8\t2\t0.4\t0\t2\n",
      "10\t5\t0.0\t1\t1 4\n",
      "11\t22\t0.0\t1\t0 22\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "27\t16\t0.0\t2\t0 1 15\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t38\t0.0\t2\t0 0 38\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t5\t0.0\t2\t0 0 5\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t7\t0.0\t2\t0 0 7\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t8\t0.0\t2\t0 0 8\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "76\t23\t0.0\t2\t0 0 23\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t3\t0.0\t2\t0 1 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t24\t0.0\t2\t0 0 24\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t9\t0.0\t2\t0 0 9\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t7\t0.0\t2\t0 0 7\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "126\t14\t0.0\t2\t0 0 14\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t23\t0.0\t2\t0 0 23\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "139\t14\t0.0\t2\t0 0 14\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t28\t0.0\t2\t0 0 28\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t22\t0.0\t2\t0 0 22\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t7\t0.0\t2\t0 0 7\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "156\t20\t0.0\t2\t0 0 20\n",
      "157\t11\t0.0\t2\t0 0 11\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "170\t2\t0.0\t2\t0 0 2\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "178\t3\t0.0\t2\t0 0 3\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "185\t2\t0.0\t2\t0 0 2\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_106-LP3-ITS-0682_342_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_106-LP3-ITS-0682_342_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_106-LP3-ITS-0682_342_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_106-LP3-ITS-0682_342_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.241 s (13.254 ¬µs/read; 4.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,186\n",
      "Reads with adapters:                       365 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,473,986 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,447,019 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 365 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.1%\n",
      "  C: 59.5%\n",
      "  G: 1.1%\n",
      "  T: 7.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t284.2\t0\t24\n",
      "4\t29\t71.0\t0\t29\n",
      "5\t56\t17.8\t0\t56\n",
      "6\t7\t4.4\t0\t7\n",
      "7\t7\t1.1\t0\t7\n",
      "8\t1\t0.3\t0\t1\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t14\t0.0\t2\t0 0 14\n",
      "33\t3\t0.0\t2\t0 0 3\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t5\t0.0\t2\t0 0 5\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t5\t0.0\t2\t0 0 5\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t7\t0.0\t2\t0 1 6\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "94\t26\t0.0\t2\t0 0 26\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t5\t0.0\t2\t0 0 5\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "144\t1\t0.0\t2\t0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t14\t0.0\t2\t0 0 14\n",
      "157\t7\t0.0\t2\t0 0 7\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t9\t0.0\t2\t0 1 8\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_107-LP3-ITS-0683_527_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_107-LP3-ITS-0683_527_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_107-LP3-ITS-0683_527_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_107-LP3-ITS-0683_527_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.250 s (13.036 ¬µs/read; 4.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,197\n",
      "Reads with adapters:                       405 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,197 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,778,297 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,736,362 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 405 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.7%\n",
      "  C: 48.9%\n",
      "  G: 1.5%\n",
      "  T: 4.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t300.0\t0\t22\n",
      "4\t10\t75.0\t0\t10\n",
      "5\t2\t18.7\t0\t2\n",
      "7\t1\t1.2\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "57\t22\t0.0\t2\t0 1 21\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "85\t22\t0.0\t2\t0 0 22\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "88\t4\t0.0\t2\t0 1 3\n",
      "89\t5\t0.0\t2\t0 0 5\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t13\t0.0\t2\t0 0 13\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t11\t0.0\t2\t0 0 11\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "100\t4\t0.0\t2\t0 0 4\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "111\t3\t0.0\t2\t0 1 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t8\t0.0\t2\t0 0 8\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t12\t0.0\t2\t0 0 12\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "122\t4\t0.0\t2\t0 0 4\n",
      "123\t9\t0.0\t2\t0 1 8\n",
      "124\t40\t0.0\t2\t0 1 39\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t7\t0.0\t2\t0 0 7\n",
      "128\t15\t0.0\t2\t0 0 15\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t4\t0.0\t2\t0 0 4\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "195\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_108-LP3-ITS-0684_310_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_108-LP3-ITS-0684_310_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_108-LP3-ITS-0684_310_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_108-LP3-ITS-0684_310_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.176 s (14.129 ¬µs/read; 4.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,450\n",
      "Reads with adapters:                       231 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,450 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,747,450 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,725,159 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 231 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.2%\n",
      "  C: 47.6%\n",
      "  G: 1.3%\n",
      "  T: 3.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t194.5\t0\t18\n",
      "4\t4\t48.6\t0\t4\n",
      "6\t2\t3.0\t0\t2\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "38\t3\t0.0\t2\t0 0 3\n",
      "46\t12\t0.0\t2\t0 1 11\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t6\t0.0\t2\t0 0 6\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "64\t4\t0.0\t2\t0 0 4\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "67\t4\t0.0\t2\t0 0 4\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t6\t0.0\t2\t0 0 6\n",
      "102\t5\t0.0\t2\t0 0 5\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t17\t0.0\t2\t0 0 17\n",
      "125\t7\t0.0\t2\t0 0 7\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t28\t0.0\t2\t0 0 28\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "139\t10\t0.0\t2\t0 0 10\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t8\t0.0\t2\t0 0 8\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_109-LP3-ITS-0685_137_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_109-LP3-ITS-0685_137_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_109-LP3-ITS-0685_137_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_109-LP3-ITS-0685_137_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.232 s (12.777 ¬µs/read; 4.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,127\n",
      "Reads with adapters:                       309 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,127 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,456,227 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,419,363 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 309 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.4%\n",
      "  C: 32.4%\n",
      "  G: 1.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t283.2\t0\t5\n",
      "4\t6\t70.8\t0\t6\n",
      "5\t2\t17.7\t0\t2\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "73\t6\t0.0\t2\t0 0 6\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "79\t8\t0.0\t2\t0 1 7\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t9\t0.0\t2\t0 0 9\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "115\t70\t0.0\t2\t0 0 70\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t9\t0.0\t2\t0 0 9\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "139\t18\t0.0\t2\t0 0 18\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t11\t0.0\t2\t0 0 11\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t29\t0.0\t2\t0 0 29\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "163\t5\t0.0\t2\t0 0 5\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_110-LP3-ITS-0686_890_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_110-LP3-ITS-0686_890_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_110-LP3-ITS-0686_890_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_110-LP3-ITS-0686_890_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.264 s (12.500 ¬µs/read; 4.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,145\n",
      "Reads with adapters:                       397 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,145 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,364,645 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,327,684 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 397 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.1%\n",
      "  C: 45.1%\n",
      "  G: 2.3%\n",
      "  T: 4.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t330.4\t0\t51\n",
      "4\t10\t82.6\t0\t10\n",
      "5\t1\t20.6\t0\t1\n",
      "7\t6\t1.3\t0\t6\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t8\t0.0\t2\t0 0 8\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t5\t0.0\t2\t0 0 5\n",
      "27\t9\t0.0\t2\t0 1 8\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 1\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "62\t1\t0.0\t2\t0 1\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t14\t0.0\t2\t0 0 14\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t13\t0.0\t2\t0 0 13\n",
      "100\t7\t0.0\t2\t0 0 7\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "104\t21\t0.0\t2\t0 1 20\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "123\t16\t0.0\t2\t0 1 15\n",
      "124\t25\t0.0\t2\t0 0 25\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t17\t0.0\t2\t0 1 16\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t51\t0.0\t2\t0 0 51\n",
      "137\t29\t0.0\t2\t0 0 29\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "147\t10\t0.0\t2\t0 0 10\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_111-LP3-ITS-0687_253_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_111-LP3-ITS-0687_253_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_111-LP3-ITS-0687_253_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_111-LP3-ITS-0687_253_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.171 s (12.085 ¬µs/read; 4.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,154\n",
      "Reads with adapters:                       204 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,260,354 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,237,843 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 204 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 48.0%\n",
      "  G: 0.0%\n",
      "  T: 2.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t55.3\t0\t1\n",
      "5\t3\t13.8\t0\t3\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "28\t3\t0.0\t2\t0 1 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t3\t0.0\t2\t0 1 2\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "57\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t7\t0.0\t2\t0 1 6\n",
      "89\t4\t0.0\t2\t0 0 4\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t8\t0.0\t2\t0 1 7\n",
      "125\t12\t0.0\t2\t0 0 12\n",
      "126\t7\t0.0\t2\t0 1 6\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t7\t0.0\t2\t0 1 6\n",
      "158\t11\t0.0\t2\t0 0 11\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "174\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_112-LP3-ITS-0688_904_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_112-LP3-ITS-0688_904_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_112-LP3-ITS-0688_904_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_112-LP3-ITS-0688_904_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.288 s (13.040 ¬µs/read; 4.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,067\n",
      "Reads with adapters:                       569 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,067 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,642,167 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,602,494 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 569 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.3%\n",
      "  C: 61.2%\n",
      "  G: 0.9%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t344.8\t0\t19\n",
      "4\t7\t86.2\t0\t7\n",
      "5\t3\t21.5\t0\t3\n",
      "6\t169\t5.4\t0\t169\n",
      "10\t2\t0.0\t1\t0 2\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 1\n",
      "26\t4\t0.0\t2\t0 0 4\n",
      "27\t26\t0.0\t2\t0 1 25\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "30\t5\t0.0\t2\t0 0 5\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t18\t0.0\t2\t0 0 18\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "83\t3\t0.0\t2\t0 0 3\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t11\t0.0\t2\t0 0 11\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t5\t0.0\t2\t0 0 5\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "100\t12\t0.0\t2\t0 2 10\n",
      "104\t29\t0.0\t2\t0 1 28\n",
      "106\t8\t0.0\t2\t0 0 8\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "110\t32\t0.0\t2\t0 0 32\n",
      "111\t15\t0.0\t2\t0 0 15\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t9\t0.0\t2\t0 1 8\n",
      "125\t12\t0.0\t2\t0 0 12\n",
      "126\t8\t0.0\t2\t0 0 8\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "135\t5\t0.0\t2\t0 0 5\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t11\t0.0\t2\t0 0 11\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "197\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_113-LP3-ITS-0689_403_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_113-LP3-ITS-0689_403_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_113-LP3-ITS-0689_403_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_113-LP3-ITS-0689_403_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.245 s (13.229 ¬µs/read; 4.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,494\n",
      "Reads with adapters:                       519 (2.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,494 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,566,694 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,526,748 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 519 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.4%\n",
      "  C: 72.6%\n",
      "  G: 1.2%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t101\t289.0\t0\t101\n",
      "4\t12\t72.2\t0\t12\n",
      "5\t7\t18.1\t0\t7\n",
      "6\t1\t4.5\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t8\t0.0\t2\t0 1 7\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "38\t5\t0.0\t2\t0 0 5\n",
      "39\t2\t0.0\t2\t0 0 2\n",
      "46\t4\t0.0\t2\t0 1 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "64\t50\t0.0\t2\t0 0 50\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "67\t8\t0.0\t2\t0 0 8\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "74\t4\t0.0\t2\t0 0 4\n",
      "75\t14\t0.0\t2\t0 0 14\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "89\t11\t0.0\t2\t0 0 11\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "99\t94\t0.0\t2\t0 0 94\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t4\t0.0\t2\t0 0 4\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t20\t0.0\t2\t0 0 20\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t7\t0.0\t2\t0 0 7\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t23\t0.0\t2\t0 0 23\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t7\t0.0\t2\t0 1 6\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "161\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "168\t2\t0.0\t2\t0 0 2\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "257\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_114-LP3-ITS-0690_541_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_114-LP3-ITS-0690_541_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_114-LP3-ITS-0690_541_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_114-LP3-ITS-0690_541_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.187 s (13.045 ¬µs/read; 4.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,339\n",
      "Reads with adapters:                       233 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,339 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,316,039 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,293,574 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 233 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.2%\n",
      "  C: 24.5%\n",
      "  G: 0.4%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t224.0\t0\t6\n",
      "4\t1\t56.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "26\t4\t0.0\t2\t0 0 4\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t16\t0.0\t2\t0 0 16\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "58\t67\t0.0\t2\t0 0 67\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "163\t30\t0.0\t2\t0 1 29\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "189\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_115-LP3-ITS-0691_255_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_115-LP3-ITS-0691_255_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_115-LP3-ITS-0691_255_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_115-LP3-ITS-0691_255_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.264 s (12.919 ¬µs/read; 4.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,471\n",
      "Reads with adapters:                       523 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,161,771 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,123,653 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 523 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.9%\n",
      "  C: 54.9%\n",
      "  G: 16.1%\n",
      "  T: 5.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t102\t319.9\t0\t102\n",
      "4\t24\t80.0\t0\t24\n",
      "5\t29\t20.0\t0\t29\n",
      "6\t10\t5.0\t0\t10\n",
      "7\t4\t1.2\t0\t4\n",
      "8\t2\t0.3\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t6\t0.0\t1\t0 6\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "23\t4\t0.0\t2\t0 0 4\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t12\t0.0\t2\t0 0 12\n",
      "30\t3\t0.0\t2\t0 0 3\n",
      "31\t3\t0.0\t2\t0 0 3\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t3\t0.0\t2\t0 0 3\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t11\t0.0\t2\t0 0 11\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t9\t0.0\t2\t0 0 9\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t5\t0.0\t2\t0 0 5\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 1\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t15\t0.0\t2\t0 1 14\n",
      "107\t5\t0.0\t2\t0 0 5\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t6\t0.0\t2\t0 0 6\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t7\t0.0\t2\t0 0 7\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t4\t0.0\t2\t0 0 4\n",
      "132\t10\t0.0\t2\t0 0 10\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t5\t0.0\t2\t0 0 5\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t16\t0.0\t2\t0 0 16\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t7\t0.0\t2\t0 0 7\n",
      "150\t18\t0.0\t2\t0 0 18\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "156\t8\t0.0\t2\t0 0 8\n",
      "157\t10\t0.0\t2\t0 0 10\n",
      "158\t15\t0.0\t2\t0 0 15\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "170\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_116-LP3-ITS-0692_1009_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_116-LP3-ITS-0692_1009_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_116-LP3-ITS-0692_1009_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_116-LP3-ITS-0692_1009_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.192 s (13.815 ¬µs/read; 4.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,924\n",
      "Reads with adapters:                       257 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,924 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,191,124 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,164,302 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 257 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.2%\n",
      "  C: 50.6%\n",
      "  G: 3.1%\n",
      "  T: 10.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t217.6\t0\t11\n",
      "4\t4\t54.4\t0\t4\n",
      "5\t8\t13.6\t0\t8\n",
      "7\t1\t0.8\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t9\t0.0\t2\t0 0 9\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "65\t4\t0.0\t2\t0 0 4\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "94\t13\t0.0\t2\t0 0 13\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "99\t5\t0.0\t2\t0 0 5\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 1\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t24\t0.0\t2\t0 0 24\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t11\t0.0\t2\t0 1 10\n",
      "151\t13\t0.0\t2\t0 0 13\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t6\t0.0\t2\t0 0 6\n",
      "157\t14\t0.0\t2\t0 0 14\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "201\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_117-LP3-ITS-0693_886_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_117-LP3-ITS-0693_886_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_117-LP3-ITS-0693_886_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_117-LP3-ITS-0693_886_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.146 s (14.207 ¬µs/read; 4.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,261\n",
      "Reads with adapters:                       188 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        10,261 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,088,561 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,071,130 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 188 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.7%\n",
      "  C: 56.9%\n",
      "  G: 1.6%\n",
      "  T: 4.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t160.3\t0\t24\n",
      "4\t5\t40.1\t0\t5\n",
      "5\t8\t10.0\t0\t8\n",
      "6\t1\t2.5\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "41\t2\t0.0\t2\t0 0 2\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "48\t4\t0.0\t2\t0 0 4\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "136\t5\t0.0\t2\t0 0 5\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t10\t0.0\t2\t0 0 10\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "156\t6\t0.0\t2\t0 0 6\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "169\t2\t0.0\t2\t0 0 2\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "187\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_118-LP3-ITS-0694_248_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_118-LP3-ITS-0694_248_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_118-LP3-ITS-0694_248_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_118-LP3-ITS-0694_248_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.266 s (12.447 ¬µs/read; 4.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,361\n",
      "Reads with adapters:                       388 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,361 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,429,661 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,395,644 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 388 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.8%\n",
      "  C: 53.9%\n",
      "  G: 1.3%\n",
      "  T: 10.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t333.8\t0\t34\n",
      "4\t31\t83.4\t0\t31\n",
      "5\t11\t20.9\t0\t11\n",
      "6\t2\t5.2\t0\t2\n",
      "7\t2\t1.3\t0\t2\n",
      "11\t2\t0.0\t1\t0 2\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "33\t2\t0.0\t2\t0 0 2\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t12\t0.0\t2\t0 0 12\n",
      "39\t6\t0.0\t2\t0 0 6\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "49\t5\t0.0\t2\t0 0 5\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "58\t22\t0.0\t2\t0 1 21\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t3\t0.0\t2\t0 0 3\n",
      "71\t4\t0.0\t2\t0 0 4\n",
      "73\t10\t0.0\t2\t0 0 10\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "76\t9\t0.0\t2\t0 1 8\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "94\t11\t0.0\t2\t0 0 11\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "106\t3\t0.0\t2\t0 0 3\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "134\t3\t0.0\t2\t0 1 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "142\t17\t0.0\t2\t0 2 15\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t17\t0.0\t2\t0 0 17\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t4\t0.0\t2\t0 1 3\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t23\t0.0\t2\t0 0 23\n",
      "157\t11\t0.0\t2\t0 0 11\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "170\t2\t0.0\t2\t0 0 2\n",
      "257\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_119-LP3-ITS-0695_918_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_119-LP3-ITS-0695_918_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_119-LP3-ITS-0695_918_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_119-LP3-ITS-0695_918_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.297 s (11.806 ¬µs/read; 5.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,129\n",
      "Reads with adapters:                       409 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,129 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,563,829 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,518,358 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 409 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.1%\n",
      "  C: 25.9%\n",
      "  G: 0.7%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t392.6\t0\t4\n",
      "4\t6\t98.2\t0\t6\n",
      "11\t1\t0.0\t1\t0 1\n",
      "25\t3\t0.0\t2\t0 0 3\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "41\t4\t0.0\t2\t0 0 4\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "95\t20\t0.0\t2\t0 0 20\n",
      "96\t64\t0.0\t2\t0 3 61\n",
      "97\t16\t0.0\t2\t0 0 16\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t6\t0.0\t2\t0 0 6\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "106\t37\t0.0\t2\t0 0 37\n",
      "107\t7\t0.0\t2\t0 1 6\n",
      "108\t3\t0.0\t2\t0 0 3\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t22\t0.0\t2\t0 0 22\n",
      "111\t41\t0.0\t2\t0 0 41\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t7\t0.0\t2\t0 0 7\n",
      "117\t13\t0.0\t2\t0 0 13\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t11\t0.0\t2\t0 0 11\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t12\t0.0\t2\t0 0 12\n",
      "139\t45\t0.0\t2\t0 0 45\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "145\t24\t0.0\t2\t0 0 24\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t6\t0.0\t2\t0 0 6\n",
      "166\t2\t0.0\t2\t0 0 2\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "189\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_120-LP3-ITS-0696_823_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_120-LP3-ITS-0696_823_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_120-LP3-ITS-0696_823_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_120-LP3-ITS-0696_823_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.504 s (10.863 ¬µs/read; 5.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,382\n",
      "Reads with adapters:                       694 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,382 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,960,982 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,874,371 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 694 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.0%\n",
      "  C: 34.1%\n",
      "  G: 0.6%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t724.7\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t71\t0.0\t2\t0 0 71\n",
      "96\t25\t0.0\t2\t0 0 25\n",
      "97\t15\t0.0\t2\t0 1 14\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t8\t0.0\t2\t0 0 8\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t102\t0.0\t2\t0 1 101\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "123\t69\t0.0\t2\t0 2 67\n",
      "125\t10\t0.0\t2\t0 0 10\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t6\t0.0\t2\t0 0 6\n",
      "128\t134\t0.0\t2\t0 2 132\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t75\t0.0\t2\t0 0 75\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "147\t12\t0.0\t2\t0 0 12\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t6\t0.0\t2\t0 0 6\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t54\t0.0\t2\t0 0 54\n",
      "159\t12\t0.0\t2\t0 0 12\n",
      "163\t9\t0.0\t2\t0 0 9\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "165\t24\t0.0\t2\t0 1 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_121-LP3-ITS-0697_375_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_121-LP3-ITS-0697_375_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_121-LP3-ITS-0697_375_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_121-LP3-ITS-0697_375_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.313 s (11.438 ¬µs/read; 5.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,331\n",
      "Reads with adapters:                       395 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,331 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,226,631 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,179,547 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 395 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.5%\n",
      "  C: 15.9%\n",
      "  G: 4.8%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t427.0\t0\t8\n",
      "4\t4\t106.8\t0\t4\n",
      "25\t9\t0.0\t2\t0 0 9\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "96\t9\t0.0\t2\t0 0 9\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t20\t0.0\t2\t0 0 20\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 1\n",
      "107\t5\t0.0\t2\t0 1 4\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "115\t67\t0.0\t2\t0 0 67\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "118\t19\t0.0\t2\t0 0 19\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t5\t0.0\t2\t0 1 4\n",
      "123\t68\t0.0\t2\t0 0 68\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "130\t7\t0.0\t2\t0 0 7\n",
      "132\t18\t0.0\t2\t0 0 18\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t13\t0.0\t2\t0 0 13\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "146\t6\t0.0\t2\t0 0 6\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t22\t0.0\t2\t0 0 22\n",
      "152\t15\t0.0\t2\t0 0 15\n",
      "153\t33\t0.0\t2\t0 2 31\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "165\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_122-LP3-ITS-0698_705_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_122-LP3-ITS-0698_705_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_122-LP3-ITS-0698_705_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_122-LP3-ITS-0698_705_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.209 s (11.983 ¬µs/read; 5.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,437\n",
      "Reads with adapters:                       259 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,437 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,248,537 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,224,275 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 259 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.1%\n",
      "  C: 54.4%\n",
      "  G: 3.5%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t272.5\t0\t4\n",
      "4\t1\t68.1\t0\t1\n",
      "6\t4\t4.3\t0\t4\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "26\t6\t0.0\t2\t0 1 5\n",
      "27\t8\t0.0\t2\t0 0 8\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "58\t42\t0.0\t2\t0 3 39\n",
      "60\t20\t0.0\t2\t0 1 19\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t46\t0.0\t2\t0 1 45\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "103\t4\t0.0\t2\t0 1 3\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "115\t9\t0.0\t2\t0 0 9\n",
      "118\t13\t0.0\t2\t0 0 13\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "128\t9\t0.0\t2\t0 0 9\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t9\t0.0\t2\t0 0 9\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "158\t10\t0.0\t2\t0 0 10\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t28\t0.0\t2\t0 0 28\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_123-LP3-ITS-0699_353_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_123-LP3-ITS-0699_353_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_123-LP3-ITS-0699_353_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_123-LP3-ITS-0699_353_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.362 s (11.027 ¬µs/read; 5.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,794\n",
      "Reads with adapters:                       557 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,870,994 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,811,206 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 557 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.3%\n",
      "  C: 17.6%\n",
      "  G: 0.7%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t512.4\t0\t4\n",
      "4\t2\t128.1\t0\t2\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 1\n",
      "25\t59\t0.0\t2\t0 2 57\n",
      "26\t5\t0.0\t2\t0 0 5\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t47\t0.0\t2\t0 1 46\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t11\t0.0\t2\t0 0 11\n",
      "101\t24\t0.0\t2\t0 0 24\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t9\t0.0\t2\t0 0 9\n",
      "106\t7\t0.0\t2\t0 1 6\n",
      "107\t30\t0.0\t2\t0 0 30\n",
      "109\t13\t0.0\t2\t0 1 12\n",
      "111\t37\t0.0\t2\t0 0 37\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t2\t0.0\t2\t0 1 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t34\t0.0\t2\t0 1 33\n",
      "117\t15\t0.0\t2\t0 0 15\n",
      "118\t8\t0.0\t2\t0 0 8\n",
      "120\t27\t0.0\t2\t0 1 26\n",
      "122\t2\t0.0\t2\t0 0 2\n",
      "123\t73\t0.0\t2\t0 1 72\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "148\t8\t0.0\t2\t0 0 8\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "163\t53\t0.0\t2\t0 0 53\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "165\t13\t0.0\t2\t0 0 13\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_124-LP3-ITS-0700_602_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_124-LP3-ITS-0700_602_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_124-LP3-ITS-0700_602_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_124-LP3-ITS-0700_602_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.228 s (11.989 ¬µs/read; 5.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,036\n",
      "Reads with adapters:                       313 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,036 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,729,836 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,693,463 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 313 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.6%\n",
      "  C: 52.7%\n",
      "  G: 6.7%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t297.4\t0\t1\n",
      "4\t4\t74.4\t0\t4\n",
      "11\t5\t0.0\t1\t0 5\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "25\t4\t0.0\t2\t0 0 4\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "31\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "101\t30\t0.0\t2\t0 0 30\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "106\t25\t0.0\t2\t0 0 25\n",
      "107\t23\t0.0\t2\t0 1 22\n",
      "108\t44\t0.0\t2\t0 0 44\n",
      "109\t19\t0.0\t2\t0 0 19\n",
      "115\t15\t0.0\t2\t0 0 15\n",
      "118\t26\t0.0\t2\t0 1 25\n",
      "120\t16\t0.0\t2\t0 0 16\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t26\t0.0\t2\t0 0 26\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "165\t48\t0.0\t2\t0 0 48\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_125-LP3-ITS-0701_690_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_125-LP3-ITS-0701_690_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_125-LP3-ITS-0701_690_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_125-LP3-ITS-0701_690_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.201 s (11.065 ¬µs/read; 5.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,185\n",
      "Reads with adapters:                       279 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,185 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,473,685 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,442,869 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 279 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.9%\n",
      "  C: 20.1%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t284.1\t0\t3\n",
      "4\t4\t71.0\t0\t4\n",
      "25\t10\t0.0\t2\t0 0 10\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "55\t4\t0.0\t2\t0 0 4\n",
      "58\t12\t0.0\t2\t0 0 12\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t17\t0.0\t2\t0 1 16\n",
      "97\t24\t0.0\t2\t0 0 24\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "104\t11\t0.0\t2\t0 0 11\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "116\t10\t0.0\t2\t0 0 10\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t10\t0.0\t2\t0 0 10\n",
      "120\t18\t0.0\t2\t0 0 18\n",
      "123\t12\t0.0\t2\t0 0 12\n",
      "128\t40\t0.0\t2\t0 0 40\n",
      "129\t16\t0.0\t2\t0 0 16\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t5\t0.0\t2\t0 0 5\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t15\t0.0\t2\t0 0 15\n",
      "138\t10\t0.0\t2\t0 0 10\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t9\t0.0\t2\t0 0 9\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "163\t7\t0.0\t2\t0 0 7\n",
      "165\t4\t0.0\t2\t0 0 4\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_126-LP3-ITS-0702_962_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_126-LP3-ITS-0702_962_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_126-LP3-ITS-0702_962_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_126-LP3-ITS-0702_962_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.270 s (11.647 ¬µs/read; 5.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,147\n",
      "Reads with adapters:                       410 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,147 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,967,247 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,921,888 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 410 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.4%\n",
      "  C: 34.1%\n",
      "  G: 1.5%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t361.7\t0\t8\n",
      "4\t13\t90.4\t0\t13\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t9\t0.0\t2\t0 0 9\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "62\t6\t0.0\t2\t0 0 6\n",
      "63\t17\t0.0\t2\t0 2 15\n",
      "67\t11\t0.0\t2\t0 1 10\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t5\t0.0\t2\t0 0 5\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t8\t0.0\t2\t0 0 8\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t53\t0.0\t2\t0 2 51\n",
      "110\t8\t0.0\t2\t0 1 7\n",
      "112\t3\t0.0\t2\t0 0 3\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t6\t0.0\t2\t0 0 6\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t8\t0.0\t2\t0 0 8\n",
      "122\t2\t0.0\t2\t0 1 1\n",
      "123\t110\t0.0\t2\t0 5 105\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "137\t39\t0.0\t2\t0 0 39\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "152\t6\t0.0\t2\t0 1 5\n",
      "153\t16\t0.0\t2\t0 1 15\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "163\t23\t0.0\t2\t0 1 22\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_127-LP3-ITS-0703_595_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_127-LP3-ITS-0703_595_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_127-LP3-ITS-0703_595_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_127-LP3-ITS-0703_595_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.463 s (11.306 ¬µs/read; 5.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,988\n",
      "Reads with adapters:                       675 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,988 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,337,388 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,261,422 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 675 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.6%\n",
      "  C: 35.0%\n",
      "  G: 0.1%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t640.4\t0\t3\n",
      "4\t2\t160.1\t0\t2\n",
      "5\t4\t40.0\t0\t4\n",
      "6\t1\t10.0\t0\t1\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "31\t7\t0.0\t2\t0 0 7\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "40\t6\t0.0\t2\t0 0 6\n",
      "49\t18\t0.0\t2\t0 0 18\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "53\t4\t0.0\t2\t0 0 4\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t35\t0.0\t2\t0 3 32\n",
      "97\t9\t0.0\t2\t0 0 9\n",
      "98\t66\t0.0\t2\t0 0 66\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "102\t6\t0.0\t2\t0 0 6\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "105\t48\t0.0\t2\t0 0 48\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t63\t0.0\t2\t0 2 61\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t30\t0.0\t2\t0 0 30\n",
      "110\t10\t0.0\t2\t0 0 10\n",
      "111\t19\t0.0\t2\t0 0 19\n",
      "112\t8\t0.0\t2\t0 0 8\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t9\t0.0\t2\t0 0 9\n",
      "116\t16\t0.0\t2\t0 0 16\n",
      "117\t28\t0.0\t2\t0 0 28\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t17\t0.0\t2\t0 0 17\n",
      "130\t26\t0.0\t2\t0 0 26\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t59\t0.0\t2\t0 0 59\n",
      "135\t4\t0.0\t2\t0 1 3\n",
      "137\t15\t0.0\t2\t0 2 13\n",
      "139\t53\t0.0\t2\t0 1 52\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t8\t0.0\t2\t0 0 8\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "165\t3\t0.0\t2\t0 0 3\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_128-LP3-ITS-0704_20_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_128-LP3-ITS-0704_20_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_128-LP3-ITS-0704_20_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_128-LP3-ITS-0704_20_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.434 s (11.719 ¬µs/read; 5.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,015\n",
      "Reads with adapters:                       693 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,015 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,141,515 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,055,918 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 693 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.4%\n",
      "  C: 15.4%\n",
      "  G: 1.2%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t7\t144.6\t0\t7\n",
      "6\t2\t9.0\t0\t2\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 1 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "88\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "95\t11\t0.0\t2\t0 0 11\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "97\t30\t0.0\t2\t0 0 30\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t43\t0.0\t2\t0 1 42\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t10\t0.0\t2\t0 0 10\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t33\t0.0\t2\t0 0 33\n",
      "109\t40\t0.0\t2\t0 0 40\n",
      "110\t18\t0.0\t2\t0 0 18\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "112\t6\t0.0\t2\t0 1 5\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t38\t0.0\t2\t1 0 37\n",
      "124\t53\t0.0\t2\t0 2 51\n",
      "125\t54\t0.0\t2\t0 0 54\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t41\t0.0\t2\t0 0 41\n",
      "129\t14\t0.0\t2\t0 0 14\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "132\t8\t0.0\t2\t0 0 8\n",
      "133\t11\t0.0\t2\t0 1 10\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t75\t0.0\t2\t0 1 74\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "141\t6\t0.0\t2\t0 0 6\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t42\t0.0\t2\t0 1 41\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "153\t46\t0.0\t2\t0 1 45\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t19\t0.0\t2\t1 0 18\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "193\t4\t0.0\t2\t0 0 4\n",
      "215\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_129-LP3-ITS-0705_843_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_129-LP3-ITS-0705_843_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_129-LP3-ITS-0705_843_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_129-LP3-ITS-0705_843_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.465 s (11.807 ¬µs/read; 5.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,375\n",
      "Reads with adapters:                       677 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,851,875 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,771,799 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 677 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.2%\n",
      "  C: 16.8%\n",
      "  G: 1.3%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t615.2\t0\t5\n",
      "4\t1\t153.8\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t4\t0.0\t2\t0 0 4\n",
      "67\t5\t0.0\t2\t0 0 5\n",
      "75\t1\t0.0\t2\t0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "85\t4\t0.0\t2\t0 0 4\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t39\t0.0\t2\t0 0 39\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t89\t0.0\t2\t0 0 89\n",
      "99\t16\t0.0\t2\t0 0 16\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "102\t6\t0.0\t2\t0 0 6\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t16\t0.0\t2\t0 0 16\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t14\t0.0\t2\t0 1 13\n",
      "107\t11\t0.0\t2\t0 0 11\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t36\t0.0\t2\t0 1 35\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t8\t0.0\t2\t0 0 8\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t68\t0.0\t2\t0 0 68\n",
      "116\t28\t0.0\t2\t0 0 28\n",
      "117\t23\t0.0\t2\t0 0 23\n",
      "118\t9\t0.0\t2\t0 0 9\n",
      "120\t20\t0.0\t2\t0 0 20\n",
      "122\t4\t0.0\t2\t0 0 4\n",
      "123\t12\t0.0\t2\t0 0 12\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "128\t24\t0.0\t2\t0 0 24\n",
      "129\t7\t0.0\t2\t0 0 7\n",
      "130\t5\t0.0\t2\t0 0 5\n",
      "131\t10\t0.0\t2\t0 0 10\n",
      "132\t9\t0.0\t2\t0 0 9\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t13\t0.0\t2\t0 0 13\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t7\t0.0\t2\t0 0 7\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t74\t0.0\t2\t0 2 72\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "179\t2\t0.0\t2\t0 0 2\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_130-LP3-ITS-0706_191_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_130-LP3-ITS-0706_191_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_130-LP3-ITS-0706_191_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_130-LP3-ITS-0706_191_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.013 s (17.117 ¬µs/read; 3.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     758\n",
      "Reads with adapters:                        10 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           758 (100.0%)\n",
      "\n",
      "Total basepairs processed:       228,158 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        227,150 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 30.0%\n",
      "  C: 30.0%\n",
      "  G: 0.0%\n",
      "  T: 40.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_131-LP3-ITS-0707_317_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_131-LP3-ITS-0707_317_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_131-LP3-ITS-0707_317_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_131-LP3-ITS-0707_317_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.300 s (10.603 ¬µs/read; 5.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,291\n",
      "Reads with adapters:                       250 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,291 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,515,591 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,486,887 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 250 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 9.6%\n",
      "  C: 90.0%\n",
      "  G: 0.4%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t442.0\t0\t1\n",
      "4\t1\t110.5\t0\t1\n",
      "31\t18\t0.0\t2\t0 0 18\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "84\t50\t0.0\t2\t0 0 50\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "118\t21\t0.0\t2\t0 0 21\n",
      "133\t142\t0.0\t2\t0 3 139\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "158\t13\t0.0\t2\t0 0 13\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_132-LP3-ITS-0708_458_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_132-LP3-ITS-0708_458_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_132-LP3-ITS-0708_458_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_132-LP3-ITS-0708_458_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (49.252 ¬µs/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     135\n",
      "Reads with adapters:                         4 (3.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           135 (100.0%)\n",
      "\n",
      "Total basepairs processed:        40,635 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         40,160 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t0.0\t0\t1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_133-LP3-ITS-0709_586_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_133-LP3-ITS-0709_586_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_133-LP3-ITS-0709_586_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_133-LP3-ITS-0709_586_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (185.233 ¬µs/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      26\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            26 (100.0%)\n",
      "\n",
      "Total basepairs processed:         7,826 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):          7,826 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_134-LP3-ITS-0710_1085_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_134-LP3-ITS-0710_1085_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_134-LP3-ITS-0710_1085_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_134-LP3-ITS-0710_1085_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.075 s (10.797 ¬µs/read; 5.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,908\n",
      "Reads with adapters:                        74 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         6,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,079,308 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,071,224 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 74 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.9%\n",
      "  C: 29.7%\n",
      "  G: 0.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t107.9\t0\t2\n",
      "98\t5\t0.0\t2\t0 0 5\n",
      "105\t30\t0.0\t2\t0 0 30\n",
      "106\t13\t0.0\t2\t0 0 13\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "116\t13\t0.0\t2\t0 0 13\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "139\t7\t0.0\t2\t0 0 7\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_135-LP3-ITS-0711_86_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_135-LP3-ITS-0711_86_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_135-LP3-ITS-0711_86_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_135-LP3-ITS-0711_86_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (36.478 ¬µs/read; 1.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     180\n",
      "Reads with adapters:                         2 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           180 (100.0%)\n",
      "\n",
      "Total basepairs processed:        54,180 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         53,896 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_136-LP3-ITS-0712_584_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_136-LP3-ITS-0712_584_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_136-LP3-ITS-0712_584_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_136-LP3-ITS-0712_584_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.011 s (19.715 ¬µs/read; 3.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     537\n",
      "Reads with adapters:                         9 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           537 (100.0%)\n",
      "\n",
      "Total basepairs processed:       161,637 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        160,547 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.8%\n",
      "  C: 11.1%\n",
      "  G: 0.0%\n",
      "  T: 11.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "129\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_137-LP3-ITS-0713_304_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_137-LP3-ITS-0713_304_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_137-LP3-ITS-0713_304_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_137-LP3-ITS-0713_304_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (21.703 ¬µs/read; 2.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     414\n",
      "Reads with adapters:                         5 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           414 (100.0%)\n",
      "\n",
      "Total basepairs processed:       124,614 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        124,178 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.0%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t0.1\t0\t1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_138-LP3-ITS-0714_771_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_138-LP3-ITS-0714_771_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_138-LP3-ITS-0714_771_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_138-LP3-ITS-0714_771_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (24.868 ¬µs/read; 2.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     331\n",
      "Reads with adapters:                         8 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           331 (100.0%)\n",
      "\n",
      "Total basepairs processed:        99,631 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         99,315 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 62.5%\n",
      "  C: 25.0%\n",
      "  G: 0.0%\n",
      "  T: 12.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t5.2\t0\t4\n",
      "4\t1\t1.3\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_139-LP3-ITS-0715_435_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_139-LP3-ITS-0715_435_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_139-LP3-ITS-0715_435_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_139-LP3-ITS-0715_435_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (21.676 ¬µs/read; 2.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     407\n",
      "Reads with adapters:                         2 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           407 (100.0%)\n",
      "\n",
      "Total basepairs processed:       122,507 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        122,261 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_140-LP3-ITS-0716_578_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_140-LP3-ITS-0716_578_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_140-LP3-ITS-0716_578_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_140-LP3-ITS-0716_578_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (54.443 ¬µs/read; 1.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     106\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           106 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,906 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         31,906 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_141-LP3-ITS-0717_395_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_141-LP3-ITS-0717_395_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_141-LP3-ITS-0717_395_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_141-LP3-ITS-0717_395_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (100.393 ¬µs/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      51\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            51 (100.0%)\n",
      "\n",
      "Total basepairs processed:        15,351 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         15,351 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_142-LP3-ITS-0718_104_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_142-LP3-ITS-0718_104_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_142-LP3-ITS-0718_104_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_142-LP3-ITS-0718_104_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (28.309 ¬µs/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     259\n",
      "Reads with adapters:                         3 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           259 (100.0%)\n",
      "\n",
      "Total basepairs processed:        77,959 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         77,644 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_143-LP3-ITS-0719_254_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_143-LP3-ITS-0719_254_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_143-LP3-ITS-0719_254_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_143-LP3-ITS-0719_254_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (51.160 ¬µs/read; 1.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     112\n",
      "Reads with adapters:                         3 (2.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           112 (100.0%)\n",
      "\n",
      "Total basepairs processed:        33,712 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         33,573 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 33.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.8\t0\t2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_144-LP3-ITS-0720_322_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_144-LP3-ITS-0720_322_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_144-LP3-ITS-0720_322_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_144-LP3-ITS-0720_322_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.225 s (10.992 ¬µs/read; 5.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,453\n",
      "Reads with adapters:                       175 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,453 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,156,353 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,135,431 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 175 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 16.6%\n",
      "  C: 76.0%\n",
      "  G: 0.0%\n",
      "  T: 7.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t319.6\t0\t4\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "81\t11\t0.0\t2\t0 0 11\n",
      "93\t17\t0.0\t2\t0 0 17\n",
      "103\t10\t0.0\t2\t0 0 10\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "133\t79\t0.0\t2\t0 0 79\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_145-LP3-ITS-0721_209_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_145-LP3-ITS-0721_209_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_145-LP3-ITS-0721_209_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_145-LP3-ITS-0721_209_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (39.887 ¬µs/read; 1.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     168\n",
      "Reads with adapters:                         3 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           168 (100.0%)\n",
      "\n",
      "Total basepairs processed:        50,568 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         50,362 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.6\t0\t1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_146-LP3-ITS-0722_220_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_146-LP3-ITS-0722_220_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_146-LP3-ITS-0722_220_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_146-LP3-ITS-0722_220_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (38.776 ¬µs/read; 1.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     161\n",
      "Reads with adapters:                         4 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           161 (100.0%)\n",
      "\n",
      "Total basepairs processed:        48,461 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         48,078 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.0%\n",
      "  C: 75.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t0.6\t0\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_147-LP3-ITS-0723_536_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_147-LP3-ITS-0723_536_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_147-LP3-ITS-0723_536_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_147-LP3-ITS-0723_536_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (63.742 ¬µs/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      85\n",
      "Reads with adapters:                         4 (4.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            85 (100.0%)\n",
      "\n",
      "Total basepairs processed:        25,585 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         25,292 bp (98.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_148-LP3-ITS-0724_768_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_148-LP3-ITS-0724_768_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_148-LP3-ITS-0724_768_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_148-LP3-ITS-0724_768_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (87.046 ¬µs/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      61\n",
      "Reads with adapters:                         3 (4.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            61 (100.0%)\n",
      "\n",
      "Total basepairs processed:        18,361 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         18,221 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 33.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.0\t0\t2\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_149-LP3-ITS-0725_1049_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_149-LP3-ITS-0725_1049_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_149-LP3-ITS-0725_1049_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_149-LP3-ITS-0725_1049_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (79.198 ¬µs/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      66\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            66 (100.0%)\n",
      "\n",
      "Total basepairs processed:        19,866 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         19,866 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_150-LP3-ITS-0726_145_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_150-LP3-ITS-0726_145_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_150-LP3-ITS-0726_145_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_150-LP3-ITS-0726_145_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (49.158 ¬µs/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     121\n",
      "Reads with adapters:                         2 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           121 (100.0%)\n",
      "\n",
      "Total basepairs processed:        36,421 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         36,356 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_151-LP3-ITS-0727_710_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_151-LP3-ITS-0727_710_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_151-LP3-ITS-0727_710_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_151-LP3-ITS-0727_710_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.011 s (19.777 ¬µs/read; 3.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     538\n",
      "Reads with adapters:                        17 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           538 (100.0%)\n",
      "\n",
      "Total basepairs processed:       161,938 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        160,432 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.6%\n",
      "  C: 23.5%\n",
      "  G: 0.0%\n",
      "  T: 5.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t8.4\t0\t4\n",
      "10\t3\t0.0\t1\t1 2\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "209\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_152-LP3-ITS-0728_872_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_152-LP3-ITS-0728_872_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_152-LP3-ITS-0728_872_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_152-LP3-ITS-0728_872_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (35.720 ¬µs/read; 1.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     178\n",
      "Reads with adapters:                         5 (2.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           178 (100.0%)\n",
      "\n",
      "Total basepairs processed:        53,578 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         53,152 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.0%\n",
      "  C: 40.0%\n",
      "  G: 0.0%\n",
      "  T: 20.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2.8\t0\t2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_153-LP3-ITS-0729_277_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_153-LP3-ITS-0729_277_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_153-LP3-ITS-0729_277_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_153-LP3-ITS-0729_277_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.200 s (9.201 ¬µs/read; 6.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,713\n",
      "Reads with adapters:                       157 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,713 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,535,613 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,506,827 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 157 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.9%\n",
      "  C: 10.8%\n",
      "  G: 0.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t339.3\t0\t4\n",
      "4\t7\t84.8\t0\t7\n",
      "6\t7\t5.3\t0\t7\n",
      "10\t1\t0.0\t1\t0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "209\t137\t0.0\t2\t0 1 136\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_154-LP3-ITS-0730_4_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_154-LP3-ITS-0730_4_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_154-LP3-ITS-0730_4_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_154-LP3-ITS-0730_4_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.115 s (7.802 ¬µs/read; 7.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,685\n",
      "Reads with adapters:                        67 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,685 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,420,185 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,410,086 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 67 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.1%\n",
      "  C: 26.9%\n",
      "  G: 1.5%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t9\t57.4\t0\t9\n",
      "6\t9\t3.6\t0\t9\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "209\t46\t0.0\t2\t0 1 45\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_155-LP3-ITS-0731_764_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_155-LP3-ITS-0731_764_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_155-LP3-ITS-0731_764_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_155-LP3-ITS-0731_764_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.118 s (7.111 ¬µs/read; 8.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,632\n",
      "Reads with adapters:                        47 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,632 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,006,232 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,005,769 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 47 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.1%\n",
      "  C: 72.3%\n",
      "  G: 4.3%\n",
      "  T: 4.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t259.9\t0\t8\n",
      "4\t15\t65.0\t0\t15\n",
      "5\t4\t16.2\t0\t4\n",
      "6\t16\t4.1\t0\t16\n",
      "7\t1\t1.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_156-LP3-ITS-0732_670_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_156-LP3-ITS-0732_670_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_156-LP3-ITS-0732_670_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_156-LP3-ITS-0732_670_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.184 s (7.574 ¬µs/read; 7.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,329\n",
      "Reads with adapters:                       254 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,323,029 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,313,164 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 254 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.3%\n",
      "  C: 59.1%\n",
      "  G: 2.0%\n",
      "  T: 10.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t147\t380.1\t0\t147\n",
      "4\t19\t95.0\t0\t19\n",
      "5\t1\t23.8\t0\t1\n",
      "6\t21\t5.9\t0\t21\n",
      "10\t23\t0.0\t1\t1 22\n",
      "209\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_157-LP3-ITS-0733_62_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_157-LP3-ITS-0733_62_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_157-LP3-ITS-0733_62_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_157-LP3-ITS-0733_62_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.230 s (9.145 ¬µs/read; 6.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,102\n",
      "Reads with adapters:                       139 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,555,702 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,530,528 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 139 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.8%\n",
      "  C: 11.5%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t392.2\t0\t3\n",
      "4\t5\t98.1\t0\t5\n",
      "5\t1\t24.5\t0\t1\n",
      "6\t10\t6.1\t0\t10\n",
      "209\t120\t0.0\t2\t0 1 119\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_158-LP3-ITS-0734_754_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_158-LP3-ITS-0734_754_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_158-LP3-ITS-0734_754_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_158-LP3-ITS-0734_754_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.267 s (9.208 ¬µs/read; 6.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,976\n",
      "Reads with adapters:                       211 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,976 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,721,776 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,682,774 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 211 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.6%\n",
      "  C: 10.4%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t452.8\t0\t6\n",
      "4\t4\t113.2\t0\t4\n",
      "5\t1\t28.3\t0\t1\n",
      "6\t13\t7.1\t0\t13\n",
      "10\t1\t0.0\t1\t0 1\n",
      "209\t185\t0.0\t2\t0 4 181\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_159-LP3-ITS-0735_288_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_159-LP3-ITS-0735_288_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_159-LP3-ITS-0735_288_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_159-LP3-ITS-0735_288_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.072 s (8.045 ¬µs/read; 7.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   9,008\n",
      "Reads with adapters:                        36 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         9,008 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,711,408 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,707,970 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 36 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t140.8\t0\t1\n",
      "4\t11\t35.2\t0\t11\n",
      "5\t1\t8.8\t0\t1\n",
      "6\t7\t2.2\t0\t7\n",
      "209\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_160-LP3-ITS-0736_1072_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_160-LP3-ITS-0736_1072_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_160-LP3-ITS-0736_1072_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_160-LP3-ITS-0736_1072_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.230 s (8.507 ¬µs/read; 7.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,081\n",
      "Reads with adapters:                       371 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,081 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,151,381 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,136,908 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 371 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.8%\n",
      "  C: 42.9%\n",
      "  G: 0.8%\n",
      "  T: 21.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t210\t423.1\t0\t210\n",
      "4\t15\t105.8\t0\t15\n",
      "5\t4\t26.4\t0\t4\n",
      "6\t21\t6.6\t0\t21\n",
      "7\t2\t1.7\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t53\t0.0\t1\t2 51\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "209\t60\t0.0\t2\t0 1 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_161-LP3-ITS-0737_1141_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_161-LP3-ITS-0737_1141_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_161-LP3-ITS-0737_1141_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_161-LP3-ITS-0737_1141_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.110 s (8.546 ¬µs/read; 7.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,864\n",
      "Reads with adapters:                        34 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,864 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,872,064 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,871,387 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 34 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 14.7%\n",
      "  C: 76.5%\n",
      "  G: 2.9%\n",
      "  T: 5.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t201.0\t0\t8\n",
      "4\t12\t50.2\t0\t12\n",
      "5\t1\t12.6\t0\t1\n",
      "6\t9\t3.1\t0\t9\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "209\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_162-LP3-ITS-0738_596_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_162-LP3-ITS-0738_596_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_162-LP3-ITS-0738_596_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_162-LP3-ITS-0738_596_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.089 s (7.916 ¬µs/read; 7.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,220\n",
      "Reads with adapters:                        75 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,220 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,377,220 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,371,145 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 75 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 54.7%\n",
      "  C: 34.7%\n",
      "  G: 0.0%\n",
      "  T: 10.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t175.3\t0\t23\n",
      "4\t6\t43.8\t0\t6\n",
      "5\t4\t11.0\t0\t4\n",
      "6\t7\t2.7\t0\t7\n",
      "7\t1\t0.7\t0\t1\n",
      "10\t6\t0.0\t1\t2 4\n",
      "209\t27\t0.0\t2\t0 0 27\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_163-LP3-ITS-0739_156_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_163-LP3-ITS-0739_156_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_163-LP3-ITS-0739_156_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_163-LP3-ITS-0739_156_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.125 s (9.036 ¬µs/read; 6.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,834\n",
      "Reads with adapters:                        96 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,834 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,164,034 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,146,151 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 96 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.6%\n",
      "  C: 8.3%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t216.2\t0\t2\n",
      "4\t2\t54.0\t0\t2\n",
      "6\t5\t3.4\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "206\t8\t0.0\t2\t0 0 8\n",
      "209\t76\t0.0\t2\t0 1 75\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_164-LP3-ITS-0740_1034_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_164-LP3-ITS-0740_1034_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_164-LP3-ITS-0740_1034_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_164-LP3-ITS-0740_1034_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.123 s (7.002 ¬µs/read; 8.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,585\n",
      "Reads with adapters:                       103 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,585 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,293,085 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,291,595 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 103 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 16.5%\n",
      "  C: 70.9%\n",
      "  G: 1.9%\n",
      "  T: 10.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t44\t274.8\t0\t44\n",
      "4\t19\t68.7\t0\t19\n",
      "5\t2\t17.2\t0\t2\n",
      "6\t25\t4.3\t0\t25\n",
      "7\t1\t1.1\t0\t1\n",
      "10\t7\t0.0\t1\t1 6\n",
      "209\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_165-LP3-ITS-0741_5_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_165-LP3-ITS-0741_5_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_165-LP3-ITS-0741_5_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_165-LP3-ITS-0741_5_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.213 s (9.648 ¬µs/read; 6.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,118\n",
      "Reads with adapters:                       172 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,118 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,657,518 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,623,481 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 172 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 94.8%\n",
      "  C: 4.1%\n",
      "  G: 0.0%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t345.6\t0\t2\n",
      "4\t3\t86.4\t0\t3\n",
      "5\t1\t21.6\t0\t1\n",
      "6\t3\t5.4\t0\t3\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "209\t160\t0.0\t2\t0 1 159\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_166-LP3-ITS-0742_848_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_166-LP3-ITS-0742_848_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_166-LP3-ITS-0742_848_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_166-LP3-ITS-0742_848_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.132 s (7.562 ¬µs/read; 7.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,517\n",
      "Reads with adapters:                        58 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,517 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,272,617 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,267,221 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 58 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 51.7%\n",
      "  C: 48.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t273.7\t0\t3\n",
      "4\t12\t68.4\t0\t12\n",
      "5\t2\t17.1\t0\t2\n",
      "6\t14\t4.3\t0\t14\n",
      "10\t2\t0.0\t1\t0 2\n",
      "209\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_167-LP3-ITS-0743_509_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_167-LP3-ITS-0743_509_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_167-LP3-ITS-0743_509_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_167-LP3-ITS-0743_509_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (53.382 ¬µs/read; 1.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     118\n",
      "Reads with adapters:                         4 (3.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           118 (100.0%)\n",
      "\n",
      "Total basepairs processed:        35,518 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         35,237 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 50.0%\n",
      "  G: 25.0%\n",
      "  T: 25.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.8\t0\t2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_168-LP3-ITS-0744_1020_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_168-LP3-ITS-0744_1020_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_168-LP3-ITS-0744_1020_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_168-LP3-ITS-0744_1020_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.229 s (9.441 ¬µs/read; 6.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,207\n",
      "Reads with adapters:                       760 (3.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,207 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,286,307 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,261,278 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 760 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 40.7%\n",
      "  G: 1.4%\n",
      "  T: 24.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t494\t378.2\t0\t494\n",
      "4\t8\t94.6\t0\t8\n",
      "5\t3\t23.6\t0\t3\n",
      "6\t17\t5.9\t0\t17\n",
      "7\t2\t1.5\t0\t2\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t22\t0.1\t0\t20 2\n",
      "10\t106\t0.0\t1\t10 96\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "209\t104\t0.0\t2\t0 1 103\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_169-LP3-ITS-0745_725_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_169-LP3-ITS-0745_725_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_169-LP3-ITS-0745_725_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_169-LP3-ITS-0745_725_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.259 s (8.337 ¬µs/read; 7.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,096\n",
      "Reads with adapters:                       335 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,096 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,359,896 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,321,876 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 335 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 62.1%\n",
      "  C: 23.3%\n",
      "  G: 0.3%\n",
      "  T: 14.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t105\t485.9\t0\t105\n",
      "4\t7\t121.5\t0\t7\n",
      "5\t2\t30.4\t0\t2\n",
      "6\t14\t7.6\t0\t14\n",
      "7\t1\t1.9\t0\t1\n",
      "9\t5\t0.1\t0\t5\n",
      "10\t22\t0.0\t1\t2 20\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "209\t177\t0.0\t2\t0 2 175\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_170-LP3-ITS-0746_1060_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_170-LP3-ITS-0746_1060_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_170-LP3-ITS-0746_1060_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_170-LP3-ITS-0746_1060_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.123 s (9.402 ¬µs/read; 6.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,068\n",
      "Reads with adapters:                       146 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,068 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,933,468 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,902,954 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 146 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 97.9%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 2.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "209\t146\t0.0\t2\t0 1 145\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_171-LP3-ITS-0747_97_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_171-LP3-ITS-0747_97_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_171-LP3-ITS-0747_97_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_171-LP3-ITS-0747_97_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.181 s (7.496 ¬µs/read; 8.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,146\n",
      "Reads with adapters:                        40 (0.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,267,946 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,267,498 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 40 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 12.5%\n",
      "  C: 87.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t377.3\t0\t1\n",
      "4\t11\t94.3\t0\t11\n",
      "5\t2\t23.6\t0\t2\n",
      "6\t24\t5.9\t0\t24\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_172-LP3-ITS-0748_164_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_172-LP3-ITS-0748_164_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_172-LP3-ITS-0748_164_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_172-LP3-ITS-0748_164_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.200 s (9.240 ¬µs/read; 6.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,682\n",
      "Reads with adapters:                       208 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,682 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,526,282 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,486,532 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 208 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.3%\n",
      "  C: 3.8%\n",
      "  G: 0.5%\n",
      "  T: 3.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t338.8\t0\t11\n",
      "6\t1\t5.3\t0\t1\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t4\t0.0\t1\t0 4\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "209\t189\t0.0\t2\t0 2 187\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_173-LP3-ITS-0749_1018_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_173-LP3-ITS-0749_1018_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_173-LP3-ITS-0749_1018_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_173-LP3-ITS-0749_1018_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.134 s (8.038 ¬µs/read; 7.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,693\n",
      "Reads with adapters:                        47 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,693 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,024,593 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,024,006 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 47 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.3%\n",
      "  C: 72.3%\n",
      "  G: 0.0%\n",
      "  T: 6.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t260.8\t0\t6\n",
      "4\t11\t65.2\t0\t11\n",
      "5\t6\t16.3\t0\t6\n",
      "6\t20\t4.1\t0\t20\n",
      "10\t1\t0.0\t1\t0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_174-LP3-ITS-0750_376_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_174-LP3-ITS-0750_376_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_174-LP3-ITS-0750_376_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_174-LP3-ITS-0750_376_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.148 s (8.665 ¬µs/read; 6.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,101\n",
      "Reads with adapters:                        70 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,101 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,147,401 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,140,917 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 70 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 55.7%\n",
      "  C: 42.9%\n",
      "  G: 0.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t267.2\t0\t3\n",
      "4\t15\t66.8\t0\t15\n",
      "5\t5\t16.7\t0\t5\n",
      "6\t15\t4.2\t0\t15\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "209\t30\t0.0\t2\t0 1 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_175-LP3-ITS-0751_168_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_175-LP3-ITS-0751_168_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_175-LP3-ITS-0751_168_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_175-LP3-ITS-0751_168_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.191 s (7.544 ¬µs/read; 7.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,348\n",
      "Reads with adapters:                       108 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,348 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,629,748 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,613,560 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 108 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 74.1%\n",
      "  C: 25.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t396.1\t0\t3\n",
      "4\t8\t99.0\t0\t8\n",
      "6\t19\t6.2\t0\t19\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "209\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_176-LP3-ITS-0752_127_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_176-LP3-ITS-0752_127_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_176-LP3-ITS-0752_127_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_176-LP3-ITS-0752_127_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.216 s (6.786 ¬µs/read; 8.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,893\n",
      "Reads with adapters:                        64 (0.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,893 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,599,793 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,599,151 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 64 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 17.2%\n",
      "  C: 79.7%\n",
      "  G: 1.6%\n",
      "  T: 1.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t498.3\t0\t1\n",
      "4\t20\t124.6\t0\t20\n",
      "5\t5\t31.1\t0\t5\n",
      "6\t32\t7.8\t0\t32\n",
      "7\t1\t1.9\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_177-LP3-ITS-0753_983_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_177-LP3-ITS-0753_983_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_177-LP3-ITS-0753_983_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_177-LP3-ITS-0753_983_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.206 s (8.556 ¬µs/read; 7.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,048\n",
      "Reads with adapters:                       122 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,048 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,238,448 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,223,005 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 122 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.8%\n",
      "  C: 23.0%\n",
      "  G: 0.0%\n",
      "  T: 3.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t375.8\t0\t11\n",
      "4\t10\t93.9\t0\t10\n",
      "5\t6\t23.5\t0\t6\n",
      "6\t13\t5.9\t0\t13\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t6\t0.0\t1\t0 6\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "209\t70\t0.0\t2\t0 1 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_178-LP3-ITS-0754_775_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_178-LP3-ITS-0754_775_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_178-LP3-ITS-0754_775_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_178-LP3-ITS-0754_775_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.157 s (9.597 ¬µs/read; 6.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,400\n",
      "Reads with adapters:                       685 (4.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,400 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,936,400 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,924,477 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 685 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.8%\n",
      "  C: 55.9%\n",
      "  G: 0.9%\n",
      "  T: 18.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t500\t256.2\t0\t500\n",
      "4\t11\t64.1\t0\t11\n",
      "5\t1\t16.0\t0\t1\n",
      "6\t10\t4.0\t0\t10\n",
      "7\t1\t1.0\t0\t1\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t14\t0.1\t0\t14\n",
      "10\t101\t0.0\t1\t7 94\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "175\t4\t0.0\t2\t0 0 4\n",
      "209\t36\t0.0\t2\t0 1 35\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "219\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_179-LP3-ITS-0755_1108_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_179-LP3-ITS-0755_1108_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_179-LP3-ITS-0755_1108_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_179-LP3-ITS-0755_1108_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.299 s (7.892 ¬µs/read; 7.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,883\n",
      "Reads with adapters:                       950 (2.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,883 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,402,783 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,384,921 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 950 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.5%\n",
      "  C: 48.1%\n",
      "  G: 1.9%\n",
      "  T: 22.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t649\t591.9\t0\t649\n",
      "4\t24\t148.0\t0\t24\n",
      "5\t7\t37.0\t0\t7\n",
      "6\t36\t9.2\t0\t36\n",
      "7\t1\t2.3\t0\t1\n",
      "9\t13\t0.1\t0\t13\n",
      "10\t151\t0.0\t1\t8 143\n",
      "11\t2\t0.0\t1\t0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "209\t66\t0.0\t2\t0 3 63\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_180-LP3-ITS-0756_483_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_180-LP3-ITS-0756_483_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_180-LP3-ITS-0756_483_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_180-LP3-ITS-0756_483_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.116 s (7.843 ¬µs/read; 7.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,851\n",
      "Reads with adapters:                        33 (0.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,851 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,470,151 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,469,475 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 33 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 6.1%\n",
      "  C: 93.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t15\t58.0\t0\t15\n",
      "6\t15\t3.6\t0\t15\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_181-LP3-ITS-0757_19_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_181-LP3-ITS-0757_19_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_181-LP3-ITS-0757_19_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_181-LP3-ITS-0757_19_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.172 s (8.741 ¬µs/read; 6.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,707\n",
      "Reads with adapters:                        94 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,707 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,931,807 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,918,887 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 94 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.2%\n",
      "  C: 29.8%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t12\t77.0\t0\t12\n",
      "5\t2\t19.2\t0\t2\n",
      "6\t17\t4.8\t0\t17\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "209\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_182-LP3-ITS-0758_385_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_182-LP3-ITS-0758_385_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_182-LP3-ITS-0758_385_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_182-LP3-ITS-0758_385_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.242 s (8.538 ¬µs/read; 7.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,298\n",
      "Reads with adapters:                       801 (2.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,298 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,517,698 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,503,636 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 801 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.7%\n",
      "  C: 48.9%\n",
      "  G: 1.4%\n",
      "  T: 18.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t515\t442.2\t0\t515\n",
      "4\t17\t110.5\t0\t17\n",
      "5\t2\t27.6\t0\t2\n",
      "6\t31\t6.9\t0\t31\n",
      "7\t1\t1.7\t0\t1\n",
      "8\t3\t0.4\t0\t3\n",
      "9\t31\t0.1\t0\t29 2\n",
      "10\t147\t0.0\t1\t10 137\n",
      "11\t2\t0.0\t1\t0 2\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "209\t47\t0.0\t2\t0 0 47\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_183-LP3-ITS-0759_680_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_183-LP3-ITS-0759_680_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_183-LP3-ITS-0759_680_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_183-LP3-ITS-0759_680_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.128 s (8.540 ¬µs/read; 7.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,931\n",
      "Reads with adapters:                        69 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,931 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,494,231 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,487,606 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 69 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 59.4%\n",
      "  C: 37.7%\n",
      "  G: 1.4%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t233.3\t0\t8\n",
      "4\t9\t58.3\t0\t9\n",
      "5\t2\t14.6\t0\t2\n",
      "6\t15\t3.6\t0\t15\n",
      "7\t1\t0.9\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "209\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_184-LP3-ITS-0760_71_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_184-LP3-ITS-0760_71_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_184-LP3-ITS-0760_71_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_184-LP3-ITS-0760_71_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.220 s (8.045 ¬µs/read; 7.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,380\n",
      "Reads with adapters:                       877 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,241,380 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,231,446 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 877 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.6%\n",
      "  C: 50.5%\n",
      "  G: 1.5%\n",
      "  T: 27.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t684\t427.8\t0\t684\n",
      "4\t9\t107.0\t0\t9\n",
      "6\t16\t6.7\t0\t16\n",
      "9\t9\t0.1\t0\t8 1\n",
      "10\t127\t0.0\t1\t8 119\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "209\t30\t0.0\t2\t0 1 29\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_185-LP3-ITS-0761_1122_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_185-LP3-ITS-0761_1122_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_185-LP3-ITS-0761_1122_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_185-LP3-ITS-0761_1122_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.142 s (9.039 ¬µs/read; 6.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,724\n",
      "Reads with adapters:                       255 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,732,924 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,722,497 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 255 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.7%\n",
      "  C: 45.1%\n",
      "  G: 0.8%\n",
      "  T: 18.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t141\t245.7\t0\t141\n",
      "4\t15\t61.4\t0\t15\n",
      "5\t4\t15.4\t0\t4\n",
      "6\t9\t3.8\t0\t9\n",
      "7\t1\t1.0\t0\t1\n",
      "8\t3\t0.2\t0\t3\n",
      "9\t7\t0.1\t0\t6 1\n",
      "10\t29\t0.0\t1\t4 25\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "209\t43\t0.0\t2\t0 2 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_186-LP3-ITS-0762_600_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_186-LP3-ITS-0762_600_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_186-LP3-ITS-0762_600_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_186-LP3-ITS-0762_600_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.106 s (6.763 ¬µs/read; 8.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,675\n",
      "Reads with adapters:                        30 (0.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,675 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,718,175 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,717,824 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 30 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 13.3%\n",
      "  C: 86.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t244.9\t0\t7\n",
      "4\t5\t61.2\t0\t5\n",
      "5\t2\t15.3\t0\t2\n",
      "6\t14\t3.8\t0\t14\n",
      "7\t1\t1.0\t0\t1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_187-LP3-ITS-0763_1011_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_187-LP3-ITS-0763_1011_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_187-LP3-ITS-0763_1011_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_187-LP3-ITS-0763_1011_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.239 s (8.958 ¬µs/read; 6.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,640\n",
      "Reads with adapters:                       185 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,640 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,018,640 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,985,955 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 185 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.9%\n",
      "  C: 13.0%\n",
      "  G: 0.0%\n",
      "  T: 2.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t416.2\t0\t2\n",
      "4\t12\t104.1\t0\t12\n",
      "5\t1\t26.0\t0\t1\n",
      "6\t13\t6.5\t0\t13\n",
      "9\t1\t0.1\t0\t0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "209\t155\t0.0\t2\t0 4 151\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_188-LP3-ITS-0764_930_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_188-LP3-ITS-0764_930_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_188-LP3-ITS-0764_930_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_188-LP3-ITS-0764_930_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.177 s (9.017 ¬µs/read; 6.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,602\n",
      "Reads with adapters:                        94 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,602 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,900,202 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,882,520 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 94 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.5%\n",
      "  C: 8.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t76.6\t0\t1\n",
      "5\t1\t19.1\t0\t1\n",
      "6\t5\t4.8\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "209\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_189-LP3-ITS-0765_937_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_189-LP3-ITS-0765_937_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_189-LP3-ITS-0765_937_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_189-LP3-ITS-0765_937_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.141 s (7.867 ¬µs/read; 7.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,895\n",
      "Reads with adapters:                        67 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,895 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,386,395 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,374,512 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 67 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.1%\n",
      "  C: 11.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t279.6\t0\t1\n",
      "4\t1\t69.9\t0\t1\n",
      "6\t7\t4.4\t0\t7\n",
      "7\t1\t1.1\t0\t1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "209\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_190-LP3-ITS-0766_1110_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_190-LP3-ITS-0766_1110_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_190-LP3-ITS-0766_1110_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_190-LP3-ITS-0766_1110_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.235 s (7.750 ¬µs/read; 7.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,333\n",
      "Reads with adapters:                       210 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,333 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,130,233 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,120,245 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 210 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.6%\n",
      "  C: 41.9%\n",
      "  G: 1.0%\n",
      "  T: 18.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t474.0\t0\t106\n",
      "4\t14\t118.5\t0\t14\n",
      "5\t4\t29.6\t0\t4\n",
      "6\t14\t7.4\t0\t14\n",
      "7\t1\t1.9\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t25\t0.0\t1\t0 25\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "209\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_191-LP3-ITS-0767_417_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_191-LP3-ITS-0767_417_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_191-LP3-ITS-0767_417_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_191-LP3-ITS-0767_417_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.146 s (8.532 ¬µs/read; 7.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,066\n",
      "Reads with adapters:                        63 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,066 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,136,866 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,129,090 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 63 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.5%\n",
      "  C: 36.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t12\t66.7\t0\t12\n",
      "5\t2\t16.7\t0\t2\n",
      "6\t11\t4.2\t0\t11\n",
      "7\t1\t1.0\t0\t1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "209\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_192-LP3-ITS-0768_167_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_192-LP3-ITS-0768_167_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_192-LP3-ITS-0768_167_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_192-LP3-ITS-0768_167_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.414 s (6.448 ¬µs/read; 9.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  64,214\n",
      "Reads with adapters:                        74 (0.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        64,214 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,328,414 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     19,327,358 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 74 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.0%\n",
      "  C: 77.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1003.3\t0\t3\n",
      "4\t25\t250.8\t0\t25\n",
      "5\t2\t62.7\t0\t2\n",
      "6\t32\t15.7\t0\t32\n",
      "9\t6\t0.2\t0\t6\n",
      "10\t1\t0.1\t1\t0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_193-LP3-ITS-0769_224_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_193-LP3-ITS-0769_224_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_193-LP3-ITS-0769_224_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_193-LP3-ITS-0769_224_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.319 s (10.924 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,183\n",
      "Reads with adapters:                       339 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,183 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,784,083 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,742,820 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 339 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.6%\n",
      "  C: 29.8%\n",
      "  G: 0.0%\n",
      "  T: 2.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t456.0\t0\t9\n",
      "4\t2\t114.0\t0\t2\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "84\t4\t0.0\t2\t0 0 4\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t6\t0.0\t2\t0 0 6\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "103\t9\t0.0\t2\t0 0 9\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "118\t127\t0.0\t2\t0 1 126\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t9\t0.0\t2\t0 1 8\n",
      "137\t39\t0.0\t2\t0 1 38\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t12\t0.0\t2\t0 0 12\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "150\t17\t0.0\t2\t0 0 17\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "155\t29\t0.0\t2\t0 1 28\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_194-LP3-ITS-0770_797_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_194-LP3-ITS-0770_797_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_194-LP3-ITS-0770_797_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_194-LP3-ITS-0770_797_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.178 s (10.480 ¬µs/read; 5.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,966\n",
      "Reads with adapters:                       179 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,966 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,106,766 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,085,769 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 179 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.3%\n",
      "  C: 28.5%\n",
      "  G: 0.0%\n",
      "  T: 2.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t265.1\t0\t9\n",
      "10\t2\t0.0\t1\t0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t100\t0.0\t2\t0 2 98\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t15\t0.0\t2\t0 1 14\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_195-LP3-ITS-0771_176_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_195-LP3-ITS-0771_176_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_195-LP3-ITS-0771_176_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_195-LP3-ITS-0771_176_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.258 s (11.225 ¬µs/read; 5.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,023\n",
      "Reads with adapters:                       275 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,929,923 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,897,246 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 275 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.5%\n",
      "  C: 32.4%\n",
      "  G: 0.4%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t359.7\t0\t9\n",
      "4\t1\t89.9\t0\t1\n",
      "9\t7\t0.1\t0\t7\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t13\t0.0\t2\t0 0 13\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "95\t2\t0.0\t2\t0 1 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t108\t0.0\t2\t0 2 106\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t45\t0.0\t2\t0 0 45\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t25\t0.0\t2\t0 1 24\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_196-LP3-ITS-0772_1132_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_196-LP3-ITS-0772_1132_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_196-LP3-ITS-0772_1132_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_196-LP3-ITS-0772_1132_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.615 s (10.497 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  58,611\n",
      "Reads with adapters:                       634 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        58,611 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,641,911 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,564,528 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 634 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.4%\n",
      "  C: 32.3%\n",
      "  G: 0.2%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t915.8\t0\t4\n",
      "4\t1\t228.9\t0\t1\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t8\t0.0\t2\t0 0 8\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "53\t4\t0.0\t2\t0 0 4\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t7\t0.0\t2\t0 0 7\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t4\t0.0\t2\t0 0 4\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "110\t4\t0.0\t2\t0 0 4\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t230\t0.0\t2\t0 5 225\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t111\t0.0\t2\t0 2 109\n",
      "134\t72\t0.0\t2\t0 2 70\n",
      "137\t40\t0.0\t2\t0 1 39\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t53\t0.0\t2\t0 0 53\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t23\t0.0\t2\t0 1 22\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_197-LP3-ITS-0773_334_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_197-LP3-ITS-0773_334_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_197-LP3-ITS-0773_334_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_197-LP3-ITS-0773_334_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.322 s (10.441 ¬µs/read; 5.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,852\n",
      "Reads with adapters:                       591 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,852 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,286,452 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,248,136 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 591 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.0%\n",
      "  C: 42.3%\n",
      "  G: 1.2%\n",
      "  T: 11.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t213\t482.1\t0\t213\n",
      "4\t1\t120.5\t0\t1\n",
      "8\t1\t0.5\t0\t1\n",
      "9\t8\t0.1\t0\t8\n",
      "10\t63\t0.0\t1\t4 59\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "43\t2\t0.0\t2\t0 0 2\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t8\t0.0\t2\t0 0 8\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t149\t0.0\t2\t0 3 146\n",
      "133\t55\t0.0\t2\t0 1 54\n",
      "134\t3\t0.0\t2\t0 1 2\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t11\t0.0\t2\t0 1 10\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t25\t0.0\t2\t0 0 25\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_198-LP3-ITS-0774_545_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_198-LP3-ITS-0774_545_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_198-LP3-ITS-0774_545_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_198-LP3-ITS-0774_545_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.234 s (11.117 ¬µs/read; 5.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,021\n",
      "Reads with adapters:                       266 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,327,321 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,295,079 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 266 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.1%\n",
      "  C: 22.6%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t328.5\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t170\t0.0\t2\t0 6 164\n",
      "133\t29\t0.0\t2\t0 0 29\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t10\t0.0\t2\t0 1 9\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_199-LP3-ITS-0775_12_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_199-LP3-ITS-0775_12_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_199-LP3-ITS-0775_12_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_199-LP3-ITS-0775_12_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.327 s (10.683 ¬µs/read; 5.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,596\n",
      "Reads with adapters:                       334 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,596 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,209,396 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,170,067 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 334 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 57.8%\n",
      "  C: 41.3%\n",
      "  G: 0.3%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t478.1\t0\t4\n",
      "4\t15\t119.5\t0\t15\n",
      "10\t1\t0.0\t1\t0 1\n",
      "22\t3\t0.0\t2\t0 0 3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t126\t0.0\t2\t0 0 126\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "133\t78\t0.0\t2\t0 0 78\n",
      "134\t14\t0.0\t2\t0 0 14\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t13\t0.0\t2\t0 0 13\n",
      "155\t17\t0.0\t2\t0 0 17\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_200-LP3-ITS-0776_855_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_200-LP3-ITS-0776_855_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_200-LP3-ITS-0776_855_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_200-LP3-ITS-0776_855_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.347 s (11.249 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,842\n",
      "Reads with adapters:                       347 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,842 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,283,442 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,239,534 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 347 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.2%\n",
      "  C: 23.6%\n",
      "  G: 0.3%\n",
      "  T: 2.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t481.9\t0\t5\n",
      "4\t1\t120.5\t0\t1\n",
      "6\t1\t7.5\t0\t1\n",
      "9\t1\t0.1\t0\t0 1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t7\t0.0\t2\t0 1 6\n",
      "99\t6\t0.0\t2\t0 0 6\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t104\t0.0\t2\t0 1 103\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t15\t0.0\t2\t0 0 15\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t17\t0.0\t2\t0 1 16\n",
      "143\t23\t0.0\t2\t0 0 23\n",
      "145\t52\t0.0\t2\t0 1 51\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t38\t0.0\t2\t0 0 38\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t22\t0.0\t2\t0 0 22\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_201-LP3-ITS-0777_903_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_201-LP3-ITS-0777_903_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_201-LP3-ITS-0777_903_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_201-LP3-ITS-0777_903_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.279 s (11.231 ¬µs/read; 5.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,811\n",
      "Reads with adapters:                       314 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,468,111 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,428,362 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 314 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.9%\n",
      "  C: 17.8%\n",
      "  G: 0.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "57\t5\t0.0\t2\t0 0 5\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t159\t0.0\t2\t0 2 157\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t14\t0.0\t2\t0 0 14\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t60\t0.0\t2\t0 1 59\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_202-LP3-ITS-0778_149_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_202-LP3-ITS-0778_149_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_202-LP3-ITS-0778_149_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_202-LP3-ITS-0778_149_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.214 s (11.691 ¬µs/read; 5.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,276\n",
      "Reads with adapters:                       165 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,276 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,501,076 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,480,741 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 165 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.9%\n",
      "  C: 23.0%\n",
      "  G: 0.0%\n",
      "  T: 3.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t285.6\t0\t3\n",
      "4\t1\t71.4\t0\t1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t91\t0.0\t2\t0 1 90\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "150\t18\t0.0\t2\t0 0 18\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_203-LP3-ITS-0779_620_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_203-LP3-ITS-0779_620_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_203-LP3-ITS-0779_620_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_203-LP3-ITS-0779_620_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.166 s (11.804 ¬µs/read; 5.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,079\n",
      "Reads with adapters:                       210 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,079 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,237,779 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,213,389 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 210 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.1%\n",
      "  C: 22.4%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t220.0\t0\t1\n",
      "6\t5\t3.4\t0\t5\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 1 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "118\t126\t0.0\t2\t0 2 124\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t20\t0.0\t2\t0 0 20\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_204-LP3-ITS-0780_329_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_204-LP3-ITS-0780_329_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_204-LP3-ITS-0780_329_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_204-LP3-ITS-0780_329_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.278 s (10.812 ¬µs/read; 5.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,738\n",
      "Reads with adapters:                       238 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,738 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,747,138 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,719,403 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 238 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.8%\n",
      "  C: 39.1%\n",
      "  G: 0.4%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t402.2\t0\t2\n",
      "6\t2\t6.3\t0\t2\n",
      "10\t2\t0.0\t1\t0 2\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t3\t0.0\t2\t0 0 3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t7\t0.0\t2\t0 0 7\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t113\t0.0\t2\t0 2 111\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t44\t0.0\t2\t0 0 44\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_205-LP3-ITS-0781_1088_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_205-LP3-ITS-0781_1088_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_205-LP3-ITS-0781_1088_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_205-LP3-ITS-0781_1088_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.010 s (18.341 ¬µs/read; 3.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     557\n",
      "Reads with adapters:                         5 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           557 (100.0%)\n",
      "\n",
      "Total basepairs processed:       167,657 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        167,037 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.0%\n",
      "  C: 40.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_206-LP3-ITS-0782_614_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_206-LP3-ITS-0782_614_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_206-LP3-ITS-0782_614_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_206-LP3-ITS-0782_614_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.319 s (10.782 ¬µs/read; 5.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,548\n",
      "Reads with adapters:                       335 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,548 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,893,948 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,852,255 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 335 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.8%\n",
      "  C: 34.3%\n",
      "  G: 0.6%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t461.7\t0\t3\n",
      "6\t4\t7.2\t0\t4\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t4\t0.0\t2\t0 0 4\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t120\t0.0\t2\t0 2 118\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "133\t58\t0.0\t2\t0 0 58\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t36\t0.0\t2\t0 1 35\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t12\t0.0\t2\t0 1 11\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_207-LP3-ITS-0783_877_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_207-LP3-ITS-0783_877_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_207-LP3-ITS-0783_877_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_207-LP3-ITS-0783_877_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.353 s (10.910 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,394\n",
      "Reads with adapters:                       374 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,394 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,750,594 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,703,664 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 374 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.5%\n",
      "  C: 33.7%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t506.2\t0\t6\n",
      "4\t1\t126.5\t0\t1\n",
      "6\t1\t7.9\t0\t1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t173\t0.0\t2\t0 4 169\n",
      "133\t67\t0.0\t2\t0 0 67\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "158\t9\t0.0\t2\t0 0 9\n",
      "159\t10\t0.0\t2\t0 1 9\n",
      "160\t42\t0.0\t2\t0 0 42\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_208-LP3-ITS-0784_553_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_208-LP3-ITS-0784_553_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_208-LP3-ITS-0784_553_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_208-LP3-ITS-0784_553_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.217 s (12.026 ¬µs/read; 4.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,020\n",
      "Reads with adapters:                       688 (3.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,020 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,424,020 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,409,188 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 688 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.4%\n",
      "  C: 43.6%\n",
      "  G: 1.3%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t412\t281.6\t0\t412\n",
      "9\t100\t0.1\t0\t91 9\n",
      "10\t79\t0.0\t1\t11 68\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t60\t0.0\t2\t0 3 57\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_209-LP3-ITS-0785_371_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_209-LP3-ITS-0785_371_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_209-LP3-ITS-0785_371_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_209-LP3-ITS-0785_371_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.357 s (10.103 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,294\n",
      "Reads with adapters:                       487 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,294 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,623,494 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,570,319 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 487 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.8%\n",
      "  C: 54.6%\n",
      "  G: 0.4%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t551.5\t0\t10\n",
      "4\t5\t137.9\t0\t5\n",
      "6\t17\t8.6\t0\t17\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "32\t21\t0.0\t2\t0 1 20\n",
      "46\t5\t0.0\t2\t0 0 5\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "79\t4\t0.0\t2\t0 0 4\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t8\t0.0\t2\t0 0 8\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t8\t0.0\t2\t0 0 8\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t186\t0.0\t2\t0 6 180\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "129\t2\t0.0\t2\t0 0 2\n",
      "133\t92\t0.0\t2\t0 3 89\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t48\t0.0\t2\t0 0 48\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_210-LP3-ITS-0786_827_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_210-LP3-ITS-0786_827_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_210-LP3-ITS-0786_827_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_210-LP3-ITS-0786_827_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.179 s (11.780 ¬µs/read; 5.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,193\n",
      "Reads with adapters:                       146 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,573,093 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,555,652 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 146 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.9%\n",
      "  C: 30.1%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t237.4\t0\t2\n",
      "4\t1\t59.3\t0\t1\n",
      "9\t5\t0.1\t0\t5\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t70\t0.0\t2\t0 2 68\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 1\n",
      "133\t27\t0.0\t2\t0 1 26\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_211-LP3-ITS-0787_756_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_211-LP3-ITS-0787_756_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_211-LP3-ITS-0787_756_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_211-LP3-ITS-0787_756_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.224 s (10.153 ¬µs/read; 5.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,015\n",
      "Reads with adapters:                       231 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,015 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,626,515 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,599,968 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 231 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.7%\n",
      "  C: 29.0%\n",
      "  G: 0.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t6\t86.0\t0\t6\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t146\t0.0\t2\t0 3 143\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t33\t0.0\t2\t0 1 32\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_212-LP3-ITS-0788_132_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_212-LP3-ITS-0788_132_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_212-LP3-ITS-0788_132_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_212-LP3-ITS-0788_132_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.367 s (10.476 ¬µs/read; 5.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,988\n",
      "Reads with adapters:                       383 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,988 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,531,388 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,485,848 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 383 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.0%\n",
      "  C: 20.1%\n",
      "  G: 0.8%\n",
      "  T: 2.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t546.7\t0\t24\n",
      "4\t2\t136.7\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t8\t0.0\t1\t1 7\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "98\t2\t0.0\t2\t0 1 1\n",
      "99\t11\t0.0\t2\t0 0 11\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t5\t0.0\t2\t0 1 4\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t219\t0.0\t2\t0 6 213\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t14\t0.0\t2\t0 1 13\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "194\t32\t0.0\t2\t0 0 32\n",
      "195\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_213-LP3-ITS-0789_37_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_213-LP3-ITS-0789_37_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_213-LP3-ITS-0789_37_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_213-LP3-ITS-0789_37_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.299 s (9.888 ¬µs/read; 6.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,238\n",
      "Reads with adapters:                       272 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,238 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,101,638 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,069,197 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 272 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.2%\n",
      "  C: 36.0%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t472.5\t0\t1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "67\t4\t0.0\t2\t0 0 4\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "99\t3\t0.0\t2\t0 1 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t127\t0.0\t2\t0 1 126\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t7\t0.0\t2\t0 0 7\n",
      "133\t51\t0.0\t2\t0 1 50\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t25\t0.0\t2\t0 0 25\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_214-LP3-ITS-0790_232_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_214-LP3-ITS-0790_232_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_214-LP3-ITS-0790_232_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_214-LP3-ITS-0790_232_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.246 s (11.414 ¬µs/read; 5.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,568\n",
      "Reads with adapters:                       235 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,491,968 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,463,249 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 235 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.6%\n",
      "  C: 28.9%\n",
      "  G: 0.4%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t337.0\t0\t2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 1 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t8\t0.0\t2\t0 0 8\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t136\t0.0\t2\t0 4 132\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t28\t0.0\t2\t0 1 27\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "151\t24\t0.0\t2\t0 0 24\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_215-LP3-ITS-0791_25_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_215-LP3-ITS-0791_25_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_215-LP3-ITS-0791_25_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_215-LP3-ITS-0791_25_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.315 s (10.584 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,717\n",
      "Reads with adapters:                       312 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,717 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,944,817 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,910,125 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 312 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.6%\n",
      "  C: 28.5%\n",
      "  G: 0.3%\n",
      "  T: 1.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t464.3\t0\t29\n",
      "4\t2\t116.1\t0\t2\n",
      "6\t9\t7.3\t0\t9\n",
      "20\t3\t0.0\t2\t0 0 3\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "118\t132\t0.0\t2\t0 3 129\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t8\t0.0\t2\t0 1 7\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "152\t10\t0.0\t2\t0 0 10\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "158\t11\t0.0\t2\t0 0 11\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t24\t0.0\t2\t0 2 22\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_216-LP3-ITS-0792_142_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_216-LP3-ITS-0792_142_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_216-LP3-ITS-0792_142_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_216-LP3-ITS-0792_142_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.848 s (10.465 ¬µs/read; 5.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  81,014\n",
      "Reads with adapters:                     1,042 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        81,014 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,385,214 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     24,288,135 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1042 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 46.9%\n",
      "  C: 47.8%\n",
      "  G: 0.6%\n",
      "  T: 4.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t186\t1265.8\t0\t186\n",
      "4\t18\t316.5\t0\t18\n",
      "8\t1\t1.2\t0\t1\n",
      "9\t23\t0.3\t0\t22 1\n",
      "10\t19\t0.1\t1\t2 17\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "43\t5\t0.0\t2\t0 0 5\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t7\t0.0\t2\t0 0 7\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "52\t5\t0.0\t2\t0 0 5\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t6\t0.0\t2\t0 0 6\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "84\t11\t0.0\t2\t0 0 11\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t6\t0.0\t2\t0 0 6\n",
      "93\t6\t0.0\t2\t0 1 5\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t8\t0.0\t2\t0 1 7\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t4\t0.0\t2\t0 0 4\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t15\t0.0\t2\t0 0 15\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t338\t0.0\t2\t0 9 329\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t184\t0.0\t2\t0 4 180\n",
      "134\t23\t0.0\t2\t0 0 23\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t21\t0.0\t2\t0 0 21\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t17\t0.0\t2\t0 0 17\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "155\t43\t0.0\t2\t0 0 43\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "160\t21\t0.0\t2\t0 1 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_217-LP3-ITS-0793_401_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_217-LP3-ITS-0793_401_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_217-LP3-ITS-0793_401_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_217-LP3-ITS-0793_401_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.351 s (11.005 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,891\n",
      "Reads with adapters:                       384 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,891 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,599,191 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,549,713 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 384 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.3%\n",
      "  C: 28.9%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t8\t0.0\t2\t0 0 8\n",
      "93\t7\t0.0\t2\t0 1 6\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "118\t148\t0.0\t2\t0 3 145\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t50\t0.0\t2\t0 0 50\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t24\t0.0\t2\t0 1 23\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t39\t0.0\t2\t0 1 38\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t10\t0.0\t2\t0 1 9\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t27\t0.0\t2\t0 1 26\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_218-LP3-ITS-0794_938_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_218-LP3-ITS-0794_938_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_218-LP3-ITS-0794_938_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_218-LP3-ITS-0794_938_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.170 s (11.552 ¬µs/read; 5.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,746\n",
      "Reads with adapters:                       148 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,746 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,438,546 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,420,729 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 148 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.5%\n",
      "  C: 34.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t230.4\t0\t1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t90\t0.0\t2\t0 3 87\n",
      "133\t23\t0.0\t2\t0 1 22\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_219-LP3-ITS-0795_858_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_219-LP3-ITS-0795_858_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_219-LP3-ITS-0795_858_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_219-LP3-ITS-0795_858_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.315 s (9.783 ¬µs/read; 6.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,231\n",
      "Reads with adapters:                       330 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,231 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,701,531 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,662,464 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 330 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.0%\n",
      "  C: 10.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t291\t0.0\t2\t0 3 288\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_220-LP3-ITS-0796_173_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_220-LP3-ITS-0796_173_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_220-LP3-ITS-0796_173_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_220-LP3-ITS-0796_173_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.236 s (11.199 ¬µs/read; 5.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,083\n",
      "Reads with adapters:                       227 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,083 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,345,983 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,321,394 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 227 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.8%\n",
      "  C: 37.4%\n",
      "  G: 0.4%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t329.4\t0\t17\n",
      "5\t1\t20.6\t0\t1\n",
      "9\t4\t0.1\t0\t3 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 1 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "84\t12\t0.0\t2\t0 0 12\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t3\t0.0\t2\t0 0 3\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t97\t0.0\t2\t0 3 94\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_221-LP3-ITS-0797_914_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_221-LP3-ITS-0797_914_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_221-LP3-ITS-0797_914_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_221-LP3-ITS-0797_914_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.139 s (10.642 ¬µs/read; 5.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,065\n",
      "Reads with adapters:                       614 (4.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,065 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,932,565 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,918,271 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 614 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 54.2%\n",
      "  G: 2.4%\n",
      "  T: 14.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t430\t204.1\t0\t430\n",
      "4\t1\t51.0\t0\t1\n",
      "9\t6\t0.0\t0\t6\n",
      "10\t76\t0.0\t1\t5 71\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "118\t85\t0.0\t2\t0 1 84\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_222-LP3-ITS-0798_14_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_222-LP3-ITS-0798_14_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_222-LP3-ITS-0798_14_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_222-LP3-ITS-0798_14_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.236 s (10.977 ¬µs/read; 5.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,476\n",
      "Reads with adapters:                       604 (2.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,476 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,464,276 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,438,842 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 604 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.6%\n",
      "  C: 13.4%\n",
      "  G: 0.2%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t392\t335.6\t0\t392\n",
      "4\t3\t83.9\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t3\t0.0\t2\t0 0 3\n",
      "51\t10\t0.0\t2\t0 1 9\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t128\t0.0\t2\t0 1 127\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "151\t18\t0.0\t2\t0 0 18\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_223-LP3-ITS-0799_1140_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_223-LP3-ITS-0799_1140_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_223-LP3-ITS-0799_1140_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_223-LP3-ITS-0799_1140_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.281 s (9.117 ¬µs/read; 6.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,854\n",
      "Reads with adapters:                     2,739 (8.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,287,054 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,273,841 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2739 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.1%\n",
      "  C: 6.5%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2699\t482.1\t0\t2699\n",
      "4\t8\t120.5\t0\t8\n",
      "5\t1\t30.1\t0\t1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "176\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_224-LP3-ITS-0800_372_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_224-LP3-ITS-0800_372_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_224-LP3-ITS-0800_372_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_224-LP3-ITS-0800_372_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.570 s (10.616 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,672\n",
      "Reads with adapters:                       587 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,672 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,155,272 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,087,974 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 587 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.4%\n",
      "  C: 25.2%\n",
      "  G: 0.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t14\t209.7\t0\t14\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t11\t0.0\t2\t0 0 11\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t34\t0.0\t2\t0 0 34\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t419\t0.0\t2\t2 4 413\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_225-LP3-ITS-0801_275_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_225-LP3-ITS-0801_275_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_225-LP3-ITS-0801_275_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_225-LP3-ITS-0801_275_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.366 s (10.278 ¬µs/read; 5.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,577\n",
      "Reads with adapters:                       618 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,577 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,708,677 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,644,801 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 618 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.3%\n",
      "  C: 32.5%\n",
      "  G: 0.5%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t139.0\t0\t2\n",
      "6\t77\t8.7\t0\t77\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "40\t1\t0.0\t2\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t6\t0.0\t2\t0 0 6\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t5\t0.0\t2\t0 0 5\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "103\t20\t0.0\t2\t0 0 20\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t7\t0.0\t2\t0 0 7\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "110\t5\t0.0\t2\t0 0 5\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "118\t365\t0.0\t2\t0 4 361\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t16\t0.0\t2\t0 0 16\n",
      "137\t15\t0.0\t2\t0 1 14\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t19\t0.0\t2\t0 0 19\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_226-LP3-ITS-0802_1083_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_226-LP3-ITS-0802_1083_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_226-LP3-ITS-0802_1083_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_226-LP3-ITS-0802_1083_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.112 s (13.835 ¬µs/read; 4.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   8,066\n",
      "Reads with adapters:                        85 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         8,066 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,427,866 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,419,434 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 85 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 62.4%\n",
      "  C: 34.1%\n",
      "  G: 0.0%\n",
      "  T: 3.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t126.0\t0\t10\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 1 1\n",
      "118\t46\t0.0\t2\t0 1 45\n",
      "132\t1\t0.0\t2\t0 1\n",
      "133\t7\t0.0\t2\t0 1 6\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_227-LP3-ITS-0803_381_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_227-LP3-ITS-0803_381_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_227-LP3-ITS-0803_381_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_227-LP3-ITS-0803_381_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.237 s (11.106 ¬µs/read; 5.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,366\n",
      "Reads with adapters:                       255 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,366 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,431,166 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,405,609 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 255 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.8%\n",
      "  C: 40.8%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t333.8\t0\t1\n",
      "5\t1\t20.9\t0\t1\n",
      "6\t27\t5.2\t0\t27\n",
      "12\t2\t0.0\t1\t0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t6\t0.0\t2\t0 0 6\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t10\t0.0\t2\t0 0 10\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t133\t0.0\t2\t0 3 130\n",
      "133\t15\t0.0\t2\t0 0 15\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_228-LP3-ITS-0804_1105_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_228-LP3-ITS-0804_1105_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_228-LP3-ITS-0804_1105_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_228-LP3-ITS-0804_1105_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.207 s (11.420 ¬µs/read; 5.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,158\n",
      "Reads with adapters:                       176 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,465,558 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,445,214 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 176 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.0%\n",
      "  C: 23.3%\n",
      "  G: 0.0%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t283.7\t0\t4\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "99\t6\t0.0\t2\t0 0 6\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "118\t117\t0.0\t2\t0 0 117\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_229-LP3-ITS-0805_1059_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_229-LP3-ITS-0805_1059_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_229-LP3-ITS-0805_1059_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_229-LP3-ITS-0805_1059_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.010 s (21.133 ¬µs/read; 2.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     451\n",
      "Reads with adapters:                         5 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           451 (100.0%)\n",
      "\n",
      "Total basepairs processed:       135,751 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        135,261 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.0%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 20.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t7.0\t0\t1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_230-LP3-ITS-0806_924_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_230-LP3-ITS-0806_924_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_230-LP3-ITS-0806_924_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_230-LP3-ITS-0806_924_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (23.558 ¬µs/read; 2.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     349\n",
      "Reads with adapters:                         9 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           349 (100.0%)\n",
      "\n",
      "Total basepairs processed:       105,049 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        104,042 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.8%\n",
      "  C: 22.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t0.0\t0\t1\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_231-LP3-ITS-0807_1031_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_231-LP3-ITS-0807_1031_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_231-LP3-ITS-0807_1031_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_231-LP3-ITS-0807_1031_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.445 s (10.195 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,642\n",
      "Reads with adapters:                       418 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,136,242 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,089,410 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 418 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.6%\n",
      "  C: 8.1%\n",
      "  G: 0.2%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t681.9\t0\t12\n",
      "5\t1\t42.6\t0\t1\n",
      "6\t3\t10.7\t0\t3\n",
      "42\t3\t0.0\t2\t0 0 3\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "81\t3\t0.0\t2\t0 0 3\n",
      "86\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "118\t352\t0.0\t2\t0 3 349\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_232-LP3-ITS-0808_750_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_232-LP3-ITS-0808_750_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_232-LP3-ITS-0808_750_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_232-LP3-ITS-0808_750_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.278 s (11.123 ¬µs/read; 5.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,023\n",
      "Reads with adapters:                       286 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,531,923 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,497,904 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 286 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.2%\n",
      "  C: 29.4%\n",
      "  G: 0.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t391.0\t0\t6\n",
      "4\t1\t97.7\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "40\t3\t0.0\t2\t0 0 3\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t11\t0.0\t2\t0 0 11\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "118\t149\t0.0\t2\t0 2 147\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t29\t0.0\t2\t0 1 28\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t16\t0.0\t2\t0 0 16\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t13\t0.0\t2\t0 0 13\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_233-LP3-ITS-0809_997_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_233-LP3-ITS-0809_997_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_233-LP3-ITS-0809_997_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_233-LP3-ITS-0809_997_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.085 s (11.689 ¬µs/read; 5.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,303\n",
      "Reads with adapters:                       112 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,303 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,198,203 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,185,956 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 112 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.3%\n",
      "  C: 19.6%\n",
      "  G: 0.0%\n",
      "  T: 16.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "79\t30\t0.0\t2\t0 1 29\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "118\t64\t0.0\t2\t0 1 63\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_234-LP3-ITS-0810_722_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_234-LP3-ITS-0810_722_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_234-LP3-ITS-0810_722_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_234-LP3-ITS-0810_722_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.137 s (10.674 ¬µs/read; 5.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,807\n",
      "Reads with adapters:                       145 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,807 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,854,907 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,838,088 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 145 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.8%\n",
      "  C: 14.5%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t200.1\t0\t4\n",
      "9\t1\t0.0\t0\t1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t113\t0.0\t2\t0 2 111\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_235-LP3-ITS-0811_9_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_235-LP3-ITS-0811_9_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_235-LP3-ITS-0811_9_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_235-LP3-ITS-0811_9_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.251 s (10.579 ¬µs/read; 5.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,769\n",
      "Reads with adapters:                       285 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,769 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,154,469 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,120,120 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 285 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 61.4%\n",
      "  C: 37.9%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t371.4\t0\t6\n",
      "6\t2\t5.8\t0\t2\n",
      "7\t1\t1.5\t0\t1\n",
      "12\t3\t0.0\t1\t1 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "59\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t138\t0.0\t2\t0 1 137\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t6\t0.0\t2\t0 0 6\n",
      "130\t5\t0.0\t2\t0 1 4\n",
      "133\t39\t0.0\t2\t0 0 39\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t12\t0.0\t2\t0 0 12\n",
      "153\t6\t0.0\t2\t0 0 6\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_236-LP3-ITS-0812_1012_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_236-LP3-ITS-0812_1012_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_236-LP3-ITS-0812_1012_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_236-LP3-ITS-0812_1012_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.182 s (10.726 ¬µs/read; 5.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,966\n",
      "Reads with adapters:                     1,067 (6.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,966 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,106,766 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,090,070 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1067 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.4%\n",
      "  C: 53.7%\n",
      "  G: 1.9%\n",
      "  T: 18.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t780\t265.1\t0\t780\n",
      "8\t38\t0.3\t0\t38\n",
      "9\t27\t0.1\t0\t27\n",
      "10\t121\t0.0\t1\t12 109\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t78\t0.0\t2\t0 0 78\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "234\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_237-LP3-ITS-0813_781_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_237-LP3-ITS-0813_781_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_237-LP3-ITS-0813_781_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_237-LP3-ITS-0813_781_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.172 s (11.814 ¬µs/read; 5.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,587\n",
      "Reads with adapters:                       999 (6.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,390,687 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,378,958 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 999 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.0%\n",
      "  C: 62.2%\n",
      "  G: 1.3%\n",
      "  T: 15.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t756\t227.9\t0\t756\n",
      "4\t13\t57.0\t0\t13\n",
      "8\t30\t0.2\t0\t30\n",
      "9\t20\t0.1\t0\t18 2\n",
      "10\t106\t0.0\t1\t3 103\n",
      "11\t1\t0.0\t1\t0 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "32\t5\t0.0\t2\t0 0 5\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "55\t1\t0.0\t2\t0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t38\t0.0\t2\t0 2 36\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t15\t0.0\t2\t0 0 15\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_238-LP3-ITS-0814_488_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_238-LP3-ITS-0814_488_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_238-LP3-ITS-0814_488_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_238-LP3-ITS-0814_488_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.135 s (11.016 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,292\n",
      "Reads with adapters:                       170 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,292 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,699,892 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,682,099 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 170 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 61.8%\n",
      "  C: 34.7%\n",
      "  G: 0.0%\n",
      "  T: 3.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t192.1\t0\t7\n",
      "4\t1\t48.0\t0\t1\n",
      "5\t5\t12.0\t0\t5\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t96\t0.0\t2\t0 4 92\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_239-LP3-ITS-0815_550_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_239-LP3-ITS-0815_550_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_239-LP3-ITS-0815_550_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_239-LP3-ITS-0815_550_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.357 s (10.216 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,973\n",
      "Reads with adapters:                       277 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,973 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,526,873 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,491,837 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 277 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 14.4%\n",
      "  C: 85.2%\n",
      "  G: 0.4%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t546.5\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "26\t6\t0.0\t2\t0 0 6\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "118\t24\t0.0\t2\t0 2 22\n",
      "119\t10\t0.0\t2\t0 0 10\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t194\t0.0\t2\t0 0 194\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_240-LP3-ITS-0816_456_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_240-LP3-ITS-0816_456_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_240-LP3-ITS-0816_456_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_240-LP3-ITS-0816_456_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (26.056 ¬µs/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     357\n",
      "Reads with adapters:                         7 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           357 (100.0%)\n",
      "\n",
      "Total basepairs processed:       107,457 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        106,834 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 71.4%\n",
      "  C: 28.6%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t5.6\t0\t2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_241-LP3-ITS-0817_261_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_241-LP3-ITS-0817_261_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_241-LP3-ITS-0817_261_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_241-LP3-ITS-0817_261_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.306 s (11.293 ¬µs/read; 5.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,090\n",
      "Reads with adapters:                       425 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,090 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,154,090 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,109,287 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 425 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.9%\n",
      "  C: 46.4%\n",
      "  G: 0.9%\n",
      "  T: 2.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t423.3\t0\t21\n",
      "4\t2\t105.8\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "32\t14\t0.0\t2\t0 0 14\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t6\t0.0\t2\t0 1 5\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "93\t9\t0.0\t2\t0 0 9\n",
      "95\t5\t0.0\t2\t0 0 5\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t12\t0.0\t2\t0 0 12\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "110\t5\t0.0\t2\t0 0 5\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t5\t0.0\t2\t0 2 3\n",
      "118\t188\t0.0\t2\t1 4 183\n",
      "123\t1\t0.0\t2\t0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t34\t0.0\t2\t0 0 34\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t35\t0.0\t2\t0 0 35\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_242-LP3-ITS-0818_974_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_242-LP3-ITS-0818_974_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_242-LP3-ITS-0818_974_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_242-LP3-ITS-0818_974_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.057 s (10.943 ¬µs/read; 5.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   5,226\n",
      "Reads with adapters:                        37 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         5,226 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,573,026 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,568,938 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 37 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.1%\n",
      "  C: 62.2%\n",
      "  G: 2.7%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "53\t9\t0.0\t2\t0 0 9\n",
      "118\t9\t0.0\t2\t0 0 9\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_243-LP3-ITS-0819_590_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_243-LP3-ITS-0819_590_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_243-LP3-ITS-0819_590_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_243-LP3-ITS-0819_590_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.158 s (11.787 ¬µs/read; 5.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,410\n",
      "Reads with adapters:                       126 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,410 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,036,410 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,021,427 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 126 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.5%\n",
      "  C: 36.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "44\t2\t0.0\t2\t0 0 2\n",
      "52\t6\t0.0\t2\t0 0 6\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t66\t0.0\t2\t0 0 66\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t28\t0.0\t2\t0 0 28\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_244-LP3-ITS-0820_69_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_244-LP3-ITS-0820_69_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_244-LP3-ITS-0820_69_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_244-LP3-ITS-0820_69_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.033 s (12.536 ¬µs/read; 4.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   2,603\n",
      "Reads with adapters:                        37 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         2,603 (100.0%)\n",
      "\n",
      "Total basepairs processed:       783,503 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        779,578 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 37 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.7%\n",
      "  C: 64.9%\n",
      "  G: 0.0%\n",
      "  T: 5.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t40.7\t0\t2\n",
      "55\t10\t0.0\t2\t0 0 10\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_245-LP3-ITS-0821_767_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_245-LP3-ITS-0821_767_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_245-LP3-ITS-0821_767_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_245-LP3-ITS-0821_767_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.032 s (19.485 ¬µs/read; 3.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   1,665\n",
      "Reads with adapters:                        10 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         1,665 (100.0%)\n",
      "\n",
      "Total basepairs processed:       501,165 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        500,162 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.0%\n",
      "  C: 30.0%\n",
      "  G: 0.0%\n",
      "  T: 10.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t26.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_246-LP3-ITS-0822_1135_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_246-LP3-ITS-0822_1135_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_246-LP3-ITS-0822_1135_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_246-LP3-ITS-0822_1135_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.221 s (11.052 ¬µs/read; 5.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,022\n",
      "Reads with adapters:                       271 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,022 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,026,622 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,998,792 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 271 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.0%\n",
      "  C: 28.8%\n",
      "  G: 0.0%\n",
      "  T: 2.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t312.8\t0\t32\n",
      "4\t1\t78.2\t0\t1\n",
      "6\t1\t4.9\t0\t1\n",
      "10\t9\t0.0\t1\t1 8\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "59\t9\t0.0\t2\t0 0 9\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t130\t0.0\t2\t1 2 127\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t39\t0.0\t2\t0 1 38\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_247-LP3-ITS-0823_153_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_247-LP3-ITS-0823_153_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_247-LP3-ITS-0823_153_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_247-LP3-ITS-0823_153_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.245 s (11.260 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,794\n",
      "Reads with adapters:                       244 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,559,994 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,531,159 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 244 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.2%\n",
      "  C: 30.3%\n",
      "  G: 0.4%\n",
      "  T: 2.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t340.5\t0\t5\n",
      "4\t1\t85.1\t0\t1\n",
      "5\t5\t21.3\t0\t5\n",
      "6\t1\t5.3\t0\t1\n",
      "8\t1\t0.3\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t122\t0.0\t2\t0 1 121\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t18\t0.0\t2\t0 1 17\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t24\t0.0\t2\t0 1 23\n",
      "155\t20\t0.0\t2\t0 0 20\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_248-LP3-ITS-0824_51_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_248-LP3-ITS-0824_51_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_248-LP3-ITS-0824_51_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_248-LP3-ITS-0824_51_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.454 s (11.097 ¬µs/read; 5.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  40,902\n",
      "Reads with adapters:                       380 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        40,902 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,311,502 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,265,929 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 380 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.2%\n",
      "  C: 11.3%\n",
      "  G: 0.5%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t639.1\t0\t2\n",
      "5\t2\t39.9\t0\t2\n",
      "10\t3\t0.0\t1\t0 3\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t8\t0.0\t2\t0 0 8\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "118\t253\t0.0\t2\t0 2 251\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "142\t70\t0.0\t2\t0 0 70\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_249-LP3-ITS-0825_1139_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_249-LP3-ITS-0825_1139_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_249-LP3-ITS-0825_1139_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_249-LP3-ITS-0825_1139_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.313 s (12.023 ¬µs/read; 4.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,023\n",
      "Reads with adapters:                       376 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,832,923 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,794,871 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 376 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 56.6%\n",
      "  C: 40.4%\n",
      "  G: 0.3%\n",
      "  T: 2.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t406.6\t0\t15\n",
      "4\t3\t101.7\t0\t3\n",
      "6\t9\t6.4\t0\t9\n",
      "7\t13\t1.6\t0\t13\n",
      "8\t13\t0.4\t0\t13\n",
      "9\t4\t0.1\t0\t4\n",
      "12\t3\t0.0\t1\t0 3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "59\t9\t0.0\t2\t0 1 8\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t10\t0.0\t2\t0 1 9\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t146\t0.0\t2\t0 5 141\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t39\t0.0\t2\t0 0 39\n",
      "137\t19\t0.0\t2\t0 0 19\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t33\t0.0\t2\t0 1 32\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_250-LP3-ITS-0826_777_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_250-LP3-ITS-0826_777_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_250-LP3-ITS-0826_777_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_250-LP3-ITS-0826_777_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.073 s (12.503 ¬µs/read; 4.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   5,872\n",
      "Reads with adapters:                        44 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         5,872 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,767,472 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,762,447 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 44 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.6%\n",
      "  C: 36.4%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t91.8\t0\t1\n",
      "52\t3\t0.0\t2\t0 1 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "118\t27\t0.0\t2\t0 1 26\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 1 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_251-LP3-ITS-0827_238_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_251-LP3-ITS-0827_238_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_251-LP3-ITS-0827_238_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_251-LP3-ITS-0827_238_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.167 s (11.625 ¬µs/read; 5.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,388\n",
      "Reads with adapters:                       202 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,388 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,330,788 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,313,315 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 202 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 64.4%\n",
      "  C: 31.2%\n",
      "  G: 1.5%\n",
      "  T: 3.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t224.8\t0\t47\n",
      "5\t1\t14.1\t0\t1\n",
      "10\t10\t0.0\t1\t2 8\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t102\t0.0\t2\t0 3 99\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 1 4\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_252-LP3-ITS-0828_726_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_252-LP3-ITS-0828_726_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_252-LP3-ITS-0828_726_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_252-LP3-ITS-0828_726_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.195 s (11.441 ¬µs/read; 5.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,032\n",
      "Reads with adapters:                       158 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,032 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,126,632 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,109,034 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 158 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.2%\n",
      "  C: 34.2%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t266.1\t0\t3\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "118\t101\t0.0\t2\t0 3 98\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_253-LP3-ITS-0829_510_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_253-LP3-ITS-0829_510_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_253-LP3-ITS-0829_510_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_253-LP3-ITS-0829_510_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.372 s (10.666 ¬µs/read; 5.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,860\n",
      "Reads with adapters:                       333 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,860 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,492,860 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,455,050 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 333 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.0%\n",
      "  C: 32.1%\n",
      "  G: 0.6%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t544.7\t0\t6\n",
      "23\t7\t0.0\t2\t0 0 7\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "60\t8\t0.0\t2\t0 0 8\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t9\t0.0\t2\t0 0 9\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t7\t0.0\t2\t0 0 7\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t9\t0.0\t2\t0 1 8\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t205\t0.0\t2\t0 0 205\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_254-LP3-ITS-0830_829_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_254-LP3-ITS-0830_829_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_254-LP3-ITS-0830_829_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_254-LP3-ITS-0830_829_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.235 s (12.221 ¬µs/read; 4.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,249\n",
      "Reads with adapters:                       238 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,793,949 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,766,433 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 238 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.5%\n",
      "  C: 24.8%\n",
      "  G: 0.0%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t300.8\t0\t2\n",
      "4\t2\t75.2\t0\t2\n",
      "10\t3\t0.0\t1\t1 2\n",
      "31\t2\t0.0\t2\t0 0 2\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t3\t0.0\t2\t0 0 3\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t158\t0.0\t2\t0 3 155\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "133\t33\t0.0\t2\t0 0 33\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_255-LP3-ITS-0831_793_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_255-LP3-ITS-0831_793_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_255-LP3-ITS-0831_793_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_255-LP3-ITS-0831_793_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (53.595 ¬µs/read; 1.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     126\n",
      "Reads with adapters:                         6 (4.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           126 (100.0%)\n",
      "\n",
      "Total basepairs processed:        37,926 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         37,318 bp (98.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.0\t0\t1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_256-LP3-ITS-0832_388_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_256-LP3-ITS-0832_388_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_256-LP3-ITS-0832_388_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_256-LP3-ITS-0832_388_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (28.375 ¬µs/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     312\n",
      "Reads with adapters:                        14 (4.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           312 (100.0%)\n",
      "\n",
      "Total basepairs processed:        93,912 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         92,721 bp (98.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 14 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.9%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 7.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t4.9\t0\t4\n",
      "9\t1\t0.0\t0\t0 1\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_257-LP3-ITS-0833_723_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_257-LP3-ITS-0833_723_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_257-LP3-ITS-0833_723_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_257-LP3-ITS-0833_723_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.011 s (24.460 ¬µs/read; 2.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     435\n",
      "Reads with adapters:                        10 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           435 (100.0%)\n",
      "\n",
      "Total basepairs processed:       130,935 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        129,940 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t6.8\t0\t2\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_258-LP3-ITS-0834_419_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_258-LP3-ITS-0834_419_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_258-LP3-ITS-0834_419_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_258-LP3-ITS-0834_419_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (31.688 ¬µs/read; 1.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     289\n",
      "Reads with adapters:                         7 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           289 (100.0%)\n",
      "\n",
      "Total basepairs processed:        86,989 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         86,474 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 7 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 57.1%\n",
      "  C: 42.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t4.5\t0\t3\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_259-LP3-ITS-0835_697_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_259-LP3-ITS-0835_697_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_259-LP3-ITS-0835_697_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_259-LP3-ITS-0835_697_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.011 s (20.014 ¬µs/read; 3.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     563\n",
      "Reads with adapters:                         8 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           563 (100.0%)\n",
      "\n",
      "Total basepairs processed:       169,463 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        168,743 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 37.5%\n",
      "  G: 0.0%\n",
      "  T: 12.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t8.8\t0\t2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "118\t4\t0.0\t2\t0 1 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_260-LP3-ITS-0836_1051_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_260-LP3-ITS-0836_1051_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_260-LP3-ITS-0836_1051_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_260-LP3-ITS-0836_1051_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.014 s (17.060 ¬µs/read; 3.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     844\n",
      "Reads with adapters:                        15 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           844 (100.0%)\n",
      "\n",
      "Total basepairs processed:       254,044 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        252,944 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 15 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 13.3%\n",
      "  C: 66.7%\n",
      "  G: 6.7%\n",
      "  T: 13.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t13.2\t0\t4\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_261-LP3-ITS-0837_374_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_261-LP3-ITS-0837_374_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_261-LP3-ITS-0837_374_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_261-LP3-ITS-0837_374_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.026 s (12.586 ¬µs/read; 4.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   2,039\n",
      "Reads with adapters:                        17 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         2,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:       613,739 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        611,848 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 94.1%\n",
      "  C: 5.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t31.9\t0\t1\n",
      "118\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_262-LP3-ITS-0838_315_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_262-LP3-ITS-0838_315_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_262-LP3-ITS-0838_315_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_262-LP3-ITS-0838_315_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.096 s (13.442 ¬µs/read; 4.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,156\n",
      "Reads with adapters:                        85 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,156 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,153,956 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,144,205 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 85 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.0%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t111.8\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "98\t4\t0.0\t2\t0 0 4\n",
      "109\t4\t0.0\t2\t0 1 3\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t45\t0.0\t2\t0 0 45\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_263-LP3-ITS-0839_80_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_263-LP3-ITS-0839_80_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_263-LP3-ITS-0839_80_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_263-LP3-ITS-0839_80_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.012 s (20.293 ¬µs/read; 2.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     600\n",
      "Reads with adapters:                        19 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           600 (100.0%)\n",
      "\n",
      "Total basepairs processed:       180,600 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        178,579 bp (98.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 19 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.4%\n",
      "  C: 31.6%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t9.4\t0\t2\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_264-LP3-ITS-0840_814_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_264-LP3-ITS-0840_814_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_264-LP3-ITS-0840_814_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_264-LP3-ITS-0840_814_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (39.121 ¬µs/read; 1.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     209\n",
      "Reads with adapters:                         8 (3.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           209 (100.0%)\n",
      "\n",
      "Total basepairs processed:        62,909 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         62,533 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t3.3\t0\t4\n",
      "10\t1\t0.0\t1\t1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_265-LP3-ITS-0841_889_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_265-LP3-ITS-0841_889_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_265-LP3-ITS-0841_889_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_265-LP3-ITS-0841_889_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.014 s (16.810 ¬µs/read; 3.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     831\n",
      "Reads with adapters:                        22 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           831 (100.0%)\n",
      "\n",
      "Total basepairs processed:       250,131 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        248,433 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 22 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.6%\n",
      "  C: 31.8%\n",
      "  G: 0.0%\n",
      "  T: 4.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t13.0\t0\t7\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t13\t0.0\t2\t0 1 12\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_266-LP3-ITS-0842_1137_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_266-LP3-ITS-0842_1137_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_266-LP3-ITS-0842_1137_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_266-LP3-ITS-0842_1137_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (40.628 ¬µs/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     188\n",
      "Reads with adapters:                         6 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           188 (100.0%)\n",
      "\n",
      "Total basepairs processed:        56,588 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         55,975 bp (98.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.9\t0\t1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_267-LP3-ITS-0843_1115_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_267-LP3-ITS-0843_1115_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_267-LP3-ITS-0843_1115_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_267-LP3-ITS-0843_1115_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.187 s (10.991 ¬µs/read; 5.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,993\n",
      "Reads with adapters:                       194 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,993 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,114,893 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,091,093 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 194 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.4%\n",
      "  C: 19.6%\n",
      "  G: 0.5%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "58\t6\t0.0\t2\t0 2 4\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t9\t0.0\t2\t0 0 9\n",
      "106\t7\t0.0\t2\t0 0 7\n",
      "107\t3\t0.0\t2\t0 1 2\n",
      "108\t4\t0.0\t2\t0 0 4\n",
      "109\t22\t0.0\t2\t0 0 22\n",
      "110\t8\t0.0\t2\t0 0 8\n",
      "111\t5\t0.0\t2\t0 0 5\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t5\t0.0\t2\t0 0 5\n",
      "117\t2\t0.0\t2\t0 1 1\n",
      "118\t22\t0.0\t2\t0 1 21\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t17\t0.0\t2\t0 1 16\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t15\t0.0\t2\t0 1 14\n",
      "137\t5\t0.0\t2\t0 0 5\n",
      "138\t12\t0.0\t2\t0 0 12\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "153\t14\t0.0\t2\t0 0 14\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "165\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_268-LP3-ITS-0844_931_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_268-LP3-ITS-0844_931_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_268-LP3-ITS-0844_931_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_268-LP3-ITS-0844_931_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.231 s (12.520 ¬µs/read; 4.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,454\n",
      "Reads with adapters:                       234 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,454 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,554,654 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,528,958 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 234 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.3%\n",
      "  C: 30.8%\n",
      "  G: 9.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t288.3\t0\t2\n",
      "4\t1\t72.1\t0\t1\n",
      "6\t1\t4.5\t0\t1\n",
      "11\t25\t0.0\t1\t0 25\n",
      "25\t21\t0.0\t2\t0 0 21\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "96\t9\t0.0\t2\t0 0 9\n",
      "97\t6\t0.0\t2\t0 0 6\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t14\t0.0\t2\t0 0 14\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "129\t19\t0.0\t2\t0 1 18\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t21\t0.0\t2\t0 0 21\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "153\t73\t0.0\t2\t0 0 73\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "209\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_269-LP3-ITS-0845_81_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_269-LP3-ITS-0845_81_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_269-LP3-ITS-0845_81_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_269-LP3-ITS-0845_81_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.416 s (11.036 ¬µs/read; 5.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,666\n",
      "Reads with adapters:                       393 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,337,466 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,290,001 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 393 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 51.7%\n",
      "  C: 47.3%\n",
      "  G: 0.8%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t588.5\t0\t6\n",
      "25\t4\t0.0\t2\t0 0 4\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "58\t9\t0.0\t2\t0 0 9\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t45\t0.0\t2\t0 1 44\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t22\t0.0\t2\t0 1 21\n",
      "107\t7\t0.0\t2\t0 0 7\n",
      "109\t26\t0.0\t2\t0 0 26\n",
      "110\t4\t0.0\t2\t0 0 4\n",
      "115\t8\t0.0\t2\t0 0 8\n",
      "118\t52\t0.0\t2\t0 0 52\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "127\t5\t0.0\t2\t0 0 5\n",
      "128\t32\t0.0\t2\t0 0 32\n",
      "130\t9\t0.0\t2\t0 0 9\n",
      "132\t15\t0.0\t2\t0 0 15\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t20\t0.0\t2\t0 0 20\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t37\t0.0\t2\t0 0 37\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "165\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_270-LP3-ITS-0846_972_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_270-LP3-ITS-0846_972_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_270-LP3-ITS-0846_972_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_270-LP3-ITS-0846_972_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.368 s (11.773 ¬µs/read; 5.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,277\n",
      "Reads with adapters:                       566 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,277 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,414,377 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,349,473 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 566 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.2%\n",
      "  C: 17.0%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t488.7\t0\t4\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "25\t5\t0.0\t2\t0 0 5\n",
      "27\t1\t0.0\t2\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "58\t30\t0.0\t2\t0 1 29\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "68\t6\t0.0\t2\t0 2 4\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t34\t0.0\t2\t0 1 33\n",
      "97\t5\t0.0\t2\t0 0 5\n",
      "98\t10\t0.0\t2\t0 0 10\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "107\t8\t0.0\t2\t0 1 7\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "110\t5\t0.0\t2\t0 0 5\n",
      "111\t12\t0.0\t2\t0 0 12\n",
      "112\t4\t0.0\t2\t0 0 4\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t12\t0.0\t2\t0 0 12\n",
      "115\t10\t0.0\t2\t0 0 10\n",
      "116\t24\t0.0\t2\t0 1 23\n",
      "117\t164\t0.0\t2\t1 5 158\n",
      "118\t6\t0.0\t2\t0 0 6\n",
      "123\t15\t0.0\t2\t0 0 15\n",
      "127\t18\t0.0\t2\t0 1 17\n",
      "128\t41\t0.0\t2\t0 0 41\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t42\t0.0\t2\t0 0 42\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t8\t0.0\t2\t0 0 8\n",
      "139\t29\t0.0\t2\t0 0 29\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t11\t0.0\t2\t0 0 11\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "163\t4\t0.0\t2\t0 0 4\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "193\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_271-LP3-ITS-0847_67_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_271-LP3-ITS-0847_67_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_271-LP3-ITS-0847_67_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_271-LP3-ITS-0847_67_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.895 s (10.610 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  84,353\n",
      "Reads with adapters:                     1,262 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        84,353 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,390,253 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     25,248,165 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1262 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.7%\n",
      "  C: 26.5%\n",
      "  G: 0.6%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t110\t1318.0\t0\t110\n",
      "4\t23\t329.5\t0\t23\n",
      "5\t4\t82.4\t0\t4\n",
      "6\t3\t20.6\t0\t3\n",
      "25\t4\t0.0\t2\t0 0 4\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "39\t2\t0.0\t2\t0 0 2\n",
      "49\t23\t0.0\t2\t0 0 23\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "75\t5\t0.0\t2\t0 0 5\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "95\t6\t0.0\t2\t0 0 6\n",
      "96\t43\t0.0\t2\t0 1 42\n",
      "97\t4\t0.0\t2\t0 0 4\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "100\t19\t0.0\t2\t0 0 19\n",
      "102\t22\t0.0\t2\t0 0 22\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t155\t0.0\t2\t0 4 151\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t7\t0.0\t2\t0 0 7\n",
      "109\t24\t0.0\t2\t0 0 24\n",
      "110\t7\t0.0\t2\t0 1 6\n",
      "111\t22\t0.0\t2\t0 0 22\n",
      "112\t8\t0.0\t2\t0 0 8\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t18\t0.0\t2\t0 1 17\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t31\t0.0\t2\t0 0 31\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t8\t0.0\t2\t0 1 7\n",
      "123\t4\t0.0\t2\t0 1 3\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "127\t1\t0.0\t2\t0 1\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t6\t0.0\t2\t0 0 6\n",
      "131\t6\t0.0\t2\t0 0 6\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t17\t0.0\t2\t0 1 16\n",
      "134\t15\t0.0\t2\t0 0 15\n",
      "135\t22\t0.0\t2\t0 0 22\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t181\t0.0\t2\t0 4 177\n",
      "139\t44\t0.0\t2\t0 0 44\n",
      "140\t17\t0.0\t2\t0 0 17\n",
      "141\t6\t0.0\t2\t0 0 6\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t14\t0.0\t2\t0 1 13\n",
      "144\t5\t0.0\t2\t0 0 5\n",
      "145\t203\t0.0\t2\t0 3 200\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "150\t85\t0.0\t2\t0 1 84\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t6\t0.0\t2\t0 0 6\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "161\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "179\t1\t0.0\t2\t0 0 1\n",
      "180\t3\t0.0\t2\t0 0 3\n",
      "182\t6\t0.0\t2\t0 0 6\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "193\t7\t0.0\t2\t0 0 7\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_272-LP3-ITS-0848_241_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_272-LP3-ITS-0848_241_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_272-LP3-ITS-0848_241_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_272-LP3-ITS-0848_241_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.485 s (11.448 ¬µs/read; 5.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,375\n",
      "Reads with adapters:                       642 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,754,875 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,688,500 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 642 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.3%\n",
      "  C: 42.5%\n",
      "  G: 13.4%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t662.1\t0\t3\n",
      "5\t1\t41.4\t0\t1\n",
      "6\t4\t10.3\t0\t4\n",
      "11\t64\t0.0\t1\t2 62\n",
      "25\t85\t0.0\t2\t0 3 82\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "32\t3\t0.0\t2\t0 0 3\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t10\t0.0\t2\t0 0 10\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t26\t0.0\t2\t0 1 25\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "101\t8\t0.0\t2\t0 0 8\n",
      "102\t9\t0.0\t2\t0 0 9\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "113\t3\t0.0\t2\t0 0 3\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t33\t0.0\t2\t0 1 32\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 1 1\n",
      "129\t62\t0.0\t2\t0 0 62\n",
      "130\t6\t0.0\t2\t0 0 6\n",
      "132\t85\t0.0\t2\t0 1 84\n",
      "133\t25\t0.0\t2\t0 0 25\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t127\t0.0\t2\t0 1 126\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t19\t0.0\t2\t0 0 19\n",
      "165\t1\t0.0\t2\t0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "201\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_273-LP3-ITS-0849_505_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_273-LP3-ITS-0849_505_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_273-LP3-ITS-0849_505_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_273-LP3-ITS-0849_505_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.484 s (10.817 ¬µs/read; 5.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,736\n",
      "Reads with adapters:                       822 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,736 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,465,536 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,361,759 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 822 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.2%\n",
      "  C: 44.9%\n",
      "  G: 4.6%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t699.0\t0\t4\n",
      "6\t1\t10.9\t0\t1\n",
      "25\t7\t0.0\t2\t0 0 7\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "40\t7\t0.0\t2\t0 0 7\n",
      "41\t14\t0.0\t2\t0 0 14\n",
      "42\t5\t0.0\t2\t0 0 5\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t45\t0.0\t2\t0 1 44\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t6\t0.0\t2\t0 0 6\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t16\t0.0\t2\t0 0 16\n",
      "104\t6\t0.0\t2\t0 0 6\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t9\t0.0\t2\t0 0 9\n",
      "110\t4\t0.0\t2\t0 0 4\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t4\t0.0\t2\t0 1 3\n",
      "115\t3\t0.0\t2\t0 0 3\n",
      "118\t5\t0.0\t2\t0 1 4\n",
      "120\t13\t0.0\t2\t0 0 13\n",
      "123\t61\t0.0\t2\t0 1 60\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t34\t0.0\t2\t0 0 34\n",
      "126\t5\t0.0\t2\t0 0 5\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t49\t0.0\t2\t0 1 48\n",
      "129\t124\t0.0\t2\t0 1 123\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "131\t11\t0.0\t2\t0 0 11\n",
      "132\t38\t0.0\t2\t0 1 37\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t21\t0.0\t2\t0 0 21\n",
      "138\t105\t0.0\t2\t0 1 104\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t4\t0.0\t2\t0 0 4\n",
      "155\t34\t0.0\t2\t0 0 34\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t85\t0.0\t2\t0 0 85\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t8\t0.0\t2\t0 0 8\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "213\t6\t0.0\t2\t0 0 6\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_274-LP3-ITS-0850_110_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_274-LP3-ITS-0850_110_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_274-LP3-ITS-0850_110_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_274-LP3-ITS-0850_110_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.082 s (12.647 ¬µs/read; 4.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,456\n",
      "Reads with adapters:                        67 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         6,456 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,943,256 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,934,234 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 67 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.8%\n",
      "  C: 56.7%\n",
      "  G: 1.5%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t100.9\t0\t2\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "123\t16\t0.0\t2\t0 0 16\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t8\t0.0\t2\t0 0 8\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "157\t25\t0.0\t2\t0 0 25\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_275-LP3-ITS-0851_1095_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_275-LP3-ITS-0851_1095_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_275-LP3-ITS-0851_1095_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_275-LP3-ITS-0851_1095_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.381 s (11.588 ¬µs/read; 5.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,887\n",
      "Reads with adapters:                       468 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,887 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,898,987 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,844,126 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 468 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 70.1%\n",
      "  C: 28.8%\n",
      "  G: 0.6%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t513.9\t0\t4\n",
      "4\t7\t128.5\t0\t7\n",
      "5\t4\t32.1\t0\t4\n",
      "6\t3\t8.0\t0\t3\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t2\t0.0\t2\t0 0 2\n",
      "64\t11\t0.0\t2\t0 1 10\n",
      "75\t3\t0.0\t2\t0 0 3\n",
      "85\t7\t0.0\t2\t0 0 7\n",
      "96\t11\t0.0\t2\t0 0 11\n",
      "98\t21\t0.0\t2\t0 0 21\n",
      "99\t6\t0.0\t2\t0 0 6\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "107\t13\t0.0\t2\t0 0 13\n",
      "109\t38\t0.0\t2\t0 0 38\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "115\t16\t0.0\t2\t0 0 16\n",
      "116\t25\t0.0\t2\t0 2 23\n",
      "117\t36\t0.0\t2\t0 0 36\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t9\t0.0\t2\t0 0 9\n",
      "125\t6\t0.0\t2\t0 0 6\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "128\t18\t0.0\t2\t0 2 16\n",
      "129\t46\t0.0\t2\t0 1 45\n",
      "131\t42\t0.0\t2\t0 0 42\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t11\t0.0\t2\t0 0 11\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t16\t0.0\t2\t0 0 16\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "163\t25\t0.0\t2\t0 1 24\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_276-LP3-ITS-0852_881_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_276-LP3-ITS-0852_881_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_276-LP3-ITS-0852_881_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_276-LP3-ITS-0852_881_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.314 s (11.589 ¬µs/read; 5.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,062\n",
      "Reads with adapters:                       388 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,062 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,145,662 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,099,276 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 388 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 62.4%\n",
      "  C: 33.8%\n",
      "  G: 2.3%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t422.8\t0\t5\n",
      "6\t3\t6.6\t0\t3\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t17\t0.0\t2\t0 0 17\n",
      "97\t4\t0.0\t2\t0 0 4\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t81\t0.0\t2\t0 1 80\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t10\t0.0\t2\t0 0 10\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t8\t0.0\t2\t0 1 7\n",
      "117\t23\t0.0\t2\t0 1 22\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "123\t12\t0.0\t2\t0 0 12\n",
      "125\t26\t0.0\t2\t0 1 25\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t16\t0.0\t2\t0 0 16\n",
      "129\t36\t0.0\t2\t0 0 36\n",
      "132\t9\t0.0\t2\t0 0 9\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "147\t29\t0.0\t2\t0 0 29\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t6\t0.0\t2\t0 0 6\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "163\t5\t0.0\t2\t0 0 5\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "213\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_277-LP3-ITS-0853_1145_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_277-LP3-ITS-0853_1145_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_277-LP3-ITS-0853_1145_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_277-LP3-ITS-0853_1145_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.427 s (11.132 ¬µs/read; 5.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,364\n",
      "Reads with adapters:                       428 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,364 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,547,564 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,494,763 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 428 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.4%\n",
      "  C: 54.2%\n",
      "  G: 0.7%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t599.4\t0\t2\n",
      "4\t1\t149.9\t0\t1\n",
      "5\t1\t37.5\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "25\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "58\t33\t0.0\t2\t0 0 33\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t11\t0.0\t2\t0 0 11\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t16\t0.0\t2\t0 1 15\n",
      "107\t54\t0.0\t2\t0 1 53\n",
      "109\t8\t0.0\t2\t0 0 8\n",
      "110\t12\t0.0\t2\t0 0 12\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "115\t13\t0.0\t2\t0 1 12\n",
      "118\t22\t0.0\t2\t0 0 22\n",
      "123\t15\t0.0\t2\t0 1 14\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t26\t0.0\t2\t0 0 26\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "138\t6\t0.0\t2\t0 0 6\n",
      "140\t3\t0.0\t2\t0 0 3\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t48\t0.0\t2\t0 0 48\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t6\t0.0\t2\t0 0 6\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t17\t0.0\t2\t0 0 17\n",
      "159\t23\t0.0\t2\t0 1 22\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t22\t0.0\t2\t0 0 22\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "165\t21\t0.0\t2\t0 0 21\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "201\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_278-LP3-ITS-0854_862_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_278-LP3-ITS-0854_862_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_278-LP3-ITS-0854_862_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_278-LP3-ITS-0854_862_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.589 s (11.219 ¬µs/read; 5.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,481\n",
      "Reads with adapters:                       861 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,481 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,796,781 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,689,325 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 861 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.1%\n",
      "  C: 32.4%\n",
      "  G: 1.3%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t820.0\t0\t4\n",
      "4\t2\t205.0\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "91\t20\t0.0\t2\t0 0 20\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t42\t0.0\t2\t0 1 41\n",
      "97\t26\t0.0\t2\t0 1 25\n",
      "98\t13\t0.0\t2\t0 0 13\n",
      "99\t3\t0.0\t2\t0 0 3\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t20\t0.0\t2\t0 1 19\n",
      "108\t51\t0.0\t2\t0 2 49\n",
      "109\t64\t0.0\t2\t0 1 63\n",
      "110\t4\t0.0\t2\t0 0 4\n",
      "111\t11\t0.0\t2\t0 1 10\n",
      "112\t3\t0.0\t2\t0 0 3\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t21\t0.0\t2\t0 1 20\n",
      "118\t36\t0.0\t2\t0 0 36\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t33\t0.0\t2\t0 0 33\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "122\t4\t0.0\t2\t0 0 4\n",
      "123\t45\t0.0\t2\t0 0 45\n",
      "128\t43\t0.0\t2\t0 0 43\n",
      "129\t9\t0.0\t2\t0 0 9\n",
      "130\t39\t0.0\t2\t0 1 38\n",
      "132\t13\t0.0\t2\t0 0 13\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "135\t7\t0.0\t2\t0 0 7\n",
      "137\t41\t0.0\t2\t0 0 41\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t109\t0.0\t2\t0 1 108\n",
      "140\t3\t0.0\t2\t0 0 3\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t6\t0.0\t2\t0 0 6\n",
      "163\t13\t0.0\t2\t0 0 13\n",
      "164\t12\t0.0\t2\t0 0 12\n",
      "165\t39\t0.0\t2\t0 1 38\n",
      "166\t26\t0.0\t2\t0 1 25\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "182\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_279-LP3-ITS-0855_645_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_279-LP3-ITS-0855_645_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_279-LP3-ITS-0855_645_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_279-LP3-ITS-0855_645_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.526 s (11.344 ¬µs/read; 5.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,365\n",
      "Reads with adapters:                       572 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,365 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,955,865 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,887,350 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 572 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.8%\n",
      "  C: 25.7%\n",
      "  G: 4.2%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t724.5\t0\t10\n",
      "4\t11\t181.1\t0\t11\n",
      "9\t1\t0.2\t0\t1\n",
      "11\t2\t0.0\t1\t1 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "67\t3\t0.0\t2\t0 0 3\n",
      "68\t4\t0.0\t2\t0 0 4\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 1 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "96\t62\t0.0\t2\t0 2 60\n",
      "97\t20\t0.0\t2\t0 2 18\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "101\t6\t0.0\t2\t0 0 6\n",
      "102\t4\t0.0\t2\t0 0 4\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "108\t3\t0.0\t2\t0 0 3\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "110\t14\t0.0\t2\t0 0 14\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "112\t15\t0.0\t2\t0 0 15\n",
      "113\t5\t0.0\t2\t0 0 5\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t18\t0.0\t2\t0 0 18\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t6\t0.0\t2\t0 0 6\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t18\t0.0\t2\t0 0 18\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t31\t0.0\t2\t0 0 31\n",
      "129\t18\t0.0\t2\t0 0 18\n",
      "130\t41\t0.0\t2\t0 1 40\n",
      "132\t23\t0.0\t2\t0 0 23\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t34\t0.0\t2\t0 0 34\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "139\t59\t0.0\t2\t0 0 59\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t6\t0.0\t2\t0 0 6\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t3\t0.0\t2\t0 0 3\n",
      "153\t26\t0.0\t2\t0 0 26\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t17\t0.0\t2\t0 0 17\n",
      "164\t9\t0.0\t2\t0 1 8\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_280-LP3-ITS-0856_169_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_280-LP3-ITS-0856_169_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_280-LP3-ITS-0856_169_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_280-LP3-ITS-0856_169_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.947 s (10.319 ¬µs/read; 5.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  91,724\n",
      "Reads with adapters:                     1,027 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        91,724 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,608,924 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     27,494,506 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1027 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 59.4%\n",
      "  C: 38.9%\n",
      "  G: 1.2%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1433.2\t0\t6\n",
      "4\t7\t358.3\t0\t7\n",
      "6\t13\t22.4\t0\t13\n",
      "7\t1\t5.6\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "10\t2\t0.1\t1\t0 2\n",
      "11\t88\t0.0\t1\t2 86\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "24\t71\t0.0\t2\t0 1 70\n",
      "25\t12\t0.0\t2\t0 0 12\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "40\t18\t0.0\t2\t0 0 18\n",
      "41\t30\t0.0\t2\t0 0 30\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t4\t0.0\t2\t0 0 4\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "85\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t71\t0.0\t2\t0 1 70\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t30\t0.0\t2\t0 0 30\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t15\t0.0\t2\t0 0 15\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t36\t0.0\t2\t0 0 36\n",
      "128\t76\t0.0\t2\t0 1 75\n",
      "129\t16\t0.0\t2\t0 0 16\n",
      "130\t23\t0.0\t2\t0 0 23\n",
      "132\t11\t0.0\t2\t0 0 11\n",
      "133\t149\t0.0\t2\t0 2 147\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "139\t2\t0.0\t2\t0 0 2\n",
      "140\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "153\t15\t0.0\t2\t0 0 15\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t31\t0.0\t2\t0 0 31\n",
      "164\t24\t0.0\t2\t0 0 24\n",
      "165\t215\t0.0\t2\t0 2 213\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "184\t8\t0.0\t2\t0 0 8\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "197\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_281-LP3-ITS-0857_654_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_281-LP3-ITS-0857_654_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_281-LP3-ITS-0857_654_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_281-LP3-ITS-0857_654_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.230 s (10.980 ¬µs/read; 5.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,988\n",
      "Reads with adapters:                       286 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,988 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,317,388 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,282,391 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 286 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 72.7%\n",
      "  C: 23.8%\n",
      "  G: 3.1%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t327.9\t0\t2\n",
      "23\t6\t0.0\t2\t0 0 6\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t4\t0.0\t2\t0 0 4\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "115\t9\t0.0\t2\t0 0 9\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t117\t0.0\t2\t0 0 117\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t30\t0.0\t2\t0 0 30\n",
      "132\t8\t0.0\t2\t0 0 8\n",
      "133\t49\t0.0\t2\t0 0 49\n",
      "135\t15\t0.0\t2\t0 0 15\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "147\t4\t0.0\t2\t0 0 4\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_282-LP3-ITS-0858_1087_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_282-LP3-ITS-0858_1087_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_282-LP3-ITS-0858_1087_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_282-LP3-ITS-0858_1087_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.829 s (10.916 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  75,951\n",
      "Reads with adapters:                     5,621 (7.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        75,951 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,861,251 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     22,730,541 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5621 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 13.0%\n",
      "  C: 85.1%\n",
      "  G: 1.8%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4628\t1186.7\t0\t4628\n",
      "4\t20\t296.7\t0\t20\n",
      "5\t1\t74.2\t0\t1\n",
      "6\t1\t18.5\t0\t1\n",
      "7\t1\t4.6\t0\t1\n",
      "11\t6\t0.0\t1\t0 6\n",
      "12\t1\t0.0\t1\t0 1\n",
      "13\t29\t0.0\t1\t0 29\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t5\t0.0\t2\t0 0 5\n",
      "31\t2\t0.0\t2\t0 0 2\n",
      "55\t7\t0.0\t2\t0 0 7\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "67\t5\t0.0\t2\t0 0 5\n",
      "68\t4\t0.0\t2\t0 0 4\n",
      "69\t3\t0.0\t2\t0 0 3\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t5\t0.0\t2\t0 0 5\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "101\t63\t0.0\t2\t0 2 61\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "104\t80\t0.0\t2\t0 0 80\n",
      "107\t15\t0.0\t2\t0 0 15\n",
      "108\t29\t0.0\t2\t0 0 29\n",
      "109\t62\t0.0\t2\t0 0 62\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "115\t57\t0.0\t2\t0 1 56\n",
      "118\t28\t0.0\t2\t0 1 27\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t101\t0.0\t2\t0 1 100\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "125\t20\t0.0\t2\t0 0 20\n",
      "128\t108\t0.0\t2\t0 2 106\n",
      "132\t49\t0.0\t2\t0 0 49\n",
      "133\t50\t0.0\t2\t0 0 50\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "140\t8\t0.0\t2\t0 0 8\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t11\t0.0\t2\t0 0 11\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t11\t0.0\t2\t0 0 11\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t32\t0.0\t2\t0 0 32\n",
      "163\t26\t0.0\t2\t0 0 26\n",
      "164\t16\t0.0\t2\t0 0 16\n",
      "165\t80\t0.0\t2\t0 1 79\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "184\t3\t0.0\t2\t0 0 3\n",
      "185\t2\t0.0\t2\t0 0 2\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_283-LP3-ITS-0859_146_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_283-LP3-ITS-0859_146_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_283-LP3-ITS-0859_146_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_283-LP3-ITS-0859_146_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.542 s (11.263 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,102\n",
      "Reads with adapters:                       689 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,102 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,478,702 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,396,794 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 689 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.0%\n",
      "  C: 13.5%\n",
      "  G: 10.9%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t751.6\t0\t4\n",
      "4\t5\t187.9\t0\t5\n",
      "5\t2\t47.0\t0\t2\n",
      "6\t5\t11.7\t0\t5\n",
      "7\t1\t2.9\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "25\t4\t0.0\t2\t0 0 4\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "41\t22\t0.0\t2\t0 0 22\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t138\t0.0\t2\t0 2 136\n",
      "97\t27\t0.0\t2\t0 1 26\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t24\t0.0\t2\t0 0 24\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "117\t10\t0.0\t2\t0 0 10\n",
      "118\t11\t0.0\t2\t1 0 10\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t6\t0.0\t2\t0 0 6\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "125\t14\t0.0\t2\t0 0 14\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t53\t0.0\t2\t0 1 52\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "132\t70\t0.0\t2\t0 2 68\n",
      "133\t51\t0.0\t2\t0 1 50\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t10\t0.0\t2\t0 0 10\n",
      "138\t14\t0.0\t2\t0 0 14\n",
      "139\t13\t0.0\t2\t0 1 12\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t69\t0.0\t2\t0 1 68\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t8\t0.0\t2\t0 0 8\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t18\t0.0\t2\t0 1 17\n",
      "163\t35\t0.0\t2\t0 0 35\n",
      "165\t24\t0.0\t2\t0 1 23\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "213\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_284-LP3-ITS-0860_398_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_284-LP3-ITS-0860_398_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_284-LP3-ITS-0860_398_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_284-LP3-ITS-0860_398_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.309 s (10.826 ¬µs/read; 5.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,568\n",
      "Reads with adapters:                       300 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,568 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,598,968 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,568,670 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 300 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.3%\n",
      "  C: 21.3%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "25\t23\t0.0\t2\t0 0 23\n",
      "58\t11\t0.0\t2\t0 0 11\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t105\t0.0\t2\t0 4 101\n",
      "101\t23\t0.0\t2\t0 1 22\n",
      "102\t7\t0.0\t2\t0 0 7\n",
      "105\t15\t0.0\t2\t0 0 15\n",
      "106\t8\t0.0\t2\t0 0 8\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t32\t0.0\t2\t0 0 32\n",
      "120\t18\t0.0\t2\t0 1 17\n",
      "122\t7\t0.0\t2\t0 0 7\n",
      "123\t16\t0.0\t2\t0 0 16\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_285-LP3-ITS-0861_367_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_285-LP3-ITS-0861_367_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_285-LP3-ITS-0861_367_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_285-LP3-ITS-0861_367_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.350 s (11.264 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,114\n",
      "Reads with adapters:                       415 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,114 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,365,314 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,318,544 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 415 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.8%\n",
      "  C: 21.4%\n",
      "  G: 0.5%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t486.2\t0\t3\n",
      "4\t17\t121.5\t0\t17\n",
      "8\t1\t0.5\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "41\t4\t0.0\t2\t0 0 4\n",
      "74\t1\t0.0\t2\t0 1\n",
      "94\t12\t0.0\t2\t0 0 12\n",
      "95\t21\t0.0\t2\t0 0 21\n",
      "96\t40\t0.0\t2\t0 0 40\n",
      "97\t6\t0.0\t2\t0 1 5\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t17\t0.0\t2\t0 0 17\n",
      "111\t37\t0.0\t2\t0 0 37\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t24\t0.0\t2\t0 0 24\n",
      "118\t15\t0.0\t2\t0 1 14\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "122\t33\t0.0\t2\t0 0 33\n",
      "123\t54\t0.0\t2\t0 0 54\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t36\t0.0\t2\t0 1 35\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "146\t36\t0.0\t2\t0 0 36\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "197\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_286-LP3-ITS-0862_748_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_286-LP3-ITS-0862_748_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_286-LP3-ITS-0862_748_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_286-LP3-ITS-0862_748_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.388 s (11.376 ¬µs/read; 5.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,110\n",
      "Reads with adapters:                       461 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,267,110 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,210,901 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 461 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.0%\n",
      "  C: 15.4%\n",
      "  G: 4.1%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t533.0\t0\t2\n",
      "6\t3\t8.3\t0\t3\n",
      "7\t1\t2.1\t0\t1\n",
      "25\t5\t0.0\t2\t0 0 5\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "56\t2\t0.0\t2\t0 1 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "78\t16\t0.0\t2\t0 1 15\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t6\t0.0\t2\t0 0 6\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t80\t0.0\t2\t0 1 79\n",
      "107\t30\t0.0\t2\t0 0 30\n",
      "108\t6\t0.0\t2\t0 0 6\n",
      "109\t13\t0.0\t2\t0 1 12\n",
      "116\t8\t0.0\t2\t0 1 7\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "126\t13\t0.0\t2\t0 0 13\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t33\t0.0\t2\t0 0 33\n",
      "129\t9\t0.0\t2\t0 0 9\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t21\t0.0\t2\t0 0 21\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "137\t54\t0.0\t2\t0 1 53\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t10\t0.0\t2\t0 0 10\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t23\t0.0\t2\t0 0 23\n",
      "153\t34\t0.0\t2\t0 0 34\n",
      "155\t9\t0.0\t2\t0 1 8\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "167\t1\t0.0\t2\t0 0 1\n",
      "197\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_287-LP3-ITS-0863_515_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_287-LP3-ITS-0863_515_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_287-LP3-ITS-0863_515_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_287-LP3-ITS-0863_515_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.340 s (11.644 ¬µs/read; 5.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,163\n",
      "Reads with adapters:                       334 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,163 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,778,063 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,740,022 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 334 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.3%\n",
      "  C: 64.7%\n",
      "  G: 2.7%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t455.7\t0\t3\n",
      "6\t1\t7.1\t0\t1\n",
      "25\t2\t0.0\t2\t0 0 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t5\t0.0\t2\t0 0 5\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t194\t0.0\t2\t0 1 193\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t6\t0.0\t2\t0 0 6\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t13\t0.0\t2\t0 0 13\n",
      "123\t16\t0.0\t2\t0 1 15\n",
      "124\t14\t0.0\t2\t0 0 14\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t12\t0.0\t2\t0 0 12\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "132\t7\t0.0\t2\t0 1 6\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "165\t19\t0.0\t2\t0 1 18\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_288-LP3-ITS-0864_121_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_288-LP3-ITS-0864_121_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_288-LP3-ITS-0864_121_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_288-LP3-ITS-0864_121_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (25.955 ¬µs/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     358\n",
      "Reads with adapters:                        10 (2.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           358 (100.0%)\n",
      "\n",
      "Total basepairs processed:       107,758 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        107,049 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t5.6\t0\t3\n",
      "6\t1\t0.1\t0\t1\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_289-LP3-ITS-0865_199_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_289-LP3-ITS-0865_199_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_289-LP3-ITS-0865_199_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_289-LP3-ITS-0865_199_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.413 s (10.792 ¬µs/read; 5.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,274\n",
      "Reads with adapters:                       713 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,274 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,520,474 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,434,557 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 713 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.0%\n",
      "  C: 55.1%\n",
      "  G: 3.5%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t598.0\t0\t4\n",
      "4\t1\t149.5\t0\t1\n",
      "7\t1\t2.3\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "58\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "74\t3\t0.0\t2\t0 0 3\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "98\t152\t0.0\t2\t0 0 152\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "101\t34\t0.0\t2\t0 0 34\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "108\t4\t0.0\t2\t0 0 4\n",
      "109\t91\t0.0\t2\t0 2 89\n",
      "110\t9\t0.0\t2\t0 0 9\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "118\t18\t0.0\t2\t0 0 18\n",
      "120\t5\t0.0\t2\t0 1 4\n",
      "123\t43\t0.0\t2\t0 0 43\n",
      "125\t15\t0.0\t2\t0 0 15\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t34\t0.0\t2\t0 0 34\n",
      "129\t34\t0.0\t2\t0 1 33\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t25\t0.0\t2\t0 0 25\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "137\t47\t0.0\t2\t0 1 46\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "141\t8\t0.0\t2\t0 1 7\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "163\t10\t0.0\t2\t0 0 10\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "165\t57\t0.0\t2\t0 1 56\n",
      "177\t2\t0.0\t2\t0 0 2\n",
      "178\t2\t0.0\t2\t0 0 2\n",
      "190\t1\t0.0\t2\t0 0 1\n",
      "213\t2\t0.0\t2\t0 0 2\n",
      "215\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_290-LP3-ITS-0866_511_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_290-LP3-ITS-0866_511_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_290-LP3-ITS-0866_511_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_290-LP3-ITS-0866_511_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.386 s (11.820 ¬µs/read; 5.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,662\n",
      "Reads with adapters:                       463 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,662 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,831,262 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,772,704 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 463 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.5%\n",
      "  C: 17.7%\n",
      "  G: 5.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t510.3\t0\t5\n",
      "4\t1\t127.6\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "25\t5\t0.0\t2\t0 0 5\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "96\t79\t0.0\t2\t0 1 78\n",
      "97\t8\t0.0\t2\t0 0 8\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t33\t0.0\t2\t0 0 33\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "111\t9\t0.0\t2\t0 0 9\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "115\t10\t0.0\t2\t0 1 9\n",
      "117\t9\t0.0\t2\t0 0 9\n",
      "118\t13\t0.0\t2\t0 1 12\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "128\t20\t0.0\t2\t0 0 20\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t24\t0.0\t2\t0 0 24\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "138\t34\t0.0\t2\t0 0 34\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "140\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t13\t0.0\t2\t0 0 13\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "153\t8\t0.0\t2\t0 0 8\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "163\t21\t0.0\t2\t0 1 20\n",
      "164\t6\t0.0\t2\t0 0 6\n",
      "165\t82\t0.0\t2\t0 1 81\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_291-LP3-ITS-0867_863_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_291-LP3-ITS-0867_863_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_291-LP3-ITS-0867_863_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_291-LP3-ITS-0867_863_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.359 s (12.776 ¬µs/read; 4.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,098\n",
      "Reads with adapters:                       569 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,098 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,457,498 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,396,477 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 569 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.6%\n",
      "  C: 46.7%\n",
      "  G: 0.4%\n",
      "  T: 8.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t39\t439.0\t0\t39\n",
      "4\t9\t109.8\t0\t9\n",
      "5\t5\t27.4\t0\t5\n",
      "6\t1\t6.9\t0\t1\n",
      "17\t1\t0.0\t1\t0 1\n",
      "19\t1\t0.0\t1\t0 0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t3\t0.0\t2\t0 0 3\n",
      "27\t12\t0.0\t2\t0 0 12\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "40\t2\t0.0\t2\t0 1 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "49\t6\t0.0\t2\t0 0 6\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t8\t0.0\t2\t0 0 8\n",
      "53\t5\t0.0\t2\t0 0 5\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "55\t4\t0.0\t2\t0 0 4\n",
      "56\t7\t0.0\t2\t0 0 7\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t16\t0.0\t2\t0 0 16\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t4\t0.0\t2\t0 1 3\n",
      "66\t6\t0.0\t2\t0 0 6\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "74\t2\t0.0\t2\t0 0 2\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "96\t5\t0.0\t2\t0 0 5\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "104\t22\t0.0\t2\t0 0 22\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t10\t0.0\t2\t0 1 9\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "114\t2\t0.0\t2\t0 0 2\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t6\t0.0\t2\t0 0 6\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t2\t0.0\t2\t0 0 2\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "138\t10\t0.0\t2\t0 0 10\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t27\t0.0\t2\t0 0 27\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "144\t4\t0.0\t2\t0 0 4\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "147\t7\t0.0\t2\t0 0 7\n",
      "148\t5\t0.0\t2\t0 0 5\n",
      "149\t9\t0.0\t2\t0 0 9\n",
      "150\t19\t0.0\t2\t0 1 18\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t16\t0.0\t2\t0 0 16\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "156\t34\t0.0\t2\t0 1 33\n",
      "157\t7\t0.0\t2\t0 0 7\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "159\t12\t0.0\t2\t0 0 12\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "166\t2\t0.0\t2\t0 0 2\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "177\t2\t0.0\t2\t0 0 2\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "181\t3\t0.0\t2\t0 0 3\n",
      "190\t2\t0.0\t2\t0 0 2\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "200\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_292-LP3-ITS-0868_187_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_292-LP3-ITS-0868_187_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_292-LP3-ITS-0868_187_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_292-LP3-ITS-0868_187_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.306 s (13.357 ¬µs/read; 4.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,899\n",
      "Reads with adapters:                       487 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,899 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,892,599 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,854,899 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 487 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.9%\n",
      "  C: 51.3%\n",
      "  G: 0.6%\n",
      "  T: 22.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t79\t357.8\t0\t79\n",
      "4\t22\t89.4\t0\t22\n",
      "5\t6\t22.4\t0\t6\n",
      "6\t17\t5.6\t0\t17\n",
      "7\t1\t1.4\t0\t1\n",
      "8\t1\t0.3\t0\t1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "30\t3\t0.0\t2\t0 0 3\n",
      "31\t1\t0.0\t2\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "33\t7\t0.0\t2\t0 0 7\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t5\t0.0\t2\t0 0 5\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t3\t0.0\t2\t0 0 3\n",
      "50\t4\t0.0\t2\t0 0 4\n",
      "51\t18\t0.0\t2\t0 0 18\n",
      "52\t27\t0.0\t2\t0 0 27\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t8\t0.0\t2\t0 0 8\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "74\t6\t0.0\t2\t0 0 6\n",
      "76\t6\t0.0\t2\t0 1 5\n",
      "78\t4\t0.0\t2\t0 1 3\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "97\t7\t0.0\t2\t0 0 7\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "102\t3\t0.0\t2\t0 0 3\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "106\t11\t0.0\t2\t0 0 11\n",
      "107\t4\t0.0\t2\t0 0 4\n",
      "108\t4\t0.0\t2\t0 0 4\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t11\t0.0\t2\t0 0 11\n",
      "118\t8\t0.0\t2\t0 1 7\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "123\t4\t0.0\t2\t0 0 4\n",
      "124\t17\t0.0\t2\t0 0 17\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t6\t0.0\t2\t0 0 6\n",
      "139\t8\t0.0\t2\t0 0 8\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t3\t0.0\t2\t0 0 3\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t6\t0.0\t2\t0 0 6\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "162\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "172\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_293-LP3-ITS-0869_594_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_293-LP3-ITS-0869_594_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_293-LP3-ITS-0869_594_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_293-LP3-ITS-0869_594_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.323 s (13.022 ¬µs/read; 4.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,769\n",
      "Reads with adapters:                       380 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,769 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,455,469 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,415,019 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 380 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.4%\n",
      "  C: 38.7%\n",
      "  G: 2.6%\n",
      "  T: 10.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t23\t387.0\t0\t23\n",
      "4\t7\t96.8\t0\t7\n",
      "5\t2\t24.2\t0\t2\n",
      "7\t4\t1.5\t0\t4\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t8\t0.0\t2\t0 0 8\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t2\t0.0\t2\t0 0 2\n",
      "34\t2\t0.0\t2\t0 1 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "38\t10\t0.0\t2\t0 0 10\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "62\t3\t0.0\t2\t0 0 3\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "101\t7\t0.0\t2\t0 1 6\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t8\t0.0\t2\t0 1 7\n",
      "105\t6\t0.0\t2\t0 0 6\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "111\t2\t0.0\t2\t0 0 2\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "114\t4\t0.0\t2\t0 0 4\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "125\t10\t0.0\t2\t0 0 10\n",
      "126\t6\t0.0\t2\t0 0 6\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t10\t0.0\t2\t0 0 10\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t14\t0.0\t2\t0 0 14\n",
      "142\t16\t0.0\t2\t0 0 16\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t4\t0.0\t2\t0 0 4\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t11\t0.0\t2\t0 2 9\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t3\t0.0\t2\t0 0 3\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "156\t42\t0.0\t2\t0 0 42\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "159\t8\t0.0\t2\t0 1 7\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "179\t2\t0.0\t2\t0 0 2\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_294-LP3-ITS-0870_574_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_294-LP3-ITS-0870_574_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_294-LP3-ITS-0870_574_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_294-LP3-ITS-0870_574_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (46.477 ¬µs/read; 1.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     132\n",
      "Reads with adapters:                        10 (7.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           132 (100.0%)\n",
      "\n",
      "Total basepairs processed:        39,732 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         39,350 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 40.0%\n",
      "  G: 0.0%\n",
      "  T: 10.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t2.1\t0\t6\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_295-LP3-ITS-0871_738_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_295-LP3-ITS-0871_738_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_295-LP3-ITS-0871_738_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_295-LP3-ITS-0871_738_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (52.832 ¬µs/read; 1.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     145\n",
      "Reads with adapters:                        10 (6.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           145 (100.0%)\n",
      "\n",
      "Total basepairs processed:        43,645 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         43,126 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t2.3\t0\t4\n",
      "10\t2\t0.0\t1\t0 2\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_296-LP3-ITS-0872_123_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_296-LP3-ITS-0872_123_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_296-LP3-ITS-0872_123_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_296-LP3-ITS-0872_123_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.323 s (12.550 ¬µs/read; 4.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,763\n",
      "Reads with adapters:                       474 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,763 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,754,663 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,711,954 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 474 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.8%\n",
      "  C: 56.5%\n",
      "  G: 0.6%\n",
      "  T: 5.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t402.5\t0\t31\n",
      "4\t21\t100.6\t0\t21\n",
      "5\t5\t25.2\t0\t5\n",
      "6\t10\t6.3\t0\t10\n",
      "7\t4\t1.6\t0\t4\n",
      "8\t1\t0.4\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t4\t0.0\t1\t0 4\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "27\t6\t0.0\t2\t0 1 5\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "30\t2\t0.0\t2\t0 1 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "33\t2\t0.0\t2\t0 0 2\n",
      "37\t2\t0.0\t2\t0 0 2\n",
      "39\t4\t0.0\t2\t0 0 4\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t29\t0.0\t2\t0 1 28\n",
      "58\t15\t0.0\t2\t0 0 15\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "62\t2\t0.0\t2\t0 0 2\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "81\t4\t0.0\t2\t0 1 3\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "86\t3\t0.0\t2\t0 0 3\n",
      "89\t59\t0.0\t2\t0 0 59\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t13\t0.0\t2\t0 0 13\n",
      "95\t4\t0.0\t2\t0 1 3\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t6\t0.0\t2\t0 0 6\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t4\t0.0\t2\t0 0 4\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t7\t0.0\t2\t0 0 7\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t12\t0.0\t2\t0 0 12\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t19\t0.0\t2\t0 0 19\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t3\t0.0\t2\t0 0 3\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "156\t26\t0.0\t2\t0 0 26\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "158\t17\t0.0\t2\t0 0 17\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "177\t3\t0.0\t2\t0 0 3\n",
      "178\t5\t0.0\t2\t0 0 5\n",
      "190\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_297-LP3-ITS-0873_250_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_297-LP3-ITS-0873_250_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_297-LP3-ITS-0873_250_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_297-LP3-ITS-0873_250_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.402 s (13.699 ¬µs/read; 4.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,358\n",
      "Reads with adapters:                       553 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,358 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,836,758 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,783,047 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 553 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.1%\n",
      "  C: 47.6%\n",
      "  G: 1.6%\n",
      "  T: 13.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t458.7\t0\t58\n",
      "4\t34\t114.7\t0\t34\n",
      "5\t3\t28.7\t0\t3\n",
      "6\t4\t7.2\t0\t4\n",
      "7\t1\t1.8\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t3\t0.0\t1\t1 2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t0 2\n",
      "21\t4\t0.0\t2\t0 1 3\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "39\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t7\t0.0\t2\t0 1 6\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t13\t0.0\t2\t0 0 13\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "65\t3\t0.0\t2\t0 0 3\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "70\t1\t0.0\t2\t0 0 1\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "74\t5\t0.0\t2\t0 0 5\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "76\t5\t0.0\t2\t0 0 5\n",
      "78\t3\t0.0\t2\t0 0 3\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t4\t0.0\t2\t0 0 4\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t4\t0.0\t2\t0 0 4\n",
      "98\t3\t0.0\t2\t0 0 3\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t6\t0.0\t2\t0 0 6\n",
      "104\t17\t0.0\t2\t0 0 17\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t10\t0.0\t2\t0 0 10\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t3\t0.0\t2\t0 0 3\n",
      "109\t6\t0.0\t2\t0 0 6\n",
      "111\t2\t0.0\t2\t0 1 1\n",
      "112\t3\t0.0\t2\t0 0 3\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "116\t11\t0.0\t2\t0 0 11\n",
      "118\t5\t0.0\t2\t0 0 5\n",
      "119\t13\t0.0\t2\t0 1 12\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t18\t0.0\t2\t0 0 18\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t8\t0.0\t2\t0 0 8\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t5\t0.0\t2\t0 0 5\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "138\t5\t0.0\t2\t0 0 5\n",
      "139\t9\t0.0\t2\t0 0 9\n",
      "142\t32\t0.0\t2\t0 0 32\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t18\t0.0\t2\t0 0 18\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t2\t0.0\t2\t0 0 2\n",
      "149\t24\t0.0\t2\t0 0 24\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t18\t0.0\t2\t0 0 18\n",
      "152\t10\t0.0\t2\t0 0 10\n",
      "153\t9\t0.0\t2\t0 0 9\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "156\t4\t0.0\t2\t0 0 4\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "159\t10\t0.0\t2\t0 0 10\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "176\t1\t0.0\t2\t0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "191\t2\t0.0\t2\t0 0 2\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "207\t1\t0.0\t2\t0 0 1\n",
      "227\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_298-LP3-ITS-0874_152_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_298-LP3-ITS-0874_152_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_298-LP3-ITS-0874_152_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_298-LP3-ITS-0874_152_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.266 s (12.441 ¬µs/read; 4.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,407\n",
      "Reads with adapters:                       404 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,407 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,443,507 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,408,050 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 404 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.6%\n",
      "  C: 57.7%\n",
      "  G: 0.7%\n",
      "  T: 4.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t334.5\t0\t24\n",
      "4\t18\t83.6\t0\t18\n",
      "5\t55\t20.9\t0\t55\n",
      "6\t13\t5.2\t0\t13\n",
      "7\t1\t1.3\t0\t1\n",
      "8\t1\t0.3\t0\t1\n",
      "11\t5\t0.0\t1\t0 5\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "56\t2\t0.0\t2\t0 1 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "63\t3\t0.0\t2\t0 0 3\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t4\t0.0\t2\t0 0 4\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t4\t0.0\t2\t0 0 4\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "81\t3\t0.0\t2\t0 0 3\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t16\t0.0\t2\t0 0 16\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "105\t2\t0.0\t2\t0 1 1\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t4\t0.0\t2\t0 0 4\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t10\t0.0\t2\t0 1 9\n",
      "134\t5\t0.0\t2\t0 0 5\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t9\t0.0\t2\t0 1 8\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t16\t0.0\t2\t0 0 16\n",
      "151\t11\t0.0\t2\t0 1 10\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t22\t0.0\t2\t0 0 22\n",
      "157\t16\t0.0\t2\t1 0 15\n",
      "158\t13\t0.0\t2\t0 0 13\n",
      "159\t17\t0.0\t2\t0 0 17\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "173\t3\t0.0\t2\t0 0 3\n",
      "178\t1\t0.0\t2\t0 0 1\n",
      "191\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_299-LP3-ITS-0875_340_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_299-LP3-ITS-0875_340_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_299-LP3-ITS-0875_340_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_299-LP3-ITS-0875_340_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.243 s (12.837 ¬µs/read; 4.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,933\n",
      "Reads with adapters:                       875 (4.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,933 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,698,833 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,668,490 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 875 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.6%\n",
      "  C: 44.5%\n",
      "  G: 0.9%\n",
      "  T: 2.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t388\t295.8\t0\t388\n",
      "4\t1\t74.0\t0\t1\n",
      "6\t210\t4.6\t0\t210\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t15\t0.0\t2\t0 0 15\n",
      "52\t7\t0.0\t2\t0 0 7\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "55\t4\t0.0\t2\t0 0 4\n",
      "56\t8\t0.0\t2\t0 0 8\n",
      "60\t20\t0.0\t2\t0 1 19\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "63\t11\t0.0\t2\t0 0 11\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t7\t0.0\t2\t0 1 6\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 1\n",
      "96\t52\t0.0\t2\t0 1 51\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "100\t3\t0.0\t2\t0 0 3\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t11\t0.0\t2\t0 0 11\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t31\t0.0\t2\t0 0 31\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "165\t2\t0.0\t2\t0 0 2\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_300-LP3-ITS-0876_717_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_300-LP3-ITS-0876_717_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_300-LP3-ITS-0876_717_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_300-LP3-ITS-0876_717_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.282 s (12.546 ¬µs/read; 4.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,462\n",
      "Reads with adapters:                     1,107 (4.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,462 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,761,062 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,715,214 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1107 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 15.7%\n",
      "  C: 77.9%\n",
      "  G: 0.8%\n",
      "  T: 5.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t351.0\t0\t46\n",
      "4\t34\t87.7\t0\t34\n",
      "5\t3\t21.9\t0\t3\n",
      "6\t650\t5.5\t0\t650\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "45\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t9\t0.0\t2\t0 1 8\n",
      "53\t12\t0.0\t2\t0 0 12\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "56\t20\t0.0\t2\t0 2 18\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "60\t5\t0.0\t2\t0 0 5\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "63\t27\t0.0\t2\t0 1 26\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "79\t4\t0.0\t2\t0 0 4\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "98\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "100\t9\t0.0\t2\t0 2 7\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t4\t0.0\t2\t0 0 4\n",
      "107\t8\t0.0\t2\t0 0 8\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t5\t0.0\t2\t0 1 4\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "124\t11\t0.0\t2\t0 0 11\n",
      "125\t4\t0.0\t2\t0 0 4\n",
      "126\t18\t0.0\t2\t0 0 18\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "130\t5\t0.0\t2\t0 0 5\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "139\t3\t0.0\t2\t0 0 3\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "145\t44\t0.0\t2\t0 0 44\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t7\t0.0\t2\t0 0 7\n",
      "149\t16\t0.0\t2\t0 0 16\n",
      "150\t5\t0.0\t2\t0 0 5\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "152\t14\t0.0\t2\t0 0 14\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "191\t3\t0.0\t2\t0 0 3\n",
      "243\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_301-LP3-ITS-0877_363_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_301-LP3-ITS-0877_363_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_301-LP3-ITS-0877_363_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_301-LP3-ITS-0877_363_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (117.392 ¬µs/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      45\n",
      "Reads with adapters:                         2 (4.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            45 (100.0%)\n",
      "\n",
      "Total basepairs processed:        13,545 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         13,539 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t0.7\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_302-LP3-ITS-0878_1044_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_302-LP3-ITS-0878_1044_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_302-LP3-ITS-0878_1044_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_302-LP3-ITS-0878_1044_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.257 s (12.845 ¬µs/read; 4.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,017\n",
      "Reads with adapters:                       477 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,017 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,025,117 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,992,831 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 477 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.8%\n",
      "  C: 68.6%\n",
      "  G: 1.0%\n",
      "  T: 3.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t312.8\t0\t20\n",
      "4\t28\t78.2\t0\t28\n",
      "5\t88\t19.5\t0\t88\n",
      "6\t25\t4.9\t0\t25\n",
      "7\t1\t1.2\t0\t1\n",
      "11\t11\t0.0\t1\t1 10\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "27\t4\t0.0\t2\t0 2 2\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 1 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "57\t31\t0.0\t2\t0 0 31\n",
      "58\t46\t0.0\t2\t0 1 45\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "63\t24\t0.0\t2\t0 0 24\n",
      "66\t8\t0.0\t2\t0 0 8\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "76\t4\t0.0\t2\t0 0 4\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "95\t1\t0.0\t2\t0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "104\t8\t0.0\t2\t0 0 8\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "139\t4\t0.0\t2\t0 0 4\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t17\t0.0\t2\t0 0 17\n",
      "151\t10\t0.0\t2\t0 0 10\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "156\t5\t0.0\t2\t0 0 5\n",
      "157\t11\t0.0\t2\t0 0 11\n",
      "158\t14\t0.0\t2\t0 0 14\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_303-LP3-ITS-0879_182_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_303-LP3-ITS-0879_182_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_303-LP3-ITS-0879_182_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_303-LP3-ITS-0879_182_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.437 s (12.129 ¬µs/read; 4.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,036\n",
      "Reads with adapters:                       771 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,036 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,846,836 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,778,394 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 771 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.2%\n",
      "  C: 47.7%\n",
      "  G: 1.3%\n",
      "  T: 10.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t125\t563.1\t0\t125\n",
      "4\t28\t140.8\t0\t28\n",
      "5\t15\t35.2\t0\t15\n",
      "6\t12\t8.8\t0\t12\n",
      "7\t2\t2.2\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t6\t0.0\t1\t1 5\n",
      "11\t2\t0.0\t1\t0 2\n",
      "12\t1\t0.0\t1\t0 1\n",
      "19\t1\t0.0\t1\t0 0 1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "27\t11\t0.0\t2\t0 1 10\n",
      "30\t13\t0.0\t2\t0 0 13\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "37\t3\t0.0\t2\t0 0 3\n",
      "39\t3\t0.0\t2\t0 0 3\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "41\t2\t0.0\t2\t0 0 2\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "47\t3\t0.0\t2\t0 0 3\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t6\t0.0\t2\t0 0 6\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t14\t0.0\t2\t0 1 13\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t9\t0.0\t2\t0 0 9\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "62\t2\t0.0\t2\t0 1 1\n",
      "63\t11\t0.0\t2\t0 1 10\n",
      "64\t2\t0.0\t2\t0 0 2\n",
      "66\t7\t0.0\t2\t0 0 7\n",
      "69\t2\t0.0\t2\t0 0 2\n",
      "71\t1\t0.0\t2\t0 0 1\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "73\t2\t0.0\t2\t0 0 2\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "78\t4\t0.0\t2\t0 0 4\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t26\t0.0\t2\t0 0 26\n",
      "95\t6\t0.0\t2\t0 0 6\n",
      "96\t20\t0.0\t2\t0 2 18\n",
      "97\t2\t0.0\t2\t0 0 2\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "100\t9\t0.0\t2\t0 0 9\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t2\t0.0\t2\t0 0 2\n",
      "107\t5\t0.0\t2\t0 0 5\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "118\t16\t0.0\t2\t0 0 16\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t7\t0.0\t2\t0 0 7\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t6\t0.0\t2\t0 0 6\n",
      "130\t3\t0.0\t2\t0 0 3\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "135\t5\t0.0\t2\t0 0 5\n",
      "136\t7\t0.0\t2\t0 1 6\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "138\t14\t0.0\t2\t0 1 13\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "140\t2\t0.0\t2\t0 0 2\n",
      "141\t2\t0.0\t2\t0 1 1\n",
      "142\t65\t0.0\t2\t1 2 62\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t6\t0.0\t2\t0 0 6\n",
      "148\t8\t0.0\t2\t0 0 8\n",
      "149\t5\t0.0\t2\t0 0 5\n",
      "150\t21\t0.0\t2\t0 0 21\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "152\t7\t0.0\t2\t0 1 6\n",
      "154\t7\t0.0\t2\t0 0 7\n",
      "156\t28\t0.0\t2\t0 0 28\n",
      "157\t9\t0.0\t2\t0 0 9\n",
      "158\t25\t0.0\t2\t0 0 25\n",
      "159\t15\t0.0\t2\t0 0 15\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t2\t0.0\t2\t0 0 2\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "165\t4\t0.0\t2\t0 0 4\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "178\t14\t0.0\t2\t0 0 14\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "183\t2\t0.0\t2\t0 0 2\n",
      "205\t1\t0.0\t2\t0 0 1\n",
      "207\t2\t0.0\t2\t0 0 2\n",
      "220\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_304-LP3-ITS-0880_638_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_304-LP3-ITS-0880_638_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_304-LP3-ITS-0880_638_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_304-LP3-ITS-0880_638_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.374 s (12.395 ¬µs/read; 4.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,210\n",
      "Reads with adapters:                       410 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,093,210 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,043,593 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 410 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.4%\n",
      "  C: 42.0%\n",
      "  G: 1.2%\n",
      "  T: 13.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t472.0\t0\t12\n",
      "4\t3\t118.0\t0\t3\n",
      "5\t1\t29.5\t0\t1\n",
      "20\t6\t0.0\t2\t0 0 6\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "41\t3\t0.0\t2\t0 0 3\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "49\t3\t0.0\t2\t0 0 3\n",
      "51\t4\t0.0\t2\t0 0 4\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "60\t3\t0.0\t2\t0 1 2\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "73\t6\t0.0\t2\t0 0 6\n",
      "75\t2\t0.0\t2\t0 0 2\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "83\t3\t0.0\t2\t0 0 3\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t6\t0.0\t2\t0 0 6\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t5\t0.0\t2\t0 0 5\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t5\t0.0\t2\t0 0 5\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t3\t0.0\t2\t0 0 3\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t17\t0.0\t2\t0 0 17\n",
      "115\t3\t0.0\t2\t0 0 3\n",
      "116\t3\t0.0\t2\t0 0 3\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t37\t0.0\t2\t0 1 36\n",
      "120\t3\t0.0\t2\t0 0 3\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t9\t0.0\t2\t0 0 9\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t3\t0.0\t2\t0 0 3\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 1 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "138\t4\t0.0\t2\t0 0 4\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t21\t0.0\t2\t0 0 21\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t19\t0.0\t2\t0 1 18\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t9\t0.0\t2\t0 0 9\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "152\t19\t0.0\t2\t0 0 19\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "156\t47\t0.0\t2\t0 0 47\n",
      "157\t5\t0.0\t2\t0 0 5\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t13\t0.0\t2\t0 0 13\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "180\t9\t0.0\t2\t0 0 9\n",
      "181\t1\t0.0\t2\t0 0 1\n",
      "182\t1\t0.0\t2\t0 0 1\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "207\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_305-LP3-ITS-0881_547_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_305-LP3-ITS-0881_547_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_305-LP3-ITS-0881_547_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_305-LP3-ITS-0881_547_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.406 s (12.164 ¬µs/read; 4.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,374\n",
      "Reads with adapters:                       602 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,374 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,045,574 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,984,182 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 602 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.4%\n",
      "  C: 49.2%\n",
      "  G: 2.0%\n",
      "  T: 8.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t53\t521.5\t0\t53\n",
      "4\t35\t130.4\t0\t35\n",
      "5\t19\t32.6\t0\t19\n",
      "6\t2\t8.1\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t5\t0.0\t1\t0 5\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "27\t9\t0.0\t2\t0 0 9\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "38\t1\t0.0\t2\t0 1\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "45\t5\t0.0\t2\t0 0 5\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "48\t2\t0.0\t2\t0 0 2\n",
      "49\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "54\t2\t0.0\t2\t0 0 2\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t3\t0.0\t2\t0 0 3\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "60\t4\t0.0\t2\t0 0 4\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "65\t1\t0.0\t2\t0 0 1\n",
      "66\t9\t0.0\t2\t0 0 9\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "70\t3\t0.0\t2\t0 1 2\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "73\t6\t0.0\t2\t0 0 6\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t4\t0.0\t2\t0 0 4\n",
      "78\t4\t0.0\t2\t0 0 4\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "89\t3\t0.0\t2\t0 0 3\n",
      "90\t7\t0.0\t2\t0 0 7\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "94\t8\t0.0\t2\t0 0 8\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "105\t3\t0.0\t2\t0 0 3\n",
      "106\t8\t0.0\t2\t0 0 8\n",
      "107\t3\t0.0\t2\t0 0 3\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "116\t9\t0.0\t2\t0 1 8\n",
      "118\t9\t0.0\t2\t0 0 9\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "125\t3\t0.0\t2\t0 0 3\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t7\t0.0\t2\t0 0 7\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "136\t11\t0.0\t2\t0 1 10\n",
      "137\t12\t0.0\t2\t0 1 11\n",
      "138\t3\t0.0\t2\t0 0 3\n",
      "139\t21\t0.0\t2\t0 0 21\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t33\t0.0\t2\t0 0 33\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "147\t10\t0.0\t2\t0 0 10\n",
      "148\t3\t0.0\t2\t0 0 3\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t21\t0.0\t2\t0 0 21\n",
      "151\t12\t0.0\t2\t0 0 12\n",
      "152\t8\t0.0\t2\t0 0 8\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t28\t0.0\t2\t0 0 28\n",
      "156\t35\t0.0\t2\t0 0 35\n",
      "157\t22\t0.0\t2\t0 0 22\n",
      "158\t12\t0.0\t2\t0 0 12\n",
      "159\t19\t0.0\t2\t0 0 19\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "162\t4\t0.0\t2\t0 0 4\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "164\t3\t0.0\t2\t0 0 3\n",
      "165\t1\t0.0\t2\t0 0 1\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "170\t1\t0.0\t2\t0 0 1\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "191\t3\t0.0\t2\t0 0 3\n",
      "195\t1\t0.0\t2\t0 0 1\n",
      "207\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_306-LP3-ITS-0882_459_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_306-LP3-ITS-0882_459_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_306-LP3-ITS-0882_459_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_306-LP3-ITS-0882_459_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.238 s (13.521 ¬µs/read; 4.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,630\n",
      "Reads with adapters:                       312 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,306,630 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,276,471 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 312 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.9%\n",
      "  C: 45.8%\n",
      "  G: 1.3%\n",
      "  T: 9.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t275.5\t0\t17\n",
      "4\t10\t68.9\t0\t10\n",
      "5\t8\t17.2\t0\t8\n",
      "6\t9\t4.3\t0\t9\n",
      "7\t1\t1.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "13\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t3\t0.0\t2\t0 0 3\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "30\t4\t0.0\t2\t0 0 4\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "35\t4\t0.0\t2\t0 0 4\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t14\t0.0\t2\t0 1 13\n",
      "57\t1\t0.0\t2\t0 0 1\n",
      "58\t5\t0.0\t2\t0 1 4\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t5\t0.0\t2\t0 1 4\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "91\t3\t0.0\t2\t0 0 3\n",
      "92\t4\t0.0\t2\t0 0 4\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "94\t11\t0.0\t2\t0 0 11\n",
      "96\t10\t0.0\t2\t0 0 10\n",
      "100\t4\t0.0\t2\t0 0 4\n",
      "104\t2\t0.0\t2\t0 1 1\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "126\t2\t0.0\t2\t0 0 2\n",
      "127\t2\t0.0\t2\t0 0 2\n",
      "128\t14\t0.0\t2\t0 0 14\n",
      "130\t4\t0.0\t2\t0 0 4\n",
      "131\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 1\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "136\t6\t0.0\t2\t0 0 6\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "138\t9\t0.0\t2\t0 1 8\n",
      "139\t11\t0.0\t2\t0 0 11\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t21\t0.0\t2\t0 0 21\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "150\t8\t0.0\t2\t0 0 8\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "156\t20\t0.0\t2\t0 0 20\n",
      "157\t3\t0.0\t2\t0 1 2\n",
      "158\t7\t0.0\t2\t0 0 7\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "168\t1\t0.0\t2\t0 0 1\n",
      "178\t2\t0.0\t2\t1 0 1\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_307-LP3-ITS-0883_607_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_307-LP3-ITS-0883_607_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_307-LP3-ITS-0883_607_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_307-LP3-ITS-0883_607_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.367 s (10.932 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,605\n",
      "Reads with adapters:                       451 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,605 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,115,105 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,058,987 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 451 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.1%\n",
      "  C: 10.2%\n",
      "  G: 0.0%\n",
      "  T: 0.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t525.1\t0\t5\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "53\t4\t0.0\t2\t0 0 4\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t295\t0.0\t2\t0 7 288\n",
      "133\t24\t0.0\t2\t0 0 24\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t4\t0.0\t2\t0 1 3\n",
      "151\t80\t0.0\t2\t0 0 80\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_308-LP3-ITS-0884_688_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_308-LP3-ITS-0884_688_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_308-LP3-ITS-0884_688_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_308-LP3-ITS-0884_688_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.188 s (10.622 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,727\n",
      "Reads with adapters:                       250 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,727 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,335,827 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,311,123 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 250 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.4%\n",
      "  C: 16.8%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t277.0\t0\t40\n",
      "6\t1\t4.3\t0\t1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "56\t7\t0.0\t2\t0 0 7\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t156\t0.0\t2\t0 1 155\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "149\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_309-LP3-ITS-0885_544_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_309-LP3-ITS-0885_544_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_309-LP3-ITS-0885_544_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_309-LP3-ITS-0885_544_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.269 s (10.197 ¬µs/read; 5.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,359\n",
      "Reads with adapters:                       461 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,934,059 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,904,387 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 461 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.0%\n",
      "  C: 14.3%\n",
      "  G: 0.2%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t208\t411.9\t0\t208\n",
      "4\t2\t103.0\t0\t2\n",
      "11\t1\t0.0\t1\t0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "59\t5\t0.0\t2\t0 0 5\n",
      "63\t13\t0.0\t2\t0 0 13\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "67\t5\t0.0\t2\t0 0 5\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "118\t176\t0.0\t2\t0 4 172\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 1 4\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_310-LP3-ITS-0886_139_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_310-LP3-ITS-0886_139_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_310-LP3-ITS-0886_139_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_310-LP3-ITS-0886_139_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.298 s (10.544 ¬µs/read; 5.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,263\n",
      "Reads with adapters:                       246 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,263 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,507,163 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,478,335 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 246 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.4%\n",
      "  C: 11.4%\n",
      "  G: 0.4%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t441.6\t0\t2\n",
      "4\t1\t110.4\t0\t1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "117\t4\t0.0\t2\t0 0 4\n",
      "118\t208\t0.0\t2\t0 2 206\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_311-LP3-ITS-0887_841_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_311-LP3-ITS-0887_841_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_311-LP3-ITS-0887_841_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_311-LP3-ITS-0887_841_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.378 s (10.092 ¬µs/read; 5.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,425\n",
      "Reads with adapters:                       325 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,425 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,264,925 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,226,125 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 325 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 98.2%\n",
      "  C: 1.8%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t584.8\t0\t3\n",
      "4\t1\t146.2\t0\t1\n",
      "118\t285\t0.0\t2\t0 2 283\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "145\t29\t0.0\t2\t0 0 29\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_312-LP3-ITS-0888_178_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_312-LP3-ITS-0888_178_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_312-LP3-ITS-0888_178_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_312-LP3-ITS-0888_178_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.233 s (11.878 ¬µs/read; 5.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,575\n",
      "Reads with adapters:                       258 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,575 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,892,075 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,868,626 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 258 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 53.1%\n",
      "  C: 45.0%\n",
      "  G: 0.8%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t305.9\t0\t5\n",
      "9\t1\t0.1\t0\t0 1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "26\t6\t0.0\t2\t0 0 6\n",
      "27\t71\t0.0\t2\t0 2 69\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "43\t4\t0.0\t2\t0 1 3\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "89\t5\t0.0\t2\t0 0 5\n",
      "90\t3\t0.0\t2\t0 0 3\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "114\t8\t0.0\t2\t0 0 8\n",
      "118\t54\t0.0\t2\t1 0 53\n",
      "119\t8\t0.0\t2\t0 1 7\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "138\t5\t0.0\t2\t0 0 5\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t24\t0.0\t2\t0 0 24\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "156\t28\t0.0\t2\t0 0 28\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "171\t1\t0.0\t2\t0 0 1\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_313-LP3-ITS-0889_879_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_313-LP3-ITS-0889_879_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_313-LP3-ITS-0889_879_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_313-LP3-ITS-0889_879_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.255 s (11.092 ¬µs/read; 5.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,987\n",
      "Reads with adapters:                       216 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,987 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,919,087 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,894,328 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 216 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.4%\n",
      "  C: 9.7%\n",
      "  G: 0.0%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t359.2\t0\t8\n",
      "10\t1\t0.0\t1\t1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "118\t189\t0.0\t2\t0 4 185\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_314-LP3-ITS-0890_115_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_314-LP3-ITS-0890_115_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_314-LP3-ITS-0890_115_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_314-LP3-ITS-0890_115_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.010 s (28.446 ¬µs/read; 2.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     357\n",
      "Reads with adapters:                        13 (3.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           357 (100.0%)\n",
      "\n",
      "Total basepairs processed:       107,457 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        106,400 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.9%\n",
      "  C: 23.1%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t5.6\t0\t2\n",
      "4\t1\t1.4\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "118\t6\t0.0\t2\t0 0 6\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_315-LP3-ITS-0891_35_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_315-LP3-ITS-0891_35_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_315-LP3-ITS-0891_35_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_315-LP3-ITS-0891_35_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.484 s (10.552 ¬µs/read; 5.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,830\n",
      "Reads with adapters:                       938 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,830 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,794,830 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,707,127 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 938 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.8%\n",
      "  C: 58.2%\n",
      "  G: 0.9%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t716.1\t0\t1\n",
      "9\t1\t0.2\t0\t1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t324\t0.0\t2\t0 2 322\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t89\t0.0\t2\t0 3 86\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "76\t26\t0.0\t2\t0 0 26\n",
      "81\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t6\t0.0\t2\t0 1 5\n",
      "97\t3\t0.0\t2\t0 0 3\n",
      "98\t11\t0.0\t2\t0 0 11\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "105\t10\t0.0\t2\t0 0 10\n",
      "106\t31\t0.0\t2\t0 2 29\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "117\t5\t0.0\t2\t0 0 5\n",
      "118\t169\t0.0\t2\t0 6 163\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "128\t11\t0.0\t2\t0 0 11\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t46\t0.0\t2\t0 0 46\n",
      "134\t16\t0.0\t2\t0 0 16\n",
      "137\t21\t0.0\t2\t0 0 21\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "147\t5\t0.0\t2\t0 0 5\n",
      "151\t122\t0.0\t2\t0 1 121\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "154\t2\t0.0\t2\t0 0 2\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_316-LP3-ITS-0892_625_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_316-LP3-ITS-0892_625_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_316-LP3-ITS-0892_625_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_316-LP3-ITS-0892_625_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.069 s (14.406 ¬µs/read; 4.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   4,793\n",
      "Reads with adapters:                        58 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         4,793 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,442,693 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,435,592 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 58 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 63.8%\n",
      "  C: 36.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t36\t0.0\t2\t0 1 35\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_317-LP3-ITS-0893_538_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_317-LP3-ITS-0893_538_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_317-LP3-ITS-0893_538_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_317-LP3-ITS-0893_538_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (61.756 ¬µs/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      91\n",
      "Reads with adapters:                         4 (4.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            91 (100.0%)\n",
      "\n",
      "Total basepairs processed:        27,391 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         27,001 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.0%\n",
      "  C: 0.0%\n",
      "  G: 25.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.4\t0\t1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_318-LP3-ITS-0894_847_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_318-LP3-ITS-0894_847_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_318-LP3-ITS-0894_847_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_318-LP3-ITS-0894_847_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.327 s (9.706 ¬µs/read; 6.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  33,711\n",
      "Reads with adapters:                       361 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        33,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,147,011 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,106,070 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 361 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.1%\n",
      "  C: 10.0%\n",
      "  G: 0.8%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t526.7\t0\t5\n",
      "4\t3\t131.7\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "52\t11\t0.0\t2\t0 0 11\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "118\t318\t0.0\t2\t0 5 313\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "133\t13\t0.0\t2\t0 0 13\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_319-LP3-ITS-0895_335_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_319-LP3-ITS-0895_335_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_319-LP3-ITS-0895_335_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_319-LP3-ITS-0895_335_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.489 s (10.054 ¬µs/read; 5.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,615\n",
      "Reads with adapters:                       489 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,615 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,633,115 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,573,099 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 489 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.4%\n",
      "  C: 9.0%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t759.6\t0\t6\n",
      "4\t2\t189.9\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t346\t0.0\t2\t0 6 340\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t35\t0.0\t2\t0 0 35\n",
      "137\t14\t0.0\t2\t0 1 13\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t36\t0.0\t2\t0 0 36\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t42\t0.0\t2\t0 0 42\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_320-LP3-ITS-0896_599_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_320-LP3-ITS-0896_599_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_320-LP3-ITS-0896_599_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_320-LP3-ITS-0896_599_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.445 s (10.190 ¬µs/read; 5.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,643\n",
      "Reads with adapters:                       427 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,643 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,136,543 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,086,681 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 427 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.4%\n",
      "  C: 6.6%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t170.5\t0\t1\n",
      "5\t2\t42.6\t0\t2\n",
      "49\t6\t0.0\t2\t0 0 6\n",
      "57\t9\t0.0\t2\t0 0 9\n",
      "118\t383\t0.0\t2\t0 5 378\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_321-LP3-ITS-0897_961_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_321-LP3-ITS-0897_961_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_321-LP3-ITS-0897_961_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_321-LP3-ITS-0897_961_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.290 s (11.220 ¬µs/read; 5.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,808\n",
      "Reads with adapters:                       680 (2.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,808 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,768,208 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,721,978 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 680 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.3%\n",
      "  C: 12.4%\n",
      "  G: 0.3%\n",
      "  T: 6.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t93\t403.2\t0\t93\n",
      "6\t128\t6.3\t0\t128\n",
      "8\t7\t0.4\t0\t7\n",
      "9\t14\t0.1\t0\t10 4\n",
      "10\t20\t0.0\t1\t1 19\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "40\t86\t0.0\t2\t0 4 82\n",
      "41\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t241\t0.0\t2\t1 3 237\n",
      "133\t9\t0.0\t2\t0 0 9\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "145\t44\t0.0\t2\t0 0 44\n",
      "150\t12\t0.0\t2\t0 0 12\n",
      "151\t11\t0.0\t2\t0 0 11\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_322-LP3-ITS-0898_941_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_322-LP3-ITS-0898_941_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_322-LP3-ITS-0898_941_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_322-LP3-ITS-0898_941_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.274 s (10.669 ¬µs/read; 5.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,704\n",
      "Reads with adapters:                       239 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,704 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,736,904 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,707,310 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 239 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 84.9%\n",
      "  C: 14.2%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t401.6\t0\t4\n",
      "5\t1\t25.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "54\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "118\t127\t0.0\t2\t0 0 127\n",
      "133\t21\t0.0\t2\t0 0 21\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "145\t62\t0.0\t2\t0 0 62\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_323-LP3-ITS-0899_133_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_323-LP3-ITS-0899_133_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_323-LP3-ITS-0899_133_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_323-LP3-ITS-0899_133_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.677 s (10.229 ¬µs/read; 5.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  66,224\n",
      "Reads with adapters:                       823 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        66,224 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,933,424 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     19,827,653 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 823 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 81.2%\n",
      "  C: 18.6%\n",
      "  G: 0.1%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1034.8\t0\t2\n",
      "10\t2\t0.1\t1\t0 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "32\t13\t0.0\t2\t0 0 13\n",
      "50\t12\t0.0\t2\t0 0 12\n",
      "56\t2\t0.0\t2\t0 0 2\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t348\t0.0\t2\t0 6 342\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t101\t0.0\t2\t0 1 100\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t103\t0.0\t2\t0 0 103\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t71\t0.0\t2\t0 2 69\n",
      "151\t68\t0.0\t2\t0 0 68\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t52\t0.0\t2\t0 1 51\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_324-LP3-ITS-0900_58_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_324-LP3-ITS-0900_58_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_324-LP3-ITS-0900_58_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_324-LP3-ITS-0900_58_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.227 s (11.089 ¬µs/read; 5.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,477\n",
      "Reads with adapters:                       215 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,477 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,163,577 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,138,109 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 215 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.2%\n",
      "  C: 9.8%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t320.0\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "59\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "118\t181\t0.0\t2\t0 3 178\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 1 3\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_325-LP3-ITS-0901_21_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_325-LP3-ITS-0901_21_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_325-LP3-ITS-0901_21_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_325-LP3-ITS-0901_21_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.056 s (25.483 ¬µs/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   2,210\n",
      "Reads with adapters:                        39 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         2,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:       665,210 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        662,650 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 39 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.9%\n",
      "  C: 59.0%\n",
      "  G: 2.6%\n",
      "  T: 2.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t34.5\t0\t18\n",
      "10\t1\t0.0\t1\t0 1\n",
      "94\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t9\t0.0\t2\t0 0 9\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_326-LP3-ITS-0902_36_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_326-LP3-ITS-0902_36_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_326-LP3-ITS-0902_36_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_326-LP3-ITS-0902_36_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.715 s (9.936 ¬µs/read; 6.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  71,992\n",
      "Reads with adapters:                       699 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        71,992 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,669,592 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     21,579,579 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 699 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.2%\n",
      "  C: 32.3%\n",
      "  G: 0.3%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1124.9\t0\t3\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "43\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "65\t2\t0.0\t2\t0 0 2\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t260\t0.0\t2\t1 5 254\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "132\t6\t0.0\t2\t0 0 6\n",
      "133\t172\t0.0\t2\t0 2 170\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t27\t0.0\t2\t0 1 26\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t162\t0.0\t2\t0 2 160\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t7\t0.0\t2\t0 0 7\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "159\t9\t0.0\t2\t0 0 9\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_327-LP3-ITS-0903_1079_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_327-LP3-ITS-0903_1079_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_327-LP3-ITS-0903_1079_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_327-LP3-ITS-0903_1079_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.156 s (10.924 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,287\n",
      "Reads with adapters:                       175 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,287 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,300,387 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,281,366 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 175 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.9%\n",
      "  C: 16.6%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t223.2\t0\t12\n",
      "10\t9\t0.0\t1\t0 9\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "96\t8\t0.0\t2\t0 0 8\n",
      "118\t110\t0.0\t2\t0 1 109\n",
      "119\t3\t0.0\t2\t0 0 3\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 1 8\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "165\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_328-LP3-ITS-0904_813_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_328-LP3-ITS-0904_813_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_328-LP3-ITS-0904_813_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_328-LP3-ITS-0904_813_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.290 s (10.933 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,501\n",
      "Reads with adapters:                       312 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,501 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,976,801 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,934,024 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 312 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 97.4%\n",
      "  C: 2.6%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "118\t118\t0.0\t2\t0 1 117\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "151\t175\t0.0\t2\t0 2 173\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_329-LP3-ITS-0905_406_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_329-LP3-ITS-0905_406_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_329-LP3-ITS-0905_406_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_329-LP3-ITS-0905_406_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (53.664 ¬µs/read; 1.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     143\n",
      "Reads with adapters:                         3 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           143 (100.0%)\n",
      "\n",
      "Total basepairs processed:        43,043 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         42,912 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t2.2\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_330-LP3-ITS-0906_213_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_330-LP3-ITS-0906_213_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_330-LP3-ITS-0906_213_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_330-LP3-ITS-0906_213_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.170 s (9.484 ¬µs/read; 6.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,895\n",
      "Reads with adapters:                     1,419 (7.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,895 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,386,395 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,378,700 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1419 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.0%\n",
      "  C: 6.9%\n",
      "  G: 0.2%\n",
      "  T: 0.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1393\t279.6\t0\t1393\n",
      "4\t5\t69.9\t0\t5\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "176\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_331-LP3-ITS-0907_98_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_331-LP3-ITS-0907_98_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_331-LP3-ITS-0907_98_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_331-LP3-ITS-0907_98_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.390 s (10.552 ¬µs/read; 5.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,932\n",
      "Reads with adapters:                       485 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,932 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,116,532 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,052,774 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 485 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.3%\n",
      "  C: 8.0%\n",
      "  G: 0.0%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t577.1\t0\t3\n",
      "10\t2\t0.0\t1\t0 2\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "118\t216\t0.0\t2\t0 2 214\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t30\t0.0\t2\t0 0 30\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "137\t17\t0.0\t2\t0 2 15\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t125\t0.0\t2\t0 4 121\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t70\t0.0\t2\t0 1 69\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_332-LP3-ITS-0908_1117_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_332-LP3-ITS-0908_1117_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_332-LP3-ITS-0908_1117_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_332-LP3-ITS-0908_1117_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.249 s (10.493 ¬µs/read; 5.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,765\n",
      "Reads with adapters:                       322 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,765 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,153,265 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,117,405 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 322 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.6%\n",
      "  C: 21.4%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t371.3\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "32\t10\t0.0\t2\t0 0 10\n",
      "51\t44\t0.0\t2\t0 1 43\n",
      "65\t4\t0.0\t2\t0 0 4\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "118\t145\t0.0\t2\t0 2 143\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "134\t11\t0.0\t2\t0 0 11\n",
      "137\t26\t0.0\t2\t0 0 26\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "143\t35\t0.0\t2\t0 1 34\n",
      "145\t23\t0.0\t2\t0 0 23\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_333-LP3-ITS-0909_171_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_333-LP3-ITS-0909_171_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_333-LP3-ITS-0909_171_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_333-LP3-ITS-0909_171_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (93.234 ¬µs/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      60\n",
      "Reads with adapters:                         6 (10.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            60 (100.0%)\n",
      "\n",
      "Total basepairs processed:        18,060 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         17,920 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t0.9\t0\t4\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_334-LP3-ITS-0910_1027_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_334-LP3-ITS-0910_1027_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_334-LP3-ITS-0910_1027_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_334-LP3-ITS-0910_1027_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.025 s (12.462 ¬µs/read; 4.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   1,966\n",
      "Reads with adapters:                        42 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         1,966 (100.0%)\n",
      "\n",
      "Total basepairs processed:       591,766 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        587,484 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 42 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.7%\n",
      "  C: 9.5%\n",
      "  G: 0.0%\n",
      "  T: 4.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t30.7\t0\t6\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t30\t0.0\t2\t0 0 30\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_335-LP3-ITS-0911_554_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_335-LP3-ITS-0911_554_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_335-LP3-ITS-0911_554_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_335-LP3-ITS-0911_554_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.295 s (10.658 ¬µs/read; 5.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,721\n",
      "Reads with adapters:                     1,023 (3.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,344,021 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,308,251 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1023 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.0%\n",
      "  C: 46.4%\n",
      "  G: 1.9%\n",
      "  T: 15.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t616\t433.1\t0\t616\n",
      "4\t1\t108.3\t0\t1\n",
      "8\t28\t0.4\t0\t28\n",
      "9\t3\t0.1\t0\t1 2\n",
      "10\t108\t0.0\t1\t6 102\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "58\t6\t0.0\t2\t0 0 6\n",
      "59\t3\t0.0\t2\t0 0 3\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "118\t172\t0.0\t2\t0 3 169\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "134\t7\t0.0\t2\t0 0 7\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "145\t22\t0.0\t2\t0 0 22\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t4\t0.0\t2\t0 1 3\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_336-LP3-ITS-0912_1081_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_336-LP3-ITS-0912_1081_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_336-LP3-ITS-0912_1081_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_336-LP3-ITS-0912_1081_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.472 s (10.111 ¬µs/read; 5.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  46,685\n",
      "Reads with adapters:                       392 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        46,685 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,052,185 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,007,384 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 392 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.6%\n",
      "  C: 8.2%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t729.5\t0\t9\n",
      "6\t1\t11.4\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "32\t6\t0.0\t2\t0 0 6\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "97\t1\t0.0\t2\t0 1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t343\t0.0\t2\t0 7 336\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_337-LP3-ITS-0913_158_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_337-LP3-ITS-0913_158_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_337-LP3-ITS-0913_158_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_337-LP3-ITS-0913_158_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.346 s (10.635 ¬µs/read; 5.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,561\n",
      "Reads with adapters:                       398 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,800,861 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,753,072 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 398 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 95.2%\n",
      "  C: 4.3%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t508.8\t0\t4\n",
      "10\t1\t0.0\t1\t0 1\n",
      "81\t3\t0.0\t2\t0 0 3\n",
      "118\t337\t0.0\t2\t0 3 334\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "145\t32\t0.0\t2\t0 0 32\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_338-LP3-ITS-0914_411_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_338-LP3-ITS-0914_411_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_338-LP3-ITS-0914_411_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_338-LP3-ITS-0914_411_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.440 s (10.560 ¬µs/read; 5.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,639\n",
      "Reads with adapters:                       414 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,639 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,533,339 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,477,424 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 414 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 93.5%\n",
      "  C: 6.0%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t650.6\t0\t4\n",
      "118\t148\t0.0\t2\t0 3 145\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 1 1\n",
      "145\t119\t0.0\t2\t0 1 118\n",
      "151\t116\t0.0\t2\t0 3 113\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_339-LP3-ITS-0915_91_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_339-LP3-ITS-0915_91_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_339-LP3-ITS-0915_91_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_339-LP3-ITS-0915_91_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.405 s (10.317 ¬µs/read; 5.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,233\n",
      "Reads with adapters:                       417 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,233 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,809,133 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,758,872 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 417 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 83.0%\n",
      "  C: 17.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t613.0\t0\t1\n",
      "6\t1\t9.6\t0\t1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "51\t11\t0.0\t2\t0 1 10\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t300\t0.0\t2\t0 9 291\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t29\t0.0\t2\t0 1 28\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t10\t0.0\t2\t0 0 10\n",
      "143\t10\t0.0\t2\t0 1 9\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t9\t0.0\t2\t0 1 8\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_340-LP3-ITS-0916_346_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_340-LP3-ITS-0916_346_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_340-LP3-ITS-0916_346_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_340-LP3-ITS-0916_346_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.200 s (11.513 ¬µs/read; 5.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,344\n",
      "Reads with adapters:                       221 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,344 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,220,544 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,194,941 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 221 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.1%\n",
      "  C: 14.9%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t271.0\t0\t2\n",
      "4\t7\t67.8\t0\t7\n",
      "9\t2\t0.1\t0\t2\n",
      "55\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t166\t0.0\t2\t0 3 163\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "159\t3\t0.0\t2\t0 0 3\n",
      "160\t10\t0.0\t2\t0 1 9\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_341-LP3-ITS-0917_624_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_341-LP3-ITS-0917_624_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_341-LP3-ITS-0917_624_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_341-LP3-ITS-0917_624_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.194 s (10.614 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,305\n",
      "Reads with adapters:                       177 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,509,805 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,488,729 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 177 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.5%\n",
      "  C: 8.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t286.0\t0\t2\n",
      "5\t1\t17.9\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "118\t132\t0.0\t2\t0 2 130\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "150\t7\t0.0\t2\t0 0 7\n",
      "151\t9\t0.0\t2\t0 1 8\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_342-LP3-ITS-0918_151_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_342-LP3-ITS-0918_151_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_342-LP3-ITS-0918_151_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_342-LP3-ITS-0918_151_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.248 s (10.940 ¬µs/read; 5.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,639\n",
      "Reads with adapters:                     3,306 (14.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,639 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,814,339 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,789,796 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3306 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.0%\n",
      "  C: 53.5%\n",
      "  G: 2.1%\n",
      "  T: 22.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2612\t353.7\t0\t2612\n",
      "4\t1\t88.4\t0\t1\n",
      "6\t3\t5.5\t0\t3\n",
      "8\t8\t0.3\t0\t8\n",
      "9\t96\t0.1\t0\t84 12\n",
      "10\t535\t0.0\t1\t31 504\n",
      "11\t1\t0.0\t1\t0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "209\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_343-LP3-ITS-0919_567_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_343-LP3-ITS-0919_567_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_343-LP3-ITS-0919_567_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_343-LP3-ITS-0919_567_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.402 s (10.949 ¬µs/read; 5.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  36,720\n",
      "Reads with adapters:                     9,058 (24.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        36,720 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,052,720 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,997,087 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9058 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.4%\n",
      "  C: 57.1%\n",
      "  G: 1.7%\n",
      "  T: 18.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7149\t573.8\t0\t7149\n",
      "4\t2\t143.4\t0\t2\n",
      "8\t26\t0.6\t0\t26\n",
      "9\t101\t0.1\t0\t95 6\n",
      "10\t1662\t0.0\t1\t92 1570\n",
      "11\t1\t0.0\t1\t0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "118\t67\t0.0\t2\t0 1 66\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t18\t0.0\t2\t0 0 18\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "175\t16\t0.0\t2\t0 0 16\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_344-LP3-ITS-0920_76_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_344-LP3-ITS-0920_76_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_344-LP3-ITS-0920_76_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_344-LP3-ITS-0920_76_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.260 s (8.822 ¬µs/read; 6.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,463\n",
      "Reads with adapters:                     3,277 (11.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,463 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,868,363 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,850,076 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3277 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.5%\n",
      "  C: 56.9%\n",
      "  G: 2.0%\n",
      "  T: 19.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2584\t460.4\t0\t2584\n",
      "4\t18\t115.1\t0\t18\n",
      "6\t11\t7.2\t0\t11\n",
      "8\t5\t0.4\t0\t5\n",
      "9\t33\t0.1\t0\t30 3\n",
      "10\t602\t0.0\t1\t41 561\n",
      "11\t2\t0.0\t1\t0 2\n",
      "118\t6\t0.0\t2\t0 0 6\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "209\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_345-LP3-ITS-0921_303_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_345-LP3-ITS-0921_303_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_345-LP3-ITS-0921_303_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_345-LP3-ITS-0921_303_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.308 s (11.025 ¬µs/read; 5.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,896\n",
      "Reads with adapters:                     5,896 (21.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,896 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,396,696 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,362,926 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5896 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.4%\n",
      "  C: 54.0%\n",
      "  G: 2.2%\n",
      "  T: 23.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4732\t435.9\t0\t4732\n",
      "4\t1\t109.0\t0\t1\n",
      "6\t4\t6.8\t0\t4\n",
      "7\t1\t1.7\t0\t1\n",
      "8\t33\t0.4\t0\t33\n",
      "9\t111\t0.1\t0\t95 16\n",
      "10\t963\t0.0\t1\t60 903\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "40\t6\t0.0\t2\t0 0 6\n",
      "118\t13\t0.0\t2\t0 1 12\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "219\t4\t0.0\t2\t0 0 4\n",
      "232\t5\t0.0\t2\t0 0 5\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t11\t0.0\t2\t0 1 10\n",
      "235\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_346-LP3-ITS-0922_1084_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_346-LP3-ITS-0922_1084_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_346-LP3-ITS-0922_1084_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_346-LP3-ITS-0922_1084_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.184 s (8.536 ¬µs/read; 7.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,566\n",
      "Reads with adapters:                     1,378 (6.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,566 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,491,366 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,482,638 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1378 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.4%\n",
      "  C: 57.3%\n",
      "  G: 1.4%\n",
      "  T: 21.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1095\t337.0\t0\t1095\n",
      "4\t13\t84.2\t0\t13\n",
      "5\t1\t21.1\t0\t1\n",
      "6\t9\t5.3\t0\t9\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t43\t0.1\t0\t40 3\n",
      "10\t201\t0.0\t1\t9 192\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "209\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_347-LP3-ITS-0923_1150_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_347-LP3-ITS-0923_1150_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_347-LP3-ITS-0923_1150_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_347-LP3-ITS-0923_1150_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.245 s (11.359 ¬µs/read; 5.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,578\n",
      "Reads with adapters:                     1,899 (8.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,578 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,494,978 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,480,554 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1899 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.7%\n",
      "  C: 46.4%\n",
      "  G: 1.2%\n",
      "  T: 20.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1329\t337.2\t0\t1329\n",
      "7\t1\t1.3\t0\t1\n",
      "8\t5\t0.3\t0\t5\n",
      "9\t184\t0.1\t0\t171 13\n",
      "10\t338\t0.0\t1\t15 323\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t29\t0.0\t2\t0 1 28\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_348-LP3-ITS-0924_674_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_348-LP3-ITS-0924_674_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_348-LP3-ITS-0924_674_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_348-LP3-ITS-0924_674_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.175 s (11.919 ¬µs/read; 5.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,717\n",
      "Reads with adapters:                     3,581 (24.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,717 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,429,817 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,412,076 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3581 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.4%\n",
      "  C: 48.2%\n",
      "  G: 1.6%\n",
      "  T: 25.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2742\t230.0\t0\t2742\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t94\t0.1\t0\t86 8\n",
      "10\t734\t0.0\t1\t52 682\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "209\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_349-LP3-ITS-0925_259_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_349-LP3-ITS-0925_259_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_349-LP3-ITS-0925_259_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_349-LP3-ITS-0925_259_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.173 s (10.822 ¬µs/read; 5.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,964\n",
      "Reads with adapters:                     3,779 (23.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,964 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,805,164 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,784,710 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3779 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.6%\n",
      "  C: 57.3%\n",
      "  G: 1.7%\n",
      "  T: 18.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2991\t249.4\t0\t2991\n",
      "4\t1\t62.4\t0\t1\n",
      "9\t67\t0.1\t0\t62 5\n",
      "10\t695\t0.0\t1\t42 653\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "175\t9\t0.0\t2\t0 0 9\n",
      "219\t2\t0.0\t2\t0 0 2\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_350-LP3-ITS-0926_38_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_350-LP3-ITS-0926_38_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_350-LP3-ITS-0926_38_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_350-LP3-ITS-0926_38_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.248 s (9.423 ¬µs/read; 6.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,313\n",
      "Reads with adapters:                     2,845 (10.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,313 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,920,213 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,905,885 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2845 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.6%\n",
      "  C: 57.9%\n",
      "  G: 1.7%\n",
      "  T: 18.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2241\t411.1\t0\t2241\n",
      "4\t9\t102.8\t0\t9\n",
      "5\t1\t25.7\t0\t1\n",
      "6\t2\t6.4\t0\t2\n",
      "7\t2\t1.6\t0\t2\n",
      "8\t38\t0.4\t0\t38\n",
      "9\t62\t0.1\t0\t59 3\n",
      "10\t479\t0.0\t1\t27 452\n",
      "11\t1\t0.0\t1\t1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "209\t6\t0.0\t2\t0 0 6\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_351-LP3-ITS-0927_44_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_351-LP3-ITS-0927_44_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_351-LP3-ITS-0927_44_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_351-LP3-ITS-0927_44_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.141 s (9.688 ¬µs/read; 6.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,549\n",
      "Reads with adapters:                     2,035 (14.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,549 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,379,249 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,368,496 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2035 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.9%\n",
      "  C: 53.4%\n",
      "  G: 2.0%\n",
      "  T: 19.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1560\t227.3\t0\t1560\n",
      "4\t7\t56.8\t0\t7\n",
      "5\t2\t14.2\t0\t2\n",
      "6\t6\t3.6\t0\t6\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t67\t0.1\t0\t60 7\n",
      "10\t383\t0.0\t1\t31 352\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "209\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_352-LP3-ITS-0928_991_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_352-LP3-ITS-0928_991_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_352-LP3-ITS-0928_991_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_352-LP3-ITS-0928_991_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.348 s (8.318 ¬µs/read; 7.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,873\n",
      "Reads with adapters:                     2,440 (5.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,873 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,603,773 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,589,742 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2440 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.6%\n",
      "  C: 56.9%\n",
      "  G: 1.6%\n",
      "  T: 17.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1825\t654.3\t0\t1825\n",
      "4\t38\t163.6\t0\t38\n",
      "5\t6\t40.9\t0\t6\n",
      "6\t38\t10.2\t0\t38\n",
      "7\t1\t2.6\t0\t1\n",
      "9\t46\t0.2\t0\t44 2\n",
      "10\t467\t0.0\t1\t15 452\n",
      "11\t3\t0.0\t1\t1 2\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "209\t12\t0.0\t2\t0 1 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_353-LP3-ITS-0929_477_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_353-LP3-ITS-0929_477_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_353-LP3-ITS-0929_477_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_353-LP3-ITS-0929_477_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.265 s (10.261 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,840\n",
      "Reads with adapters:                     1,274 (4.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,777,840 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,735,888 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1274 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.3%\n",
      "  C: 43.8%\n",
      "  G: 1.8%\n",
      "  T: 22.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t882\t403.8\t0\t882\n",
      "4\t1\t100.9\t0\t1\n",
      "7\t1\t1.6\t0\t1\n",
      "8\t5\t0.4\t0\t5\n",
      "9\t33\t0.1\t0\t30 3\n",
      "10\t173\t0.0\t1\t12 161\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "208\t1\t0.0\t2\t0 0 1\n",
      "209\t176\t0.0\t2\t0 0 176\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_354-LP3-ITS-0930_218_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_354-LP3-ITS-0930_218_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_354-LP3-ITS-0930_218_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_354-LP3-ITS-0930_218_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.217 s (11.071 ¬µs/read; 5.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,607\n",
      "Reads with adapters:                     2,382 (12.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,607 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,901,707 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,884,876 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2382 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.5%\n",
      "  C: 55.1%\n",
      "  G: 1.3%\n",
      "  T: 21.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1873\t306.4\t0\t1873\n",
      "4\t2\t76.6\t0\t2\n",
      "7\t1\t1.2\t0\t1\n",
      "8\t6\t0.3\t0\t6\n",
      "9\t94\t0.1\t0\t89 5\n",
      "10\t364\t0.0\t1\t22 342\n",
      "11\t1\t0.0\t1\t0 1\n",
      "40\t3\t0.0\t2\t1 0 2\n",
      "118\t12\t0.0\t2\t0 0 12\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "175\t11\t0.0\t2\t0 0 11\n",
      "212\t6\t0.0\t2\t0 0 6\n",
      "216\t2\t0.0\t2\t0 0 2\n",
      "219\t4\t0.0\t2\t0 0 4\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_355-LP3-ITS-0931_308_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_355-LP3-ITS-0931_308_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_355-LP3-ITS-0931_308_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_355-LP3-ITS-0931_308_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.293 s (11.032 ¬µs/read; 5.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,571\n",
      "Reads with adapters:                     3,984 (15.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,571 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,997,871 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,964,286 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3984 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.0%\n",
      "  C: 50.6%\n",
      "  G: 1.7%\n",
      "  T: 23.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3087\t415.2\t0\t3087\n",
      "4\t3\t103.8\t0\t3\n",
      "7\t1\t1.6\t0\t1\n",
      "8\t27\t0.4\t0\t27\n",
      "9\t110\t0.1\t0\t107 3\n",
      "10\t674\t0.0\t1\t39 635\n",
      "11\t2\t0.0\t1\t0 2\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "209\t75\t0.0\t2\t0 1 74\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_356-LP3-ITS-0932_732_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_356-LP3-ITS-0932_732_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_356-LP3-ITS-0932_732_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_356-LP3-ITS-0932_732_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.178 s (10.842 ¬µs/read; 5.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,453\n",
      "Reads with adapters:                     2,031 (12.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,453 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,952,353 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,942,311 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2031 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.8%\n",
      "  C: 48.2%\n",
      "  G: 1.4%\n",
      "  T: 25.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1532\t257.1\t0\t1532\n",
      "4\t6\t64.3\t0\t6\n",
      "5\t1\t16.1\t0\t1\n",
      "6\t7\t4.0\t0\t7\n",
      "9\t83\t0.1\t0\t79 4\n",
      "10\t397\t0.0\t1\t22 375\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_357-LP3-ITS-0933_160_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_357-LP3-ITS-0933_160_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_357-LP3-ITS-0933_160_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_357-LP3-ITS-0933_160_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.256 s (9.384 ¬µs/read; 6.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,277\n",
      "Reads with adapters:                     1,646 (6.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,277 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,210,377 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,202,919 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1646 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.2%\n",
      "  C: 51.5%\n",
      "  G: 1.0%\n",
      "  T: 25.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1277\t426.2\t0\t1277\n",
      "4\t19\t106.6\t0\t19\n",
      "5\t2\t26.6\t0\t2\n",
      "6\t9\t6.7\t0\t9\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t87\t0.1\t0\t76 11\n",
      "10\t249\t0.0\t1\t12 237\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_358-LP3-ITS-0934_405_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_358-LP3-ITS-0934_405_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_358-LP3-ITS-0934_405_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_358-LP3-ITS-0934_405_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.250 s (11.530 ¬µs/read; 5.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,718\n",
      "Reads with adapters:                     5,639 (26.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,718 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,537,118 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,503,318 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5639 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.7%\n",
      "  C: 60.2%\n",
      "  G: 2.2%\n",
      "  T: 18.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4578\t339.3\t0\t4578\n",
      "4\t5\t84.8\t0\t5\n",
      "9\t68\t0.1\t0\t64 4\n",
      "10\t923\t0.0\t1\t43 880\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t38\t0.0\t2\t0 1 37\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t15\t0.0\t2\t0 0 15\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_359-LP3-ITS-0935_744_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_359-LP3-ITS-0935_744_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_359-LP3-ITS-0935_744_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_359-LP3-ITS-0935_744_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.153 s (10.109 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,170\n",
      "Reads with adapters:                     2,003 (13.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,170 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,566,170 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,542,785 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2003 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.9%\n",
      "  C: 58.1%\n",
      "  G: 1.0%\n",
      "  T: 19.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1580\t237.0\t0\t1580\n",
      "4\t1\t59.3\t0\t1\n",
      "6\t1\t3.7\t0\t1\n",
      "7\t1\t0.9\t0\t1\n",
      "8\t1\t0.2\t0\t1\n",
      "9\t23\t0.1\t0\t20 3\n",
      "10\t322\t0.0\t1\t26 296\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "209\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_360-LP3-ITS-0936_221_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_360-LP3-ITS-0936_221_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_360-LP3-ITS-0936_221_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_360-LP3-ITS-0936_221_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.304 s (10.908 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,863\n",
      "Reads with adapters:                     3,758 (13.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,863 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,386,763 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,349,047 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3758 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.8%\n",
      "  C: 53.9%\n",
      "  G: 1.9%\n",
      "  T: 21.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2953\t435.4\t0\t2953\n",
      "4\t2\t108.8\t0\t2\n",
      "6\t1\t6.8\t0\t1\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t110\t0.1\t0\t101 9\n",
      "10\t584\t0.0\t1\t33 551\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "209\t103\t0.0\t2\t0 0 103\n",
      "210\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_361-LP3-ITS-0937_959_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_361-LP3-ITS-0937_959_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_361-LP3-ITS-0937_959_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_361-LP3-ITS-0937_959_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.521 s (10.601 ¬µs/read; 5.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  49,121\n",
      "Reads with adapters:                     6,635 (13.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        49,121 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,785,421 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,690,884 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6635 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.7%\n",
      "  C: 49.3%\n",
      "  G: 1.7%\n",
      "  T: 22.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4936\t767.5\t0\t4936\n",
      "4\t4\t191.9\t0\t4\n",
      "8\t4\t0.7\t0\t4\n",
      "9\t116\t0.2\t0\t103 13\n",
      "10\t1252\t0.0\t1\t63 1189\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t47\t0.0\t2\t0 2 45\n",
      "133\t6\t0.0\t2\t0 0 6\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t24\t0.0\t2\t0 0 24\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "186\t1\t0.0\t2\t0 0 1\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t34\t0.0\t2\t0 1 33\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t159\t0.0\t2\t0 2 157\n",
      "235\t34\t0.0\t2\t0 0 34\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_362-LP3-ITS-0938_1120_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_362-LP3-ITS-0938_1120_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_362-LP3-ITS-0938_1120_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_362-LP3-ITS-0938_1120_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.138 s (11.594 ¬µs/read; 5.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,878\n",
      "Reads with adapters:                     1,041 (8.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,878 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,575,278 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,569,636 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1041 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.2%\n",
      "  C: 50.8%\n",
      "  G: 1.9%\n",
      "  T: 20.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t768\t185.6\t0\t768\n",
      "4\t6\t46.4\t0\t6\n",
      "5\t4\t11.6\t0\t4\n",
      "6\t4\t2.9\t0\t4\n",
      "8\t7\t0.2\t0\t7\n",
      "9\t32\t0.0\t0\t29 3\n",
      "10\t216\t0.0\t1\t10 206\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "209\t2\t0.0\t2\t0 0 2\n",
      "230\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_363-LP3-ITS-0939_195_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_363-LP3-ITS-0939_195_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_363-LP3-ITS-0939_195_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_363-LP3-ITS-0939_195_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (61.867 ¬µs/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     119\n",
      "Reads with adapters:                         5 (4.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           119 (100.0%)\n",
      "\n",
      "Total basepairs processed:        35,819 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         35,527 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.0%\n",
      "  C: 40.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.9\t0\t2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_364-LP3-ITS-0940_384_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_364-LP3-ITS-0940_384_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_364-LP3-ITS-0940_384_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_364-LP3-ITS-0940_384_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (65.013 ¬µs/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      85\n",
      "Reads with adapters:                         6 (7.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            85 (100.0%)\n",
      "\n",
      "Total basepairs processed:        25,585 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         25,432 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 16.7%\n",
      "  G: 0.0%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.3\t0\t2\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_365-LP3-ITS-0941_190_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_365-LP3-ITS-0941_190_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_365-LP3-ITS-0941_190_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_365-LP3-ITS-0941_190_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (59.560 ¬µs/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     106\n",
      "Reads with adapters:                         9 (8.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           106 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,906 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         31,534 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.4%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 22.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t1.7\t0\t6\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_366-LP3-ITS-0942_229_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_366-LP3-ITS-0942_229_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_366-LP3-ITS-0942_229_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_366-LP3-ITS-0942_229_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (45.602 ¬µs/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     146\n",
      "Reads with adapters:                        12 (8.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           146 (100.0%)\n",
      "\n",
      "Total basepairs processed:        43,946 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         43,624 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 33.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2.3\t0\t7\n",
      "9\t1\t0.0\t0\t0 1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_367-LP3-ITS-0943_1021_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_367-LP3-ITS-0943_1021_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_367-LP3-ITS-0943_1021_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_367-LP3-ITS-0943_1021_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (43.331 ¬µs/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     167\n",
      "Reads with adapters:                        11 (6.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           167 (100.0%)\n",
      "\n",
      "Total basepairs processed:        50,267 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         49,959 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.4%\n",
      "  C: 63.6%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2.6\t0\t7\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_368-LP3-ITS-0944_111_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_368-LP3-ITS-0944_111_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_368-LP3-ITS-0944_111_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_368-LP3-ITS-0944_111_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (41.004 ¬µs/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     193\n",
      "Reads with adapters:                         6 (3.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           193 (100.0%)\n",
      "\n",
      "Total basepairs processed:        58,093 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         57,952 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 16.7%\n",
      "  C: 83.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t3.0\t0\t4\n",
      "11\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_369-LP3-ITS-0945_779_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_369-LP3-ITS-0945_779_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_369-LP3-ITS-0945_779_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_369-LP3-ITS-0945_779_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.282 s (9.548 ¬µs/read; 6.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,483\n",
      "Reads with adapters:                     2,262 (7.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,874,383 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,857,253 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2262 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.0%\n",
      "  C: 7.5%\n",
      "  G: 0.2%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2192\t460.7\t0\t2192\n",
      "4\t7\t115.2\t0\t7\n",
      "5\t1\t28.8\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "176\t56\t0.0\t2\t0 0 56\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_370-LP3-ITS-0946_733_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_370-LP3-ITS-0946_733_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_370-LP3-ITS-0946_733_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_370-LP3-ITS-0946_733_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.284 s (10.287 ¬µs/read; 5.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,601\n",
      "Reads with adapters:                       263 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,601 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,307,901 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,273,015 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 263 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.1%\n",
      "  C: 46.4%\n",
      "  G: 0.0%\n",
      "  T: 1.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t431.3\t0\t7\n",
      "10\t1\t0.0\t1\t0 1\n",
      "87\t5\t0.0\t2\t0 0 5\n",
      "100\t2\t0.0\t2\t0 0 2\n",
      "104\t15\t0.0\t2\t0 1 14\n",
      "105\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "113\t2\t0.0\t2\t0 0 2\n",
      "115\t7\t0.0\t2\t0 0 7\n",
      "117\t6\t0.0\t2\t0 0 6\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "121\t3\t0.0\t2\t0 0 3\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "128\t4\t0.0\t2\t0 0 4\n",
      "132\t8\t0.0\t2\t0 0 8\n",
      "133\t31\t0.0\t2\t0 0 31\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "141\t13\t0.0\t2\t0 1 12\n",
      "142\t23\t0.0\t2\t0 1 22\n",
      "143\t57\t0.0\t2\t0 1 56\n",
      "144\t9\t0.0\t2\t0 0 9\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t47\t0.0\t2\t0 0 47\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_371-LP3-ITS-0947_453_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_371-LP3-ITS-0947_453_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_371-LP3-ITS-0947_453_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_371-LP3-ITS-0947_453_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.532 s (10.138 ¬µs/read; 5.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,447\n",
      "Reads with adapters:                     2,011 (3.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,447 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,786,547 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,701,444 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2011 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.9%\n",
      "  C: 10.9%\n",
      "  G: 0.0%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t819.5\t0\t5\n",
      "5\t4\t51.2\t0\t4\n",
      "6\t1416\t12.8\t0\t1416\n",
      "7\t5\t3.2\t0\t5\n",
      "10\t3\t0.1\t1\t0 3\n",
      "32\t6\t0.0\t2\t0 0 6\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "62\t4\t0.0\t2\t0 0 4\n",
      "103\t34\t0.0\t2\t0 1 33\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "113\t10\t0.0\t2\t0 0 10\n",
      "116\t4\t0.0\t2\t0 0 4\n",
      "117\t19\t0.0\t2\t0 1 18\n",
      "118\t80\t0.0\t2\t0 2 78\n",
      "119\t7\t0.0\t2\t0 0 7\n",
      "120\t5\t0.0\t2\t0 0 5\n",
      "121\t17\t0.0\t2\t0 0 17\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t60\t0.0\t2\t0 1 59\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "141\t9\t0.0\t2\t0 0 9\n",
      "142\t80\t0.0\t2\t0 0 80\n",
      "143\t134\t0.0\t2\t0 0 134\n",
      "144\t15\t0.0\t2\t0 0 15\n",
      "145\t7\t0.0\t2\t0 0 7\n",
      "146\t5\t0.0\t2\t0 1 4\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "155\t55\t0.0\t2\t0 0 55\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t4\t0.0\t2\t0 0 4\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "209\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_372-LP3-ITS-0948_591_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_372-LP3-ITS-0948_591_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_372-LP3-ITS-0948_591_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_372-LP3-ITS-0948_591_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.227 s (10.867 ¬µs/read; 5.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,930\n",
      "Reads with adapters:                       399 (1.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,930 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,299,930 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,268,230 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 399 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.2%\n",
      "  C: 24.6%\n",
      "  G: 0.3%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t327.0\t0\t7\n",
      "6\t169\t5.1\t0\t169\n",
      "13\t1\t0.0\t1\t0 1\n",
      "45\t3\t0.0\t2\t0 0 3\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t26\t0.0\t2\t0 0 26\n",
      "121\t5\t0.0\t2\t0 0 5\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "132\t5\t0.0\t2\t0 0 5\n",
      "133\t23\t0.0\t2\t0 0 23\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "141\t2\t0.0\t2\t0 0 2\n",
      "142\t24\t0.0\t2\t0 0 24\n",
      "143\t57\t0.0\t2\t0 0 57\n",
      "144\t7\t0.0\t2\t0 0 7\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "149\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t45\t0.0\t2\t0 1 44\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "185\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_373-LP3-ITS-0949_1098_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_373-LP3-ITS-0949_1098_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_373-LP3-ITS-0949_1098_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_373-LP3-ITS-0949_1098_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.188 s (10.619 ¬µs/read; 5.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,705\n",
      "Reads with adapters:                       202 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,705 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,329,205 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,304,671 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 202 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.6%\n",
      "  C: 9.4%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t276.6\t0\t2\n",
      "10\t2\t0.0\t1\t0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "115\t2\t0.0\t2\t0 0 2\n",
      "118\t148\t0.0\t2\t0 2 146\n",
      "121\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t26\t0.0\t2\t0 0 26\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_374-LP3-ITS-0950_1074_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_374-LP3-ITS-0950_1074_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_374-LP3-ITS-0950_1074_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_374-LP3-ITS-0950_1074_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.312 s (10.968 ¬µs/read; 5.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,422\n",
      "Reads with adapters:                       502 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,422 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,555,022 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,503,774 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 502 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.7%\n",
      "  C: 68.5%\n",
      "  G: 1.4%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t444.1\t0\t25\n",
      "6\t1\t6.9\t0\t1\n",
      "10\t4\t0.0\t1\t0 4\n",
      "12\t1\t0.0\t1\t0 1\n",
      "53\t3\t0.0\t2\t0 0 3\n",
      "62\t158\t0.0\t2\t0 2 156\n",
      "63\t4\t0.0\t2\t0 0 4\n",
      "81\t13\t0.0\t2\t0 0 13\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 1\n",
      "92\t7\t0.0\t2\t0 0 7\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "105\t5\t0.0\t2\t0 0 5\n",
      "108\t6\t0.0\t2\t0 0 6\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "111\t4\t0.0\t2\t0 0 4\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "113\t4\t0.0\t2\t0 0 4\n",
      "115\t4\t0.0\t2\t0 0 4\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t16\t0.0\t2\t0 0 16\n",
      "120\t2\t0.0\t2\t0 0 2\n",
      "123\t20\t0.0\t2\t0 0 20\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t49\t0.0\t2\t0 1 48\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "141\t4\t0.0\t2\t0 0 4\n",
      "142\t12\t0.0\t2\t0 0 12\n",
      "143\t57\t0.0\t2\t0 0 57\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "155\t74\t0.0\t2\t0 1 73\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_375-LP3-ITS-0951_499_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_375-LP3-ITS-0951_499_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_375-LP3-ITS-0951_499_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_375-LP3-ITS-0951_499_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.559 s (10.669 ¬µs/read; 5.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,377\n",
      "Reads with adapters:                       702 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,377 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,765,477 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,684,668 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 702 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.1%\n",
      "  C: 73.5%\n",
      "  G: 0.3%\n",
      "  T: 0.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t818.4\t0\t11\n",
      "4\t3\t204.6\t0\t3\n",
      "5\t1\t51.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "53\t8\t0.0\t2\t0 0 8\n",
      "62\t60\t0.0\t2\t0 0 60\n",
      "71\t21\t0.0\t2\t0 1 20\n",
      "72\t1\t0.0\t2\t0 0 1\n",
      "87\t28\t0.0\t2\t0 0 28\n",
      "92\t22\t0.0\t2\t0 0 22\n",
      "101\t10\t0.0\t2\t0 0 10\n",
      "104\t21\t0.0\t2\t0 1 20\n",
      "106\t14\t0.0\t2\t0 0 14\n",
      "107\t37\t0.0\t2\t0 0 37\n",
      "112\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t24\t0.0\t2\t0 0 24\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "121\t4\t0.0\t2\t0 1 3\n",
      "123\t135\t0.0\t2\t0 1 134\n",
      "128\t26\t0.0\t2\t0 0 26\n",
      "132\t9\t0.0\t2\t0 0 9\n",
      "133\t92\t0.0\t2\t0 0 92\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t36\t0.0\t2\t0 0 36\n",
      "143\t71\t0.0\t2\t0 0 71\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "155\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_376-LP3-ITS-0952_1019_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_376-LP3-ITS-0952_1019_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_376-LP3-ITS-0952_1019_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_376-LP3-ITS-0952_1019_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.449 s (10.141 ¬µs/read; 5.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,248\n",
      "Reads with adapters:                       310 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,248 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,318,648 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,280,502 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 310 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.0%\n",
      "  C: 57.7%\n",
      "  G: 1.0%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t691.4\t0\t11\n",
      "4\t5\t172.8\t0\t5\n",
      "10\t2\t0.0\t1\t0 2\n",
      "100\t37\t0.0\t2\t0 1 36\n",
      "101\t7\t0.0\t2\t0 0 7\n",
      "104\t7\t0.0\t2\t0 0 7\n",
      "118\t12\t0.0\t2\t0 0 12\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "123\t19\t0.0\t2\t0 0 19\n",
      "130\t2\t0.0\t2\t0 0 2\n",
      "133\t87\t0.0\t2\t0 0 87\n",
      "136\t3\t0.0\t2\t0 0 3\n",
      "142\t43\t0.0\t2\t0 0 43\n",
      "143\t45\t0.0\t2\t0 0 45\n",
      "144\t6\t0.0\t2\t0 0 6\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_377-LP3-ITS-0953_616_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_377-LP3-ITS-0953_616_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_377-LP3-ITS-0953_616_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_377-LP3-ITS-0953_616_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (42.416 ¬µs/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     178\n",
      "Reads with adapters:                        16 (9.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           178 (100.0%)\n",
      "\n",
      "Total basepairs processed:        53,578 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         53,315 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.2%\n",
      "  C: 56.2%\n",
      "  G: 0.0%\n",
      "  T: 12.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t2.8\t0\t11\n",
      "9\t2\t0.0\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_378-LP3-ITS-0954_805_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_378-LP3-ITS-0954_805_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_378-LP3-ITS-0954_805_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_378-LP3-ITS-0954_805_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (27.234 ¬µs/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     312\n",
      "Reads with adapters:                        16 (5.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           312 (100.0%)\n",
      "\n",
      "Total basepairs processed:        93,912 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         93,646 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.5%\n",
      "  C: 56.2%\n",
      "  G: 0.0%\n",
      "  T: 6.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t4.9\t0\t13\n",
      "10\t1\t0.0\t1\t0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_379-LP3-ITS-0955_131_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_379-LP3-ITS-0955_131_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_379-LP3-ITS-0955_131_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_379-LP3-ITS-0955_131_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (39.746 ¬µs/read; 1.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     185\n",
      "Reads with adapters:                        17 (9.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           185 (100.0%)\n",
      "\n",
      "Total basepairs processed:        55,685 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         54,989 bp (98.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 17 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.1%\n",
      "  C: 29.4%\n",
      "  G: 5.9%\n",
      "  T: 17.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t2.9\t0\t9\n",
      "6\t2\t0.0\t0\t2\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_380-LP3-ITS-0956_113_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_380-LP3-ITS-0956_113_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_380-LP3-ITS-0956_113_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_380-LP3-ITS-0956_113_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (59.387 ¬µs/read; 1.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     114\n",
      "Reads with adapters:                         5 (4.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           114 (100.0%)\n",
      "\n",
      "Total basepairs processed:        34,314 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         34,043 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.0%\n",
      "  C: 20.0%\n",
      "  G: 0.0%\n",
      "  T: 20.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.8\t0\t2\n",
      "10\t1\t0.0\t1\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_381-LP3-ITS-0957_808_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_381-LP3-ITS-0957_808_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_381-LP3-ITS-0957_808_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_381-LP3-ITS-0957_808_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (72.091 ¬µs/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      89\n",
      "Reads with adapters:                         4 (4.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            89 (100.0%)\n",
      "\n",
      "Total basepairs processed:        26,789 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         26,540 bp (99.1%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 25.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.4\t0\t1\n",
      "10\t1\t0.0\t1\t1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_382-LP3-ITS-0958_1143_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_382-LP3-ITS-0958_1143_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_382-LP3-ITS-0958_1143_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_382-LP3-ITS-0958_1143_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (50.564 ¬µs/read; 1.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     147\n",
      "Reads with adapters:                         8 (5.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           147 (100.0%)\n",
      "\n",
      "Total basepairs processed:        44,247 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         43,965 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.5%\n",
      "  C: 62.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t2.3\t0\t4\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_383-LP3-ITS-0959_901_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_383-LP3-ITS-0959_901_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_383-LP3-ITS-0959_901_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_383-LP3-ITS-0959_901_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (42.451 ¬µs/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     171\n",
      "Reads with adapters:                        11 (6.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           171 (100.0%)\n",
      "\n",
      "Total basepairs processed:        51,471 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         50,932 bp (99.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.5%\n",
      "  C: 36.4%\n",
      "  G: 9.1%\n",
      "  T: 9.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t2.7\t0\t6\n",
      "10\t2\t0.0\t1\t1 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_384-LP3-ITS-0960_628_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_384-LP3-ITS-0960_628_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_384-LP3-ITS-0960_628_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_384-LP3-ITS-0960_628_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (28.600 ¬µs/read; 2.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     305\n",
      "Reads with adapters:                         6 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           305 (100.0%)\n",
      "\n",
      "Total basepairs processed:        91,805 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         91,658 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.7%\n",
      "  C: 33.3%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t4.8\t0\t3\n",
      "10\t2\t0.0\t1\t0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_385-LP3-ITS-0961_75_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_385-LP3-ITS-0961_75_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_385-LP3-ITS-0961_75_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_385-LP3-ITS-0961_75_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.217 s (10.808 ¬µs/read; 5.55 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,118\n",
      "Reads with adapters:                       298 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,118 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,055,518 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,052,106 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 298 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.2%\n",
      "  C: 3.4%\n",
      "  G: 0.0%\n",
      "  T: 17.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t67\t314.3\t0\t67\n",
      "4\t1\t78.6\t0\t1\n",
      "6\t2\t4.9\t0\t2\n",
      "9\t218\t0.1\t0\t203 15\n",
      "10\t4\t0.0\t1\t0 4\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "219\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_386-LP3-ITS-0962_528_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_386-LP3-ITS-0962_528_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_386-LP3-ITS-0962_528_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_386-LP3-ITS-0962_528_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (24.438 ¬µs/read; 2.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     375\n",
      "Reads with adapters:                         8 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           375 (100.0%)\n",
      "\n",
      "Total basepairs processed:       112,875 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        111,339 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 87.5%\n",
      "  C: 12.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t5.9\t0\t1\n",
      "219\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_387-LP3-ITS-0963_953_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_387-LP3-ITS-0963_953_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_387-LP3-ITS-0963_953_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_387-LP3-ITS-0963_953_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.222 s (10.376 ¬µs/read; 5.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,382\n",
      "Reads with adapters:                       117 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,382 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,435,982 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,415,740 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 117 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.0%\n",
      "  C: 0.9%\n",
      "  G: 0.0%\n",
      "  T: 64.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t5.2\t0\t1\n",
      "9\t1\t0.1\t0\t0 1\n",
      "152\t74\t0.0\t2\t0 0 74\n",
      "219\t41\t0.0\t2\t0 3 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_388-LP3-ITS-0964_1104_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_388-LP3-ITS-0964_1104_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_388-LP3-ITS-0964_1104_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_388-LP3-ITS-0964_1104_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.149 s (8.040 ¬µs/read; 7.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,580\n",
      "Reads with adapters:                       334 (1.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,580 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,592,580 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,588,032 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 334 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.2%\n",
      "  C: 3.6%\n",
      "  G: 0.3%\n",
      "  T: 6.9%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t295\t290.3\t0\t295\n",
      "4\t2\t72.6\t0\t2\n",
      "5\t1\t18.1\t0\t1\n",
      "6\t10\t4.5\t0\t10\n",
      "9\t3\t0.1\t0\t3\n",
      "152\t22\t0.0\t2\t0 0 22\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_389-LP3-ITS-0965_429_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_389-LP3-ITS-0965_429_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_389-LP3-ITS-0965_429_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_389-LP3-ITS-0965_429_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.199 s (8.250 ¬µs/read; 7.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,077\n",
      "Reads with adapters:                     1,115 (4.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,077 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,247,177 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,243,565 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1115 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 96.1%\n",
      "  C: 3.4%\n",
      "  G: 0.0%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1087\t376.2\t0\t1087\n",
      "4\t24\t94.1\t0\t24\n",
      "5\t1\t23.5\t0\t1\n",
      "6\t1\t5.9\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_390-LP3-ITS-0966_640_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_390-LP3-ITS-0966_640_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_390-LP3-ITS-0966_640_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_390-LP3-ITS-0966_640_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.163 s (9.346 ¬µs/read; 6.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,397\n",
      "Reads with adapters:                     2,262 (13.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,397 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,236,497 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,228,586 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2262 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.0%\n",
      "  C: 7.4%\n",
      "  G: 0.1%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2237\t271.8\t0\t2237\n",
      "4\t9\t68.0\t0\t9\n",
      "5\t1\t17.0\t0\t1\n",
      "6\t5\t4.2\t0\t5\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_391-LP3-ITS-0967_960_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_391-LP3-ITS-0967_960_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_391-LP3-ITS-0967_960_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_391-LP3-ITS-0967_960_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (69.645 ¬µs/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      88\n",
      "Reads with adapters:                         2 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            88 (100.0%)\n",
      "\n",
      "Total basepairs processed:        26,488 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         26,480 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t0.3\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_392-LP3-ITS-0968_242_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_392-LP3-ITS-0968_242_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_392-LP3-ITS-0968_242_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_392-LP3-ITS-0968_242_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (69.185 ¬µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      82\n",
      "Reads with adapters:                         1 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            82 (100.0%)\n",
      "\n",
      "Total basepairs processed:        24,682 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         24,679 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 100.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.3\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_393-LP3-ITS-0969_424_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_393-LP3-ITS-0969_424_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_393-LP3-ITS-0969_424_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_393-LP3-ITS-0969_424_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.193 s (10.142 ¬µs/read; 5.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,984\n",
      "Reads with adapters:                       274 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,984 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,714,184 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,707,886 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 274 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.4%\n",
      "  C: 14.2%\n",
      "  G: 1.1%\n",
      "  T: 18.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t296.6\t0\t88\n",
      "4\t5\t74.2\t0\t5\n",
      "5\t3\t18.5\t0\t3\n",
      "6\t20\t4.6\t0\t20\n",
      "7\t1\t1.2\t0\t1\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t124\t0.1\t0\t116 8\n",
      "10\t6\t0.0\t1\t0 6\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "175\t15\t0.0\t2\t0 0 15\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "219\t4\t0.0\t2\t0 0 4\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_394-LP3-ITS-0970_534_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_394-LP3-ITS-0970_534_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_394-LP3-ITS-0970_534_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_394-LP3-ITS-0970_534_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.174 s (9.231 ¬µs/read; 6.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,872\n",
      "Reads with adapters:                       133 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,872 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,680,472 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,673,774 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 133 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.2%\n",
      "  C: 23.3%\n",
      "  G: 0.0%\n",
      "  T: 10.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t294.9\t0\t42\n",
      "4\t13\t73.7\t0\t13\n",
      "6\t17\t4.6\t0\t17\n",
      "7\t2\t1.2\t0\t2\n",
      "9\t27\t0.1\t0\t24 3\n",
      "10\t3\t0.0\t1\t0 3\n",
      "175\t5\t0.0\t2\t0 0 5\n",
      "219\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_395-LP3-ITS-0971_286_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_395-LP3-ITS-0971_286_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_395-LP3-ITS-0971_286_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_395-LP3-ITS-0971_286_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.156 s (8.397 ¬µs/read; 7.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,595\n",
      "Reads with adapters:                       103 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,595 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,597,095 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,586,556 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 103 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.0%\n",
      "  C: 29.1%\n",
      "  G: 0.0%\n",
      "  T: 3.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t290.5\t0\t12\n",
      "4\t9\t72.6\t0\t9\n",
      "5\t2\t18.2\t0\t2\n",
      "6\t21\t4.5\t0\t21\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "151\t9\t0.0\t2\t0 0 9\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "175\t17\t0.0\t2\t0 0 17\n",
      "219\t8\t0.0\t2\t0 0 8\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t8\t0.0\t2\t0 0 8\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_396-LP3-ITS-0972_927_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_396-LP3-ITS-0972_927_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_396-LP3-ITS-0972_927_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_396-LP3-ITS-0972_927_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.137 s (9.213 ¬µs/read; 6.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,840\n",
      "Reads with adapters:                        53 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,466,840 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,462,895 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 53 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 34.0%\n",
      "  C: 32.1%\n",
      "  G: 0.0%\n",
      "  T: 34.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t231.9\t0\t10\n",
      "4\t1\t58.0\t0\t1\n",
      "5\t1\t14.5\t0\t1\n",
      "6\t11\t3.6\t0\t11\n",
      "7\t2\t0.9\t0\t2\n",
      "9\t4\t0.1\t0\t4\n",
      "10\t1\t0.0\t1\t0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "152\t14\t0.0\t2\t0 0 14\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "219\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_397-LP3-ITS-0973_618_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_397-LP3-ITS-0973_618_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_397-LP3-ITS-0973_618_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_397-LP3-ITS-0973_618_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.196 s (7.296 ¬µs/read; 8.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,886\n",
      "Reads with adapters:                       105 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,886 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,092,686 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,088,835 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 105 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.6%\n",
      "  C: 48.6%\n",
      "  G: 1.0%\n",
      "  T: 1.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t420.1\t0\t7\n",
      "4\t7\t105.0\t0\t7\n",
      "5\t16\t26.3\t0\t16\n",
      "6\t54\t6.6\t0\t54\n",
      "7\t1\t1.6\t0\t1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "175\t10\t0.0\t2\t0 0 10\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_398-LP3-ITS-0974_577_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_398-LP3-ITS-0974_577_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_398-LP3-ITS-0974_577_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_398-LP3-ITS-0974_577_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.210 s (7.376 ¬µs/read; 8.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,404\n",
      "Reads with adapters:                       109 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,404 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,549,604 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,543,851 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 109 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.5%\n",
      "  C: 37.6%\n",
      "  G: 0.0%\n",
      "  T: 12.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t443.8\t0\t43\n",
      "4\t1\t111.0\t0\t1\n",
      "5\t1\t27.7\t0\t1\n",
      "6\t17\t6.9\t0\t17\n",
      "9\t1\t0.1\t0\t0 1\n",
      "10\t9\t0.0\t1\t0 9\n",
      "30\t5\t0.0\t2\t0 0 5\n",
      "33\t5\t0.0\t2\t0 1 4\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "175\t13\t0.0\t2\t0 0 13\n",
      "233\t4\t0.0\t2\t0 2 2\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_399-LP3-ITS-0975_462_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_399-LP3-ITS-0975_462_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_399-LP3-ITS-0975_462_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_399-LP3-ITS-0975_462_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.216 s (8.124 ¬µs/read; 7.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,572\n",
      "Reads with adapters:                        68 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,572 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,998,172 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,994,201 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 68 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.6%\n",
      "  C: 50.0%\n",
      "  G: 1.5%\n",
      "  T: 2.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t415.2\t0\t18\n",
      "4\t2\t103.8\t0\t2\n",
      "5\t1\t25.9\t0\t1\n",
      "6\t25\t6.5\t0\t25\n",
      "7\t1\t1.6\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "76\t1\t0.0\t2\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "175\t5\t0.0\t2\t0 0 5\n",
      "212\t3\t0.0\t2\t0 0 3\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t3\t0.0\t2\t0 0 3\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_400-LP3-ITS-0976_735_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_400-LP3-ITS-0976_735_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_400-LP3-ITS-0976_735_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_400-LP3-ITS-0976_735_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.098 s (11.205 ¬µs/read; 5.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   8,767\n",
      "Reads with adapters:                        83 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         8,767 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,638,867 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,634,905 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 83 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.7%\n",
      "  C: 24.1%\n",
      "  G: 1.2%\n",
      "  T: 6.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t137.0\t0\t24\n",
      "4\t20\t34.2\t0\t20\n",
      "5\t3\t8.6\t0\t3\n",
      "6\t9\t2.1\t0\t9\n",
      "9\t1\t0.0\t0\t0 1\n",
      "10\t5\t0.0\t1\t1 4\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "175\t7\t0.0\t2\t0 0 7\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_401-LP3-ITS-0977_648_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_401-LP3-ITS-0977_648_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_401-LP3-ITS-0977_648_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_401-LP3-ITS-0977_648_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.167 s (7.911 ¬µs/read; 7.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,072\n",
      "Reads with adapters:                       133 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,072 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,342,672 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,337,087 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 133 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 49.6%\n",
      "  C: 29.3%\n",
      "  G: 2.3%\n",
      "  T: 18.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t47\t329.2\t0\t47\n",
      "4\t2\t82.3\t0\t2\n",
      "5\t1\t20.6\t0\t1\n",
      "6\t26\t5.1\t0\t26\n",
      "7\t1\t1.3\t0\t1\n",
      "9\t6\t0.1\t0\t4 2\n",
      "10\t21\t0.0\t1\t1 20\n",
      "30\t2\t0.0\t2\t0 0 2\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "94\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "175\t10\t0.0\t2\t0 0 10\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_402-LP3-ITS-0978_788_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_402-LP3-ITS-0978_788_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_402-LP3-ITS-0978_788_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_402-LP3-ITS-0978_788_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (93.351 ¬µs/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      57\n",
      "Reads with adapters:                         0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            57 (100.0%)\n",
      "\n",
      "Total basepairs processed:        17,157 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         17,157 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_403-LP3-ITS-0979_492_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_403-LP3-ITS-0979_492_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_403-LP3-ITS-0979_492_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_403-LP3-ITS-0979_492_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.130 s (7.493 ¬µs/read; 8.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,375\n",
      "Reads with adapters:                        66 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,229,875 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,226,013 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 66 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.9%\n",
      "  C: 53.0%\n",
      "  G: 0.0%\n",
      "  T: 3.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t271.5\t0\t9\n",
      "4\t4\t67.9\t0\t4\n",
      "5\t2\t17.0\t0\t2\n",
      "6\t28\t4.2\t0\t28\n",
      "7\t3\t1.1\t0\t3\n",
      "10\t2\t0.0\t1\t0 2\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "175\t6\t0.0\t2\t0 0 6\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "234\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_404-LP3-ITS-0980_345_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_404-LP3-ITS-0980_345_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_404-LP3-ITS-0980_345_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_404-LP3-ITS-0980_345_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (76.593 ¬µs/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      79\n",
      "Reads with adapters:                         1 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            79 (100.0%)\n",
      "\n",
      "Total basepairs processed:        23,779 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         23,637 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_405-LP3-ITS-0981_150_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_405-LP3-ITS-0981_150_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_405-LP3-ITS-0981_150_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_405-LP3-ITS-0981_150_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.198 s (7.419 ¬µs/read; 8.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,704\n",
      "Reads with adapters:                       268 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,704 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,037,904 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,032,344 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 268 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 56.3%\n",
      "  C: 31.3%\n",
      "  G: 2.2%\n",
      "  T: 10.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t121\t417.2\t0\t121\n",
      "4\t63\t104.3\t0\t63\n",
      "5\t2\t26.1\t0\t2\n",
      "6\t28\t6.5\t0\t28\n",
      "7\t1\t1.6\t0\t1\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t5\t0.1\t0\t5\n",
      "10\t16\t0.0\t1\t2 14\n",
      "11\t1\t0.0\t1\t0 1\n",
      "30\t3\t0.0\t2\t0 0 3\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "37\t1\t0.0\t2\t0 1\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "175\t10\t0.0\t2\t0 0 10\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_406-LP3-ITS-0982_990_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_406-LP3-ITS-0982_990_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_406-LP3-ITS-0982_990_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_406-LP3-ITS-0982_990_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.173 s (7.889 ¬µs/read; 7.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,918\n",
      "Reads with adapters:                        54 (0.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,918 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,597,318 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,594,683 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 54 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.7%\n",
      "  C: 57.4%\n",
      "  G: 1.9%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t342.5\t0\t5\n",
      "4\t3\t85.6\t0\t3\n",
      "5\t3\t21.4\t0\t3\n",
      "6\t28\t5.4\t0\t28\n",
      "10\t3\t0.0\t1\t0 3\n",
      "175\t7\t0.0\t2\t0 0 7\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_407-LP3-ITS-0983_72_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_407-LP3-ITS-0983_72_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_407-LP3-ITS-0983_72_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_407-LP3-ITS-0983_72_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.171 s (6.939 ¬µs/read; 8.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,597\n",
      "Reads with adapters:                       118 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,597 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,403,697 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,400,944 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 118 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.8%\n",
      "  C: 69.5%\n",
      "  G: 0.8%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t384.3\t0\t12\n",
      "4\t60\t96.1\t0\t60\n",
      "5\t9\t24.0\t0\t9\n",
      "6\t22\t6.0\t0\t22\n",
      "10\t1\t0.0\t1\t0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "175\t11\t0.0\t2\t0 0 11\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_408-LP3-ITS-0984_672_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_408-LP3-ITS-0984_672_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_408-LP3-ITS-0984_672_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_408-LP3-ITS-0984_672_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.146 s (9.819 ¬µs/read; 6.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,901\n",
      "Reads with adapters:                       120 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,901 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,485,201 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,475,733 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 120 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 73.3%\n",
      "  C: 18.3%\n",
      "  G: 2.5%\n",
      "  T: 5.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t232.8\t0\t27\n",
      "4\t21\t58.2\t0\t21\n",
      "6\t13\t3.6\t0\t13\n",
      "7\t1\t0.9\t0\t1\n",
      "10\t5\t0.0\t1\t1 4\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "76\t8\t0.0\t2\t0 0 8\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "151\t8\t0.0\t2\t0 0 8\n",
      "175\t13\t0.0\t2\t0 0 13\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t10\t0.0\t2\t0 0 10\n",
      "235\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_409-LP3-ITS-0985_530_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_409-LP3-ITS-0985_530_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_409-LP3-ITS-0985_530_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_409-LP3-ITS-0985_530_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.155 s (7.343 ¬µs/read; 8.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,106\n",
      "Reads with adapters:                        56 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,106 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,352,906 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,350,575 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 56 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.5%\n",
      "  C: 62.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t329.8\t0\t2\n",
      "4\t14\t82.4\t0\t14\n",
      "5\t3\t20.6\t0\t3\n",
      "6\t22\t5.2\t0\t22\n",
      "7\t2\t1.3\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "175\t5\t0.0\t2\t0 0 5\n",
      "219\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_410-LP3-ITS-0986_399_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_410-LP3-ITS-0986_399_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_410-LP3-ITS-0986_399_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_410-LP3-ITS-0986_399_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.150 s (8.730 ¬µs/read; 6.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,198\n",
      "Reads with adapters:                       279 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,198 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,176,598 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,164,213 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 279 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.9%\n",
      "  C: 48.4%\n",
      "  G: 1.4%\n",
      "  T: 18.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t182\t268.7\t0\t182\n",
      "5\t1\t16.8\t0\t1\n",
      "6\t9\t4.2\t0\t9\n",
      "9\t4\t0.1\t0\t4\n",
      "10\t23\t0.0\t1\t0 23\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "31\t1\t0.0\t2\t0 0 1\n",
      "42\t2\t0.0\t2\t0 0 2\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "124\t7\t0.0\t2\t0 0 7\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "175\t9\t0.0\t2\t0 0 9\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "233\t7\t0.0\t2\t0 0 7\n",
      "234\t17\t0.0\t2\t0 0 17\n",
      "235\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_411-LP3-ITS-0987_518_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_411-LP3-ITS-0987_518_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_411-LP3-ITS-0987_518_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_411-LP3-ITS-0987_518_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.262 s (8.599 ¬µs/read; 6.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,471\n",
      "Reads with adapters:                       525 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,471 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,171,771 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,159,358 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 525 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.4%\n",
      "  C: 45.3%\n",
      "  G: 1.3%\n",
      "  T: 21.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t328\t476.1\t0\t328\n",
      "4\t12\t119.0\t0\t12\n",
      "5\t5\t29.8\t0\t5\n",
      "6\t37\t7.4\t0\t37\n",
      "7\t1\t1.9\t0\t1\n",
      "9\t10\t0.1\t0\t10\n",
      "10\t65\t0.0\t1\t5 60\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t2\t0.0\t2\t0 0 2\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "30\t7\t0.0\t2\t0 0 7\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "175\t24\t0.0\t2\t0 0 24\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "219\t5\t0.0\t2\t0 0 5\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "233\t4\t0.0\t2\t0 0 4\n",
      "234\t4\t0.0\t2\t0 0 4\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_412-LP3-ITS-0988_184_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_412-LP3-ITS-0988_184_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_412-LP3-ITS-0988_184_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_412-LP3-ITS-0988_184_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.042 s (10.721 ¬µs/read; 5.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   3,922\n",
      "Reads with adapters:                        16 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         3,922 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,180,522 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,179,398 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.8%\n",
      "  C: 18.8%\n",
      "  G: 0.0%\n",
      "  T: 12.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t61.3\t0\t4\n",
      "6\t3\t1.0\t0\t3\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_413-LP3-ITS-0989_994_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_413-LP3-ITS-0989_994_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_413-LP3-ITS-0989_994_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_413-LP3-ITS-0989_994_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.266 s (8.950 ¬µs/read; 6.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  29,706\n",
      "Reads with adapters:                     2,754 (9.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        29,706 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,941,506 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,928,724 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2754 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.8%\n",
      "  C: 8.6%\n",
      "  G: 0.1%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2714\t464.2\t0\t2714\n",
      "4\t11\t116.0\t0\t11\n",
      "5\t2\t29.0\t0\t2\n",
      "10\t1\t0.0\t1\t1\n",
      "176\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_414-LP3-ITS-0990_791_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_414-LP3-ITS-0990_791_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_414-LP3-ITS-0990_791_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_414-LP3-ITS-0990_791_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.136 s (8.364 ¬µs/read; 7.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,210\n",
      "Reads with adapters:                       170 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,210 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,879,210 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,876,461 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 170 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.8%\n",
      "  C: 48.8%\n",
      "  G: 0.0%\n",
      "  T: 12.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t115\t253.3\t0\t115\n",
      "4\t5\t63.3\t0\t5\n",
      "6\t14\t4.0\t0\t14\n",
      "7\t1\t1.0\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t23\t0.0\t1\t1 22\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "175\t6\t0.0\t2\t0 0 6\n",
      "234\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_415-LP3-ITS-0991_593_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_415-LP3-ITS-0991_593_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_415-LP3-ITS-0991_593_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_415-LP3-ITS-0991_593_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.256 s (7.389 ¬µs/read; 8.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  34,622\n",
      "Reads with adapters:                       434 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        34,622 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,421,222 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,418,872 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 434 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 54.1%\n",
      "  C: 35.3%\n",
      "  G: 1.2%\n",
      "  T: 9.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t242\t541.0\t0\t242\n",
      "4\t129\t135.2\t0\t129\n",
      "5\t3\t33.8\t0\t3\n",
      "6\t27\t8.5\t0\t27\n",
      "7\t3\t2.1\t0\t3\n",
      "9\t3\t0.1\t0\t3\n",
      "10\t24\t0.0\t1\t0 24\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_416-LP3-ITS-0992_615_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_416-LP3-ITS-0992_615_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_416-LP3-ITS-0992_615_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_416-LP3-ITS-0992_615_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.135 s (9.049 ¬µs/read; 6.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,961\n",
      "Reads with adapters:                       299 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,961 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,503,261 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,501,682 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 299 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.8%\n",
      "  C: 36.5%\n",
      "  G: 1.3%\n",
      "  T: 13.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t164\t233.8\t0\t164\n",
      "4\t91\t58.4\t0\t91\n",
      "5\t1\t14.6\t0\t1\n",
      "6\t14\t3.7\t0\t14\n",
      "9\t6\t0.1\t0\t5 1\n",
      "10\t20\t0.0\t1\t2 18\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_417-LP3-ITS-0993_1129_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_417-LP3-ITS-0993_1129_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_417-LP3-ITS-0993_1129_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_417-LP3-ITS-0993_1129_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.012 s (14.600 ¬µs/read; 4.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     790\n",
      "Reads with adapters:                         4 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           790 (100.0%)\n",
      "\n",
      "Total basepairs processed:       237,790 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        237,458 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 25.0%\n",
      "  G: 0.0%\n",
      "  T: 25.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t12.3\t0\t1\n",
      "5\t1\t0.8\t0\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_418-LP3-ITS-0994_1025_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_418-LP3-ITS-0994_1025_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_418-LP3-ITS-0994_1025_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_418-LP3-ITS-0994_1025_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.067 s (15.226 ¬µs/read; 3.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   4,390\n",
      "Reads with adapters:                        49 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         4,390 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,321,390 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,318,020 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 49 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.8%\n",
      "  C: 8.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "42\t33\t0.0\t2\t0 0 33\n",
      "124\t16\t0.0\t2\t0 1 15\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_419-LP3-ITS-0995_393_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_419-LP3-ITS-0995_393_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_419-LP3-ITS-0995_393_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_419-LP3-ITS-0995_393_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.338 s (8.050 ¬µs/read; 7.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,015\n",
      "Reads with adapters:                       150 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,015 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,646,515 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,633,947 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 150 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 11.3%\n",
      "  C: 88.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t2\t41.0\t0\t2\n",
      "6\t26\t10.3\t0\t26\n",
      "7\t1\t2.6\t0\t1\n",
      "89\t38\t0.0\t2\t0 1 37\n",
      "102\t70\t0.0\t2\t0 1 69\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "164\t3\t0.0\t2\t0 1 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_420-LP3-ITS-0996_759_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_420-LP3-ITS-0996_759_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_420-LP3-ITS-0996_759_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_420-LP3-ITS-0996_759_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.005 s (98.056 ¬µs/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      54\n",
      "Reads with adapters:                         2 (3.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            54 (100.0%)\n",
      "\n",
      "Total basepairs processed:        16,254 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         16,133 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t0.8\t0\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_421-LP3-ITS-0997_211_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_421-LP3-ITS-0997_211_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_421-LP3-ITS-0997_211_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_421-LP3-ITS-0997_211_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (58.713 ¬µs/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     115\n",
      "Reads with adapters:                         4 (3.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           115 (100.0%)\n",
      "\n",
      "Total basepairs processed:        34,615 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         34,360 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.8\t0\t1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_422-LP3-ITS-0998_965_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_422-LP3-ITS-0998_965_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_422-LP3-ITS-0998_965_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_422-LP3-ITS-0998_965_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.105 s (8.979 ¬µs/read; 6.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,652\n",
      "Reads with adapters:                        23 (0.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,652 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,507,252 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,504,727 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 23 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 56.5%\n",
      "  C: 43.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t182.1\t0\t1\n",
      "6\t6\t2.8\t0\t6\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "175\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_423-LP3-ITS-0999_671_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_423-LP3-ITS-0999_671_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_423-LP3-ITS-0999_671_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_423-LP3-ITS-0999_671_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.204 s (8.155 ¬µs/read; 7.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,035\n",
      "Reads with adapters:                        93 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,035 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,535,535 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,528,432 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 93 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.7%\n",
      "  C: 73.1%\n",
      "  G: 2.2%\n",
      "  T: 1.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t391.2\t0\t2\n",
      "4\t10\t97.8\t0\t10\n",
      "5\t2\t24.4\t0\t2\n",
      "6\t22\t6.1\t0\t22\n",
      "7\t1\t1.5\t0\t1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t39\t0.0\t2\t0 0 39\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "175\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_424-LP3-ITS-1000_1121_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_424-LP3-ITS-1000_1121_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_424-LP3-ITS-1000_1121_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_424-LP3-ITS-1000_1121_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.021 s (18.432 ¬µs/read; 3.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   1,123\n",
      "Reads with adapters:                        10 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         1,123 (100.0%)\n",
      "\n",
      "Total basepairs processed:       338,023 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        336,633 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 10 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.0%\n",
      "  C: 10.0%\n",
      "  G: 0.0%\n",
      "  T: 10.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t0.3\t0\t1\n",
      "7\t1\t0.1\t0\t1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "175\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_425-LP3-ITS-1001_898_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_425-LP3-ITS-1001_898_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_425-LP3-ITS-1001_898_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_425-LP3-ITS-1001_898_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.128 s (7.312 ¬µs/read; 8.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,498\n",
      "Reads with adapters:                        68 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,498 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,266,898 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,264,008 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 68 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.2%\n",
      "  C: 45.6%\n",
      "  G: 0.0%\n",
      "  T: 16.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t273.4\t0\t30\n",
      "5\t2\t17.1\t0\t2\n",
      "6\t16\t4.3\t0\t16\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "175\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_426-LP3-ITS-1002_354_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_426-LP3-ITS-1002_354_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_426-LP3-ITS-1002_354_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_426-LP3-ITS-1002_354_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.059 s (7.889 ¬µs/read; 7.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,497\n",
      "Reads with adapters:                        13 (0.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,497 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,256,597 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,256,190 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 13 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 15.4%\n",
      "  C: 61.5%\n",
      "  G: 0.0%\n",
      "  T: 23.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t117.1\t0\t3\n",
      "6\t8\t1.8\t0\t8\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_427-LP3-ITS-1003_566_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_427-LP3-ITS-1003_566_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_427-LP3-ITS-1003_566_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_427-LP3-ITS-1003_566_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.126 s (9.115 ¬µs/read; 6.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,796\n",
      "Reads with adapters:                        34 (0.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,152,596 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,151,404 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 34 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t215.6\t0\t3\n",
      "4\t8\t53.9\t0\t8\n",
      "5\t1\t13.5\t0\t1\n",
      "6\t16\t3.4\t0\t16\n",
      "175\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_428-LP3-ITS-1004_969_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_428-LP3-ITS-1004_969_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_428-LP3-ITS-1004_969_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_428-LP3-ITS-1004_969_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.060 s (14.251 ¬µs/read; 4.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   4,183\n",
      "Reads with adapters:                        60 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         4,183 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,259,083 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,257,253 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 60 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 38.3%\n",
      "  C: 46.7%\n",
      "  G: 3.3%\n",
      "  T: 11.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t65.4\t0\t32\n",
      "6\t2\t1.0\t0\t2\n",
      "7\t1\t0.3\t0\t1\n",
      "9\t2\t0.0\t0\t2\n",
      "10\t3\t0.0\t1\t0 3\n",
      "42\t10\t0.0\t2\t0 0 10\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_429-LP3-ITS-1005_604_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_429-LP3-ITS-1005_604_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_429-LP3-ITS-1005_604_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_429-LP3-ITS-1005_604_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.034 s (9.939 ¬µs/read; 6.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   3,419\n",
      "Reads with adapters:                         4 (0.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         3,419 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,029,119 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,028,926 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.0%\n",
      "  C: 50.0%\n",
      "  G: 25.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t3\t0.8\t0\t3\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_430-LP3-ITS-1006_470_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_430-LP3-ITS-1006_470_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_430-LP3-ITS-1006_470_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_430-LP3-ITS-1006_470_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.029 s (13.227 ¬µs/read; 4.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   2,229\n",
      "Reads with adapters:                         8 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         2,229 (100.0%)\n",
      "\n",
      "Total basepairs processed:       670,929 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        670,516 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 62.5%\n",
      "  C: 37.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t34.8\t0\t1\n",
      "6\t3\t0.5\t0\t3\n",
      "33\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_431-LP3-ITS-1007_612_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_431-LP3-ITS-1007_612_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_431-LP3-ITS-1007_612_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_431-LP3-ITS-1007_612_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.050 s (10.951 ¬µs/read; 5.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   4,593\n",
      "Reads with adapters:                     1,091 (23.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         4,593 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,382,493 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,378,113 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1091 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 9.7%\n",
      "  C: 89.3%\n",
      "  G: 0.7%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t197\t71.8\t0\t197\n",
      "4\t883\t17.9\t0\t883\n",
      "5\t1\t4.5\t0\t1\n",
      "6\t5\t1.1\t0\t5\n",
      "9\t1\t0.0\t0\t1\n",
      "11\t1\t0.0\t1\t0 1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "14\t1\t0.0\t1\t0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_432-LP3-ITS-1008_452_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_432-LP3-ITS-1008_452_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_432-LP3-ITS-1008_452_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_432-LP3-ITS-1008_452_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.137 s (9.027 ¬µs/read; 6.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,155\n",
      "Reads with adapters:                     1,983 (13.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,155 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,561,655 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,553,045 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1983 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 96.7%\n",
      "  C: 2.6%\n",
      "  G: 0.3%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1932\t236.8\t0\t1932\n",
      "4\t9\t59.2\t0\t9\n",
      "5\t2\t14.8\t0\t2\n",
      "6\t2\t3.7\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "73\t36\t0.0\t2\t0 3 33\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_433-LP3-ITS-1009_864_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_433-LP3-ITS-1009_864_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_433-LP3-ITS-1009_864_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_433-LP3-ITS-1009_864_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (38.835 ¬µs/read; 1.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     194\n",
      "Reads with adapters:                         8 (4.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           194 (100.0%)\n",
      "\n",
      "Total basepairs processed:        58,394 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         57,579 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t3.0\t0\t1\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_434-LP3-ITS-1010_368_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_434-LP3-ITS-1010_368_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_434-LP3-ITS-1010_368_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_434-LP3-ITS-1010_368_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.012 s (12.793 ¬µs/read; 4.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     964\n",
      "Reads with adapters:                         1 (0.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           964 (100.0%)\n",
      "\n",
      "Total basepairs processed:       290,164 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        290,077 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 0.0%\n",
      "  C: 100.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_435-LP3-ITS-1011_634_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_435-LP3-ITS-1011_634_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_435-LP3-ITS-1011_634_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_435-LP3-ITS-1011_634_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.094 s (7.515 ¬µs/read; 7.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,567\n",
      "Reads with adapters:                       147 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,567 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,782,667 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,780,864 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 147 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.4%\n",
      "  C: 54.4%\n",
      "  G: 2.0%\n",
      "  T: 23.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t99\t196.4\t0\t99\n",
      "4\t5\t49.1\t0\t5\n",
      "5\t2\t12.3\t0\t2\n",
      "6\t15\t3.1\t0\t15\n",
      "7\t1\t0.8\t0\t1\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t17\t0.0\t1\t0 17\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "175\t3\t0.0\t2\t0 0 3\n",
      "212\t2\t0.0\t2\t0 0 2\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_436-LP3-ITS-1012_833_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_436-LP3-ITS-1012_833_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_436-LP3-ITS-1012_833_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_436-LP3-ITS-1012_833_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.066 s (8.325 ¬µs/read; 7.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,978\n",
      "Reads with adapters:                        62 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,978 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,401,378 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,400,695 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 62 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.9%\n",
      "  C: 53.2%\n",
      "  G: 1.6%\n",
      "  T: 11.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t27\t124.7\t0\t27\n",
      "4\t3\t31.2\t0\t3\n",
      "5\t2\t7.8\t0\t2\n",
      "6\t18\t1.9\t0\t18\n",
      "9\t2\t0.0\t0\t2\n",
      "10\t7\t0.0\t1\t0 7\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_437-LP3-ITS-1013_713_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_437-LP3-ITS-1013_713_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_437-LP3-ITS-1013_713_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_437-LP3-ITS-1013_713_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.137 s (8.725 ¬µs/read; 6.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,648\n",
      "Reads with adapters:                       130 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,710,048 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,707,705 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 130 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.5%\n",
      "  C: 50.8%\n",
      "  G: 0.0%\n",
      "  T: 20.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t82\t244.5\t0\t82\n",
      "4\t1\t61.1\t0\t1\n",
      "6\t14\t3.8\t0\t14\n",
      "9\t3\t0.1\t0\t3\n",
      "10\t19\t0.0\t1\t1 18\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "175\t4\t0.0\t2\t0 0 4\n",
      "219\t2\t0.0\t2\t0 0 2\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_438-LP3-ITS-1014_637_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_438-LP3-ITS-1014_637_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_438-LP3-ITS-1014_637_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_438-LP3-ITS-1014_637_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.086 s (9.599 ¬µs/read; 6.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   8,950\n",
      "Reads with adapters:                        39 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         8,950 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,693,950 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,692,441 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 39 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.9%\n",
      "  C: 41.0%\n",
      "  G: 0.0%\n",
      "  T: 23.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t139.8\t0\t14\n",
      "5\t1\t8.7\t0\t1\n",
      "6\t11\t2.2\t0\t11\n",
      "9\t3\t0.0\t0\t3\n",
      "10\t2\t0.0\t1\t0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "175\t5\t0.0\t2\t0 0 5\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_439-LP3-ITS-1015_940_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_439-LP3-ITS-1015_940_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_439-LP3-ITS-1015_940_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_439-LP3-ITS-1015_940_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.141 s (8.997 ¬µs/read; 6.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,650\n",
      "Reads with adapters:                       154 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,650 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,710,650 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,706,692 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 154 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.3%\n",
      "  C: 55.2%\n",
      "  G: 1.9%\n",
      "  T: 17.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t104\t244.5\t0\t104\n",
      "4\t1\t61.1\t0\t1\n",
      "6\t9\t3.8\t0\t9\n",
      "9\t3\t0.1\t0\t2 1\n",
      "10\t15\t0.0\t1\t0 15\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "33\t2\t0.0\t2\t0 0 2\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "175\t16\t0.0\t2\t0 0 16\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_440-LP3-ITS-1016_893_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_440-LP3-ITS-1016_893_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_440-LP3-ITS-1016_893_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_440-LP3-ITS-1016_893_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.059 s (8.109 ¬µs/read; 7.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,247\n",
      "Reads with adapters:                        50 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,247 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,181,347 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,180,189 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 50 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.0%\n",
      "  C: 56.0%\n",
      "  G: 0.0%\n",
      "  T: 22.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t37\t113.2\t0\t37\n",
      "6\t4\t1.8\t0\t4\n",
      "10\t3\t0.0\t1\t0 3\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "175\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_441-LP3-ITS-1017_652_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_441-LP3-ITS-1017_652_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_441-LP3-ITS-1017_652_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_441-LP3-ITS-1017_652_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.023 s (10.460 ¬µs/read; 5.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   2,158\n",
      "Reads with adapters:                         8 (0.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         2,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:       649,558 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        649,389 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.5%\n",
      "  C: 62.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t33.7\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_442-LP3-ITS-1018_380_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_442-LP3-ITS-1018_380_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_442-LP3-ITS-1018_380_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_442-LP3-ITS-1018_380_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.030 s (13.104 ¬µs/read; 4.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   2,312\n",
      "Reads with adapters:                        26 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         2,312 (100.0%)\n",
      "\n",
      "Total basepairs processed:       695,912 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        694,913 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 26 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.3%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 7.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t36.1\t0\t3\n",
      "9\t3\t0.0\t0\t2 1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "42\t7\t0.0\t2\t0 0 7\n",
      "44\t11\t0.0\t2\t0 0 11\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_443-LP3-ITS-1019_8_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_443-LP3-ITS-1019_8_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_443-LP3-ITS-1019_8_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_443-LP3-ITS-1019_8_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.021 s (11.148 ¬µs/read; 5.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   1,906\n",
      "Reads with adapters:                       287 (15.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         1,906 (100.0%)\n",
      "\n",
      "Total basepairs processed:       573,706 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        572,577 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 287 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 15.0%\n",
      "  C: 57.5%\n",
      "  G: 1.0%\n",
      "  T: 26.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t248\t29.8\t0\t248\n",
      "6\t1\t0.5\t0\t1\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t37\t0.0\t1\t2 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_444-LP3-ITS-1020_1062_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_444-LP3-ITS-1020_1062_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_444-LP3-ITS-1020_1062_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_444-LP3-ITS-1020_1062_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (26.769 ¬µs/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     333\n",
      "Reads with adapters:                         3 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           333 (100.0%)\n",
      "\n",
      "Total basepairs processed:       100,233 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        100,096 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_445-LP3-ITS-1021_923_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_445-LP3-ITS-1021_923_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_445-LP3-ITS-1021_923_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_445-LP3-ITS-1021_923_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (61.176 ¬µs/read; 0.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     108\n",
      "Reads with adapters:                         4 (3.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           108 (100.0%)\n",
      "\n",
      "Total basepairs processed:        32,508 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         32,495 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.0%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 25.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t1.7\t0\t3\n",
      "4\t1\t0.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_446-LP3-ITS-1022_230_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_446-LP3-ITS-1022_230_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_446-LP3-ITS-1022_230_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_446-LP3-ITS-1022_230_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.042 s (16.822 ¬µs/read; 3.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   2,503\n",
      "Reads with adapters:                        20 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         2,503 (100.0%)\n",
      "\n",
      "Total basepairs processed:       753,403 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        752,033 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 20 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.0%\n",
      "  C: 75.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t39.1\t0\t1\n",
      "6\t1\t0.6\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "53\t13\t0.0\t2\t0 0 13\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_447-LP3-ITS-1023_31_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_447-LP3-ITS-1023_31_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_447-LP3-ITS-1023_31_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_447-LP3-ITS-1023_31_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.145 s (9.292 ¬µs/read; 6.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,645\n",
      "Reads with adapters:                       416 (2.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,645 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,709,145 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,704,198 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 416 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.7%\n",
      "  C: 56.7%\n",
      "  G: 1.7%\n",
      "  T: 14.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t294\t244.5\t0\t294\n",
      "5\t1\t15.3\t0\t1\n",
      "6\t15\t3.8\t0\t15\n",
      "7\t1\t1.0\t0\t1\n",
      "9\t4\t0.1\t0\t4\n",
      "10\t73\t0.0\t1\t4 69\n",
      "11\t1\t0.0\t1\t0 1\n",
      "42\t11\t0.0\t2\t0 1 10\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "175\t11\t0.0\t2\t0 0 11\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_448-LP3-ITS-1024_539_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_448-LP3-ITS-1024_539_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_448-LP3-ITS-1024_539_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_448-LP3-ITS-1024_539_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.244 s (10.983 ¬µs/read; 5.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,262\n",
      "Reads with adapters:                     2,082 (9.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,700,862 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,677,476 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2082 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.9%\n",
      "  C: 57.3%\n",
      "  G: 2.0%\n",
      "  T: 19.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1684\t347.8\t0\t1684\n",
      "6\t3\t5.4\t0\t3\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t22\t0.1\t0\t22\n",
      "10\t225\t0.0\t1\t8 217\n",
      "11\t2\t0.0\t1\t0 2\n",
      "42\t68\t0.0\t2\t0 2 66\n",
      "105\t11\t0.0\t2\t0 0 11\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "175\t56\t0.0\t2\t0 1 55\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_449-LP3-ITS-1025_760_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_449-LP3-ITS-1025_760_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_449-LP3-ITS-1025_760_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_449-LP3-ITS-1025_760_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.166 s (9.530 ¬µs/read; 6.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,446\n",
      "Reads with adapters:                     1,208 (6.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,446 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,251,246 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,240,641 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1208 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.5%\n",
      "  C: 44.4%\n",
      "  G: 1.0%\n",
      "  T: 31.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t928\t272.6\t0\t928\n",
      "6\t17\t4.3\t0\t17\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t24\t0.1\t0\t23 1\n",
      "10\t204\t0.0\t1\t10 194\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "175\t24\t0.0\t2\t0 0 24\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_450-LP3-ITS-1026_572_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_450-LP3-ITS-1026_572_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_450-LP3-ITS-1026_572_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_450-LP3-ITS-1026_572_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.009 s (22.693 ¬µs/read; 2.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     388\n",
      "Reads with adapters:                         3 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           388 (100.0%)\n",
      "\n",
      "Total basepairs processed:       116,788 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        116,623 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 100.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t0.4\t0\t1\n",
      "9\t1\t0.0\t0\t1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_451-LP3-ITS-1027_1038_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_451-LP3-ITS-1027_1038_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_451-LP3-ITS-1027_1038_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_451-LP3-ITS-1027_1038_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.037 s (9.005 ¬µs/read; 6.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   4,160\n",
      "Reads with adapters:                        22 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         4,160 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,252,160 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,251,568 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 22 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.8%\n",
      "  C: 68.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t65.0\t0\t2\n",
      "4\t13\t16.2\t0\t13\n",
      "6\t1\t1.0\t0\t1\n",
      "9\t2\t0.0\t0\t2\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_452-LP3-ITS-1028_471_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_452-LP3-ITS-1028_471_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_452-LP3-ITS-1028_471_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_452-LP3-ITS-1028_471_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.012 s (15.866 ¬µs/read; 3.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     778\n",
      "Reads with adapters:                         5 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           778 (100.0%)\n",
      "\n",
      "Total basepairs processed:       234,178 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        233,915 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.0%\n",
      "  C: 40.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t12.2\t0\t1\n",
      "9\t2\t0.0\t0\t1 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_453-LP3-ITS-1029_605_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_453-LP3-ITS-1029_605_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_453-LP3-ITS-1029_605_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_453-LP3-ITS-1029_605_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.159 s (10.108 ¬µs/read; 5.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,740\n",
      "Reads with adapters:                     1,367 (8.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,740 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,737,740 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,727,837 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1367 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 82.2%\n",
      "  C: 17.3%\n",
      "  G: 0.1%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1189\t245.9\t0\t1189\n",
      "4\t145\t61.5\t0\t145\n",
      "14\t1\t0.0\t1\t0 1\n",
      "175\t6\t0.0\t2\t0 0 6\n",
      "176\t24\t0.0\t2\t0 0 24\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_454-LP3-ITS-1030_988_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_454-LP3-ITS-1030_988_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_454-LP3-ITS-1030_988_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_454-LP3-ITS-1030_988_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.111 s (9.151 ¬µs/read; 6.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,100\n",
      "Reads with adapters:                       999 (8.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,100 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,642,100 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,636,273 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 999 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.3%\n",
      "  C: 7.8%\n",
      "  G: 0.4%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t960\t189.1\t0\t960\n",
      "4\t22\t47.3\t0\t22\n",
      "5\t1\t11.8\t0\t1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "176\t13\t0.0\t2\t0 0 13\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_455-LP3-ITS-1031_869_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_455-LP3-ITS-1031_869_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_455-LP3-ITS-1031_869_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_455-LP3-ITS-1031_869_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.089 s (7.669 ¬µs/read; 7.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,604\n",
      "Reads with adapters:                       102 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,604 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,492,804 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,491,553 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 102 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.6%\n",
      "  C: 61.8%\n",
      "  G: 0.0%\n",
      "  T: 16.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t181.3\t0\t74\n",
      "5\t1\t11.3\t0\t1\n",
      "6\t7\t2.8\t0\t7\n",
      "9\t2\t0.0\t0\t2\n",
      "10\t12\t0.0\t1\t0 12\n",
      "11\t1\t0.0\t1\t0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "175\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_456-LP3-ITS-1032_948_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_456-LP3-ITS-1032_948_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_456-LP3-ITS-1032_948_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_456-LP3-ITS-1032_948_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.152 s (8.251 ¬µs/read; 7.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,422\n",
      "Reads with adapters:                        55 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,422 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,545,022 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,542,974 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 55 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.7%\n",
      "  C: 63.6%\n",
      "  G: 0.0%\n",
      "  T: 3.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t287.8\t0\t13\n",
      "4\t1\t72.0\t0\t1\n",
      "5\t2\t18.0\t0\t2\n",
      "6\t26\t4.5\t0\t26\n",
      "9\t3\t0.1\t0\t3\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "175\t7\t0.0\t2\t0 1 6\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_457-LP3-ITS-1033_619_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_457-LP3-ITS-1033_619_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_457-LP3-ITS-1033_619_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_457-LP3-ITS-1033_619_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.178 s (7.389 ¬µs/read; 8.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,044\n",
      "Reads with adapters:                        80 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,044 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,237,244 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,234,164 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 80 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.5%\n",
      "  C: 42.5%\n",
      "  G: 2.5%\n",
      "  T: 22.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t375.7\t0\t34\n",
      "5\t2\t23.5\t0\t2\n",
      "6\t23\t5.9\t0\t23\n",
      "10\t6\t0.0\t1\t0 6\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "175\t11\t0.0\t2\t0 0 11\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_458-LP3-ITS-1034_359_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_458-LP3-ITS-1034_359_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_458-LP3-ITS-1034_359_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_458-LP3-ITS-1034_359_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.115 s (9.174 ¬µs/read; 6.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,498\n",
      "Reads with adapters:                       150 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,498 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,761,898 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,752,799 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 150 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.7%\n",
      "  C: 28.0%\n",
      "  G: 1.3%\n",
      "  T: 18.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t195.3\t0\t70\n",
      "4\t2\t48.8\t0\t2\n",
      "5\t1\t12.2\t0\t1\n",
      "6\t8\t3.1\t0\t8\n",
      "7\t2\t0.8\t0\t2\n",
      "9\t2\t0.0\t0\t2\n",
      "10\t16\t0.0\t1\t0 16\n",
      "76\t8\t0.0\t2\t0 1 7\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "175\t18\t0.0\t2\t0 0 18\n",
      "212\t4\t0.0\t2\t0 1 3\n",
      "232\t5\t0.0\t2\t0 0 5\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_459-LP3-ITS-1035_865_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_459-LP3-ITS-1035_865_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_459-LP3-ITS-1035_865_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_459-LP3-ITS-1035_865_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.109 s (10.906 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  10,035\n",
      "Reads with adapters:                       202 (2.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        10,035 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,020,535 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,017,207 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 202 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.7%\n",
      "  C: 52.0%\n",
      "  G: 0.5%\n",
      "  T: 19.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t141\t156.8\t0\t141\n",
      "4\t16\t39.2\t0\t16\n",
      "6\t9\t2.4\t0\t9\n",
      "10\t20\t0.0\t1\t1 19\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "76\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "175\t4\t0.0\t2\t0 1 3\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_460-LP3-ITS-1036_1099_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_460-LP3-ITS-1036_1099_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_460-LP3-ITS-1036_1099_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_460-LP3-ITS-1036_1099_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.126 s (8.485 ¬µs/read; 7.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,898\n",
      "Reads with adapters:                       217 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,898 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,484,298 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,474,424 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 217 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.4%\n",
      "  C: 40.1%\n",
      "  G: 1.8%\n",
      "  T: 9.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t103\t232.8\t0\t103\n",
      "4\t25\t58.2\t0\t25\n",
      "5\t1\t14.5\t0\t1\n",
      "6\t9\t3.6\t0\t9\n",
      "9\t3\t0.1\t0\t3\n",
      "10\t26\t0.0\t1\t1 25\n",
      "76\t5\t0.0\t2\t0 0 5\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "175\t26\t0.0\t2\t0 1 25\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t6\t0.0\t2\t0 0 6\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_461-LP3-ITS-1037_134_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_461-LP3-ITS-1037_134_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_461-LP3-ITS-1037_134_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_461-LP3-ITS-1037_134_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.217 s (8.206 ¬µs/read; 7.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,433\n",
      "Reads with adapters:                     1,153 (4.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,433 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,956,333 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,947,408 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1153 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 12.8%\n",
      "  C: 84.6%\n",
      "  G: 0.9%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t187\t413.0\t0\t187\n",
      "4\t905\t103.3\t0\t905\n",
      "5\t3\t25.8\t0\t3\n",
      "6\t15\t6.5\t0\t15\n",
      "8\t2\t0.4\t0\t2\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t9\t0.0\t1\t0 9\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t2\t0.0\t1\t1 1\n",
      "13\t2\t0.0\t1\t0 2\n",
      "14\t2\t0.0\t1\t0 2\n",
      "175\t19\t0.0\t2\t0 0 19\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_462-LP3-ITS-1038_683_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_462-LP3-ITS-1038_683_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_462-LP3-ITS-1038_683_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_462-LP3-ITS-1038_683_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.133 s (9.076 ¬µs/read; 6.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,653\n",
      "Reads with adapters:                        98 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,410,553 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,405,047 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 98 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.9%\n",
      "  C: 51.0%\n",
      "  G: 1.0%\n",
      "  T: 4.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t229.0\t0\t16\n",
      "4\t33\t57.2\t0\t33\n",
      "5\t1\t14.3\t0\t1\n",
      "6\t7\t3.6\t0\t7\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t4\t0.0\t1\t1 3\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t7\t0.0\t2\t0 0 7\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "175\t20\t0.0\t2\t0 0 20\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_463-LP3-ITS-1039_95_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_463-LP3-ITS-1039_95_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_463-LP3-ITS-1039_95_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_463-LP3-ITS-1039_95_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.188 s (8.486 ¬µs/read; 7.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,098\n",
      "Reads with adapters:                        77 (0.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,098 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,651,498 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,646,473 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 77 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.2%\n",
      "  C: 45.5%\n",
      "  G: 1.3%\n",
      "  T: 9.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t345.3\t0\t35\n",
      "4\t2\t86.3\t0\t2\n",
      "6\t8\t5.4\t0\t8\n",
      "7\t1\t1.3\t0\t1\n",
      "10\t4\t0.0\t1\t0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "175\t24\t0.0\t2\t0 0 24\n",
      "232\t1\t0.0\t2\t0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_464-LP3-ITS-1040_644_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_464-LP3-ITS-1040_644_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_464-LP3-ITS-1040_644_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_464-LP3-ITS-1040_644_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.138 s (7.958 ¬µs/read; 7.54 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,300\n",
      "Reads with adapters:                       241 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,300 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,207,300 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,205,264 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 241 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.5%\n",
      "  C: 53.1%\n",
      "  G: 1.7%\n",
      "  T: 13.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t175\t270.3\t0\t175\n",
      "4\t21\t67.6\t0\t21\n",
      "5\t3\t16.9\t0\t3\n",
      "6\t17\t4.2\t0\t17\n",
      "9\t2\t0.1\t0\t2\n",
      "10\t16\t0.0\t1\t1 15\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "175\t2\t0.0\t2\t0 0 2\n",
      "176\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_465-LP3-ITS-1041_734_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_465-LP3-ITS-1041_734_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_465-LP3-ITS-1041_734_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_465-LP3-ITS-1041_734_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.146 s (8.574 ¬µs/read; 7.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  16,989\n",
      "Reads with adapters:                       233 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        16,989 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,113,689 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,111,965 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 233 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 68.7%\n",
      "  C: 24.9%\n",
      "  G: 0.9%\n",
      "  T: 5.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t92\t265.5\t0\t92\n",
      "4\t101\t66.4\t0\t101\n",
      "5\t3\t16.6\t0\t3\n",
      "6\t21\t4.1\t0\t21\n",
      "7\t1\t1.0\t0\t1\n",
      "9\t2\t0.1\t0\t2\n",
      "10\t9\t0.0\t1\t0 9\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_466-LP3-ITS-1042_57_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_466-LP3-ITS-1042_57_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_466-LP3-ITS-1042_57_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_466-LP3-ITS-1042_57_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.106 s (7.436 ¬µs/read; 8.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,211\n",
      "Reads with adapters:                       116 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,211 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,277,511 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,276,881 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 116 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 65.5%\n",
      "  C: 27.6%\n",
      "  G: 0.0%\n",
      "  T: 6.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t60\t222.0\t0\t60\n",
      "4\t29\t55.5\t0\t29\n",
      "5\t1\t13.9\t0\t1\n",
      "6\t17\t3.5\t0\t17\n",
      "7\t1\t0.9\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t6\t0.0\t1\t0 6\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_467-LP3-ITS-1043_1116_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_467-LP3-ITS-1043_1116_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_467-LP3-ITS-1043_1116_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_467-LP3-ITS-1043_1116_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.170 s (8.186 ¬µs/read; 7.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,730\n",
      "Reads with adapters:                       151 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,730 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,239,730 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,237,154 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 151 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 57.6%\n",
      "  C: 35.8%\n",
      "  G: 0.0%\n",
      "  T: 6.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t49\t323.9\t0\t49\n",
      "4\t47\t81.0\t0\t47\n",
      "5\t3\t20.2\t0\t3\n",
      "6\t29\t5.1\t0\t29\n",
      "7\t2\t1.3\t0\t2\n",
      "10\t10\t0.0\t1\t1 9\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "175\t9\t0.0\t2\t0 0 9\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_468-LP3-ITS-1044_766_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_468-LP3-ITS-1044_766_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_468-LP3-ITS-1044_766_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_468-LP3-ITS-1044_766_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.119 s (7.605 ¬µs/read; 7.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,584\n",
      "Reads with adapters:                        78 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,584 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,690,784 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,686,124 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 78 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 43.6%\n",
      "  G: 1.3%\n",
      "  T: 5.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t243.5\t0\t18\n",
      "4\t24\t60.9\t0\t24\n",
      "5\t2\t15.2\t0\t2\n",
      "6\t8\t3.8\t0\t8\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "175\t9\t0.0\t2\t0 0 9\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t8\t0.0\t2\t0 0 8\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_469-LP3-ITS-1045_59_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_469-LP3-ITS-1045_59_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_469-LP3-ITS-1045_59_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_469-LP3-ITS-1045_59_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.252 s (7.707 ¬µs/read; 7.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,728\n",
      "Reads with adapters:                       178 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,728 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,851,128 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,845,279 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 178 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.5%\n",
      "  C: 46.6%\n",
      "  G: 0.0%\n",
      "  T: 7.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t52\t511.4\t0\t52\n",
      "4\t55\t127.8\t0\t55\n",
      "5\t2\t32.0\t0\t2\n",
      "6\t34\t8.0\t0\t34\n",
      "7\t1\t2.0\t0\t1\n",
      "10\t5\t0.0\t1\t1 4\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "35\t1\t0.0\t2\t0 0 1\n",
      "76\t2\t0.0\t2\t0 1 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "175\t13\t0.0\t2\t0 0 13\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "234\t5\t0.0\t2\t0 0 5\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_470-LP3-ITS-1046_755_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_470-LP3-ITS-1046_755_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_470-LP3-ITS-1046_755_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_470-LP3-ITS-1046_755_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (44.674 ¬µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     171\n",
      "Reads with adapters:                         5 (2.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           171 (100.0%)\n",
      "\n",
      "Total basepairs processed:        51,471 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         51,104 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 40.0%\n",
      "  C: 60.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t2.7\t0\t3\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_471-LP3-ITS-1047_129_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_471-LP3-ITS-1047_129_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_471-LP3-ITS-1047_129_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_471-LP3-ITS-1047_129_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.219 s (7.821 ¬µs/read; 7.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,963\n",
      "Reads with adapters:                       158 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,963 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,416,863 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,413,580 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 158 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.5%\n",
      "  C: 53.8%\n",
      "  G: 0.6%\n",
      "  T: 12.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t436.9\t0\t91\n",
      "5\t2\t27.3\t0\t2\n",
      "6\t18\t6.8\t0\t18\n",
      "9\t5\t0.1\t0\t4 1\n",
      "10\t28\t0.0\t1\t1 27\n",
      "175\t12\t0.0\t2\t0 1 11\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_472-LP3-ITS-1048_1028_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_472-LP3-ITS-1048_1028_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_472-LP3-ITS-1048_1028_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_472-LP3-ITS-1048_1028_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.181 s (8.122 ¬µs/read; 7.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,265\n",
      "Reads with adapters:                       157 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,265 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,701,765 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,699,297 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 157 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.0%\n",
      "  C: 59.2%\n",
      "  G: 1.9%\n",
      "  T: 17.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t108\t347.9\t0\t108\n",
      "4\t1\t87.0\t0\t1\n",
      "5\t5\t21.7\t0\t5\n",
      "6\t15\t5.4\t0\t15\n",
      "7\t1\t1.4\t0\t1\n",
      "9\t3\t0.1\t0\t3\n",
      "10\t15\t0.0\t1\t1 14\n",
      "11\t1\t0.0\t1\t0 1\n",
      "175\t4\t0.0\t2\t0 0 4\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "298\t2\t0.0\t2\t0 0 2\n",
      "300\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_473-LP3-ITS-1049_474_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_473-LP3-ITS-1049_474_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_473-LP3-ITS-1049_474_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_473-LP3-ITS-1049_474_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.116 s (8.080 ¬µs/read; 7.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,312\n",
      "Reads with adapters:                       126 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,312 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,307,912 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,306,320 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 126 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.8%\n",
      "  C: 58.7%\n",
      "  G: 3.2%\n",
      "  T: 18.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t223.6\t0\t77\n",
      "6\t26\t3.5\t0\t26\n",
      "7\t1\t0.9\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t15\t0.0\t1\t1 14\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_474-LP3-ITS-1050_949_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_474-LP3-ITS-1050_949_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_474-LP3-ITS-1050_949_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_474-LP3-ITS-1050_949_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.177 s (8.631 ¬µs/read; 6.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,541\n",
      "Reads with adapters:                       149 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,541 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,182,841 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,175,497 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 149 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.6%\n",
      "  C: 43.6%\n",
      "  G: 0.0%\n",
      "  T: 12.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t76\t321.0\t0\t76\n",
      "4\t11\t80.2\t0\t11\n",
      "6\t13\t5.0\t0\t13\n",
      "9\t3\t0.1\t0\t3\n",
      "10\t11\t0.0\t1\t1 10\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "174\t1\t0.0\t2\t0 0 1\n",
      "175\t18\t0.0\t2\t0 0 18\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t2\t0.0\t2\t0 1 1\n",
      "234\t11\t0.0\t2\t0 0 11\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_475-LP3-ITS-1051_987_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_475-LP3-ITS-1051_987_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_475-LP3-ITS-1051_987_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_475-LP3-ITS-1051_987_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.344 s (7.637 ¬µs/read; 7.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  45,082\n",
      "Reads with adapters:                       231 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        45,082 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,569,682 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,557,672 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 231 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 48.5%\n",
      "  C: 35.9%\n",
      "  G: 1.7%\n",
      "  T: 13.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t704.4\t0\t77\n",
      "4\t23\t176.1\t0\t23\n",
      "5\t1\t44.0\t0\t1\n",
      "6\t46\t11.0\t0\t46\n",
      "7\t4\t2.8\t0\t4\n",
      "9\t3\t0.2\t0\t3\n",
      "10\t17\t0.0\t1\t3 14\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "175\t35\t0.0\t2\t0 0 35\n",
      "212\t2\t0.0\t2\t0 0 2\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_476-LP3-ITS-1052_521_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_476-LP3-ITS-1052_521_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_476-LP3-ITS-1052_521_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_476-LP3-ITS-1052_521_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.225 s (8.257 ¬µs/read; 7.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,192\n",
      "Reads with adapters:                       133 (0.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,192 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,184,792 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,175,991 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 133 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 57.9%\n",
      "  C: 30.8%\n",
      "  G: 0.8%\n",
      "  T: 10.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t424.9\t0\t45\n",
      "4\t26\t106.2\t0\t26\n",
      "5\t2\t26.6\t0\t2\n",
      "6\t13\t6.6\t0\t13\n",
      "10\t3\t0.0\t1\t0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "173\t1\t0.0\t2\t0 0 1\n",
      "175\t27\t0.0\t2\t0 0 27\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "233\t3\t0.0\t2\t0 0 3\n",
      "234\t7\t0.0\t2\t0 0 7\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_477-LP3-ITS-1053_234_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_477-LP3-ITS-1053_234_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_477-LP3-ITS-1053_234_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_477-LP3-ITS-1053_234_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.231 s (8.388 ¬µs/read; 7.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,575\n",
      "Reads with adapters:                       265 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,575 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,300,075 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,287,967 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 265 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.8%\n",
      "  C: 39.2%\n",
      "  G: 0.4%\n",
      "  T: 16.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t128\t430.9\t0\t128\n",
      "4\t33\t107.7\t0\t33\n",
      "6\t27\t6.7\t0\t27\n",
      "7\t1\t1.7\t0\t1\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t3\t0.1\t0\t3\n",
      "10\t12\t0.0\t1\t0 12\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "175\t36\t0.0\t2\t0 0 36\n",
      "212\t3\t0.0\t2\t0 0 3\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t10\t0.0\t2\t0 0 10\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_478-LP3-ITS-1054_627_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_478-LP3-ITS-1054_627_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_478-LP3-ITS-1054_627_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_478-LP3-ITS-1054_627_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.226 s (8.278 ¬µs/read; 7.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,329\n",
      "Reads with adapters:                       169 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,226,029 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,214,035 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 169 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 58.0%\n",
      "  C: 33.1%\n",
      "  G: 1.8%\n",
      "  T: 7.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t427.0\t0\t65\n",
      "4\t19\t106.8\t0\t19\n",
      "6\t11\t6.7\t0\t11\n",
      "7\t1\t1.7\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t9\t0.0\t1\t0 9\n",
      "11\t1\t0.0\t1\t1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "76\t3\t0.0\t2\t0 0 3\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "175\t30\t0.0\t2\t0 1 29\n",
      "212\t2\t0.0\t2\t0 0 2\n",
      "216\t2\t0.0\t2\t0 0 2\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "233\t6\t0.0\t2\t0 0 6\n",
      "234\t10\t0.0\t2\t0 1 9\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_479-LP3-ITS-1055_120_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_479-LP3-ITS-1055_120_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_479-LP3-ITS-1055_120_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_479-LP3-ITS-1055_120_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.283 s (9.035 ¬µs/read; 6.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,270\n",
      "Reads with adapters:                       388 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,270 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,412,270 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,398,218 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 388 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.4%\n",
      "  C: 45.1%\n",
      "  G: 1.8%\n",
      "  T: 15.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t224\t488.6\t0\t224\n",
      "5\t1\t30.5\t0\t1\n",
      "6\t24\t7.6\t0\t24\n",
      "7\t4\t1.9\t0\t4\n",
      "9\t6\t0.1\t0\t5 1\n",
      "10\t57\t0.0\t1\t7 50\n",
      "25\t1\t0.0\t2\t0 0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "33\t1\t0.0\t2\t0 0 1\n",
      "34\t1\t0.0\t2\t0 0 1\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "146\t3\t0.0\t2\t0 0 3\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "175\t33\t0.0\t2\t0 1 32\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "233\t5\t0.0\t2\t0 0 5\n",
      "234\t10\t0.0\t2\t0 0 10\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_480-LP3-ITS-1056_186_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_480-LP3-ITS-1056_186_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_480-LP3-ITS-1056_186_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_480-LP3-ITS-1056_186_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.398 s (7.691 ¬µs/read; 7.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  51,762\n",
      "Reads with adapters:                       765 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        51,762 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,580,362 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,569,055 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 765 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.8%\n",
      "  C: 54.2%\n",
      "  G: 0.9%\n",
      "  T: 20.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t561\t808.8\t0\t561\n",
      "4\t5\t202.2\t0\t5\n",
      "5\t1\t50.5\t0\t1\n",
      "6\t28\t12.6\t0\t28\n",
      "7\t1\t3.2\t0\t1\n",
      "9\t16\t0.2\t0\t16\n",
      "10\t100\t0.0\t1\t3 97\n",
      "11\t1\t0.0\t1\t1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "30\t3\t0.0\t2\t0 0 3\n",
      "31\t2\t0.0\t2\t0 0 2\n",
      "33\t3\t0.0\t2\t0 0 3\n",
      "35\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "175\t20\t0.0\t2\t0 0 20\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "232\t4\t0.0\t2\t0 0 4\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t7\t0.0\t2\t0 0 7\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_481-LP3-ITS-1057_954_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_481-LP3-ITS-1057_954_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_481-LP3-ITS-1057_954_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_481-LP3-ITS-1057_954_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.196 s (10.796 ¬µs/read; 5.56 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  18,154\n",
      "Reads with adapters:                       247 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        18,154 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,464,354 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,438,019 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 247 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.3%\n",
      "  C: 72.5%\n",
      "  G: 0.8%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t283.7\t0\t4\n",
      "57\t58\t0.0\t2\t0 1 57\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "81\t7\t0.0\t2\t0 1 6\n",
      "84\t16\t0.0\t2\t0 0 16\n",
      "87\t17\t0.0\t2\t0 2 15\n",
      "96\t17\t0.0\t2\t0 0 17\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t25\t0.0\t2\t0 0 25\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "133\t26\t0.0\t2\t0 0 26\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t59\t0.0\t2\t0 1 58\n",
      "156\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_482-LP3-ITS-1058_360_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_482-LP3-ITS-1058_360_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_482-LP3-ITS-1058_360_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_482-LP3-ITS-1058_360_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.240 s (11.459 ¬µs/read; 5.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,961\n",
      "Reads with adapters:                       352 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,961 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,309,261 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,286,741 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 352 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.7%\n",
      "  C: 63.4%\n",
      "  G: 1.7%\n",
      "  T: 14.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t161\t327.5\t0\t161\n",
      "4\t1\t81.9\t0\t1\n",
      "10\t18\t0.0\t1\t0 18\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t2\t0.0\t2\t0 0 2\n",
      "84\t9\t0.0\t2\t0 1 8\n",
      "86\t3\t0.0\t2\t0 0 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t29\t0.0\t2\t0 1 28\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t99\t0.0\t2\t0 2 97\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "158\t4\t0.0\t2\t0 0 4\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_483-LP3-ITS-1059_118_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_483-LP3-ITS-1059_118_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_483-LP3-ITS-1059_118_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_483-LP3-ITS-1059_118_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.010 s (65.381 ¬µs/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     150\n",
      "Reads with adapters:                         4 (2.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           150 (100.0%)\n",
      "\n",
      "Total basepairs processed:        45,150 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         44,844 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.0%\n",
      "  C: 75.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t2.3\t0\t2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_484-LP3-ITS-1060_559_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_484-LP3-ITS-1060_559_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_484-LP3-ITS-1060_559_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_484-LP3-ITS-1060_559_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.139 s (11.035 ¬µs/read; 5.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,574\n",
      "Reads with adapters:                       220 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,574 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,784,774 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,766,578 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 220 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 51.4%\n",
      "  C: 44.5%\n",
      "  G: 1.8%\n",
      "  T: 2.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t196.5\t0\t5\n",
      "5\t5\t12.3\t0\t5\n",
      "20\t69\t0.0\t2\t0 3 66\n",
      "80\t9\t0.0\t2\t0 1 8\n",
      "86\t19\t0.0\t2\t0 0 19\n",
      "118\t63\t0.0\t2\t0 0 63\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "124\t4\t0.0\t2\t0 0 4\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_485-LP3-ITS-1061_0_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_485-LP3-ITS-1061_0_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_485-LP3-ITS-1061_0_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_485-LP3-ITS-1061_0_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.233 s (11.541 ¬µs/read; 5.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,146\n",
      "Reads with adapters:                       328 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,063,946 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,024,418 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 328 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 15.2%\n",
      "  C: 71.6%\n",
      "  G: 0.9%\n",
      "  T: 12.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t314.8\t0\t4\n",
      "4\t1\t78.7\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "76\t2\t0.0\t2\t0 0 2\n",
      "77\t3\t0.0\t2\t0 0 3\n",
      "78\t16\t0.0\t2\t0 1 15\n",
      "79\t8\t0.0\t2\t0 1 7\n",
      "80\t3\t0.0\t2\t0 2 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "82\t3\t0.0\t2\t0 0 3\n",
      "83\t12\t0.0\t2\t0 1 11\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "92\t3\t0.0\t2\t0 0 3\n",
      "93\t83\t0.0\t2\t0 0 83\n",
      "94\t3\t0.0\t2\t0 0 3\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "143\t10\t0.0\t2\t0 0 10\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "155\t133\t0.0\t2\t0 3 130\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_486-LP3-ITS-1062_677_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_486-LP3-ITS-1062_677_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_486-LP3-ITS-1062_677_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_486-LP3-ITS-1062_677_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.235 s (11.322 ¬µs/read; 5.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,790\n",
      "Reads with adapters:                       224 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,790 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,257,790 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,241,402 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 224 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 26.3%\n",
      "  C: 71.0%\n",
      "  G: 0.9%\n",
      "  T: 1.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t324.8\t0\t9\n",
      "8\t35\t0.3\t0\t35\n",
      "12\t61\t0.0\t1\t1 60\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "91\t1\t0.0\t2\t0 0 1\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "118\t40\t0.0\t2\t0 1 39\n",
      "133\t45\t0.0\t2\t0 0 45\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_487-LP3-ITS-1063_206_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_487-LP3-ITS-1063_206_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_487-LP3-ITS-1063_206_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_487-LP3-ITS-1063_206_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.237 s (11.495 ¬µs/read; 5.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,574\n",
      "Reads with adapters:                     2,135 (10.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,574 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,192,774 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,167,226 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2135 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 19.1%\n",
      "  C: 58.3%\n",
      "  G: 1.2%\n",
      "  T: 21.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1663\t321.5\t0\t1663\n",
      "4\t2\t80.4\t0\t2\n",
      "6\t2\t5.0\t0\t2\n",
      "8\t26\t0.3\t0\t26\n",
      "9\t14\t0.1\t0\t13 1\n",
      "10\t294\t0.0\t1\t5 289\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "80\t5\t0.0\t2\t0 0 5\n",
      "81\t3\t0.0\t2\t0 0 3\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "83\t2\t0.0\t2\t0 0 2\n",
      "86\t9\t0.0\t2\t0 0 9\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "118\t23\t0.0\t2\t0 1 22\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "133\t22\t0.0\t2\t0 0 22\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "155\t46\t0.0\t2\t0 0 46\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_488-LP3-ITS-1064_853_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_488-LP3-ITS-1064_853_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_488-LP3-ITS-1064_853_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_488-LP3-ITS-1064_853_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.445 s (10.511 ¬µs/read; 5.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,378\n",
      "Reads with adapters:                       255 (0.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,378 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,755,778 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,722,653 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 255 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.0%\n",
      "  C: 74.1%\n",
      "  G: 0.0%\n",
      "  T: 7.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t165.5\t0\t1\n",
      "5\t1\t41.4\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "83\t6\t0.0\t2\t0 2 4\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "96\t5\t0.0\t2\t0 0 5\n",
      "99\t14\t0.0\t2\t0 0 14\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "108\t7\t0.0\t2\t0 0 7\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "117\t7\t0.0\t2\t0 0 7\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "128\t11\t0.0\t2\t0 0 11\n",
      "130\t5\t0.0\t2\t0 0 5\n",
      "133\t88\t0.0\t2\t0 2 86\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t6\t0.0\t2\t0 1 5\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t73\t0.0\t2\t0 1 72\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_489-LP3-ITS-1065_291_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_489-LP3-ITS-1065_291_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_489-LP3-ITS-1065_291_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_489-LP3-ITS-1065_291_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.154 s (10.730 ¬µs/read; 5.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,309\n",
      "Reads with adapters:                       180 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,307,009 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,285,303 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 180 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 16.1%\n",
      "  C: 78.9%\n",
      "  G: 2.2%\n",
      "  T: 2.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t223.6\t0\t1\n",
      "6\t2\t3.5\t0\t2\n",
      "9\t2\t0.1\t0\t2\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t3\t0.0\t2\t0 0 3\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t74\t0.0\t2\t0 0 74\n",
      "96\t6\t0.0\t2\t0 0 6\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "143\t12\t0.0\t2\t0 0 12\n",
      "155\t62\t0.0\t2\t0 1 61\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "169\t1\t0.0\t2\t0 0 1\n",
      "175\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_490-LP3-ITS-1066_939_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_490-LP3-ITS-1066_939_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_490-LP3-ITS-1066_939_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_490-LP3-ITS-1066_939_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.232 s (11.044 ¬µs/read; 5.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,015\n",
      "Reads with adapters:                       218 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,015 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,325,515 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,304,104 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 218 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.2%\n",
      "  C: 50.0%\n",
      "  G: 0.5%\n",
      "  T: 12.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t328.4\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t35\t0.0\t2\t0 3 32\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "57\t22\t0.0\t2\t0 1 21\n",
      "79\t5\t0.0\t2\t0 0 5\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t7\t0.0\t2\t0 0 7\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "96\t9\t0.0\t2\t0 0 9\n",
      "101\t6\t0.0\t2\t0 0 6\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "118\t31\t0.0\t2\t0 1 30\n",
      "124\t12\t0.0\t2\t0 0 12\n",
      "133\t36\t0.0\t2\t0 1 35\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "155\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_491-LP3-ITS-1067_198_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_491-LP3-ITS-1067_198_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_491-LP3-ITS-1067_198_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_491-LP3-ITS-1067_198_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.265 s (11.397 ¬µs/read; 5.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,257\n",
      "Reads with adapters:                       388 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,000,357 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,959,879 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 388 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 9.5%\n",
      "  C: 87.1%\n",
      "  G: 0.8%\n",
      "  T: 2.6%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'C' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t363.4\t0\t6\n",
      "4\t1\t90.8\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "73\t91\t0.0\t2\t0 0 91\n",
      "83\t8\t0.0\t2\t0 0 8\n",
      "87\t100\t0.0\t2\t0 0 100\n",
      "88\t7\t0.0\t2\t0 0 7\n",
      "93\t14\t0.0\t2\t0 0 14\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t12\t0.0\t2\t0 1 11\n",
      "133\t70\t0.0\t2\t0 0 70\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t13\t0.0\t2\t0 0 13\n",
      "155\t55\t0.0\t2\t0 0 55\n",
      "166\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_492-LP3-ITS-1068_673_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_492-LP3-ITS-1068_673_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_492-LP3-ITS-1068_673_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_492-LP3-ITS-1068_673_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.243 s (11.750 ¬µs/read; 5.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,646\n",
      "Reads with adapters:                       300 (1.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,646 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,214,446 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,174,792 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 300 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 77.7%\n",
      "  C: 16.3%\n",
      "  G: 0.3%\n",
      "  T: 5.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "44\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "79\t10\t0.0\t2\t0 0 10\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t2\t0.0\t2\t0 1 1\n",
      "84\t12\t0.0\t2\t0 0 12\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "113\t1\t0.0\t2\t0 0 1\n",
      "118\t28\t0.0\t2\t0 0 28\n",
      "119\t7\t0.0\t2\t0 0 7\n",
      "121\t4\t0.0\t2\t0 0 4\n",
      "122\t5\t0.0\t2\t0 0 5\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t8\t0.0\t2\t0 0 8\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t11\t0.0\t2\t0 0 11\n",
      "145\t144\t0.0\t2\t1 1 142\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_493-LP3-ITS-1069_882_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_493-LP3-ITS-1069_882_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_493-LP3-ITS-1069_882_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_493-LP3-ITS-1069_882_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.390 s (10.357 ¬µs/read; 5.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  37,688\n",
      "Reads with adapters:                       470 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        37,688 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,344,088 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,286,848 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 470 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.5%\n",
      "  C: 54.5%\n",
      "  G: 0.4%\n",
      "  T: 9.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t1\t0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "66\t1\t0.0\t2\t0 0 1\n",
      "67\t7\t0.0\t2\t0 0 7\n",
      "80\t13\t0.0\t2\t0 0 13\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t11\t0.0\t2\t0 1 10\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "87\t10\t0.0\t2\t0 0 10\n",
      "96\t23\t0.0\t2\t0 1 22\n",
      "99\t10\t0.0\t2\t0 0 10\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "103\t35\t0.0\t2\t0 0 35\n",
      "106\t18\t0.0\t2\t0 0 18\n",
      "108\t38\t0.0\t2\t0 0 38\n",
      "111\t13\t0.0\t2\t0 0 13\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "122\t10\t0.0\t2\t0 0 10\n",
      "123\t17\t0.0\t2\t0 1 16\n",
      "124\t23\t0.0\t2\t0 1 22\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "133\t97\t0.0\t2\t0 3 94\n",
      "135\t3\t0.0\t2\t0 0 3\n",
      "136\t1\t0.0\t2\t0 0 1\n",
      "141\t3\t0.0\t2\t0 0 3\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t33\t0.0\t2\t0 1 32\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t14\t0.0\t2\t0 0 14\n",
      "155\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_494-LP3-ITS-1070_22_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_494-LP3-ITS-1070_22_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_494-LP3-ITS-1070_22_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_494-LP3-ITS-1070_22_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.301 s (10.697 ¬µs/read; 5.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,184\n",
      "Reads with adapters:                       764 (2.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,184 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,483,384 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,451,902 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 764 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 13.1%\n",
      "  C: 77.0%\n",
      "  G: 0.3%\n",
      "  T: 9.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t260\t440.4\t0\t260\n",
      "4\t226\t110.1\t0\t226\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t3\t0.1\t0\t2 1\n",
      "10\t52\t0.0\t1\t5 47\n",
      "11\t2\t0.0\t1\t0 2\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "94\t5\t0.0\t2\t0 0 5\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t5\t0.0\t2\t0 0 5\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "123\t24\t0.0\t2\t0 0 24\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t103\t0.0\t2\t0 1 102\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t14\t0.0\t2\t0 0 14\n",
      "155\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_495-LP3-ITS-1071_977_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_495-LP3-ITS-1071_977_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_495-LP3-ITS-1071_977_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_495-LP3-ITS-1071_977_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.121 s (10.535 ¬µs/read; 5.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  11,454\n",
      "Reads with adapters:                       119 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        11,454 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,447,654 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,435,291 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 119 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.1%\n",
      "  C: 44.5%\n",
      "  G: 0.0%\n",
      "  T: 3.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t179.0\t0\t6\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "55\t18\t0.0\t2\t0 0 18\n",
      "77\t2\t0.0\t2\t0 0 2\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "87\t8\t0.0\t2\t0 0 8\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "118\t37\t0.0\t2\t0 1 36\n",
      "133\t10\t0.0\t2\t0 0 10\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t2\t0.0\t2\t0 0 2\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "155\t13\t0.0\t2\t0 1 12\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_496-LP3-ITS-1072_649_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_496-LP3-ITS-1072_649_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_496-LP3-ITS-1072_649_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_496-LP3-ITS-1072_649_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.253 s (11.443 ¬µs/read; 5.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,087\n",
      "Reads with adapters:                       148 (0.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,087 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,648,187 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,629,339 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 148 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.6%\n",
      "  C: 73.0%\n",
      "  G: 0.0%\n",
      "  T: 3.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t345.1\t0\t2\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "80\t2\t0.0\t2\t0 0 2\n",
      "83\t5\t0.0\t2\t0 0 5\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "93\t22\t0.0\t2\t0 0 22\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "118\t17\t0.0\t2\t0 0 17\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t3\t0.0\t2\t0 1 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t50\t0.0\t2\t0 0 50\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_497-LP3-ITS-1073_946_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_497-LP3-ITS-1073_946_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_497-LP3-ITS-1073_946_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_497-LP3-ITS-1073_946_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.264 s (10.735 ¬µs/read; 5.59 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,639\n",
      "Reads with adapters:                       337 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,639 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,416,339 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,375,362 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 337 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.2%\n",
      "  C: 70.0%\n",
      "  G: 0.6%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t385.0\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "73\t30\t0.0\t2\t0 1 29\n",
      "75\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "91\t8\t0.0\t2\t0 0 8\n",
      "93\t23\t0.0\t2\t0 0 23\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "104\t14\t0.0\t2\t1 0 13\n",
      "109\t17\t0.0\t2\t0 0 17\n",
      "118\t31\t0.0\t2\t0 1 30\n",
      "124\t10\t0.0\t2\t0 0 10\n",
      "133\t108\t0.0\t2\t0 0 108\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t16\t0.0\t2\t0 0 16\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t52\t0.0\t2\t1 0 51\n",
      "157\t3\t0.0\t2\t0 0 3\n",
      "163\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_498-LP3-ITS-1074_49_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_498-LP3-ITS-1074_49_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_498-LP3-ITS-1074_49_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_498-LP3-ITS-1074_49_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.223 s (10.916 ¬µs/read; 5.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,432\n",
      "Reads with adapters:                       431 (2.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,432 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,150,032 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,117,146 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 431 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 67.1%\n",
      "  C: 26.0%\n",
      "  G: 0.2%\n",
      "  T: 6.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t319.2\t0\t7\n",
      "5\t1\t20.0\t0\t1\n",
      "9\t2\t0.1\t0\t2\n",
      "20\t50\t0.0\t2\t0 1 49\n",
      "51\t174\t0.0\t2\t0 1 173\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "80\t3\t0.0\t2\t0 0 3\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "83\t3\t0.0\t2\t0 1 2\n",
      "87\t21\t0.0\t2\t0 0 21\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "97\t8\t0.0\t2\t0 0 8\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "108\t12\t0.0\t2\t0 0 12\n",
      "112\t3\t0.0\t2\t0 1 2\n",
      "117\t14\t0.0\t2\t0 0 14\n",
      "118\t65\t0.0\t2\t0 0 65\n",
      "123\t14\t0.0\t2\t0 0 14\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t8\t0.0\t2\t0 0 8\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "155\t15\t0.0\t2\t0 0 15\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "162\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_499-LP3-ITS-1075_28_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_499-LP3-ITS-1075_28_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_499-LP3-ITS-1075_28_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_499-LP3-ITS-1075_28_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.335 s (8.465 ¬µs/read; 7.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,561\n",
      "Reads with adapters:                     3,735 (9.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,907,861 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,891,762 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3735 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 91.9%\n",
      "  C: 7.5%\n",
      "  G: 0.2%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3689\t618.1\t0\t3689\n",
      "4\t12\t154.5\t0\t12\n",
      "5\t2\t38.6\t0\t2\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "176\t23\t0.0\t2\t0 1 22\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_500-LP3-ITS-1076_1113_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_500-LP3-ITS-1076_1113_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_500-LP3-ITS-1076_1113_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_500-LP3-ITS-1076_1113_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.250 s (11.401 ¬µs/read; 5.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,937\n",
      "Reads with adapters:                       347 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,937 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,603,037 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,561,922 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 347 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.4%\n",
      "  C: 79.3%\n",
      "  G: 1.2%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "27\t9\t0.0\t2\t0 2 7\n",
      "53\t6\t0.0\t2\t0 0 6\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "56\t16\t0.0\t2\t0 3 13\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "79\t13\t0.0\t2\t0 0 13\n",
      "82\t2\t0.0\t2\t0 0 2\n",
      "83\t3\t0.0\t2\t0 1 2\n",
      "84\t8\t0.0\t2\t0 0 8\n",
      "87\t29\t0.0\t2\t0 1 28\n",
      "89\t5\t0.0\t2\t0 0 5\n",
      "93\t22\t0.0\t2\t0 0 22\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t4\t0.0\t2\t0 0 4\n",
      "101\t3\t0.0\t2\t0 0 3\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "110\t3\t0.0\t2\t0 0 3\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "114\t3\t0.0\t2\t0 0 3\n",
      "118\t15\t0.0\t2\t0 0 15\n",
      "124\t5\t0.0\t2\t0 0 5\n",
      "127\t6\t0.0\t2\t0 0 6\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "133\t58\t0.0\t2\t0 0 58\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "140\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t97\t0.0\t2\t0 3 94\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_501-LP3-ITS-1077_1048_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_501-LP3-ITS-1077_1048_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_501-LP3-ITS-1077_1048_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_501-LP3-ITS-1077_1048_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.412 s (10.592 ¬µs/read; 5.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  38,928\n",
      "Reads with adapters:                       384 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        38,928 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,717,328 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,667,616 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 384 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.7%\n",
      "  C: 69.3%\n",
      "  G: 0.3%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t608.2\t0\t5\n",
      "4\t1\t152.1\t0\t1\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "60\t1\t0.0\t2\t0 1\n",
      "78\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "96\t17\t0.0\t2\t0 0 17\n",
      "104\t4\t0.0\t2\t0 0 4\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "111\t3\t0.0\t2\t0 0 3\n",
      "118\t53\t0.0\t2\t0 0 53\n",
      "123\t12\t0.0\t2\t0 1 11\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t199\t0.0\t2\t0 2 197\n",
      "136\t4\t0.0\t2\t0 0 4\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t61\t0.0\t2\t0 1 60\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "158\t3\t0.0\t2\t0 0 3\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_502-LP3-ITS-1078_568_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_502-LP3-ITS-1078_568_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_502-LP3-ITS-1078_568_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_502-LP3-ITS-1078_568_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.088 s (12.611 ¬µs/read; 4.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,939\n",
      "Reads with adapters:                       347 (5.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         6,939 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,088,639 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,082,347 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 347 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 13.5%\n",
      "  C: 68.6%\n",
      "  G: 0.9%\n",
      "  T: 17.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t242\t108.4\t0\t242\n",
      "6\t1\t1.7\t0\t1\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t41\t0.0\t1\t5 36\n",
      "27\t21\t0.0\t2\t0 0 21\n",
      "61\t3\t0.0\t2\t0 0 3\n",
      "93\t17\t0.0\t2\t0 0 17\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t18\t0.0\t2\t0 0 18\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_503-LP3-ITS-1079_1066_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_503-LP3-ITS-1079_1066_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_503-LP3-ITS-1079_1066_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_503-LP3-ITS-1079_1066_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.348 s (9.668 ¬µs/read; 6.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,972\n",
      "Reads with adapters:                       447 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,972 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,827,572 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,788,529 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 447 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.6%\n",
      "  C: 66.0%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t562.1\t0\t3\n",
      "4\t2\t140.5\t0\t2\n",
      "7\t1\t2.2\t0\t1\n",
      "51\t218\t0.0\t2\t0 2 216\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t4\t0.0\t2\t0 1 3\n",
      "85\t3\t0.0\t2\t0 0 3\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "117\t3\t0.0\t2\t0 0 3\n",
      "118\t106\t0.0\t2\t0 3 103\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t4\t0.0\t2\t0 0 4\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "133\t29\t0.0\t2\t0 1 28\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "155\t27\t0.0\t2\t0 1 26\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_504-LP3-ITS-1080_1076_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_504-LP3-ITS-1080_1076_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_504-LP3-ITS-1080_1076_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_504-LP3-ITS-1080_1076_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.580 s (10.551 ¬µs/read; 5.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  54,959\n",
      "Reads with adapters:                       574 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        54,959 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,542,659 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,480,937 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 574 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 18.8%\n",
      "  C: 77.7%\n",
      "  G: 0.7%\n",
      "  T: 2.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t858.7\t0\t3\n",
      "4\t2\t214.7\t0\t2\n",
      "10\t1\t0.1\t1\t0 1\n",
      "73\t3\t0.0\t2\t0 0 3\n",
      "81\t108\t0.0\t2\t0 1 107\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t20\t0.0\t2\t0 3 17\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t86\t0.0\t2\t0 3 83\n",
      "87\t118\t0.0\t2\t0 0 118\n",
      "88\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "102\t5\t0.0\t2\t0 0 5\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t75\t0.0\t2\t0 1 74\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t23\t0.0\t2\t0 0 23\n",
      "151\t2\t0.0\t2\t0 1 1\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t87\t0.0\t2\t0 1 86\n",
      "156\t22\t0.0\t2\t0 0 22\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_505-LP3-ITS-1081_835_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_505-LP3-ITS-1081_835_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_505-LP3-ITS-1081_835_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_505-LP3-ITS-1081_835_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.161 s (12.249 ¬µs/read; 4.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,175\n",
      "Reads with adapters:                       410 (3.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,175 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,965,675 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,946,008 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 410 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.2%\n",
      "  C: 62.0%\n",
      "  G: 1.2%\n",
      "  T: 14.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t216\t205.9\t0\t216\n",
      "9\t12\t0.1\t0\t10 2\n",
      "10\t48\t0.0\t1\t3 45\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "93\t7\t0.0\t2\t0 0 7\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t3\t0.0\t2\t0 0 3\n",
      "109\t4\t0.0\t2\t0 0 4\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "123\t8\t0.0\t2\t0 0 8\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "133\t17\t0.0\t2\t0 0 17\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t11\t0.0\t2\t0 0 11\n",
      "155\t65\t0.0\t2\t0 1 64\n",
      "156\t1\t0.0\t2\t0 0 1\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_506-LP3-ITS-1082_850_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_506-LP3-ITS-1082_850_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_506-LP3-ITS-1082_850_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_506-LP3-ITS-1082_850_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.244 s (11.073 ¬µs/read; 5.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,031\n",
      "Reads with adapters:                       480 (2.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,031 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,631,331 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,606,762 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 480 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.8%\n",
      "  C: 52.3%\n",
      "  G: 0.8%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t75\t344.2\t0\t75\n",
      "4\t1\t86.1\t0\t1\n",
      "5\t188\t21.5\t0\t188\n",
      "6\t2\t5.4\t0\t2\n",
      "51\t18\t0.0\t2\t0 0 18\n",
      "60\t12\t0.0\t2\t0 0 12\n",
      "66\t2\t0.0\t2\t0 0 2\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t6\t0.0\t2\t0 0 6\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t133\t0.0\t2\t0 1 132\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "120\t4\t0.0\t2\t0 0 4\n",
      "121\t8\t0.0\t2\t0 0 8\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_507-LP3-ITS-1083_77_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_507-LP3-ITS-1083_77_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_507-LP3-ITS-1083_77_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_507-LP3-ITS-1083_77_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.495 s (9.002 ¬µs/read; 6.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  54,989\n",
      "Reads with adapters:                     4,310 (7.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        54,989 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,551,689 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,511,917 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4310 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.7%\n",
      "  C: 8.7%\n",
      "  G: 0.1%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4007\t859.2\t0\t4007\n",
      "4\t5\t214.8\t0\t5\n",
      "5\t2\t53.7\t0\t2\n",
      "10\t4\t0.1\t1\t0 4\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "58\t22\t0.0\t2\t0 1 21\n",
      "60\t61\t0.0\t2\t0 3 58\n",
      "80\t26\t0.0\t2\t0 0 26\n",
      "82\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t74\t0.0\t2\t0 0 74\n",
      "85\t1\t0.0\t2\t0 1\n",
      "116\t7\t0.0\t2\t0 0 7\n",
      "118\t24\t0.0\t2\t0 1 23\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t25\t0.0\t2\t0 0 25\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t12\t0.0\t2\t0 0 12\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "155\t19\t0.0\t2\t0 0 19\n",
      "176\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_508-LP3-ITS-1084_1004_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_508-LP3-ITS-1084_1004_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_508-LP3-ITS-1084_1004_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_508-LP3-ITS-1084_1004_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.115 s (14.552 ¬µs/read; 4.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   7,905\n",
      "Reads with adapters:                        94 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         7,905 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,379,405 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,366,940 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 94 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.0%\n",
      "  C: 67.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t15\t0.0\t2\t0 0 15\n",
      "124\t3\t0.0\t2\t0 0 3\n",
      "127\t3\t0.0\t2\t0 0 3\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t39\t0.0\t2\t0 0 39\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 1 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_509-LP3-ITS-1085_1016_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_509-LP3-ITS-1085_1016_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_509-LP3-ITS-1085_1016_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_509-LP3-ITS-1085_1016_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.251 s (10.257 ¬µs/read; 5.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,433\n",
      "Reads with adapters:                       289 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,433 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,354,333 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,318,355 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 289 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.1%\n",
      "  C: 76.1%\n",
      "  G: 0.0%\n",
      "  T: 1.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t381.8\t0\t3\n",
      "10\t1\t0.0\t1\t0 1\n",
      "61\t12\t0.0\t2\t0 0 12\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "83\t3\t0.0\t2\t0 0 3\n",
      "92\t4\t0.0\t2\t0 0 4\n",
      "93\t19\t0.0\t2\t0 0 19\n",
      "96\t11\t0.0\t2\t0 0 11\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t28\t0.0\t2\t0 0 28\n",
      "123\t27\t0.0\t2\t0 0 27\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "128\t5\t0.0\t2\t0 0 5\n",
      "133\t107\t0.0\t2\t0 0 107\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t11\t0.0\t2\t0 0 11\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t31\t0.0\t2\t0 1 30\n",
      "162\t11\t0.0\t2\t0 0 11\n",
      "163\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_510-LP3-ITS-1086_208_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_510-LP3-ITS-1086_208_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_510-LP3-ITS-1086_208_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_510-LP3-ITS-1086_208_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.115 s (13.094 ¬µs/read; 4.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   8,776\n",
      "Reads with adapters:                        78 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         8,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,641,576 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,633,179 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 78 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.2%\n",
      "  C: 60.3%\n",
      "  G: 1.3%\n",
      "  T: 1.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t137.1\t0\t7\n",
      "4\t6\t34.3\t0\t6\n",
      "10\t1\t0.0\t1\t0 1\n",
      "76\t1\t0.0\t2\t0 0 1\n",
      "87\t7\t0.0\t2\t0 0 7\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "104\t2\t0.0\t2\t0 0 2\n",
      "111\t1\t0.0\t2\t0 0 1\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "124\t6\t0.0\t2\t0 0 6\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "155\t16\t0.0\t2\t0 0 16\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_511-LP3-ITS-1087_1033_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_511-LP3-ITS-1087_1033_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_511-LP3-ITS-1087_1033_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_511-LP3-ITS-1087_1033_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.464 s (9.785 ¬µs/read; 6.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  47,442\n",
      "Reads with adapters:                       356 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        47,442 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,280,042 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,240,337 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 356 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.1%\n",
      "  C: 70.5%\n",
      "  G: 0.6%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t42\t741.3\t0\t42\n",
      "4\t2\t185.3\t0\t2\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t34\t0.0\t2\t0 0 34\n",
      "92\t4\t0.0\t2\t0 0 4\n",
      "93\t13\t0.0\t2\t0 0 13\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "113\t5\t0.0\t2\t0 0 5\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t18\t0.0\t2\t0 0 18\n",
      "122\t7\t0.0\t2\t0 0 7\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t133\t0.0\t2\t0 1 132\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t10\t0.0\t2\t0 0 10\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "155\t63\t0.0\t2\t0 0 63\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_512-LP3-ITS-1088_1100_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_512-LP3-ITS-1088_1100_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_512-LP3-ITS-1088_1100_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_512-LP3-ITS-1088_1100_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.578 s (10.361 ¬µs/read; 5.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  55,787\n",
      "Reads with adapters:                       502 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        55,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,791,887 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     16,734,712 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 502 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 66.3%\n",
      "  C: 32.3%\n",
      "  G: 0.2%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t871.7\t0\t1\n",
      "4\t1\t217.9\t0\t1\n",
      "67\t2\t0.0\t2\t0 0 2\n",
      "79\t4\t0.0\t2\t0 1 3\n",
      "80\t1\t0.0\t2\t0 0 1\n",
      "81\t2\t0.0\t2\t0 0 2\n",
      "86\t59\t0.0\t2\t0 1 58\n",
      "87\t55\t0.0\t2\t0 0 55\n",
      "89\t11\t0.0\t2\t0 1 10\n",
      "93\t5\t0.0\t2\t0 1 4\n",
      "96\t4\t0.0\t2\t0 0 4\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "101\t7\t0.0\t2\t0 0 7\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t207\t0.0\t2\t0 7 200\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "121\t1\t0.0\t2\t0 0 1\n",
      "123\t20\t0.0\t2\t0 0 20\n",
      "133\t66\t0.0\t2\t0 0 66\n",
      "135\t5\t0.0\t2\t0 0 5\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "155\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_513-LP3-ITS-1089_260_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_513-LP3-ITS-1089_260_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_513-LP3-ITS-1089_260_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_513-LP3-ITS-1089_260_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.366 s (10.429 ¬µs/read; 5.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,113\n",
      "Reads with adapters:                       812 (2.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,569,013 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,513,133 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 812 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.8%\n",
      "  C: 77.8%\n",
      "  G: 0.4%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t548.6\t0\t21\n",
      "4\t314\t137.2\t0\t314\n",
      "10\t1\t0.0\t1\t0 1\n",
      "54\t47\t0.0\t2\t0 1 46\n",
      "56\t55\t0.0\t2\t0 1 54\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "78\t8\t0.0\t2\t0 0 8\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "86\t9\t0.0\t2\t0 0 9\n",
      "87\t11\t0.0\t2\t0 0 11\n",
      "93\t9\t0.0\t2\t0 0 9\n",
      "96\t2\t0.0\t2\t0 0 2\n",
      "99\t11\t0.0\t2\t0 1 10\n",
      "104\t13\t0.0\t2\t0 0 13\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t63\t0.0\t2\t0 1 62\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "122\t4\t0.0\t2\t0 0 4\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t87\t0.0\t2\t0 1 86\n",
      "135\t2\t0.0\t2\t0 0 2\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t4\t0.0\t2\t0 0 4\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "154\t5\t0.0\t2\t0 1 4\n",
      "155\t125\t0.0\t2\t0 1 124\n",
      "159\t5\t0.0\t2\t0 1 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_514-LP3-ITS-1090_446_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_514-LP3-ITS-1090_446_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_514-LP3-ITS-1090_446_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_514-LP3-ITS-1090_446_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.209 s (11.015 ¬µs/read; 5.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,006\n",
      "Reads with adapters:                     2,169 (11.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        19,006 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,720,806 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,696,352 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2169 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.6%\n",
      "  C: 56.2%\n",
      "  G: 1.4%\n",
      "  T: 21.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1696\t297.0\t0\t1696\n",
      "5\t3\t18.6\t0\t3\n",
      "6\t1\t4.6\t0\t1\n",
      "9\t8\t0.1\t0\t8\n",
      "10\t333\t0.0\t1\t20 313\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "87\t2\t0.0\t2\t0 0 2\n",
      "93\t7\t0.0\t2\t0 0 7\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "118\t52\t0.0\t2\t0 0 52\n",
      "133\t28\t0.0\t2\t0 0 28\n",
      "142\t7\t0.0\t2\t0 1 6\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t9\t0.0\t2\t1 1 7\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "198\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_515-LP3-ITS-1091_639_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_515-LP3-ITS-1091_639_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_515-LP3-ITS-1091_639_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_515-LP3-ITS-1091_639_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.959 s (10.442 ¬µs/read; 5.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  91,796\n",
      "Reads with adapters:                    12,000 (13.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        91,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,630,596 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     27,501,815 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 12000 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.8%\n",
      "  C: 60.0%\n",
      "  G: 2.0%\n",
      "  T: 15.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9333\t1434.3\t0\t9333\n",
      "4\t7\t358.6\t0\t7\n",
      "7\t9\t5.6\t0\t9\n",
      "8\t7\t1.4\t0\t7\n",
      "9\t121\t0.4\t0\t106 15\n",
      "10\t1822\t0.1\t1\t82 1740\n",
      "11\t1\t0.0\t1\t0 1\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "32\t233\t0.0\t2\t0 10 223\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "86\t1\t0.0\t2\t0 0 1\n",
      "89\t18\t0.0\t2\t0 0 18\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "118\t88\t0.0\t2\t0 1 87\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t46\t0.0\t2\t0 0 46\n",
      "143\t16\t0.0\t2\t0 0 16\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t6\t0.0\t2\t0 0 6\n",
      "151\t112\t0.0\t2\t0 1 111\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t15\t0.0\t2\t0 0 15\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "184\t2\t0.0\t2\t0 0 2\n",
      "191\t1\t0.0\t2\t0 0 1\n",
      "232\t8\t0.0\t2\t0 0 8\n",
      "234\t83\t0.0\t2\t0 1 82\n",
      "235\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_516-LP3-ITS-1092_1005_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_516-LP3-ITS-1092_1005_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_516-LP3-ITS-1092_1005_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_516-LP3-ITS-1092_1005_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.238 s (11.822 ¬µs/read; 5.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,130\n",
      "Reads with adapters:                       606 (3.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,130 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,059,130 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,026,611 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 606 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.6%\n",
      "  C: 45.4%\n",
      "  G: 1.5%\n",
      "  T: 15.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t285\t314.5\t0\t285\n",
      "4\t1\t78.6\t0\t1\n",
      "8\t1\t0.3\t0\t1\n",
      "9\t13\t0.1\t0\t12 1\n",
      "10\t71\t0.0\t1\t3 68\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t86\t0.0\t2\t0 0 86\n",
      "123\t10\t0.0\t2\t0 0 10\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "133\t63\t0.0\t2\t0 1 62\n",
      "134\t6\t0.0\t2\t0 0 6\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t14\t0.0\t2\t0 0 14\n",
      "151\t10\t0.0\t2\t0 1 9\n",
      "152\t5\t0.0\t2\t0 0 5\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "159\t6\t0.0\t2\t0 0 6\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_517-LP3-ITS-1093_245_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_517-LP3-ITS-1093_245_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_517-LP3-ITS-1093_245_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_517-LP3-ITS-1093_245_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.154 s (11.760 ¬µs/read; 5.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,076\n",
      "Reads with adapters:                     2,336 (17.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,076 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,935,876 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,919,396 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2336 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.6%\n",
      "  C: 59.2%\n",
      "  G: 1.3%\n",
      "  T: 18.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1878\t204.3\t0\t1878\n",
      "4\t2\t51.1\t0\t2\n",
      "8\t2\t0.2\t0\t2\n",
      "9\t47\t0.0\t0\t46 1\n",
      "10\t361\t0.0\t1\t21 340\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "118\t6\t0.0\t2\t0 0 6\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "151\t14\t0.0\t2\t0 0 14\n",
      "175\t5\t0.0\t2\t0 0 5\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "219\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_518-LP3-ITS-1094_561_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_518-LP3-ITS-1094_561_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_518-LP3-ITS-1094_561_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_518-LP3-ITS-1094_561_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.437 s (10.402 ¬µs/read; 5.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  41,979\n",
      "Reads with adapters:                       340 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        41,979 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,635,679 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,593,909 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 340 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.2%\n",
      "  C: 55.9%\n",
      "  G: 0.3%\n",
      "  T: 0.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t655.9\t0\t5\n",
      "4\t7\t164.0\t0\t7\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "63\t1\t0.0\t2\t0 0 1\n",
      "81\t1\t0.0\t2\t0 0 1\n",
      "84\t2\t0.0\t2\t0 0 2\n",
      "91\t2\t0.0\t2\t0 0 2\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "108\t2\t0.0\t2\t0 0 2\n",
      "118\t97\t0.0\t2\t0 1 96\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "123\t28\t0.0\t2\t0 0 28\n",
      "133\t102\t0.0\t2\t0 0 102\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t13\t0.0\t2\t0 0 13\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t8\t0.0\t2\t0 1 7\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "166\t1\t0.0\t2\t0 0 1\n",
      "180\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_519-LP3-ITS-1095_1007_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_519-LP3-ITS-1095_1007_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_519-LP3-ITS-1095_1007_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_519-LP3-ITS-1095_1007_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.548 s (10.311 ¬µs/read; 5.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,187\n",
      "Reads with adapters:                       411 (0.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,187 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,009,287 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,955,361 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 411 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 69.3%\n",
      "  C: 30.2%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4\t831.0\t0\t4\n",
      "5\t1\t51.9\t0\t1\n",
      "7\t1\t3.2\t0\t1\n",
      "10\t1\t0.1\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "61\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "107\t2\t0.0\t2\t0 0 2\n",
      "115\t1\t0.0\t2\t0 0 1\n",
      "118\t131\t0.0\t2\t0 0 131\n",
      "120\t1\t0.0\t2\t0 0 1\n",
      "123\t20\t0.0\t2\t0 0 20\n",
      "131\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t62\t0.0\t2\t0 1 61\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t36\t0.0\t2\t0 0 36\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "145\t13\t0.0\t2\t0 0 13\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "151\t52\t0.0\t2\t0 0 52\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t23\t0.0\t2\t0 1 22\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "160\t22\t0.0\t2\t0 0 22\n",
      "164\t10\t0.0\t2\t0 1 9\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_520-LP3-ITS-1096_1078_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_520-LP3-ITS-1096_1078_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_520-LP3-ITS-1096_1078_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_520-LP3-ITS-1096_1078_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.575 s (10.852 ¬µs/read; 5.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  52,969\n",
      "Reads with adapters:                       620 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        52,969 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,943,669 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,857,415 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 620 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.8%\n",
      "  C: 9.2%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t827.6\t0\t1\n",
      "46\t4\t0.0\t2\t0 0 4\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "52\t2\t0.0\t2\t0 0 2\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "67\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "90\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t121\t0.0\t2\t0 2 119\n",
      "123\t5\t0.0\t2\t0 0 5\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "133\t20\t0.0\t2\t0 0 20\n",
      "134\t22\t0.0\t2\t0 0 22\n",
      "137\t43\t0.0\t2\t0 0 43\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t7\t0.0\t2\t0 0 7\n",
      "145\t12\t0.0\t2\t0 0 12\n",
      "149\t1\t0.0\t2\t0 0 1\n",
      "150\t167\t0.0\t2\t0 1 166\n",
      "151\t155\t0.0\t2\t0 3 152\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t23\t0.0\t2\t0 0 23\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_521-LP3-ITS-1097_348_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_521-LP3-ITS-1097_348_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_521-LP3-ITS-1097_348_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_521-LP3-ITS-1097_348_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.327 s (11.973 ¬µs/read; 5.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,293\n",
      "Reads with adapters:                       476 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,293 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,215,193 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,177,037 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 476 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 45.8%\n",
      "  C: 45.0%\n",
      "  G: 2.5%\n",
      "  T: 6.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t133\t426.5\t0\t133\n",
      "10\t23\t0.0\t1\t0 23\n",
      "47\t20\t0.0\t2\t0 1 19\n",
      "53\t2\t0.0\t2\t0 0 2\n",
      "54\t1\t0.0\t2\t0 0 1\n",
      "93\t14\t0.0\t2\t0 0 14\n",
      "98\t1\t0.0\t2\t0 0 1\n",
      "101\t2\t0.0\t2\t0 0 2\n",
      "102\t1\t0.0\t2\t0 0 1\n",
      "103\t7\t0.0\t2\t0 0 7\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "107\t6\t0.0\t2\t0 0 6\n",
      "118\t178\t0.0\t2\t1 2 175\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "133\t45\t0.0\t2\t0 0 45\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "139\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 0 7\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_522-LP3-ITS-1098_293_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_522-LP3-ITS-1098_293_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_522-LP3-ITS-1098_293_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_522-LP3-ITS-1098_293_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.290 s (11.480 ¬µs/read; 5.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  25,254\n",
      "Reads with adapters:                     1,153 (4.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        25,254 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,601,454 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,548,361 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1153 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 39.8%\n",
      "  C: 43.8%\n",
      "  G: 1.4%\n",
      "  T: 15.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t683\t394.6\t0\t683\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t22\t0.1\t0\t22\n",
      "10\t165\t0.0\t1\t12 153\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t81\t0.0\t2\t0 1 80\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t27\t0.0\t2\t0 2 25\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t17\t0.0\t2\t0 0 17\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t9\t0.0\t2\t0 0 9\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t45\t0.0\t2\t0 2 43\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t77\t0.0\t2\t0 1 76\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_523-LP3-ITS-1099_712_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_523-LP3-ITS-1099_712_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_523-LP3-ITS-1099_712_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_523-LP3-ITS-1099_712_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.484 s (10.934 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  44,235\n",
      "Reads with adapters:                     2,355 (5.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        44,235 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,314,735 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     13,279,721 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2355 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.6%\n",
      "  C: 44.4%\n",
      "  G: 1.9%\n",
      "  T: 12.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1385\t691.2\t0\t1385\n",
      "4\t7\t172.8\t0\t7\n",
      "8\t2\t0.7\t0\t2\n",
      "9\t378\t0.2\t0\t362 16\n",
      "10\t395\t0.0\t1\t32 363\n",
      "11\t1\t0.0\t1\t0 1\n",
      "37\t20\t0.0\t2\t0 1 19\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "118\t42\t0.0\t2\t0 1 41\n",
      "128\t3\t0.0\t2\t0 0 3\n",
      "133\t37\t0.0\t2\t0 1 36\n",
      "134\t3\t0.0\t2\t0 0 3\n",
      "142\t16\t0.0\t2\t0 1 15\n",
      "145\t11\t0.0\t2\t0 0 11\n",
      "146\t2\t0.0\t2\t0 0 2\n",
      "151\t22\t0.0\t2\t0 0 22\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t11\t0.0\t2\t0 0 11\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "164\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_524-LP3-ITS-1100_854_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_524-LP3-ITS-1100_854_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_524-LP3-ITS-1100_854_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_524-LP3-ITS-1100_854_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.202 s (11.667 ¬µs/read; 5.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,347\n",
      "Reads with adapters:                       218 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,347 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,221,447 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,194,395 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 218 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 60.6%\n",
      "  C: 39.0%\n",
      "  G: 0.0%\n",
      "  T: 0.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t2\t67.8\t0\t2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "52\t4\t0.0\t2\t0 0 4\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "118\t104\t0.0\t2\t0 2 102\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t7\t0.0\t2\t0 0 7\n",
      "133\t47\t0.0\t2\t0 1 46\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t8\t0.0\t2\t0 0 8\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t8\t0.0\t2\t0 0 8\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "160\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_525-LP3-ITS-1101_408_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_525-LP3-ITS-1101_408_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_525-LP3-ITS-1101_408_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_525-LP3-ITS-1101_408_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.292 s (11.187 ¬µs/read; 5.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,134\n",
      "Reads with adapters:                       253 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,134 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,866,334 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,835,525 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 253 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 52.2%\n",
      "  C: 46.2%\n",
      "  G: 0.4%\n",
      "  T: 1.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t408.3\t0\t1\n",
      "6\t1\t6.4\t0\t1\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "46\t3\t0.0\t2\t0 0 3\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "107\t1\t0.0\t2\t0 0 1\n",
      "118\t106\t0.0\t2\t0 0 106\n",
      "123\t29\t0.0\t2\t0 0 29\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t63\t0.0\t2\t0 1 62\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t9\t0.0\t2\t0 0 9\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "150\t2\t0.0\t2\t0 0 2\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "152\t3\t0.0\t2\t0 1 2\n",
      "155\t9\t0.0\t2\t0 0 9\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_526-LP3-ITS-1102_464_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_526-LP3-ITS-1102_464_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_526-LP3-ITS-1102_464_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_526-LP3-ITS-1102_464_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.213 s (11.964 ¬µs/read; 5.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,781\n",
      "Reads with adapters:                       282 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,781 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,352,081 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,335,366 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 282 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 85.1%\n",
      "  C: 12.4%\n",
      "  G: 0.0%\n",
      "  T: 2.5%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t34\t277.8\t0\t34\n",
      "4\t1\t69.5\t0\t1\n",
      "8\t89\t0.3\t0\t89\n",
      "9\t24\t0.1\t0\t21 3\n",
      "10\t10\t0.0\t1\t1 9\n",
      "21\t2\t0.0\t2\t0 0 2\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "118\t76\t0.0\t2\t0 2 74\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "133\t3\t0.0\t2\t0 1 2\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "148\t11\t0.0\t2\t0 1 10\n",
      "150\t19\t0.0\t2\t0 0 19\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_527-LP3-ITS-1103_273_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_527-LP3-ITS-1103_273_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_527-LP3-ITS-1103_273_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_527-LP3-ITS-1103_273_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.470 s (10.888 ¬µs/read; 5.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  43,142\n",
      "Reads with adapters:                       379 (0.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        43,142 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,985,742 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,946,389 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 379 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 54.6%\n",
      "  C: 41.2%\n",
      "  G: 0.3%\n",
      "  T: 4.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t674.1\t0\t58\n",
      "8\t2\t0.7\t0\t2\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t8\t0.0\t1\t0 8\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t9\t0.0\t2\t0 0 9\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "78\t2\t0.0\t2\t0 0 2\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t4\t0.0\t2\t0 0 4\n",
      "93\t8\t0.0\t2\t0 0 8\n",
      "95\t2\t0.0\t2\t0 0 2\n",
      "103\t5\t0.0\t2\t0 0 5\n",
      "110\t2\t0.0\t2\t0 0 2\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t118\t0.0\t2\t0 4 114\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t37\t0.0\t2\t0 0 37\n",
      "142\t42\t0.0\t2\t0 0 42\n",
      "143\t15\t0.0\t2\t0 0 15\n",
      "145\t9\t0.0\t2\t0 0 9\n",
      "146\t1\t0.0\t2\t0 0 1\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "153\t3\t0.0\t2\t0 1 2\n",
      "155\t24\t0.0\t2\t0 0 24\n",
      "159\t8\t0.0\t2\t0 0 8\n",
      "184\t1\t0.0\t2\t0 0 1\n",
      "194\t5\t0.0\t2\t0 1 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_528-LP3-ITS-1104_1001_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_528-LP3-ITS-1104_1001_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_528-LP3-ITS-1104_1001_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_528-LP3-ITS-1104_1001_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.700 s (10.115 ¬µs/read; 5.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  69,204\n",
      "Reads with adapters:                       818 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        69,204 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,830,404 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     20,768,811 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 818 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.1%\n",
      "  C: 42.1%\n",
      "  G: 0.5%\n",
      "  T: 7.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t298\t1081.3\t0\t298\n",
      "8\t1\t1.1\t0\t1\n",
      "9\t3\t0.3\t0\t3\n",
      "10\t44\t0.1\t1\t1 43\n",
      "26\t1\t0.0\t2\t0 0 1\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "93\t4\t0.0\t2\t0 0 4\n",
      "118\t228\t0.0\t2\t0 4 224\n",
      "133\t96\t0.0\t2\t0 0 96\n",
      "134\t18\t0.0\t2\t0 0 18\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "142\t4\t0.0\t2\t0 0 4\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "145\t65\t0.0\t2\t0 1 64\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t20\t0.0\t2\t0 0 20\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t6\t0.0\t2\t0 0 6\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_529-LP3-ITS-1105_897_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_529-LP3-ITS-1105_897_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_529-LP3-ITS-1105_897_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_529-LP3-ITS-1105_897_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.342 s (10.936 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,306\n",
      "Reads with adapters:                       365 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,306 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,423,106 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,376,381 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 365 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 61.6%\n",
      "  C: 38.1%\n",
      "  G: 0.0%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t489.2\t0\t3\n",
      "7\t1\t1.9\t0\t1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "50\t2\t0.0\t2\t0 0 2\n",
      "51\t3\t0.0\t2\t0 0 3\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "79\t3\t0.0\t2\t0 0 3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t5\t0.0\t2\t0 0 5\n",
      "95\t3\t0.0\t2\t0 0 3\n",
      "103\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t147\t0.0\t2\t0 2 145\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "125\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t59\t0.0\t2\t0 0 59\n",
      "142\t22\t0.0\t2\t0 0 22\n",
      "143\t8\t0.0\t2\t0 0 8\n",
      "148\t2\t0.0\t2\t0 1 1\n",
      "151\t40\t0.0\t2\t0 0 40\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "155\t44\t0.0\t2\t0 0 44\n",
      "158\t5\t0.0\t2\t0 0 5\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "183\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_530-LP3-ITS-1106_489_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_530-LP3-ITS-1106_489_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_530-LP3-ITS-1106_489_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_530-LP3-ITS-1106_489_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.316 s (11.278 ¬µs/read; 5.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,982\n",
      "Reads with adapters:                       267 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,982 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,422,582 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,390,612 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 267 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.0%\n",
      "  C: 23.2%\n",
      "  G: 0.4%\n",
      "  T: 0.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t437.2\t0\t2\n",
      "5\t1\t27.3\t0\t1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "52\t1\t0.0\t2\t0 0 1\n",
      "57\t2\t0.0\t2\t0 0 2\n",
      "79\t2\t0.0\t2\t0 0 2\n",
      "85\t4\t0.0\t2\t0 0 4\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t180\t0.0\t2\t0 3 177\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t25\t0.0\t2\t0 0 25\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t7\t0.0\t2\t0 1 6\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "155\t8\t0.0\t2\t0 0 8\n",
      "159\t4\t0.0\t2\t0 0 4\n",
      "160\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_531-LP3-ITS-1107_214_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_531-LP3-ITS-1107_214_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_531-LP3-ITS-1107_214_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_531-LP3-ITS-1107_214_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.533 s (10.931 ¬µs/read; 5.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  48,742\n",
      "Reads with adapters:                     5,549 (11.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        48,742 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,671,342 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,611,726 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5549 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 27.4%\n",
      "  C: 58.2%\n",
      "  G: 2.1%\n",
      "  T: 12.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4083\t761.6\t0\t4083\n",
      "4\t1\t190.4\t0\t1\n",
      "7\t1\t3.0\t0\t1\n",
      "8\t124\t0.7\t0\t124\n",
      "9\t159\t0.2\t0\t149 10\n",
      "10\t902\t0.0\t1\t58 844\n",
      "11\t1\t0.0\t1\t0 1\n",
      "12\t1\t0.0\t1\t0 1\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "51\t5\t0.0\t2\t0 0 5\n",
      "73\t1\t0.0\t2\t0 0 1\n",
      "84\t3\t0.0\t2\t0 0 3\n",
      "96\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 0 1\n",
      "118\t111\t0.0\t2\t0 2 109\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "134\t22\t0.0\t2\t0 1 21\n",
      "137\t22\t0.0\t2\t0 0 22\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t69\t0.0\t2\t0 0 69\n",
      "151\t19\t0.0\t2\t0 0 19\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_532-LP3-ITS-1108_473_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_532-LP3-ITS-1108_473_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_532-LP3-ITS-1108_473_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_532-LP3-ITS-1108_473_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.178 s (13.472 ¬µs/read; 4.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  13,237\n",
      "Reads with adapters:                     1,558 (11.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        13,237 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,984,337 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,966,092 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1558 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.5%\n",
      "  C: 45.8%\n",
      "  G: 1.9%\n",
      "  T: 20.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1111\t206.8\t0\t1111\n",
      "8\t2\t0.2\t0\t2\n",
      "9\t59\t0.1\t0\t59\n",
      "10\t302\t0.0\t1\t27 275\n",
      "11\t3\t0.0\t1\t0 3\n",
      "79\t1\t0.0\t2\t0 0 1\n",
      "99\t1\t0.0\t2\t0 0 1\n",
      "118\t37\t0.0\t2\t0 0 37\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "133\t5\t0.0\t2\t0 0 5\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "157\t1\t0.0\t2\t0 0 1\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "175\t10\t0.0\t2\t0 1 9\n",
      "216\t3\t0.0\t2\t0 0 3\n",
      "219\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_533-LP3-ITS-1109_309_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_533-LP3-ITS-1109_309_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_533-LP3-ITS-1109_309_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_533-LP3-ITS-1109_309_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.358 s (11.059 ¬µs/read; 5.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,414\n",
      "Reads with adapters:                       382 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,414 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,756,614 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,705,798 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 382 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 80.4%\n",
      "  C: 18.8%\n",
      "  G: 0.0%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t506.5\t0\t3\n",
      "6\t2\t7.9\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "45\t1\t0.0\t2\t0 0 1\n",
      "47\t2\t0.0\t2\t0 0 2\n",
      "51\t2\t0.0\t2\t0 0 2\n",
      "53\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "105\t1\t0.0\t2\t0 0 1\n",
      "116\t1\t0.0\t2\t0 0 1\n",
      "118\t85\t0.0\t2\t0 1 84\n",
      "128\t2\t0.0\t2\t0 0 2\n",
      "133\t32\t0.0\t2\t0 0 32\n",
      "134\t26\t0.0\t2\t0 0 26\n",
      "137\t7\t0.0\t2\t0 0 7\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t160\t0.0\t2\t0 1 159\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "153\t18\t0.0\t2\t0 0 18\n",
      "155\t21\t0.0\t2\t0 0 21\n",
      "157\t2\t0.0\t2\t0 0 2\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_534-LP3-ITS-1110_543_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_534-LP3-ITS-1110_543_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_534-LP3-ITS-1110_543_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_534-LP3-ITS-1110_543_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.102 s (15.119 ¬µs/read; 3.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,728\n",
      "Reads with adapters:                       381 (5.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         6,728 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,025,128 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      2,018,141 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 381 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 90.0%\n",
      "  C: 9.4%\n",
      "  G: 0.3%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t337\t105.1\t0\t337\n",
      "4\t1\t26.3\t0\t1\n",
      "103\t4\t0.0\t2\t0 0 4\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "134\t4\t0.0\t2\t0 0 4\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_535-LP3-ITS-1111_1149_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_535-LP3-ITS-1111_1149_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_535-LP3-ITS-1111_1149_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_535-LP3-ITS-1111_1149_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 9.624 s (10.071 ¬µs/read; 5.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                 955,678\n",
      "Reads with adapters:                     9,703 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):       955,678 (100.0%)\n",
      "\n",
      "Total basepairs processed:   287,659,078 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):    286,349,566 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9703 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.9%\n",
      "  C: 23.7%\n",
      "  G: 0.1%\n",
      "  T: 0.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t14932.5\t0\t12\n",
      "4\t1\t3733.1\t0\t1\n",
      "5\t1\t933.3\t0\t1\n",
      "6\t144\t233.3\t0\t144\n",
      "7\t1\t58.3\t0\t1\n",
      "8\t1\t14.6\t0\t1\n",
      "9\t1\t3.6\t0\t1\n",
      "10\t1\t0.9\t1\t0 1\n",
      "19\t1\t0.0\t1\t0 0 1\n",
      "20\t4\t0.0\t2\t0 0 4\n",
      "32\t11\t0.0\t2\t0 1 10\n",
      "46\t17\t0.0\t2\t0 0 17\n",
      "47\t12\t0.0\t2\t0 0 12\n",
      "49\t4\t0.0\t2\t0 0 4\n",
      "50\t5\t0.0\t2\t0 0 5\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "53\t6\t0.0\t2\t0 0 6\n",
      "55\t2\t0.0\t2\t0 0 2\n",
      "57\t4\t0.0\t2\t0 0 4\n",
      "58\t15\t0.0\t2\t0 0 15\n",
      "59\t23\t0.0\t2\t0 1 22\n",
      "60\t113\t0.0\t2\t0 4 109\n",
      "61\t2\t0.0\t2\t0 0 2\n",
      "65\t15\t0.0\t2\t0 0 15\n",
      "69\t4\t0.0\t2\t0 0 4\n",
      "84\t23\t0.0\t2\t0 0 23\n",
      "86\t2\t0.0\t2\t0 0 2\n",
      "87\t21\t0.0\t2\t0 0 21\n",
      "92\t1\t0.0\t2\t0 0 1\n",
      "93\t20\t0.0\t2\t0 0 20\n",
      "95\t4\t0.0\t2\t0 0 4\n",
      "101\t1\t0.0\t2\t0 0 1\n",
      "103\t2\t0.0\t2\t0 0 2\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "110\t12\t0.0\t2\t0 0 12\n",
      "115\t3\t0.0\t2\t0 0 3\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t1122\t0.0\t2\t0 18 1104\n",
      "119\t4\t0.0\t2\t0 0 4\n",
      "123\t11\t0.0\t2\t0 0 11\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "128\t49\t0.0\t2\t0 0 49\n",
      "132\t35\t0.0\t2\t0 1 34\n",
      "133\t1158\t0.0\t2\t0 13 1145\n",
      "134\t644\t0.0\t2\t0 11 633\n",
      "135\t1\t0.0\t2\t0 1\n",
      "136\t2\t0.0\t2\t0 0 2\n",
      "137\t585\t0.0\t2\t0 10 575\n",
      "138\t2\t0.0\t2\t0 0 2\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t139\t0.0\t2\t0 1 138\n",
      "143\t73\t0.0\t2\t0 0 73\n",
      "144\t8\t0.0\t2\t0 0 8\n",
      "145\t4255\t0.0\t2\t1 64 4190\n",
      "146\t20\t0.0\t2\t0 0 20\n",
      "147\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t178\t0.0\t2\t0 2 176\n",
      "152\t4\t0.0\t2\t0 0 4\n",
      "153\t236\t0.0\t2\t0 3 233\n",
      "154\t7\t0.0\t2\t0 0 7\n",
      "155\t629\t0.0\t2\t0 5 624\n",
      "156\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "163\t2\t0.0\t2\t0 0 2\n",
      "164\t27\t0.0\t2\t0 0 27\n",
      "180\t2\t0.0\t2\t0 0 2\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_536-LP3-ITS-1112_646_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_536-LP3-ITS-1112_646_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_536-LP3-ITS-1112_646_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_536-LP3-ITS-1112_646_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.673 s (11.226 ¬µs/read; 5.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  59,993\n",
      "Reads with adapters:                       650 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        59,993 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,057,893 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,978,639 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 650 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.1%\n",
      "  C: 23.7%\n",
      "  G: 0.5%\n",
      "  T: 0.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t937.4\t0\t3\n",
      "5\t51\t58.6\t0\t51\n",
      "6\t1\t14.6\t0\t1\n",
      "11\t2\t0.0\t1\t0 2\n",
      "32\t4\t0.0\t2\t0 0 4\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "58\t7\t0.0\t2\t0 0 7\n",
      "59\t7\t0.0\t2\t0 0 7\n",
      "60\t11\t0.0\t2\t0 1 10\n",
      "63\t5\t0.0\t2\t0 0 5\n",
      "87\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t98\t0.0\t2\t0 3 95\n",
      "122\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "126\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "133\t47\t0.0\t2\t0 0 47\n",
      "134\t39\t0.0\t2\t0 0 39\n",
      "137\t24\t0.0\t2\t0 0 24\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t290\t0.0\t2\t0 6 284\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "152\t2\t0.0\t2\t0 0 2\n",
      "153\t19\t0.0\t2\t0 1 18\n",
      "155\t18\t0.0\t2\t0 0 18\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_537-LP3-ITS-1113_490_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_537-LP3-ITS-1113_490_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_537-LP3-ITS-1113_490_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_537-LP3-ITS-1113_490_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.173 s (11.035 ¬µs/read; 5.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,642\n",
      "Reads with adapters:                       222 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,708,242 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,679,398 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 222 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.8%\n",
      "  C: 7.2%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t244.4\t0\t3\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t4\t0.0\t1\t0 4\n",
      "46\t2\t0.0\t2\t0 0 2\n",
      "60\t3\t0.0\t2\t0 0 3\n",
      "118\t65\t0.0\t2\t0 0 65\n",
      "133\t7\t0.0\t2\t0 0 7\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t2\t0.0\t2\t0 0 2\n",
      "145\t125\t0.0\t2\t0 2 123\n",
      "151\t6\t0.0\t2\t0 0 6\n",
      "153\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_538-LP3-ITS-1114_136_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_538-LP3-ITS-1114_136_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_538-LP3-ITS-1114_136_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_538-LP3-ITS-1114_136_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.095 s (14.594 ¬µs/read; 4.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,512\n",
      "Reads with adapters:                        74 (1.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):         6,512 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,960,112 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      1,950,130 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 74 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.7%\n",
      "  C: 18.9%\n",
      "  G: 0.0%\n",
      "  T: 1.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t101.8\t0\t3\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "118\t5\t0.0\t2\t0 1 4\n",
      "133\t11\t0.0\t2\t0 0 11\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t50\t0.0\t2\t0 1 49\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_539-LP3-ITS-1115_1064_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_539-LP3-ITS-1115_1064_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_539-LP3-ITS-1115_1064_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_539-LP3-ITS-1115_1064_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.364 s (10.123 ¬µs/read; 5.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,942\n",
      "Reads with adapters:                       488 (1.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,942 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,818,542 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,753,867 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 488 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 88.1%\n",
      "  C: 10.9%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t561.6\t0\t2\n",
      "10\t2\t0.0\t1\t0 2\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "85\t1\t0.0\t2\t0 0 1\n",
      "95\t5\t0.0\t2\t0 0 5\n",
      "118\t168\t0.0\t2\t0 0 168\n",
      "119\t2\t0.0\t2\t0 0 2\n",
      "133\t37\t0.0\t2\t0 1 36\n",
      "135\t7\t0.0\t2\t0 0 7\n",
      "137\t12\t0.0\t2\t0 0 12\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t240\t0.0\t2\t0 2 238\n",
      "151\t7\t0.0\t2\t0 0 7\n",
      "153\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_540-LP3-ITS-1116_780_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_540-LP3-ITS-1116_780_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_540-LP3-ITS-1116_780_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_540-LP3-ITS-1116_780_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.211 s (12.183 ¬µs/read; 4.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,358\n",
      "Reads with adapters:                       291 (1.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,358 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,224,758 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,184,687 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 291 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 78.0%\n",
      "  C: 22.0%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t271.2\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "50\t1\t0.0\t2\t0 0 1\n",
      "51\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "60\t2\t0.0\t2\t0 0 2\n",
      "69\t1\t0.0\t2\t0 0 1\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "118\t19\t0.0\t2\t0 0 19\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t39\t0.0\t2\t0 1 38\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "145\t196\t0.0\t2\t0 3 193\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t14\t0.0\t2\t0 0 14\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_541-LP3-ITS-1117_975_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_541-LP3-ITS-1117_975_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_541-LP3-ITS-1117_975_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_541-LP3-ITS-1117_975_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.353 s (10.722 ¬µs/read; 5.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,884\n",
      "Reads with adapters:                       511 (1.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,884 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,898,084 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,830,061 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 511 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 79.8%\n",
      "  C: 19.2%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t10\t513.8\t0\t10\n",
      "7\t1\t2.0\t0\t1\n",
      "10\t3\t0.0\t1\t0 3\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "51\t7\t0.0\t2\t0 0 7\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "58\t2\t0.0\t2\t0 0 2\n",
      "60\t1\t0.0\t2\t0 0 1\n",
      "83\t3\t0.0\t2\t0 0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "110\t1\t0.0\t2\t0 1\n",
      "118\t24\t0.0\t2\t0 1 23\n",
      "124\t1\t0.0\t2\t0 0 1\n",
      "128\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t61\t0.0\t2\t0 0 61\n",
      "134\t76\t0.0\t2\t0 1 75\n",
      "137\t69\t0.0\t2\t0 2 67\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "142\t18\t0.0\t2\t0 0 18\n",
      "143\t17\t0.0\t2\t0 0 17\n",
      "145\t189\t0.0\t2\t0 3 186\n",
      "146\t4\t0.0\t2\t0 0 4\n",
      "151\t5\t0.0\t2\t0 0 5\n",
      "155\t11\t0.0\t2\t0 0 11\n",
      "164\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_542-LP3-ITS-1118_338_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_542-LP3-ITS-1118_338_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_542-LP3-ITS-1118_338_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_542-LP3-ITS-1118_338_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.190 s (10.746 ¬µs/read; 5.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  17,717\n",
      "Reads with adapters:                       210 (1.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        17,717 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,332,817 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,309,433 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 210 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 76.7%\n",
      "  C: 19.5%\n",
      "  G: 0.5%\n",
      "  T: 3.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t276.8\t0\t32\n",
      "10\t5\t0.0\t1\t0 5\n",
      "46\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "118\t57\t0.0\t2\t0 1 56\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t12\t0.0\t2\t0 0 12\n",
      "137\t6\t0.0\t2\t0 0 6\n",
      "141\t1\t0.0\t2\t0 0 1\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t13\t0.0\t2\t0 0 13\n",
      "145\t70\t0.0\t2\t0 0 70\n",
      "151\t2\t0.0\t2\t0 0 2\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "177\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_543-LP3-ITS-1119_1107_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_543-LP3-ITS-1119_1107_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_543-LP3-ITS-1119_1107_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_543-LP3-ITS-1119_1107_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.021 s (32.713 ¬µs/read; 1.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     648\n",
      "Reads with adapters:                        21 (3.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           648 (100.0%)\n",
      "\n",
      "Total basepairs processed:       195,048 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        193,730 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 21 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 47.6%\n",
      "  C: 52.4%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t10.1\t0\t3\n",
      "10\t2\t0.0\t1\t0 2\n",
      "36\t7\t0.0\t2\t0 0 7\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "99\t2\t0.0\t2\t0 0 2\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_544-LP3-ITS-1120_867_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_544-LP3-ITS-1120_867_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_544-LP3-ITS-1120_867_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_544-LP3-ITS-1120_867_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (28.875 ¬µs/read; 2.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     278\n",
      "Reads with adapters:                         8 (2.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           278 (100.0%)\n",
      "\n",
      "Total basepairs processed:        83,678 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         83,387 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 12.5%\n",
      "  C: 87.5%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t4.3\t0\t5\n",
      "10\t1\t0.0\t1\t0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_545-LP3-ITS-1121_316_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_545-LP3-ITS-1121_316_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_545-LP3-ITS-1121_316_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_545-LP3-ITS-1121_316_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.006 s (58.925 ¬µs/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     106\n",
      "Reads with adapters:                         4 (3.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           106 (100.0%)\n",
      "\n",
      "Total basepairs processed:        31,906 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         31,657 bp (99.2%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 75.0%\n",
      "  C: 0.0%\n",
      "  G: 0.0%\n",
      "  T: 25.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t1.7\t0\t1\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_546-LP3-ITS-1122_10_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_546-LP3-ITS-1122_10_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_546-LP3-ITS-1122_10_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_546-LP3-ITS-1122_10_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (44.690 ¬µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     149\n",
      "Reads with adapters:                        11 (7.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           149 (100.0%)\n",
      "\n",
      "Total basepairs processed:        44,849 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         44,572 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 11 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 36.4%\n",
      "  C: 54.5%\n",
      "  G: 0.0%\n",
      "  T: 9.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t2.3\t0\t7\n",
      "10\t2\t0.0\t1\t1 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_547-LP3-ITS-1123_684_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_547-LP3-ITS-1123_684_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_547-LP3-ITS-1123_684_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_547-LP3-ITS-1123_684_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (58.873 ¬µs/read; 1.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     118\n",
      "Reads with adapters:                         9 (7.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           118 (100.0%)\n",
      "\n",
      "Total basepairs processed:        35,518 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         35,022 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 44.4%\n",
      "  C: 44.4%\n",
      "  G: 0.0%\n",
      "  T: 11.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t1.8\t0\t5\n",
      "9\t1\t0.0\t0\t1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "145\t1\t0.0\t2\t0 0 1\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_548-LP3-ITS-1124_54_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_548-LP3-ITS-1124_54_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_548-LP3-ITS-1124_54_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_548-LP3-ITS-1124_54_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.007 s (67.755 ¬µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      98\n",
      "Reads with adapters:                         3 (3.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):            98 (100.0%)\n",
      "\n",
      "Total basepairs processed:        29,498 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         29,374 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t1.5\t0\t2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_549-LP3-ITS-1125_313_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_549-LP3-ITS-1125_313_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_549-LP3-ITS-1125_313_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_549-LP3-ITS-1125_313_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.222 s (8.940 ¬µs/read; 6.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  24,824\n",
      "Reads with adapters:                     1,792 (7.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        24,824 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,472,024 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      7,462,369 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1792 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 22.0%\n",
      "  C: 53.5%\n",
      "  G: 1.4%\n",
      "  T: 23.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1397\t387.9\t0\t1397\n",
      "4\t1\t97.0\t0\t1\n",
      "5\t2\t24.2\t0\t2\n",
      "6\t21\t6.1\t0\t21\n",
      "7\t4\t1.5\t0\t4\n",
      "8\t1\t0.4\t0\t1\n",
      "9\t45\t0.1\t0\t40 5\n",
      "10\t307\t0.0\t1\t19 288\n",
      "11\t3\t0.0\t1\t0 3\n",
      "118\t4\t0.0\t2\t0 0 4\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "209\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_550-LP3-ITS-1126_942_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_550-LP3-ITS-1126_942_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_550-LP3-ITS-1126_942_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_550-LP3-ITS-1126_942_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.192 s (8.425 ¬µs/read; 7.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  22,834\n",
      "Reads with adapters:                       963 (4.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        22,834 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,873,034 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,865,593 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 963 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.6%\n",
      "  C: 51.4%\n",
      "  G: 1.1%\n",
      "  T: 17.9%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t667\t356.8\t0\t667\n",
      "4\t3\t89.2\t0\t3\n",
      "5\t2\t22.3\t0\t2\n",
      "6\t23\t5.6\t0\t23\n",
      "7\t9\t1.4\t0\t9\n",
      "8\t3\t0.3\t0\t3\n",
      "9\t69\t0.1\t0\t66 3\n",
      "10\t169\t0.0\t1\t8 161\n",
      "11\t3\t0.0\t1\t0 3\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "175\t3\t0.0\t2\t0 0 3\n",
      "206\t9\t0.0\t2\t0 0 9\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_551-LP3-ITS-1127_936_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_551-LP3-ITS-1127_936_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_551-LP3-ITS-1127_936_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_551-LP3-ITS-1127_936_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.257 s (11.054 ¬µs/read; 5.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  23,282\n",
      "Reads with adapters:                     1,338 (5.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        23,282 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,007,882 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,988,829 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1338 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 35.8%\n",
      "  C: 48.4%\n",
      "  G: 1.5%\n",
      "  T: 14.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t839\t363.8\t0\t839\n",
      "4\t6\t90.9\t0\t6\n",
      "5\t1\t22.7\t0\t1\n",
      "6\t13\t5.7\t0\t13\n",
      "7\t129\t1.4\t0\t129\n",
      "8\t39\t0.4\t0\t39\n",
      "9\t39\t0.1\t0\t35 4\n",
      "10\t174\t0.0\t1\t9 165\n",
      "11\t1\t0.0\t1\t1\n",
      "13\t1\t0.0\t1\t0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "108\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t43\t0.0\t2\t0 1 42\n",
      "132\t2\t0.0\t2\t0 0 2\n",
      "133\t14\t0.0\t2\t0 0 14\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "145\t10\t0.0\t2\t0 0 10\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "175\t8\t0.0\t2\t0 0 8\n",
      "216\t2\t0.0\t2\t0 0 2\n",
      "219\t2\t0.0\t2\t0 0 2\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_552-LP3-ITS-1128_183_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_552-LP3-ITS-1128_183_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_552-LP3-ITS-1128_183_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_552-LP3-ITS-1128_183_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.344 s (8.074 ¬µs/read; 7.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  42,661\n",
      "Reads with adapters:                       566 (1.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        42,661 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,840,961 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     12,836,032 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 566 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.5%\n",
      "  C: 66.4%\n",
      "  G: 1.2%\n",
      "  T: 11.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t390\t666.6\t0\t390\n",
      "4\t1\t166.6\t0\t1\n",
      "5\t2\t41.7\t0\t2\n",
      "6\t48\t10.4\t0\t48\n",
      "7\t1\t2.6\t0\t1\n",
      "9\t28\t0.2\t0\t25 3\n",
      "10\t52\t0.0\t1\t0 52\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "34\t30\t0.0\t2\t0 3 27\n",
      "36\t1\t0.0\t2\t0 0 1\n",
      "104\t1\t0.0\t2\t0 0 1\n",
      "118\t8\t0.0\t2\t0 0 8\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_553-LP3-ITS-1129_1123_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_553-LP3-ITS-1129_1123_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_553-LP3-ITS-1129_1123_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_553-LP3-ITS-1129_1123_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.008 s (31.867 ¬µs/read; 1.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     263\n",
      "Reads with adapters:                        12 (4.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           263 (100.0%)\n",
      "\n",
      "Total basepairs processed:        79,163 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         78,976 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 12 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 58.3%\n",
      "  G: 0.0%\n",
      "  T: 8.3%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t4.1\t0\t8\n",
      "10\t3\t0.0\t1\t0 3\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_554-LP3-ITS-1130_587_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_554-LP3-ITS-1130_587_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_554-LP3-ITS-1130_587_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_554-LP3-ITS-1130_587_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.193 s (10.172 ¬µs/read; 5.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  19,022\n",
      "Reads with adapters:                       193 (1.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   1 (0.0%)\n",
      "Reads written (passing filters):        19,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,725,622 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      5,701,810 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 193 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 89.1%\n",
      "  C: 9.3%\n",
      "  G: 0.0%\n",
      "  T: 1.0%\n",
      "  none/other: 0.5%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t297.2\t0\t7\n",
      "4\t1\t74.3\t0\t1\n",
      "5\t1\t18.6\t0\t1\n",
      "73\t16\t0.0\t2\t0 1 15\n",
      "74\t1\t0.0\t2\t0 0 1\n",
      "109\t2\t0.0\t2\t0 0 2\n",
      "116\t2\t0.0\t2\t0 0 2\n",
      "118\t7\t0.0\t2\t0 0 7\n",
      "127\t68\t0.0\t2\t0 1 67\n",
      "128\t41\t0.0\t2\t0 0 41\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "143\t6\t0.0\t2\t0 0 6\n",
      "144\t5\t0.0\t2\t0 0 5\n",
      "145\t3\t0.0\t2\t0 0 3\n",
      "158\t27\t0.0\t2\t0 1 26\n",
      "159\t2\t0.0\t2\t0 0 2\n",
      "219\t1\t0.0\t2\t0 0 1\n",
      "301\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_555-LP3-ITS-1131_742_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_555-LP3-ITS-1131_742_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_555-LP3-ITS-1131_742_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_555-LP3-ITS-1131_742_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.010 s (23.743 ¬µs/read; 2.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     424\n",
      "Reads with adapters:                        16 (3.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           424 (100.0%)\n",
      "\n",
      "Total basepairs processed:       127,624 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        127,441 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 37.5%\n",
      "  C: 50.0%\n",
      "  G: 0.0%\n",
      "  T: 12.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t6.6\t0\t12\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_556-LP3-ITS-1132_778_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_556-LP3-ITS-1132_778_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_556-LP3-ITS-1132_778_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_556-LP3-ITS-1132_778_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.020 s (97.278 ¬µs/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     205\n",
      "Reads with adapters:                         9 (4.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           205 (100.0%)\n",
      "\n",
      "Total basepairs processed:        61,705 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):         60,848 bp (98.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 9 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 33.3%\n",
      "  C: 66.7%\n",
      "  G: 0.0%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t3.2\t0\t2\n",
      "10\t1\t0.0\t1\t0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "124\t2\t0.0\t2\t0 0 2\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "209\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_557-LP3-ITS-1133_686_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_557-LP3-ITS-1133_686_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_557-LP3-ITS-1133_686_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_557-LP3-ITS-1133_686_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.319 s (11.106 ¬µs/read; 5.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  28,728\n",
      "Reads with adapters:                     1,483 (5.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        28,728 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,647,128 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,605,405 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1483 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 31.0%\n",
      "  C: 46.4%\n",
      "  G: 1.8%\n",
      "  T: 20.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t967\t448.9\t0\t967\n",
      "4\t3\t112.2\t0\t3\n",
      "6\t1\t7.0\t0\t1\n",
      "7\t5\t1.8\t0\t5\n",
      "8\t3\t0.4\t0\t3\n",
      "9\t19\t0.1\t0\t19\n",
      "10\t185\t0.0\t1\t16 169\n",
      "11\t3\t0.0\t1\t0 3\n",
      "12\t3\t0.0\t1\t1 2\n",
      "50\t3\t0.0\t2\t0 0 3\n",
      "72\t2\t0.0\t2\t0 0 2\n",
      "84\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t2\t0.0\t2\t0 0 2\n",
      "97\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t185\t0.0\t2\t0 3 182\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "129\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t41\t0.0\t2\t0 0 41\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t3\t0.0\t2\t0 0 3\n",
      "142\t19\t0.0\t2\t0 0 19\n",
      "143\t4\t0.0\t2\t0 0 4\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t5\t0.0\t2\t0 0 5\n",
      "160\t7\t0.0\t2\t0 0 7\n",
      "164\t4\t0.0\t2\t0 0 4\n",
      "180\t1\t0.0\t2\t0 0 1\n",
      "188\t1\t0.0\t2\t0 0 1\n",
      "189\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_558-LP3-ITS-1134_500_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_558-LP3-ITS-1134_500_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_558-LP3-ITS-1134_500_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_558-LP3-ITS-1134_500_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.181 s (14.587 ¬µs/read; 4.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  12,436\n",
      "Reads with adapters:                     1,087 (8.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        12,436 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,743,236 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      3,723,724 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1087 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 32.3%\n",
      "  C: 49.2%\n",
      "  G: 2.4%\n",
      "  T: 16.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t733\t194.3\t0\t733\n",
      "4\t1\t48.6\t0\t1\n",
      "7\t1\t0.8\t0\t1\n",
      "8\t3\t0.2\t0\t3\n",
      "9\t31\t0.0\t0\t30 1\n",
      "10\t215\t0.0\t1\t12 203\n",
      "11\t1\t0.0\t1\t0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t39\t0.0\t2\t0 0 39\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "133\t8\t0.0\t2\t0 0 8\n",
      "134\t1\t0.0\t2\t0 0 1\n",
      "137\t4\t0.0\t2\t0 0 4\n",
      "142\t1\t0.0\t2\t0 0 1\n",
      "145\t3\t0.0\t2\t0 1 2\n",
      "150\t4\t0.0\t2\t0 0 4\n",
      "151\t18\t0.0\t2\t0 1 17\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t7\t0.0\t2\t0 0 7\n",
      "232\t3\t0.0\t2\t0 0 3\n",
      "234\t6\t0.0\t2\t0 0 6\n",
      "235\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_559-LP3-ITS-1135_361_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_559-LP3-ITS-1135_361_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_559-LP3-ITS-1135_361_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_559-LP3-ITS-1135_361_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.398 s (7.499 ¬µs/read; 8.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  53,020\n",
      "Reads with adapters:                     1,274 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        53,020 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,959,020 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     15,951,061 bp (100.0%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1274 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 54.2%\n",
      "  G: 1.5%\n",
      "  T: 15.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t884\t828.4\t0\t884\n",
      "4\t7\t207.1\t0\t7\n",
      "5\t6\t51.8\t0\t6\n",
      "6\t50\t12.9\t0\t50\n",
      "7\t2\t3.2\t0\t2\n",
      "8\t23\t0.8\t0\t23\n",
      "9\t94\t0.2\t0\t88 6\n",
      "10\t194\t0.1\t1\t9 185\n",
      "49\t1\t0.0\t2\t0 0 1\n",
      "58\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 2\n",
      "138\t1\t0.0\t2\t0 0 1\n",
      "145\t5\t0.0\t2\t0 1 4\n",
      "154\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_560-LP3-ITS-1136_1042_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_560-LP3-ITS-1136_1042_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_560-LP3-ITS-1136_1042_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_560-LP3-ITS-1136_1042_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.505 s (10.532 ¬µs/read; 5.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  47,941\n",
      "Reads with adapters:                     5,415 (11.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        47,941 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,430,241 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     14,363,225 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 5415 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 23.2%\n",
      "  C: 61.5%\n",
      "  G: 1.6%\n",
      "  T: 13.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t4245\t749.1\t0\t4245\n",
      "4\t4\t187.3\t0\t4\n",
      "5\t3\t46.8\t0\t3\n",
      "7\t1\t2.9\t0\t1\n",
      "9\t34\t0.2\t0\t32 2\n",
      "10\t718\t0.0\t1\t49 669\n",
      "11\t1\t0.0\t1\t1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "31\t2\t0.0\t2\t0 0 2\n",
      "32\t9\t0.0\t2\t0 0 9\n",
      "33\t13\t0.0\t2\t0 2 11\n",
      "35\t5\t0.0\t2\t0 0 5\n",
      "36\t41\t0.0\t2\t0 1 40\n",
      "37\t13\t0.0\t2\t0 0 13\n",
      "38\t2\t0.0\t2\t0 0 2\n",
      "40\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "59\t1\t0.0\t2\t0 0 1\n",
      "64\t1\t0.0\t2\t0 0 1\n",
      "112\t1\t0.0\t2\t0 0 1\n",
      "114\t1\t0.0\t2\t0 0 1\n",
      "118\t115\t0.0\t2\t0 1 114\n",
      "125\t2\t0.0\t2\t0 0 2\n",
      "132\t12\t0.0\t2\t0 0 12\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t6\t0.0\t2\t0 1 5\n",
      "135\t1\t0.0\t2\t0 0 1\n",
      "137\t17\t0.0\t2\t0 0 17\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t3\t0.0\t2\t0 0 3\n",
      "145\t51\t0.0\t2\t0 0 51\n",
      "147\t2\t0.0\t2\t0 0 2\n",
      "148\t1\t0.0\t2\t0 0 1\n",
      "150\t3\t0.0\t2\t0 1 2\n",
      "151\t32\t0.0\t2\t0 1 31\n",
      "155\t3\t0.0\t2\t0 0 3\n",
      "158\t2\t0.0\t2\t0 0 2\n",
      "159\t5\t0.0\t2\t0 0 5\n",
      "160\t22\t0.0\t2\t0 0 22\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "212\t4\t0.0\t2\t0 0 4\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t1\t0.0\t2\t0 0 1\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_561-LP3-ITS-1137_548_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_561-LP3-ITS-1137_548_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_561-LP3-ITS-1137_548_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_561-LP3-ITS-1137_548_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.439 s (11.254 ¬µs/read; 5.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  39,015\n",
      "Reads with adapters:                     4,017 (10.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        39,015 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,743,515 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     11,673,877 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 4017 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.5%\n",
      "  C: 49.4%\n",
      "  G: 2.1%\n",
      "  T: 19.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2858\t609.6\t0\t2858\n",
      "4\t1\t152.4\t0\t1\n",
      "9\t41\t0.1\t0\t36 5\n",
      "10\t822\t0.0\t1\t52 770\n",
      "89\t1\t0.0\t2\t0 0 1\n",
      "117\t1\t0.0\t2\t0 0 1\n",
      "118\t50\t0.0\t2\t0 0 50\n",
      "119\t1\t0.0\t2\t0 0 1\n",
      "123\t3\t0.0\t2\t0 0 3\n",
      "132\t9\t0.0\t2\t0 0 9\n",
      "133\t27\t0.0\t2\t0 0 27\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t2\t0.0\t2\t0 0 2\n",
      "142\t6\t0.0\t2\t0 0 6\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t27\t0.0\t2\t0 0 27\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t4\t0.0\t2\t0 0 4\n",
      "160\t3\t0.0\t2\t0 0 3\n",
      "164\t1\t0.0\t2\t0 0 1\n",
      "175\t31\t0.0\t2\t0 1 30\n",
      "219\t26\t0.0\t2\t0 0 26\n",
      "231\t1\t0.0\t2\t0 0 1\n",
      "232\t21\t0.0\t2\t0 1 20\n",
      "234\t73\t0.0\t2\t0 0 73\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_562-LP3-ITS-1138_1101_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_562-LP3-ITS-1138_1101_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_562-LP3-ITS-1138_1101_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_562-LP3-ITS-1138_1101_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.210 s (9.801 ¬µs/read; 6.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  21,448\n",
      "Reads with adapters:                     1,474 (6.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        21,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,455,848 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,447,128 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 1474 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 47.8%\n",
      "  G: 0.6%\n",
      "  T: 22.6%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1049\t335.1\t0\t1049\n",
      "4\t1\t83.8\t0\t1\n",
      "5\t4\t20.9\t0\t4\n",
      "6\t30\t5.2\t0\t30\n",
      "7\t3\t1.3\t0\t3\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t47\t0.1\t0\t44 3\n",
      "10\t324\t0.0\t1\t19 305\n",
      "11\t5\t0.0\t1\t0 5\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "151\t1\t0.0\t2\t0 0 1\n",
      "204\t3\t0.0\t2\t0 0 3\n",
      "205\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_563-LP3-ITS-1139_621_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_563-LP3-ITS-1139_621_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_563-LP3-ITS-1139_621_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_563-LP3-ITS-1139_621_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.638 s (10.446 ¬µs/read; 5.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  61,063\n",
      "Reads with adapters:                     2,519 (4.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        61,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,379,963 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     18,330,020 bp (99.7%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2519 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 41.0%\n",
      "  C: 43.8%\n",
      "  G: 1.5%\n",
      "  T: 13.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1475\t954.1\t0\t1475\n",
      "4\t6\t238.5\t0\t6\n",
      "7\t5\t3.7\t0\t5\n",
      "8\t4\t0.9\t0\t4\n",
      "9\t371\t0.2\t0\t344 27\n",
      "10\t381\t0.1\t1\t10 371\n",
      "11\t1\t0.0\t1\t0 1\n",
      "32\t2\t0.0\t2\t0 0 2\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "48\t1\t0.0\t2\t0 0 1\n",
      "62\t1\t0.0\t2\t0 0 1\n",
      "83\t1\t0.0\t2\t0 0 1\n",
      "118\t103\t0.0\t2\t0 5 98\n",
      "127\t1\t0.0\t2\t0 0 1\n",
      "132\t1\t0.0\t2\t0 0 1\n",
      "133\t60\t0.0\t2\t0 0 60\n",
      "134\t19\t0.0\t2\t0 1 18\n",
      "137\t31\t0.0\t2\t0 0 31\n",
      "142\t5\t0.0\t2\t0 0 5\n",
      "143\t2\t0.0\t2\t0 0 2\n",
      "145\t5\t0.0\t2\t0 0 5\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "158\t1\t0.0\t2\t0 0 1\n",
      "160\t8\t0.0\t2\t0 0 8\n",
      "232\t1\t0.0\t2\t0 0 1\n",
      "234\t22\t0.0\t2\t0 0 22\n",
      "235\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_564-LP3-ITS-1140_571_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_564-LP3-ITS-1140_571_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_564-LP3-ITS-1140_571_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_564-LP3-ITS-1140_571_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.171 s (12.130 ¬µs/read; 4.95 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  14,115\n",
      "Reads with adapters:                       340 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        14,115 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,248,615 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,228,005 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 340 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 56.5%\n",
      "  C: 26.2%\n",
      "  G: 1.2%\n",
      "  T: 16.2%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t150\t220.5\t0\t150\n",
      "8\t3\t0.2\t0\t3\n",
      "9\t39\t0.1\t0\t31 8\n",
      "10\t36\t0.0\t1\t1 35\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "118\t34\t0.0\t2\t0 0 34\n",
      "123\t2\t0.0\t2\t0 0 2\n",
      "133\t4\t0.0\t2\t0 0 4\n",
      "142\t3\t0.0\t2\t0 0 3\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "151\t3\t0.0\t2\t0 0 3\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "175\t14\t0.0\t2\t0 0 14\n",
      "216\t12\t0.0\t2\t0 0 12\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t21\t0.0\t2\t0 0 21\n",
      "235\t9\t0.0\t2\t0 0 9\n",
      "236\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_565-LP3-ITS-1141_1151_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_565-LP3-ITS-1141_1151_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_565-LP3-ITS-1141_1151_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_565-LP3-ITS-1141_1151_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.308 s (8.587 ¬µs/read; 6.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  35,917\n",
      "Reads with adapters:                     2,648 (7.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        35,917 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,811,017 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     10,798,473 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2648 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.8%\n",
      "  C: 49.1%\n",
      "  G: 1.6%\n",
      "  T: 27.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2077\t561.2\t0\t2077\n",
      "4\t2\t140.3\t0\t2\n",
      "5\t2\t35.1\t0\t2\n",
      "6\t47\t8.8\t0\t47\n",
      "7\t3\t2.2\t0\t3\n",
      "8\t10\t0.5\t0\t10\n",
      "9\t85\t0.1\t0\t79 6\n",
      "10\t412\t0.0\t1\t20 392\n",
      "11\t4\t0.0\t1\t0 4\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "133\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "209\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_566-LP3-ITS-1142_445_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_566-LP3-ITS-1142_445_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_566-LP3-ITS-1142_445_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_566-LP3-ITS-1142_445_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.179 s (8.709 ¬µs/read; 6.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  20,565\n",
      "Reads with adapters:                     2,020 (9.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        20,565 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,190,065 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      6,179,212 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2020 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 28.2%\n",
      "  C: 50.7%\n",
      "  G: 2.0%\n",
      "  T: 19.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1469\t321.3\t0\t1469\n",
      "4\t1\t80.3\t0\t1\n",
      "5\t1\t20.1\t0\t1\n",
      "6\t16\t5.0\t0\t16\n",
      "7\t1\t1.3\t0\t1\n",
      "8\t2\t0.3\t0\t2\n",
      "9\t29\t0.1\t0\t28 1\n",
      "10\t487\t0.0\t1\t30 457\n",
      "11\t8\t0.0\t1\t1 7\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "206\t1\t0.0\t2\t0 0 1\n",
      "209\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_567-LP3-ITS-1143_225_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_567-LP3-ITS-1143_225_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_567-LP3-ITS-1143_225_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_567-LP3-ITS-1143_225_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.269 s (8.583 ¬µs/read; 6.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  31,294\n",
      "Reads with adapters:                     2,530 (8.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        31,294 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,419,494 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,403,983 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2530 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 92.8%\n",
      "  C: 6.8%\n",
      "  G: 0.2%\n",
      "  T: 0.2%\n",
      "  none/other: 0.0%\n",
      "WARNING:\n",
      "    The adapter is preceded by 'A' extremely often.\n",
      "    The provided adapter sequence could be incomplete at its 5' end.\n",
      "    Ignore this warning when trimming primers.\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2473\t489.0\t0\t2473\n",
      "4\t5\t122.2\t0\t5\n",
      "6\t1\t7.6\t0\t1\n",
      "10\t2\t0.0\t1\t0 2\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "123\t1\t0.0\t2\t0 0 1\n",
      "133\t6\t0.0\t2\t0 1 5\n",
      "145\t6\t0.0\t2\t0 0 6\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "176\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "WARNING:\n",
      "    One or more of your adapter sequences may be incomplete.\n",
      "    Please see the detailed output above.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_568-LP3-ITS-1144_413_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_568-LP3-ITS-1144_413_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_568-LP3-ITS-1144_413_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_568-LP3-ITS-1144_413_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.488 s (8.376 ¬µs/read; 7.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  58,252\n",
      "Reads with adapters:                     6,152 (10.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        58,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,533,852 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     17,508,159 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 6152 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 14.0%\n",
      "  C: 70.9%\n",
      "  G: 1.4%\n",
      "  T: 13.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5344\t910.2\t0\t5344\n",
      "4\t4\t227.5\t0\t4\n",
      "5\t5\t56.9\t0\t5\n",
      "6\t48\t14.2\t0\t48\n",
      "7\t4\t3.6\t0\t4\n",
      "8\t9\t0.9\t0\t9\n",
      "9\t69\t0.2\t0\t69\n",
      "10\t654\t0.1\t1\t42 612\n",
      "11\t3\t0.0\t1\t0 3\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "155\t1\t0.0\t2\t0 0 1\n",
      "209\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_569-LP3-ITS-1145_29_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_569-LP3-ITS-1145_29_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_569-LP3-ITS-1145_29_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_569-LP3-ITS-1145_29_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.379 s (11.613 ¬µs/read; 5.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  32,606\n",
      "Reads with adapters:                     2,272 (7.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        32,606 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,814,406 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,746,750 bp (99.3%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2272 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 42.9%\n",
      "  C: 39.9%\n",
      "  G: 1.1%\n",
      "  T: 16.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1309\t509.5\t0\t1309\n",
      "6\t6\t8.0\t0\t6\n",
      "7\t118\t2.0\t0\t118\n",
      "8\t31\t0.5\t0\t31\n",
      "9\t68\t0.1\t0\t64 4\n",
      "10\t344\t0.0\t1\t22 322\n",
      "11\t2\t0.0\t1\t0 2\n",
      "68\t1\t0.0\t2\t0 0 1\n",
      "77\t1\t0.0\t2\t0 0 1\n",
      "100\t1\t0.0\t2\t0 0 1\n",
      "106\t1\t0.0\t2\t0 0 1\n",
      "118\t97\t0.0\t2\t0 1 96\n",
      "130\t1\t0.0\t2\t0 0 1\n",
      "133\t16\t0.0\t2\t0 0 16\n",
      "134\t9\t0.0\t2\t0 0 9\n",
      "137\t34\t0.0\t2\t0 0 34\n",
      "142\t19\t0.0\t2\t0 0 19\n",
      "143\t9\t0.0\t2\t0 0 9\n",
      "145\t94\t0.0\t2\t0 0 94\n",
      "150\t31\t0.0\t2\t0 1 30\n",
      "151\t4\t0.0\t2\t0 0 4\n",
      "152\t1\t0.0\t2\t0 0 1\n",
      "153\t1\t0.0\t2\t0 0 1\n",
      "155\t12\t0.0\t2\t0 0 12\n",
      "158\t6\t0.0\t2\t0 0 6\n",
      "160\t2\t0.0\t2\t0 0 2\n",
      "175\t3\t0.0\t2\t0 0 3\n",
      "212\t1\t0.0\t2\t0 0 1\n",
      "216\t5\t0.0\t2\t0 0 5\n",
      "232\t12\t0.0\t2\t0 0 12\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "234\t30\t0.0\t2\t0 1 29\n",
      "235\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_570-LP3-ITS-1146_825_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_570-LP3-ITS-1146_825_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_570-LP3-ITS-1146_825_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_570-LP3-ITS-1146_825_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.237 s (8.835 ¬µs/read; 6.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  26,873\n",
      "Reads with adapters:                     2,287 (8.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        26,873 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,088,773 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,078,101 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2287 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 20.2%\n",
      "  C: 53.8%\n",
      "  G: 1.3%\n",
      "  T: 24.7%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1822\t419.9\t0\t1822\n",
      "6\t22\t6.6\t0\t22\n",
      "7\t3\t1.6\t0\t3\n",
      "8\t10\t0.4\t0\t10\n",
      "9\t49\t0.1\t0\t48 1\n",
      "10\t371\t0.0\t1\t26 345\n",
      "11\t5\t0.0\t1\t0 5\n",
      "56\t1\t0.0\t2\t0 0 1\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "209\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_571-LP3-ITS-1147_632_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_571-LP3-ITS-1147_632_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_571-LP3-ITS-1147_632_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_571-LP3-ITS-1147_632_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.779 s (10.610 ¬µs/read; 5.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  73,425\n",
      "Reads with adapters:                    11,903 (16.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        73,425 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,100,925 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):     21,987,857 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 11903 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 21.8%\n",
      "  C: 60.0%\n",
      "  G: 1.8%\n",
      "  T: 16.4%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9347\t1147.3\t0\t9347\n",
      "4\t4\t286.8\t0\t4\n",
      "5\t1\t71.7\t0\t1\n",
      "6\t3\t17.9\t0\t3\n",
      "7\t7\t4.5\t0\t7\n",
      "8\t6\t1.1\t0\t6\n",
      "9\t68\t0.3\t0\t58 10\n",
      "10\t2015\t0.1\t1\t117 1898\n",
      "11\t4\t0.0\t1\t0 4\n",
      "12\t4\t0.0\t1\t0 4\n",
      "38\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "93\t1\t0.0\t2\t0 0 1\n",
      "95\t1\t0.0\t2\t0 0 1\n",
      "96\t3\t0.0\t2\t0 0 3\n",
      "118\t84\t0.0\t2\t0 1 83\n",
      "132\t4\t0.0\t2\t0 0 4\n",
      "133\t57\t0.0\t2\t1 0 56\n",
      "134\t13\t0.0\t2\t0 0 13\n",
      "136\t3\t0.0\t2\t0 1 2\n",
      "137\t20\t0.0\t2\t0 0 20\n",
      "142\t11\t0.0\t2\t0 0 11\n",
      "143\t5\t0.0\t2\t0 0 5\n",
      "144\t1\t0.0\t2\t0 0 1\n",
      "145\t89\t0.0\t2\t0 1 88\n",
      "150\t27\t0.0\t2\t0 0 27\n",
      "151\t77\t0.0\t2\t0 0 77\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "159\t1\t0.0\t2\t0 0 1\n",
      "160\t6\t0.0\t2\t0 0 6\n",
      "175\t5\t0.0\t2\t0 0 5\n",
      "204\t2\t0.0\t2\t0 0 2\n",
      "216\t9\t0.0\t2\t0 0 9\n",
      "219\t3\t0.0\t2\t0 0 3\n",
      "232\t2\t0.0\t2\t0 0 2\n",
      "234\t15\t0.0\t2\t0 0 15\n",
      "235\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_572-LP3-ITS-1148_968_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_572-LP3-ITS-1148_968_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_572-LP3-ITS-1148_968_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_572-LP3-ITS-1148_968_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.165 s (10.468 ¬µs/read; 5.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  15,771\n",
      "Reads with adapters:                       980 (6.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        15,771 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,747,071 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      4,737,503 bp (99.8%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 980 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 29.0%\n",
      "  C: 47.1%\n",
      "  G: 1.7%\n",
      "  T: 22.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t690\t246.4\t0\t690\n",
      "6\t19\t3.9\t0\t19\n",
      "7\t2\t1.0\t0\t2\n",
      "8\t2\t0.2\t0\t2\n",
      "9\t31\t0.1\t0\t31\n",
      "10\t208\t0.0\t1\t19 189\n",
      "11\t2\t0.0\t1\t0 2\n",
      "55\t1\t0.0\t2\t0 0 1\n",
      "118\t3\t0.0\t2\t0 0 3\n",
      "175\t1\t0.0\t2\t0 0 1\n",
      "209\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_573-LP3-ITS-1149_1026_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_573-LP3-ITS-1149_1026_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_573-LP3-ITS-1149_1026_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_573-LP3-ITS-1149_1026_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.292 s (10.559 ¬µs/read; 5.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  27,653\n",
      "Reads with adapters:                     2,878 (10.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        27,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,323,553 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      8,286,400 bp (99.6%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2878 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 25.7%\n",
      "  C: 48.7%\n",
      "  G: 1.4%\n",
      "  T: 24.1%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2149\t432.1\t0\t2149\n",
      "7\t3\t1.7\t0\t3\n",
      "8\t6\t0.4\t0\t6\n",
      "9\t85\t0.1\t0\t78 7\n",
      "10\t465\t0.0\t1\t34 431\n",
      "11\t1\t0.0\t1\t0 1\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "89\t2\t0.0\t2\t0 0 2\n",
      "117\t2\t0.0\t2\t0 0 2\n",
      "118\t61\t0.0\t2\t0 0 61\n",
      "132\t3\t0.0\t2\t0 0 3\n",
      "133\t19\t0.0\t2\t0 0 19\n",
      "134\t2\t0.0\t2\t0 0 2\n",
      "137\t1\t0.0\t2\t0 0 1\n",
      "142\t14\t0.0\t2\t0 0 14\n",
      "143\t1\t0.0\t2\t0 0 1\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "150\t3\t0.0\t2\t0 0 3\n",
      "151\t16\t0.0\t2\t0 0 16\n",
      "155\t2\t0.0\t2\t0 0 2\n",
      "160\t1\t0.0\t2\t0 0 1\n",
      "175\t10\t0.0\t2\t0 0 10\n",
      "216\t1\t0.0\t2\t0 0 1\n",
      "219\t2\t0.0\t2\t0 0 2\n",
      "233\t2\t0.0\t2\t0 0 2\n",
      "234\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_574-LP3-ITS-1150_421_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_574-LP3-ITS-1150_421_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_574-LP3-ITS-1150_421_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_574-LP3-ITS-1150_421_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.259 s (8.537 ¬µs/read; 7.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                  30,308\n",
      "Reads with adapters:                     2,204 (7.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):        30,308 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,122,708 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):      9,109,962 bp (99.9%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 2204 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 24.0%\n",
      "  C: 56.7%\n",
      "  G: 1.5%\n",
      "  T: 17.8%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1688\t473.6\t0\t1688\n",
      "5\t2\t29.6\t0\t2\n",
      "6\t21\t7.4\t0\t21\n",
      "7\t5\t1.8\t0\t5\n",
      "8\t3\t0.5\t0\t3\n",
      "9\t47\t0.1\t0\t45 2\n",
      "10\t420\t0.0\t1\t25 395\n",
      "11\t4\t0.0\t1\t0 4\n",
      "118\t1\t0.0\t2\t0 0 1\n",
      "194\t1\t0.0\t2\t0 0 1\n",
      "209\t12\t0.0\t2\t0 1 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_575-LP3-ITS-1151_64_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_575-LP3-ITS-1151_64_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_575-LP3-ITS-1151_64_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_575-LP3-ITS-1151_64_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.028 s (28.189 ¬µs/read; 2.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     985\n",
      "Reads with adapters:                        22 (2.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           985 (100.0%)\n",
      "\n",
      "Total basepairs processed:       296,485 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        294,878 bp (99.5%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 22 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 50.0%\n",
      "  C: 45.5%\n",
      "  G: 0.0%\n",
      "  T: 4.5%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t15.4\t0\t5\n",
      "7\t1\t0.1\t0\t1\n",
      "9\t1\t0.0\t0\t1\n",
      "10\t3\t0.0\t1\t1 2\n",
      "90\t2\t0.0\t2\t0 0 2\n",
      "109\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t2\t0.0\t2\t0 0 2\n",
      "145\t2\t0.0\t2\t0 0 2\n",
      "155\t3\t0.0\t2\t0 1 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_576-LP3-ITS-1152_311_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_576-LP3-ITS-1152_311_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.10.14\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/processes/16494-1732629237.182453@svens/tmp/q2-OutPath-1_g76so9/364526_576-LP3-ITS-1152_311_L001_R1_001.fastq.gz --adapter AACTTTARYCRAAGGATCTC /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/57423b24-a117-4fa5-8467-c8ece8970919/data/364526_576-LP3-ITS-1152_311_L001_R1_001.fastq.gz\n",
      "Processing single-end reads on 1 core ...\n",
      "Finished in 0.012 s (17.900 ¬µs/read; 3.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     674\n",
      "Reads with adapters:                        16 (2.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads written (passing filters):           674 (100.0%)\n",
      "\n",
      "Total basepairs processed:       202,874 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "Total written (filtered):        201,697 bp (99.4%)\n",
      "\n",
      "=== Adapter 1 ===\n",
      "\n",
      "Sequence: AACTTTARYCRAAGGATCTC; Type: regular 3'; Length: 20; Trimmed: 16 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20 bp: 2\n",
      "\n",
      "Bases preceding removed adapters:\n",
      "  A: 43.8%\n",
      "  C: 50.0%\n",
      "  G: 6.2%\n",
      "  T: 0.0%\n",
      "  none/other: 0.0%\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t10.5\t0\t5\n",
      "10\t2\t0.0\t1\t0 2\n",
      "37\t1\t0.0\t2\t0 0 1\n",
      "87\t1\t0.0\t2\t0 0 1\n",
      "118\t2\t0.0\t2\t0 0 2\n",
      "133\t3\t0.0\t2\t0 0 3\n",
      "150\t1\t0.0\t2\t0 0 1\n",
      "233\t1\t0.0\t2\t0 0 1\n",
      "Saved SampleData[SequencesWithQuality] to: ITS/single-end/ITS-revcomp-trimmed.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t9m20.738s\n",
      "user\t7m25.936s\n",
      "sys\t1m12.539s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime cutadapt trim-single \\\n",
    "    --i-demultiplexed-sequences ITS/single-end/ITS-demux-single-end.qza \\\n",
    "    --p-adapter AACTTTARYCRAAGGATCTC \\\n",
    "    --o-trimmed-sequences ITS/single-end/ITS-revcomp-trimmed.qza \\\n",
    "    --verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: ITS/single-end/ITS-revcomp-trimmed-denoised-table.qza\n",
      "Saved FeatureData[Sequence] to: ITS/single-end/ITS-revcomp-trimmed-denoised-rep-seqs.qza\n",
      "Saved SampleData[DADA2Stats] to: ITS/single-end/ITS-revcomp-trimmed-denoised-stats.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t9m1.237s\n",
      "user\t19m14.519s\n",
      "sys\t0m19.803s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS/single-end/ITS-revcomp-trimmed-denoised-rep-seqs.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m8.093s\n",
      "user\t0m5.835s\n",
      "sys\t0m1.308s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS/single-end/ITS-revcomp-trimmed-denoised-table.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m7.212s\n",
      "user\t0m5.600s\n",
      "sys\t0m1.313s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS/single-end/ITS-revcomp-trimmed-denoised-stats.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m6.313s\n",
      "user\t0m4.933s\n",
      "sys\t0m1.178s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-single \\\n",
    "    --i-demultiplexed-seqs ITS/single-end/ITS-revcomp-trimmed.qza \\\n",
    "    --p-trunc-len 0 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences ITS/single-end/ITS-revcomp-trimmed-denoised-rep-seqs.qza \\\n",
    "    --o-table ITS/single-end/ITS-revcomp-trimmed-denoised-table.qza \\\n",
    "    --o-denoising-stats ITS/single-end/ITS-revcomp-trimmed-denoised-stats.qza\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data ITS/single-end/ITS-revcomp-trimmed-denoised-rep-seqs.qza \\\n",
    "    --o-visualization ITS/single-end/ITS-revcomp-trimmed-denoised-rep-seqs.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table ITS/single-end/ITS-revcomp-trimmed-denoised-table.qza \\\n",
    "    --o-visualization ITS/single-end/ITS-revcomp-trimmed-denoised-table.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file ITS/single-end/ITS-revcomp-trimmed-denoised-stats.qza \\\n",
    "    --o-visualization ITS/single-end/ITS-revcomp-trimmed-denoised-stats.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application(s). This may print messages to stdout and/or stderr.\n",
      "The command(s) being run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: run_dada.R --input_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/qiime2/svens/data/401ed144-77b7-4d67-87f0-37485df4f98a/data --output_path /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpp7nacakv/output.tsv.biom --output_track /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpp7nacakv/track.tsv --filtered_directory /var/folders/qv/m0nf9nf10_3_dt4rl46kwy5r0000gn/T/tmpp7nacakv --truncation_length 190 --trim_left 0 --max_expected_errors 2.0 --truncation_quality_score 2 --max_length Inf --pooling_method independent --chimera_method consensus --min_parental_fold 1.0 --allow_one_off False --num_threads 5 --learn_min_reads 1000000 --homopolymer_gap_penalty NULL --band_size 16\n",
      "\n",
      "R version 4.3.3 (2024-02-29) \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Lade n√∂tiges Paket: Rcpp\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DADA2: 1.30.0 / Rcpp: 1.0.13.1 / RcppParallel: 5.1.9 \n",
      "2) Filtering ................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      "3) Learning Error Rates\n",
      "197883480 total bases in 1041492 reads from 27 samples will be used for learning the error rates.\n",
      "4) Denoise samples \n",
      "................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................\n",
      "5) Remove chimeras (method = consensus)\n",
      "6) Report read numbers through the pipeline\n",
      "7) Write output\n",
      "Saved FeatureTable[Frequency] to: ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-table.qza\n",
      "Saved FeatureData[Sequence] to: ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-rep-seqs.qza\n",
      "Saved SampleData[DADA2Stats] to: ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-stats.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t8m49.882s\n",
      "user\t22m19.458s\n",
      "sys\t0m24.935s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-rep-seqs.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m11.751s\n",
      "user\t0m9.495s\n",
      "sys\t0m1.278s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-table.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m8.102s\n",
      "user\t0m6.512s\n",
      "sys\t0m1.314s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-stats.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m6.275s\n",
      "user\t0m4.897s\n",
      "sys\t0m1.186s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-single \\\n",
    "    --i-demultiplexed-seqs ITS/single-end/revcomp-trimmed/ITS-revcomp-trimmed.qza \\\n",
    "    --p-trunc-len 190 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-rep-seqs.qza \\\n",
    "    --o-table ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-table.qza \\\n",
    "    --o-denoising-stats ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-stats.qza \\\n",
    "    --verbose\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-rep-seqs.qza \\\n",
    "    --o-visualization ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-rep-seqs.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-table.qza \\\n",
    "    --o-visualization ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-table.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-stats.qza \\\n",
    "    --o-visualization ITS/single-end/Final/ITS-revcomp-trimmed-denoised-trunc190-stats.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.5 ITS Paired-End"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: paired-end/paired-dada-table-180.qza\n",
      "Saved FeatureData[Sequence] to: paired-end/paired-dada-rep-seqs-180.qza\n",
      "Saved SampleData[DADA2Stats] to: paired-end/paired-dada-stats-180.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t18m29.574s\n",
      "user\t40m48.438s\n",
      "sys\t0m48.067s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-rep-seqs-180.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m16.352s\n",
      "user\t0m9.106s\n",
      "sys\t0m1.257s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-table-180.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m13.113s\n",
      "user\t0m6.510s\n",
      "sys\t0m1.322s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-stats-180.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m11.574s\n",
      "user\t0m5.152s\n",
      "sys\t0m1.166s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: paired-end/paired-dada-table-190.qza\n",
      "Saved FeatureData[Sequence] to: paired-end/paired-dada-rep-seqs-190.qza\n",
      "Saved SampleData[DADA2Stats] to: paired-end/paired-dada-stats-190.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t125m17.030s\n",
      "user\t42m6.044s\n",
      "sys\t0m48.548s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-rep-seqs-190.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m17.662s\n",
      "user\t0m10.055s\n",
      "sys\t0m1.514s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-table-190.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m13.474s\n",
      "user\t0m6.661s\n",
      "sys\t0m1.510s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-stats-190.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m11.434s\n",
      "user\t0m4.967s\n",
      "sys\t0m1.255s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: paired-end/paired-dada-table-200.qza\n",
      "Saved FeatureData[Sequence] to: paired-end/paired-dada-rep-seqs-200.qza\n",
      "Saved SampleData[DADA2Stats] to: paired-end/paired-dada-stats-200.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t19m51.698s\n",
      "user\t43m22.761s\n",
      "sys\t1m11.662s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-rep-seqs-200.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m18.714s\n",
      "user\t0m10.742s\n",
      "sys\t0m1.800s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-table-200.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m14.035s\n",
      "user\t0m6.976s\n",
      "sys\t0m1.739s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-stats-200.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m11.694s\n",
      "user\t0m5.108s\n",
      "sys\t0m1.351s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: paired-end/paired-dada-table-210.qza\n",
      "Saved FeatureData[Sequence] to: paired-end/paired-dada-rep-seqs-210.qza\n",
      "Saved SampleData[DADA2Stats] to: paired-end/paired-dada-stats-210.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t19m42.321s\n",
      "user\t41m59.870s\n",
      "sys\t1m11.082s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-rep-seqs-210.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m18.339s\n",
      "user\t0m10.475s\n",
      "sys\t0m1.708s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-table-210.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m13.834s\n",
      "user\t0m6.829s\n",
      "sys\t0m1.678s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-stats-210.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m11.627s\n",
      "user\t0m5.031s\n",
      "sys\t0m1.344s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: paired-end/paired-dada-table-220.qza\n",
      "Saved FeatureData[Sequence] to: paired-end/paired-dada-rep-seqs-220.qza\n",
      "Saved SampleData[DADA2Stats] to: paired-end/paired-dada-stats-220.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t17m51.453s\n",
      "user\t38m32.283s\n",
      "sys\t1m3.862s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-rep-seqs-220.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m17.206s\n",
      "user\t0m9.514s\n",
      "sys\t0m1.614s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-table-220.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m13.686s\n",
      "user\t0m6.718s\n",
      "sys\t0m1.607s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: paired-end/paired-dada-stats-220.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m11.766s\n",
      "user\t0m5.179s\n",
      "sys\t0m1.338s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir/artefacts\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs paired-end/ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 180 \\\n",
    "    --p-trunc-len-r 180 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences paired-end/paired-dada-rep-seqs-180.qza \\\n",
    "    --o-table paired-end/paired-dada-table-180.qza \\\n",
    "    --o-denoising-stats paired-end/paired-dada-stats-180.qza\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data paired-end/paired-dada-rep-seqs-180.qza \\\n",
    "    --o-visualization paired-end/paired-dada-rep-seqs-180.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table paired-end/paired-dada-table-180.qza \\\n",
    "    --o-visualization paired-end/paired-dada-table-180.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file paired-end/paired-dada-stats-180.qza \\\n",
    "    --o-visualization paired-end/paired-dada-stats-180.qzv\n",
    "    \n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs paired-end/ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 190 \\\n",
    "    --p-trunc-len-r 190 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences paired-end/paired-dada-rep-seqs-190.qza \\\n",
    "    --o-table paired-end/paired-dada-table-190.qza \\\n",
    "    --o-denoising-stats paired-end/paired-dada-stats-190.qza\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data paired-end/paired-dada-rep-seqs-190.qza \\\n",
    "    --o-visualization paired-end/paired-dada-rep-seqs-190.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table paired-end/paired-dada-table-190.qza \\\n",
    "    --o-visualization paired-end/paired-dada-table-190.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file paired-end/paired-dada-stats-190.qza \\\n",
    "    --o-visualization paired-end/paired-dada-stats-190.qzv\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs paired-end/ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 200 \\\n",
    "    --p-trunc-len-r 200 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences paired-end/paired-dada-rep-seqs-200.qza \\\n",
    "    --o-table paired-end/paired-dada-table-200.qza \\\n",
    "    --o-denoising-stats paired-end/paired-dada-stats-200.qza\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data paired-end/paired-dada-rep-seqs-200.qza \\\n",
    "    --o-visualization paired-end/paired-dada-rep-seqs-200.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table paired-end/paired-dada-table-200.qza \\\n",
    "    --o-visualization paired-end/paired-dada-table-200.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file paired-end/paired-dada-stats-200.qza \\\n",
    "    --o-visualization paired-end/paired-dada-stats-200.qzv\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs paired-end/ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 210 \\\n",
    "    --p-trunc-len-r 210 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences paired-end/paired-dada-rep-seqs-210.qza \\\n",
    "    --o-table paired-end/paired-dada-table-210.qza \\\n",
    "    --o-denoising-stats paired-end/paired-dada-stats-210.qza\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data paired-end/paired-dada-rep-seqs-210.qza \\\n",
    "    --o-visualization paired-end/paired-dada-rep-seqs-210.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table paired-end/paired-dada-table-210.qza \\\n",
    "    --o-visualization paired-end/paired-dada-table-210.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file paired-end/paired-dada-stats-210.qza \\\n",
    "    --o-visualization paired-end/paired-dada-stats-210.qzv\n",
    "\n",
    "time qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs paired-end/ITS-demux-paired-end.qza \\\n",
    "    --p-trunc-len-f 220 \\\n",
    "    --p-trunc-len-r 220 \\\n",
    "    --p-n-threads 5 \\\n",
    "    --o-representative-sequences paired-end/paired-dada-rep-seqs-220.qza \\\n",
    "    --o-table paired-end/paired-dada-table-220.qza \\\n",
    "    --o-denoising-stats paired-end/paired-dada-stats-220.qza\n",
    "\n",
    "time qiime feature-table tabulate-seqs \\\n",
    "    --i-data paired-end/paired-dada-rep-seqs-220.qza \\\n",
    "    --o-visualization paired-end/paired-dada-rep-seqs-220.qzv\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table paired-end/paired-dada-table-220.qza \\\n",
    "    --o-visualization paired-end/paired-dada-table-220.qzv\n",
    "\n",
    "time qiime metadata tabulate \\\n",
    "    --m-input-file paired-end/paired-dada-stats-220.qza \\\n",
    "    --o-visualization paired-end/paired-dada-stats-220.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 5. Filtering \"NOT-USE\" samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[811547, 811426, 811305, 811548, 811427, 811306, 811549, 811428, 811307, 811429, 811308, 811543, 811422, 811301, 811544, 811423, 811302, 811545, 811424, 811303, 811546, 811425, 811304, 811309, 811550, 811551, 811430, 811431, 811310, 811553, 811432, 811311, 811536, 811415, 811537, 811416, 811538, 811417, 811418, 811532, 811411, 811533, 811412, 811534, 811413, 811535, 811414, 811419, 811540, 811541, 811420, 811542, 811421, 811300, 811569, 811327, 811206, 811328, 811207, 811329, 811208, 811565, 811444, 811323, 811202, 811566, 811445, 811203, 811567, 811568, 811326, 811205, 811572, 811330, 811573, 811331, 811574, 811332, 811211, 811575, 811454, 811212, 811570, 811571, 811558, 811316, 811559, 811438, 811317, 811439, 811318, 811319, 811554, 811312, 811555, 811313, 811556, 811314, 811557, 811315, 811561, 811440, 811562, 811441, 811320, 811563, 811442, 811321, 811200, 811564, 811443, 811322, 811201, 811560, 811624, 811503, 811625, 811504, 811626, 811505, 811627, 811506, 811620, 811621, 811500, 811501, 811502, 811628, 811507, 811629, 811508, 811509, 811630, 811613, 811610, 811611, 811612, 811618, 811619, 811525, 811404, 811526, 811405, 811527, 811406, 811528, 811407, 811642, 811521, 811400, 811643, 811522, 811401, 811523, 811524, 811403, 811529, 811408, 811409, 811530, 811531, 811410, 811635, 811636, 811515, 811637, 811516, 811638, 811517, 811631, 811510, 811632, 811511, 811633, 811512, 811634, 811513, 811639, 811518, 811519, 811640, 811641, 811520, 811602, 811603, 811604, 811605, 811600, 811601, 811606, 811607, 811608, 811609, 811190, 811191, 811070, 811192, 811071, 811193, 811072, 811198, 811077, 811199, 811079, 811194, 811073, 811195, 811074, 811196, 811075, 811197, 811076, 811180, 811181, 811182, 811187, 811188, 811189, 811183, 811184, 811185, 811186, 811091, 811092, 811093, 811094, 811090, 811099, 811095, 811096, 811097, 811080, 811081, 811083, 811088, 811089, 811084, 811086, 811087, 811389, 811268, 811269, 811390, 811391, 811270, 811396, 811275, 811397, 811276, 811277, 811399, 811278, 811392, 811271, 811393, 811272, 811394, 811273, 811395, 811274, 811499, 811378, 811257, 811136, 811379, 811258, 811137, 811259, 811138, 811139, 811380, 811385, 811264, 811386, 811265, 811387, 811266, 811388, 811267, 811381, 811260, 811382, 811261, 811140, 811383, 811262, 811141, 811384, 811263, 811142, 811169, 811290, 811291, 811170, 811292, 811171, 811297, 811176, 811298, 811177, 811299, 811178, 811179, 811293, 811172, 811294, 811173, 811295, 811174, 811296, 811175, 811279, 811158, 811159, 811280, 811281, 811160, 811286, 811165, 811287, 811166, 811288, 811167, 811289, 811168, 811282, 811161, 811283, 811162, 811284, 811163, 811285, 811164, 811349, 811228, 811107, 811229, 811108, 811109, 811587, 811466, 811345, 811224, 811103, 811588, 811467, 811346, 811225, 811589, 811468, 811347, 811226, 811105, 811469, 811348, 811227, 811106, 811594, 811352, 811231, 811110, 811595, 811474, 811353, 811232, 811111, 811596, 811475, 811354, 811233, 811597, 811476, 811355, 811234, 811113, 811590, 811591, 811470, 811592, 811471, 811350, 811593, 811472, 811351, 811230, 811459, 811338, 811339, 811576, 811455, 811334, 811213, 811577, 811456, 811335, 811214, 811578, 811457, 811336, 811215, 811579, 811458, 811337, 811583, 811462, 811341, 811584, 811463, 811342, 811100, 811585, 811464, 811343, 811222, 811586, 811465, 811344, 811223, 811580, 811581, 811582, 811340, 811488, 811367, 811246, 811125, 811368, 811247, 811126, 811369, 811248, 811249, 811490, 811495, 811374, 811253, 811496, 811375, 811254, 811497, 811376, 811255, 811498, 811377, 811256, 811135, 811491, 811492, 811371, 811250, 811493, 811372, 811251, 811494, 811373, 811252, 811239, 811118, 811119, 811598, 811477, 811356, 811235, 811114, 811599, 811478, 811357, 811236, 811115, 811479, 811358, 811237, 811359, 811238, 811117, 811484, 811242, 811121, 811485, 811243, 811122, 811486, 811365, 811244, 811123, 811487, 811366, 811245, 811480, 811481, 811360, 811482, 811361, 811240, 811483, 811362, 811241, 811120, 810579, 810696, 810576, 810697, 810575, 810698, 810578, 810699, 810577, 810583, 810582, 810585, 810584, 810581, 810580, 810689, 810569, 810685, 810565, 810686, 810687, 810567, 810688, 810566, 810692, 810693, 810571, 810694, 810574, 810695, 810573, 810690, 810570, 810691, 810598, 810597, 810587, 810586, 810589, 810588, 810594, 810593, 810596, 810595, 810590, 810592, 810591, 810898, 810777, 810656, 810536, 810899, 810778, 810657, 810535, 810779, 810658, 810538, 810659, 810537, 810894, 810773, 810652, 810532, 810895, 810774, 810653, 810531, 810896, 810775, 810654, 810534, 810897, 810776, 810655, 810533, 810539, 810781, 810660, 810782, 810661, 810541, 810783, 810662, 810540, 810887, 810766, 810645, 810525, 810888, 810767, 810646, 810524, 810889, 810768, 810647, 810769, 810648, 810526, 810883, 810762, 810641, 810521, 810884, 810763, 810642, 810520, 810885, 810643, 810523, 810886, 810765, 810644, 810522, 810649, 810529, 810528, 810890, 810891, 810770, 810892, 810771, 810650, 810530, 810893, 810651, 810799, 810678, 810558, 810679, 810557, 810559, 810795, 810674, 810554, 810796, 810675, 810797, 810676, 810556, 810798, 810677, 810555, 810681, 810561, 810682, 810560, 810683, 810563, 810684, 810562, 810680, 810788, 810667, 810547, 810789, 810668, 810546, 810669, 810549, 810548, 810784, 810663, 810543, 810785, 810664, 810786, 810665, 810545, 810787, 810666, 810544, 810791, 810670, 810550, 810792, 810671, 810793, 810672, 810552, 810794, 810673, 810551, 810790, 810975, 810854, 810733, 810612, 810976, 810855, 810613, 810977, 810735, 810614, 810978, 810736, 810615, 810971, 810850, 810972, 810851, 810730, 810973, 810731, 810610, 810974, 810853, 810732, 810611, 810979, 810858, 810737, 810616, 810859, 810738, 810617, 810739, 810618, 810619, 810980, 810981, 810860, 810964, 810722, 810965, 810844, 810723, 810602, 810966, 810845, 810724, 810603, 810967, 810846, 810725, 810604, 810960, 810961, 810840, 810962, 810841, 810720, 810963, 810842, 810721, 810600, 810609, 810968, 810847, 810726, 810605, 810969, 810727, 810606, 810849, 810728, 810607, 810729, 810608, 810970, 810997, 810876, 810755, 810634, 810514, 810998, 810877, 810635, 810513, 810999, 810878, 810757, 810636, 810516, 810879, 810758, 810637, 810515, 810993, 810751, 810630, 810510, 810994, 810873, 810752, 810631, 810995, 810874, 810753, 810632, 810512, 810996, 810875, 810754, 810633, 810511, 810759, 810638, 810518, 810639, 810517, 810519, 810880, 810881, 810760, 810882, 810761, 810640, 810986, 810865, 810744, 810623, 810503, 810987, 810866, 810745, 810624, 810502, 810988, 810867, 810746, 810505, 810989, 810868, 810747, 810626, 810504, 810982, 810861, 810740, 810983, 810862, 810741, 810620, 810984, 810863, 810742, 810621, 810501, 810985, 810864, 810743, 810622, 810500, 810869, 810748, 810627, 810507, 810749, 810628, 810506, 810629, 810509, 810508, 810990, 810991, 810870, 810992, 810871, 810750, 810931, 810810, 810932, 810933, 810934, 810813, 810930, 810939, 810818, 810819, 810935, 810814, 810936, 810815, 810937, 810816, 810938, 810817, 810921, 810922, 810801, 810923, 810928, 810929, 810808, 810924, 810803, 810925, 810804, 810926, 810805, 810927, 810806, 810953, 810832, 810711, 810954, 810833, 810712, 810955, 810834, 810713, 810956, 810835, 810714, 810950, 810951, 810830, 810952, 810831, 810710, 810719, 810957, 810715, 810958, 810837, 810716, 810959, 810838, 810717, 810839, 810718, 810942, 810821, 810700, 810943, 810822, 810701, 810944, 810823, 810702, 810945, 810824, 810703, 810940, 810941, 810820, 810829, 810708, 810709, 810946, 810825, 810704, 810947, 810826, 810705, 810948, 810827, 810706, 810949, 810828, 810707, 810910, 810911, 810912, 810917, 810913, 810914, 810915, 810916, 810900, 810901, 810906, 810907, 810909, 810902, 810903, 810904, 810905, 811059, 811060, 811061, 811066, 811067, 811068, 811062, 811063, 811064, 811065, 811026, 811027, 811028, 811029, 811033, 811034, 811035, 811036, 811030, 811031, 811032, 811019, 811015, 811016, 811017, 811018, 811022, 811023, 811024, 811025, 811020, 811021, 811048, 811049, 811050, 811055, 811056, 811057, 811058, 811051, 811052, 811053, 811054, 811037, 811038, 811039, 811044, 811045, 811046, 811047, 811040, 811041, 811042, 811043, 810499, 810498, 810495, 810494, 810497, 810496, 810493, 811008, 811009, 811004, 811005, 811006, 811007, 811011, 811012, 811013, 811014, 811010, 811000, 811001, 811002, 811003]\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Load the CSV file into a DataFrame\n",
    "csv_file = '/Users/svens/Documents/MicroTerroir/ITS_demux_lp.csv'  # Replace with your CSV file name\n",
    "data = pd.read_csv(csv_file)\n",
    "\n",
    "# Filter rows where 'Project' is NOT 'NOT-USE'\n",
    "filtered_data = data[data['Project'] != 'NOT-USE']\n",
    "\n",
    "# Extract the Sample IDs\n",
    "sample_ids = filtered_data['Id'].tolist()  # Replace 'Sample ID' with the actual column name if different\n",
    "\n",
    "# Save to a text file or print\n",
    "with open('/Users/svens/Documents/MicroTerroir/filtered_sample_ids.txt', 'w') as output_file:\n",
    "    for sample_id in sample_ids:\n",
    "        output_file.write(str(sample_id) + '\\n')\n",
    "\n",
    "# Print the list (optional)\n",
    "print(sample_ids)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: artefacts/ITS/single-end/dada-table-190-samfilt.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m6.024s\n",
      "user\t0m4.962s\n",
      "sys\t0m0.866s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: artefacts/ITS/single-end/dada-table-190-samfilt.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m7.990s\n",
      "user\t0m6.349s\n",
      "sys\t0m1.362s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir\n",
    "\n",
    "time qiime feature-table filter-samples \\\n",
    "  --i-table artefacts/ITS/single-end/dada-table-190.qza \\\n",
    "  --m-metadata-file ITS-metadata.tsv \\\n",
    "  --p-where \"[Project] != 'NOT-USE'\" \\\n",
    "  --o-filtered-table artefacts/ITS/single-end/dada-table-190-samfilt.qza\n",
    "\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table artefacts/ITS/single-end/dada-table-190-samfilt.qza \\\n",
    "    --o-visualization artefacts/ITS/single-end/dada-table-190-samfilt.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: artefacts/16S/denoised/dada-table-samfilt.qza\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m7.070s\n",
      "user\t0m4.912s\n",
      "sys\t0m0.926s\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: artefacts/16S/denoised/dada-table-samfilt.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m7.983s\n",
      "user\t0m6.363s\n",
      "sys\t0m1.382s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir\n",
    "\n",
    "time qiime feature-table filter-samples \\\n",
    "  --i-table artefacts/16S/denoised/dada-table.qza \\\n",
    "  --m-metadata-file 16S-samfilt-metadata.tsv \\\n",
    "  --p-where \"[Project] != 'NOT-USE'\" \\\n",
    "  --o-filtered-table artefacts/16S/denoised/dada-table-samfilt.qza\n",
    "\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table artefacts/16S/denoised/dada-table-samfilt.qza \\\n",
    "    --o-visualization artefacts/16S/denoised/dada-table-samfilt.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved Visualization to: artefacts/16S/denoised/dada-table.qzv\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "real\t0m8.127s\n",
      "user\t0m6.423s\n",
      "sys\t0m1.351s\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "cd /Users/svens/Documents/MicroTerroir\n",
    "\n",
    "time qiime feature-table summarize \\\n",
    "    --i-table artefacts/16S/denoised/dada-table.qza \\\n",
    "    --o-visualization artefacts/16S/denoised/dada-table.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "qiime2-amplicon-2024.10",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.14"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
