{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Prepare the Microbiome Feature Tables for Analysis with Metabolomics data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import biom\n",
    "import os\n",
    "from qiime2 import Visualization\n",
    "import re\n",
    "\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "## function to rename columns \n",
    "\n",
    "def rename_columns_by_taxonomy(df):\n",
    "    def extract_reliable_taxon(taxon_str):\n",
    "        levels = taxon_str.split(';')\n",
    "        \n",
    "        # Remove 'sh' level if present\n",
    "        levels = [level for level in levels if not level.startswith('sh__')]\n",
    "        \n",
    "        # Iterate backwards through taxonomic levels\n",
    "        for level in reversed(levels):\n",
    "            taxon_name = re.sub(r'^[a-z]__', '', level)  # Remove prefix (e.g., 's__')\n",
    "            if taxon_name and not any(keyword in taxon_name.lower() for keyword in [\"metagenome\", \"unidentified\", \"uncultured\"]):\n",
    "                return taxon_name\n",
    "        \n",
    "        # If all levels are unreliable, return the highest available taxon\n",
    "        return re.sub(r'^[a-z]__', '', levels[0]) if levels else \"Unknown\"\n",
    "    \n",
    "    df_renamed = df.rename(columns={col: extract_reliable_taxon(col) for col in df.columns})\n",
    "    return df_renamed"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# ITS\n",
    "\n",
    "### 1. Subset to PostMLF \n",
    "\n",
    "|          \t| ASV table \t| 90% closed ref OUTs \t| OTUs collapsed on genus \t|\n",
    "|----------\t|-----------\t|---------------------\t|-------------------------\t|\n",
    "| features \t| 333       \t| 97                  \t| 68                      \t|\n",
    "\n",
    "\n",
    "> rarefy to the min. 2835 to keep all features\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!mkdir /home/lfloerl/cloud/lfloerl/Microterroir/LC-MS_data/Results/MicrobiomeMetabolome/ITS\n",
    "\n",
    "os.chdir('/home/lfloerl/cloud/lfloerl/Microterroir/LC-MS_data/Results/MicrobiomeMetabolome/ITS')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 1.1. Rarefied ASV table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Error while terminating subprocess (pid=3110671): \n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# 1. Subset to PostMLF \n",
    "qiime feature-table filter-samples \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/mv_filtered_table.qza \\\n",
    "    --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv \\\n",
    "    --p-where \"[sample_type]='Post-MLF'\" \\\n",
    "    --o-filtered-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/mv_PostMLF_filtered_table.qza\n",
    "\n",
    "qiime feature-table filter-seqs \\\n",
    "    --i-data /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/Sven-denoising-test/ITS/revcomp-trimmed-denoised/with-adjusted-parameters/ITS-revcomp-trimmed-ER-0.05-denoised-trunc-190-PFA-4.0-maxee-4.0-rep-seqs.qza \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/mv_PostMLF_filtered_table.qza \\\n",
    "    --o-filtered-data /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/mv_PostMLF_filtered_rep_seqs.qza\n",
    "\n",
    "qiime feature-table summarize --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/spatio-temporal/microvinification/mv_PostMLF_filtered_table.qza --o-visualization /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/spatio-temporal/microvinification/mv_PostMLF_filtered_table.qzv\n",
    "\n",
    "# 2. Rarefy \n",
    "qiime feature-table rarefy \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/spatio-temporal/microvinification/mv_PostMLF_filtered_table.qza \\\n",
    "    --p-sampling-depth 2835 \\\n",
    "    --o-rarefied-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/spatio-temporal/microvinification/mv_PostMLF_rarefied2835.qza\n",
    "\n",
    "# 3. Export \n",
    "qiime tools export --input-path /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/spatio-temporal/microvinification/mv_PostMLF_rarefied2835.qza --output-path mv_PostMLF_rarefied2835"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "> **333 features**\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/0a35b1d7-086a-4454-837a-386855cc50e6')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 0a35b1d7-086a-4454-837a-386855cc50e6>"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load('/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/spatio-temporal/microvinification/mv_PostMLF_filtered_table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Ascomycota</th>\n",
       "      <th>Pseudaegerita_sp</th>\n",
       "      <th>Ascomycota</th>\n",
       "      <th>Alternaria_subcucurbitae</th>\n",
       "      <th>Ramimonilia_apicalis</th>\n",
       "      <th>Ascomycota</th>\n",
       "      <th>Malassezia_restricta</th>\n",
       "      <th>Fungi_sp</th>\n",
       "      <th>Alternaria_eureka</th>\n",
       "      <th>...</th>\n",
       "      <th>Phaeococcomyces_sp</th>\n",
       "      <th>Microstroma_bacarum</th>\n",
       "      <th>Alternaria</th>\n",
       "      <th>Ascomycota</th>\n",
       "      <th>Gallowayella_poeltii</th>\n",
       "      <th>Hanseniaspora_uvarum</th>\n",
       "      <th>Cladosporium_herbarum</th>\n",
       "      <th>Nothophaeotheca_mirabibensis</th>\n",
       "      <th>Hanseniaspora_uvarum</th>\n",
       "      <th>Cladosporium_herbarum</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PostMLF_2023_Plot4</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PostMLF_2023_Plot13</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PostMLF_2023_Plot9</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PostMLF_2023_Plot5</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PostMLF_2021_Plot12</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 306 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 index  Ascomycota  Pseudaegerita_sp  Ascomycota  \\\n",
       "0   PostMLF_2023_Plot4           0                 0           0   \n",
       "1  PostMLF_2023_Plot13           0                 0           0   \n",
       "2   PostMLF_2023_Plot9           0                 0           0   \n",
       "3   PostMLF_2023_Plot5           0                 0           0   \n",
       "4  PostMLF_2021_Plot12           0                 0           0   \n",
       "\n",
       "   Alternaria_subcucurbitae  Ramimonilia_apicalis  Ascomycota  \\\n",
       "0                         0                     0           0   \n",
       "1                         0                     0           0   \n",
       "2                         0                     0           0   \n",
       "3                         0                     0           0   \n",
       "4                         0                     0           0   \n",
       "\n",
       "   Malassezia_restricta  Fungi_sp  Alternaria_eureka  ...  Phaeococcomyces_sp  \\\n",
       "0                     0         0                  0  ...                   0   \n",
       "1                     0         0                  0  ...                   0   \n",
       "2                     0         0                  0  ...                   0   \n",
       "3                     0         0                  0  ...                   0   \n",
       "4                     0         0                  0  ...                   0   \n",
       "\n",
       "   Microstroma_bacarum  Alternaria  Ascomycota  Gallowayella_poeltii  \\\n",
       "0                    0           0           0                     0   \n",
       "1                    0           0           0                     0   \n",
       "2                    0           0           0                     0   \n",
       "3                    0           0           0                     0   \n",
       "4                    0           0           0                     0   \n",
       "\n",
       "   Hanseniaspora_uvarum  Cladosporium_herbarum  Nothophaeotheca_mirabibensis  \\\n",
       "0                     0                      0                             0   \n",
       "1                     0                      0                             0   \n",
       "2                     0                      0                             0   \n",
       "3                     0                      0                             0   \n",
       "4                     0                      0                             0   \n",
       "\n",
       "   Hanseniaspora_uvarum  Cladosporium_herbarum  \n",
       "0                     0                      0  \n",
       "1                     0                      0  \n",
       "2                     0                      0  \n",
       "3                     0                      0  \n",
       "4                     0                      0  \n",
       "\n",
       "[5 rows x 306 columns]"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load the BIOM table\n",
    "biom_table = biom.load_table(\"mv_PostMLF_rarefied2835/feature-table.biom\")\n",
    "df_ITS = biom_table.to_dataframe()\n",
    "\n",
    "# load the metadata to rename the samples accordingly \n",
    "ITS_md = pd.read_csv('/home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv', sep='\\t')\n",
    "df_ITS.columns = ['PostMLF_' + str(ITS_md.set_index('id').loc[col, 'Year']) + '_Plot' + str(ITS_md.loc[ITS_md['id'] == col, 'Plot'].values[0]) for col in df_ITS.columns]\n",
    "\n",
    "# load the taxonomy to rename the features\n",
    "ITS_taxonomy = pd.read_csv('/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/taxonomy/taxonomy.tsv', sep='\\t')\n",
    "ITS_taxonomy.set_index('Feature ID', inplace=True)\n",
    "# Rename the index of df_ITS based on the 'Taxon' column of ITS_taxonomy\n",
    "df_ITS.index = df_ITS.index.map(ITS_taxonomy['Taxon'])\n",
    "\n",
    "# transpose and reset index\n",
    "df_ITS = df_ITS.T.reset_index()\n",
    "\n",
    "# rename columns \n",
    "df_ITS = rename_columns_by_taxonomy(df_ITS)\n",
    "\n",
    "df_ITS.to_csv('PostMLF_ASVs_rarefied_labled.tsv', sep='\\t')\n",
    "\n",
    "df_ITS.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 1.2. Closed Ref OTU clustering \n",
    "\n",
    "> note. UNITE release_10.05.2021"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: PostMLF_closedOTU_table.qza\n",
      "Saved FeatureData[Sequence] to: PostMLF_closedOTU_seq.qza\n",
      "Saved FeatureData[Sequence] to: PostMLF_closedOTU_unmatched_seq.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "qiime vsearch cluster-features-closed-reference \\\n",
    "    --i-sequences /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/mv_PostMLF_filtered_rep_seqs.qza \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/ITS/lavaux/mv_PostMLF_filtered_table.qza \\\n",
    "    --i-reference-sequences /home/lfloerl/public/Data/Databases/QIIME2/UNITE/sh_qiime_release_10.05.2021/unite-ver8-99-seqs-10.05.2021.qza \\\n",
    "    --p-perc-identity 0.9 \\\n",
    "    --p-threads 10 \\\n",
    "    --o-clustered-table PostMLF_closedOTU_table.qza \\\n",
    "    --o-clustered-sequences PostMLF_closedOTU_seq.qza \\\n",
    "    --o-unmatched-sequences PostMLF_closedOTU_unmatched_seq.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: PostMLF_closedOTU_table.qzv\u001b[0m\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime feature-table summarize --i-table PostMLF_closedOTU_table.qza --o-visualization PostMLF_closedOTU_table.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "> **97 features**\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/b50bd29d-01d9-4234-a7bb-057a942d1788')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: b50bd29d-01d9-4234-a7bb-057a942d1788>"
      ]
     },
     "execution_count": 60,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load('PostMLF_closedOTU_table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: PostMLF_closedOTU_rarefied_table.qza\n",
      "Exported PostMLF_closedOTU_rarefied_table.qza as BIOMV210DirFmt to directory PostMLF_closedOTU_rarefied_table\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# 2. Rarefy \n",
    "qiime feature-table rarefy \\\n",
    "    --i-table PostMLF_closedOTU_table.qza \\\n",
    "    --p-sampling-depth 2835 \\\n",
    "    --o-rarefied-table PostMLF_closedOTU_rarefied_table.qza\n",
    "\n",
    "# 3. Export \n",
    "qiime tools export --input-path PostMLF_closedOTU_rarefied_table.qza --output-path PostMLF_closedOTU_rarefied_table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported /home/lfloerl/public/Data/Databases/QIIME2/UNITE/sh_qiime_release_10.05.2021/sh_taxonomy_qiime_ver8_99_10.05.2021.qza as TSVTaxonomyDirectoryFormat to directory unite-ver8-99-taxonomy\u001b[0m\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "# export corresponding taxonomy\n",
    "!qiime tools export --input-path /home/lfloerl/public/Data/Databases/QIIME2/UNITE/sh_qiime_release_10.05.2021/sh_taxonomy_qiime_ver8_99_10.05.2021.qza --output-path unite-ver8-99-taxonomy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>364525_567-LP3-ITS-0567</th>\n",
       "      <th>364525_570-LP3-ITS-0570</th>\n",
       "      <th>364525_571-LP3-ITS-0571</th>\n",
       "      <th>364525_572-LP3-ITS-0572</th>\n",
       "      <th>364526_154-LP3-ITS-0730</th>\n",
       "      <th>364526_156-LP3-ITS-0732</th>\n",
       "      <th>364526_159-LP3-ITS-0735</th>\n",
       "      <th>364526_163-LP3-ITS-0739</th>\n",
       "      <th>364526_165-LP3-ITS-0741</th>\n",
       "      <th>364526_188-LP3-ITS-0764</th>\n",
       "      <th>...</th>\n",
       "      <th>364526_549-LP3-ITS-1125</th>\n",
       "      <th>364526_550-LP3-ITS-1126</th>\n",
       "      <th>364526_559-LP3-ITS-1135</th>\n",
       "      <th>364526_562-LP3-ITS-1138</th>\n",
       "      <th>364526_565-LP3-ITS-1141</th>\n",
       "      <th>364526_566-LP3-ITS-1142</th>\n",
       "      <th>364526_568-LP3-ITS-1144</th>\n",
       "      <th>364526_570-LP3-ITS-1146</th>\n",
       "      <th>364526_572-LP3-ITS-1148</th>\n",
       "      <th>364526_574-LP3-ITS-1150</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>SH1846583.08FU_UDB0119152_reps</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SH1685246.08FU_UDB0736585_reps</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SH1744677.08FU_MN509277_reps</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SH1688430.08FU_KC584249_refs</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>SH1729342.08FU_AY843135_refs</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                364525_567-LP3-ITS-0567  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364525_570-LP3-ITS-0570  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364525_571-LP3-ITS-0571  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364525_572-LP3-ITS-0572  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_154-LP3-ITS-0730  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_156-LP3-ITS-0732  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_159-LP3-ITS-0735  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_163-LP3-ITS-0739  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_165-LP3-ITS-0741  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_188-LP3-ITS-0764  ...  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0  ...   \n",
       "SH1685246.08FU_UDB0736585_reps                        0  ...   \n",
       "SH1744677.08FU_MN509277_reps                          0  ...   \n",
       "SH1688430.08FU_KC584249_refs                          0  ...   \n",
       "SH1729342.08FU_AY843135_refs                          0  ...   \n",
       "\n",
       "                                364526_549-LP3-ITS-1125  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_550-LP3-ITS-1126  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_559-LP3-ITS-1135  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_562-LP3-ITS-1138  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_565-LP3-ITS-1141  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_566-LP3-ITS-1142  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_568-LP3-ITS-1144  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_570-LP3-ITS-1146  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_572-LP3-ITS-1148  \\\n",
       "SH1846583.08FU_UDB0119152_reps                        0   \n",
       "SH1685246.08FU_UDB0736585_reps                        0   \n",
       "SH1744677.08FU_MN509277_reps                          0   \n",
       "SH1688430.08FU_KC584249_refs                          0   \n",
       "SH1729342.08FU_AY843135_refs                          0   \n",
       "\n",
       "                                364526_574-LP3-ITS-1150  \n",
       "SH1846583.08FU_UDB0119152_reps                        0  \n",
       "SH1685246.08FU_UDB0736585_reps                        0  \n",
       "SH1744677.08FU_MN509277_reps                          0  \n",
       "SH1688430.08FU_KC584249_refs                          0  \n",
       "SH1729342.08FU_AY843135_refs                          0  \n",
       "\n",
       "[5 rows x 32 columns]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load the BIOM table\n",
    "biom_table = biom.load_table(\"PostMLF_closedOTU_rarefied_table/feature-table.biom\")\n",
    "df_ITS = biom_table.to_dataframe()\n",
    "\n",
    "df_ITS.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Dothideomycetes</th>\n",
       "      <th>Hyaloscypha</th>\n",
       "      <th>Cladosporium_austrohemisphaericum</th>\n",
       "      <th>Alternaria_subcucurbitae</th>\n",
       "      <th>Ramimonilia_apicalis</th>\n",
       "      <th>Malassezia_restricta</th>\n",
       "      <th>Ascomycota</th>\n",
       "      <th>Alternaria_eureka</th>\n",
       "      <th>Rhodotorula_graminis</th>\n",
       "      <th>...</th>\n",
       "      <th>Paracamarosporium_hawaiiense</th>\n",
       "      <th>Pleosporaceae</th>\n",
       "      <th>Melampsora_epitea</th>\n",
       "      <th>Hormonema_macrosporum</th>\n",
       "      <th>Claviceps_purpurea</th>\n",
       "      <th>Chaetothyriales</th>\n",
       "      <th>Endoconidioma_populi</th>\n",
       "      <th>Phaeococcomyces</th>\n",
       "      <th>Acremonium</th>\n",
       "      <th>Gallowayella_poeltii</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PostMLF_2023_Plot4</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PostMLF_2023_Plot13</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PostMLF_2023_Plot9</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PostMLF_2023_Plot5</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PostMLF_2021_Plot12</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 89 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 index  Dothideomycetes  Hyaloscypha  \\\n",
       "0   PostMLF_2023_Plot4                0            0   \n",
       "1  PostMLF_2023_Plot13                0            0   \n",
       "2   PostMLF_2023_Plot9                0            0   \n",
       "3   PostMLF_2023_Plot5                0            0   \n",
       "4  PostMLF_2021_Plot12                0            0   \n",
       "\n",
       "   Cladosporium_austrohemisphaericum  Alternaria_subcucurbitae  \\\n",
       "0                                  0                         0   \n",
       "1                                  0                         0   \n",
       "2                                  0                         0   \n",
       "3                                  0                         0   \n",
       "4                                  0                         0   \n",
       "\n",
       "   Ramimonilia_apicalis  Malassezia_restricta  Ascomycota  Alternaria_eureka  \\\n",
       "0                     0                     0           0                  0   \n",
       "1                     0                     0           0                  0   \n",
       "2                     0                     0           0                  0   \n",
       "3                     0                     0           0                  0   \n",
       "4                     0                     0           0                  0   \n",
       "\n",
       "   Rhodotorula_graminis  ...  Paracamarosporium_hawaiiense  Pleosporaceae  \\\n",
       "0                     0  ...                             0              0   \n",
       "1                     0  ...                             0              0   \n",
       "2                     0  ...                             0              0   \n",
       "3                     0  ...                             0              0   \n",
       "4                     0  ...                             0              0   \n",
       "\n",
       "   Melampsora_epitea  Hormonema_macrosporum  Claviceps_purpurea  \\\n",
       "0                  0                      0                   0   \n",
       "1                  0                      0                   0   \n",
       "2                  0                      0                   0   \n",
       "3                  0                      0                   0   \n",
       "4                  0                      0                   0   \n",
       "\n",
       "   Chaetothyriales  Endoconidioma_populi  Phaeococcomyces  Acremonium  \\\n",
       "0                0                     0                0           0   \n",
       "1                0                     0                0           0   \n",
       "2                0                     0                0           0   \n",
       "3                0                     0                0           0   \n",
       "4                0                     0                0           0   \n",
       "\n",
       "   Gallowayella_poeltii  \n",
       "0                     0  \n",
       "1                     0  \n",
       "2                     0  \n",
       "3                     0  \n",
       "4                     0  \n",
       "\n",
       "[5 rows x 89 columns]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load the BIOM table\n",
    "biom_table = biom.load_table(\"PostMLF_closedOTU_rarefied_table/feature-table.biom\")\n",
    "df_ITS = biom_table.to_dataframe()\n",
    "\n",
    "# load the metadata to rename the samples accordingly \n",
    "ITS_md = pd.read_csv('/home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv', sep='\\t')\n",
    "df_ITS.columns = ['PostMLF_' + str(ITS_md.set_index('id').loc[col, 'Year']) + '_Plot' + str(ITS_md.loc[ITS_md['id'] == col, 'Plot'].values[0]) for col in df_ITS.columns]\n",
    "\n",
    "# load the taxonomy to rename the features\n",
    "ITS_taxonomy = pd.read_csv('unite-ver8-99-taxonomy/taxonomy.tsv', sep='\\t')\n",
    "ITS_taxonomy.set_index('Feature ID', inplace=True)\n",
    "# Rename the index of df_ITS based on the 'Taxon' column of ITS_taxonomy\n",
    "df_ITS.index = df_ITS.index.map(ITS_taxonomy['Taxon'])\n",
    "\n",
    "# transpose and reset index\n",
    "df_ITS = df_ITS.T.reset_index()\n",
    "\n",
    "# rename columns \n",
    "df_ITS = rename_columns_by_taxonomy(df_ITS)\n",
    "\n",
    "df_ITS.to_csv('PostMLF_cOTUs_rarefied_labled.tsv', sep='\\t')\n",
    "\n",
    "df_ITS.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 1.3. Collapse on Genus Level"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: PostMLF_closedOTU_rarefied_genus_table.qza\n",
      "Saved Visualization to: PostMLF_closedOTU_rarefied_genus_table.qzv\n",
      "Exported PostMLF_closedOTU_rarefied_genus_table.qza as BIOMV210DirFmt to directory PostMLF_closedOTU_rarefied_genus_table\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "qiime taxa collapse \\\n",
    "    --i-table PostMLF_closedOTU_rarefied_table.qza \\\n",
    "    --i-taxonomy /home/lfloerl/public/Data/Databases/QIIME2/UNITE/sh_qiime_release_10.05.2021/sh_taxonomy_qiime_ver8_99_10.05.2021.qza \\\n",
    "    --p-level 6 \\\n",
    "    --o-collapsed-table PostMLF_closedOTU_rarefied_genus_table.qza\n",
    "    \n",
    "qiime feature-table summarize --i-table PostMLF_closedOTU_rarefied_genus_table.qza --o-visualization PostMLF_closedOTU_rarefied_genus_table.qzv\n",
    "\n",
    "qiime tools export --input-path PostMLF_closedOTU_rarefied_genus_table.qza --output-path PostMLF_closedOTU_rarefied_genus_table"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "> **68 features** "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/612af571-1b18-4a67-bf87-f7b7cefcfa85')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 612af571-1b18-4a67-bf87-f7b7cefcfa85>"
      ]
     },
     "execution_count": 63,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load('PostMLF_closedOTU_rarefied_genus_table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Dothideomycetes</th>\n",
       "      <th>Hyaloscypha</th>\n",
       "      <th>Cladosporium</th>\n",
       "      <th>Alternaria</th>\n",
       "      <th>Ramimonilia</th>\n",
       "      <th>Malassezia</th>\n",
       "      <th>Ascomycota</th>\n",
       "      <th>Rhodotorula</th>\n",
       "      <th>Hanseniaspora</th>\n",
       "      <th>...</th>\n",
       "      <th>Dothiorella</th>\n",
       "      <th>Paracamarosporium</th>\n",
       "      <th>Pleosporaceae</th>\n",
       "      <th>Hormonema</th>\n",
       "      <th>Claviceps</th>\n",
       "      <th>Chaetothyriales</th>\n",
       "      <th>Endoconidioma</th>\n",
       "      <th>Phaeococcomyces</th>\n",
       "      <th>Acremonium</th>\n",
       "      <th>Gallowayella</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PostMLF_2023_Plot4</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2349.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PostMLF_2023_Plot13</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>421.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PostMLF_2023_Plot9</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2641.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PostMLF_2023_Plot5</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2722.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PostMLF_2021_Plot12</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 69 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 index  Dothideomycetes  Hyaloscypha  Cladosporium  \\\n",
       "0   PostMLF_2023_Plot4                0            0             0   \n",
       "1  PostMLF_2023_Plot13                0            0             0   \n",
       "2   PostMLF_2023_Plot9                0            0             0   \n",
       "3   PostMLF_2023_Plot5                0            0             0   \n",
       "4  PostMLF_2021_Plot12                0            0             0   \n",
       "\n",
       "   Alternaria  Ramimonilia  Malassezia  Ascomycota  Rhodotorula  \\\n",
       "0           0            0           0           0            0   \n",
       "1           0            0           0           0            0   \n",
       "2           0            0           0           0            0   \n",
       "3           0            0           0           0            0   \n",
       "4           0            0           0           0            0   \n",
       "\n",
       "   Hanseniaspora  ...  Dothiorella  Paracamarosporium  Pleosporaceae  \\\n",
       "0         2349.0  ...            0                  0              0   \n",
       "1          421.0  ...            0                  0              0   \n",
       "2         2641.0  ...            0                  0              0   \n",
       "3         2722.0  ...            0                  0              0   \n",
       "4              0  ...            0                  0              0   \n",
       "\n",
       "   Hormonema  Claviceps  Chaetothyriales  Endoconidioma  Phaeococcomyces  \\\n",
       "0          0          0                0              0                0   \n",
       "1          0          0                0              0                0   \n",
       "2          0          0                0              0                0   \n",
       "3          0          0                0              0                0   \n",
       "4          0          0                0              0                0   \n",
       "\n",
       "   Acremonium  Gallowayella  \n",
       "0           0             0  \n",
       "1           0             0  \n",
       "2           0             0  \n",
       "3           0             0  \n",
       "4           0             0  \n",
       "\n",
       "[5 rows x 69 columns]"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load the BIOM table\n",
    "biom_table = biom.load_table(\"PostMLF_closedOTU_rarefied_genus_table/feature-table.biom\")\n",
    "df_ITS = biom_table.to_dataframe()\n",
    "\n",
    "# load the metadata to rename the samples accordingly \n",
    "ITS_md = pd.read_csv('/home/lfloerl/microterroir/Microbiome/Metadata/ITS_lavaux.tsv', sep='\\t')\n",
    "df_ITS.columns = ['PostMLF_' + str(ITS_md.set_index('id').loc[col, 'Year']) + '_Plot' + str(ITS_md.loc[ITS_md['id'] == col, 'Plot'].values[0]) for col in df_ITS.columns]\n",
    "\n",
    "# transpose and reset index\n",
    "df_ITS = df_ITS.T.reset_index()\n",
    "\n",
    "# rename columns \n",
    "df_ITS = rename_columns_by_taxonomy(df_ITS)\n",
    "\n",
    "df_ITS.to_csv('PostMLF_cOTUs_genus_rarefied_labled.tsv', sep='\\t')\n",
    "\n",
    "df_ITS.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<hr>\n",
    "\n",
    "# 16S\n",
    "\n",
    "|          \t| ASV table \t| 99% closed ref OUTs \t| OTUs collapsed on genus \t|\n",
    "|----------\t|-----------\t|---------------------\t|-------------------------\t|\n",
    "| features \t| 326       \t| 203                 \t| 56                      \t|\n",
    "\n",
    "### 1. Subset to PostMLF \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!mkdir /home/lfloerl/cloud/lfloerl/Microterroir/LC-MS_data/Results/MicrobiomeMetabolome/16S\n",
    "\n",
    "os.chdir('/home/lfloerl/cloud/lfloerl/Microterroir/LC-MS_data/Results/MicrobiomeMetabolome/16S')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 1.1. Rarefied ASV table\n",
    "\n",
    "> rarefy to 100 to keep as many as possible\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qza\n",
      "Saved FeatureData[Sequence] to: /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_rep_seqs.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# 1. Subset to PostMLF \n",
    "qiime feature-table filter-samples \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_filtered_table.qza \\\n",
    "    --m-metadata-file /home/lfloerl/microterroir/Microbiome/Metadata/16S_lavaux.tsv \\\n",
    "    --p-where \"[sample_type]='Post-MLF'\" \\\n",
    "    --o-filtered-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qza\n",
    "\n",
    "qiime feature-table filter-seqs \\\n",
    "    --i-data /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/bac-dada2-single/dada-rep-seqs-220-ee4-fa4.qza \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qza \\\n",
    "    --o-filtered-data /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_rep_seqs.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime feature-table summarize --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qza --o-visualization /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "> **326 features**\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/fe38fe97-645e-41c3-838a-a2fbacc68efc')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: fe38fe97-645e-41c3-838a-a2fbacc68efc>"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load('/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_rarefied.qza\n",
      "Exported /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_rarefied.qza as BIOMV210DirFmt to directory mv_PostMLF_rarefied\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# 2. Rarefy \n",
    "qiime feature-table rarefy \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qza \\\n",
    "    --p-sampling-depth 100 \\\n",
    "    --o-rarefied-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_rarefied.qza\n",
    "\n",
    "# 3. Export \n",
    "qiime tools export --input-path /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_rarefied.qza --output-path mv_PostMLF_rarefied"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/taxonomy.qza as TSVTaxonomyDirectoryFormat to directory /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/taxonomy\u001b[0m\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "#!qiime tools export --input-path /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/taxonomy.qza --output-path /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/taxonomy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Pirellulaceae</th>\n",
       "      <th>Escherichia_sp.</th>\n",
       "      <th>Caldilineaceae</th>\n",
       "      <th>Tepidisphaeraceae</th>\n",
       "      <th>Solirubrobacter</th>\n",
       "      <th>Acetobacter_cerevisiae</th>\n",
       "      <th>Anaerolineaceae</th>\n",
       "      <th>Actinomycetospora</th>\n",
       "      <th>Gemmataceae</th>\n",
       "      <th>...</th>\n",
       "      <th>Rhodobacteraceae</th>\n",
       "      <th>RB41</th>\n",
       "      <th>TK10</th>\n",
       "      <th>Vicinamibacteraceae</th>\n",
       "      <th>A4b</th>\n",
       "      <th>Vicinamibacteraceae</th>\n",
       "      <th>Enterobacteriaceae</th>\n",
       "      <th>Vicinamibacteraceae</th>\n",
       "      <th>Vicinamibacterales</th>\n",
       "      <th>JG30-KF-CM45</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PostMLF_2023_Plot4</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PostMLF_2023_Plot13</td>\n",
       "      <td>0</td>\n",
       "      <td>5.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PostMLF_2023_Plot9</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PostMLF_2023_Plot5</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PostMLF_2021_Plot17</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 150 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 index  Pirellulaceae  Escherichia_sp.  Caldilineaceae  \\\n",
       "0   PostMLF_2023_Plot4              0                0               0   \n",
       "1  PostMLF_2023_Plot13              0              5.0               0   \n",
       "2   PostMLF_2023_Plot9              0                0               0   \n",
       "3   PostMLF_2023_Plot5              0                0               0   \n",
       "4  PostMLF_2021_Plot17            1.0                0               0   \n",
       "\n",
       "   Tepidisphaeraceae  Solirubrobacter  Acetobacter_cerevisiae  \\\n",
       "0                  0                0                       0   \n",
       "1                  0                0                       0   \n",
       "2                  0                0                       0   \n",
       "3                  0                0                       0   \n",
       "4                  0              1.0                       0   \n",
       "\n",
       "   Anaerolineaceae  Actinomycetospora  Gemmataceae  ...  Rhodobacteraceae  \\\n",
       "0                0                  0            0  ...                 0   \n",
       "1                0                  0            0  ...                 0   \n",
       "2                0                  0            0  ...                 0   \n",
       "3                0                  0            0  ...                 0   \n",
       "4                0                  0            0  ...                 0   \n",
       "\n",
       "   RB41  TK10  Vicinamibacteraceae  A4b  Vicinamibacteraceae  \\\n",
       "0     0     0                    0    0                    0   \n",
       "1     0     0                    0    0                    0   \n",
       "2     0     0                    0    0                    0   \n",
       "3     0     0                    0    0                    0   \n",
       "4     0     0                    0    0                    0   \n",
       "\n",
       "   Enterobacteriaceae  Vicinamibacteraceae  Vicinamibacterales  JG30-KF-CM45  \n",
       "0                   0                    0                   0             0  \n",
       "1                 1.0                    0                   0             0  \n",
       "2                   0                    0                   0             0  \n",
       "3                   0                    0                   0             0  \n",
       "4                   0                    0                   0             0  \n",
       "\n",
       "[5 rows x 150 columns]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load the BIOM table\n",
    "biom_table = biom.load_table(\"mv_PostMLF_rarefied/feature-table.biom\")\n",
    "df_16S = biom_table.to_dataframe()\n",
    "\n",
    "\n",
    "# load the metadata to rename the samples accordingly \n",
    "Bac_md = pd.read_csv('/home/lfloerl/microterroir/Microbiome/Metadata/16S_lavaux.tsv', sep='\\t')\n",
    "df_16S.columns = ['PostMLF_' + str(Bac_md.set_index('id').loc[col, 'Year']) + '_Plot' + str(Bac_md.loc[Bac_md['id'] == col, 'Plot'].values[0]) for col in df_16S.columns]\n",
    "\n",
    "# load the taxonomy to rename the features\n",
    "Bac_taxonomy = pd.read_csv('/home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/taxonomy/taxonomy.tsv', sep='\\t')\n",
    "Bac_taxonomy.set_index('Feature ID', inplace=True)\n",
    "# Rename the index of df_ITS based on the 'Taxon' column of ITS_taxonomy\n",
    "df_16S.index = df_16S.index.map(Bac_taxonomy['Taxon'])\n",
    "\n",
    "\n",
    "# transpose and reset index\n",
    "df_16S = df_16S.T.reset_index()\n",
    "\n",
    "# rename columns \n",
    "df_16S = rename_columns_by_taxonomy(df_16S)\n",
    "\n",
    "df_16S.to_csv('PostMLF_ASVs_rarefied_labled.tsv', sep='\\t')\n",
    "\n",
    "df_16S.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 1.2. Closed Ref OTU clustering "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: PostMLF_closedOTU_table.qza\n",
      "Saved FeatureData[Sequence] to: PostMLF_closedOTU_seq.qza\n",
      "Saved FeatureData[Sequence] to: PostMLF_closedOTU_unmatched_seq.qza\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "qiime vsearch cluster-features-closed-reference \\\n",
    "    --i-sequences /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_rep_seqs.qza \\\n",
    "    --i-table /home/lfloerl/cloud/lfloerl/Microterroir/artifacts/16S/lavaux/mv_PostMLF_filtered_table.qza \\\n",
    "    --i-reference-sequences /home/lfloerl/public/Data/Databases/QIIME2/SILVA/silva-138-99-seqs.qza \\\n",
    "    --p-perc-identity 0.99 \\\n",
    "    --p-threads 10 \\\n",
    "    --o-clustered-table PostMLF_closedOTU_table.qza \\\n",
    "    --o-clustered-sequences PostMLF_closedOTU_seq.qza \\\n",
    "    --o-unmatched-sequences PostMLF_closedOTU_unmatched_seq.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: PostMLF_closedOTU_table.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime feature-table summarize --i-table PostMLF_closedOTU_table.qza --o-visualization PostMLF_closedOTU_table.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "> **203 features**\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/d4741a0e-ae6e-4b60-956e-58a30af5d7cb')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: d4741a0e-ae6e-4b60-956e-58a30af5d7cb>"
      ]
     },
     "execution_count": 71,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load('PostMLF_closedOTU_table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: PostMLF_closedOTU_rarefied_table.qza\n",
      "Exported PostMLF_closedOTU_rarefied_table.qza as BIOMV210DirFmt to directory PostMLF_closedOTU_rarefied_table\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# 2. Rarefy \n",
    "qiime feature-table rarefy \\\n",
    "    --i-table PostMLF_closedOTU_table.qza \\\n",
    "    --p-sampling-depth 100 \\\n",
    "    --o-rarefied-table PostMLF_closedOTU_rarefied_table.qza\n",
    "\n",
    "# 3. Export \n",
    "qiime tools export --input-path PostMLF_closedOTU_rarefied_table.qza --output-path PostMLF_closedOTU_rarefied_table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported /home/lfloerl/public/Data/Databases/QIIME2/SILVA/silva-138-99-tax.qza as TSVTaxonomyDirectoryFormat to directory silva-138-99-tax\u001b[0m\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "# export corresponding taxonomy\n",
    "!qiime tools export --input-path /home/lfloerl/public/Data/Databases/QIIME2/SILVA/silva-138-99-tax.qza --output-path silva-138-99-tax"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>s__Escherichia_coli</th>\n",
       "      <th>s__bacterium_Ellin6048</th>\n",
       "      <th>s__Acetobacter_cerevisiae</th>\n",
       "      <th>g__Actinomycetospora</th>\n",
       "      <th>s__Oenococcus_oeni</th>\n",
       "      <th>g__Pirellula</th>\n",
       "      <th>g__Rokubacteriales</th>\n",
       "      <th>g__Acetobacter</th>\n",
       "      <th>f__Vicinamibacteraceae</th>\n",
       "      <th>...</th>\n",
       "      <th>s__Mycobacterium_sp.</th>\n",
       "      <th>f__Pirellulaceae</th>\n",
       "      <th>s__Ruminiclostridium_hungatei</th>\n",
       "      <th>g__TK10</th>\n",
       "      <th>g__Vicinamibacteraceae</th>\n",
       "      <th>g__A4b</th>\n",
       "      <th>o__Gaiellales</th>\n",
       "      <th>g__Pirellula</th>\n",
       "      <th>s__Salmonella_enterica</th>\n",
       "      <th>g__Vicinamibacteraceae</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PostMLF_2023_Plot4</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>84.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PostMLF_2023_Plot13</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>69.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PostMLF_2023_Plot9</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>97.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PostMLF_2023_Plot5</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>100.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PostMLF_2021_Plot17</td>\n",
       "      <td>0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 93 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 index   s__Escherichia_coli   s__bacterium_Ellin6048  \\\n",
       "0   PostMLF_2023_Plot4                     0                        0   \n",
       "1  PostMLF_2023_Plot13                   4.0                        0   \n",
       "2   PostMLF_2023_Plot9                     0                        0   \n",
       "3   PostMLF_2023_Plot5                     0                        0   \n",
       "4  PostMLF_2021_Plot17                     0                      2.0   \n",
       "\n",
       "    s__Acetobacter_cerevisiae   g__Actinomycetospora   s__Oenococcus_oeni  \\\n",
       "0                           0                      0                 84.0   \n",
       "1                           0                      0                 69.0   \n",
       "2                           0                      0                 97.0   \n",
       "3                           0                      0                100.0   \n",
       "4                           0                      0                  1.0   \n",
       "\n",
       "    g__Pirellula   g__Rokubacteriales   g__Acetobacter  \\\n",
       "0              0                    0                0   \n",
       "1              0                    0                0   \n",
       "2              0                    0                0   \n",
       "3              0                    0                0   \n",
       "4              0                    0                0   \n",
       "\n",
       "    f__Vicinamibacteraceae  ...   s__Mycobacterium_sp.   f__Pirellulaceae  \\\n",
       "0                        0  ...                      0                  0   \n",
       "1                        0  ...                      0                  0   \n",
       "2                        0  ...                      0                  0   \n",
       "3                        0  ...                      0                  0   \n",
       "4                        0  ...                    1.0                  0   \n",
       "\n",
       "    s__Ruminiclostridium_hungatei   g__TK10   g__Vicinamibacteraceae   g__A4b  \\\n",
       "0                               0         0                        0        0   \n",
       "1                               0         0                        0        0   \n",
       "2                               0         0                        0        0   \n",
       "3                               0         0                        0        0   \n",
       "4                               0         0                        0        0   \n",
       "\n",
       "    o__Gaiellales   g__Pirellula   s__Salmonella_enterica  \\\n",
       "0               0              0                        0   \n",
       "1             1.0              0                      2.0   \n",
       "2               0              0                        0   \n",
       "3               0              0                        0   \n",
       "4               0              0                        0   \n",
       "\n",
       "    g__Vicinamibacteraceae  \n",
       "0                        0  \n",
       "1                        0  \n",
       "2                        0  \n",
       "3                        0  \n",
       "4                        0  \n",
       "\n",
       "[5 rows x 93 columns]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load the BIOM table\n",
    "biom_table = biom.load_table(\"PostMLF_closedOTU_rarefied_table/feature-table.biom\")\n",
    "df_16S = biom_table.to_dataframe()\n",
    "\n",
    "\n",
    "# load the metadata to rename the samples accordingly \n",
    "Bac_md = pd.read_csv('/home/lfloerl/microterroir/Microbiome/Metadata/16S_lavaux.tsv', sep='\\t')\n",
    "df_16S.columns = ['PostMLF_' + str(Bac_md.set_index('id').loc[col, 'Year']) + '_Plot' + str(Bac_md.loc[Bac_md['id'] == col, 'Plot'].values[0]) for col in df_16S.columns]\n",
    "\n",
    "# load the taxonomy to rename the features\n",
    "Bac_taxonomy = pd.read_csv('silva-138-99-tax/taxonomy.tsv', sep='\\t')\n",
    "Bac_taxonomy.set_index('Feature ID', inplace=True)\n",
    "# Rename the index of df_ITS based on the 'Taxon' column of ITS_taxonomy\n",
    "df_16S.index = df_16S.index.map(Bac_taxonomy['Taxon'])\n",
    "\n",
    "# transpose and reset index\n",
    "df_16S = df_16S.T.reset_index()\n",
    "\n",
    "# rename columns \n",
    "df_16S = rename_columns_by_taxonomy(df_16S)\n",
    "\n",
    "df_16S.to_csv('PostMLF_cOTUs_rarefied_labled.tsv', sep='\\t')\n",
    "\n",
    "df_16S.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 1.3. Collapse on Genus Level"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: PostMLF_closedOTU_rarefied_genus_table.qza\n",
      "Saved Visualization to: PostMLF_closedOTU_rarefied_genus_table.qzv\n",
      "Exported PostMLF_closedOTU_rarefied_genus_table.qza as BIOMV210DirFmt to directory PostMLF_closedOTU_rarefied_genus_table\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "qiime taxa collapse \\\n",
    "    --i-table PostMLF_closedOTU_rarefied_table.qza \\\n",
    "    --i-taxonomy /home/lfloerl/public/Data/Databases/QIIME2/SILVA/silva-138-99-tax.qza \\\n",
    "    --p-level 6 \\\n",
    "    --o-collapsed-table PostMLF_closedOTU_rarefied_genus_table.qza\n",
    "    \n",
    "qiime feature-table summarize --i-table PostMLF_closedOTU_rarefied_genus_table.qza --o-visualization PostMLF_closedOTU_rarefied_genus_table.qzv\n",
    "\n",
    "qiime tools export --input-path PostMLF_closedOTU_rarefied_genus_table.qza --output-path PostMLF_closedOTU_rarefied_genus_table"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "> **56 features** "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter server extension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/scratch/lfloerl/tmpdata/qiime2/lfloerl/data/431001cc-5cdf-4243-aa34-f1e0d135f926')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 431001cc-5cdf-4243-aa34-f1e0d135f926>"
      ]
     },
     "execution_count": 75,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load('PostMLF_closedOTU_rarefied_genus_table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>index</th>\n",
       "      <th>Escherichia-Shigella</th>\n",
       "      <th>Solirubrobacter</th>\n",
       "      <th>Acetobacter</th>\n",
       "      <th>Actinomycetospora</th>\n",
       "      <th>Oenococcus</th>\n",
       "      <th>Pirellula</th>\n",
       "      <th>Rokubacteriales</th>\n",
       "      <th>Vicinamibacteraceae</th>\n",
       "      <th>Pedosphaeraceae</th>\n",
       "      <th>...</th>\n",
       "      <th>Achromobacter</th>\n",
       "      <th>Chthoniobacter</th>\n",
       "      <th>Actinoplanes</th>\n",
       "      <th>JGI_0001001-H03</th>\n",
       "      <th>Gluconobacter</th>\n",
       "      <th>Lactobacillus</th>\n",
       "      <th>Mycobacterium</th>\n",
       "      <th>Ruminiclostridium</th>\n",
       "      <th>A4b</th>\n",
       "      <th>Salmonella</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PostMLF_2023_Plot4</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>13.0</td>\n",
       "      <td>0</td>\n",
       "      <td>84.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PostMLF_2023_Plot13</td>\n",
       "      <td>4.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>69.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PostMLF_2023_Plot9</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0</td>\n",
       "      <td>97.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PostMLF_2023_Plot5</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>100.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PostMLF_2021_Plot17</td>\n",
       "      <td>0</td>\n",
       "      <td>2.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 57 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 index  Escherichia-Shigella  Solirubrobacter  Acetobacter  \\\n",
       "0   PostMLF_2023_Plot4                     0                0         13.0   \n",
       "1  PostMLF_2023_Plot13                   4.0                0            0   \n",
       "2   PostMLF_2023_Plot9                     0                0          3.0   \n",
       "3   PostMLF_2023_Plot5                     0                0            0   \n",
       "4  PostMLF_2021_Plot17                     0              2.0            0   \n",
       "\n",
       "   Actinomycetospora  Oenococcus  Pirellula  Rokubacteriales  \\\n",
       "0                  0        84.0          0                0   \n",
       "1                  0        69.0          0                0   \n",
       "2                  0        97.0          0                0   \n",
       "3                  0       100.0          0                0   \n",
       "4                  0         1.0          0              1.0   \n",
       "\n",
       "   Vicinamibacteraceae  Pedosphaeraceae  ...  Achromobacter  Chthoniobacter  \\\n",
       "0                    0                0  ...            1.0               0   \n",
       "1                    0                0  ...              0               0   \n",
       "2                    0                0  ...              0               0   \n",
       "3                    0                0  ...              0               0   \n",
       "4                    0                0  ...              0               0   \n",
       "\n",
       "   Actinoplanes  JGI_0001001-H03  Gluconobacter  Lactobacillus  Mycobacterium  \\\n",
       "0             0                0              0              0              0   \n",
       "1             0                0              0            2.0              0   \n",
       "2             0                0              0              0              0   \n",
       "3             0                0              0              0              0   \n",
       "4             0                0              0              0            1.0   \n",
       "\n",
       "   Ruminiclostridium  A4b  Salmonella  \n",
       "0                  0    0           0  \n",
       "1                  0    0         2.0  \n",
       "2                  0    0           0  \n",
       "3                  0    0           0  \n",
       "4                  0    0           0  \n",
       "\n",
       "[5 rows x 57 columns]"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load the BIOM table\n",
    "biom_table = biom.load_table(\"PostMLF_closedOTU_rarefied_genus_table/feature-table.biom\")\n",
    "df_16S = biom_table.to_dataframe()\n",
    "\n",
    "# load the metadata to rename the samples accordingly \n",
    "Bac_md = pd.read_csv('/home/lfloerl/microterroir/Microbiome/Metadata/16S_lavaux.tsv', sep='\\t')\n",
    "df_16S.columns = ['PostMLF_' + str(Bac_md.set_index('id').loc[col, 'Year']) + '_Plot' + str(Bac_md.loc[Bac_md['id'] == col, 'Plot'].values[0]) for col in df_16S.columns]\n",
    "\n",
    "# transpose and reset index\n",
    "df_16S = df_16S.T.reset_index()\n",
    "\n",
    "# rename columns \n",
    "df_16S = rename_columns_by_taxonomy(df_16S)\n",
    "\n",
    "df_16S.to_csv('PostMLF_cOTUs_genus_rarefied_labled.tsv', sep='\\t')\n",
    "\n",
    "df_16S.head()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "qiime2-amplicon-2024.10",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.14"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
